{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 74,
   "metadata": {},
   "outputs": [],
   "source": [
    "#Dependencies\n",
    "from sklearn.model_selection import train_test_split\n",
    "from pyspark.sql import SparkSession\n",
    "from pyspark.sql.functions import col, udf\n",
    "from pyspark.sql.types import IntegerType\n",
    "from pyspark import SparkContext, since\n",
    "from pyspark.ml.feature import CountVectorizer, StopWordsRemover, HashingTF, IDF, Tokenizer, StringIndexer"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 75,
   "metadata": {},
   "outputs": [],
   "source": [
    "spark = SparkSession.builder.appName('nlp').getOrCreate()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 78,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "+------+-------------------+-------------------+---------+----------+-----------+--------------------+----------+------------+-----------------+--------------------+--------------------+------------------+----------------+----------------------+--------------------+-------------------+--------------------+-------------------+--------------------+--------------------+----------+\n",
      "|Agency|    Budget End Date|  Budget Start Date|CFDA code|Department|Fiscal Year|           Key Words|  Latitude|   Longitude|Organization City|Organization Country|   Organization Name|Organization State|Organization ZIP|Principal Investigator|    Project Abstract|   Project End Date|      Project Number| Project Start Date|       Project Title|           Seed Term|Total Cost|\n",
      "+------+-------------------+-------------------+---------+----------+-----------+--------------------+----------+------------+-----------------+--------------------+--------------------+------------------+----------------+----------------------+--------------------+-------------------+--------------------+-------------------+--------------------+--------------------+----------+\n",
      "|   NSF|2015-08-31T00:00:00|2012-09-01T00:00:00|   47.074|       NSF|       2012| Acer; Address; A...|42.7871725|  -86.101102|          HOLLAND|       UNITED STATES|        HOPE COLLEGE|                MI|      49422-9000|         BEST, AARON A|Advances in seque...|2015-08-31T00:00:00|             1229585|2012-09-01T00:00:00|MRI: ACQUISITION ...|next+generation+s...|  171877.0|\n",
      "|   NIH|2016-08-31T00:00:00|2012-09-21T00:00:00|      847|       HHS|       2012| base; Base Seque...| 42.334978|  -71.105987|           BOSTON|       UNITED STATES|BRIGHAM AND WOMEN...|                MA|       021156110|    GOESSLING, WOLFRAM|DESCRIPTION (prov...|2016-08-31T00:00:00|     1R01DK095721-01|2012-09-21T00:00:00|MUTANT MAPPING AN...|next+generation+s...|  530443.0|\n",
      "|   NIH|               null|               null|     null|       HHS|       2012| base; Biological...| 44.055948|  -92.525906|        ROCHESTER|       UNITED STATES|         MAYO CLINIC|                MN|       559050001|  WEINSHILBOUM, RIC...|OVERALL SUMMARY A...|               null|5U19GM061388-13 (...|               null|NEXT GENERATION D...|next+generation+s...|  482387.0|\n",
      "|   NIH|2013-10-31T00:00:00|2011-08-15T00:00:00|      855|       HHS|       2012| Accounting; Affe...| 41.302711|  -72.931986|        NEW HAVEN|       UNITED STATES|     YALE UNIVERSITY|                CT|       065208327|     CACCONE, ADALGISA|DESCRIPTION (prov...|2013-10-31T00:00:00|     5R21AI094615-02|2011-08-15T00:00:00|NEXT GENERATION S...|next+generation+s...|  209013.0|\n",
      "|   NIH|               null|               null|     null|       HHS|       2012| Address; Adopted...| 35.366248|  -79.429394|      CHAPEL HILL|       UNITED STATES|UNIVERSITY OF NOR...|                NC|       275990001|  MARZLUFF, WILLIAM F.|Next Generation S...|               null|5P30CA016086-36 (...|               null|NEXT GENERATION S...|next+generation+s...|  270636.0|\n",
      "|   NIH|2012-03-05T00:00:00|2011-02-01T00:00:00|      859|       HHS|       2012| Actinobacteria c...| 37.240049| -119.503236|         LA JOLLA|       UNITED STATES|UNIVERSITY OF CAL...|                CA|       920930934|      BERNHARDT, PETER|DESCRIPTION (prov...|2012-03-05T00:00:00|     5F32GM096711-02|2011-02-01T00:00:00|NEXT-GENERATION S...|next+generation+s...|    8124.0|\n",
      "|   NIH|2017-04-30T00:00:00|2012-08-09T00:00:00|     null|       HHS|       2012| Adoption; Appoin...| 39.298737|  -76.594342|        BALTIMORE|       UNITED STATES|THE JOHNS HOPKINS...|                MD|       212051832|  YEGNASUBRAMANIAN,...|To maintain and b...|2017-04-30T00:00:00|2P30CA006973-49 (...|2012-08-09T00:00:00|NEXT GENERATION S...|next+generation+s...|  229477.0|\n",
      "|   NIH|2015-06-30T00:00:00|2010-07-15T00:00:00|      859|       HHS|       2012| Acute Myelocytic...| 38.648424|  -90.309068|      SAINT LOUIS|       UNITED STATES|WASHINGTON UNIVER...|                MO|       631304862|      MARDIS, ELAINE R|Pharmacogenomics ...|2015-06-30T00:00:00|     5U01GM097119-03|2010-07-15T00:00:00|NEXT GENERATION S...|next+generation+s...|  572671.0|\n",
      "|   NIH|               null|               null|     null|       HHS|       2012| Address; Adopted...| 35.366248|  -79.429394|      CHAPEL HILL|       UNITED STATES|UNIVERSITY OF NOR...|                NC|       275990001|  MARZLUFF, WILLIAM F.|Next Generation S...|               null|3P30CA016086-36S1...|               null|NEXT GENERATION S...|next+generation+s...|    2552.0|\n",
      "|   NIH|2015-02-28T00:00:00|2010-09-26T00:00:00|      172|       HHS|       2012| abstracting; Alu...| 40.481906|  -74.457066|       PISCATAWAY|       UNITED STATES|RUTGERS THE ST UN...|                NJ|       088543925|        XING, JINCHUAN|Project Summary/A...|2015-02-28T00:00:00|     4R00HG005846-03|2010-09-26T00:00:00|HIGH-THROUGHPUT M...|next+generation+s...|  249000.0|\n",
      "|   NIH|2013-08-31T00:00:00|2011-08-23T00:00:00|      172|       HHS|       2012| Address; Allelot...| 42.748478|  -84.483276|     EAST LANSING|       UNITED STATES|MICHIGAN STATE UN...|                MI|       488242600|       BROWN, C. TITUS|DESCRIPTION (prov...|2013-08-31T00:00:00|     5R25HG006243-02|2011-08-23T00:00:00|ANALYZING NEXT-GE...|next+generation+s...|   53653.0|\n",
      "|   NIH|2014-08-31T00:00:00|2011-09-01T00:00:00|      395|       HHS|       2012| Adopted; antican...| 38.648424|  -90.309068|      SAINT LOUIS|       UNITED STATES|WASHINGTON UNIVER...|                MO|       631304862|  MAHER, CHRISTOPHER A|PROJECT SUMMARY/A...|2014-08-31T00:00:00|     5R00CA149182-04|2011-09-01T00:00:00|CHARACTERIZATION ...|next+generation+s...|  231830.0|\n",
      "|   NSF|2016-07-31T00:00:00|2012-08-15T00:00:00|   47.049|       NSF|       2012| Algorithms; anal...| 40.092617|  -88.247723|        CHAMPAIGN|       UNITED STATES|UNIVERSITY OF ILL...|                IL|      61820-7473|              MA, PING|Recently, the eme...|2016-07-31T00:00:00|             1222718|2012-08-15T00:00:00|COLLABORATIVE RES...|next+generation+s...|  372098.0|\n",
      "|   NSF|2016-07-31T00:00:00|2012-08-15T00:00:00|   47.049|       NSF|       2012| Algorithms; anal...| 33.968143|-117.3358533|        RIVERSIDE|       UNITED STATES|UNIVERSITY OF CAL...|                CA|      92521-1000|          CUI, XINPING|Recently, the eme...|2016-07-31T00:00:00|             1223107|2012-08-15T00:00:00|COLLABORATIVE RES...|next+generation+s...|  532114.0|\n",
      "|   NIH|2015-02-28T00:00:00|2010-06-01T00:00:00|      172|       HHS|       2012| Address; Algorit...|  42.33625|  -71.070668|           BOSTON|       UNITED STATES|BOSTON UNIVERSITY...|                MA|       021182340|  JOHNSON, WILLIAM ...|DESCRIPTION (prov...|2015-02-28T00:00:00|     7R01HG005692-03|2010-06-01T00:00:00|STATISTICAL TOOLS...|next+generation+s...|  336105.0|\n",
      "|   NSF|2012-08-31T00:00:00|2009-09-01T00:00:00|   47.074|       NSF|       2009| Achievement; Bac...|30.6058424| -96.3689349|  COLLEGE STATION|       UNITED STATES|TEXAS A & M UNIVE...|                TX|      77845-0000|     DICKMAN, MARTIN B|During the past 1...|2012-08-31T00:00:00|             0923502|2009-09-01T00:00:00|MRI: ACQUISITION ...|next+generation+s...|  612019.0|\n",
      "|   NSF|2012-08-31T00:00:00|2009-09-01T00:00:00|   47.074|       NSF|       2009| Actinium; Addres...|40.4777023| -74.4254953|    NEW BRUNSWICK|       UNITED STATES|RUTGERS THE ST UN...|                NJ|      08901-8559|        DOONER, HUGO K|PI: Hugo K. Doone...|2012-08-31T00:00:00|             0923950|2009-09-01T00:00:00|FACILE PRODUCTION...|next+generation+s...| 1242967.0|\n",
      "|   NSF|2014-08-31T00:00:00|2011-09-01T00:00:00|   47.050|       NSF|       2011| Address; Animals...|41.8084314| -72.2495231|           STORRS|       UNITED STATES|UNIVERSITY OF CON...|                CT|      06269-1133|     MCMANUS, GEORGE B|The Ocean's bioma...|2014-08-31T00:00:00|             1130033|2011-09-01T00:00:00|COLLABORATIVE RES...|next+generation+s...|  408830.0|\n",
      "|   NSF|2014-08-31T00:00:00|2011-09-01T00:00:00|   47.050|       NSF|       2011| Address; Animals...|42.3177887|  -72.635903|      NORTHAMPTON|       UNITED STATES|       SMITH COLLEGE|                MA|      01063-6304|         KATZ, LAURA A|The Ocean's bioma...|2014-08-31T00:00:00|             1129734|2011-09-01T00:00:00|COLLABORATIVE RES...|next+generation+s...|  421638.0|\n",
      "|   NSF|2011-09-30T00:00:00|2010-10-15T00:00:00|   47.074|       NSF|       2011| Area; Basic Scie...|32.8328112|-117.2712717|         LA JOLLA|       UNITED STATES|UNIVERSITY OF CAL...|                CA|      92093-0210|   BARTLETT, DOUGLAS H|This grant will s...|2011-09-30T00:00:00|             1068643|2010-10-15T00:00:00|US-EC WORKSHOP ON...|next+generation+s...|   25000.0|\n",
      "+------+-------------------+-------------------+---------+----------+-----------+--------------------+----------+------------+-----------------+--------------------+--------------------+------------------+----------------+----------------------+--------------------+-------------------+--------------------+-------------------+--------------------+--------------------+----------+\n",
      "only showing top 20 rows\n",
      "\n"
     ]
    }
   ],
   "source": [
    "#Read in CSV \n",
    "df = spark.read.format(\"csv\").option(\"header\", \"true\").load(\"FedReporter.csv\")\n",
    "df.show()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 92,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "28303"
      ]
     },
     "execution_count": 92,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df.count()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 93,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Remove duplicate projects\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 94,
   "metadata": {
    "scrolled": false
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "Tokenizer_41c9831f753c21ee2acb"
      ]
     },
     "execution_count": 94,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# Abstracts\n",
    "# ---------------\n",
    "\n",
    "#Tokenize words in abstracts\n",
    "tokenizer = Tokenizer(inputCol=\"Project Abstract\", outputCol=\"abstract_words_raw\")\n",
    "tokenizer"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 95,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Create a function to return the length of a list\n",
    "def abstract_word_list_length(abstract_word_list):\n",
    "    return len(abstract_word_list)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 96,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "<function __main__.abstract_word_list_length>"
      ]
     },
     "execution_count": 96,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# Create a user defined function \n",
    "count_tokens = udf(abstract_word_list_length, IntegerType())\n",
    "count_tokens"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 97,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+------+\n",
      "|Project Abstract                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           |abstract_words_raw                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |tokens|\n",
      "+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+------+\n",
      "|Advances in sequencing technology have dramatically shifted the way science is performed in many fields of Biology, which has also shifted the way education of the next generation of scientists must be conducted. However, the generation of sequencing data has been restricted to large laboratories, consortia of laboratories, and core sequencing facilities, limiting the exposure of undergraduates to these technologies. The recent introduction of benchtop next generation sequencing platforms promises to make moderately high throughput sequencing available to many individual research laboratories and smaller institutions. This Major Research Instrumentation grant supports the acquisition of an Ion Torrent Personal Genome Machine (Ion PGM) sequencing platform by Hope College. The Ion PGM will have significant impact on undergraduate research and training at Hope College by making affordable, high throughput sequencing capacity available to faculty and undergraduate researchers for use in established and emerging research and education programs. These programs include research focusing on: 1) the genome sequences of viruses that infect bacteria (Mycobacteriophage) that are discovered by first year college students as part of introductory biology laboratory courses; 2) long-term environmental studies of the local Lake Macatawa Watershed to identify and track microbial sources of contamination through whole genome sequencing of novel bacterial isolates; 3) advancing understanding of fundamental gene expression mechanisms of the mammalian parasite Giardia lamblia; 4) understanding phylogenetic and biogeographic relationships among maples found across the World; and 5) testing potential horizontal gene transfers of plastid genes between parasitic plants and their photosynthetic hosts. The projects advance scientific understanding of a wide range of biological and environmental problems that can be addressed through DNA sequencing at different scales. Each project will be made more effective by the in house availability of the Ion PGM, which is uniquely versatile in being able to generate sequence data at vastly different scales in a cost effective manner.<br/><br/>This sequencing platform will allow Hope College to effectively train undergraduates in the design of experiments requiring sequencing data; usage of next generation sequencing technology to generate necessary data; and analysis, interpretation, and dissemination of large scale data sets in both research laboratory and classroom laboratory settings. The Ion PGM will immediately enhance projects and programs that impact over 100 undergraduates during each academic year. First year students will be impacted through Hope College's highly successful yearlong course in phage genomics research, which is designed to retain high quality students in STEM fields. Students at all levels will participate in the research laboratories of the PI?s and their collaborators. Students in upper level Biology courses will use the technology in classroom laboratory-based research projects. Hope College?s established summer research programs will allow us to broaden exposure of students across the nation and of students from underrepresented groups in sciences in the use of leading edge sequencing technology. Faculty will be impacted through their ability to take advantage of next generation sequencing technology to advance projects into new and innovative areas and generate preliminary data for sustaining extramurally funded research programs. Student-faculty collaborative research will be disseminated to the scientific community through local, regional, and national/international scientific conferences and peer-reviewed publications and to the general public through community-based popular seminars. Sequence data will be deposited to appropriate openly accessible databases. Regional and national collaborators will use the Ion PGM, broadening its impact beyond Hope College to students in West Michigan and throughout the nation.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             |[advances, in, sequencing, technology, have, dramatically, shifted, the, way, science, is, performed, in, many, fields, of, biology,, which, has, also, shifted, the, way, education, of, the, next, generation, of, scientists, must, be, conducted., however,, the, generation, of, sequencing, data, has, been, restricted, to, large, laboratories,, consortia, of, laboratories,, and, core, sequencing, facilities,, limiting, the, exposure, of, undergraduates, to, these, technologies., the, recent, introduction, of, benchtop, next, generation, sequencing, platforms, promises, to, make, moderately, high, throughput, sequencing, available, to, many, individual, research, laboratories, and, smaller, institutions., this, major, research, instrumentation, grant, supports, the, acquisition, of, an, ion, torrent, personal, genome, machine, (ion, pgm), sequencing, platform, by, hope, college., the, ion, pgm, will, have, significant, impact, on, undergraduate, research, and, training, at, hope, college, by, making, affordable,, high, throughput, sequencing, capacity, available, to, faculty, and, undergraduate, researchers, for, use, in, established, and, emerging, research, and, education, programs., these, programs, include, research, focusing, on:, 1), the, genome, sequences, of, viruses, that, infect, bacteria, (mycobacteriophage), that, are, discovered, by, first, year, college, students, as, part, of, introductory, biology, laboratory, courses;, 2), long-term, environmental, studies, of, the, local, lake, macatawa, watershed, to, identify, and, track, microbial, sources, of, contamination, through, whole, genome, sequencing, of, novel, bacterial, isolates;, 3), advancing, understanding, of, fundamental, gene, expression, mechanisms, of, the, mammalian, parasite, giardia, lamblia;, 4), understanding, phylogenetic, and, biogeographic, relationships, among, maples, found, across, the, world;, and, 5), testing, potential, horizontal, gene, transfers, of, plastid, genes, between, parasitic, plants, and, their, photosynthetic, hosts., the, projects, advance, scientific, understanding, of, a, wide, range, of, biological, and, environmental, problems, that, can, be, addressed, through, dna, sequencing, at, different, scales., each, project, will, be, made, more, effective, by, the, in, house, availability, of, the, ion, pgm,, which, is, uniquely, versatile, in, being, able, to, generate, sequence, data, at, vastly, different, scales, in, a, cost, effective, manner.<br/><br/>this, sequencing, platform, will, allow, hope, college, to, effectively, train, undergraduates, in, the, design, of, experiments, requiring, sequencing, data;, usage, of, next, generation, sequencing, technology, to, generate, necessary, data;, and, analysis,, interpretation,, and, dissemination, of, large, scale, data, sets, in, both, research, laboratory, and, classroom, laboratory, settings., the, ion, pgm, will, immediately, enhance, projects, and, programs, that, impact, over, 100, undergraduates, during, each, academic, year., first, year, students, will, be, impacted, through, hope, college's, highly, successful, yearlong, course, in, phage, genomics, research,, which, is, designed, to, retain, high, quality, students, in, stem, fields., students, at, all, levels, will, participate, in, the, research, laboratories, of, the, pi?s, and, their, collaborators., students, in, upper, level, biology, courses, will, use, the, technology, in, classroom, laboratory-based, research, projects., hope, college?s, established, summer, research, programs, will, allow, us, to, broaden, exposure, of, students, across, the, nation, and, of, students, from, underrepresented, groups, in, sciences, in, the, use, of, leading, edge, sequencing, technology., faculty, will, be, impacted, through, their, ability, to, take, advantage, of, next, generation, sequencing, technology, to, advance, projects, into, new, and, innovative, areas, and, generate, preliminary, data, for, sustaining, extramurally, funded, research, programs., student-faculty, collaborative, research, will, be, disseminated, to, the, scientific, community, through, local,, regional,, and, national/international, scientific, conferences, and, peer-reviewed, publications, and, to, the, general, public, through, community-based, popular, seminars., sequence, data, will, be, deposited, to, appropriate, openly, accessible, databases., regional, and, national, collaborators, will, use, the, ion, pgm,, broadening, its, impact, beyond, hope, college, to, students, in, west, michigan, and, throughout, the, nation.]                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          |557   |\n",
      "|DESCRIPTION (provided by applicant): The power of the zebrafish system stems from its utility as a developmental biology model combined with the ease of its genetic manipulation and experimentation. Our understanding of key genetic mechanisms of vertebrate development has been propelled by the phenotypic characterization, genetic mapping and positional cloning of induced and spontaneous mutations in zebrafish. However, the potential of this system has not been fully realized, as inefficient microsatellite-based mapping remains the primary method in the field. We propose to apply technological and computational advances of present day genomics to genetic mapping in the zebrafish system. Specifically, we propose to develop a method for rapid and accurate mapping of recessive zebrafish mutants using Next Generation Sequencing (NGS) of pooled samples. We also propose to investigate parameters of screen design and sample analysis to optimize the use of this protocol. Finally, we aim to develop methods for identification of the causal mutation among the variants discovered within the mapping interval. Application of NGS technology, complemented by specifically developed computational techniques, will provide an efficient, accurate and inexpensive method for genetic mapping in zebrafish. This approach will enable the simultaneous identification of informative genetic markers, mapping of the mutation position, and potential identification of the causal sequence change in a single experiment. The data obtained in these genomic analyses and the methods developed will be made available to the zebrafish community. Importantly, these approaches will also be widely applicable to genetic analysis of other model systems.  PUBLIC HEALTH RELEVANCE: Analysis of zebrafish mutants has enabled the identification of genes contributing to fundamental biological processes, including human diseases; however, the methods used for mutant mapping and gene discovery are inefficient. Here, we propose a fast and cost-effective method for genetic mapping and mutation identification using Next Generation Sequencing. This will facilitate the more rapid discovery of gene function and the translation of this knowledge to biomedical investigation.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  |[description, (provided, by, applicant):, the, power, of, the, zebrafish, system, stems, from, its, utility, as, a, developmental, biology, model, combined, with, the, ease, of, its, genetic, manipulation, and, experimentation., our, understanding, of, key, genetic, mechanisms, of, vertebrate, development, has, been, propelled, by, the, phenotypic, characterization,, genetic, mapping, and, positional, cloning, of, induced, and, spontaneous, mutations, in, zebrafish., however,, the, potential, of, this, system, has, not, been, fully, realized,, as, inefficient, microsatellite-based, mapping, remains, the, primary, method, in, the, field., we, propose, to, apply, technological, and, computational, advances, of, present, day, genomics, to, genetic, mapping, in, the, zebrafish, system., specifically,, we, propose, to, develop, a, method, for, rapid, and, accurate, mapping, of, recessive, zebrafish, mutants, using, next, generation, sequencing, (ngs), of, pooled, samples., we, also, propose, to, investigate, parameters, of, screen, design, and, sample, analysis, to, optimize, the, use, of, this, protocol., finally,, we, aim, to, develop, methods, for, identification, of, the, causal, mutation, among, the, variants, discovered, within, the, mapping, interval., application, of, ngs, technology,, complemented, by, specifically, developed, computational, techniques,, will, provide, an, efficient,, accurate, and, inexpensive, method, for, genetic, mapping, in, zebrafish., this, approach, will, enable, the, simultaneous, identification, of, informative, genetic, markers,, mapping, of, the, mutation, position,, and, potential, identification, of, the, causal, sequence, change, in, a, single, experiment., the, data, obtained, in, these, genomic, analyses, and, the, methods, developed, will, be, made, available, to, the, zebrafish, community., importantly,, these, approaches, will, also, be, widely, applicable, to, genetic, analysis, of, other, model, systems., , public, health, relevance:, analysis, of, zebrafish, mutants, has, enabled, the, identification, of, genes, contributing, to, fundamental, biological, processes,, including, human, diseases;, however,, the, methods, used, for, mutant, mapping, and, gene, discovery, are, inefficient., here,, we, propose, a, fast, and, cost-effective, method, for, genetic, mapping, and, mutation, identification, using, next, generation, sequencing., this, will, facilitate, the, more, rapid, discovery, of, gene, function, and, the, translation, of, this, knowledge, to, biomedical, investigation.]                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |317   |\n",
      "|OVERALL SUMMARY AND SPECIFIC AIMS: NETWORK RESOURCE This component of the Mayo Clinic PGRN renewal application represents a request for the funding of a 'Network Resource' designed to provide the entire PGRN with access to the latest techniques In the rapidly evolving field of 'Next Generation' DNA sequencing. Access would be provided through a recurring, peer-reviewed process available to all PGRN Centers similar to the current 'PGRN-RIKEN' process for obtaining access to pharmacogenomic GWAS genotyping through the PGRN-RIKEN collaboration. This proposed 'PGRN Nehwork Resource' is one of three coordinated applications from current PGRN Centers, all with similar goals - clearly indicating an unmet need in pharmacogenomic research - a need emphasized by the letters of support from current PGRN Centers that are attached to this application. Those letters were sent by eight different PGRN research centers. Although pharmacoqenomics has always required access to DNA sequencing, that need has been accentuated bv recent dramatic changes in DNA seguencing technology and bv the eguallv dramatic outcomes of the application of genome-wide approaches to pharmacogenomics. Each of these proposals is designed to provide a 'link' between the PGRN and a large, experienced Genomics Center with expertise in both the application and the evaluation of DNA sequencing technology. The issue is not whether pharmacogenomics as a discipline requires access to the very latest in rapidly evolving DNA sequencing techniques, but only how to achieve that goal in a way that will best advance the science while being affordable and open to all PGRN Centers. Specifically, the Mayo PGRN is proposing that a formal collaborative relationship be established with the Baylor College of Medicine (BCM)-Human Genome Sequencing Center (HGSC), one of three NHGRIsupported large genome centers - with an open, protocol-based selection process similar to the well established and highly successful PGRN-RIKEN process for the selection of PGRN pharmacogenomic GWAS genotyping projects performed in collaboration with RIKEN. The initial focus of the Mayo PGRN-sponsored 'Next Generation' DNA sequencing Network Resource would be on sequencing large contiguous regions of the genome. In some cases, this will involve complete resequencing of large genes or tandemly repeated gene families or even entire gene families, e.g., the 'CYPome', all cytochrome P450 genes. In other cases, it wilt involve focused resequencing across 'SNP signals' observed during pharmacogenomic GWA studies - in many cases, GWA studies made possible by the PGRN-RIKEN collaboration, thus building on a foundation established by that very successful program. Since the review panel for these applications will evaluate a series of Network Resource applications for access to Next Generation DNA sequencing, it should be emphasized that the Mayo PGRN looks upon these efforts as complementary. Therefore, it is possible that reviewers may conclude that some 'combination' of these proposals should be supported. All of these proposals share;  A recognition of the critical need for access to rapidly evolving DNA sequencing techniques in order to advance the science of pharmacogenomics.  A recognition that established Genome Centers have the requisite experience, expertise and infrastructure required to make this effort succeed.  A recognition that the 'process' for access to Next Generation DNA sequencing will have to be open to all PGRN centers, based on scientific significance and feasibility. The cost associated with these studies would be covered primarily by funding available to the 'Network Resource'.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |[overall, summary, and, specific, aims:, network, resource, this, component, of, the, mayo, clinic, pgrn, renewal, application, represents, a, request, for, the, funding, of, a, 'network, resource', designed, to, provide, the, entire, pgrn, with, access, to, the, latest, techniques, in, the, rapidly, evolving, field, of, 'next, generation', dna, sequencing., access, would, be, provided, through, a, recurring,, peer-reviewed, process, available, to, all, pgrn, centers, similar, to, the, current, 'pgrn-riken', process, for, obtaining, access, to, pharmacogenomic, gwas, genotyping, through, the, pgrn-riken, collaboration., this, proposed, 'pgrn, nehwork, resource', is, one, of, three, coordinated, applications, from, current, pgrn, centers,, all, with, similar, goals, -, clearly, indicating, an, unmet, need, in, pharmacogenomic, research, -, a, need, emphasized, by, the, letters, of, support, from, current, pgrn, centers, that, are, attached, to, this, application., those, letters, were, sent, by, eight, different, pgrn, research, centers., although, pharmacoqenomics, has, always, required, access, to, dna, sequencing,, that, need, has, been, accentuated, bv, recent, dramatic, changes, in, dna, seguencing, technology, and, bv, the, eguallv, dramatic, outcomes, of, the, application, of, genome-wide, approaches, to, pharmacogenomics., each, of, these, proposals, is, designed, to, provide, a, 'link', between, the, pgrn, and, a, large,, experienced, genomics, center, with, expertise, in, both, the, application, and, the, evaluation, of, dna, sequencing, technology., the, issue, is, not, whether, pharmacogenomics, as, a, discipline, requires, access, to, the, very, latest, in, rapidly, evolving, dna, sequencing, techniques,, but, only, how, to, achieve, that, goal, in, a, way, that, will, best, advance, the, science, while, being, affordable, and, open, to, all, pgrn, centers., specifically,, the, mayo, pgrn, is, proposing, that, a, formal, collaborative, relationship, be, established, with, the, baylor, college, of, medicine, (bcm)-human, genome, sequencing, center, (hgsc),, one, of, three, nhgrisupported, large, genome, centers, -, with, an, open,, protocol-based, selection, process, similar, to, the, well, established, and, highly, successful, pgrn-riken, process, for, the, selection, of, pgrn, pharmacogenomic, gwas, genotyping, projects, performed, in, collaboration, with, riken., the, initial, focus, of, the, mayo, pgrn-sponsored, 'next, generation', dna, sequencing, network, resource, would, be, on, sequencing, large, contiguous, regions, of, the, genome., in, some, cases,, this, will, involve, complete, resequencing, of, large, genes, or, tandemly, repeated, gene, families, or, even, entire, gene, families,, e.g.,, the, 'cypome',, all, cytochrome, p450, genes., in, other, cases,, it, wilt, involve, focused, resequencing, across, 'snp, signals', observed, during, pharmacogenomic, gwa, studies, -, in, many, cases,, gwa, studies, made, possible, by, the, pgrn-riken, collaboration,, thus, building, on, a, foundation, established, by, that, very, successful, program., since, the, review, panel, for, these, applications, will, evaluate, a, series, of, network, resource, applications, for, access, to, next, generation, dna, sequencing,, it, should, be, emphasized, that, the, mayo, pgrn, looks, upon, these, efforts, as, complementary., therefore,, it, is, possible, that, reviewers, may, conclude, that, some, 'combination', of, these, proposals, should, be, supported., all, of, these, proposals, share;, , a, recognition, of, the, critical, need, for, access, to, rapidly, evolving, dna, sequencing, techniques, in, order, to, advance, the, science, of, pharmacogenomics., , a, recognition, that, established, genome, centers, have, the, requisite, experience,, expertise, and, infrastructure, required, to, make, this, effort, succeed., , a, recognition, that, the, 'process', for, access, to, next, generation, dna, sequencing, will, have, to, be, open, to, all, pgrn, centers,, based, on, scientific, significance, and, feasibility., the, cost, associated, with, these, studies, would, be, covered, primarily, by, funding, available, to, the, 'network, resource'.]                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |548   |\n",
      "|DESCRIPTION (provided by applicant): Human African trypanosomiasis, also known as sleeping sickness, is a zoonotic disease affecting thousands of people across sub-Saharan Africa and periodic epidemics have caused significant social and economic disruption. The trypanosome responsible for sleeping sickness is morphologically indistinguishable from closely related parasites found in animals, leading to confusion regarding the origins of human disease outbreaks. This project seeks to improve the tools available for understanding the epidemiology of this neglected disease. The immediate aims are to develop a robust and broadly applicable panel of genetic markers and to validate these in a pilot study in Uganda, a disease-endemic country currently experiencing a severe epidemic. This project will employ next-generation sequencing of multiple trypanosome genomes, selected to encompass the complicated evolutionary relationships observed among trypanosomes in preliminary studies. The genetic polymorphisms identified from genome comparisons will be evaluated for their ability to resolve fine scale population structure and to distinguish between human-infective and animal-restricted parasites in the Ugandan disease focus. In the longer term, the markers developed in this project will provide a foundation for cataloguing the full diversity of trypanosomes in nature, for identifying the pool of trypanosomes that are most relevant for human disease, and for characterizing the relationships among trypanosome populations distributed across eastern Africa. In addition, comparisons among the genomes of animal-restricted and human-disease causing trypanosomes will provide insight into the functional genetic differences conferring human infectivity.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          |[description, (provided, by, applicant):, human, african, trypanosomiasis,, also, known, as, sleeping, sickness,, is, a, zoonotic, disease, affecting, thousands, of, people, across, sub-saharan, africa, and, periodic, epidemics, have, caused, significant, social, and, economic, disruption., the, trypanosome, responsible, for, sleeping, sickness, is, morphologically, indistinguishable, from, closely, related, parasites, found, in, animals,, leading, to, confusion, regarding, the, origins, of, human, disease, outbreaks., this, project, seeks, to, improve, the, tools, available, for, understanding, the, epidemiology, of, this, neglected, disease., the, immediate, aims, are, to, develop, a, robust, and, broadly, applicable, panel, of, genetic, markers, and, to, validate, these, in, a, pilot, study, in, uganda,, a, disease-endemic, country, currently, experiencing, a, severe, epidemic., this, project, will, employ, next-generation, sequencing, of, multiple, trypanosome, genomes,, selected, to, encompass, the, complicated, evolutionary, relationships, observed, among, trypanosomes, in, preliminary, studies., the, genetic, polymorphisms, identified, from, genome, comparisons, will, be, evaluated, for, their, ability, to, resolve, fine, scale, population, structure, and, to, distinguish, between, human-infective, and, animal-restricted, parasites, in, the, ugandan, disease, focus., in, the, longer, term,, the, markers, developed, in, this, project, will, provide, a, foundation, for, cataloguing, the, full, diversity, of, trypanosomes, in, nature,, for, identifying, the, pool, of, trypanosomes, that, are, most, relevant, for, human, disease,, and, for, characterizing, the, relationships, among, trypanosome, populations, distributed, across, eastern, africa., in, addition,, comparisons, among, the, genomes, of, animal-restricted, and, human-disease, causing, trypanosomes, will, provide, insight, into, the, functional, genetic, differences, conferring, human, infectivity.]                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          |234   |\n",
      "|Next Generation Sequencing and Genotyping Core Facility. The goal of the Next Gen Sequencing component is to make 'next generation' sequencing and related services widely available to UNC LCCC members at cost-effective prices. Services include: whole genome shotgun sequencing of human and model organisms, chromatin-IP sample sequencing, measuring RNA expression with both digital gene expression and RNA-seq, SNP typing, and polymorphism screens. The component adds value to the cancer center by putting complex and expensive DNA/RNA sequencing technology within easy reach of UNC LCCC members. Strongly integrated genomics and bioinformatics groups interact with the user to ensure robust data that is archived appropriately for future use. This component is key in UNC LCCC's participation in The Cancer Genome Atlas project and serving LCCC needs. Future plans Include expanding staff and equipment so that a full array of Next Gen sequencing and services may be provided in a timely manner, as well as the acquisition of the next generation of technology.   The goal of the DNA Sequencing component is to produce high quality sequencing and genotyping data rapidly and at a reasonable cost for UNC-CH researchers using state-of-the-art technologies as well as provide technical support to enhance the value of results produced. Services include DNA sequencing and microsatellite genotyping. The component adds value to the cancer center by producing high-quality data at a competitive price while providing the benefits of a local resource to assist in: scientific strategies, sample preparation, sequencing through difficult regions, and assistance in data interpretation. This component has a wide user base with over 220 different laboratories as users in 2009. Future plans include continued use of existing equipment, reduction of chemistry costs, and evaluation of future sequencing technology options such as long read single molecule sequencing technologies. The goal of the High Throughput Genotyping component is to produce high-quality genotyping data rapidly and at a reasonable cost for researchers using state-of-the-art technologies. Services include SNP genotyping, copy number variation and DNA methylation profiling analysis. New technologies and/or applications are being explored to increase the core's competitive edge and to allow for Genome Wide Association Study on rare variants. These core components are used by multiple members and in each core, peer reviewed member used exceeds 70%. The proposed budget of $241,797 is less than 5% of the core's operating cost. The budget provides salary stability for vital core staff who work with members to adopt these novel technologies.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |[next, generation, sequencing, and, genotyping, core, facility., the, goal, of, the, next, gen, sequencing, component, is, to, make, 'next, generation', sequencing, and, related, services, widely, available, to, unc, lccc, members, at, cost-effective, prices., services, include:, whole, genome, shotgun, sequencing, of, human, and, model, organisms,, chromatin-ip, sample, sequencing,, measuring, rna, expression, with, both, digital, gene, expression, and, rna-seq,, snp, typing,, and, polymorphism, screens., the, component, adds, value, to, the, cancer, center, by, putting, complex, and, expensive, dna/rna, sequencing, technology, within, easy, reach, of, unc, lccc, members., strongly, integrated, genomics, and, bioinformatics, groups, interact, with, the, user, to, ensure, robust, data, that, is, archived, appropriately, for, future, use., this, component, is, key, in, unc, lccc's, participation, in, the, cancer, genome, atlas, project, and, serving, lccc, needs., future, plans, include, expanding, staff, and, equipment, so, that, a, full, array, of, next, gen, sequencing, and, services, may, be, provided, in, a, timely, manner,, as, well, as, the, acquisition, of, the, next, generation, of, technology., , , the, goal, of, the, dna, sequencing, component, is, to, produce, high, quality, sequencing, and, genotyping, data, rapidly, and, at, a, reasonable, cost, for, unc-ch, researchers, using, state-of-the-art, technologies, as, well, as, provide, technical, support, to, enhance, the, value, of, results, produced., services, include, dna, sequencing, and, microsatellite, genotyping., the, component, adds, value, to, the, cancer, center, by, producing, high-quality, data, at, a, competitive, price, while, providing, the, benefits, of, a, local, resource, to, assist, in:, scientific, strategies,, sample, preparation,, sequencing, through, difficult, regions,, and, assistance, in, data, interpretation., this, component, has, a, wide, user, base, with, over, 220, different, laboratories, as, users, in, 2009., future, plans, include, continued, use, of, existing, equipment,, reduction, of, chemistry, costs,, and, evaluation, of, future, sequencing, technology, options, such, as, long, read, single, molecule, sequencing, technologies., the, goal, of, the, high, throughput, genotyping, component, is, to, produce, high-quality, genotyping, data, rapidly, and, at, a, reasonable, cost, for, researchers, using, state-of-the-art, technologies., services, include, snp, genotyping,, copy, number, variation, and, dna, methylation, profiling, analysis., new, technologies, and/or, applications, are, being, explored, to, increase, the, core's, competitive, edge, and, to, allow, for, genome, wide, association, study, on, rare, variants., these, core, components, are, used, by, multiple, members, and, in, each, core,, peer, reviewed, member, used, exceeds, 70%., the, proposed, budget, of, $241,797, is, less, than, 5%, of, the, core's, operating, cost., the, budget, provides, salary, stability, for, vital, core, staff, who, work, with, members, to, adopt, these, novel, technologies.]                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          |404   |\n",
      "|DESCRIPTION (provided by applicant): Actinomycetes have a tremendous potential to produce natural products with medicinal value. The long-term goal of this project is to develop novel agents for the treatment of human disease facilitated by the application of next-generation DNA sequencing to rapidly access actinomycete genetic information. The marine Streptomyces sp. CNH-189 is a producer of structurally fascinating meroterpenoids with potent activity against a broad panel of multi-drug resistant Staphylococcus aureus strains. The overall research objectives are to beta-test Illumina DNA sequencing for rapidly accessing natural product gene cluster sequences in Streptomyces sp. CNH-189, and to use genetics and biochemistry to link meroterpenoid biosynthesis to the identified gene clusters. Preliminary studies include the construction of genomic DNA fragment libraries that are ready for paired-end Illumina sequencing, and the isolation of a fosmid clone that contains a large part of the biosynthetic gene cluster for meroterpenoid biosynthesis. Through a series of experiments, I propose to evaluate how Illumina sequencing can be used to find gene clusters in Streptomyces sp. CNH-189 and other actinomycetes, to obtain a complete coverage of the gene cluster, and to probe the biosynthesis through in vitro biochemistry, gene disruption, and comparative metabolite profiling. The unique chemical structures and biological activities of the natural products isolated from Streptomyces sp. CNH-189 may lead to new chemotherapies and fundamental biochemical knowledge.  PUBLIC HEALTH RELEVANCE: Many pharmaceuticals currently used in the clinic are natural product based. Therefore, natural product discovery, understanding of biosynthetic pathways, and engineering the production of natural product analogs are key for the development of new human chemotherapies. Next-generation DNA sequencing and short read assembly ofers a new and potentialy general platform for accessing gene clusters that could facilitate the genome mining and understanding of clustered biosynthetic pathways in microorganisms. The proposed research will for the first time evaluate Illumina sequencing as a platform for rapidly accessing biosynthetic gene clusters in actinomycetes. Successful completion of this research may advance the field of natural product discovery and biosynthesis.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              |[description, (provided, by, applicant):, actinomycetes, have, a, tremendous, potential, to, produce, natural, products, with, medicinal, value., the, long-term, goal, of, this, project, is, to, develop, novel, agents, for, the, treatment, of, human, disease, facilitated, by, the, application, of, next-generation, dna, sequencing, to, rapidly, access, actinomycete, genetic, information., the, marine, streptomyces, sp., cnh-189, is, a, producer, of, structurally, fascinating, meroterpenoids, with, potent, activity, against, a, broad, panel, of, multi-drug, resistant, staphylococcus, aureus, strains., the, overall, research, objectives, are, to, beta-test, illumina, dna, sequencing, for, rapidly, accessing, natural, product, gene, cluster, sequences, in, streptomyces, sp., cnh-189,, and, to, use, genetics, and, biochemistry, to, link, meroterpenoid, biosynthesis, to, the, identified, gene, clusters., preliminary, studies, include, the, construction, of, genomic, dna, fragment, libraries, that, are, ready, for, paired-end, illumina, sequencing,, and, the, isolation, of, a, fosmid, clone, that, contains, a, large, part, of, the, biosynthetic, gene, cluster, for, meroterpenoid, biosynthesis., through, a, series, of, experiments,, i, propose, to, evaluate, how, illumina, sequencing, can, be, used, to, find, gene, clusters, in, streptomyces, sp., cnh-189, and, other, actinomycetes,, to, obtain, a, complete, coverage, of, the, gene, cluster,, and, to, probe, the, biosynthesis, through, in, vitro, biochemistry,, gene, disruption,, and, comparative, metabolite, profiling., the, unique, chemical, structures, and, biological, activities, of, the, natural, products, isolated, from, streptomyces, sp., cnh-189, may, lead, to, new, chemotherapies, and, fundamental, biochemical, knowledge., , public, health, relevance:, many, pharmaceuticals, currently, used, in, the, clinic, are, natural, product, based., therefore,, natural, product, discovery,, understanding, of, biosynthetic, pathways,, and, engineering, the, production, of, natural, product, analogs, are, key, for, the, development, of, new, human, chemotherapies., next-generation, dna, sequencing, and, short, read, assembly, ofers, a, new, and, potentialy, general, platform, for, accessing, gene, clusters, that, could, facilitate, the, genome, mining, and, understanding, of, clustered, biosynthetic, pathways, in, microorganisms., the, proposed, research, will, for, the, first, time, evaluate, illumina, sequencing, as, a, platform, for, rapidly, accessing, biosynthetic, gene, clusters, in, actinomycetes., successful, completion, of, this, research, may, advance, the, field, of, natural, product, discovery, and, biosynthesis.]                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              |330   |\n",
      "|To maintain and build on the Sidney Kimmel Comprehensive Cancer Centers (SKCCC) record of excellence in the fields of cancer genetics and epigenetics, the SKCCC established a new Next Generation Sequencing Core (NGSC) as a shared resource in January 2009. A product of a convergence of advances in molecular biology, engineering, computer science, and bioinformatics. Next Generation Sequencing features the ability to sequence billions of base pairs of DNA in a single run, at a cost that is several orders of magnitude less than previous gold standard sequencing technologies. Next Generation Sequencing is an extremely versatile technology and can be used to investigate a multitude of genomic processes in a previously unimaginable genome-wide and unbiased fashion. In order to facilitate widespread adoption of this relatively new technology, rather than administering the Core as a 'black-box' facility, the Core operates as a 'collaborative Core' in which users can consult and collaborate with NGSC faculty and staff to: 1) identify the optimal molecular biology approaches (prior to sequencing library construction) to synapse their research questions with the capabilities of the Core, 2) prepare the appropriate libraries and carry out massively parallel sequencing, and 3) carry out the appropriate primary, secondary, and tertiary bioinformatics analyses to analyze and interpret the results. The NGSC features Applied Biosystems SOUD Next Generation Sequencing instruments capable of generating nearly 100 Gigabases of sequencing data in a single run, state-of-the-art equipment for sequencing library preparation and quality control, and powerful computational resources including a server with ten compute nodes (each containing Quad Core Intel Xeon processors and 16 Gigabytes of memory) and 100 terabytes of storage for dedicated bioinformatics analyses. The Core is currently administered by a faculty director overseeing all activities, a faculty Co-Director overseeing bioinformatics support, a laboratory manager, and a bioinformatics manager. The Core's goal is to be as flexible as possible in assisting researchers in exploring all aspects of cancer genetics and epigenetics, including, but not limited to, germline and somatic variation/mutation, genomic structural variations/alterations, transcriptome analysis, transcription factor binding sites, chromatin modifications, and DNA methylation. Lay: The Next Generation Sequencing Core provides cutting edge services that harness the most recent technological advances to allow large scale genomic sequencing of many hundreds of millions of DNA molecules in parallel at about 100,000 times less cost than previous sequencing technologies. These services are poised to rapidly accelerate the pace of discovery and clinical translation in cancer molecular genetics research. SKCCC Managed Shared Resource                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            |[to, maintain, and, build, on, the, sidney, kimmel, comprehensive, cancer, centers, (skccc), record, of, excellence, in, the, fields, of, cancer, genetics, and, epigenetics,, the, skccc, established, a, new, next, generation, sequencing, core, (ngsc), as, a, shared, resource, in, january, 2009., a, product, of, a, convergence, of, advances, in, molecular, biology,, engineering,, computer, science,, and, bioinformatics., next, generation, sequencing, features, the, ability, to, sequence, billions, of, base, pairs, of, dna, in, a, single, run,, at, a, cost, that, is, several, orders, of, magnitude, less, than, previous, gold, standard, sequencing, technologies., next, generation, sequencing, is, an, extremely, versatile, technology, and, can, be, used, to, investigate, a, multitude, of, genomic, processes, in, a, previously, unimaginable, genome-wide, and, unbiased, fashion., in, order, to, facilitate, widespread, adoption, of, this, relatively, new, technology,, rather, than, administering, the, core, as, a, 'black-box', facility,, the, core, operates, as, a, 'collaborative, core', in, which, users, can, consult, and, collaborate, with, ngsc, faculty, and, staff, to:, 1), identify, the, optimal, molecular, biology, approaches, (prior, to, sequencing, library, construction), to, synapse, their, research, questions, with, the, capabilities, of, the, core,, 2), prepare, the, appropriate, libraries, and, carry, out, massively, parallel, sequencing,, and, 3), carry, out, the, appropriate, primary,, secondary,, and, tertiary, bioinformatics, analyses, to, analyze, and, interpret, the, results., the, ngsc, features, applied, biosystems, soud, next, generation, sequencing, instruments, capable, of, generating, nearly, 100, gigabases, of, sequencing, data, in, a, single, run,, state-of-the-art, equipment, for, sequencing, library, preparation, and, quality, control,, and, powerful, computational, resources, including, a, server, with, ten, compute, nodes, (each, containing, quad, core, intel, xeon, processors, and, 16, gigabytes, of, memory), and, 100, terabytes, of, storage, for, dedicated, bioinformatics, analyses., the, core, is, currently, administered, by, a, faculty, director, overseeing, all, activities,, a, faculty, co-director, overseeing, bioinformatics, support,, a, laboratory, manager,, and, a, bioinformatics, manager., the, core's, goal, is, to, be, as, flexible, as, possible, in, assisting, researchers, in, exploring, all, aspects, of, cancer, genetics, and, epigenetics,, including,, but, not, limited, to,, germline, and, somatic, variation/mutation,, genomic, structural, variations/alterations,, transcriptome, analysis,, transcription, factor, binding, sites,, chromatin, modifications,, and, dna, methylation., lay:, the, next, generation, sequencing, core, provides, cutting, edge, services, that, harness, the, most, recent, technological, advances, to, allow, large, scale, genomic, sequencing, of, many, hundreds, of, millions, of, dna, molecules, in, parallel, at, about, 100,000, times, less, cost, than, previous, sequencing, technologies., these, services, are, poised, to, rapidly, accelerate, the, pace, of, discovery, and, clinical, translation, in, cancer, molecular, genetics, research., skccc, managed, shared, resource]                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              |408   |\n",
      "|Pharmacogenomics began with a focus on the functional impact of a small number of variants in a single drug metabolism gene. As the field matures it has become clear that advances in understanding the genetic basis for drug effect must take a look beyond the genes that we expect to be involved in a drug pathway. The coincident completion of analysis on pharmacogenetic studies using genome-wide association (GWA) or other approaches to identify commonly altered genomic regions, and the ascendance of next-generation sequencing and analysis methods provides a compelling framework by which we can anticipate increasing the knowledge of rare mutations in affected cohorts. This is especially challenging in diseases such as cancer, heart failure, or mental retardation syndromes, where complex variation in copy number and sequence occur simultaneously within a given gene. Targeting these genomic regions of interest is a technology exercise that has been developing over the past several years, and typically involves the following steps once these regions have been identified; 1) designing primers or probes to the genes in the regions or interest or tiled across the entire locus/loci, 2) carrying out the selected targeting method using these probes, either by PCR or capture, 3) isolating the DNA that results, 4) sequencing the DNA using one of three commercially available next-generation sequencing platforms, and 5) aligning the resulting reads to the human reference genome and analyzing the aligned reads to identify single nucleotide or indel variants specific to the affected cohort. Downstream functional studies may further enhance our understanding of how these variants play a role in disease susceptibility, onset or severity. As an alternative to, or in concert with these targeted sequencing studies, light sequence coverage (4X) of the entire genome with paired end sequencing reads can be accomplished quickly and inexpensively by next generation sequencing platforms. When coupled with careful analysis of read pair mapping, these reads can be used to identify regions of amplification or deletion (copy number alteration or CNA), and other structural variations (inversions, cryptic translocations) that may be vital to the understanding of pharmacogenetic traits. Several years ago, in anticipation that targeted re-sequencing would be applicable to mutation discovery in cancer samples, our laboratory developed and scaled to high-throughput, PCR-based capabilities for this purpose. These capabilities include the full suite of activities needed for a 'pipeline', including automated PCR primer design, primer validation, and 384 well automated PCR reaction assembly, post-reaction processing and sequencing assembly at low volume to minimize DNA usage and cost. Commensurate with the need to analyze the capillary traces resulting from these efforts, we developed an automated analysis pipeline capable of identifying high quality single-nucleotide and indel variants, predicting the impact of the identified mutations on protein structure, and databasing the resulting information for downstream analyses such as correlation of mutational profiles across samples, correlation to clinical data elements, and so on. The use ofthis pipeline has resulted in an expansion of our understanding of mutations occurring in lung adenocarcinoma, acute myeloid leukemia, and glioblastoma multiforme.  Coincident with our PCR pipeline development work, we also began investigating targeted capture by solution-phase hybridization. Based on our early success, and taking advantage of new technology such as the capability to produce up to 55,000 oligos that could be cleaved from the synthesis substrate (Agilent) as well as the massively parallel sequencing capability of next-generation platforms, we developed a novel capture method based on our original BAC-based capture. We refer to this method as 'WUCap' (Washington University Capture), and now have incorporated it into the sequencing production activities at our center. This approach has been applied to targeted capture and sequencing in the 1,000 Genomes project (pilot 3 studies), to the sequencing of 6,000 genes in ovarian cancer samples for The Cancer Genome Atlas (TCGA), and will soon be applied to a set of 500 genes suspected in retinitis pigmentosa, as well as to our NHGRI-sponsored medical sequencing project to investigate GWAS peaks resulting from case/control studies of metabolic syndrome in over 7,000 genomes.|[pharmacogenomics, began, with, a, focus, on, the, functional, impact, of, a, small, number, of, variants, in, a, single, drug, metabolism, gene., as, the, field, matures, it, has, become, clear, that, advances, in, understanding, the, genetic, basis, for, drug, effect, must, take, a, look, beyond, the, genes, that, we, expect, to, be, involved, in, a, drug, pathway., the, coincident, completion, of, analysis, on, pharmacogenetic, studies, using, genome-wide, association, (gwa), or, other, approaches, to, identify, commonly, altered, genomic, regions,, and, the, ascendance, of, next-generation, sequencing, and, analysis, methods, provides, a, compelling, framework, by, which, we, can, anticipate, increasing, the, knowledge, of, rare, mutations, in, affected, cohorts., this, is, especially, challenging, in, diseases, such, as, cancer,, heart, failure,, or, mental, retardation, syndromes,, where, complex, variation, in, copy, number, and, sequence, occur, simultaneously, within, a, given, gene., targeting, these, genomic, regions, of, interest, is, a, technology, exercise, that, has, been, developing, over, the, past, several, years,, and, typically, involves, the, following, steps, once, these, regions, have, been, identified;, 1), designing, primers, or, probes, to, the, genes, in, the, regions, or, interest, or, tiled, across, the, entire, locus/loci,, 2), carrying, out, the, selected, targeting, method, using, these, probes,, either, by, pcr, or, capture,, 3), isolating, the, dna, that, results,, 4), sequencing, the, dna, using, one, of, three, commercially, available, next-generation, sequencing, platforms,, and, 5), aligning, the, resulting, reads, to, the, human, reference, genome, and, analyzing, the, aligned, reads, to, identify, single, nucleotide, or, indel, variants, specific, to, the, affected, cohort., downstream, functional, studies, may, further, enhance, our, understanding, of, how, these, variants, play, a, role, in, disease, susceptibility,, onset, or, severity., as, an, alternative, to,, or, in, concert, with, these, targeted, sequencing, studies,, light, sequence, coverage, (4x), of, the, entire, genome, with, paired, end, sequencing, reads, can, be, accomplished, quickly, and, inexpensively, by, next, generation, sequencing, platforms., when, coupled, with, careful, analysis, of, read, pair, mapping,, these, reads, can, be, used, to, identify, regions, of, amplification, or, deletion, (copy, number, alteration, or, cna),, and, other, structural, variations, (inversions,, cryptic, translocations), that, may, be, vital, to, the, understanding, of, pharmacogenetic, traits., several, years, ago,, in, anticipation, that, targeted, re-sequencing, would, be, applicable, to, mutation, discovery, in, cancer, samples,, our, laboratory, developed, and, scaled, to, high-throughput,, pcr-based, capabilities, for, this, purpose., these, capabilities, include, the, full, suite, of, activities, needed, for, a, 'pipeline',, including, automated, pcr, primer, design,, primer, validation,, and, 384, well, automated, pcr, reaction, assembly,, post-reaction, processing, and, sequencing, assembly, at, low, volume, to, minimize, dna, usage, and, cost., commensurate, with, the, need, to, analyze, the, capillary, traces, resulting, from, these, efforts,, we, developed, an, automated, analysis, pipeline, capable, of, identifying, high, quality, single-nucleotide, and, indel, variants,, predicting, the, impact, of, the, identified, mutations, on, protein, structure,, and, databasing, the, resulting, information, for, downstream, analyses, such, as, correlation, of, mutational, profiles, across, samples,, correlation, to, clinical, data, elements,, and, so, on., the, use, ofthis, pipeline, has, resulted, in, an, expansion, of, our, understanding, of, mutations, occurring, in, lung, adenocarcinoma,, acute, myeloid, leukemia,, and, glioblastoma, multiforme., , coincident, with, our, pcr, pipeline, development, work,, we, also, began, investigating, targeted, capture, by, solution-phase, hybridization., based, on, our, early, success,, and, taking, advantage, of, new, technology, such, as, the, capability, to, produce, up, to, 55,000, oligos, that, could, be, cleaved, from, the, synthesis, substrate, (agilent), as, well, as, the, massively, parallel, sequencing, capability, of, next-generation, platforms,, we, developed, a, novel, capture, method, based, on, our, original, bac-based, capture., we, refer, to, this, method, as, 'wucap', (washington, university, capture),, and, now, have, incorporated, it, into, the, sequencing, production, activities, at, our, center., this, approach, has, been, applied, to, targeted, capture, and, sequencing, in, the, 1,000, genomes, project, (pilot, 3, studies),, to, the, sequencing, of, 6,000, genes, in, ovarian, cancer, samples, for, the, cancer, genome, atlas, (tcga),, and, will, soon, be, applied, to, a, set, of, 500, genes, suspected, in, retinitis, pigmentosa,, as, well, as, to, our, nhgri-sponsored, medical, sequencing, project, to, investigate, gwas, peaks, resulting, from, case/control, studies, of, metabolic, syndrome, in, over, 7,000, genomes.]|666   |\n",
      "|Next Generation Sequencing and Genotyping Core Facility. The goal of the Next Gen Sequencing component is to make 'next generation' sequencing and related services widely available to UNC LCCC members at cost-effective prices. Services include: whole genome shotgun sequencing of human and model organisms, chromatin-IP sample sequencing, measuring RNA expression with both digital gene expression and RNA-seq, SNP typing, and polymorphism screens. The component adds value to the cancer center by putting complex and expensive DNA/RNA sequencing technology within easy reach of UNC LCCC members. Strongly integrated genomics and bioinformatics groups interact with the user to ensure robust data that is archived appropriately for future use. This component is key in UNC LCCC's participation in The Cancer Genome Atlas project and serving LCCC needs. Future plans Include expanding staff and equipment so that a full array of Next Gen sequencing and services may be provided in a timely manner, as well as the acquisition of the next generation of technology.   The goal of the DNA Sequencing component is to produce high quality sequencing and genotyping data rapidly and at a reasonable cost for UNC-CH researchers using state-of-the-art technologies as well as provide technical support to enhance the value of results produced. Services include DNA sequencing and microsatellite genotyping. The component adds value to the cancer center by producing high-quality data at a competitive price while providing the benefits of a local resource to assist in: scientific strategies, sample preparation, sequencing through difficult regions, and assistance in data interpretation. This component has a wide user base with over 220 different laboratories as users in 2009. Future plans include continued use of existing equipment, reduction of chemistry costs, and evaluation of future sequencing technology options such as long read single molecule sequencing technologies. The goal of the High Throughput Genotyping component is to produce high-quality genotyping data rapidly and at a reasonable cost for researchers using state-of-the-art technologies. Services include SNP genotyping, copy number variation and DNA methylation profiling analysis. New technologies and/or applications are being explored to increase the core's competitive edge and to allow for Genome Wide Association Study on rare variants. These core components are used by multiple members and in each core, peer reviewed member used exceeds 70%. The proposed budget of $241,797 is less than 5% of the core's operating cost. The budget provides salary stability for vital core staff who work with members to adopt these novel technologies.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |[next, generation, sequencing, and, genotyping, core, facility., the, goal, of, the, next, gen, sequencing, component, is, to, make, 'next, generation', sequencing, and, related, services, widely, available, to, unc, lccc, members, at, cost-effective, prices., services, include:, whole, genome, shotgun, sequencing, of, human, and, model, organisms,, chromatin-ip, sample, sequencing,, measuring, rna, expression, with, both, digital, gene, expression, and, rna-seq,, snp, typing,, and, polymorphism, screens., the, component, adds, value, to, the, cancer, center, by, putting, complex, and, expensive, dna/rna, sequencing, technology, within, easy, reach, of, unc, lccc, members., strongly, integrated, genomics, and, bioinformatics, groups, interact, with, the, user, to, ensure, robust, data, that, is, archived, appropriately, for, future, use., this, component, is, key, in, unc, lccc's, participation, in, the, cancer, genome, atlas, project, and, serving, lccc, needs., future, plans, include, expanding, staff, and, equipment, so, that, a, full, array, of, next, gen, sequencing, and, services, may, be, provided, in, a, timely, manner,, as, well, as, the, acquisition, of, the, next, generation, of, technology., , , the, goal, of, the, dna, sequencing, component, is, to, produce, high, quality, sequencing, and, genotyping, data, rapidly, and, at, a, reasonable, cost, for, unc-ch, researchers, using, state-of-the-art, technologies, as, well, as, provide, technical, support, to, enhance, the, value, of, results, produced., services, include, dna, sequencing, and, microsatellite, genotyping., the, component, adds, value, to, the, cancer, center, by, producing, high-quality, data, at, a, competitive, price, while, providing, the, benefits, of, a, local, resource, to, assist, in:, scientific, strategies,, sample, preparation,, sequencing, through, difficult, regions,, and, assistance, in, data, interpretation., this, component, has, a, wide, user, base, with, over, 220, different, laboratories, as, users, in, 2009., future, plans, include, continued, use, of, existing, equipment,, reduction, of, chemistry, costs,, and, evaluation, of, future, sequencing, technology, options, such, as, long, read, single, molecule, sequencing, technologies., the, goal, of, the, high, throughput, genotyping, component, is, to, produce, high-quality, genotyping, data, rapidly, and, at, a, reasonable, cost, for, researchers, using, state-of-the-art, technologies., services, include, snp, genotyping,, copy, number, variation, and, dna, methylation, profiling, analysis., new, technologies, and/or, applications, are, being, explored, to, increase, the, core's, competitive, edge, and, to, allow, for, genome, wide, association, study, on, rare, variants., these, core, components, are, used, by, multiple, members, and, in, each, core,, peer, reviewed, member, used, exceeds, 70%., the, proposed, budget, of, $241,797, is, less, than, 5%, of, the, core's, operating, cost., the, budget, provides, salary, stability, for, vital, core, staff, who, work, with, members, to, adopt, these, novel, technologies.]                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          |404   |\n",
      "|Project Summary/Abstract Mobile DNA elements are DNA segments that can mobilize from one genomic location to another,some making copies of themselves during the process. Mobile elements are ubiquitous in eukaryotic genomes,and roughly 50% of the human genome consists of mobile, repetitive DNA sequences such as Alu and LINE1elements. Mobile elements have profoundly influenced primate genomic evolution, and alterations caused bymobile elements have been identified in more than 100 human genetic disorders. In addition, mobile elementsare highly useful as genetic markers in tracing relationships of populations and species because of their uniquemanner of propagating in the genome. Therefore, it is of great interest to understand the biology of mobileelements in human populations. Currently available genotyping methods, however, provide only limited insightinto the distribution of Alu and L1 elements in the human genome. In this proposal, I will first describe a novel, high-throughput method to genotype a large number ofindividuals for Alu insertion polymorphisms. Implementation of this method will permit, for the first time, rapid,inexpensive genotyping of whole families of mobile elements not only in humans but in any organism. Next, Ioutline my plans to construct a computational pipeline that can efficiently handle and analyze the high-throughput genomic data generated by my genotyping method. Finally I describe my plan to apply thegenotyping procedure to 540 HapMap samples and use my computational pipeline to analyze the results.Specifically, I will use the insertion polymorphism data to assess the genomic diversity associated with mobileelement insertions, as well as to answer questions related to mobile element biology and to reconstruct ancienthuman demographic history. In addition, the Alu insertion polymorphism dataset generated in this project willprovide a valuable supplementary resource of structural variants for the HapMap and 1000 Genomes Projects. My long-term goal is to pursue a career in human genomic variation and population genetics researchwith an emphasis on mobile elements. Using a combination of computational and experimental tools, Iwill study the impact of mobile DNA elements on the human genome and on genetic diversity among humanpopulations. This award will provide the necessary training for me to become an independent investigator ontwo fronts: I will acquire essential training in computational biology and bioinformatics to handle large-scalegenomic data, including data generated from next-generation sequencing technology. In addition, researchtraining offered by the institution and my mentor will allow me to acquire knowledge about research resources,grant writing, responsible conduct in research, and other information necessary to become a successful,independent investigator.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        |[project, summary/abstract, mobile, dna, elements, are, dna, segments, that, can, mobilize, from, one, genomic, location, to, another,some, making, copies, of, themselves, during, the, process., mobile, elements, are, ubiquitous, in, eukaryotic, genomes,and, roughly, 50%, of, the, human, genome, consists, of, mobile,, repetitive, dna, sequences, such, as, alu, and, line1elements., mobile, elements, have, profoundly, influenced, primate, genomic, evolution,, and, alterations, caused, bymobile, elements, have, been, identified, in, more, than, 100, human, genetic, disorders., in, addition,, mobile, elementsare, highly, useful, as, genetic, markers, in, tracing, relationships, of, populations, and, species, because, of, their, uniquemanner, of, propagating, in, the, genome., therefore,, it, is, of, great, interest, to, understand, the, biology, of, mobileelements, in, human, populations., currently, available, genotyping, methods,, however,, provide, only, limited, insightinto, the, distribution, of, alu, and, l1, elements, in, the, human, genome., in, this, proposal,, i, will, first, describe, a, novel,, high-throughput, method, to, genotype, a, large, number, ofindividuals, for, alu, insertion, polymorphisms., implementation, of, this, method, will, permit,, for, the, first, time,, rapid,inexpensive, genotyping, of, whole, families, of, mobile, elements, not, only, in, humans, but, in, any, organism., next,, ioutline, my, plans, to, construct, a, computational, pipeline, that, can, efficiently, handle, and, analyze, the, high-throughput, genomic, data, generated, by, my, genotyping, method., finally, i, describe, my, plan, to, apply, thegenotyping, procedure, to, 540, hapmap, samples, and, use, my, computational, pipeline, to, analyze, the, results.specifically,, i, will, use, the, insertion, polymorphism, data, to, assess, the, genomic, diversity, associated, with, mobileelement, insertions,, as, well, as, to, answer, questions, related, to, mobile, element, biology, and, to, reconstruct, ancienthuman, demographic, history., in, addition,, the, alu, insertion, polymorphism, dataset, generated, in, this, project, willprovide, a, valuable, supplementary, resource, of, structural, variants, for, the, hapmap, and, 1000, genomes, projects., my, long-term, goal, is, to, pursue, a, career, in, human, genomic, variation, and, population, genetics, researchwith, an, emphasis, on, mobile, elements., using, a, combination, of, computational, and, experimental, tools,, iwill, study, the, impact, of, mobile, dna, elements, on, the, human, genome, and, on, genetic, diversity, among, humanpopulations., this, award, will, provide, the, necessary, training, for, me, to, become, an, independent, investigator, ontwo, fronts:, i, will, acquire, essential, training, in, computational, biology, and, bioinformatics, to, handle, large-scalegenomic, data,, including, data, generated, from, next-generation, sequencing, technology., in, addition,, researchtraining, offered, by, the, institution, and, my, mentor, will, allow, me, to, acquire, knowledge, about, research, resources,grant, writing,, responsible, conduct, in, research,, and, other, information, necessary, to, become, a, successful,independent, investigator.]                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  |400   |\n",
      "|DESCRIPTION (provided by applicant): Modern biomedical research is increasingly making use of genome-scale from next-generation sequencing platforms, including Roche 454, Illumina GA2, and ABI SOLiD. These platforms make it possible for individual labs to quickly and cheaply generate vast amounts of genomic and transcriptomic data from de novo sequencing, resequencing, ChIP-seq, mRNA-seq, and allelotyping experiments. Despite this ability to generate large data sets, biomedical researchers are rarely trained in the computational and statistical techniques necessary to make sense of this data. Thus, many researchers must rely on others - often computational scientists with little biological training - to design and implement appropriate data reduction and data mining techniques. Moreover, most institutions do not have access to computational resources necessary to run these analyses. Our specific aims are to help bridge this gap in a short, two-week course, by teaching biomedical researchers to (1) run analyses on remote UNIX servers hosted in the Amazon Web Services 'cloud'; (2) perform mapping and assembly on large short-read data sets; (3) tackle specific biological problems with existing short-read data; and (4) design computational pipelines capable of addressing their own research questions. All specific aims will be accompanied by in-depth hands-on practical training in the relevant techniques. Our experience is that this practical training leads to a substantial improvement in the basic computational sophistication of participants. This short course will help train the current and next generation of independent biomedical re- searchers in basic computational thinking and procedure, as well as teaching them how to make use of scalable Internet computing resources for their own research. Our end goal is increase the efficiency and sophistication with which biomedical researchers make use of novel sequencing technologies.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |[description, (provided, by, applicant):, modern, biomedical, research, is, increasingly, making, use, of, genome-scale, from, next-generation, sequencing, platforms,, including, roche, 454,, illumina, ga2,, and, abi, solid., these, platforms, make, it, possible, for, individual, labs, to, quickly, and, cheaply, generate, vast, amounts, of, genomic, and, transcriptomic, data, from, de, novo, sequencing,, resequencing,, chip-seq,, mrna-seq,, and, allelotyping, experiments., despite, this, ability, to, generate, large, data, sets,, biomedical, researchers, are, rarely, trained, in, the, computational, and, statistical, techniques, necessary, to, make, sense, of, this, data., thus,, many, researchers, must, rely, on, others, -, often, computational, scientists, with, little, biological, training, -, to, design, and, implement, appropriate, data, reduction, and, data, mining, techniques., moreover,, most, institutions, do, not, have, access, to, computational, resources, necessary, to, run, these, analyses., our, specific, aims, are, to, help, bridge, this, gap, in, a, short,, two-week, course,, by, teaching, biomedical, researchers, to, (1), run, analyses, on, remote, unix, servers, hosted, in, the, amazon, web, services, 'cloud';, (2), perform, mapping, and, assembly, on, large, short-read, data, sets;, (3), tackle, specific, biological, problems, with, existing, short-read, data;, and, (4), design, computational, pipelines, capable, of, addressing, their, own, research, questions., all, specific, aims, will, be, accompanied, by, in-depth, hands-on, practical, training, in, the, relevant, techniques., our, experience, is, that, this, practical, training, leads, to, a, substantial, improvement, in, the, basic, computational, sophistication, of, participants., this, short, course, will, help, train, the, current, and, next, generation, of, independent, biomedical, re-, searchers, in, basic, computational, thinking, and, procedure,, as, well, as, teaching, them, how, to, make, use, of, scalable, internet, computing, resources, for, their, own, research., our, end, goal, is, increase, the, efficiency, and, sophistication, with, which, biomedical, researchers, make, use, of, novel, sequencing, technologies.]                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |280   |\n",
      "|PROJECT SUMMARY/ABSTRACT The impact of high-throughput genomics technologies on personalized medicine and novel cancertreatments will ultimately be determined by the ability to interpret the data accordingly. Therefore, theapplicant's long-term objective is to translate genome-based knowledge, through integrative analysis of high-throughput genomic technologies, into the development of novel diagnostic, prognostic, and therapeuticstrategies in cancer. As an initial step towards this endeavor, this proposal focuses on the central hypothesisthat common carcinomas harbor driving recurrent gene fusion mutations that are hidden by non-specificaberrations. We believe that Next Generation Sequencing of cancer transcriptomes may elucidate thesedriving gene fusions and therefore established the following specific aims to accomplish this: (1) employintegrative analysis of genomics data to detect novel gene fusions in ETS negative prostate tumors, (2)further develop the bioinformatic approaches to prioritize casual chimeras and apply these methods inmelanoma and pancreatic cancer, and (3) implement a sequencing based gene- and exon-level expressionoutlier profile analysis to assess the prevalence, and nominate novel, gene fusions in publicly available datacollections. Overall, if the aims of this application are achieved we may reveal novel pathognomomic targets incancer that can be translated into the development of a targeted therapy. Even the successful discovery of asingle target, whether discovered directly from the proposed work or uncovered by another lab leveraging themethodologies developed as a result of this proposal, will have significant impact on how we treat cancer. The Michigan Center for Translational Pathology has in-depth experience using bioinformatics andgenomics for studying gene expression, particularly gene fusions, yielding numerous biomarkers andtherapeutic targets in prostate cancer and therefore serves as an ideal environment for supporting theapplicant. This can be exemplified by the applicant's extensive preliminary data, which has led to multiple highimpact first author manuscripts. Given Dr. Chinnaiyan's excellent record of mentoring biostatisticians andbioinformaticians as they transition into successful independent investigators, he will provide a similar careerdevelopment plan for the current applicant. The first specific aim, to be completed during the Mentored phase,will focus on detecting gene fusions in prostate cancer and therefore offer critical integrative analysis, cancerbiology, and translational research training. The focus of the independent phase will be placed on sequenceanalysis of chimera detection in newly established cancer transcriptome projects and RNA-Seq based outlierexpression analysis to reveal novel therapeutic targets, which will have minimal overlap with, but complements,the mentor's research on prostate gene fusions. Overall, an NIH Pathway to Independence Award will beindispensable for the applicant to become an independent investigator in the much needed field of integrativecancer biology to continue translating genome-based discovering into widely adopting clinical applications.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |[project, summary/abstract, the, impact, of, high-throughput, genomics, technologies, on, personalized, medicine, and, novel, cancertreatments, will, ultimately, be, determined, by, the, ability, to, interpret, the, data, accordingly., therefore,, theapplicant's, long-term, objective, is, to, translate, genome-based, knowledge,, through, integrative, analysis, of, high-throughput, genomic, technologies,, into, the, development, of, novel, diagnostic,, prognostic,, and, therapeuticstrategies, in, cancer., as, an, initial, step, towards, this, endeavor,, this, proposal, focuses, on, the, central, hypothesisthat, common, carcinomas, harbor, driving, recurrent, gene, fusion, mutations, that, are, hidden, by, non-specificaberrations., we, believe, that, next, generation, sequencing, of, cancer, transcriptomes, may, elucidate, thesedriving, gene, fusions, and, therefore, established, the, following, specific, aims, to, accomplish, this:, (1), employintegrative, analysis, of, genomics, data, to, detect, novel, gene, fusions, in, ets, negative, prostate, tumors,, (2)further, develop, the, bioinformatic, approaches, to, prioritize, casual, chimeras, and, apply, these, methods, inmelanoma, and, pancreatic, cancer,, and, (3), implement, a, sequencing, based, gene-, and, exon-level, expressionoutlier, profile, analysis, to, assess, the, prevalence,, and, nominate, novel,, gene, fusions, in, publicly, available, datacollections., overall,, if, the, aims, of, this, application, are, achieved, we, may, reveal, novel, pathognomomic, targets, incancer, that, can, be, translated, into, the, development, of, a, targeted, therapy., even, the, successful, discovery, of, asingle, target,, whether, discovered, directly, from, the, proposed, work, or, uncovered, by, another, lab, leveraging, themethodologies, developed, as, a, result, of, this, proposal,, will, have, significant, impact, on, how, we, treat, cancer., the, michigan, center, for, translational, pathology, has, in-depth, experience, using, bioinformatics, andgenomics, for, studying, gene, expression,, particularly, gene, fusions,, yielding, numerous, biomarkers, andtherapeutic, targets, in, prostate, cancer, and, therefore, serves, as, an, ideal, environment, for, supporting, theapplicant., this, can, be, exemplified, by, the, applicant's, extensive, preliminary, data,, which, has, led, to, multiple, highimpact, first, author, manuscripts., given, dr., chinnaiyan's, excellent, record, of, mentoring, biostatisticians, andbioinformaticians, as, they, transition, into, successful, independent, investigators,, he, will, provide, a, similar, careerdevelopment, plan, for, the, current, applicant., the, first, specific, aim,, to, be, completed, during, the, mentored, phase,will, focus, on, detecting, gene, fusions, in, prostate, cancer, and, therefore, offer, critical, integrative, analysis,, cancerbiology,, and, translational, research, training., the, focus, of, the, independent, phase, will, be, placed, on, sequenceanalysis, of, chimera, detection, in, newly, established, cancer, transcriptome, projects, and, rna-seq, based, outlierexpression, analysis, to, reveal, novel, therapeutic, targets,, which, will, have, minimal, overlap, with,, but, complements,the, mentor's, research, on, prostate, gene, fusions., overall,, an, nih, pathway, to, independence, award, will, beindispensable, for, the, applicant, to, become, an, independent, investigator, in, the, much, needed, field, of, integrativecancer, biology, to, continue, translating, genome-based, discovering, into, widely, adopting, clinical, applications.]                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              |418   |\n",
      "|Recently, the emerging new field of metagenomics facilitated by the advent of next-generation sequencing technology enables genome sequencing of unculturable and often unknown microbes in natural environments, offering researchers an unprecedented opportunity to delineate bio-diversity of any microbial organism. Mining single nucleotide polymorphisms (SNPs) from metagenomic sequencing data offers an unique opportunity to rapidly and accurately detect known or novel strains related to multiple biothreat agents. While the sequencing technologies are evolving at unprecedented speed, researchers engaged in this enterprise are facing major computational, algorithmic and statistical challenges in the analysis of the massive metagenomic data. It is clear that both current analytical and computational methods are inadequate for this challenge. In this project, the investigator and his colleagues develop a family of statistically sound and computationally efficient algorithms to detect SNPs from metagenomic data to characterize microbial diversity in natural environments. <br/><br/>The proposed project provides the national security and biodefense agencies new tools for rapid and accurate detection of biothreat agents. It also provides researchers in microbiology with new tools for producing abundant, high throughput SNPs for detailed analysis of the genetic basis of microbial diversity and evolution. Since this informatics tool can be used to study a wide variety of microbial communities, it helps accelerating scientific advancements of our knowledge in microbiology and evolution. The multidisciplinary nature of the project will promote collaboration between biologists, computer scientists and statistician. The multidisciplinary nature of the project will also provide postdoctoral fellows and graduate students training in statistics, genomics and scientific computing through hands-on experience.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           |[recently,, the, emerging, new, field, of, metagenomics, facilitated, by, the, advent, of, next-generation, sequencing, technology, enables, genome, sequencing, of, unculturable, and, often, unknown, microbes, in, natural, environments,, offering, researchers, an, unprecedented, opportunity, to, delineate, bio-diversity, of, any, microbial, organism., mining, single, nucleotide, polymorphisms, (snps), from, metagenomic, sequencing, data, offers, an, unique, opportunity, to, rapidly, and, accurately, detect, known, or, novel, strains, related, to, multiple, biothreat, agents., while, the, sequencing, technologies, are, evolving, at, unprecedented, speed,, researchers, engaged, in, this, enterprise, are, facing, major, computational,, algorithmic, and, statistical, challenges, in, the, analysis, of, the, massive, metagenomic, data., it, is, clear, that, both, current, analytical, and, computational, methods, are, inadequate, for, this, challenge., in, this, project,, the, investigator, and, his, colleagues, develop, a, family, of, statistically, sound, and, computationally, efficient, algorithms, to, detect, snps, from, metagenomic, data, to, characterize, microbial, diversity, in, natural, environments., <br/><br/>the, proposed, project, provides, the, national, security, and, biodefense, agencies, new, tools, for, rapid, and, accurate, detection, of, biothreat, agents., it, also, provides, researchers, in, microbiology, with, new, tools, for, producing, abundant,, high, throughput, snps, for, detailed, analysis, of, the, genetic, basis, of, microbial, diversity, and, evolution., since, this, informatics, tool, can, be, used, to, study, a, wide, variety, of, microbial, communities,, it, helps, accelerating, scientific, advancements, of, our, knowledge, in, microbiology, and, evolution., the, multidisciplinary, nature, of, the, project, will, promote, collaboration, between, biologists,, computer, scientists, and, statistician., the, multidisciplinary, nature, of, the, project, will, also, provide, postdoctoral, fellows, and, graduate, students, training, in, statistics,, genomics, and, scientific, computing, through, hands-on, experience.]                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |255   |\n",
      "|Recently, the emerging new field of metagenomics facilitated by the advent of next-generation sequencing technology enables genome sequencing of unculturable and often unknown microbes in natural environments, offering researchers an unprecedented opportunity to delineate bio-diversity of any microbial organism. Mining single nucleotide polymorphisms (SNPs) from metagenomic sequencing data offers an unique opportunity to rapidly and accurately detect known or novel strains related to multiple biothreat agents. While the sequencing technologies are evolving at unprecedented speed, researchers engaged in this enterprise are facing major computational, algorithmic and statistical challenges in the analysis of the massive metagenomic data. It is clear that both current analytical and computational methods are inadequate for this challenge. In this project, the investigator and his colleagues develop a family of statistically sound and computationally efficient algorithms to detect SNPs from metagenomic data to characterize microbial diversity in natural environments. <br/><br/>The proposed project provides the national security and biodefense agencies new tools for rapid and accurate detection of biothreat agents. It also provides researchers in microbiology with new tools for producing abundant, high throughput SNPs for detailed analysis of the genetic basis of microbial diversity and evolution. Since this informatics tool can be used to study a wide variety of microbial communities, it helps accelerating scientific advancements of our knowledge in microbiology and evolution. The multidisciplinary nature of the project will promote collaboration between biologists, computer scientists and statistician. The multidisciplinary nature of the project will also provide postdoctoral fellows and graduate students training in statistics, genomics and scientific computing through hands-on experience.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           |[recently,, the, emerging, new, field, of, metagenomics, facilitated, by, the, advent, of, next-generation, sequencing, technology, enables, genome, sequencing, of, unculturable, and, often, unknown, microbes, in, natural, environments,, offering, researchers, an, unprecedented, opportunity, to, delineate, bio-diversity, of, any, microbial, organism., mining, single, nucleotide, polymorphisms, (snps), from, metagenomic, sequencing, data, offers, an, unique, opportunity, to, rapidly, and, accurately, detect, known, or, novel, strains, related, to, multiple, biothreat, agents., while, the, sequencing, technologies, are, evolving, at, unprecedented, speed,, researchers, engaged, in, this, enterprise, are, facing, major, computational,, algorithmic, and, statistical, challenges, in, the, analysis, of, the, massive, metagenomic, data., it, is, clear, that, both, current, analytical, and, computational, methods, are, inadequate, for, this, challenge., in, this, project,, the, investigator, and, his, colleagues, develop, a, family, of, statistically, sound, and, computationally, efficient, algorithms, to, detect, snps, from, metagenomic, data, to, characterize, microbial, diversity, in, natural, environments., <br/><br/>the, proposed, project, provides, the, national, security, and, biodefense, agencies, new, tools, for, rapid, and, accurate, detection, of, biothreat, agents., it, also, provides, researchers, in, microbiology, with, new, tools, for, producing, abundant,, high, throughput, snps, for, detailed, analysis, of, the, genetic, basis, of, microbial, diversity, and, evolution., since, this, informatics, tool, can, be, used, to, study, a, wide, variety, of, microbial, communities,, it, helps, accelerating, scientific, advancements, of, our, knowledge, in, microbiology, and, evolution., the, multidisciplinary, nature, of, the, project, will, promote, collaboration, between, biologists,, computer, scientists, and, statistician., the, multidisciplinary, nature, of, the, project, will, also, provide, postdoctoral, fellows, and, graduate, students, training, in, statistics,, genomics, and, scientific, computing, through, hands-on, experience.]                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |255   |\n",
      "|DESCRIPTION (provided by applicant): Next-generation sequencing technologies are capable of producing tens of millions of sequence reads during each instrument run, and are quickly being applied in diverse types of experiments (e.g. RNA-Seq, miRNA-Seq, ChIP-Seq, BS-seq, CNV-Seq) to address biomedical questions by cost-effectively generating genome-wide datasets. While sequencing has been promoted as overcoming longstanding limitations of microarray-based studies, its data files are much larger than for microarrays, and its diverse data types raise similar as well as novel statistical and computational challenges. There is a pressing need for statistical and computational tools to address what leaders in the field have stated are the largest problems: data analysis and data integration. We propose to develop a comprehensive and coordinated set of statistical methods for high throughput sequencing (HTS) that directly address many important data analysis problems in epigenomics. Specifically we plan to address the following computational and statistical challenges facing researchers conducting HTS experiments: 1) develop sensitive statistical methods for the analysis of ChIP-seq data both for single- and paired-end-tag runs, particularly the focusing on applications in genome-wide profiling of nucleosome positions. 2) develop statistical methods for the analysis of BS-seq data, producing base-level DNA methylation profiles. 3) develop new statistical tools and methods for data integration in order to gain new biological insights about global transcription and regulation. We also plan to apply these approaches to a variety of high throughput sequencing data sets to demonstrate the relevance and utility of our methods. We plan to work with stimulated STAT1 and STAT3 data, and data from the ETS transcription factor family and its cofactors, for which we have already gathered significant data through our collaborations, including transcription factors, histone marks, DNAse I hypersensitivity and gene expression.  PUBLIC HEALTH RELEVANCE: We propose to develop a comprehensive and coordinated set of statistical methods for high throughput sequencing (HTS) that directly address many important data analysis problems in epigenomics. In particular, we plan to integrate data from multiple sources including expression, transcription factor binding, nucleosome positioning, histone marks and DNA methylation to better understand the mechanisms that regulate the behavior of a cell. Much of our proposal involves not just the development of new statistical and computational methods, but also the design, implementation and delivery of software tools that support these ideas. The many useful applications of next-generation sequencing with assure that or well- developed methods will have a broad impact in molecular biology, specifically in transcription regulation, chromatin dynamics, development, and cancer.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 |[description, (provided, by, applicant):, next-generation, sequencing, technologies, are, capable, of, producing, tens, of, millions, of, sequence, reads, during, each, instrument, run,, and, are, quickly, being, applied, in, diverse, types, of, experiments, (e.g., rna-seq,, mirna-seq,, chip-seq,, bs-seq,, cnv-seq), to, address, biomedical, questions, by, cost-effectively, generating, genome-wide, datasets., while, sequencing, has, been, promoted, as, overcoming, longstanding, limitations, of, microarray-based, studies,, its, data, files, are, much, larger, than, for, microarrays,, and, its, diverse, data, types, raise, similar, as, well, as, novel, statistical, and, computational, challenges., there, is, a, pressing, need, for, statistical, and, computational, tools, to, address, what, leaders, in, the, field, have, stated, are, the, largest, problems:, data, analysis, and, data, integration., we, propose, to, develop, a, comprehensive, and, coordinated, set, of, statistical, methods, for, high, throughput, sequencing, (hts), that, directly, address, many, important, data, analysis, problems, in, epigenomics., specifically, we, plan, to, address, the, following, computational, and, statistical, challenges, facing, researchers, conducting, hts, experiments:, 1), develop, sensitive, statistical, methods, for, the, analysis, of, chip-seq, data, both, for, single-, and, paired-end-tag, runs,, particularly, the, focusing, on, applications, in, genome-wide, profiling, of, nucleosome, positions., 2), develop, statistical, methods, for, the, analysis, of, bs-seq, data,, producing, base-level, dna, methylation, profiles., 3), develop, new, statistical, tools, and, methods, for, data, integration, in, order, to, gain, new, biological, insights, about, global, transcription, and, regulation., we, also, plan, to, apply, these, approaches, to, a, variety, of, high, throughput, sequencing, data, sets, to, demonstrate, the, relevance, and, utility, of, our, methods., we, plan, to, work, with, stimulated, stat1, and, stat3, data,, and, data, from, the, ets, transcription, factor, family, and, its, cofactors,, for, which, we, have, already, gathered, significant, data, through, our, collaborations,, including, transcription, factors,, histone, marks,, dnase, i, hypersensitivity, and, gene, expression., , public, health, relevance:, we, propose, to, develop, a, comprehensive, and, coordinated, set, of, statistical, methods, for, high, throughput, sequencing, (hts), that, directly, address, many, important, data, analysis, problems, in, epigenomics., in, particular,, we, plan, to, integrate, data, from, multiple, sources, including, expression,, transcription, factor, binding,, nucleosome, positioning,, histone, marks, and, dna, methylation, to, better, understand, the, mechanisms, that, regulate, the, behavior, of, a, cell., much, of, our, proposal, involves, not, just, the, development, of, new, statistical, and, computational, methods,, but, also, the, design,, implementation, and, delivery, of, software, tools, that, support, these, ideas., the, many, useful, applications, of, next-generation, sequencing, with, assure, that, or, well-, developed, methods, will, have, a, broad, impact, in, molecular, biology,, specifically, in, transcription, regulation,, chromatin, dynamics,, development,, and, cancer.]                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                |411   |\n",
      "|During the past 10 years, there has been a major revolution in the biological/life sciences as a result of the remarkable development and subsequent generation of a vast quantity of data from genome sequencing efforts. Genome sequencing is a process that determines the identity and order of an organism's DNA; thus providing a blueprint for the genetic code underlying all of the processes of life.   This has been highlighted by the recently completed draft sequence of the human genome, a notable technical achievement made possible by the development of powerful computational hardware and software.  The availability of complete genome sequences is providing the foundation for understanding how organisms (from microbes to humans) function, develop, evolve and interact with pathogens and assorted environmental cues.  This has led to a transformational change in the field of biology stemming from the development of next-generation ultra high-throughput, ultra low-cost DNA sequencing technology ( e.g. Roche 454), which serves as the basis for this proposal. What used to take several months, can now be done in several hours. The Roche 454 FLX will be used by more than 60 faculty members and their research staff, including post-doctoral fellows, graduate and undergraduate students at Texas A&M University.  Applications and research programs include:  dissection of sorghum drought tolerance, functional and metagenomic analysis of biological communities for cattle and rumen  bacteria, disease resistance  and  an analysis of microflora in ocean drilling programs, among others.  Dissemination of data and project outcomes will be available on the Institute for Plant Genomics and Biotechnology website (http://ipgb.tamu.edu), research publications, relevant scientific meetings and during our scheduled campus-wide annual meeting.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |[during, the, past, 10, years,, there, has, been, a, major, revolution, in, the, biological/life, sciences, as, a, result, of, the, remarkable, development, and, subsequent, generation, of, a, vast, quantity, of, data, from, genome, sequencing, efforts., genome, sequencing, is, a, process, that, determines, the, identity, and, order, of, an, organism's, dna;, thus, providing, a, blueprint, for, the, genetic, code, underlying, all, of, the, processes, of, life., , , this, has, been, highlighted, by, the, recently, completed, draft, sequence, of, the, human, genome,, a, notable, technical, achievement, made, possible, by, the, development, of, powerful, computational, hardware, and, software., , the, availability, of, complete, genome, sequences, is, providing, the, foundation, for, understanding, how, organisms, (from, microbes, to, humans), function,, develop,, evolve, and, interact, with, pathogens, and, assorted, environmental, cues., , this, has, led, to, a, transformational, change, in, the, field, of, biology, stemming, from, the, development, of, next-generation, ultra, high-throughput,, ultra, low-cost, dna, sequencing, technology, (, e.g., roche, 454),, which, serves, as, the, basis, for, this, proposal., what, used, to, take, several, months,, can, now, be, done, in, several, hours., the, roche, 454, flx, will, be, used, by, more, than, 60, faculty, members, and, their, research, staff,, including, post-doctoral, fellows,, graduate, and, undergraduate, students, at, texas, a&m, university., , applications, and, research, programs, include:, , dissection, of, sorghum, drought, tolerance,, functional, and, metagenomic, analysis, of, biological, communities, for, cattle, and, rumen, , bacteria,, disease, resistance, , and, , an, analysis, of, microflora, in, ocean, drilling, programs,, among, others., , dissemination, of, data, and, project, outcomes, will, be, available, on, the, institute, for, plant, genomics, and, biotechnology, website, (http://ipgb.tamu.edu),, research, publications,, relevant, scientific, meetings, and, during, our, scheduled, campus-wide, annual, meeting.]                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           |277   |\n",
      "|PI: Hugo K. Dooner (Rutgers University)<br/>CoPIs: Chunguang Du (Montclair State University) and Randall A. Kerstetter (Rutgers University)<br/><br/>Transposable element (TE) insertions cause mutations that help researchers to elucidate gene function.  Collections of insertions that have been sequence-indexed (i.e., in which the host DNA adjacent to the insertion is known) are valuable resources in organisms with a sequenced genome and have been identified by the maize community as essential to fully exploit the maize genome sequence just generated.  Recently, the feasibility of combining high-throughput DNA sequencing with efficient multi-dimensional pooling strategies to rapidly and simultaneously sequence and index hundreds of new insertions has been demonstrated.  This project will develop and use a general method for generating, sequencing, and indexing Ac/Ds transposable element insertions in maize that is rapid, accurate, and cost-effective by taking advantage of next-generation sequencing technology. Specifically, the project will:  (1) sequence-index a collection of existing lines that contain a unique transposed Ac element; (2) complete a set of 120 roughly equidistant transgenic Ds elements that serve as launching platforms and carry easily scored markers that will allow simple visual selection of element transposition from any region of the genome and, thus, enable researchers to generate regional gene knock-out collections, (3) sequence-index 10,000 Ds element insertion sites from model platforms using a novel method that should be applicable to any collection of insertions produced in a common genetic background; and (4) develop a web-searchable database of insertion site sequences cross-referenced to lines that will be freely available from the Maize Genetics Stock Center (http://maizecoop.cropsci.uiuc.edu).  All relevant information from this project will be accessible from MaizeGDB (http://www.maizegdb.org).<br/><br/>This project addresses a critical need in that it will deliver a sequence-indexed reverse genetics resource, considered essential for researchers to fully exploit the maize genome sequence.  It will integrate next-generation sequencing technology at Rutgers with bioinformatic sequence analysis at Montclair State University, a predominantly undergraduate institution in NJ.  Students at MSU will assist with the assembly, annotation, and mapping of short reads as putative Ac-adjacent sequences and with the design of primers for validation.  The project will also provide research training opportunities for MSU informatics and molecular biology students, many of whom are from underrepresented groups.  Students at both Rutgers and MSU will also participate in the project as summer interns in the molecular biology lab and maize genetics nursery.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         |[pi:, hugo, k., dooner, (rutgers, university)<br/>copis:, chunguang, du, (montclair, state, university), and, randall, a., kerstetter, (rutgers, university)<br/><br/>transposable, element, (te), insertions, cause, mutations, that, help, researchers, to, elucidate, gene, function., , collections, of, insertions, that, have, been, sequence-indexed, (i.e.,, in, which, the, host, dna, adjacent, to, the, insertion, is, known), are, valuable, resources, in, organisms, with, a, sequenced, genome, and, have, been, identified, by, the, maize, community, as, essential, to, fully, exploit, the, maize, genome, sequence, just, generated., , recently,, the, feasibility, of, combining, high-throughput, dna, sequencing, with, efficient, multi-dimensional, pooling, strategies, to, rapidly, and, simultaneously, sequence, and, index, hundreds, of, new, insertions, has, been, demonstrated., , this, project, will, develop, and, use, a, general, method, for, generating,, sequencing,, and, indexing, ac/ds, transposable, element, insertions, in, maize, that, is, rapid,, accurate,, and, cost-effective, by, taking, advantage, of, next-generation, sequencing, technology., specifically,, the, project, will:, , (1), sequence-index, a, collection, of, existing, lines, that, contain, a, unique, transposed, ac, element;, (2), complete, a, set, of, 120, roughly, equidistant, transgenic, ds, elements, that, serve, as, launching, platforms, and, carry, easily, scored, markers, that, will, allow, simple, visual, selection, of, element, transposition, from, any, region, of, the, genome, and,, thus,, enable, researchers, to, generate, regional, gene, knock-out, collections,, (3), sequence-index, 10,000, ds, element, insertion, sites, from, model, platforms, using, a, novel, method, that, should, be, applicable, to, any, collection, of, insertions, produced, in, a, common, genetic, background;, and, (4), develop, a, web-searchable, database, of, insertion, site, sequences, cross-referenced, to, lines, that, will, be, freely, available, from, the, maize, genetics, stock, center, (http://maizecoop.cropsci.uiuc.edu)., , all, relevant, information, from, this, project, will, be, accessible, from, maizegdb, (http://www.maizegdb.org).<br/><br/>this, project, addresses, a, critical, need, in, that, it, will, deliver, a, sequence-indexed, reverse, genetics, resource,, considered, essential, for, researchers, to, fully, exploit, the, maize, genome, sequence., , it, will, integrate, next-generation, sequencing, technology, at, rutgers, with, bioinformatic, sequence, analysis, at, montclair, state, university,, a, predominantly, undergraduate, institution, in, nj., , students, at, msu, will, assist, with, the, assembly,, annotation,, and, mapping, of, short, reads, as, putative, ac-adjacent, sequences, and, with, the, design, of, primers, for, validation., , the, project, will, also, provide, research, training, opportunities, for, msu, informatics, and, molecular, biology, students,, many, of, whom, are, from, underrepresented, groups., , students, at, both, rutgers, and, msu, will, also, participate, in, the, project, as, summer, interns, in, the, molecular, biology, lab, and, maize, genetics, nursery.]                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        |395   |\n",
      "|The Ocean's biomass and diversity are predominantly microbial, yet this aspect of diversity remains underexplored. Efforts in recent years have begun to document microbial diversity in marine systems, and to elucidate the processes that structure assemblages across space and time. This project focuses on two important sister clades of microbial eukaryotes, the oligotrich and choreotrich ciliates. These organisms comprise a major component of planktonic food webs as they graze on phytoplankton, and are in turn eaten by zooplankton and larval fish.<br/><br/>Earlier molecular work on ciliate diversity relied on light microscopy, construction of clone libraries and Sanger sequencing. This revealed a high degree of cryptic diversity (similar species that are genetically distinct), which is surprising, given the long-held idea that all microbes are globally distributed and that few species exist, at least as compared to animals and plants. This past work also showed that ciliate assemblages contain a few highly abundant forms and many rare ones, consistent with the concept of a 'rare biosphere'. However, these methods are limited by high costs of both labor and materials, so that efforts to sample any local assemblage comprehensively usually resulted in undersaturation (repeated sampling continued to uncover new species). Next generation approaches are needed to truly assess the depths of biodiversity in planktonic ciliates.<br/><br/>This project brings together investigators with strengths in ecology, taxonomy and oceanography (PI McManus) and in molecular evolution, systematics and bioinformatics (PI Katz). Pyrosequencing will be used to sample the oligotrichs and choreotrichs 'to exhaustion' in coastal environments. Denaturing gradient gel electrophoresis (DGGE), a technique that generates a fingerprint of the diversity in a sample, will be used to pre-select samples for pyrosequencing based on where strong gradients are observed in the composition of assemblages in relation to environmental factors (density fronts, thermoclines, etc.). Using these approaches, combined with an informatics pipeline already in place, this project addresses three specific objectives:<br/><br/>Objective 1. Determine the spatial scale of variability in ciliate diversity by measuring how ciliate assemblages change over meter, kilometer, 100 km, and basin scales.<br/>Objective 2. Assess the contributions of different size classes of ciliates to overall assemblage diversity.<br/>Objective 3. Experimentally evaluate factors that control the temporal shift of individual species from rarity to commonness in a natural assemblage, and vice versa.<br/><br/>BROADER IMPACTS: A program to involve K-12 teachers in research, initiated under a previous award, will be continued. For the past three years, teachers have been hosted at the UConn marine labs, where they conducted independent research, learned to sample and enumerate plankton, and developed teaching resources for subsequent use in the classroom. In this project the program will be adapted to accommodate the shorter, more intense experiences most teachers prefer, and to incorporate an instructional component in broader marine sciences.  In addition to the teacher program, this project will involve training of postdoctoral fellows and undergraduates. This will include cross-disciplinary work through exchanges between the PI laboratories, with a focus on experimental design, data collection/analysis, and manuscript preparation. Both PIs are committed to involving students from underrepresented groups in their research, and PI Katz maintains intensive recruiting efforts in this area. Hence, the project will further increase the diversity of researchers studying biological oceanography.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |[the, ocean's, biomass, and, diversity, are, predominantly, microbial,, yet, this, aspect, of, diversity, remains, underexplored., efforts, in, recent, years, have, begun, to, document, microbial, diversity, in, marine, systems,, and, to, elucidate, the, processes, that, structure, assemblages, across, space, and, time., this, project, focuses, on, two, important, sister, clades, of, microbial, eukaryotes,, the, oligotrich, and, choreotrich, ciliates., these, organisms, comprise, a, major, component, of, planktonic, food, webs, as, they, graze, on, phytoplankton,, and, are, in, turn, eaten, by, zooplankton, and, larval, fish.<br/><br/>earlier, molecular, work, on, ciliate, diversity, relied, on, light, microscopy,, construction, of, clone, libraries, and, sanger, sequencing., this, revealed, a, high, degree, of, cryptic, diversity, (similar, species, that, are, genetically, distinct),, which, is, surprising,, given, the, long-held, idea, that, all, microbes, are, globally, distributed, and, that, few, species, exist,, at, least, as, compared, to, animals, and, plants., this, past, work, also, showed, that, ciliate, assemblages, contain, a, few, highly, abundant, forms, and, many, rare, ones,, consistent, with, the, concept, of, a, 'rare, biosphere'., however,, these, methods, are, limited, by, high, costs, of, both, labor, and, materials,, so, that, efforts, to, sample, any, local, assemblage, comprehensively, usually, resulted, in, undersaturation, (repeated, sampling, continued, to, uncover, new, species)., next, generation, approaches, are, needed, to, truly, assess, the, depths, of, biodiversity, in, planktonic, ciliates.<br/><br/>this, project, brings, together, investigators, with, strengths, in, ecology,, taxonomy, and, oceanography, (pi, mcmanus), and, in, molecular, evolution,, systematics, and, bioinformatics, (pi, katz)., pyrosequencing, will, be, used, to, sample, the, oligotrichs, and, choreotrichs, 'to, exhaustion', in, coastal, environments., denaturing, gradient, gel, electrophoresis, (dgge),, a, technique, that, generates, a, fingerprint, of, the, diversity, in, a, sample,, will, be, used, to, pre-select, samples, for, pyrosequencing, based, on, where, strong, gradients, are, observed, in, the, composition, of, assemblages, in, relation, to, environmental, factors, (density, fronts,, thermoclines,, etc.)., using, these, approaches,, combined, with, an, informatics, pipeline, already, in, place,, this, project, addresses, three, specific, objectives:<br/><br/>objective, 1., determine, the, spatial, scale, of, variability, in, ciliate, diversity, by, measuring, how, ciliate, assemblages, change, over, meter,, kilometer,, 100, km,, and, basin, scales.<br/>objective, 2., assess, the, contributions, of, different, size, classes, of, ciliates, to, overall, assemblage, diversity.<br/>objective, 3., experimentally, evaluate, factors, that, control, the, temporal, shift, of, individual, species, from, rarity, to, commonness, in, a, natural, assemblage,, and, vice, versa.<br/><br/>broader, impacts:, a, program, to, involve, k-12, teachers, in, research,, initiated, under, a, previous, award,, will, be, continued., for, the, past, three, years,, teachers, have, been, hosted, at, the, uconn, marine, labs,, where, they, conducted, independent, research,, learned, to, sample, and, enumerate, plankton,, and, developed, teaching, resources, for, subsequent, use, in, the, classroom., in, this, project, the, program, will, be, adapted, to, accommodate, the, shorter,, more, intense, experiences, most, teachers, prefer,, and, to, incorporate, an, instructional, component, in, broader, marine, sciences., , in, addition, to, the, teacher, program,, this, project, will, involve, training, of, postdoctoral, fellows, and, undergraduates., this, will, include, cross-disciplinary, work, through, exchanges, between, the, pi, laboratories,, with, a, focus, on, experimental, design,, data, collection/analysis,, and, manuscript, preparation., both, pis, are, committed, to, involving, students, from, underrepresented, groups, in, their, research,, and, pi, katz, maintains, intensive, recruiting, efforts, in, this, area., hence,, the, project, will, further, increase, the, diversity, of, researchers, studying, biological, oceanography.]                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          |527   |\n",
      "|The Ocean's biomass and diversity are predominantly microbial, yet this aspect of diversity remains underexplored. Efforts in recent years have begun to document microbial diversity in marine systems, and to elucidate the processes that structure assemblages across space and time. This project focuses on two important sister clades of microbial eukaryotes, the oligotrich and choreotrich ciliates. These organisms comprise a major component of planktonic food webs as they graze on phytoplankton, and are in turn eaten by zooplankton and larval fish.<br/><br/>Earlier molecular work on ciliate diversity relied on light microscopy, construction of clone libraries and Sanger sequencing. This revealed a high degree of cryptic diversity (similar species that are genetically distinct), which is surprising, given the long-held idea that all microbes are globally distributed and that few species exist, at least as compared to animals and plants. This past work also showed that ciliate assemblages contain a few highly abundant forms and many rare ones, consistent with the concept of a 'rare biosphere'. However, these methods are limited by high costs of both labor and materials, so that efforts to sample any local assemblage comprehensively usually resulted in undersaturation (repeated sampling continued to uncover new species). Next generation approaches are needed to truly assess the depths of biodiversity in planktonic ciliates.<br/><br/>This project brings together investigators with strengths in ecology, taxonomy and oceanography (PI McManus) and in molecular evolution, systematics and bioinformatics (PI Katz). Pyrosequencing will be used to sample the oligotrichs and choreotrichs 'to exhaustion' in coastal environments. Denaturing gradient gel electrophoresis (DGGE), a technique that generates a fingerprint of the diversity in a sample, will be used to pre-select samples for pyrosequencing based on where strong gradients are observed in the composition of assemblages in relation to environmental factors (density fronts, thermoclines, etc.). Using these approaches, combined with an informatics pipeline already in place, this project addresses three specific objectives:<br/><br/>Objective 1. Determine the spatial scale of variability in ciliate diversity by measuring how ciliate assemblages change over meter, kilometer, 100 km, and basin scales.<br/>Objective 2. Assess the contributions of different size classes of ciliates to overall assemblage diversity.<br/>Objective 3. Experimentally evaluate factors that control the temporal shift of individual species from rarity to commonness in a natural assemblage, and vice versa.<br/><br/>BROADER IMPACTS: A program to involve K-12 teachers in research, initiated under a previous award, will be continued. For the past three years, teachers have been hosted at the UConn marine labs, where they conducted independent research, learned to sample and enumerate plankton, and developed teaching resources for subsequent use in the classroom. In this project the program will be adapted to accommodate the shorter, more intense experiences most teachers prefer, and to incorporate an instructional component in broader marine sciences.  In addition to the teacher program, this project will involve training of postdoctoral fellows and undergraduates. This will include cross-disciplinary work through exchanges between the PI laboratories, with a focus on experimental design, data collection/analysis, and manuscript preparation. Both PIs are committed to involving students from underrepresented groups in their research, and PI Katz maintains intensive recruiting efforts in this area. Hence, the project will further increase the diversity of researchers studying biological oceanography.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |[the, ocean's, biomass, and, diversity, are, predominantly, microbial,, yet, this, aspect, of, diversity, remains, underexplored., efforts, in, recent, years, have, begun, to, document, microbial, diversity, in, marine, systems,, and, to, elucidate, the, processes, that, structure, assemblages, across, space, and, time., this, project, focuses, on, two, important, sister, clades, of, microbial, eukaryotes,, the, oligotrich, and, choreotrich, ciliates., these, organisms, comprise, a, major, component, of, planktonic, food, webs, as, they, graze, on, phytoplankton,, and, are, in, turn, eaten, by, zooplankton, and, larval, fish.<br/><br/>earlier, molecular, work, on, ciliate, diversity, relied, on, light, microscopy,, construction, of, clone, libraries, and, sanger, sequencing., this, revealed, a, high, degree, of, cryptic, diversity, (similar, species, that, are, genetically, distinct),, which, is, surprising,, given, the, long-held, idea, that, all, microbes, are, globally, distributed, and, that, few, species, exist,, at, least, as, compared, to, animals, and, plants., this, past, work, also, showed, that, ciliate, assemblages, contain, a, few, highly, abundant, forms, and, many, rare, ones,, consistent, with, the, concept, of, a, 'rare, biosphere'., however,, these, methods, are, limited, by, high, costs, of, both, labor, and, materials,, so, that, efforts, to, sample, any, local, assemblage, comprehensively, usually, resulted, in, undersaturation, (repeated, sampling, continued, to, uncover, new, species)., next, generation, approaches, are, needed, to, truly, assess, the, depths, of, biodiversity, in, planktonic, ciliates.<br/><br/>this, project, brings, together, investigators, with, strengths, in, ecology,, taxonomy, and, oceanography, (pi, mcmanus), and, in, molecular, evolution,, systematics, and, bioinformatics, (pi, katz)., pyrosequencing, will, be, used, to, sample, the, oligotrichs, and, choreotrichs, 'to, exhaustion', in, coastal, environments., denaturing, gradient, gel, electrophoresis, (dgge),, a, technique, that, generates, a, fingerprint, of, the, diversity, in, a, sample,, will, be, used, to, pre-select, samples, for, pyrosequencing, based, on, where, strong, gradients, are, observed, in, the, composition, of, assemblages, in, relation, to, environmental, factors, (density, fronts,, thermoclines,, etc.)., using, these, approaches,, combined, with, an, informatics, pipeline, already, in, place,, this, project, addresses, three, specific, objectives:<br/><br/>objective, 1., determine, the, spatial, scale, of, variability, in, ciliate, diversity, by, measuring, how, ciliate, assemblages, change, over, meter,, kilometer,, 100, km,, and, basin, scales.<br/>objective, 2., assess, the, contributions, of, different, size, classes, of, ciliates, to, overall, assemblage, diversity.<br/>objective, 3., experimentally, evaluate, factors, that, control, the, temporal, shift, of, individual, species, from, rarity, to, commonness, in, a, natural, assemblage,, and, vice, versa.<br/><br/>broader, impacts:, a, program, to, involve, k-12, teachers, in, research,, initiated, under, a, previous, award,, will, be, continued., for, the, past, three, years,, teachers, have, been, hosted, at, the, uconn, marine, labs,, where, they, conducted, independent, research,, learned, to, sample, and, enumerate, plankton,, and, developed, teaching, resources, for, subsequent, use, in, the, classroom., in, this, project, the, program, will, be, adapted, to, accommodate, the, shorter,, more, intense, experiences, most, teachers, prefer,, and, to, incorporate, an, instructional, component, in, broader, marine, sciences., , in, addition, to, the, teacher, program,, this, project, will, involve, training, of, postdoctoral, fellows, and, undergraduates., this, will, include, cross-disciplinary, work, through, exchanges, between, the, pi, laboratories,, with, a, focus, on, experimental, design,, data, collection/analysis,, and, manuscript, preparation., both, pis, are, committed, to, involving, students, from, underrepresented, groups, in, their, research,, and, pi, katz, maintains, intensive, recruiting, efforts, in, this, area., hence,, the, project, will, further, increase, the, diversity, of, researchers, studying, biological, oceanography.]                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          |527   |\n",
      "|This grant will support a workshop focused on developing ideas for a training course in marine microbial genomics, consistent with recommendations of the US-European Commission Taskforce on Biotechnology Research. Leading scientists from the US and Europe will meet in October, 2010 in Washington, DC. These experts will discuss recent scientific progress in marine microbiology that has accrued from modern genomic approaches, as well as opportunities and challenges for continued progress given the fast pace of genome technology advances. Analyses of the state of the science and technology will then frame consideration of the most effective ways to train next-generation scientists who will capitalize on these advances and tackle the technological challenges, a prerequisite for continued progress in this important scientific area. <br/><br/>Workshop sessions will examine the computational methods used to process large amounts of DNA sequence information, current and future DNA sequencing methodologies and tools being used to connect gene and genome sequence information with functional properties of the relevant microbes. The outcome of the presentations and discussions will be a workshop report that provides an overview of critical issues and provides recommendations from each session. It will also include a basic description of an advanced training course on the topic of genomic analyses of marine microbes.<br/><br/>Broader impacts:  this entire grant is focused on the needs of the United States and the European Commission to continue to work together in the field of marine microbial genomics in order to expedite advances in basic sciences and in biotechnology.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |[this, grant, will, support, a, workshop, focused, on, developing, ideas, for, a, training, course, in, marine, microbial, genomics,, consistent, with, recommendations, of, the, us-european, commission, taskforce, on, biotechnology, research., leading, scientists, from, the, us, and, europe, will, meet, in, october,, 2010, in, washington,, dc., these, experts, will, discuss, recent, scientific, progress, in, marine, microbiology, that, has, accrued, from, modern, genomic, approaches,, as, well, as, opportunities, and, challenges, for, continued, progress, given, the, fast, pace, of, genome, technology, advances., analyses, of, the, state, of, the, science, and, technology, will, then, frame, consideration, of, the, most, effective, ways, to, train, next-generation, scientists, who, will, capitalize, on, these, advances, and, tackle, the, technological, challenges,, a, prerequisite, for, continued, progress, in, this, important, scientific, area., <br/><br/>workshop, sessions, will, examine, the, computational, methods, used, to, process, large, amounts, of, dna, sequence, information,, current, and, future, dna, sequencing, methodologies, and, tools, being, used, to, connect, gene, and, genome, sequence, information, with, functional, properties, of, the, relevant, microbes., the, outcome, of, the, presentations, and, discussions, will, be, a, workshop, report, that, provides, an, overview, of, critical, issues, and, provides, recommendations, from, each, session., it, will, also, include, a, basic, description, of, an, advanced, training, course, on, the, topic, of, genomic, analyses, of, marine, microbes.<br/><br/>broader, impacts:, , this, entire, grant, is, focused, on, the, needs, of, the, united, states, and, the, european, commission, to, continue, to, work, together, in, the, field, of, marine, microbial, genomics, in, order, to, expedite, advances, in, basic, sciences, and, in, biotechnology.]                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       |248   |\n",
      "+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+------+\n",
      "only showing top 20 rows\n",
      "\n"
     ]
    }
   ],
   "source": [
    "# Transform dataframe\n",
    "abstract_tokenized_df = tokenizer.transform(df)\n",
    "\n",
    "# Select the needed columns and don't truncate results\n",
    "abstract_tokenized_df.select(\"Project Abstract\", \"abstract_words_raw\")\\\n",
    "    .withColumn(\"tokens\", count_tokens(col(\"abstract_words_raw\"))).show(truncate=False)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 98,
   "metadata": {
    "scrolled": true
   },
   "outputs": [],
   "source": [
    "#instantiate remover\n",
    "remover = StopWordsRemover(inputCol=\"abstract_words_raw\", outputCol=\"abstract_words_filtered\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 99,
   "metadata": {},
   "outputs": [],
   "source": [
    "# transform and show filtered data\n",
    "abstract_filtered_df = remover.transform(abstract_tokenized_df)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 100,
   "metadata": {},
   "outputs": [],
   "source": [
    "# #TF/IDF work-round \n",
    "# ------------\n",
    "#String indexer to check vocab_size\n",
    "\n",
    "# indexer = StringIndexer(inputCol=\"abstract_words_filtered\", outputCol=\"abstract_words_Indexed\")\n",
    "# indexed = indexer.fit(abstract_filtered_df).transform(abstract_filtered_df)\n",
    "# indexed.show()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 101,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Row(abstract_words_filtered=['advances', 'sequencing', 'technology', 'dramatically', 'shifted', 'way', 'science', 'performed', 'many', 'fields', 'biology,', 'also', 'shifted', 'way', 'education', 'next', 'generation', 'scientists', 'must', 'conducted.', 'however,', 'generation', 'sequencing', 'data', 'restricted', 'large', 'laboratories,', 'consortia', 'laboratories,', 'core', 'sequencing', 'facilities,', 'limiting', 'exposure', 'undergraduates', 'technologies.', 'recent', 'introduction', 'benchtop', 'next', 'generation', 'sequencing', 'platforms', 'promises', 'make', 'moderately', 'high', 'throughput', 'sequencing', 'available', 'many', 'individual', 'research', 'laboratories', 'smaller', 'institutions.', 'major', 'research', 'instrumentation', 'grant', 'supports', 'acquisition', 'ion', 'torrent', 'personal', 'genome', 'machine', '(ion', 'pgm)', 'sequencing', 'platform', 'hope', 'college.', 'ion', 'pgm', 'significant', 'impact', 'undergraduate', 'research', 'training', 'hope', 'college', 'making', 'affordable,', 'high', 'throughput', 'sequencing', 'capacity', 'available', 'faculty', 'undergraduate', 'researchers', 'use', 'established', 'emerging', 'research', 'education', 'programs.', 'programs', 'include', 'research', 'focusing', 'on:', '1)', 'genome', 'sequences', 'viruses', 'infect', 'bacteria', '(mycobacteriophage)', 'discovered', 'first', 'year', 'college', 'students', 'part', 'introductory', 'biology', 'laboratory', 'courses;', '2)', 'long-term', 'environmental', 'studies', 'local', 'lake', 'macatawa', 'watershed', 'identify', 'track', 'microbial', 'sources', 'contamination', 'whole', 'genome', 'sequencing', 'novel', 'bacterial', 'isolates;', '3)', 'advancing', 'understanding', 'fundamental', 'gene', 'expression', 'mechanisms', 'mammalian', 'parasite', 'giardia', 'lamblia;', '4)', 'understanding', 'phylogenetic', 'biogeographic', 'relationships', 'among', 'maples', 'found', 'across', 'world;', '5)', 'testing', 'potential', 'horizontal', 'gene', 'transfers', 'plastid', 'genes', 'parasitic', 'plants', 'photosynthetic', 'hosts.', 'projects', 'advance', 'scientific', 'understanding', 'wide', 'range', 'biological', 'environmental', 'problems', 'addressed', 'dna', 'sequencing', 'different', 'scales.', 'project', 'made', 'effective', 'house', 'availability', 'ion', 'pgm,', 'uniquely', 'versatile', 'able', 'generate', 'sequence', 'data', 'vastly', 'different', 'scales', 'cost', 'effective', 'manner.<br/><br/>this', 'sequencing', 'platform', 'allow', 'hope', 'college', 'effectively', 'train', 'undergraduates', 'design', 'experiments', 'requiring', 'sequencing', 'data;', 'usage', 'next', 'generation', 'sequencing', 'technology', 'generate', 'necessary', 'data;', 'analysis,', 'interpretation,', 'dissemination', 'large', 'scale', 'data', 'sets', 'research', 'laboratory', 'classroom', 'laboratory', 'settings.', 'ion', 'pgm', 'immediately', 'enhance', 'projects', 'programs', 'impact', '100', 'undergraduates', 'academic', 'year.', 'first', 'year', 'students', 'impacted', 'hope', \"college's\", 'highly', 'successful', 'yearlong', 'course', 'phage', 'genomics', 'research,', 'designed', 'retain', 'high', 'quality', 'students', 'stem', 'fields.', 'students', 'levels', 'participate', 'research', 'laboratories', 'pi?s', 'collaborators.', 'students', 'upper', 'level', 'biology', 'courses', 'use', 'technology', 'classroom', 'laboratory-based', 'research', 'projects.', 'hope', 'college?s', 'established', 'summer', 'research', 'programs', 'allow', 'us', 'broaden', 'exposure', 'students', 'across', 'nation', 'students', 'underrepresented', 'groups', 'sciences', 'use', 'leading', 'edge', 'sequencing', 'technology.', 'faculty', 'impacted', 'ability', 'take', 'advantage', 'next', 'generation', 'sequencing', 'technology', 'advance', 'projects', 'new', 'innovative', 'areas', 'generate', 'preliminary', 'data', 'sustaining', 'extramurally', 'funded', 'research', 'programs.', 'student-faculty', 'collaborative', 'research', 'disseminated', 'scientific', 'community', 'local,', 'regional,', 'national/international', 'scientific', 'conferences', 'peer-reviewed', 'publications', 'general', 'public', 'community-based', 'popular', 'seminars.', 'sequence', 'data', 'deposited', 'appropriate', 'openly', 'accessible', 'databases.', 'regional', 'national', 'collaborators', 'use', 'ion', 'pgm,', 'broadening', 'impact', 'beyond', 'hope', 'college', 'students', 'west', 'michigan', 'throughout', 'nation.'], cv_Features=SparseVector(50663, {2: 2.0, 4: 11.0, 5: 1.0, 7: 1.0, 8: 1.0, 9: 1.0, 10: 1.0, 13: 14.0, 20: 5.0, 21: 1.0, 24: 1.0, 27: 1.0, 29: 3.0, 34: 1.0, 35: 4.0, 37: 1.0, 38: 1.0, 42: 1.0, 49: 1.0, 54: 2.0, 58: 3.0, 61: 1.0, 62: 2.0, 63: 2.0, 65: 3.0, 78: 1.0, 90: 2.0, 101: 2.0, 103: 1.0, 104: 1.0, 109: 1.0, 125: 1.0, 128: 4.0, 130: 2.0, 141: 1.0, 145: 1.0, 148: 1.0, 157: 2.0, 158: 1.0, 161: 3.0, 163: 1.0, 166: 1.0, 171: 3.0, 172: 1.0, 176: 1.0, 177: 1.0, 180: 2.0, 189: 3.0, 198: 1.0, 199: 2.0, 201: 1.0, 202: 1.0, 207: 8.0, 218: 1.0, 227: 1.0, 237: 1.0, 239: 1.0, 240: 2.0, 244: 1.0, 252: 1.0, 257: 1.0, 258: 1.0, 259: 2.0, 262: 1.0, 278: 3.0, 285: 5.0, 292: 1.0, 312: 1.0, 315: 1.0, 319: 1.0, 320: 1.0, 331: 2.0, 335: 1.0, 349: 1.0, 357: 3.0, 366: 1.0, 368: 1.0, 377: 2.0, 394: 1.0, 395: 1.0, 409: 2.0, 426: 1.0, 427: 1.0, 432: 1.0, 433: 4.0, 444: 1.0, 454: 2.0, 455: 1.0, 462: 1.0, 464: 1.0, 470: 1.0, 484: 1.0, 491: 2.0, 499: 1.0, 524: 1.0, 525: 2.0, 534: 1.0, 549: 3.0, 569: 1.0, 570: 1.0, 575: 1.0, 593: 1.0, 606: 1.0, 615: 1.0, 651: 1.0, 693: 1.0, 700: 1.0, 708: 1.0, 709: 2.0, 720: 1.0, 721: 1.0, 733: 1.0, 737: 1.0, 738: 1.0, 741: 2.0, 758: 1.0, 797: 1.0, 803: 1.0, 809: 1.0, 825: 1.0, 828: 1.0, 839: 1.0, 922: 1.0, 929: 1.0, 950: 1.0, 963: 1.0, 968: 1.0, 1028: 1.0, 1139: 1.0, 1171: 1.0, 1208: 1.0, 1248: 1.0, 1276: 1.0, 1278: 1.0, 1279: 1.0, 1289: 1.0, 1302: 2.0, 1303: 1.0, 1311: 1.0, 1349: 1.0, 1372: 1.0, 1436: 1.0, 1460: 1.0, 1474: 1.0, 1499: 2.0, 1502: 1.0, 1519: 1.0, 1526: 1.0, 1549: 1.0, 1785: 6.0, 1823: 1.0, 1843: 1.0, 1911: 1.0, 1980: 1.0, 2012: 1.0, 2054: 1.0, 2116: 1.0, 2146: 1.0, 2272: 1.0, 2299: 1.0, 2320: 1.0, 2368: 1.0, 2397: 1.0, 2401: 1.0, 2406: 1.0, 2435: 2.0, 2474: 1.0, 2609: 1.0, 2633: 1.0, 2712: 2.0, 2804: 1.0, 2856: 1.0, 2977: 4.0, 3237: 1.0, 3304: 1.0, 3555: 1.0, 3573: 1.0, 3672: 1.0, 3682: 1.0, 3715: 1.0, 3966: 1.0, 4107: 1.0, 4151: 3.0, 4229: 1.0, 4340: 1.0, 4401: 1.0, 4425: 1.0, 4524: 1.0, 4535: 1.0, 4599: 1.0, 4677: 1.0, 4684: 1.0, 4721: 1.0, 4868: 1.0, 4893: 1.0, 4937: 1.0, 5043: 1.0, 5466: 1.0, 5737: 1.0, 5952: 1.0, 6038: 2.0, 6166: 1.0, 6441: 1.0, 6451: 1.0, 6764: 1.0, 6908: 1.0, 6912: 1.0, 7188: 1.0, 7438: 1.0, 7517: 1.0, 7568: 5.0, 7570: 1.0, 8044: 1.0, 8436: 1.0, 9796: 1.0, 10890: 2.0, 11212: 1.0, 11275: 2.0, 11580: 1.0, 11937: 1.0, 12273: 1.0, 13372: 1.0, 14504: 1.0, 14709: 1.0, 14840: 1.0, 14883: 2.0, 15633: 1.0, 19155: 1.0, 19356: 2.0, 19764: 1.0, 20283: 2.0, 20987: 1.0, 23117: 1.0, 23986: 2.0, 25444: 1.0, 26868: 1.0, 26931: 1.0, 27204: 1.0, 27348: 1.0, 29339: 1.0, 30609: 1.0, 30652: 1.0, 30714: 1.0, 30785: 1.0, 30820: 1.0, 30904: 1.0, 30964: 1.0, 30965: 1.0, 31214: 1.0, 31332: 1.0, 31340: 1.0, 31372: 1.0, 31765: 1.0, 31928: 1.0, 32003: 1.0, 32119: 1.0, 32302: 1.0, 32466: 1.0, 32478: 1.0, 32490: 1.0, 32504: 1.0, 32611: 1.0}))\n",
      "Row(abstract_words_filtered=['description', '(provided', 'applicant):', 'power', 'zebrafish', 'system', 'stems', 'utility', 'developmental', 'biology', 'model', 'combined', 'ease', 'genetic', 'manipulation', 'experimentation.', 'understanding', 'key', 'genetic', 'mechanisms', 'vertebrate', 'development', 'propelled', 'phenotypic', 'characterization,', 'genetic', 'mapping', 'positional', 'cloning', 'induced', 'spontaneous', 'mutations', 'zebrafish.', 'however,', 'potential', 'system', 'fully', 'realized,', 'inefficient', 'microsatellite-based', 'mapping', 'remains', 'primary', 'method', 'field.', 'propose', 'apply', 'technological', 'computational', 'advances', 'present', 'day', 'genomics', 'genetic', 'mapping', 'zebrafish', 'system.', 'specifically,', 'propose', 'develop', 'method', 'rapid', 'accurate', 'mapping', 'recessive', 'zebrafish', 'mutants', 'using', 'next', 'generation', 'sequencing', '(ngs)', 'pooled', 'samples.', 'also', 'propose', 'investigate', 'parameters', 'screen', 'design', 'sample', 'analysis', 'optimize', 'use', 'protocol.', 'finally,', 'aim', 'develop', 'methods', 'identification', 'causal', 'mutation', 'among', 'variants', 'discovered', 'within', 'mapping', 'interval.', 'application', 'ngs', 'technology,', 'complemented', 'specifically', 'developed', 'computational', 'techniques,', 'provide', 'efficient,', 'accurate', 'inexpensive', 'method', 'genetic', 'mapping', 'zebrafish.', 'approach', 'enable', 'simultaneous', 'identification', 'informative', 'genetic', 'markers,', 'mapping', 'mutation', 'position,', 'potential', 'identification', 'causal', 'sequence', 'change', 'single', 'experiment.', 'data', 'obtained', 'genomic', 'analyses', 'methods', 'developed', 'made', 'available', 'zebrafish', 'community.', 'importantly,', 'approaches', 'also', 'widely', 'applicable', 'genetic', 'analysis', 'model', 'systems.', '', 'public', 'health', 'relevance:', 'analysis', 'zebrafish', 'mutants', 'enabled', 'identification', 'genes', 'contributing', 'fundamental', 'biological', 'processes,', 'including', 'human', 'diseases;', 'however,', 'methods', 'used', 'mutant', 'mapping', 'gene', 'discovery', 'inefficient.', 'here,', 'propose', 'fast', 'cost-effective', 'method', 'genetic', 'mapping', 'mutation', 'identification', 'using', 'next', 'generation', 'sequencing.', 'facilitate', 'rapid', 'discovery', 'gene', 'function', 'translation', 'knowledge', 'biomedical', 'investigation.'], cv_Features=SparseVector(50663, {0: 1.0, 2: 2.0, 6: 1.0, 12: 1.0, 13: 1.0, 14: 1.0, 15: 1.0, 16: 2.0, 17: 1.0, 19: 1.0, 20: 1.0, 21: 2.0, 22: 1.0, 24: 1.0, 25: 1.0, 29: 1.0, 30: 8.0, 35: 1.0, 38: 1.0, 40: 3.0, 41: 2.0, 42: 1.0, 48: 1.0, 50: 4.0, 51: 1.0, 54: 1.0, 59: 3.0, 60: 1.0, 61: 2.0, 62: 1.0, 64: 1.0, 72: 1.0, 79: 2.0, 83: 1.0, 89: 1.0, 91: 1.0, 96: 2.0, 104: 1.0, 110: 2.0, 125: 2.0, 126: 1.0, 131: 1.0, 137: 2.0, 145: 1.0, 146: 1.0, 149: 1.0, 185: 1.0, 187: 1.0, 199: 1.0, 202: 1.0, 204: 5.0, 208: 1.0, 211: 1.0, 222: 1.0, 233: 1.0, 237: 1.0, 241: 4.0, 243: 1.0, 277: 1.0, 285: 2.0, 292: 1.0, 345: 1.0, 390: 1.0, 416: 1.0, 430: 1.0, 432: 1.0, 433: 2.0, 435: 1.0, 446: 1.0, 460: 1.0, 474: 1.0, 511: 2.0, 552: 1.0, 558: 1.0, 564: 2.0, 568: 1.0, 585: 1.0, 595: 1.0, 682: 1.0, 683: 9.0, 711: 2.0, 712: 1.0, 720: 1.0, 721: 1.0, 743: 1.0, 885: 1.0, 907: 1.0, 937: 1.0, 972: 1.0, 973: 2.0, 996: 1.0, 1039: 1.0, 1116: 1.0, 1198: 1.0, 1235: 1.0, 1250: 1.0, 1332: 3.0, 1353: 5.0, 1354: 1.0, 1415: 2.0, 1493: 1.0, 1527: 1.0, 1599: 1.0, 1764: 1.0, 1819: 1.0, 1844: 1.0, 1870: 1.0, 1914: 1.0, 1989: 1.0, 2122: 1.0, 2550: 1.0, 3107: 1.0, 3195: 1.0, 3313: 1.0, 3396: 1.0, 3613: 1.0, 3704: 1.0, 4039: 1.0, 4044: 1.0, 4157: 1.0, 4870: 1.0, 4939: 1.0, 5499: 1.0, 5860: 1.0, 6029: 1.0, 6192: 1.0, 7117: 1.0, 7311: 1.0, 7777: 1.0, 8032: 1.0, 8092: 1.0, 8317: 1.0, 11282: 1.0, 12231: 1.0, 13744: 1.0, 13800: 1.0, 14738: 1.0, 15008: 2.0, 15400: 1.0, 15659: 1.0, 18752: 1.0, 23311: 1.0, 24425: 1.0, 31134: 1.0}))\n",
      "Row(abstract_words_filtered=['overall', 'summary', 'specific', 'aims:', 'network', 'resource', 'component', 'mayo', 'clinic', 'pgrn', 'renewal', 'application', 'represents', 'request', 'funding', \"'network\", \"resource'\", 'designed', 'provide', 'entire', 'pgrn', 'access', 'latest', 'techniques', 'rapidly', 'evolving', 'field', \"'next\", \"generation'\", 'dna', 'sequencing.', 'access', 'provided', 'recurring,', 'peer-reviewed', 'process', 'available', 'pgrn', 'centers', 'similar', 'current', \"'pgrn-riken'\", 'process', 'obtaining', 'access', 'pharmacogenomic', 'gwas', 'genotyping', 'pgrn-riken', 'collaboration.', 'proposed', \"'pgrn\", 'nehwork', \"resource'\", 'one', 'three', 'coordinated', 'applications', 'current', 'pgrn', 'centers,', 'similar', 'goals', '-', 'clearly', 'indicating', 'unmet', 'need', 'pharmacogenomic', 'research', '-', 'need', 'emphasized', 'letters', 'support', 'current', 'pgrn', 'centers', 'attached', 'application.', 'letters', 'sent', 'eight', 'different', 'pgrn', 'research', 'centers.', 'although', 'pharmacoqenomics', 'always', 'required', 'access', 'dna', 'sequencing,', 'need', 'accentuated', 'bv', 'recent', 'dramatic', 'changes', 'dna', 'seguencing', 'technology', 'bv', 'eguallv', 'dramatic', 'outcomes', 'application', 'genome-wide', 'approaches', 'pharmacogenomics.', 'proposals', 'designed', 'provide', \"'link'\", 'pgrn', 'large,', 'experienced', 'genomics', 'center', 'expertise', 'application', 'evaluation', 'dna', 'sequencing', 'technology.', 'issue', 'whether', 'pharmacogenomics', 'discipline', 'requires', 'access', 'latest', 'rapidly', 'evolving', 'dna', 'sequencing', 'techniques,', 'achieve', 'goal', 'way', 'best', 'advance', 'science', 'affordable', 'open', 'pgrn', 'centers.', 'specifically,', 'mayo', 'pgrn', 'proposing', 'formal', 'collaborative', 'relationship', 'established', 'baylor', 'college', 'medicine', '(bcm)-human', 'genome', 'sequencing', 'center', '(hgsc),', 'one', 'three', 'nhgrisupported', 'large', 'genome', 'centers', '-', 'open,', 'protocol-based', 'selection', 'process', 'similar', 'well', 'established', 'highly', 'successful', 'pgrn-riken', 'process', 'selection', 'pgrn', 'pharmacogenomic', 'gwas', 'genotyping', 'projects', 'performed', 'collaboration', 'riken.', 'initial', 'focus', 'mayo', 'pgrn-sponsored', \"'next\", \"generation'\", 'dna', 'sequencing', 'network', 'resource', 'sequencing', 'large', 'contiguous', 'regions', 'genome.', 'cases,', 'involve', 'complete', 'resequencing', 'large', 'genes', 'tandemly', 'repeated', 'gene', 'families', 'even', 'entire', 'gene', 'families,', 'e.g.,', \"'cypome',\", 'cytochrome', 'p450', 'genes.', 'cases,', 'wilt', 'involve', 'focused', 'resequencing', 'across', \"'snp\", \"signals'\", 'observed', 'pharmacogenomic', 'gwa', 'studies', '-', 'many', 'cases,', 'gwa', 'studies', 'made', 'possible', 'pgrn-riken', 'collaboration,', 'thus', 'building', 'foundation', 'established', 'successful', 'program.', 'since', 'review', 'panel', 'applications', 'evaluate', 'series', 'network', 'resource', 'applications', 'access', 'next', 'generation', 'dna', 'sequencing,', 'emphasized', 'mayo', 'pgrn', 'looks', 'upon', 'efforts', 'complementary.', 'therefore,', 'possible', 'reviewers', 'may', 'conclude', \"'combination'\", 'proposals', 'supported.', 'proposals', 'share;', '', 'recognition', 'critical', 'need', 'access', 'rapidly', 'evolving', 'dna', 'sequencing', 'techniques', 'order', 'advance', 'science', 'pharmacogenomics.', '', 'recognition', 'established', 'genome', 'centers', 'requisite', 'experience,', 'expertise', 'infrastructure', 'required', 'make', 'effort', 'succeed.', '', 'recognition', \"'process'\", 'access', 'next', 'generation', 'dna', 'sequencing', 'open', 'pgrn', 'centers,', 'based', 'scientific', 'significance', 'feasibility.', 'cost', 'associated', 'studies', 'covered', 'primarily', 'funding', 'available', \"'network\", \"resource'.\"], cv_Features=SparseVector(50663, {2: 2.0, 4: 2.0, 8: 9.0, 9: 3.0, 11: 1.0, 13: 7.0, 24: 1.0, 25: 2.0, 45: 1.0, 47: 1.0, 52: 1.0, 53: 2.0, 55: 1.0, 58: 3.0, 63: 1.0, 81: 1.0, 82: 1.0, 84: 1.0, 88: 1.0, 91: 3.0, 98: 1.0, 101: 1.0, 108: 2.0, 111: 2.0, 112: 1.0, 128: 1.0, 130: 3.0, 131: 1.0, 141: 1.0, 144: 3.0, 145: 1.0, 148: 1.0, 161: 1.0, 188: 1.0, 199: 2.0, 203: 3.0, 220: 1.0, 235: 2.0, 240: 1.0, 244: 2.0, 247: 1.0, 266: 4.0, 281: 8.0, 284: 1.0, 285: 2.0, 289: 1.0, 293: 2.0, 319: 2.0, 322: 2.0, 327: 1.0, 331: 4.0, 348: 1.0, 356: 1.0, 357: 1.0, 360: 1.0, 365: 1.0, 375: 1.0, 392: 4.0, 402: 4.0, 409: 2.0, 417: 1.0, 432: 1.0, 433: 2.0, 438: 1.0, 440: 3.0, 444: 1.0, 447: 1.0, 467: 1.0, 468: 1.0, 472: 3.0, 476: 2.0, 486: 3.0, 510: 1.0, 522: 1.0, 524: 2.0, 527: 3.0, 530: 1.0, 531: 1.0, 562: 2.0, 568: 1.0, 570: 1.0, 575: 1.0, 577: 1.0, 599: 1.0, 600: 1.0, 644: 1.0, 653: 1.0, 669: 2.0, 712: 1.0, 713: 1.0, 718: 1.0, 730: 2.0, 741: 1.0, 742: 1.0, 745: 1.0, 746: 4.0, 750: 1.0, 758: 1.0, 793: 1.0, 829: 1.0, 840: 1.0, 850: 2.0, 887: 3.0, 894: 2.0, 925: 1.0, 932: 1.0, 955: 1.0, 984: 2.0, 1022: 1.0, 1060: 1.0, 1143: 2.0, 1174: 1.0, 1206: 1.0, 1228: 1.0, 1281: 2.0, 1294: 1.0, 1384: 1.0, 1501: 12.0, 1625: 1.0, 1740: 2.0, 1916: 1.0, 1951: 1.0, 2272: 1.0, 2285: 3.0, 2300: 3.0, 2332: 2.0, 2403: 2.0, 2490: 1.0, 2503: 1.0, 2527: 1.0, 2550: 1.0, 2667: 1.0, 2746: 4.0, 2791: 1.0, 2794: 1.0, 2833: 1.0, 2888: 1.0, 2897: 1.0, 2977: 1.0, 3086: 2.0, 3464: 1.0, 3529: 2.0, 3530: 3.0, 3554: 4.0, 3570: 1.0, 3682: 1.0, 3706: 1.0, 3710: 1.0, 3739: 1.0, 3758: 1.0, 3769: 1.0, 3804: 2.0, 4119: 1.0, 4183: 1.0, 4316: 3.0, 4385: 2.0, 4416: 1.0, 4459: 2.0, 4469: 2.0, 4557: 1.0, 4561: 1.0, 4743: 1.0, 4891: 3.0, 4936: 1.0, 5009: 2.0, 5228: 1.0, 5494: 1.0, 5861: 1.0, 6353: 1.0, 6364: 2.0, 6506: 2.0, 6509: 2.0, 6530: 2.0, 7685: 1.0, 7903: 1.0, 8022: 1.0, 8327: 1.0, 8759: 1.0, 9834: 1.0, 10741: 1.0, 11461: 1.0, 11474: 1.0, 11648: 1.0, 11652: 1.0, 11654: 1.0, 11657: 1.0, 11658: 1.0, 11659: 1.0, 11660: 1.0, 11669: 1.0, 11670: 1.0, 11672: 1.0, 11673: 1.0, 11686: 1.0, 11687: 1.0, 11688: 1.0, 11692: 1.0, 11693: 1.0, 11694: 1.0, 11696: 1.0, 11697: 1.0, 11698: 1.0, 11714: 1.0, 11719: 1.0, 11724: 1.0, 11726: 1.0, 11731: 1.0, 11734: 1.0, 11740: 1.0, 11744: 1.0}))\n"
     ]
    }
   ],
   "source": [
    "# Generate Term Frequency (TF)\n",
    "\n",
    "# fit a CountVectorizerModel from the corpus.\n",
    "cv = CountVectorizer(inputCol=\"abstract_words_filtered\", outputCol=\"cv_Features\", vocabSize=100000)\n",
    "cv_model = cv.fit(abstract_filtered_df)\n",
    "cv_result = cv_model.transform(abstract_filtered_df)\n",
    "\n",
    "# Select the needed columns and don't truncate results\n",
    "for words in cv_result.select(\"abstract_words_filtered\",\"cv_Features\").take(3):\n",
    "    print(words)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 102,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Row(idf_features=SparseVector(50663, {2: 1.6419, 4: 8.7403, 5: 1.003, 7: 1.6404, 8: 1.3756, 9: 0.7737, 10: 0.8027, 13: 20.1663, 20: 5.3513, 21: 0.8351, 24: 1.2127, 27: 1.0057, 29: 2.8801, 34: 1.1249, 35: 3.9023, 37: 1.0098, 38: 1.1757, 42: 0.8133, 49: 2.0758, 54: 3.0154, 58: 4.8335, 61: 1.3413, 62: 3.1883, 63: 2.5538, 65: 4.5462, 78: 2.0851, 90: 3.0813, 101: 3.5613, 103: 1.6609, 104: 1.774, 109: 1.619, 125: 1.6652, 128: 7.7222, 130: 3.4986, 141: 1.68, 145: 2.2249, 148: 1.813, 157: 3.7237, 158: 1.9209, 161: 6.1081, 163: 1.9843, 166: 2.3579, 171: 5.6967, 172: 2.1248, 176: 1.8206, 177: 1.8133, 180: 5.5038, 189: 6.2957, 198: 1.9944, 199: 4.0192, 201: 2.9839, 202: 2.2084, 207: 22.8698, 218: 2.3475, 227: 2.1576, 237: 2.164, 239: 2.1868, 240: 4.6147, 244: 2.0306, 252: 2.1009, 257: 2.2023, 258: 2.3354, 259: 4.2508, 262: 2.0499, 278: 7.1201, 285: 11.2756, 292: 2.2484, 312: 2.4073, 315: 2.4356, 319: 2.2602, 320: 2.3292, 331: 4.5593, 335: 3.145, 349: 2.4152, 357: 7.6163, 366: 2.3913, 368: 2.7112, 377: 5.5027, 394: 2.5827, 395: 2.5559, 409: 4.9704, 426: 2.4776, 427: 2.6685, 432: 2.4168, 433: 10.7585, 444: 2.5655, 454: 5.4741, 455: 2.9369, 462: 2.6039, 464: 2.9625, 470: 2.5808, 484: 2.5664, 491: 6.678, 499: 2.8416, 524: 2.7257, 525: 5.5794, 534: 2.6752, 549: 7.9197, 569: 2.6534, 570: 2.7011, 575: 2.8711, 593: 2.7541, 606: 2.8013, 615: 2.9523, 651: 2.8843, 693: 3.003, 700: 3.4199, 708: 2.8356, 709: 5.9539, 720: 2.86, 721: 2.9784, 733: 3.1056, 737: 3.0822, 738: 3.0429, 741: 5.5736, 758: 2.9342, 797: 2.9035, 803: 3.1937, 809: 2.8593, 825: 3.0715, 828: 2.9302, 839: 3.1368, 922: 3.407, 929: 3.2288, 950: 3.0745, 963: 3.3105, 968: 3.0593, 1028: 3.3571, 1139: 3.4286, 1171: 3.3057, 1208: 3.495, 1248: 3.8522, 1276: 3.555, 1278: 3.4253, 1279: 3.3714, 1289: 4.3619, 1302: 7.8279, 1303: 3.6715, 1311: 3.3541, 1349: 3.3807, 1372: 3.6591, 1436: 3.4395, 1460: 4.2643, 1474: 3.8912, 1499: 7.0853, 1502: 3.7375, 1519: 3.7287, 1526: 3.6577, 1549: 3.58, 1785: 22.2757, 1823: 3.745, 1843: 4.1572, 1911: 3.8114, 1980: 3.8656, 2012: 3.8455, 2054: 3.9874, 2116: 4.1195, 2146: 3.9069, 2272: 3.9354, 2299: 3.9798, 2320: 3.95, 2368: 4.2346, 2397: 4.0811, 2401: 4.2201, 2406: 4.0543, 2435: 8.2304, 2474: 4.0791, 2609: 4.1594, 2633: 4.1527, 2712: 8.3879, 2804: 4.2081, 2856: 4.2058, 2977: 17.6057, 3237: 4.4101, 3304: 4.7333, 3555: 4.5304, 3573: 4.918, 3672: 4.511, 3682: 4.5142, 3715: 4.5272, 3966: 4.5843, 4107: 4.6486, 4151: 15.781, 4229: 4.6786, 4340: 4.7414, 4401: 4.7743, 4425: 4.7373, 4524: 4.7743, 4535: 5.0523, 4599: 5.3381, 4677: 5.115, 4684: 4.8801, 4721: 4.8302, 4868: 4.8617, 4893: 4.8755, 4937: 4.8942, 5043: 5.2535, 5466: 5.3604, 5737: 5.0634, 5952: 5.1209, 6038: 10.3512, 6166: 6.0611, 6441: 5.5234, 6451: 5.2603, 6764: 5.3308, 6908: 5.368, 6912: 5.368, 7188: 5.4224, 7438: 5.4716, 7517: 5.8319, 7568: 29.0405, 7570: 5.4886, 8044: 6.0461, 8436: 5.6969, 9796: 6.1732, 10890: 11.6638, 11212: 5.8813, 11275: 11.8139, 11580: 5.9603, 11937: 6.0764, 12273: 6.1236, 13372: 6.3589, 14504: 6.5131, 14709: 6.5619, 14840: 6.5872, 14883: 13.5085, 15633: 6.7244, 19155: 7.0319, 19356: 15.3716, 19764: 7.1153, 20283: 16.1071, 20987: 7.3604, 23117: 7.7659, 23986: 16.6097, 25444: 7.7659, 26868: 7.9482, 26931: 7.9482, 27204: 7.9482, 27348: 7.9482, 29339: 8.1713, 30609: 8.3048, 30652: 8.3048, 30714: 8.3048, 30785: 8.3048, 30820: 8.3048, 30904: 8.3048, 30964: 8.3048, 30965: 8.3048, 31214: 8.3048, 31332: 8.3048, 31340: 8.3048, 31372: 8.3048, 31765: 8.3048, 31928: 8.3048, 32003: 8.3048, 32119: 8.3048, 32302: 8.3048, 32466: 8.3048, 32478: 8.3048, 32490: 8.3048, 32504: 8.3048, 32611: 8.3048}), abstract_words_filtered=['advances', 'sequencing', 'technology', 'dramatically', 'shifted', 'way', 'science', 'performed', 'many', 'fields', 'biology,', 'also', 'shifted', 'way', 'education', 'next', 'generation', 'scientists', 'must', 'conducted.', 'however,', 'generation', 'sequencing', 'data', 'restricted', 'large', 'laboratories,', 'consortia', 'laboratories,', 'core', 'sequencing', 'facilities,', 'limiting', 'exposure', 'undergraduates', 'technologies.', 'recent', 'introduction', 'benchtop', 'next', 'generation', 'sequencing', 'platforms', 'promises', 'make', 'moderately', 'high', 'throughput', 'sequencing', 'available', 'many', 'individual', 'research', 'laboratories', 'smaller', 'institutions.', 'major', 'research', 'instrumentation', 'grant', 'supports', 'acquisition', 'ion', 'torrent', 'personal', 'genome', 'machine', '(ion', 'pgm)', 'sequencing', 'platform', 'hope', 'college.', 'ion', 'pgm', 'significant', 'impact', 'undergraduate', 'research', 'training', 'hope', 'college', 'making', 'affordable,', 'high', 'throughput', 'sequencing', 'capacity', 'available', 'faculty', 'undergraduate', 'researchers', 'use', 'established', 'emerging', 'research', 'education', 'programs.', 'programs', 'include', 'research', 'focusing', 'on:', '1)', 'genome', 'sequences', 'viruses', 'infect', 'bacteria', '(mycobacteriophage)', 'discovered', 'first', 'year', 'college', 'students', 'part', 'introductory', 'biology', 'laboratory', 'courses;', '2)', 'long-term', 'environmental', 'studies', 'local', 'lake', 'macatawa', 'watershed', 'identify', 'track', 'microbial', 'sources', 'contamination', 'whole', 'genome', 'sequencing', 'novel', 'bacterial', 'isolates;', '3)', 'advancing', 'understanding', 'fundamental', 'gene', 'expression', 'mechanisms', 'mammalian', 'parasite', 'giardia', 'lamblia;', '4)', 'understanding', 'phylogenetic', 'biogeographic', 'relationships', 'among', 'maples', 'found', 'across', 'world;', '5)', 'testing', 'potential', 'horizontal', 'gene', 'transfers', 'plastid', 'genes', 'parasitic', 'plants', 'photosynthetic', 'hosts.', 'projects', 'advance', 'scientific', 'understanding', 'wide', 'range', 'biological', 'environmental', 'problems', 'addressed', 'dna', 'sequencing', 'different', 'scales.', 'project', 'made', 'effective', 'house', 'availability', 'ion', 'pgm,', 'uniquely', 'versatile', 'able', 'generate', 'sequence', 'data', 'vastly', 'different', 'scales', 'cost', 'effective', 'manner.<br/><br/>this', 'sequencing', 'platform', 'allow', 'hope', 'college', 'effectively', 'train', 'undergraduates', 'design', 'experiments', 'requiring', 'sequencing', 'data;', 'usage', 'next', 'generation', 'sequencing', 'technology', 'generate', 'necessary', 'data;', 'analysis,', 'interpretation,', 'dissemination', 'large', 'scale', 'data', 'sets', 'research', 'laboratory', 'classroom', 'laboratory', 'settings.', 'ion', 'pgm', 'immediately', 'enhance', 'projects', 'programs', 'impact', '100', 'undergraduates', 'academic', 'year.', 'first', 'year', 'students', 'impacted', 'hope', \"college's\", 'highly', 'successful', 'yearlong', 'course', 'phage', 'genomics', 'research,', 'designed', 'retain', 'high', 'quality', 'students', 'stem', 'fields.', 'students', 'levels', 'participate', 'research', 'laboratories', 'pi?s', 'collaborators.', 'students', 'upper', 'level', 'biology', 'courses', 'use', 'technology', 'classroom', 'laboratory-based', 'research', 'projects.', 'hope', 'college?s', 'established', 'summer', 'research', 'programs', 'allow', 'us', 'broaden', 'exposure', 'students', 'across', 'nation', 'students', 'underrepresented', 'groups', 'sciences', 'use', 'leading', 'edge', 'sequencing', 'technology.', 'faculty', 'impacted', 'ability', 'take', 'advantage', 'next', 'generation', 'sequencing', 'technology', 'advance', 'projects', 'new', 'innovative', 'areas', 'generate', 'preliminary', 'data', 'sustaining', 'extramurally', 'funded', 'research', 'programs.', 'student-faculty', 'collaborative', 'research', 'disseminated', 'scientific', 'community', 'local,', 'regional,', 'national/international', 'scientific', 'conferences', 'peer-reviewed', 'publications', 'general', 'public', 'community-based', 'popular', 'seminars.', 'sequence', 'data', 'deposited', 'appropriate', 'openly', 'accessible', 'databases.', 'regional', 'national', 'collaborators', 'use', 'ion', 'pgm,', 'broadening', 'impact', 'beyond', 'hope', 'college', 'students', 'west', 'michigan', 'throughout', 'nation.'])\n",
      "Row(idf_features=SparseVector(50663, {0: 0.5493, 2: 1.6419, 6: 0.8571, 12: 0.7058, 13: 1.4404, 14: 0.2706, 15: 0.2765, 16: 1.5655, 17: 0.285, 19: 0.9223, 20: 1.0703, 21: 1.6702, 22: 1.3289, 24: 1.2127, 25: 0.8137, 29: 0.96, 30: 10.7127, 35: 0.9756, 38: 1.1757, 40: 3.8678, 41: 2.315, 42: 0.8133, 48: 1.2309, 50: 4.3509, 51: 0.8589, 54: 1.5077, 59: 5.1472, 60: 1.419, 61: 2.6825, 62: 1.5942, 64: 1.3804, 72: 1.5405, 79: 3.016, 83: 1.8098, 89: 1.5185, 91: 1.5101, 96: 3.2722, 104: 1.774, 110: 3.19, 125: 3.3303, 126: 1.9805, 131: 1.7519, 137: 4.7316, 145: 2.2249, 146: 1.8156, 149: 2.3191, 185: 2.6232, 187: 1.932, 199: 2.0096, 202: 2.2084, 204: 10.334, 208: 2.0488, 211: 2.5105, 222: 2.0766, 233: 2.3561, 237: 2.164, 241: 9.5163, 243: 2.1991, 277: 2.4097, 285: 4.5102, 292: 2.2484, 345: 2.55, 390: 2.4621, 416: 3.1199, 430: 2.5325, 432: 2.4168, 433: 5.3793, 435: 2.4038, 446: 2.634, 460: 2.5692, 474: 2.6907, 511: 5.4118, 552: 2.6439, 558: 2.5655, 564: 5.7064, 568: 2.7001, 585: 2.8001, 595: 2.6985, 682: 2.8254, 683: 29.0671, 711: 5.8134, 712: 3.0884, 720: 2.86, 721: 2.9784, 743: 2.8718, 885: 3.034, 907: 3.008, 937: 3.2782, 972: 3.227, 973: 6.9439, 996: 3.1716, 1039: 3.5174, 1116: 3.348, 1198: 3.2725, 1235: 3.4528, 1250: 3.348, 1332: 11.7524, 1353: 22.5709, 1354: 3.3817, 1415: 7.5817, 1493: 3.6227, 1527: 3.539, 1599: 3.5966, 1764: 3.6646, 1819: 3.9481, 1844: 3.8757, 1870: 3.9354, 1914: 3.9536, 1989: 3.8406, 2122: 4.0543, 2550: 4.098, 3107: 4.2975, 3195: 4.9933, 3313: 4.7056, 3396: 5.4066, 3613: 5.1882, 3704: 4.537, 4039: 4.6267, 4044: 4.645, 4157: 4.9277, 4870: 5.009, 4939: 4.9475, 5499: 5.0523, 5860: 5.1328, 6029: 5.1509, 6192: 5.2073, 7117: 5.3987, 7311: 5.4467, 7777: 5.5234, 8032: 5.5503, 8092: 5.5503, 8317: 5.5873, 11282: 6.2434, 12231: 6.1236, 13744: 6.5372, 13800: 6.4006, 14738: 6.5619, 15008: 14.1454, 15400: 6.6954, 15659: 6.8168, 18752: 6.9549, 23311: 7.6117, 24425: 7.6858, 31134: 8.3048}), abstract_words_filtered=['description', '(provided', 'applicant):', 'power', 'zebrafish', 'system', 'stems', 'utility', 'developmental', 'biology', 'model', 'combined', 'ease', 'genetic', 'manipulation', 'experimentation.', 'understanding', 'key', 'genetic', 'mechanisms', 'vertebrate', 'development', 'propelled', 'phenotypic', 'characterization,', 'genetic', 'mapping', 'positional', 'cloning', 'induced', 'spontaneous', 'mutations', 'zebrafish.', 'however,', 'potential', 'system', 'fully', 'realized,', 'inefficient', 'microsatellite-based', 'mapping', 'remains', 'primary', 'method', 'field.', 'propose', 'apply', 'technological', 'computational', 'advances', 'present', 'day', 'genomics', 'genetic', 'mapping', 'zebrafish', 'system.', 'specifically,', 'propose', 'develop', 'method', 'rapid', 'accurate', 'mapping', 'recessive', 'zebrafish', 'mutants', 'using', 'next', 'generation', 'sequencing', '(ngs)', 'pooled', 'samples.', 'also', 'propose', 'investigate', 'parameters', 'screen', 'design', 'sample', 'analysis', 'optimize', 'use', 'protocol.', 'finally,', 'aim', 'develop', 'methods', 'identification', 'causal', 'mutation', 'among', 'variants', 'discovered', 'within', 'mapping', 'interval.', 'application', 'ngs', 'technology,', 'complemented', 'specifically', 'developed', 'computational', 'techniques,', 'provide', 'efficient,', 'accurate', 'inexpensive', 'method', 'genetic', 'mapping', 'zebrafish.', 'approach', 'enable', 'simultaneous', 'identification', 'informative', 'genetic', 'markers,', 'mapping', 'mutation', 'position,', 'potential', 'identification', 'causal', 'sequence', 'change', 'single', 'experiment.', 'data', 'obtained', 'genomic', 'analyses', 'methods', 'developed', 'made', 'available', 'zebrafish', 'community.', 'importantly,', 'approaches', 'also', 'widely', 'applicable', 'genetic', 'analysis', 'model', 'systems.', '', 'public', 'health', 'relevance:', 'analysis', 'zebrafish', 'mutants', 'enabled', 'identification', 'genes', 'contributing', 'fundamental', 'biological', 'processes,', 'including', 'human', 'diseases;', 'however,', 'methods', 'used', 'mutant', 'mapping', 'gene', 'discovery', 'inefficient.', 'here,', 'propose', 'fast', 'cost-effective', 'method', 'genetic', 'mapping', 'mutation', 'identification', 'using', 'next', 'generation', 'sequencing.', 'facilitate', 'rapid', 'discovery', 'gene', 'function', 'translation', 'knowledge', 'biomedical', 'investigation.'])\n",
      "Row(idf_features=SparseVector(50663, {2: 1.6419, 4: 1.5891, 8: 12.3805, 9: 2.321, 11: 0.7721, 13: 10.0831, 24: 1.2127, 25: 1.6274, 45: 1.1525, 47: 1.1903, 52: 1.2016, 53: 2.372, 55: 1.0729, 58: 4.8335, 63: 1.2769, 81: 1.5428, 82: 1.8006, 84: 1.4788, 88: 1.4663, 91: 4.5303, 98: 1.8566, 101: 1.7807, 108: 4.0045, 111: 3.1334, 112: 1.7191, 128: 1.9306, 130: 5.2479, 131: 1.7519, 141: 1.68, 144: 5.3257, 145: 2.2249, 148: 1.813, 161: 2.036, 188: 1.9296, 199: 4.0192, 203: 7.741, 220: 2.1, 235: 4.371, 240: 2.3073, 244: 4.0612, 247: 2.0794, 266: 9.5484, 281: 21.2712, 284: 2.2724, 285: 4.5102, 289: 2.1953, 293: 4.6055, 319: 4.5204, 322: 4.5593, 327: 2.463, 331: 9.1187, 348: 2.2932, 356: 2.2434, 357: 2.5388, 360: 2.2534, 365: 2.6174, 375: 2.3206, 392: 10.7171, 402: 10.0561, 409: 4.9704, 417: 2.6444, 432: 2.4168, 433: 5.3793, 438: 2.5105, 440: 7.9586, 444: 2.5655, 447: 2.4518, 467: 2.6374, 468: 2.6281, 472: 8.8794, 476: 5.2689, 486: 8.0767, 510: 2.5532, 522: 2.6286, 524: 5.4513, 527: 8.0025, 530: 2.8526, 531: 2.6767, 562: 5.816, 568: 2.7001, 570: 2.7011, 575: 2.8711, 577: 2.6706, 599: 2.7931, 600: 2.7469, 644: 2.9422, 653: 2.8453, 669: 5.6712, 712: 3.0884, 713: 2.7715, 718: 2.8612, 730: 5.7523, 741: 2.7868, 742: 2.8326, 745: 2.7547, 746: 12.3659, 750: 3.1183, 758: 2.9342, 793: 2.9389, 829: 2.9093, 840: 3.1741, 850: 6.547, 887: 9.7677, 894: 6.3567, 925: 3.2477, 932: 3.0631, 955: 3.1851, 984: 6.4972, 1022: 3.1775, 1060: 3.1582, 1143: 7.5943, 1174: 3.4428, 1206: 3.5342, 1228: 3.6687, 1281: 8.0564, 1294: 3.3622, 1384: 3.48, 1501: 71.5236, 1625: 3.6201, 1740: 8.4163, 1916: 4.0262, 1951: 3.7831, 2272: 3.9354, 2285: 12.8308, 2300: 12.7929, 2332: 8.3508, 2403: 8.3554, 2490: 4.0625, 2503: 4.0811, 2527: 4.2975, 2550: 4.098, 2667: 4.1174, 2746: 19.8906, 2791: 4.2081, 2794: 4.187, 2833: 4.2744, 2888: 4.2201, 2897: 4.237, 2977: 4.4014, 3086: 8.9964, 3464: 4.9132, 3529: 9.419, 3530: 15.1238, 3554: 22.9596, 3570: 4.5207, 3682: 4.5142, 3706: 4.5672, 3710: 4.671, 3739: 4.6018, 3758: 4.6195, 3769: 4.547, 3804: 10.9772, 4119: 4.6376, 4183: 4.6862, 4316: 16.6237, 4385: 10.1379, 4416: 5.2401, 4459: 10.1949, 4469: 10.2065, 4557: 4.8391, 4561: 4.7997, 4743: 4.8481, 4891: 17.8809, 4936: 4.8848, 5009: 10.6762, 5228: 4.9778, 5494: 5.0143, 5861: 5.1091, 6353: 5.2603, 6364: 11.8665, 6506: 11.9206, 6509: 11.9206, 6530: 11.9206, 7685: 5.5146, 7903: 5.5594, 8022: 5.5412, 8327: 5.5873, 8759: 5.6456, 9834: 5.8199, 10741: 5.729, 11461: 5.8813, 11474: 5.894, 11648: 5.9603, 11652: 5.9603, 11654: 5.9603, 11657: 5.9603, 11658: 5.9603, 11659: 5.9603, 11660: 5.9603, 11669: 5.9603, 11670: 5.9603, 11672: 5.9603, 11673: 5.9603, 11686: 5.9603, 11687: 5.9603, 11688: 5.9603, 11692: 5.9603, 11693: 5.9603, 11694: 5.9603, 11696: 5.9603, 11697: 5.9603, 11698: 5.9603, 11714: 5.9603, 11719: 5.9603, 11724: 5.9603, 11726: 5.9603, 11731: 5.9603, 11734: 5.9603, 11740: 5.9603, 11744: 5.9603}), abstract_words_filtered=['overall', 'summary', 'specific', 'aims:', 'network', 'resource', 'component', 'mayo', 'clinic', 'pgrn', 'renewal', 'application', 'represents', 'request', 'funding', \"'network\", \"resource'\", 'designed', 'provide', 'entire', 'pgrn', 'access', 'latest', 'techniques', 'rapidly', 'evolving', 'field', \"'next\", \"generation'\", 'dna', 'sequencing.', 'access', 'provided', 'recurring,', 'peer-reviewed', 'process', 'available', 'pgrn', 'centers', 'similar', 'current', \"'pgrn-riken'\", 'process', 'obtaining', 'access', 'pharmacogenomic', 'gwas', 'genotyping', 'pgrn-riken', 'collaboration.', 'proposed', \"'pgrn\", 'nehwork', \"resource'\", 'one', 'three', 'coordinated', 'applications', 'current', 'pgrn', 'centers,', 'similar', 'goals', '-', 'clearly', 'indicating', 'unmet', 'need', 'pharmacogenomic', 'research', '-', 'need', 'emphasized', 'letters', 'support', 'current', 'pgrn', 'centers', 'attached', 'application.', 'letters', 'sent', 'eight', 'different', 'pgrn', 'research', 'centers.', 'although', 'pharmacoqenomics', 'always', 'required', 'access', 'dna', 'sequencing,', 'need', 'accentuated', 'bv', 'recent', 'dramatic', 'changes', 'dna', 'seguencing', 'technology', 'bv', 'eguallv', 'dramatic', 'outcomes', 'application', 'genome-wide', 'approaches', 'pharmacogenomics.', 'proposals', 'designed', 'provide', \"'link'\", 'pgrn', 'large,', 'experienced', 'genomics', 'center', 'expertise', 'application', 'evaluation', 'dna', 'sequencing', 'technology.', 'issue', 'whether', 'pharmacogenomics', 'discipline', 'requires', 'access', 'latest', 'rapidly', 'evolving', 'dna', 'sequencing', 'techniques,', 'achieve', 'goal', 'way', 'best', 'advance', 'science', 'affordable', 'open', 'pgrn', 'centers.', 'specifically,', 'mayo', 'pgrn', 'proposing', 'formal', 'collaborative', 'relationship', 'established', 'baylor', 'college', 'medicine', '(bcm)-human', 'genome', 'sequencing', 'center', '(hgsc),', 'one', 'three', 'nhgrisupported', 'large', 'genome', 'centers', '-', 'open,', 'protocol-based', 'selection', 'process', 'similar', 'well', 'established', 'highly', 'successful', 'pgrn-riken', 'process', 'selection', 'pgrn', 'pharmacogenomic', 'gwas', 'genotyping', 'projects', 'performed', 'collaboration', 'riken.', 'initial', 'focus', 'mayo', 'pgrn-sponsored', \"'next\", \"generation'\", 'dna', 'sequencing', 'network', 'resource', 'sequencing', 'large', 'contiguous', 'regions', 'genome.', 'cases,', 'involve', 'complete', 'resequencing', 'large', 'genes', 'tandemly', 'repeated', 'gene', 'families', 'even', 'entire', 'gene', 'families,', 'e.g.,', \"'cypome',\", 'cytochrome', 'p450', 'genes.', 'cases,', 'wilt', 'involve', 'focused', 'resequencing', 'across', \"'snp\", \"signals'\", 'observed', 'pharmacogenomic', 'gwa', 'studies', '-', 'many', 'cases,', 'gwa', 'studies', 'made', 'possible', 'pgrn-riken', 'collaboration,', 'thus', 'building', 'foundation', 'established', 'successful', 'program.', 'since', 'review', 'panel', 'applications', 'evaluate', 'series', 'network', 'resource', 'applications', 'access', 'next', 'generation', 'dna', 'sequencing,', 'emphasized', 'mayo', 'pgrn', 'looks', 'upon', 'efforts', 'complementary.', 'therefore,', 'possible', 'reviewers', 'may', 'conclude', \"'combination'\", 'proposals', 'supported.', 'proposals', 'share;', '', 'recognition', 'critical', 'need', 'access', 'rapidly', 'evolving', 'dna', 'sequencing', 'techniques', 'order', 'advance', 'science', 'pharmacogenomics.', '', 'recognition', 'established', 'genome', 'centers', 'requisite', 'experience,', 'expertise', 'infrastructure', 'required', 'make', 'effort', 'succeed.', '', 'recognition', \"'process'\", 'access', 'next', 'generation', 'dna', 'sequencing', 'open', 'pgrn', 'centers,', 'based', 'scientific', 'significance', 'feasibility.', 'cost', 'associated', 'studies', 'covered', 'primarily', 'funding', 'available', \"'network\", \"resource'.\"])\n",
      "Row(idf_features=SparseVector(50663, {6: 3.4284, 13: 1.4404, 14: 0.2706, 15: 0.2765, 17: 0.285, 21: 0.8351, 25: 1.6274, 27: 3.0172, 29: 0.96, 30: 4.0173, 31: 3.1834, 32: 1.0804, 41: 1.1575, 58: 1.6112, 68: 1.2561, 70: 1.6693, 87: 1.6071, 109: 1.619, 110: 1.595, 123: 1.7821, 127: 1.7644, 135: 1.7874, 139: 2.0417, 140: 1.7577, 158: 1.9209, 163: 1.9843, 167: 2.1131, 191: 2.1169, 199: 2.0096, 202: 6.6251, 224: 2.3741, 227: 2.1576, 239: 2.1868, 240: 4.6147, 255: 2.0854, 256: 2.2335, 271: 2.4946, 321: 2.481, 326: 2.2387, 328: 2.2531, 364: 2.3883, 373: 3.961, 413: 2.4469, 478: 3.694, 539: 3.0503, 540: 2.8218, 613: 2.8398, 644: 2.9422, 652: 2.8945, 653: 2.8453, 657: 3.3212, 685: 5.9155, 703: 2.852, 723: 3.2041, 852: 2.9342, 854: 3.0414, 886: 3.0311, 904: 3.212, 946: 3.0946, 947: 3.4539, 963: 3.3105, 993: 3.093, 998: 3.351, 1061: 3.2495, 1086: 3.2933, 1279: 6.7428, 1312: 3.5587, 1334: 3.3964, 1355: 3.655, 1390: 3.594, 1412: 3.4318, 1527: 3.539, 1543: 3.6482, 1579: 3.6307, 1620: 3.6618, 1755: 3.93, 1916: 4.0262, 2132: 3.8622, 2305: 3.9629, 2323: 3.9874, 2687: 10.2655, 2800: 4.2975, 2881: 4.2543, 2968: 4.3591, 3163: 4.3591, 3167: 4.3674, 3391: 4.7174, 3971: 5.2401, 4147: 5.0037, 4514: 4.7701, 4615: 9.9761, 4646: 4.817, 4849: 4.8895, 5257: 4.9881, 5883: 5.1756, 6025: 5.2138, 6127: 5.2138, 6163: 5.1756, 6746: 5.3308, 7295: 5.4386, 7604: 5.4972, 10831: 26.8976, 11358: 5.844, 11454: 6.4896, 11561: 20.4503, 12213: 6.0919, 13384: 6.2995, 13801: 13.7669, 13919: 6.4441, 14259: 6.4896, 15137: 6.6398, 15199: 7.0727, 15491: 6.6954, 18721: 6.9549, 19164: 15.3716, 19425: 7.0727, 19890: 7.4782, 20427: 7.255, 20636: 7.3063, 22159: 7.5427, 23758: 7.6117, 24203: 7.6858, 24382: 7.6858, 24743: 7.6858, 24852: 7.6858, 25232: 7.6858}), abstract_words_filtered=['description', '(provided', 'applicant):', 'human', 'african', 'trypanosomiasis,', 'also', 'known', 'sleeping', 'sickness,', 'zoonotic', 'disease', 'affecting', 'thousands', 'people', 'across', 'sub-saharan', 'africa', 'periodic', 'epidemics', 'caused', 'significant', 'social', 'economic', 'disruption.', 'trypanosome', 'responsible', 'sleeping', 'sickness', 'morphologically', 'indistinguishable', 'closely', 'related', 'parasites', 'found', 'animals,', 'leading', 'confusion', 'regarding', 'origins', 'human', 'disease', 'outbreaks.', 'project', 'seeks', 'improve', 'tools', 'available', 'understanding', 'epidemiology', 'neglected', 'disease.', 'immediate', 'aims', 'develop', 'robust', 'broadly', 'applicable', 'panel', 'genetic', 'markers', 'validate', 'pilot', 'study', 'uganda,', 'disease-endemic', 'country', 'currently', 'experiencing', 'severe', 'epidemic.', 'project', 'employ', 'next-generation', 'sequencing', 'multiple', 'trypanosome', 'genomes,', 'selected', 'encompass', 'complicated', 'evolutionary', 'relationships', 'observed', 'among', 'trypanosomes', 'preliminary', 'studies.', 'genetic', 'polymorphisms', 'identified', 'genome', 'comparisons', 'evaluated', 'ability', 'resolve', 'fine', 'scale', 'population', 'structure', 'distinguish', 'human-infective', 'animal-restricted', 'parasites', 'ugandan', 'disease', 'focus.', 'longer', 'term,', 'markers', 'developed', 'project', 'provide', 'foundation', 'cataloguing', 'full', 'diversity', 'trypanosomes', 'nature,', 'identifying', 'pool', 'trypanosomes', 'relevant', 'human', 'disease,', 'characterizing', 'relationships', 'among', 'trypanosome', 'populations', 'distributed', 'across', 'eastern', 'africa.', 'addition,', 'comparisons', 'among', 'genomes', 'animal-restricted', 'human-disease', 'causing', 'trypanosomes', 'provide', 'insight', 'functional', 'genetic', 'differences', 'conferring', 'human', 'infectivity.'])\n",
      "Row(idf_features=SparseVector(50663, {0: 1.0987, 2: 0.821, 5: 2.0061, 6: 0.8571, 8: 4.1268, 10: 0.8027, 13: 17.2854, 16: 1.5655, 20: 5.3513, 25: 0.8137, 27: 1.0057, 28: 1.779, 32: 1.0804, 35: 0.9756, 36: 5.0253, 37: 1.0098, 45: 1.1525, 47: 1.1903, 52: 2.4033, 55: 3.2188, 58: 4.8335, 64: 2.7608, 65: 3.0308, 72: 1.5405, 78: 6.2553, 79: 1.508, 86: 1.5433, 95: 1.576, 101: 1.7807, 108: 4.0045, 112: 1.7191, 115: 1.7669, 126: 1.9805, 127: 1.7644, 128: 3.8611, 132: 1.8172, 135: 1.7874, 145: 2.2249, 150: 1.9204, 157: 1.8619, 161: 2.036, 182: 4.4258, 190: 2.1075, 198: 7.9776, 199: 2.0096, 208: 2.0488, 219: 2.0125, 234: 3.0829, 285: 4.5102, 309: 2.269, 329: 2.2361, 338: 2.6634, 344: 2.4199, 362: 9.3739, 368: 2.7112, 370: 4.8711, 374: 7.627, 394: 5.1653, 420: 2.5949, 427: 2.6685, 433: 10.7585, 440: 5.3057, 444: 2.5655, 446: 5.268, 454: 2.737, 464: 2.9625, 467: 18.462, 470: 5.1616, 472: 2.9598, 477: 2.7267, 485: 2.6752, 486: 2.6922, 489: 15.9087, 562: 2.908, 575: 5.7423, 577: 2.6706, 597: 2.9119, 613: 2.8398, 665: 3.0051, 687: 2.873, 708: 2.8356, 714: 3.0938, 755: 2.9054, 774: 3.164, 791: 3.2468, 822: 3.235, 839: 3.1368, 840: 3.1741, 930: 3.1159, 1049: 3.4731, 1066: 3.2577, 1116: 3.348, 1143: 15.1885, 1145: 3.4584, 1215: 7.16, 1269: 6.8768, 1293: 3.4973, 1306: 3.6214, 1334: 3.3964, 1360: 3.6618, 1373: 3.5185, 1398: 7.1473, 1457: 3.7054, 1499: 3.5427, 1502: 11.2126, 1512: 3.8178, 1528: 3.6771, 1535: 3.5102, 1628: 3.6057, 1690: 3.6267, 1702: 4.0222, 1720: 3.8791, 1844: 3.8757, 1854: 3.9445, 1879: 3.9051, 1889: 3.961, 2003: 3.8572, 2005: 4.5878, 2030: 4.6267, 2106: 3.8588, 2146: 3.9069, 2270: 4.1893, 2272: 3.9354, 2297: 4.0707, 2383: 4.194, 2411: 4.113, 2466: 4.0422, 2493: 8.5539, 2775: 4.2395, 2812: 9.1141, 2915: 4.2693, 2985: 4.3564, 3024: 4.547, 3091: 13.8479, 3094: 4.547, 3095: 4.6124, 3215: 4.34, 3258: 4.4608, 3263: 4.3536, 3438: 4.7785, 3474: 4.5239, 3636: 9.1826, 3667: 4.5672, 3721: 9.4427, 3767: 4.6598, 3783: 4.9726, 3927: 4.574, 3980: 4.6978, 3982: 9.3879, 4172: 9.5654, 4207: 4.671, 4340: 4.7414, 4385: 5.069, 4469: 5.1033, 4531: 9.7696, 4610: 4.804, 4648: 4.8214, 4722: 4.8302, 5139: 5.8563, 5243: 5.1033, 5260: 5.3529, 5472: 5.3529, 5694: 10.8449, 5864: 17.7998, 5941: 5.1209, 6139: 5.1756, 7071: 5.4467, 7561: 5.4886, 8844: 5.6556, 9722: 19.0768, 10786: 5.7399, 10803: 6.1903, 10945: 5.844, 11257: 5.8199, 11816: 6.0461, 12025: 6.0611, 12156: 6.2077, 12200: 13.6991, 12508: 6.1564, 13042: 6.2254, 13374: 6.2995, 13662: 6.3589, 14827: 6.5872, 15748: 6.7543, 16062: 6.7543, 18851: 6.9927, 19090: 7.0319, 19118: 7.0319, 19124: 7.0319, 19274: 7.0319, 19372: 7.0319}), abstract_words_filtered=['next', 'generation', 'sequencing', 'genotyping', 'core', 'facility.', 'goal', 'next', 'gen', 'sequencing', 'component', 'make', \"'next\", \"generation'\", 'sequencing', 'related', 'services', 'widely', 'available', 'unc', 'lccc', 'members', 'cost-effective', 'prices.', 'services', 'include:', 'whole', 'genome', 'shotgun', 'sequencing', 'human', 'model', 'organisms,', 'chromatin-ip', 'sample', 'sequencing,', 'measuring', 'rna', 'expression', 'digital', 'gene', 'expression', 'rna-seq,', 'snp', 'typing,', 'polymorphism', 'screens.', 'component', 'adds', 'value', 'cancer', 'center', 'putting', 'complex', 'expensive', 'dna/rna', 'sequencing', 'technology', 'within', 'easy', 'reach', 'unc', 'lccc', 'members.', 'strongly', 'integrated', 'genomics', 'bioinformatics', 'groups', 'interact', 'user', 'ensure', 'robust', 'data', 'archived', 'appropriately', 'future', 'use.', 'component', 'key', 'unc', \"lccc's\", 'participation', 'cancer', 'genome', 'atlas', 'project', 'serving', 'lccc', 'needs.', 'future', 'plans', 'include', 'expanding', 'staff', 'equipment', 'full', 'array', 'next', 'gen', 'sequencing', 'services', 'may', 'provided', 'timely', 'manner,', 'well', 'acquisition', 'next', 'generation', 'technology.', '', '', 'goal', 'dna', 'sequencing', 'component', 'produce', 'high', 'quality', 'sequencing', 'genotyping', 'data', 'rapidly', 'reasonable', 'cost', 'unc-ch', 'researchers', 'using', 'state-of-the-art', 'technologies', 'well', 'provide', 'technical', 'support', 'enhance', 'value', 'results', 'produced.', 'services', 'include', 'dna', 'sequencing', 'microsatellite', 'genotyping.', 'component', 'adds', 'value', 'cancer', 'center', 'producing', 'high-quality', 'data', 'competitive', 'price', 'providing', 'benefits', 'local', 'resource', 'assist', 'in:', 'scientific', 'strategies,', 'sample', 'preparation,', 'sequencing', 'difficult', 'regions,', 'assistance', 'data', 'interpretation.', 'component', 'wide', 'user', 'base', '220', 'different', 'laboratories', 'users', '2009.', 'future', 'plans', 'include', 'continued', 'use', 'existing', 'equipment,', 'reduction', 'chemistry', 'costs,', 'evaluation', 'future', 'sequencing', 'technology', 'options', 'long', 'read', 'single', 'molecule', 'sequencing', 'technologies.', 'goal', 'high', 'throughput', 'genotyping', 'component', 'produce', 'high-quality', 'genotyping', 'data', 'rapidly', 'reasonable', 'cost', 'researchers', 'using', 'state-of-the-art', 'technologies.', 'services', 'include', 'snp', 'genotyping,', 'copy', 'number', 'variation', 'dna', 'methylation', 'profiling', 'analysis.', 'new', 'technologies', 'and/or', 'applications', 'explored', 'increase', \"core's\", 'competitive', 'edge', 'allow', 'genome', 'wide', 'association', 'study', 'rare', 'variants.', 'core', 'components', 'used', 'multiple', 'members', 'core,', 'peer', 'reviewed', 'member', 'used', 'exceeds', '70%.', 'proposed', 'budget', '$241,797', 'less', '5%', \"core's\", 'operating', 'cost.', 'budget', 'provides', 'salary', 'stability', 'vital', 'core', 'staff', 'work', 'members', 'adopt', 'novel', 'technologies.'])\n",
      "Row(idf_features=SparseVector(50663, {0: 0.5493, 2: 6.5677, 4: 2.3837, 6: 1.7142, 8: 5.5024, 9: 0.7737, 10: 2.408, 12: 0.7058, 13: 7.2022, 14: 0.2706, 15: 0.2765, 17: 0.285, 19: 0.9223, 27: 1.0057, 29: 1.9201, 30: 1.3391, 31: 1.0611, 35: 0.9756, 37: 1.0098, 41: 1.1575, 42: 0.8133, 45: 2.3049, 47: 1.1903, 50: 1.0877, 51: 0.8589, 55: 1.0729, 58: 1.6112, 61: 1.3413, 63: 1.2769, 64: 2.7608, 73: 1.5784, 83: 1.8098, 87: 1.6071, 90: 1.5406, 91: 1.5101, 104: 1.774, 122: 2.0274, 130: 1.7493, 151: 1.7754, 153: 1.7957, 158: 1.9209, 166: 2.3579, 178: 2.2087, 198: 1.9944, 208: 2.0488, 217: 2.1412, 223: 2.1934, 252: 2.1009, 281: 2.6589, 284: 4.5449, 292: 2.2484, 303: 2.6039, 319: 2.2602, 328: 2.2531, 356: 2.2434, 360: 2.2534, 366: 2.3913, 370: 2.4356, 390: 2.4621, 404: 2.8189, 407: 2.968, 409: 2.4852, 438: 2.5105, 440: 7.9586, 451: 2.5473, 466: 2.7862, 505: 2.7681, 511: 2.7059, 522: 2.6286, 539: 6.1007, 562: 2.908, 582: 22.7275, 593: 2.7541, 656: 3.0355, 670: 2.8183, 678: 2.8718, 698: 3.2423, 709: 5.9539, 732: 2.9177, 872: 3.0868, 910: 2.9853, 911: 3.3996, 1020: 14.1415, 1026: 6.4416, 1042: 3.3242, 1045: 3.4166, 1049: 3.4731, 1060: 3.1582, 1137: 3.3922, 1156: 3.8656, 1161: 3.3105, 1238: 3.8455, 1282: 3.3975, 1449: 3.6468, 1887: 4.4101, 1916: 4.0262, 1917: 3.9931, 2013: 4.0938, 2084: 21.2965, 2131: 4.1174, 2251: 3.9372, 2308: 13.2478, 2322: 3.95, 2413: 4.0183, 2622: 4.1731, 2688: 5.8081, 2708: 4.1415, 2729: 9.4586, 2779: 4.201, 2823: 4.1916, 3148: 4.4396, 3692: 4.574, 3710: 4.671, 4150: 4.7785, 4786: 5.3987, 5079: 4.9326, 5464: 11.3728, 5641: 5.2269, 5666: 5.1328, 6154: 5.2949, 6158: 5.1882, 6171: 5.7182, 6496: 5.2672, 6555: 24.9016, 6631: 5.4066, 7478: 5.4716, 7962: 17.9223, 8733: 6.0023, 9192: 5.6658, 10921: 5.9603, 11259: 28.2908, 11396: 28.8249, 11421: 12.7591, 11925: 6.1732, 12215: 6.2077, 12413: 21.6187, 14657: 14.4125, 15015: 6.6132, 15154: 6.8835, 15297: 6.6672, 15578: 6.7244, 17330: 7.0319, 17526: 6.8835, 18772: 7.0319, 20025: 7.2062, 20095: 7.2062, 20097: 7.2062, 20138: 7.2062, 20174: 7.2062, 20199: 7.2062, 20218: 7.2062, 20238: 7.2062, 20244: 7.2062, 20252: 7.2062, 20270: 7.2062, 20306: 7.2062, 20335: 7.2062}), abstract_words_filtered=['description', '(provided', 'applicant):', 'actinomycetes', 'tremendous', 'potential', 'produce', 'natural', 'products', 'medicinal', 'value.', 'long-term', 'goal', 'project', 'develop', 'novel', 'agents', 'treatment', 'human', 'disease', 'facilitated', 'application', 'next-generation', 'dna', 'sequencing', 'rapidly', 'access', 'actinomycete', 'genetic', 'information.', 'marine', 'streptomyces', 'sp.', 'cnh-189', 'producer', 'structurally', 'fascinating', 'meroterpenoids', 'potent', 'activity', 'broad', 'panel', 'multi-drug', 'resistant', 'staphylococcus', 'aureus', 'strains.', 'overall', 'research', 'objectives', 'beta-test', 'illumina', 'dna', 'sequencing', 'rapidly', 'accessing', 'natural', 'product', 'gene', 'cluster', 'sequences', 'streptomyces', 'sp.', 'cnh-189,', 'use', 'genetics', 'biochemistry', 'link', 'meroterpenoid', 'biosynthesis', 'identified', 'gene', 'clusters.', 'preliminary', 'studies', 'include', 'construction', 'genomic', 'dna', 'fragment', 'libraries', 'ready', 'paired-end', 'illumina', 'sequencing,', 'isolation', 'fosmid', 'clone', 'contains', 'large', 'part', 'biosynthetic', 'gene', 'cluster', 'meroterpenoid', 'biosynthesis.', 'series', 'experiments,', 'propose', 'evaluate', 'illumina', 'sequencing', 'used', 'find', 'gene', 'clusters', 'streptomyces', 'sp.', 'cnh-189', 'actinomycetes,', 'obtain', 'complete', 'coverage', 'gene', 'cluster,', 'probe', 'biosynthesis', 'vitro', 'biochemistry,', 'gene', 'disruption,', 'comparative', 'metabolite', 'profiling.', 'unique', 'chemical', 'structures', 'biological', 'activities', 'natural', 'products', 'isolated', 'streptomyces', 'sp.', 'cnh-189', 'may', 'lead', 'new', 'chemotherapies', 'fundamental', 'biochemical', 'knowledge.', '', 'public', 'health', 'relevance:', 'many', 'pharmaceuticals', 'currently', 'used', 'clinic', 'natural', 'product', 'based.', 'therefore,', 'natural', 'product', 'discovery,', 'understanding', 'biosynthetic', 'pathways,', 'engineering', 'production', 'natural', 'product', 'analogs', 'key', 'development', 'new', 'human', 'chemotherapies.', 'next-generation', 'dna', 'sequencing', 'short', 'read', 'assembly', 'ofers', 'new', 'potentialy', 'general', 'platform', 'accessing', 'gene', 'clusters', 'facilitate', 'genome', 'mining', 'understanding', 'clustered', 'biosynthetic', 'pathways', 'microorganisms.', 'proposed', 'research', 'first', 'time', 'evaluate', 'illumina', 'sequencing', 'platform', 'rapidly', 'accessing', 'biosynthetic', 'gene', 'clusters', 'actinomycetes.', 'successful', 'completion', 'research', 'may', 'advance', 'field', 'natural', 'product', 'discovery', 'biosynthesis.'])\n",
      "Row(idf_features=SparseVector(50663, {4: 0.7946, 8: 4.1268, 10: 1.6053, 13: 15.8449, 18: 1.1638, 20: 1.0703, 26: 3.0367, 34: 1.1249, 36: 6.7004, 48: 1.2309, 54: 1.5077, 55: 1.0729, 62: 1.5942, 63: 1.2769, 64: 1.3804, 78: 12.5107, 83: 5.4294, 92: 2.0363, 100: 2.4572, 126: 3.961, 128: 1.9306, 130: 1.7493, 131: 1.7519, 137: 2.3658, 148: 1.813, 157: 1.8619, 163: 1.9843, 164: 1.8669, 168: 2.2999, 176: 1.8206, 177: 1.8133, 180: 8.2557, 189: 2.0986, 218: 2.3475, 219: 2.0125, 220: 2.1, 222: 2.0766, 225: 2.2639, 229: 2.1009, 233: 2.3561, 262: 2.0499, 285: 11.2756, 303: 7.8118, 318: 2.5455, 327: 2.463, 328: 2.2531, 331: 2.2797, 334: 2.3442, 355: 2.5215, 372: 2.4283, 390: 2.4621, 394: 2.5827, 398: 2.4797, 433: 13.4482, 440: 2.6529, 464: 2.9625, 472: 5.9196, 475: 2.686, 483: 2.7794, 487: 5.1027, 489: 6.3635, 497: 5.5702, 511: 2.7059, 516: 2.585, 556: 2.5992, 562: 2.908, 572: 3.1215, 575: 5.7423, 643: 6.2982, 665: 12.0204, 669: 2.8356, 673: 3.0791, 720: 5.7199, 731: 2.9442, 734: 2.8007, 743: 2.8718, 746: 3.0915, 774: 3.164, 797: 5.8069, 846: 2.9112, 856: 2.9598, 924: 3.0837, 929: 3.2288, 930: 6.2317, 951: 3.0593, 963: 3.3105, 981: 3.5966, 982: 3.2208, 999: 3.2866, 1017: 6.5247, 1043: 3.2459, 1045: 3.4166, 1054: 3.3632, 1058: 6.5247, 1073: 3.2468, 1209: 3.533, 1269: 3.4384, 1283: 3.4188, 1372: 3.6591, 1377: 3.7555, 1394: 3.8809, 1481: 3.628, 1502: 7.4751, 1512: 3.8178, 1578: 3.628, 1702: 4.0222, 1743: 3.6983, 1816: 3.6926, 1839: 3.8572, 1841: 3.717, 1861: 4.0707, 1973: 8.0843, 1980: 7.7311, 2031: 4.3427, 2033: 3.9372, 2043: 3.8488, 2084: 4.2593, 2122: 4.0543, 2188: 4.0707, 2225: 3.9427, 2296: 3.95, 2309: 4.1195, 2318: 3.961, 2402: 4.308, 2573: 4.2346, 2597: 4.1549, 2682: 4.4336, 2707: 4.1415, 2713: 4.6304, 2828: 4.2129, 2921: 4.2643, 2988: 4.3319, 3029: 4.2949, 3137: 4.3427, 3142: 4.5983, 3158: 4.5537, 3258: 4.4608, 3373: 4.6053, 3469: 4.4336, 3512: 4.6786, 3766: 4.5638, 4113: 4.6413, 4128: 4.6486, 4157: 4.9277, 4172: 4.7827, 4230: 4.7293, 4340: 4.7414, 4387: 4.7954, 4393: 4.7373, 4435: 4.9676, 4835: 4.8848, 4875: 4.8663, 4942: 4.8895, 5047: 5.0803, 5056: 4.9425, 5169: 5.0917, 5204: 4.9726, 5396: 4.9933, 5427: 4.9933, 5694: 5.4224, 5705: 5.069, 5768: 5.0746, 5942: 5.1268, 6048: 5.1448, 6136: 5.1694, 6461: 5.2603, 6765: 5.3308, 6913: 11.1006, 7364: 5.455, 7518: 5.6258, 7932: 5.5234, 8399: 5.5968, 8487: 6.4896, 11304: 12.0921, 11500: 5.894, 11736: 6.4896, 11756: 5.9741, 11937: 6.0764, 12508: 6.1564, 12742: 6.2077, 12804: 6.2077, 13389: 6.2995, 14318: 6.4896, 17141: 6.8168, 18615: 7.255, 18728: 6.9549, 19062: 15.3716, 19309: 15.3716, 19396: 15.3716, 19479: 7.0727, 19959: 7.1597, 21024: 7.3604, 21446: 7.4175, 22305: 7.5427, 23499: 7.6117, 23953: 7.6858, 24188: 7.6858, 24242: 7.6858, 24312: 7.6858, 24496: 7.6858, 24501: 7.6858, 24578: 7.6858, 24606: 7.6858, 24611: 7.6858, 24626: 7.6858, 24643: 7.6858, 24710: 7.6858, 24793: 7.6858, 24834: 7.6858, 24954: 7.6858, 25030: 7.6858, 25082: 7.6858}), abstract_words_filtered=['maintain', 'build', 'sidney', 'kimmel', 'comprehensive', 'cancer', 'centers', '(skccc)', 'record', 'excellence', 'fields', 'cancer', 'genetics', 'epigenetics,', 'skccc', 'established', 'new', 'next', 'generation', 'sequencing', 'core', '(ngsc)', 'shared', 'resource', 'january', '2009.', 'product', 'convergence', 'advances', 'molecular', 'biology,', 'engineering,', 'computer', 'science,', 'bioinformatics.', 'next', 'generation', 'sequencing', 'features', 'ability', 'sequence', 'billions', 'base', 'pairs', 'dna', 'single', 'run,', 'cost', 'several', 'orders', 'magnitude', 'less', 'previous', 'gold', 'standard', 'sequencing', 'technologies.', 'next', 'generation', 'sequencing', 'extremely', 'versatile', 'technology', 'used', 'investigate', 'multitude', 'genomic', 'processes', 'previously', 'unimaginable', 'genome-wide', 'unbiased', 'fashion.', 'order', 'facilitate', 'widespread', 'adoption', 'relatively', 'new', 'technology,', 'rather', 'administering', 'core', \"'black-box'\", 'facility,', 'core', 'operates', \"'collaborative\", \"core'\", 'users', 'consult', 'collaborate', 'ngsc', 'faculty', 'staff', 'to:', '1)', 'identify', 'optimal', 'molecular', 'biology', 'approaches', '(prior', 'sequencing', 'library', 'construction)', 'synapse', 'research', 'questions', 'capabilities', 'core,', '2)', 'prepare', 'appropriate', 'libraries', 'carry', 'massively', 'parallel', 'sequencing,', '3)', 'carry', 'appropriate', 'primary,', 'secondary,', 'tertiary', 'bioinformatics', 'analyses', 'analyze', 'interpret', 'results.', 'ngsc', 'features', 'applied', 'biosystems', 'soud', 'next', 'generation', 'sequencing', 'instruments', 'capable', 'generating', 'nearly', '100', 'gigabases', 'sequencing', 'data', 'single', 'run,', 'state-of-the-art', 'equipment', 'sequencing', 'library', 'preparation', 'quality', 'control,', 'powerful', 'computational', 'resources', 'including', 'server', 'ten', 'compute', 'nodes', '(each', 'containing', 'quad', 'core', 'intel', 'xeon', 'processors', '16', 'gigabytes', 'memory)', '100', 'terabytes', 'storage', 'dedicated', 'bioinformatics', 'analyses.', 'core', 'currently', 'administered', 'faculty', 'director', 'overseeing', 'activities,', 'faculty', 'co-director', 'overseeing', 'bioinformatics', 'support,', 'laboratory', 'manager,', 'bioinformatics', 'manager.', \"core's\", 'goal', 'flexible', 'possible', 'assisting', 'researchers', 'exploring', 'aspects', 'cancer', 'genetics', 'epigenetics,', 'including,', 'limited', 'to,', 'germline', 'somatic', 'variation/mutation,', 'genomic', 'structural', 'variations/alterations,', 'transcriptome', 'analysis,', 'transcription', 'factor', 'binding', 'sites,', 'chromatin', 'modifications,', 'dna', 'methylation.', 'lay:', 'next', 'generation', 'sequencing', 'core', 'provides', 'cutting', 'edge', 'services', 'harness', 'recent', 'technological', 'advances', 'allow', 'large', 'scale', 'genomic', 'sequencing', 'many', 'hundreds', 'millions', 'dna', 'molecules', 'parallel', '100,000', 'times', 'less', 'cost', 'previous', 'sequencing', 'technologies.', 'services', 'poised', 'rapidly', 'accelerate', 'pace', 'discovery', 'clinical', 'translation', 'cancer', 'molecular', 'genetics', 'research.', 'skccc', 'managed', 'shared', 'resource'])\n",
      "Row(idf_features=SparseVector(50663, {0: 0.5493, 6: 0.8571, 8: 4.1268, 9: 2.321, 10: 0.8027, 11: 0.7721, 13: 17.2854, 16: 2.3483, 18: 1.1638, 19: 0.9223, 20: 1.0703, 21: 0.8351, 24: 4.851, 27: 2.0115, 29: 3.8401, 30: 1.3391, 31: 1.0611, 34: 3.3747, 35: 0.9756, 36: 5.0253, 37: 1.0098, 39: 1.2025, 40: 5.1571, 45: 2.3049, 48: 1.2309, 52: 3.6049, 53: 1.186, 58: 4.8335, 59: 1.7157, 62: 3.1883, 64: 1.3804, 65: 1.5154, 66: 1.6186, 70: 3.3386, 72: 1.5405, 83: 3.6196, 84: 2.9575, 87: 1.6071, 89: 1.5185, 110: 4.7851, 111: 1.5667, 113: 1.842, 115: 5.3007, 118: 3.1549, 120: 1.7527, 124: 1.6753, 126: 3.961, 128: 3.8611, 131: 1.7519, 132: 1.8172, 152: 7.1996, 164: 3.7337, 165: 2.0017, 169: 2.071, 171: 3.7978, 176: 1.8206, 177: 1.8133, 185: 7.8696, 187: 1.932, 188: 1.9296, 189: 2.0986, 193: 2.2093, 195: 2.088, 198: 1.9944, 199: 2.0096, 206: 2.3209, 209: 2.064, 211: 7.5316, 219: 2.0125, 222: 2.0766, 233: 2.3561, 240: 4.6147, 241: 7.1372, 249: 2.4667, 260: 2.2013, 262: 2.0499, 274: 8.9351, 285: 2.2551, 288: 2.2683, 301: 3.1783, 305: 2.3486, 311: 2.8278, 318: 2.5455, 327: 2.463, 330: 2.3267, 338: 2.6634, 351: 2.5006, 356: 2.2434, 363: 2.4609, 365: 10.4696, 368: 2.7112, 370: 2.4356, 372: 4.8566, 389: 2.489, 391: 2.6355, 402: 2.514, 413: 2.4469, 433: 2.6896, 434: 5.205, 441: 2.4941, 464: 2.9625, 475: 2.686, 477: 2.7267, 501: 2.7619, 505: 2.7681, 511: 2.7059, 513: 10.7069, 532: 2.6634, 533: 2.9923, 539: 9.151, 540: 2.8218, 554: 2.6439, 555: 2.7486, 606: 2.8013, 613: 2.8398, 618: 2.772, 638: 2.6922, 654: 2.7937, 655: 2.7117, 656: 6.071, 672: 2.8477, 675: 5.5227, 704: 6.0858, 717: 2.9987, 720: 2.86, 730: 5.7523, 731: 2.9442, 732: 2.9177, 744: 3.1682, 768: 3.9318, 771: 2.9791, 776: 2.9316, 783: 9.3476, 791: 3.2468, 809: 2.8593, 812: 3.346, 828: 2.9302, 886: 3.0311, 902: 3.515, 968: 3.0593, 1016: 3.349, 1021: 3.7068, 1049: 3.4731, 1054: 6.7264, 1058: 3.2623, 1064: 3.2059, 1079: 3.3704, 1094: 6.4451, 1110: 3.169, 1115: 3.5257, 1179: 10.5272, 1181: 3.3838, 1204: 5.4386, 1216: 3.2763, 1221: 11.2037, 1226: 3.3028, 1237: 3.5476, 1238: 3.8455, 1241: 3.739, 1245: 3.655, 1284: 15.4153, 1292: 3.5649, 1306: 3.6214, 1332: 3.9175, 1349: 3.3807, 1388: 3.4595, 1401: 3.5281, 1416: 3.607, 1419: 14.8563, 1520: 3.6591, 1527: 3.539, 1556: 7.6583, 1594: 3.751, 1600: 3.6785, 1624: 3.9874, 1630: 4.0342, 1704: 3.7631, 1714: 7.9977, 1740: 4.2081, 1799: 3.7155, 1939: 3.8639, 1945: 3.8488, 1981: 3.834, 1993: 4.2493, 1995: 7.8421, 2030: 4.6267, 2131: 4.1174, 2133: 3.8689, 2134: 3.9246, 2158: 3.9246, 2200: 4.1371, 2309: 4.1195, 2320: 3.95, 2345: 4.4824, 2383: 4.194, 2429: 4.2346, 2448: 4.1685, 2470: 4.0302, 2476: 8.1707, 2477: 4.2322, 2531: 4.0707, 2554: 4.0666, 2584: 4.1504, 2621: 4.3454, 2719: 4.2543, 2787: 4.2419, 2819: 4.4824, 2880: 4.5436, 2935: 4.2395, 2982: 4.2618, 3031: 4.5809, 3137: 4.3427, 3200: 4.6376, 3236: 4.3986, 3237: 4.4101, 3263: 4.3536, 3289: 4.3871, 3317: 4.3786, 3356: 4.5174, 3424: 4.5207, 3464: 4.9132, 3489: 4.4426, 3521: 4.4547, 3669: 4.5304, 3980: 4.6978, 4122: 4.8989, 4190: 4.7095, 4228: 4.6786, 4234: 4.6978, 4485: 5.0413, 4529: 5.1328, 4820: 4.8481, 4902: 5.1448, 4999: 5.1882, 5177: 5.0413, 5333: 4.9778, 5480: 10.1834, 5500: 5.1756, 5636: 5.2073, 6157: 5.1756, 6303: 11.0117, 6396: 5.2401, 6640: 5.302, 6739: 10.861, 6801: 5.7075, 6910: 5.3909, 7016: 5.3832, 7028: 5.3987, 7640: 5.5058, 7713: 5.5146, 8233: 5.6969, 8610: 5.9603, 8627: 5.6356, 9452: 6.1076, 10754: 5.7621, 11262: 5.8199, 11703: 5.9603, 11727: 6.0461, 11773: 5.9741, 12045: 6.0611, 12590: 12.8012, 13492: 6.3387, 13623: 6.3589, 13731: 6.3796, 13732: 6.3796, 13756: 6.4006, 13873: 6.4221, 14135: 6.4666, 14188: 6.5131, 15147: 6.6398, 15568: 6.7244, 16520: 6.785, 16650: 14.8351, 17301: 6.8496, 18853: 6.9927, 19075: 7.4175, 19254: 15.3716, 19674: 7.1153, 19927: 7.1597, 20324: 7.2062, 20364: 7.3604, 20503: 7.255, 20579: 7.3063, 20933: 7.3604, 21198: 7.4175, 22028: 7.5427, 22295: 7.5427, 22800: 7.6117, 23046: 7.6117, 23614: 7.6117, 23933: 7.6858, 23978: 7.6858, 24044: 7.6858, 24053: 7.6858, 24090: 7.6858, 24129: 7.6858, 24145: 7.6858, 24164: 7.6858, 24237: 7.6858, 24348: 7.6858, 24391: 7.6858, 24533: 7.6858, 24596: 7.6858, 24622: 7.6858, 24730: 7.6858, 24794: 7.6858, 24871: 7.6858, 24978: 7.6858, 25061: 7.6858, 25088: 7.6858, 25113: 7.6858, 25185: 7.6858, 25234: 7.6858, 25270: 7.6858}), abstract_words_filtered=['pharmacogenomics', 'began', 'focus', 'functional', 'impact', 'small', 'number', 'variants', 'single', 'drug', 'metabolism', 'gene.', 'field', 'matures', 'become', 'clear', 'advances', 'understanding', 'genetic', 'basis', 'drug', 'effect', 'must', 'take', 'look', 'beyond', 'genes', 'expect', 'involved', 'drug', 'pathway.', 'coincident', 'completion', 'analysis', 'pharmacogenetic', 'studies', 'using', 'genome-wide', 'association', '(gwa)', 'approaches', 'identify', 'commonly', 'altered', 'genomic', 'regions,', 'ascendance', 'next-generation', 'sequencing', 'analysis', 'methods', 'provides', 'compelling', 'framework', 'anticipate', 'increasing', 'knowledge', 'rare', 'mutations', 'affected', 'cohorts.', 'especially', 'challenging', 'diseases', 'cancer,', 'heart', 'failure,', 'mental', 'retardation', 'syndromes,', 'complex', 'variation', 'copy', 'number', 'sequence', 'occur', 'simultaneously', 'within', 'given', 'gene.', 'targeting', 'genomic', 'regions', 'interest', 'technology', 'exercise', 'developing', 'past', 'several', 'years,', 'typically', 'involves', 'following', 'steps', 'regions', 'identified;', '1)', 'designing', 'primers', 'probes', 'genes', 'regions', 'interest', 'tiled', 'across', 'entire', 'locus/loci,', '2)', 'carrying', 'selected', 'targeting', 'method', 'using', 'probes,', 'either', 'pcr', 'capture,', '3)', 'isolating', 'dna', 'results,', '4)', 'sequencing', 'dna', 'using', 'one', 'three', 'commercially', 'available', 'next-generation', 'sequencing', 'platforms,', '5)', 'aligning', 'resulting', 'reads', 'human', 'reference', 'genome', 'analyzing', 'aligned', 'reads', 'identify', 'single', 'nucleotide', 'indel', 'variants', 'specific', 'affected', 'cohort.', 'downstream', 'functional', 'studies', 'may', 'enhance', 'understanding', 'variants', 'play', 'role', 'disease', 'susceptibility,', 'onset', 'severity.', 'alternative', 'to,', 'concert', 'targeted', 'sequencing', 'studies,', 'light', 'sequence', 'coverage', '(4x)', 'entire', 'genome', 'paired', 'end', 'sequencing', 'reads', 'accomplished', 'quickly', 'inexpensively', 'next', 'generation', 'sequencing', 'platforms.', 'coupled', 'careful', 'analysis', 'read', 'pair', 'mapping,', 'reads', 'used', 'identify', 'regions', 'amplification', 'deletion', '(copy', 'number', 'alteration', 'cna),', 'structural', 'variations', '(inversions,', 'cryptic', 'translocations)', 'may', 'vital', 'understanding', 'pharmacogenetic', 'traits.', 'several', 'years', 'ago,', 'anticipation', 'targeted', 're-sequencing', 'applicable', 'mutation', 'discovery', 'cancer', 'samples,', 'laboratory', 'developed', 'scaled', 'high-throughput,', 'pcr-based', 'capabilities', 'purpose.', 'capabilities', 'include', 'full', 'suite', 'activities', 'needed', \"'pipeline',\", 'including', 'automated', 'pcr', 'primer', 'design,', 'primer', 'validation,', '384', 'well', 'automated', 'pcr', 'reaction', 'assembly,', 'post-reaction', 'processing', 'sequencing', 'assembly', 'low', 'volume', 'minimize', 'dna', 'usage', 'cost.', 'commensurate', 'need', 'analyze', 'capillary', 'traces', 'resulting', 'efforts,', 'developed', 'automated', 'analysis', 'pipeline', 'capable', 'identifying', 'high', 'quality', 'single-nucleotide', 'indel', 'variants,', 'predicting', 'impact', 'identified', 'mutations', 'protein', 'structure,', 'databasing', 'resulting', 'information', 'downstream', 'analyses', 'correlation', 'mutational', 'profiles', 'across', 'samples,', 'correlation', 'clinical', 'data', 'elements,', 'on.', 'use', 'ofthis', 'pipeline', 'resulted', 'expansion', 'understanding', 'mutations', 'occurring', 'lung', 'adenocarcinoma,', 'acute', 'myeloid', 'leukemia,', 'glioblastoma', 'multiforme.', '', 'coincident', 'pcr', 'pipeline', 'development', 'work,', 'also', 'began', 'investigating', 'targeted', 'capture', 'solution-phase', 'hybridization.', 'based', 'early', 'success,', 'taking', 'advantage', 'new', 'technology', 'capability', 'produce', '55,000', 'oligos', 'cleaved', 'synthesis', 'substrate', '(agilent)', 'well', 'massively', 'parallel', 'sequencing', 'capability', 'next-generation', 'platforms,', 'developed', 'novel', 'capture', 'method', 'based', 'original', 'bac-based', 'capture.', 'refer', 'method', \"'wucap'\", '(washington', 'university', 'capture),', 'incorporated', 'sequencing', 'production', 'activities', 'center.', 'approach', 'applied', 'targeted', 'capture', 'sequencing', '1,000', 'genomes', 'project', '(pilot', '3', 'studies),', 'sequencing', '6,000', 'genes', 'ovarian', 'cancer', 'samples', 'cancer', 'genome', 'atlas', '(tcga),', 'soon', 'applied', 'set', '500', 'genes', 'suspected', 'retinitis', 'pigmentosa,', 'well', 'nhgri-sponsored', 'medical', 'sequencing', 'project', 'investigate', 'gwas', 'peaks', 'resulting', 'case/control', 'studies', 'metabolic', 'syndrome', '7,000', 'genomes.'])\n",
      "Row(idf_features=SparseVector(50663, {0: 1.0987, 2: 0.821, 5: 2.0061, 6: 0.8571, 8: 4.1268, 10: 0.8027, 13: 17.2854, 16: 1.5655, 20: 5.3513, 25: 0.8137, 27: 1.0057, 28: 1.779, 32: 1.0804, 35: 0.9756, 36: 5.0253, 37: 1.0098, 45: 1.1525, 47: 1.1903, 52: 2.4033, 55: 3.2188, 58: 4.8335, 64: 2.7608, 65: 3.0308, 72: 1.5405, 78: 6.2553, 79: 1.508, 86: 1.5433, 95: 1.576, 101: 1.7807, 108: 4.0045, 112: 1.7191, 115: 1.7669, 126: 1.9805, 127: 1.7644, 128: 3.8611, 132: 1.8172, 135: 1.7874, 145: 2.2249, 150: 1.9204, 157: 1.8619, 161: 2.036, 182: 4.4258, 190: 2.1075, 198: 7.9776, 199: 2.0096, 208: 2.0488, 219: 2.0125, 234: 3.0829, 285: 4.5102, 309: 2.269, 329: 2.2361, 338: 2.6634, 344: 2.4199, 362: 9.3739, 368: 2.7112, 370: 4.8711, 374: 7.627, 394: 5.1653, 420: 2.5949, 427: 2.6685, 433: 10.7585, 440: 5.3057, 444: 2.5655, 446: 5.268, 454: 2.737, 464: 2.9625, 467: 18.462, 470: 5.1616, 472: 2.9598, 477: 2.7267, 485: 2.6752, 486: 2.6922, 489: 15.9087, 562: 2.908, 575: 5.7423, 577: 2.6706, 597: 2.9119, 613: 2.8398, 665: 3.0051, 687: 2.873, 708: 2.8356, 714: 3.0938, 755: 2.9054, 774: 3.164, 791: 3.2468, 822: 3.235, 839: 3.1368, 840: 3.1741, 930: 3.1159, 1049: 3.4731, 1066: 3.2577, 1116: 3.348, 1143: 15.1885, 1145: 3.4584, 1215: 7.16, 1269: 6.8768, 1293: 3.4973, 1306: 3.6214, 1334: 3.3964, 1360: 3.6618, 1373: 3.5185, 1398: 7.1473, 1457: 3.7054, 1499: 3.5427, 1502: 11.2126, 1512: 3.8178, 1528: 3.6771, 1535: 3.5102, 1628: 3.6057, 1690: 3.6267, 1702: 4.0222, 1720: 3.8791, 1844: 3.8757, 1854: 3.9445, 1879: 3.9051, 1889: 3.961, 2003: 3.8572, 2005: 4.5878, 2030: 4.6267, 2106: 3.8588, 2146: 3.9069, 2270: 4.1893, 2272: 3.9354, 2297: 4.0707, 2383: 4.194, 2411: 4.113, 2466: 4.0422, 2493: 8.5539, 2775: 4.2395, 2812: 9.1141, 2915: 4.2693, 2985: 4.3564, 3024: 4.547, 3091: 13.8479, 3094: 4.547, 3095: 4.6124, 3215: 4.34, 3258: 4.4608, 3263: 4.3536, 3438: 4.7785, 3474: 4.5239, 3636: 9.1826, 3667: 4.5672, 3721: 9.4427, 3767: 4.6598, 3783: 4.9726, 3927: 4.574, 3980: 4.6978, 3982: 9.3879, 4172: 9.5654, 4207: 4.671, 4340: 4.7414, 4385: 5.069, 4469: 5.1033, 4531: 9.7696, 4610: 4.804, 4648: 4.8214, 4722: 4.8302, 5139: 5.8563, 5243: 5.1033, 5260: 5.3529, 5472: 5.3529, 5694: 10.8449, 5864: 17.7998, 5941: 5.1209, 6139: 5.1756, 7071: 5.4467, 7561: 5.4886, 8844: 5.6556, 9722: 19.0768, 10786: 5.7399, 10803: 6.1903, 10945: 5.844, 11257: 5.8199, 11816: 6.0461, 12025: 6.0611, 12156: 6.2077, 12200: 13.6991, 12508: 6.1564, 13042: 6.2254, 13374: 6.2995, 13662: 6.3589, 14827: 6.5872, 15748: 6.7543, 16062: 6.7543, 18851: 6.9927, 19090: 7.0319, 19118: 7.0319, 19124: 7.0319, 19274: 7.0319, 19372: 7.0319}), abstract_words_filtered=['next', 'generation', 'sequencing', 'genotyping', 'core', 'facility.', 'goal', 'next', 'gen', 'sequencing', 'component', 'make', \"'next\", \"generation'\", 'sequencing', 'related', 'services', 'widely', 'available', 'unc', 'lccc', 'members', 'cost-effective', 'prices.', 'services', 'include:', 'whole', 'genome', 'shotgun', 'sequencing', 'human', 'model', 'organisms,', 'chromatin-ip', 'sample', 'sequencing,', 'measuring', 'rna', 'expression', 'digital', 'gene', 'expression', 'rna-seq,', 'snp', 'typing,', 'polymorphism', 'screens.', 'component', 'adds', 'value', 'cancer', 'center', 'putting', 'complex', 'expensive', 'dna/rna', 'sequencing', 'technology', 'within', 'easy', 'reach', 'unc', 'lccc', 'members.', 'strongly', 'integrated', 'genomics', 'bioinformatics', 'groups', 'interact', 'user', 'ensure', 'robust', 'data', 'archived', 'appropriately', 'future', 'use.', 'component', 'key', 'unc', \"lccc's\", 'participation', 'cancer', 'genome', 'atlas', 'project', 'serving', 'lccc', 'needs.', 'future', 'plans', 'include', 'expanding', 'staff', 'equipment', 'full', 'array', 'next', 'gen', 'sequencing', 'services', 'may', 'provided', 'timely', 'manner,', 'well', 'acquisition', 'next', 'generation', 'technology.', '', '', 'goal', 'dna', 'sequencing', 'component', 'produce', 'high', 'quality', 'sequencing', 'genotyping', 'data', 'rapidly', 'reasonable', 'cost', 'unc-ch', 'researchers', 'using', 'state-of-the-art', 'technologies', 'well', 'provide', 'technical', 'support', 'enhance', 'value', 'results', 'produced.', 'services', 'include', 'dna', 'sequencing', 'microsatellite', 'genotyping.', 'component', 'adds', 'value', 'cancer', 'center', 'producing', 'high-quality', 'data', 'competitive', 'price', 'providing', 'benefits', 'local', 'resource', 'assist', 'in:', 'scientific', 'strategies,', 'sample', 'preparation,', 'sequencing', 'difficult', 'regions,', 'assistance', 'data', 'interpretation.', 'component', 'wide', 'user', 'base', '220', 'different', 'laboratories', 'users', '2009.', 'future', 'plans', 'include', 'continued', 'use', 'existing', 'equipment,', 'reduction', 'chemistry', 'costs,', 'evaluation', 'future', 'sequencing', 'technology', 'options', 'long', 'read', 'single', 'molecule', 'sequencing', 'technologies.', 'goal', 'high', 'throughput', 'genotyping', 'component', 'produce', 'high-quality', 'genotyping', 'data', 'rapidly', 'reasonable', 'cost', 'researchers', 'using', 'state-of-the-art', 'technologies.', 'services', 'include', 'snp', 'genotyping,', 'copy', 'number', 'variation', 'dna', 'methylation', 'profiling', 'analysis.', 'new', 'technologies', 'and/or', 'applications', 'explored', 'increase', \"core's\", 'competitive', 'edge', 'allow', 'genome', 'wide', 'association', 'study', 'rare', 'variants.', 'core', 'components', 'used', 'multiple', 'members', 'core,', 'peer', 'reviewed', 'member', 'used', 'exceeds', '70%.', 'proposed', 'budget', '$241,797', 'less', '5%', \"core's\", 'operating', 'cost.', 'budget', 'provides', 'salary', 'stability', 'vital', 'core', 'staff', 'work', 'members', 'adopt', 'novel', 'technologies.'])\n",
      "Row(idf_features=SparseVector(50663, {4: 0.7946, 6: 5.1426, 8: 5.5024, 13: 1.4404, 16: 0.7828, 20: 3.2108, 25: 1.6274, 27: 2.0115, 30: 4.0173, 32: 1.0804, 35: 1.9511, 48: 1.2309, 49: 4.1515, 52: 1.2016, 53: 1.186, 54: 4.5231, 55: 1.0729, 58: 3.2223, 81: 1.5428, 83: 9.049, 87: 1.6071, 90: 3.0813, 115: 1.7669, 124: 1.6753, 125: 1.6652, 130: 1.7493, 135: 1.7874, 137: 9.4632, 138: 1.7507, 141: 1.68, 157: 1.8619, 166: 2.3579, 171: 1.8989, 185: 2.6232, 187: 1.932, 199: 2.0096, 202: 2.2084, 206: 2.3209, 214: 2.0848, 224: 2.3741, 238: 2.2377, 241: 4.7581, 252: 2.1009, 255: 6.2562, 271: 2.4946, 300: 2.2477, 303: 2.6039, 314: 2.4742, 318: 2.5455, 328: 2.2531, 338: 2.6634, 364: 2.3883, 367: 2.4026, 406: 2.8718, 421: 2.4498, 427: 2.6685, 441: 4.9883, 443: 5.1822, 450: 7.5056, 460: 2.5692, 472: 2.9598, 475: 5.372, 482: 2.7868, 495: 2.8977, 522: 2.6286, 539: 3.0503, 540: 2.8218, 548: 22.8429, 556: 2.5992, 569: 2.6534, 641: 2.8655, 651: 5.7686, 665: 3.0051, 675: 2.7613, 685: 2.9577, 722: 2.7597, 723: 6.4082, 747: 3.2234, 775: 2.9395, 792: 3.1574, 847: 3.3797, 852: 2.9342, 877: 3.164, 924: 3.0837, 932: 6.1262, 950: 3.0745, 1055: 3.2568, 1095: 3.4156, 1119: 3.3154, 1143: 11.3914, 1176: 3.4708, 1179: 7.0181, 1205: 3.7213, 1212: 3.5686, 1228: 3.6687, 1265: 7.1298, 1275: 3.4406, 1279: 3.3714, 1286: 3.4562, 1304: 3.4927, 1331: 3.4997, 1426: 3.6673, 1468: 3.6161, 1505: 3.5674, 1508: 3.8035, 1536: 3.5476, 1634: 3.9741, 1725: 3.6968, 1771: 4.3591, 1895: 41.6587, 1933: 3.8389, 1980: 3.8656, 2009: 4.0302, 2213: 8.4839, 2220: 4.0728, 2272: 3.9354, 2295: 3.93, 2453: 4.7536, 2493: 4.2769, 2678: 4.1305, 2683: 4.4547, 2785: 4.2568, 2856: 4.2058, 2909: 4.3509, 2924: 4.4396, 2937: 4.4824, 3164: 4.3212, 3293: 4.3758, 3318: 4.7056, 3332: 4.5239, 3342: 4.4547, 3492: 15.3804, 3615: 4.557, 3705: 4.5239, 3783: 9.9453, 3792: 4.5537, 4053: 4.6748, 4223: 4.7174, 4310: 5.069, 4371: 4.7253, 4421: 4.8391, 4649: 10.0393, 4866: 4.8617, 5102: 5.2949, 5198: 4.9625, 5468: 5.009, 5637: 25.1981, 5651: 5.115, 6046: 5.1388, 6360: 6.0313, 6630: 5.302, 7091: 5.4972, 7178: 11.6399, 8504: 5.6969, 10744: 5.729, 11986: 6.0461, 13513: 6.3796, 13691: 6.3796, 14204: 6.4896, 14262: 6.4896, 14379: 6.4896, 14460: 6.4896, 15541: 6.7244, 19547: 7.1153, 19587: 7.1153, 19599: 7.1153, 19609: 7.1153, 19611: 7.1153, 19612: 7.1153, 19613: 7.1153, 19620: 7.1153, 19629: 7.1153, 19631: 7.1153, 19647: 7.1153, 19662: 7.1153, 19685: 7.1153, 19688: 7.1153, 19691: 7.1153, 19712: 7.1153, 19738: 7.1153, 19742: 7.1153, 19752: 7.1153, 19756: 7.1153, 19759: 7.1153, 19769: 7.1153}), abstract_words_filtered=['project', 'summary/abstract', 'mobile', 'dna', 'elements', 'dna', 'segments', 'mobilize', 'one', 'genomic', 'location', 'another,some', 'making', 'copies', 'process.', 'mobile', 'elements', 'ubiquitous', 'eukaryotic', 'genomes,and', 'roughly', '50%', 'human', 'genome', 'consists', 'mobile,', 'repetitive', 'dna', 'sequences', 'alu', 'line1elements.', 'mobile', 'elements', 'profoundly', 'influenced', 'primate', 'genomic', 'evolution,', 'alterations', 'caused', 'bymobile', 'elements', 'identified', '100', 'human', 'genetic', 'disorders.', 'addition,', 'mobile', 'elementsare', 'highly', 'useful', 'genetic', 'markers', 'tracing', 'relationships', 'populations', 'species', 'uniquemanner', 'propagating', 'genome.', 'therefore,', 'great', 'interest', 'understand', 'biology', 'mobileelements', 'human', 'populations.', 'currently', 'available', 'genotyping', 'methods,', 'however,', 'provide', 'limited', 'insightinto', 'distribution', 'alu', 'l1', 'elements', 'human', 'genome.', 'proposal,', 'first', 'describe', 'novel,', 'high-throughput', 'method', 'genotype', 'large', 'number', 'ofindividuals', 'alu', 'insertion', 'polymorphisms.', 'implementation', 'method', 'permit,', 'first', 'time,', 'rapid,inexpensive', 'genotyping', 'whole', 'families', 'mobile', 'elements', 'humans', 'organism.', 'next,', 'ioutline', 'plans', 'construct', 'computational', 'pipeline', 'efficiently', 'handle', 'analyze', 'high-throughput', 'genomic', 'data', 'generated', 'genotyping', 'method.', 'finally', 'describe', 'plan', 'apply', 'thegenotyping', 'procedure', '540', 'hapmap', 'samples', 'use', 'computational', 'pipeline', 'analyze', 'results.specifically,', 'use', 'insertion', 'polymorphism', 'data', 'assess', 'genomic', 'diversity', 'associated', 'mobileelement', 'insertions,', 'well', 'answer', 'questions', 'related', 'mobile', 'element', 'biology', 'reconstruct', 'ancienthuman', 'demographic', 'history.', 'addition,', 'alu', 'insertion', 'polymorphism', 'dataset', 'generated', 'project', 'willprovide', 'valuable', 'supplementary', 'resource', 'structural', 'variants', 'hapmap', '1000', 'genomes', 'projects.', 'long-term', 'goal', 'pursue', 'career', 'human', 'genomic', 'variation', 'population', 'genetics', 'researchwith', 'emphasis', 'mobile', 'elements.', 'using', 'combination', 'computational', 'experimental', 'tools,', 'iwill', 'study', 'impact', 'mobile', 'dna', 'elements', 'human', 'genome', 'genetic', 'diversity', 'among', 'humanpopulations.', 'award', 'provide', 'necessary', 'training', 'become', 'independent', 'investigator', 'ontwo', 'fronts:', 'acquire', 'essential', 'training', 'computational', 'biology', 'bioinformatics', 'handle', 'large-scalegenomic', 'data,', 'including', 'data', 'generated', 'next-generation', 'sequencing', 'technology.', 'addition,', 'researchtraining', 'offered', 'institution', 'mentor', 'allow', 'acquire', 'knowledge', 'research', 'resources,grant', 'writing,', 'responsible', 'conduct', 'research,', 'information', 'necessary', 'become', 'successful,independent', 'investigator.'])\n",
      "Row(idf_features=SparseVector(50663, {4: 1.5891, 11: 2.3164, 13: 2.8809, 14: 0.2706, 15: 0.2765, 17: 0.285, 20: 5.3513, 35: 2.9267, 37: 1.0098, 48: 1.2309, 49: 6.2273, 52: 1.2016, 55: 1.0729, 63: 1.2769, 68: 2.5122, 83: 1.8098, 104: 3.5479, 130: 3.4986, 137: 14.1948, 144: 1.7752, 150: 1.9204, 156: 4.0949, 163: 1.9843, 212: 2.7096, 232: 4.1081, 233: 2.3561, 237: 4.3281, 256: 2.2335, 270: 2.2663, 278: 4.7467, 281: 2.6589, 285: 2.2551, 297: 2.1248, 312: 2.4073, 314: 2.4742, 316: 2.1467, 322: 2.2797, 324: 2.301, 344: 2.4199, 345: 12.7501, 392: 5.3586, 394: 10.3307, 398: 2.4797, 403: 2.3453, 433: 2.6896, 444: 10.2621, 449: 2.5294, 455: 2.9369, 483: 5.5588, 489: 3.1817, 499: 2.8416, 537: 2.699, 539: 3.0503, 562: 2.908, 584: 2.8856, 606: 2.8013, 651: 5.7686, 669: 2.8356, 678: 2.8718, 683: 3.2297, 687: 2.873, 724: 2.7783, 731: 2.9442, 737: 3.0822, 767: 2.8786, 797: 2.9035, 833: 3.5463, 899: 2.9797, 950: 3.0745, 985: 3.0993, 1020: 3.5354, 1025: 3.1911, 1031: 3.1207, 1074: 3.3663, 1112: 3.5293, 1135: 3.266, 1139: 3.4286, 1182: 3.594, 1232: 3.464, 1238: 3.8455, 1245: 3.655, 1448: 3.5079, 1502: 3.7375, 1555: 3.751, 1561: 3.813, 1577: 3.739, 1676: 4.5537, 1688: 3.636, 1712: 3.6605, 1772: 7.5322, 1780: 3.6897, 1816: 3.6926, 2026: 3.8809, 2228: 3.9798, 2532: 8.2964, 2629: 4.1023, 2742: 4.3674, 2773: 4.4366, 2804: 4.2081, 2848: 4.201, 2923: 4.6267, 3018: 4.557, 3236: 4.3986, 3295: 9.1073, 3434: 4.4336, 3493: 4.4426, 3556: 9.3725, 3974: 4.7785, 4017: 4.6018, 4235: 4.7618, 4264: 5.0143, 4297: 4.804, 4373: 4.7333, 4482: 4.8709, 4528: 4.7743, 4552: 5.3091, 5362: 5.3235, 5480: 5.0917, 5673: 5.2401, 6158: 5.1882, 6618: 5.4224, 6629: 11.2713, 6724: 5.3235, 7680: 5.5146, 7707: 5.7075, 8028: 5.5779, 8421: 5.92, 8772: 5.6556, 10877: 12.7179, 10890: 5.8319, 11632: 5.9467, 11796: 6.1732, 11946: 6.0313, 12832: 6.2077, 13151: 6.2434, 13793: 6.4006, 14508: 6.5131, 14533: 6.5131, 14611: 6.5372, 14625: 6.5372, 14640: 6.5372, 14665: 6.5372, 18866: 6.9927, 20128: 7.2062, 20223: 7.2062, 20267: 7.2062}), abstract_words_filtered=['description', '(provided', 'applicant):', 'modern', 'biomedical', 'research', 'increasingly', 'making', 'use', 'genome-scale', 'next-generation', 'sequencing', 'platforms,', 'including', 'roche', '454,', 'illumina', 'ga2,', 'abi', 'solid.', 'platforms', 'make', 'possible', 'individual', 'labs', 'quickly', 'cheaply', 'generate', 'vast', 'amounts', 'genomic', 'transcriptomic', 'data', 'de', 'novo', 'sequencing,', 'resequencing,', 'chip-seq,', 'mrna-seq,', 'allelotyping', 'experiments.', 'despite', 'ability', 'generate', 'large', 'data', 'sets,', 'biomedical', 'researchers', 'rarely', 'trained', 'computational', 'statistical', 'techniques', 'necessary', 'make', 'sense', 'data.', 'thus,', 'many', 'researchers', 'must', 'rely', 'others', '-', 'often', 'computational', 'scientists', 'little', 'biological', 'training', '-', 'design', 'implement', 'appropriate', 'data', 'reduction', 'data', 'mining', 'techniques.', 'moreover,', 'institutions', 'access', 'computational', 'resources', 'necessary', 'run', 'analyses.', 'specific', 'aims', 'help', 'bridge', 'gap', 'short,', 'two-week', 'course,', 'teaching', 'biomedical', 'researchers', '(1)', 'run', 'analyses', 'remote', 'unix', 'servers', 'hosted', 'amazon', 'web', 'services', \"'cloud';\", '(2)', 'perform', 'mapping', 'assembly', 'large', 'short-read', 'data', 'sets;', '(3)', 'tackle', 'specific', 'biological', 'problems', 'existing', 'short-read', 'data;', '(4)', 'design', 'computational', 'pipelines', 'capable', 'addressing', 'research', 'questions.', 'specific', 'aims', 'accompanied', 'in-depth', 'hands-on', 'practical', 'training', 'relevant', 'techniques.', 'experience', 'practical', 'training', 'leads', 'substantial', 'improvement', 'basic', 'computational', 'sophistication', 'participants.', 'short', 'course', 'help', 'train', 'current', 'next', 'generation', 'independent', 'biomedical', 're-', 'searchers', 'basic', 'computational', 'thinking', 'procedure,', 'well', 'teaching', 'make', 'use', 'scalable', 'internet', 'computing', 'resources', 'research.', 'end', 'goal', 'increase', 'efficiency', 'sophistication', 'biomedical', 'researchers', 'make', 'use', 'novel', 'sequencing', 'technologies.'])\n",
      "Row(idf_features=SparseVector(50663, {2: 6.5677, 4: 1.5891, 11: 1.5443, 13: 2.8809, 16: 0.7828, 18: 1.1638, 19: 1.8446, 20: 2.1405, 25: 0.8137, 27: 1.0057, 36: 6.7004, 37: 5.049, 40: 5.1571, 41: 1.1575, 45: 2.3049, 47: 1.1903, 54: 1.5077, 57: 1.3142, 59: 1.7157, 68: 2.5122, 74: 12.7133, 83: 1.8098, 84: 2.9575, 88: 1.4663, 90: 3.0813, 91: 1.5101, 95: 1.576, 98: 1.8566, 108: 2.0022, 109: 1.619, 110: 1.595, 127: 1.7644, 129: 1.7562, 131: 1.7519, 144: 1.7752, 145: 4.4498, 154: 1.9141, 158: 1.9209, 160: 3.9883, 163: 1.9843, 171: 3.7978, 179: 4.5823, 181: 2.9382, 182: 2.2129, 188: 3.8592, 195: 2.088, 199: 2.0096, 211: 2.5105, 221: 2.0371, 252: 2.1009, 261: 2.1718, 268: 2.6213, 274: 2.2338, 285: 2.2551, 286: 4.9243, 300: 2.2477, 302: 2.5609, 313: 2.7392, 314: 7.4227, 316: 2.1467, 319: 4.5204, 325: 3.026, 331: 4.5593, 340: 4.4793, 356: 2.2434, 357: 2.5388, 384: 2.3875, 403: 2.3453, 417: 2.6444, 421: 2.4498, 433: 2.6896, 441: 2.4941, 443: 5.1822, 445: 2.5514, 449: 2.5294, 460: 2.5692, 488: 2.5455, 501: 2.7619, 506: 2.7289, 509: 2.686, 511: 2.7059, 512: 2.529, 522: 2.6286, 527: 2.6675, 530: 2.8526, 544: 2.7311, 572: 3.1215, 618: 2.772, 622: 2.7602, 641: 2.8655, 642: 2.7464, 646: 3.2857, 655: 2.7117, 660: 2.7602, 665: 3.0051, 679: 8.6758, 718: 2.8612, 719: 2.8926, 721: 2.9784, 722: 2.7597, 748: 2.8743, 813: 2.9282, 827: 2.953, 843: 3.0267, 858: 2.9132, 877: 3.164, 888: 3.1032, 891: 3.2716, 921: 3.1516, 929: 3.2288, 991: 6.3364, 1057: 7.0347, 1072: 3.1565, 1074: 3.3663, 1096: 3.1716, 1116: 3.348, 1120: 3.3252, 1167: 3.34, 1185: 3.422, 1201: 3.7025, 1331: 3.4997, 1452: 3.5079, 1481: 3.628, 1506: 3.8422, 1516: 3.6954, 1530: 3.6018, 1545: 3.5636, 1553: 3.6122, 1590: 3.6135, 1592: 3.5674, 1596: 3.8275, 1651: 4.4426, 1680: 3.9228, 1697: 3.6855, 1749: 3.8673, 1765: 3.9536, 1807: 3.6926, 1924: 3.9855, 1962: 4.5304, 2015: 3.8809, 2079: 4.1963, 2120: 4.0523, 2164: 3.9193, 2228: 3.9798, 2274: 3.939, 2342: 4.1594, 2843: 4.5843, 2899: 4.3564, 2999: 4.4856, 3037: 4.3454, 3216: 4.4366, 3244: 4.6413, 3496: 4.5142, 3673: 9.0609, 3792: 4.5537, 3842: 4.6673, 3968: 4.634, 3997: 4.6089, 4004: 5.6456, 4393: 4.7373, 4642: 4.8391, 4994: 4.9084, 5101: 4.9524, 5162: 5.2269, 5198: 4.9625, 5746: 5.069, 5773: 5.1509, 6134: 5.1694, 6216: 5.1882, 6401: 5.2401, 6515: 11.9206, 6580: 5.2879, 6621: 5.3308, 6855: 5.3529, 6897: 5.368, 6900: 5.3604, 6917: 5.368, 7139: 22.8726, 7245: 5.7399, 8044: 6.0461, 8134: 5.5594, 8161: 5.5594, 8783: 6.2077, 9050: 5.6658, 13149: 6.2434, 15712: 6.7543, 19525: 7.0727, 20136: 7.2062, 20416: 7.255, 20740: 7.3063, 21034: 7.3604, 22099: 7.5427, 22302: 7.5427, 22911: 7.6117, 22980: 7.6117, 26801: 7.8529, 29503: 8.1713, 29594: 8.1713, 30049: 8.1713, 30625: 8.3048, 30690: 8.3048, 30729: 8.3048, 30731: 8.3048, 30766: 8.3048, 30779: 8.3048, 30787: 8.3048, 30826: 8.3048, 30849: 8.3048, 30940: 8.3048, 30950: 8.3048, 31363: 8.3048, 31463: 8.3048, 31483: 8.3048, 31488: 8.3048, 31594: 8.3048, 31677: 8.3048, 31800: 8.3048, 31872: 8.3048, 31895: 8.3048, 31899: 8.3048, 31974: 8.3048, 32009: 8.3048, 32143: 8.3048, 32214: 8.3048, 32262: 8.3048, 32293: 8.3048, 32392: 8.3048, 32393: 8.3048, 32576: 8.3048, 32647: 8.3048}), abstract_words_filtered=['project', 'summary/abstract', 'impact', 'high-throughput', 'genomics', 'technologies', 'personalized', 'medicine', 'novel', 'cancertreatments', 'ultimately', 'determined', 'ability', 'interpret', 'data', 'accordingly.', 'therefore,', \"theapplicant's\", 'long-term', 'objective', 'translate', 'genome-based', 'knowledge,', 'integrative', 'analysis', 'high-throughput', 'genomic', 'technologies,', 'development', 'novel', 'diagnostic,', 'prognostic,', 'therapeuticstrategies', 'cancer.', 'initial', 'step', 'towards', 'endeavor,', 'proposal', 'focuses', 'central', 'hypothesisthat', 'common', 'carcinomas', 'harbor', 'driving', 'recurrent', 'gene', 'fusion', 'mutations', 'hidden', 'non-specificaberrations.', 'believe', 'next', 'generation', 'sequencing', 'cancer', 'transcriptomes', 'may', 'elucidate', 'thesedriving', 'gene', 'fusions', 'therefore', 'established', 'following', 'specific', 'aims', 'accomplish', 'this:', '(1)', 'employintegrative', 'analysis', 'genomics', 'data', 'detect', 'novel', 'gene', 'fusions', 'ets', 'negative', 'prostate', 'tumors,', '(2)further', 'develop', 'bioinformatic', 'approaches', 'prioritize', 'casual', 'chimeras', 'apply', 'methods', 'inmelanoma', 'pancreatic', 'cancer,', '(3)', 'implement', 'sequencing', 'based', 'gene-', 'exon-level', 'expressionoutlier', 'profile', 'analysis', 'assess', 'prevalence,', 'nominate', 'novel,', 'gene', 'fusions', 'publicly', 'available', 'datacollections.', 'overall,', 'aims', 'application', 'achieved', 'may', 'reveal', 'novel', 'pathognomomic', 'targets', 'incancer', 'translated', 'development', 'targeted', 'therapy.', 'even', 'successful', 'discovery', 'asingle', 'target,', 'whether', 'discovered', 'directly', 'proposed', 'work', 'uncovered', 'another', 'lab', 'leveraging', 'themethodologies', 'developed', 'result', 'proposal,', 'significant', 'impact', 'treat', 'cancer.', 'michigan', 'center', 'translational', 'pathology', 'in-depth', 'experience', 'using', 'bioinformatics', 'andgenomics', 'studying', 'gene', 'expression,', 'particularly', 'gene', 'fusions,', 'yielding', 'numerous', 'biomarkers', 'andtherapeutic', 'targets', 'prostate', 'cancer', 'therefore', 'serves', 'ideal', 'environment', 'supporting', 'theapplicant.', 'exemplified', \"applicant's\", 'extensive', 'preliminary', 'data,', 'led', 'multiple', 'highimpact', 'first', 'author', 'manuscripts.', 'given', 'dr.', \"chinnaiyan's\", 'excellent', 'record', 'mentoring', 'biostatisticians', 'andbioinformaticians', 'transition', 'successful', 'independent', 'investigators,', 'provide', 'similar', 'careerdevelopment', 'plan', 'current', 'applicant.', 'first', 'specific', 'aim,', 'completed', 'mentored', 'phase,will', 'focus', 'detecting', 'gene', 'fusions', 'prostate', 'cancer', 'therefore', 'offer', 'critical', 'integrative', 'analysis,', 'cancerbiology,', 'translational', 'research', 'training.', 'focus', 'independent', 'phase', 'placed', 'sequenceanalysis', 'chimera', 'detection', 'newly', 'established', 'cancer', 'transcriptome', 'projects', 'rna-seq', 'based', 'outlierexpression', 'analysis', 'reveal', 'novel', 'therapeutic', 'targets,', 'minimal', 'overlap', 'with,', 'complements,the', \"mentor's\", 'research', 'prostate', 'gene', 'fusions.', 'overall,', 'nih', 'pathway', 'independence', 'award', 'beindispensable', 'applicant', 'become', 'independent', 'investigator', 'much', 'needed', 'field', 'integrativecancer', 'biology', 'continue', 'translating', 'genome-based', 'discovering', 'widely', 'adopting', 'clinical', 'applications.'])\n",
      "Row(idf_features=SparseVector(50663, {10: 2.408, 13: 5.7618, 20: 2.1405, 21: 1.6702, 25: 0.8137, 27: 3.0172, 30: 1.3391, 32: 1.0804, 37: 1.0098, 40: 2.5785, 41: 1.1575, 47: 1.1903, 49: 2.0758, 58: 1.6112, 59: 1.7157, 64: 1.3804, 65: 1.5154, 103: 1.6609, 126: 1.9805, 127: 1.7644, 128: 1.9306, 135: 1.7874, 137: 2.3658, 140: 1.7577, 143: 1.7673, 144: 1.7752, 145: 2.2249, 153: 1.7957, 161: 4.072, 167: 4.2261, 182: 2.2129, 187: 1.932, 201: 11.9357, 207: 2.8587, 212: 2.7096, 219: 4.025, 265: 2.2126, 289: 2.1953, 313: 2.7392, 330: 2.3267, 356: 2.2434, 394: 7.748, 395: 2.5559, 440: 2.6529, 448: 4.8938, 453: 2.9701, 454: 2.737, 470: 2.5808, 499: 2.8416, 514: 2.7641, 537: 2.699, 539: 3.0503, 550: 2.6399, 564: 2.8532, 582: 6.4936, 584: 2.8856, 634: 3.1565, 664: 2.9476, 711: 2.9067, 723: 6.4082, 738: 3.0429, 782: 2.9874, 784: 2.9483, 793: 2.9389, 844: 3.6374, 873: 3.0563, 877: 3.164, 1016: 3.349, 1019: 3.164, 1044: 3.2596, 1155: 6.5358, 1169: 7.1473, 1199: 6.6444, 1201: 7.4051, 1225: 7.6453, 1288: 3.5402, 1319: 3.3663, 1377: 3.7555, 1507: 3.886, 1573: 15.1734, 1600: 3.6785, 1631: 3.7155, 1664: 3.8003, 1695: 3.7753, 1755: 3.93, 1783: 3.8791, 1842: 3.7831, 1868: 7.7379, 2002: 3.9427, 2003: 3.8572, 2285: 4.2769, 2629: 4.1023, 2705: 4.1504, 2745: 4.2297, 2789: 4.4307, 2790: 4.7253, 2931: 4.237, 2954: 4.47, 3164: 4.3212, 3222: 4.3619, 3252: 4.8127, 3417: 4.4396, 3446: 9.1006, 3475: 4.7495, 3592: 9.3421, 3714: 4.8617, 3940: 4.574, 4000: 4.5948, 4150: 4.7785, 4344: 4.7174, 4433: 4.7536, 5016: 4.9229, 5543: 5.025, 5673: 5.2401, 5739: 5.3455, 6158: 5.1882, 6587: 5.2879, 6898: 5.3604, 8656: 5.8319, 10819: 5.7849, 11639: 5.9467, 11643: 5.9603, 13285: 6.2805, 13377: 6.2995, 14117: 14.1454, 15581: 6.7244, 16699: 6.8168, 17279: 6.8496, 17524: 6.8835, 19055: 7.0319, 19532: 7.0727, 20035: 7.2062, 20198: 7.2062, 20243: 7.2062}), abstract_words_filtered=['recently,', 'emerging', 'new', 'field', 'metagenomics', 'facilitated', 'advent', 'next-generation', 'sequencing', 'technology', 'enables', 'genome', 'sequencing', 'unculturable', 'often', 'unknown', 'microbes', 'natural', 'environments,', 'offering', 'researchers', 'unprecedented', 'opportunity', 'delineate', 'bio-diversity', 'microbial', 'organism.', 'mining', 'single', 'nucleotide', 'polymorphisms', '(snps)', 'metagenomic', 'sequencing', 'data', 'offers', 'unique', 'opportunity', 'rapidly', 'accurately', 'detect', 'known', 'novel', 'strains', 'related', 'multiple', 'biothreat', 'agents.', 'sequencing', 'technologies', 'evolving', 'unprecedented', 'speed,', 'researchers', 'engaged', 'enterprise', 'facing', 'major', 'computational,', 'algorithmic', 'statistical', 'challenges', 'analysis', 'massive', 'metagenomic', 'data.', 'clear', 'current', 'analytical', 'computational', 'methods', 'inadequate', 'challenge.', 'project,', 'investigator', 'colleagues', 'develop', 'family', 'statistically', 'sound', 'computationally', 'efficient', 'algorithms', 'detect', 'snps', 'metagenomic', 'data', 'characterize', 'microbial', 'diversity', 'natural', 'environments.', '<br/><br/>the', 'proposed', 'project', 'provides', 'national', 'security', 'biodefense', 'agencies', 'new', 'tools', 'rapid', 'accurate', 'detection', 'biothreat', 'agents.', 'also', 'provides', 'researchers', 'microbiology', 'new', 'tools', 'producing', 'abundant,', 'high', 'throughput', 'snps', 'detailed', 'analysis', 'genetic', 'basis', 'microbial', 'diversity', 'evolution.', 'since', 'informatics', 'tool', 'used', 'study', 'wide', 'variety', 'microbial', 'communities,', 'helps', 'accelerating', 'scientific', 'advancements', 'knowledge', 'microbiology', 'evolution.', 'multidisciplinary', 'nature', 'project', 'promote', 'collaboration', 'biologists,', 'computer', 'scientists', 'statistician.', 'multidisciplinary', 'nature', 'project', 'also', 'provide', 'postdoctoral', 'fellows', 'graduate', 'students', 'training', 'statistics,', 'genomics', 'scientific', 'computing', 'hands-on', 'experience.'])\n",
      "Row(idf_features=SparseVector(50663, {10: 2.408, 13: 5.7618, 20: 2.1405, 21: 1.6702, 25: 0.8137, 27: 3.0172, 30: 1.3391, 32: 1.0804, 37: 1.0098, 40: 2.5785, 41: 1.1575, 47: 1.1903, 49: 2.0758, 58: 1.6112, 59: 1.7157, 64: 1.3804, 65: 1.5154, 103: 1.6609, 126: 1.9805, 127: 1.7644, 128: 1.9306, 135: 1.7874, 137: 2.3658, 140: 1.7577, 143: 1.7673, 144: 1.7752, 145: 2.2249, 153: 1.7957, 161: 4.072, 167: 4.2261, 182: 2.2129, 187: 1.932, 201: 11.9357, 207: 2.8587, 212: 2.7096, 219: 4.025, 265: 2.2126, 289: 2.1953, 313: 2.7392, 330: 2.3267, 356: 2.2434, 394: 7.748, 395: 2.5559, 440: 2.6529, 448: 4.8938, 453: 2.9701, 454: 2.737, 470: 2.5808, 499: 2.8416, 514: 2.7641, 537: 2.699, 539: 3.0503, 550: 2.6399, 564: 2.8532, 582: 6.4936, 584: 2.8856, 634: 3.1565, 664: 2.9476, 711: 2.9067, 723: 6.4082, 738: 3.0429, 782: 2.9874, 784: 2.9483, 793: 2.9389, 844: 3.6374, 873: 3.0563, 877: 3.164, 1016: 3.349, 1019: 3.164, 1044: 3.2596, 1155: 6.5358, 1169: 7.1473, 1199: 6.6444, 1201: 7.4051, 1225: 7.6453, 1288: 3.5402, 1319: 3.3663, 1377: 3.7555, 1507: 3.886, 1573: 15.1734, 1600: 3.6785, 1631: 3.7155, 1664: 3.8003, 1695: 3.7753, 1755: 3.93, 1783: 3.8791, 1842: 3.7831, 1868: 7.7379, 2002: 3.9427, 2003: 3.8572, 2285: 4.2769, 2629: 4.1023, 2705: 4.1504, 2745: 4.2297, 2789: 4.4307, 2790: 4.7253, 2931: 4.237, 2954: 4.47, 3164: 4.3212, 3222: 4.3619, 3252: 4.8127, 3417: 4.4396, 3446: 9.1006, 3475: 4.7495, 3592: 9.3421, 3714: 4.8617, 3940: 4.574, 4000: 4.5948, 4150: 4.7785, 4344: 4.7174, 4433: 4.7536, 5016: 4.9229, 5543: 5.025, 5673: 5.2401, 5739: 5.3455, 6158: 5.1882, 6587: 5.2879, 6898: 5.3604, 8656: 5.8319, 10819: 5.7849, 11639: 5.9467, 11643: 5.9603, 13285: 6.2805, 13377: 6.2995, 14117: 14.1454, 15581: 6.7244, 16699: 6.8168, 17279: 6.8496, 17524: 6.8835, 19055: 7.0319, 19532: 7.0727, 20035: 7.2062, 20198: 7.2062, 20243: 7.2062}), abstract_words_filtered=['recently,', 'emerging', 'new', 'field', 'metagenomics', 'facilitated', 'advent', 'next-generation', 'sequencing', 'technology', 'enables', 'genome', 'sequencing', 'unculturable', 'often', 'unknown', 'microbes', 'natural', 'environments,', 'offering', 'researchers', 'unprecedented', 'opportunity', 'delineate', 'bio-diversity', 'microbial', 'organism.', 'mining', 'single', 'nucleotide', 'polymorphisms', '(snps)', 'metagenomic', 'sequencing', 'data', 'offers', 'unique', 'opportunity', 'rapidly', 'accurately', 'detect', 'known', 'novel', 'strains', 'related', 'multiple', 'biothreat', 'agents.', 'sequencing', 'technologies', 'evolving', 'unprecedented', 'speed,', 'researchers', 'engaged', 'enterprise', 'facing', 'major', 'computational,', 'algorithmic', 'statistical', 'challenges', 'analysis', 'massive', 'metagenomic', 'data.', 'clear', 'current', 'analytical', 'computational', 'methods', 'inadequate', 'challenge.', 'project,', 'investigator', 'colleagues', 'develop', 'family', 'statistically', 'sound', 'computationally', 'efficient', 'algorithms', 'detect', 'snps', 'metagenomic', 'data', 'characterize', 'microbial', 'diversity', 'natural', 'environments.', '<br/><br/>the', 'proposed', 'project', 'provides', 'national', 'security', 'biodefense', 'agencies', 'new', 'tools', 'rapid', 'accurate', 'detection', 'biothreat', 'agents.', 'also', 'provides', 'researchers', 'microbiology', 'new', 'tools', 'producing', 'abundant,', 'high', 'throughput', 'snps', 'detailed', 'analysis', 'genetic', 'basis', 'microbial', 'diversity', 'evolution.', 'since', 'informatics', 'tool', 'used', 'study', 'wide', 'variety', 'microbial', 'communities,', 'helps', 'accelerating', 'scientific', 'advancements', 'knowledge', 'microbiology', 'evolution.', 'multidisciplinary', 'nature', 'project', 'promote', 'collaboration', 'biologists,', 'computer', 'scientists', 'statistician.', 'multidisciplinary', 'nature', 'project', 'also', 'provide', 'postdoctoral', 'fellows', 'graduate', 'students', 'training', 'statistics,', 'genomics', 'scientific', 'computing', 'hands-on', 'experience.'])\n",
      "Row(idf_features=SparseVector(50663, {0: 0.5493, 2: 0.821, 8: 2.7512, 10: 2.408, 12: 0.7058, 13: 8.6427, 14: 0.2706, 15: 0.2765, 17: 0.285, 19: 0.9223, 20: 12.8431, 21: 1.6702, 26: 1.0122, 37: 1.0098, 38: 1.1757, 40: 6.4463, 41: 5.7876, 42: 0.8133, 43: 2.2931, 48: 2.4617, 50: 2.1754, 51: 0.8589, 52: 1.2016, 57: 1.3142, 59: 10.2944, 62: 1.5942, 63: 3.8307, 65: 4.5462, 92: 10.1815, 95: 1.576, 100: 2.4572, 104: 1.774, 109: 1.619, 110: 1.595, 112: 1.7191, 127: 1.7644, 131: 1.7519, 137: 9.4632, 138: 1.7507, 147: 8.8949, 155: 1.8235, 167: 6.3392, 171: 1.8989, 176: 1.8206, 177: 1.8133, 182: 2.2129, 195: 2.088, 212: 24.3865, 218: 2.3475, 220: 2.1, 229: 4.2017, 234: 6.1659, 258: 2.3354, 262: 2.0499, 265: 2.2126, 267: 2.2197, 269: 4.8288, 272: 2.3486, 286: 2.4621, 290: 2.2551, 327: 4.9259, 334: 4.6884, 345: 2.55, 346: 2.5049, 353: 6.0917, 356: 2.2434, 372: 2.4283, 389: 4.978, 394: 2.5827, 402: 2.514, 414: 5.187, 421: 9.7991, 445: 5.1027, 451: 2.5473, 454: 8.2111, 460: 2.5692, 481: 3.0466, 486: 5.3845, 488: 2.5455, 502: 2.5079, 527: 2.6675, 539: 6.1007, 544: 2.7311, 554: 2.6439, 571: 2.6964, 578: 2.8386, 595: 5.397, 597: 2.9119, 610: 2.8326, 641: 5.7311, 659: 2.863, 664: 2.9476, 671: 3.2688, 719: 5.7851, 726: 3.9855, 731: 2.9442, 737: 6.1644, 747: 3.2234, 775: 2.9395, 776: 2.9316, 782: 2.9874, 862: 4.0604, 882: 2.9415, 924: 3.0837, 982: 3.2208, 1001: 3.18, 1032: 3.4264, 1111: 3.3019, 1208: 3.495, 1284: 3.8538, 1304: 3.4927, 1311: 3.3541, 1345: 3.5513, 1352: 3.4307, 1400: 8.7072, 1404: 3.5439, 1443: 3.6701, 1453: 3.6031, 1625: 7.2402, 1675: 3.8947, 1704: 3.7631, 1707: 3.7097, 1730: 3.636, 1764: 3.6646, 1830: 3.8774, 1901: 3.8243, 1913: 4.2543, 2003: 7.7143, 2162: 3.8757, 2188: 4.0707, 2299: 3.9798, 2341: 3.961, 2660: 4.2518, 2734: 4.1482, 2936: 4.2769, 3236: 4.3986, 3553: 4.4669, 3571: 4.4762, 3714: 4.8617, 3724: 4.537, 4101: 4.6978, 4207: 4.671, 4264: 5.0143, 4491: 4.7618, 4884: 5.1091, 5420: 5.0413, 5988: 5.1268, 5993: 5.2138, 6092: 10.6616, 7899: 5.5234, 8182: 5.5873, 8600: 5.6258, 8783: 6.2077, 9865: 5.6969, 9902: 5.7964, 10503: 6.3189, 10970: 12.2798, 11304: 6.0461, 11399: 5.92, 11538: 5.92, 11629: 5.9467, 11851: 6.0023, 12171: 6.0919, 13024: 6.2995, 13195: 6.2434, 13463: 6.3387, 13692: 6.6954, 13971: 6.4441, 14303: 6.4896, 14523: 6.5131, 14838: 6.8168, 15200: 6.6672, 15275: 6.785, 16071: 6.7543, 17340: 6.8835, 17404: 6.8835, 17412: 6.8835, 17415: 6.8835, 17418: 6.8835, 17421: 6.8835, 17536: 6.8835, 18612: 6.9549}), abstract_words_filtered=['description', '(provided', 'applicant):', 'next-generation', 'sequencing', 'technologies', 'capable', 'producing', 'tens', 'millions', 'sequence', 'reads', 'instrument', 'run,', 'quickly', 'applied', 'diverse', 'types', 'experiments', '(e.g.', 'rna-seq,', 'mirna-seq,', 'chip-seq,', 'bs-seq,', 'cnv-seq)', 'address', 'biomedical', 'questions', 'cost-effectively', 'generating', 'genome-wide', 'datasets.', 'sequencing', 'promoted', 'overcoming', 'longstanding', 'limitations', 'microarray-based', 'studies,', 'data', 'files', 'much', 'larger', 'microarrays,', 'diverse', 'data', 'types', 'raise', 'similar', 'well', 'novel', 'statistical', 'computational', 'challenges.', 'pressing', 'need', 'statistical', 'computational', 'tools', 'address', 'leaders', 'field', 'stated', 'largest', 'problems:', 'data', 'analysis', 'data', 'integration.', 'propose', 'develop', 'comprehensive', 'coordinated', 'set', 'statistical', 'methods', 'high', 'throughput', 'sequencing', '(hts)', 'directly', 'address', 'many', 'important', 'data', 'analysis', 'problems', 'epigenomics.', 'specifically', 'plan', 'address', 'following', 'computational', 'statistical', 'challenges', 'facing', 'researchers', 'conducting', 'hts', 'experiments:', '1)', 'develop', 'sensitive', 'statistical', 'methods', 'analysis', 'chip-seq', 'data', 'single-', 'paired-end-tag', 'runs,', 'particularly', 'focusing', 'applications', 'genome-wide', 'profiling', 'nucleosome', 'positions.', '2)', 'develop', 'statistical', 'methods', 'analysis', 'bs-seq', 'data,', 'producing', 'base-level', 'dna', 'methylation', 'profiles.', '3)', 'develop', 'new', 'statistical', 'tools', 'methods', 'data', 'integration', 'order', 'gain', 'new', 'biological', 'insights', 'global', 'transcription', 'regulation.', 'also', 'plan', 'apply', 'approaches', 'variety', 'high', 'throughput', 'sequencing', 'data', 'sets', 'demonstrate', 'relevance', 'utility', 'methods.', 'plan', 'work', 'stimulated', 'stat1', 'stat3', 'data,', 'data', 'ets', 'transcription', 'factor', 'family', 'cofactors,', 'already', 'gathered', 'significant', 'data', 'collaborations,', 'including', 'transcription', 'factors,', 'histone', 'marks,', 'dnase', 'hypersensitivity', 'gene', 'expression.', '', 'public', 'health', 'relevance:', 'propose', 'develop', 'comprehensive', 'coordinated', 'set', 'statistical', 'methods', 'high', 'throughput', 'sequencing', '(hts)', 'directly', 'address', 'many', 'important', 'data', 'analysis', 'problems', 'epigenomics.', 'particular,', 'plan', 'integrate', 'data', 'multiple', 'sources', 'including', 'expression,', 'transcription', 'factor', 'binding,', 'nucleosome', 'positioning,', 'histone', 'marks', 'dna', 'methylation', 'better', 'understand', 'mechanisms', 'regulate', 'behavior', 'cell.', 'much', 'proposal', 'involves', 'development', 'new', 'statistical', 'computational', 'methods,', 'also', 'design,', 'implementation', 'delivery', 'software', 'tools', 'support', 'ideas.', 'many', 'useful', 'applications', 'next-generation', 'sequencing', 'assure', 'well-', 'developed', 'methods', 'broad', 'impact', 'molecular', 'biology,', 'specifically', 'transcription', 'regulation,', 'chromatin', 'dynamics,', 'development,', 'cancer.'])\n",
      "Row(idf_features=SparseVector(50663, {0: 5.4933, 4: 2.3837, 6: 0.8571, 8: 1.3756, 13: 4.3213, 19: 2.7668, 20: 2.1405, 27: 1.0057, 29: 0.96, 30: 1.3391, 31: 1.0611, 40: 2.5785, 48: 1.2309, 54: 1.5077, 58: 4.8335, 62: 1.5942, 64: 2.7608, 70: 1.6693, 103: 1.6609, 104: 1.774, 128: 1.9306, 137: 2.3658, 145: 2.2249, 161: 2.036, 164: 3.7337, 166: 2.3579, 180: 2.7519, 199: 2.0096, 202: 2.2084, 207: 2.8587, 220: 2.1, 225: 2.2639, 236: 2.0423, 247: 2.0794, 256: 2.2335, 261: 2.1718, 266: 2.3871, 285: 2.2551, 299: 2.2932, 329: 4.4722, 330: 2.3267, 356: 2.2434, 374: 2.5423, 377: 2.7513, 418: 2.9422, 432: 2.4168, 438: 2.5105, 453: 2.9701, 486: 2.6922, 539: 3.0503, 549: 2.6399, 599: 2.7931, 644: 2.9422, 669: 2.8356, 672: 2.8477, 740: 2.9994, 809: 2.8593, 816: 3.013, 822: 3.235, 856: 2.9598, 885: 3.034, 1070: 3.3551, 1072: 3.1565, 1099: 3.3776, 1136: 3.2698, 1216: 3.2763, 1303: 3.6715, 1380: 3.5838, 1528: 3.6771, 1549: 3.58, 1553: 3.6122, 1573: 5.0578, 1754: 3.8723, 1788: 3.7213, 1862: 3.7722, 1898: 3.9855, 1986: 4.2129, 2011: 4.1415, 2026: 3.8809, 2083: 3.939, 2116: 4.1195, 2143: 4.547, 2160: 4.0938, 2201: 3.9354, 2280: 3.9228, 2435: 4.1152, 2466: 4.0422, 2519: 4.3027, 2813: 4.2518, 2899: 4.3564, 2954: 4.47, 2966: 4.511, 3072: 4.574, 3120: 4.4277, 3146: 4.4336, 3453: 4.4608, 3541: 4.4639, 3743: 4.9829, 4798: 4.9881, 4852: 5.0037, 5033: 4.918, 5136: 4.9475, 5462: 5.274, 5743: 5.069, 5835: 5.1756, 6173: 5.3832, 6215: 5.1882, 6975: 5.9333, 7433: 5.4633, 7445: 5.4633, 7603: 6.2254, 7885: 5.5234, 8095: 5.8563, 8233: 5.6969, 8421: 11.8401, 8478: 5.616, 8587: 5.6658, 8773: 5.8081, 9506: 5.729, 10961: 5.8319, 11163: 5.7964, 11246: 5.8199, 12085: 6.0764, 12150: 6.2077, 13471: 6.3387, 14275: 6.9927, 14761: 6.5619, 15442: 6.6954, 15527: 6.7244, 16846: 14.3194, 17700: 6.9186, 18607: 6.9549, 19930: 7.4175, 20124: 7.2062, 20824: 7.3063, 24517: 7.6858, 24901: 7.6858, 25488: 7.7659, 26140: 7.7659, 26550: 7.8529, 27819: 8.0535, 27920: 8.0535, 28195: 8.0535, 28201: 8.0535, 28778: 8.0535, 28829: 8.0535, 29040: 8.0535, 29105: 8.0535}), abstract_words_filtered=['past', '10', 'years,', 'major', 'revolution', 'biological/life', 'sciences', 'result', 'remarkable', 'development', 'subsequent', 'generation', 'vast', 'quantity', 'data', 'genome', 'sequencing', 'efforts.', 'genome', 'sequencing', 'process', 'determines', 'identity', 'order', \"organism's\", 'dna;', 'thus', 'providing', 'blueprint', 'genetic', 'code', 'underlying', 'processes', 'life.', '', '', 'highlighted', 'recently', 'completed', 'draft', 'sequence', 'human', 'genome,', 'notable', 'technical', 'achievement', 'made', 'possible', 'development', 'powerful', 'computational', 'hardware', 'software.', '', 'availability', 'complete', 'genome', 'sequences', 'providing', 'foundation', 'understanding', 'organisms', '(from', 'microbes', 'humans)', 'function,', 'develop,', 'evolve', 'interact', 'pathogens', 'assorted', 'environmental', 'cues.', '', 'led', 'transformational', 'change', 'field', 'biology', 'stemming', 'development', 'next-generation', 'ultra', 'high-throughput,', 'ultra', 'low-cost', 'dna', 'sequencing', 'technology', '(', 'e.g.', 'roche', '454),', 'serves', 'basis', 'proposal.', 'used', 'take', 'several', 'months,', 'done', 'several', 'hours.', 'roche', '454', 'flx', 'used', '60', 'faculty', 'members', 'research', 'staff,', 'including', 'post-doctoral', 'fellows,', 'graduate', 'undergraduate', 'students', 'texas', 'a&m', 'university.', '', 'applications', 'research', 'programs', 'include:', '', 'dissection', 'sorghum', 'drought', 'tolerance,', 'functional', 'metagenomic', 'analysis', 'biological', 'communities', 'cattle', 'rumen', '', 'bacteria,', 'disease', 'resistance', '', '', 'analysis', 'microflora', 'ocean', 'drilling', 'programs,', 'among', 'others.', '', 'dissemination', 'data', 'project', 'outcomes', 'available', 'institute', 'plant', 'genomics', 'biotechnology', 'website', '(http://ipgb.tamu.edu),', 'research', 'publications,', 'relevant', 'scientific', 'meetings', 'scheduled', 'campus-wide', 'annual', 'meeting.'])\n",
      "Row(idf_features=SparseVector(50663, {0: 4.9439, 2: 1.6419, 4: 0.7946, 8: 2.7512, 10: 0.8027, 13: 4.3213, 16: 0.7828, 21: 1.6702, 25: 0.8137, 26: 2.0244, 27: 6.0344, 30: 1.3391, 35: 0.9756, 37: 1.0098, 40: 1.2893, 41: 2.315, 49: 2.0758, 54: 3.0154, 58: 6.4447, 62: 4.7825, 63: 1.2769, 79: 1.508, 87: 1.6071, 88: 1.4663, 108: 2.0022, 124: 1.6753, 128: 1.9306, 153: 1.7957, 154: 1.9141, 157: 1.8619, 166: 4.7159, 170: 2.4148, 194: 2.061, 199: 2.0096, 207: 5.7175, 211: 2.5105, 214: 4.1697, 228: 2.168, 232: 2.054, 237: 2.164, 241: 4.7581, 243: 2.1991, 256: 2.2335, 270: 2.2663, 278: 2.3734, 297: 2.1248, 303: 7.8118, 315: 2.4356, 316: 2.1467, 344: 2.4199, 389: 2.489, 394: 7.748, 402: 2.514, 403: 2.3453, 438: 2.5105, 440: 2.6529, 443: 2.5911, 452: 5.544, 483: 2.7794, 490: 2.6203, 493: 5.4513, 500: 2.8526, 506: 2.7289, 514: 2.7641, 539: 6.1007, 548: 3.2633, 552: 5.2877, 561: 2.8118, 562: 2.908, 568: 2.7001, 579: 2.7903, 593: 2.7541, 633: 2.8919, 659: 2.863, 678: 2.8718, 683: 3.2297, 685: 2.9577, 777: 2.8386, 788: 2.8699, 811: 2.897, 828: 2.9302, 837: 2.9302, 850: 3.2735, 891: 3.2716, 898: 2.9923, 992: 3.4188, 1014: 6.5376, 1017: 3.2623, 1028: 3.3571, 1031: 3.1207, 1046: 3.2243, 1070: 3.3551, 1083: 3.3601, 1119: 3.3154, 1205: 3.7213, 1213: 3.6482, 1214: 3.5269, 1231: 3.4362, 1284: 3.8538, 1296: 7.0323, 1319: 3.3663, 1371: 3.3996, 1426: 14.6694, 1446: 3.4869, 1457: 3.7054, 1474: 3.8912, 1527: 3.539, 1654: 3.8538, 1822: 21.7956, 1839: 3.8572, 1844: 3.8757, 2012: 3.8455, 2111: 3.8639, 2193: 3.8895, 2202: 3.9893, 2272: 3.9354, 2283: 4.2643, 2351: 4.0124, 2368: 4.2346, 2378: 3.9798, 2435: 4.1152, 2440: 4.1283, 2447: 4.0163, 2637: 4.1754, 2804: 8.4163, 2999: 4.4856, 3289: 4.3871, 3473: 4.9229, 3492: 15.3804, 3573: 4.918, 3736: 4.547, 4056: 4.6748, 4190: 4.7095, 4313: 5.274, 4321: 5.2009, 4391: 5.0578, 4402: 16.3649, 4433: 4.7536, 4637: 4.8755, 5470: 6.5872, 5769: 5.8199, 5795: 5.086, 6644: 5.302, 6708: 6.2805, 6910: 5.3909, 6930: 5.368, 7288: 5.4386, 8504: 5.6969, 10842: 5.7509, 11340: 5.844, 11485: 5.894, 11727: 6.0461, 12105: 6.0764, 12110: 6.0764, 12229: 6.1903, 13653: 48.3212, 14335: 6.4896, 14409: 6.4896, 14561: 6.5131, 15636: 6.7244, 16567: 6.8168, 17358: 6.8835, 18789: 6.9549, 19931: 7.1597, 21078: 16.1071, 21208: 16.1071, 21271: 16.1071, 21272: 16.1071, 21380: 7.8529, 21488: 16.1071, 23630: 7.6117, 26956: 7.9482, 27525: 8.0535, 27557: 8.0535, 27592: 8.0535, 27595: 8.0535, 27661: 8.0535, 27711: 8.0535, 27758: 8.0535, 27838: 8.0535, 27845: 8.0535, 27987: 8.0535, 28019: 8.0535, 28029: 8.0535, 28077: 8.0535, 28132: 8.0535, 28235: 8.0535, 28306: 8.0535, 28493: 8.0535, 28579: 8.0535, 28592: 8.0535, 28686: 8.0535, 28877: 8.0535, 28921: 8.0535, 29076: 8.0535, 29164: 8.0535, 29244: 8.0535}), abstract_words_filtered=['pi:', 'hugo', 'k.', 'dooner', '(rutgers', 'university)<br/>copis:', 'chunguang', 'du', '(montclair', 'state', 'university)', 'randall', 'a.', 'kerstetter', '(rutgers', 'university)<br/><br/>transposable', 'element', '(te)', 'insertions', 'cause', 'mutations', 'help', 'researchers', 'elucidate', 'gene', 'function.', '', 'collections', 'insertions', 'sequence-indexed', '(i.e.,', 'host', 'dna', 'adjacent', 'insertion', 'known)', 'valuable', 'resources', 'organisms', 'sequenced', 'genome', 'identified', 'maize', 'community', 'essential', 'fully', 'exploit', 'maize', 'genome', 'sequence', 'generated.', '', 'recently,', 'feasibility', 'combining', 'high-throughput', 'dna', 'sequencing', 'efficient', 'multi-dimensional', 'pooling', 'strategies', 'rapidly', 'simultaneously', 'sequence', 'index', 'hundreds', 'new', 'insertions', 'demonstrated.', '', 'project', 'develop', 'use', 'general', 'method', 'generating,', 'sequencing,', 'indexing', 'ac/ds', 'transposable', 'element', 'insertions', 'maize', 'rapid,', 'accurate,', 'cost-effective', 'taking', 'advantage', 'next-generation', 'sequencing', 'technology.', 'specifically,', 'project', 'will:', '', '(1)', 'sequence-index', 'collection', 'existing', 'lines', 'contain', 'unique', 'transposed', 'ac', 'element;', '(2)', 'complete', 'set', '120', 'roughly', 'equidistant', 'transgenic', 'ds', 'elements', 'serve', 'launching', 'platforms', 'carry', 'easily', 'scored', 'markers', 'allow', 'simple', 'visual', 'selection', 'element', 'transposition', 'region', 'genome', 'and,', 'thus,', 'enable', 'researchers', 'generate', 'regional', 'gene', 'knock-out', 'collections,', '(3)', 'sequence-index', '10,000', 'ds', 'element', 'insertion', 'sites', 'model', 'platforms', 'using', 'novel', 'method', 'applicable', 'collection', 'insertions', 'produced', 'common', 'genetic', 'background;', '(4)', 'develop', 'web-searchable', 'database', 'insertion', 'site', 'sequences', 'cross-referenced', 'lines', 'freely', 'available', 'maize', 'genetics', 'stock', 'center', '(http://maizecoop.cropsci.uiuc.edu).', '', 'relevant', 'information', 'project', 'accessible', 'maizegdb', '(http://www.maizegdb.org).<br/><br/>this', 'project', 'addresses', 'critical', 'need', 'deliver', 'sequence-indexed', 'reverse', 'genetics', 'resource,', 'considered', 'essential', 'researchers', 'fully', 'exploit', 'maize', 'genome', 'sequence.', '', 'integrate', 'next-generation', 'sequencing', 'technology', 'rutgers', 'bioinformatic', 'sequence', 'analysis', 'montclair', 'state', 'university,', 'predominantly', 'undergraduate', 'institution', 'nj.', '', 'students', 'msu', 'assist', 'assembly,', 'annotation,', 'mapping', 'short', 'reads', 'putative', 'ac-adjacent', 'sequences', 'design', 'primers', 'validation.', '', 'project', 'also', 'provide', 'research', 'training', 'opportunities', 'msu', 'informatics', 'molecular', 'biology', 'students,', 'many', 'underrepresented', 'groups.', '', 'students', 'rutgers', 'msu', 'also', 'participate', 'project', 'summer', 'interns', 'molecular', 'biology', 'lab', 'maize', 'genetics', 'nursery.'])\n",
      "Row(idf_features=SparseVector(50663, {0: 0.5493, 10: 0.8027, 11: 0.7721, 16: 0.7828, 20: 1.0703, 21: 0.8351, 26: 2.0244, 27: 6.0344, 33: 0.9868, 35: 0.9756, 43: 1.1466, 46: 3.6248, 49: 2.0758, 59: 1.7157, 63: 1.2769, 64: 2.7608, 65: 3.0308, 69: 1.373, 71: 3.716, 80: 1.6612, 84: 1.4788, 95: 4.7281, 101: 1.7807, 103: 1.6609, 104: 1.774, 110: 1.595, 111: 3.1334, 125: 1.6652, 131: 1.7519, 139: 2.0417, 141: 1.68, 148: 1.813, 150: 1.9204, 188: 1.9296, 198: 1.9944, 201: 5.9679, 206: 2.3209, 207: 2.8587, 216: 2.3372, 225: 2.2639, 238: 2.2377, 240: 2.3073, 284: 2.2724, 285: 2.2551, 288: 2.2683, 300: 4.4955, 312: 2.4073, 314: 2.4742, 342: 2.3802, 360: 2.2534, 377: 2.7513, 394: 2.5827, 429: 2.4514, 433: 2.6896, 435: 2.4038, 446: 7.9019, 467: 5.2749, 474: 2.6907, 480: 2.4848, 483: 2.7794, 487: 2.5514, 495: 8.693, 503: 2.6624, 506: 2.7289, 512: 2.529, 556: 2.5992, 561: 2.8118, 569: 7.9601, 582: 3.2468, 594: 2.6665, 600: 8.2408, 618: 2.772, 634: 3.1565, 653: 2.8453, 655: 2.7117, 665: 3.0051, 672: 5.6954, 708: 2.8356, 712: 3.0884, 723: 25.6329, 764: 2.7955, 787: 2.8774, 791: 3.2468, 798: 2.9916, 801: 2.844, 816: 3.013, 820: 2.86, 839: 3.1368, 882: 2.9415, 885: 3.034, 958: 3.4652, 963: 3.3105, 974: 3.1741, 977: 3.1623, 980: 3.1191, 984: 6.4972, 1023: 3.2623, 1045: 3.4166, 1070: 3.3551, 1083: 3.3601, 1110: 3.169, 1179: 3.5091, 1183: 3.8162, 1216: 3.2763, 1244: 4.201, 1262: 3.3807, 1299: 3.344, 1336: 3.5079, 1381: 3.4754, 1462: 3.5293, 1474: 3.8912, 1488: 3.5825, 1507: 3.886, 1518: 4.0402, 1566: 3.6482, 1583: 3.6701, 1590: 3.6135, 1611: 3.5587, 1614: 3.6174, 1631: 3.7155, 1650: 3.8472, 1659: 3.7272, 1704: 3.7631, 1734: 3.8066, 1837: 3.7083, 1854: 3.9445, 1887: 13.2304, 1899: 3.736, 1931: 3.7893, 1934: 3.7753, 1935: 3.7924, 1980: 3.8656, 1997: 3.7956, 2245: 4.0645, 2279: 3.95, 2297: 4.0707, 2322: 3.95, 2410: 4.0462, 2413: 4.0183, 2417: 4.3843, 2442: 4.1327, 2514: 4.0564, 2648: 4.1109, 2658: 4.3238, 2737: 4.3292, 2882: 4.47, 2954: 4.47, 2982: 4.2618, 3042: 4.2846, 3100: 4.2949, 3129: 4.3265, 3135: 4.308, 3182: 4.6939, 3239: 4.6901, 3267: 9.5154, 3295: 4.5537, 3449: 4.4277, 3507: 4.4486, 3515: 4.4517, 3523: 4.7618, 3549: 4.4793, 3619: 4.5174, 3631: 4.5304, 3807: 9.4506, 3821: 4.5706, 4067: 4.645, 4427: 4.7743, 4433: 4.7536, 4503: 4.766, 4564: 4.7912, 4576: 4.7997, 4637: 4.8755, 4641: 4.9229, 4670: 4.817, 4735: 4.8571, 5167: 4.9524, 5651: 5.115, 5779: 5.115, 6038: 5.1756, 6223: 5.1882, 6326: 5.2203, 6371: 5.2335, 6391: 5.2401, 6414: 5.2535, 6692: 5.3604, 7019: 5.3832, 7604: 5.4972, 7619: 5.5146, 8856: 5.7509, 9823: 18.1833, 11420: 5.8813, 11434: 5.8813, 11524: 5.92, 11592: 5.9467, 11636: 5.9467, 12075: 6.0764, 12220: 6.1564, 12638: 6.3189, 12832: 6.2077, 13354: 6.2995, 13591: 30.7432, 13592: 30.7432, 13686: 6.3796, 13726: 6.3796, 13972: 6.4441, 14825: 6.6954, 14856: 6.9927, 15670: 6.785, 16331: 6.7543, 16358: 14.4125, 16520: 6.785, 16663: 6.8168, 16743: 6.8168, 17150: 14.6126, 17214: 6.8496, 17311: 6.8496, 18610: 15.0854, 19458: 7.0727, 19533: 7.0727, 19594: 7.1153, 19606: 7.1153, 19780: 7.1597, 20002: 7.1597, 20094: 7.2062, 20151: 7.2062, 20629: 7.3063, 20985: 7.3604, 21103: 7.4175, 21430: 7.4175, 21676: 7.4782, 23076: 7.6117, 23125: 7.6117, 23235: 7.6117, 23664: 7.6117, 23943: 7.6858, 23949: 7.6858, 24003: 7.6858, 24028: 7.6858, 24066: 7.6858, 24084: 7.6858, 24118: 7.6858, 24127: 7.6858, 24132: 7.6858, 24143: 7.6858, 24182: 7.6858, 24210: 7.6858, 24263: 7.6858, 24281: 7.6858, 24302: 7.6858, 24331: 7.6858, 24337: 7.6858, 24387: 7.6858, 24465: 7.6858, 24588: 7.6858, 24602: 7.6858, 24641: 7.6858, 24662: 7.6858, 24681: 7.6858, 24686: 7.6858, 24727: 7.6858, 24735: 7.6858, 24738: 7.6858, 24740: 7.6858, 24765: 7.6858, 24782: 7.6858, 24864: 7.6858, 24899: 7.6858, 24927: 7.6858, 24949: 7.6858, 24955: 7.6858, 24967: 7.6858, 24979: 7.6858, 24982: 7.6858, 25083: 7.6858, 25125: 7.6858, 25137: 7.6858, 25178: 7.6858, 25184: 7.6858, 25245: 7.6858, 25246: 7.6858, 25256: 7.6858, 25274: 7.6858}), abstract_words_filtered=[\"ocean's\", 'biomass', 'diversity', 'predominantly', 'microbial,', 'yet', 'aspect', 'diversity', 'remains', 'underexplored.', 'efforts', 'recent', 'years', 'begun', 'document', 'microbial', 'diversity', 'marine', 'systems,', 'elucidate', 'processes', 'structure', 'assemblages', 'across', 'space', 'time.', 'project', 'focuses', 'two', 'important', 'sister', 'clades', 'microbial', 'eukaryotes,', 'oligotrich', 'choreotrich', 'ciliates.', 'organisms', 'comprise', 'major', 'component', 'planktonic', 'food', 'webs', 'graze', 'phytoplankton,', 'turn', 'eaten', 'zooplankton', 'larval', 'fish.<br/><br/>earlier', 'molecular', 'work', 'ciliate', 'diversity', 'relied', 'light', 'microscopy,', 'construction', 'clone', 'libraries', 'sanger', 'sequencing.', 'revealed', 'high', 'degree', 'cryptic', 'diversity', '(similar', 'species', 'genetically', 'distinct),', 'surprising,', 'given', 'long-held', 'idea', 'microbes', 'globally', 'distributed', 'species', 'exist,', 'least', 'compared', 'animals', 'plants.', 'past', 'work', 'also', 'showed', 'ciliate', 'assemblages', 'contain', 'highly', 'abundant', 'forms', 'many', 'rare', 'ones,', 'consistent', 'concept', \"'rare\", \"biosphere'.\", 'however,', 'methods', 'limited', 'high', 'costs', 'labor', 'materials,', 'efforts', 'sample', 'local', 'assemblage', 'comprehensively', 'usually', 'resulted', 'undersaturation', '(repeated', 'sampling', 'continued', 'uncover', 'new', 'species).', 'next', 'generation', 'approaches', 'needed', 'truly', 'assess', 'depths', 'biodiversity', 'planktonic', 'ciliates.<br/><br/>this', 'project', 'brings', 'together', 'investigators', 'strengths', 'ecology,', 'taxonomy', 'oceanography', '(pi', 'mcmanus)', 'molecular', 'evolution,', 'systematics', 'bioinformatics', '(pi', 'katz).', 'pyrosequencing', 'used', 'sample', 'oligotrichs', 'choreotrichs', \"'to\", \"exhaustion'\", 'coastal', 'environments.', 'denaturing', 'gradient', 'gel', 'electrophoresis', '(dgge),', 'technique', 'generates', 'fingerprint', 'diversity', 'sample,', 'used', 'pre-select', 'samples', 'pyrosequencing', 'based', 'strong', 'gradients', 'observed', 'composition', 'assemblages', 'relation', 'environmental', 'factors', '(density', 'fronts,', 'thermoclines,', 'etc.).', 'using', 'approaches,', 'combined', 'informatics', 'pipeline', 'already', 'place,', 'project', 'addresses', 'three', 'specific', 'objectives:<br/><br/>objective', '1.', 'determine', 'spatial', 'scale', 'variability', 'ciliate', 'diversity', 'measuring', 'ciliate', 'assemblages', 'change', 'meter,', 'kilometer,', '100', 'km,', 'basin', 'scales.<br/>objective', '2.', 'assess', 'contributions', 'different', 'size', 'classes', 'ciliates', 'overall', 'assemblage', 'diversity.<br/>objective', '3.', 'experimentally', 'evaluate', 'factors', 'control', 'temporal', 'shift', 'individual', 'species', 'rarity', 'commonness', 'natural', 'assemblage,', 'vice', 'versa.<br/><br/>broader', 'impacts:', 'program', 'involve', 'k-12', 'teachers', 'research,', 'initiated', 'previous', 'award,', 'continued.', 'past', 'three', 'years,', 'teachers', 'hosted', 'uconn', 'marine', 'labs,', 'conducted', 'independent', 'research,', 'learned', 'sample', 'enumerate', 'plankton,', 'developed', 'teaching', 'resources', 'subsequent', 'use', 'classroom.', 'project', 'program', 'adapted', 'accommodate', 'shorter,', 'intense', 'experiences', 'teachers', 'prefer,', 'incorporate', 'instructional', 'component', 'broader', 'marine', 'sciences.', '', 'addition', 'teacher', 'program,', 'project', 'involve', 'training', 'postdoctoral', 'fellows', 'undergraduates.', 'include', 'cross-disciplinary', 'work', 'exchanges', 'pi', 'laboratories,', 'focus', 'experimental', 'design,', 'data', 'collection/analysis,', 'manuscript', 'preparation.', 'pis', 'committed', 'involving', 'students', 'underrepresented', 'groups', 'research,', 'pi', 'katz', 'maintains', 'intensive', 'recruiting', 'efforts', 'area.', 'hence,', 'project', 'increase', 'diversity', 'researchers', 'studying', 'biological', 'oceanography.'])\n",
      "Row(idf_features=SparseVector(50663, {0: 0.5493, 10: 0.8027, 11: 0.7721, 16: 0.7828, 20: 1.0703, 21: 0.8351, 26: 2.0244, 27: 6.0344, 33: 0.9868, 35: 0.9756, 43: 1.1466, 46: 3.6248, 49: 2.0758, 59: 1.7157, 63: 1.2769, 64: 2.7608, 65: 3.0308, 69: 1.373, 71: 3.716, 80: 1.6612, 84: 1.4788, 95: 4.7281, 101: 1.7807, 103: 1.6609, 104: 1.774, 110: 1.595, 111: 3.1334, 125: 1.6652, 131: 1.7519, 139: 2.0417, 141: 1.68, 148: 1.813, 150: 1.9204, 188: 1.9296, 198: 1.9944, 201: 5.9679, 206: 2.3209, 207: 2.8587, 216: 2.3372, 225: 2.2639, 238: 2.2377, 240: 2.3073, 284: 2.2724, 285: 2.2551, 288: 2.2683, 300: 4.4955, 312: 2.4073, 314: 2.4742, 342: 2.3802, 360: 2.2534, 377: 2.7513, 394: 2.5827, 429: 2.4514, 433: 2.6896, 435: 2.4038, 446: 7.9019, 467: 5.2749, 474: 2.6907, 480: 2.4848, 483: 2.7794, 487: 2.5514, 495: 8.693, 503: 2.6624, 506: 2.7289, 512: 2.529, 556: 2.5992, 561: 2.8118, 569: 7.9601, 582: 3.2468, 594: 2.6665, 600: 8.2408, 618: 2.772, 634: 3.1565, 653: 2.8453, 655: 2.7117, 665: 3.0051, 672: 5.6954, 708: 2.8356, 712: 3.0884, 723: 25.6329, 764: 2.7955, 787: 2.8774, 791: 3.2468, 798: 2.9916, 801: 2.844, 816: 3.013, 820: 2.86, 839: 3.1368, 882: 2.9415, 885: 3.034, 958: 3.4652, 963: 3.3105, 974: 3.1741, 977: 3.1623, 980: 3.1191, 984: 6.4972, 1023: 3.2623, 1045: 3.4166, 1070: 3.3551, 1083: 3.3601, 1110: 3.169, 1179: 3.5091, 1183: 3.8162, 1216: 3.2763, 1244: 4.201, 1262: 3.3807, 1299: 3.344, 1336: 3.5079, 1381: 3.4754, 1462: 3.5293, 1474: 3.8912, 1488: 3.5825, 1507: 3.886, 1518: 4.0402, 1566: 3.6482, 1583: 3.6701, 1590: 3.6135, 1611: 3.5587, 1614: 3.6174, 1631: 3.7155, 1650: 3.8472, 1659: 3.7272, 1704: 3.7631, 1734: 3.8066, 1837: 3.7083, 1854: 3.9445, 1887: 13.2304, 1899: 3.736, 1931: 3.7893, 1934: 3.7753, 1935: 3.7924, 1980: 3.8656, 1997: 3.7956, 2245: 4.0645, 2279: 3.95, 2297: 4.0707, 2322: 3.95, 2410: 4.0462, 2413: 4.0183, 2417: 4.3843, 2442: 4.1327, 2514: 4.0564, 2648: 4.1109, 2658: 4.3238, 2737: 4.3292, 2882: 4.47, 2954: 4.47, 2982: 4.2618, 3042: 4.2846, 3100: 4.2949, 3129: 4.3265, 3135: 4.308, 3182: 4.6939, 3239: 4.6901, 3267: 9.5154, 3295: 4.5537, 3449: 4.4277, 3507: 4.4486, 3515: 4.4517, 3523: 4.7618, 3549: 4.4793, 3619: 4.5174, 3631: 4.5304, 3807: 9.4506, 3821: 4.5706, 4067: 4.645, 4427: 4.7743, 4433: 4.7536, 4503: 4.766, 4564: 4.7912, 4576: 4.7997, 4637: 4.8755, 4641: 4.9229, 4670: 4.817, 4735: 4.8571, 5167: 4.9524, 5651: 5.115, 5779: 5.115, 6038: 5.1756, 6223: 5.1882, 6326: 5.2203, 6371: 5.2335, 6391: 5.2401, 6414: 5.2535, 6692: 5.3604, 7019: 5.3832, 7604: 5.4972, 7619: 5.5146, 8856: 5.7509, 9823: 18.1833, 11420: 5.8813, 11434: 5.8813, 11524: 5.92, 11592: 5.9467, 11636: 5.9467, 12075: 6.0764, 12220: 6.1564, 12638: 6.3189, 12832: 6.2077, 13354: 6.2995, 13591: 30.7432, 13592: 30.7432, 13686: 6.3796, 13726: 6.3796, 13972: 6.4441, 14825: 6.6954, 14856: 6.9927, 15670: 6.785, 16331: 6.7543, 16358: 14.4125, 16520: 6.785, 16663: 6.8168, 16743: 6.8168, 17150: 14.6126, 17214: 6.8496, 17311: 6.8496, 18610: 15.0854, 19458: 7.0727, 19533: 7.0727, 19594: 7.1153, 19606: 7.1153, 19780: 7.1597, 20002: 7.1597, 20094: 7.2062, 20151: 7.2062, 20629: 7.3063, 20985: 7.3604, 21103: 7.4175, 21430: 7.4175, 21676: 7.4782, 23076: 7.6117, 23125: 7.6117, 23235: 7.6117, 23664: 7.6117, 23943: 7.6858, 23949: 7.6858, 24003: 7.6858, 24028: 7.6858, 24066: 7.6858, 24084: 7.6858, 24118: 7.6858, 24127: 7.6858, 24132: 7.6858, 24143: 7.6858, 24182: 7.6858, 24210: 7.6858, 24263: 7.6858, 24281: 7.6858, 24302: 7.6858, 24331: 7.6858, 24337: 7.6858, 24387: 7.6858, 24465: 7.6858, 24588: 7.6858, 24602: 7.6858, 24641: 7.6858, 24662: 7.6858, 24681: 7.6858, 24686: 7.6858, 24727: 7.6858, 24735: 7.6858, 24738: 7.6858, 24740: 7.6858, 24765: 7.6858, 24782: 7.6858, 24864: 7.6858, 24899: 7.6858, 24927: 7.6858, 24949: 7.6858, 24955: 7.6858, 24967: 7.6858, 24979: 7.6858, 24982: 7.6858, 25083: 7.6858, 25125: 7.6858, 25137: 7.6858, 25178: 7.6858, 25184: 7.6858, 25245: 7.6858, 25246: 7.6858, 25256: 7.6858, 25274: 7.6858}), abstract_words_filtered=[\"ocean's\", 'biomass', 'diversity', 'predominantly', 'microbial,', 'yet', 'aspect', 'diversity', 'remains', 'underexplored.', 'efforts', 'recent', 'years', 'begun', 'document', 'microbial', 'diversity', 'marine', 'systems,', 'elucidate', 'processes', 'structure', 'assemblages', 'across', 'space', 'time.', 'project', 'focuses', 'two', 'important', 'sister', 'clades', 'microbial', 'eukaryotes,', 'oligotrich', 'choreotrich', 'ciliates.', 'organisms', 'comprise', 'major', 'component', 'planktonic', 'food', 'webs', 'graze', 'phytoplankton,', 'turn', 'eaten', 'zooplankton', 'larval', 'fish.<br/><br/>earlier', 'molecular', 'work', 'ciliate', 'diversity', 'relied', 'light', 'microscopy,', 'construction', 'clone', 'libraries', 'sanger', 'sequencing.', 'revealed', 'high', 'degree', 'cryptic', 'diversity', '(similar', 'species', 'genetically', 'distinct),', 'surprising,', 'given', 'long-held', 'idea', 'microbes', 'globally', 'distributed', 'species', 'exist,', 'least', 'compared', 'animals', 'plants.', 'past', 'work', 'also', 'showed', 'ciliate', 'assemblages', 'contain', 'highly', 'abundant', 'forms', 'many', 'rare', 'ones,', 'consistent', 'concept', \"'rare\", \"biosphere'.\", 'however,', 'methods', 'limited', 'high', 'costs', 'labor', 'materials,', 'efforts', 'sample', 'local', 'assemblage', 'comprehensively', 'usually', 'resulted', 'undersaturation', '(repeated', 'sampling', 'continued', 'uncover', 'new', 'species).', 'next', 'generation', 'approaches', 'needed', 'truly', 'assess', 'depths', 'biodiversity', 'planktonic', 'ciliates.<br/><br/>this', 'project', 'brings', 'together', 'investigators', 'strengths', 'ecology,', 'taxonomy', 'oceanography', '(pi', 'mcmanus)', 'molecular', 'evolution,', 'systematics', 'bioinformatics', '(pi', 'katz).', 'pyrosequencing', 'used', 'sample', 'oligotrichs', 'choreotrichs', \"'to\", \"exhaustion'\", 'coastal', 'environments.', 'denaturing', 'gradient', 'gel', 'electrophoresis', '(dgge),', 'technique', 'generates', 'fingerprint', 'diversity', 'sample,', 'used', 'pre-select', 'samples', 'pyrosequencing', 'based', 'strong', 'gradients', 'observed', 'composition', 'assemblages', 'relation', 'environmental', 'factors', '(density', 'fronts,', 'thermoclines,', 'etc.).', 'using', 'approaches,', 'combined', 'informatics', 'pipeline', 'already', 'place,', 'project', 'addresses', 'three', 'specific', 'objectives:<br/><br/>objective', '1.', 'determine', 'spatial', 'scale', 'variability', 'ciliate', 'diversity', 'measuring', 'ciliate', 'assemblages', 'change', 'meter,', 'kilometer,', '100', 'km,', 'basin', 'scales.<br/>objective', '2.', 'assess', 'contributions', 'different', 'size', 'classes', 'ciliates', 'overall', 'assemblage', 'diversity.<br/>objective', '3.', 'experimentally', 'evaluate', 'factors', 'control', 'temporal', 'shift', 'individual', 'species', 'rarity', 'commonness', 'natural', 'assemblage,', 'vice', 'versa.<br/><br/>broader', 'impacts:', 'program', 'involve', 'k-12', 'teachers', 'research,', 'initiated', 'previous', 'award,', 'continued.', 'past', 'three', 'years,', 'teachers', 'hosted', 'uconn', 'marine', 'labs,', 'conducted', 'independent', 'research,', 'learned', 'sample', 'enumerate', 'plankton,', 'developed', 'teaching', 'resources', 'subsequent', 'use', 'classroom.', 'project', 'program', 'adapted', 'accommodate', 'shorter,', 'intense', 'experiences', 'teachers', 'prefer,', 'incorporate', 'instructional', 'component', 'broader', 'marine', 'sciences.', '', 'addition', 'teacher', 'program,', 'project', 'involve', 'training', 'postdoctoral', 'fellows', 'undergraduates.', 'include', 'cross-disciplinary', 'work', 'exchanges', 'pi', 'laboratories,', 'focus', 'experimental', 'design,', 'data', 'collection/analysis,', 'manuscript', 'preparation.', 'pis', 'committed', 'involving', 'students', 'underrepresented', 'groups', 'research,', 'pi', 'katz', 'maintains', 'intensive', 'recruiting', 'efforts', 'area.', 'hence,', 'project', 'increase', 'diversity', 'researchers', 'studying', 'biological', 'oceanography.'])\n",
      "Row(idf_features=SparseVector(50663, {0: 0.5493, 2: 0.821, 8: 2.7512, 13: 1.4404, 14: 0.2706, 21: 0.8351, 43: 1.1466, 49: 4.1515, 52: 1.2016, 58: 3.2223, 59: 1.7157, 62: 3.1883, 64: 2.7608, 70: 1.6693, 83: 3.6196, 88: 1.4663, 95: 1.576, 112: 1.7191, 124: 1.6753, 128: 3.8611, 130: 1.7493, 137: 2.3658, 144: 1.7752, 145: 2.2249, 148: 1.813, 156: 4.0949, 161: 4.072, 165: 2.0017, 167: 2.1131, 198: 1.9944, 201: 5.9679, 219: 4.025, 220: 2.1, 227: 2.1576, 231: 2.0902, 233: 4.7121, 256: 2.2335, 257: 2.2023, 259: 2.1254, 266: 2.3871, 349: 4.8304, 356: 2.2434, 362: 2.3435, 398: 2.4797, 429: 2.4514, 455: 5.8737, 457: 7.5173, 468: 5.2562, 493: 2.7257, 499: 5.6832, 509: 2.686, 524: 2.7257, 539: 3.0503, 573: 2.9289, 608: 2.9611, 655: 2.7117, 702: 2.9125, 720: 5.7199, 730: 2.8761, 782: 2.9874, 807: 2.906, 898: 2.9923, 915: 3.3632, 986: 3.198, 1139: 3.4286, 1147: 3.4439, 1242: 3.4406, 1303: 3.6715, 1386: 3.5902, 1448: 3.5079, 1887: 17.6405, 1934: 3.7753, 2105: 3.9193, 2114: 3.9463, 2232: 4.3106, 2250: 3.9855, 2297: 8.1415, 2410: 4.0462, 2519: 4.3027, 2713: 4.6304, 3135: 4.308, 3434: 4.4336, 3560: 4.645, 3592: 4.671, 3664: 11.0825, 4013: 5.0358, 4157: 4.9277, 4502: 4.7954, 4556: 5.0358, 4878: 4.8663, 5158: 4.9524, 5834: 11.1188, 5860: 5.1328, 6566: 5.3308, 7019: 5.3832, 7322: 5.4467, 7389: 5.5323, 7620: 5.8319, 7680: 5.5146, 11603: 5.9467, 11994: 6.0461, 12226: 6.5619, 12278: 6.1236, 12683: 6.3796, 13295: 6.2805, 14787: 6.5619, 14904: 6.5872, 15282: 6.8496, 16536: 6.785, 22004: 7.5427, 22996: 7.6117, 23655: 7.6117, 24395: 16.6097, 26966: 7.9482, 27035: 7.9482, 29868: 8.1713, 30610: 8.3048, 30941: 8.3048, 30954: 8.3048, 31648: 8.3048}), abstract_words_filtered=['grant', 'support', 'workshop', 'focused', 'developing', 'ideas', 'training', 'course', 'marine', 'microbial', 'genomics,', 'consistent', 'recommendations', 'us-european', 'commission', 'taskforce', 'biotechnology', 'research.', 'leading', 'scientists', 'us', 'europe', 'meet', 'october,', '2010', 'washington,', 'dc.', 'experts', 'discuss', 'recent', 'scientific', 'progress', 'marine', 'microbiology', 'accrued', 'modern', 'genomic', 'approaches,', 'well', 'opportunities', 'challenges', 'continued', 'progress', 'given', 'fast', 'pace', 'genome', 'technology', 'advances.', 'analyses', 'state', 'science', 'technology', 'frame', 'consideration', 'effective', 'ways', 'train', 'next-generation', 'scientists', 'capitalize', 'advances', 'tackle', 'technological', 'challenges,', 'prerequisite', 'continued', 'progress', 'important', 'scientific', 'area.', '<br/><br/>workshop', 'sessions', 'examine', 'computational', 'methods', 'used', 'process', 'large', 'amounts', 'dna', 'sequence', 'information,', 'current', 'future', 'dna', 'sequencing', 'methodologies', 'tools', 'used', 'connect', 'gene', 'genome', 'sequence', 'information', 'functional', 'properties', 'relevant', 'microbes.', 'outcome', 'presentations', 'discussions', 'workshop', 'report', 'provides', 'overview', 'critical', 'issues', 'provides', 'recommendations', 'session.', 'also', 'include', 'basic', 'description', 'advanced', 'training', 'course', 'topic', 'genomic', 'analyses', 'marine', 'microbes.<br/><br/>broader', 'impacts:', '', 'entire', 'grant', 'focused', 'needs', 'united', 'states', 'european', 'commission', 'continue', 'work', 'together', 'field', 'marine', 'microbial', 'genomics', 'order', 'expedite', 'advances', 'basic', 'sciences', 'biotechnology.'])\n"
     ]
    }
   ],
   "source": [
    "#Generate IDF Model (Inverse Document Frequency) \n",
    "idf = IDF(inputCol=\"cv_Features\", outputCol=\"idf_features\")\n",
    "idfModel = idf.fit(cv_result)\n",
    "rescaledData = idfModel.transform(cv_result)\n",
    "\n",
    "for features_label in rescaledData.select(\"idf_features\", \"abstract_words_filtered\").take(20):\n",
    "    print(features_label)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 103,
   "metadata": {},
   "outputs": [],
   "source": [
    "spark.stop()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.6.4"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
