{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "#load dependencies\n",
    "import requests\n",
    "import json\n",
    "import pandas as pd\n",
    "import time"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style>\n",
       "    .dataframe thead tr:only-child th {\n",
       "        text-align: right;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: left;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>keyterm</th>\n",
       "      <th>client_list_frequency</th>\n",
       "      <th>NIH_result_output</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>next-gen sequencing</td>\n",
       "      <td>31</td>\n",
       "      <td>1778</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>RNA sequencing</td>\n",
       "      <td>14</td>\n",
       "      <td>6953</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>RNAseq</td>\n",
       "      <td>17</td>\n",
       "      <td>4369</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>RNA seq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>ribonucleic acid sequencing</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>mouse</td>\n",
       "      <td>9</td>\n",
       "      <td>na</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>mtDNAseq</td>\n",
       "      <td>8</td>\n",
       "      <td>na</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>mtDNA seq</td>\n",
       "      <td>1</td>\n",
       "      <td>48</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>mtDNA sequencing</td>\n",
       "      <td>na</td>\n",
       "      <td>69</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>mitochondrial dna sequencing</td>\n",
       "      <td>1</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>mitochondrial DNA sequencing</td>\n",
       "      <td>8</td>\n",
       "      <td>218</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>TCR sequencing</td>\n",
       "      <td>4</td>\n",
       "      <td>148</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>TCRseq</td>\n",
       "      <td>3</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>TCR seq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>T Cell Receptor Sequencing</td>\n",
       "      <td>na</td>\n",
       "      <td>2157</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15</th>\n",
       "      <td>epigenetics</td>\n",
       "      <td>4</td>\n",
       "      <td>672</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>16</th>\n",
       "      <td>amplicon sequencing</td>\n",
       "      <td>5</td>\n",
       "      <td>143</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>17</th>\n",
       "      <td>amplicon seq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18</th>\n",
       "      <td>CHIP sequencing</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19</th>\n",
       "      <td>CHIP seq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>20</th>\n",
       "      <td>ChIP-sequencing</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>21</th>\n",
       "      <td>ChIP-seq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>22</th>\n",
       "      <td>ChIPseq</td>\n",
       "      <td>2</td>\n",
       "      <td>48</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>23</th>\n",
       "      <td>immunology</td>\n",
       "      <td>3</td>\n",
       "      <td>2828</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>24</th>\n",
       "      <td>mitochondrial drug toxicity</td>\n",
       "      <td>2</td>\n",
       "      <td>230</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>25</th>\n",
       "      <td>DNA seq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>26</th>\n",
       "      <td>DNAseq</td>\n",
       "      <td>1</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>27</th>\n",
       "      <td>DNA sequencing</td>\n",
       "      <td>1</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>28</th>\n",
       "      <td>smRNA seq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>29</th>\n",
       "      <td>smRNAseq</td>\n",
       "      <td>1</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>38</th>\n",
       "      <td>transposon sequencing</td>\n",
       "      <td>1</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>39</th>\n",
       "      <td>Tn-seq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>40</th>\n",
       "      <td>transposon seq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>41</th>\n",
       "      <td>tn seq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>42</th>\n",
       "      <td>tn sequencing</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>43</th>\n",
       "      <td>blood</td>\n",
       "      <td>1</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>44</th>\n",
       "      <td>bird</td>\n",
       "      <td>1</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>45</th>\n",
       "      <td>autism</td>\n",
       "      <td>1</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>46</th>\n",
       "      <td>gene expression</td>\n",
       "      <td>1</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>47</th>\n",
       "      <td>Pig</td>\n",
       "      <td>1</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>48</th>\n",
       "      <td>IBD</td>\n",
       "      <td>1</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>49</th>\n",
       "      <td>mycobacterium</td>\n",
       "      <td>1</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>50</th>\n",
       "      <td>female health</td>\n",
       "      <td>1</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>51</th>\n",
       "      <td>microbial genetics</td>\n",
       "      <td>1</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>52</th>\n",
       "      <td>human</td>\n",
       "      <td>2</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>53</th>\n",
       "      <td>rat</td>\n",
       "      <td>1</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>54</th>\n",
       "      <td>microbiome</td>\n",
       "      <td>1</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>55</th>\n",
       "      <td>microbiome sequencing</td>\n",
       "      <td>1</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>56</th>\n",
       "      <td>genomics</td>\n",
       "      <td>1</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>57</th>\n",
       "      <td>regenerative medicine</td>\n",
       "      <td>1</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>58</th>\n",
       "      <td>developmental and regenerative biology</td>\n",
       "      <td>1</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>59</th>\n",
       "      <td>bacterial genetics</td>\n",
       "      <td>1</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>60</th>\n",
       "      <td>zebrafish</td>\n",
       "      <td>1</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>61</th>\n",
       "      <td>stem cell biology</td>\n",
       "      <td>1</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>62</th>\n",
       "      <td>species identification</td>\n",
       "      <td>1</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>63</th>\n",
       "      <td>saliva</td>\n",
       "      <td>1</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>64</th>\n",
       "      <td>prostate cancer</td>\n",
       "      <td>1</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>65</th>\n",
       "      <td>phylogenetics</td>\n",
       "      <td>1</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>66</th>\n",
       "      <td>parkinson's disease</td>\n",
       "      <td>1</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>67</th>\n",
       "      <td>leukemia</td>\n",
       "      <td>1</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>68 rows Ã— 3 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                   keyterm client_list_frequency  \\\n",
       "0                      next-gen sequencing                    31   \n",
       "1                           RNA sequencing                    14   \n",
       "2                                   RNAseq                    17   \n",
       "3                                  RNA seq                   NaN   \n",
       "4              ribonucleic acid sequencing                   NaN   \n",
       "5                                    mouse                     9   \n",
       "6                                 mtDNAseq                     8   \n",
       "7                                mtDNA seq                     1   \n",
       "8                         mtDNA sequencing                    na   \n",
       "9             mitochondrial dna sequencing                     1   \n",
       "10            mitochondrial DNA sequencing                     8   \n",
       "11                          TCR sequencing                     4   \n",
       "12                                  TCRseq                     3   \n",
       "13                                 TCR seq                   NaN   \n",
       "14              T Cell Receptor Sequencing                   na    \n",
       "15                             epigenetics                     4   \n",
       "16                     amplicon sequencing                     5   \n",
       "17                            amplicon seq                   NaN   \n",
       "18                         CHIP sequencing                   NaN   \n",
       "19                                CHIP seq                   NaN   \n",
       "20                         ChIP-sequencing                   NaN   \n",
       "21                                ChIP-seq                   NaN   \n",
       "22                                 ChIPseq                     2   \n",
       "23                              immunology                     3   \n",
       "24             mitochondrial drug toxicity                     2   \n",
       "25                                 DNA seq                   NaN   \n",
       "26                                  DNAseq                     1   \n",
       "27                          DNA sequencing                     1   \n",
       "28                               smRNA seq                   NaN   \n",
       "29                                smRNAseq                     1   \n",
       "..                                     ...                   ...   \n",
       "38                   transposon sequencing                     1   \n",
       "39                                  Tn-seq                   NaN   \n",
       "40                          transposon seq                   NaN   \n",
       "41                                  tn seq                   NaN   \n",
       "42                           tn sequencing                   NaN   \n",
       "43                                   blood                     1   \n",
       "44                                    bird                     1   \n",
       "45                                  autism                     1   \n",
       "46                         gene expression                     1   \n",
       "47                                     Pig                     1   \n",
       "48                                     IBD                     1   \n",
       "49                           mycobacterium                     1   \n",
       "50                           female health                     1   \n",
       "51                      microbial genetics                     1   \n",
       "52                                   human                     2   \n",
       "53                                     rat                     1   \n",
       "54                              microbiome                     1   \n",
       "55                   microbiome sequencing                     1   \n",
       "56                                genomics                     1   \n",
       "57                   regenerative medicine                     1   \n",
       "58  developmental and regenerative biology                     1   \n",
       "59                      bacterial genetics                     1   \n",
       "60                               zebrafish                     1   \n",
       "61                       stem cell biology                     1   \n",
       "62                  species identification                     1   \n",
       "63                                  saliva                     1   \n",
       "64                         prostate cancer                     1   \n",
       "65                           phylogenetics                     1   \n",
       "66                     parkinson's disease                     1   \n",
       "67                                leukemia                     1   \n",
       "\n",
       "   NIH_result_output  \n",
       "0               1778  \n",
       "1               6953  \n",
       "2               4369  \n",
       "3                NaN  \n",
       "4                NaN  \n",
       "5                 na  \n",
       "6                 na  \n",
       "7                 48  \n",
       "8                 69  \n",
       "9                NaN  \n",
       "10               218  \n",
       "11               148  \n",
       "12                 1  \n",
       "13               NaN  \n",
       "14              2157  \n",
       "15               672  \n",
       "16               143  \n",
       "17               NaN  \n",
       "18               NaN  \n",
       "19               NaN  \n",
       "20               NaN  \n",
       "21               NaN  \n",
       "22                48  \n",
       "23              2828  \n",
       "24               230  \n",
       "25               NaN  \n",
       "26               NaN  \n",
       "27               NaN  \n",
       "28               NaN  \n",
       "29               NaN  \n",
       "..               ...  \n",
       "38               NaN  \n",
       "39               NaN  \n",
       "40               NaN  \n",
       "41               NaN  \n",
       "42               NaN  \n",
       "43               NaN  \n",
       "44               NaN  \n",
       "45               NaN  \n",
       "46               NaN  \n",
       "47               NaN  \n",
       "48               NaN  \n",
       "49               NaN  \n",
       "50               NaN  \n",
       "51               NaN  \n",
       "52               NaN  \n",
       "53               NaN  \n",
       "54               NaN  \n",
       "55               NaN  \n",
       "56               NaN  \n",
       "57               NaN  \n",
       "58               NaN  \n",
       "59               NaN  \n",
       "60               NaN  \n",
       "61               NaN  \n",
       "62               NaN  \n",
       "63               NaN  \n",
       "64               NaN  \n",
       "65               NaN  \n",
       "66               NaN  \n",
       "67               NaN  \n",
       "\n",
       "[68 rows x 3 columns]"
      ]
     },
     "execution_count": 2,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# key term list\n",
    "key_terms_df = pd.read_csv(\"client_keyword_results.csv\")\n",
    "key_terms = key_terms_df['keyterm']\n",
    "key_terms_df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 38,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['https://api.federalreporter.nih.gov/v1/projects/search?query=text:next-gen+sequencing',\n",
       " 'https://api.federalreporter.nih.gov/v1/projects/search?query=text:RNA+sequencing',\n",
       " 'https://api.federalreporter.nih.gov/v1/projects/search?query=text:RNAseq',\n",
       " 'https://api.federalreporter.nih.gov/v1/projects/search?query=text:RNA+seq',\n",
       " 'https://api.federalreporter.nih.gov/v1/projects/search?query=text:ribonucleic+acid+sequencing']"
      ]
     },
     "execution_count": 38,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "url_list = []\n",
    "\n",
    "for word in key_terms:\n",
    "    word = word.strip().replace(' ', '+')\n",
    "    url = f\"https://api.federalreporter.nih.gov/v1/projects/search?query=text:{word}\"\n",
    "    url_list.append(url)\n",
    "\n",
    "url_list[:5]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 37,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[{'projectNumber': '5K08NS075094-02', 'fy': 2012, 'title': 'NEXT-GENERATION SEQUENCING APPROACHES FOR NOVEL GENE DISCOVERY IN ALS', 'department': 'HHS', 'agency': 'NIH', 'ic': 'NINDS', 'totalCostAmount': 157414, 'nihApplId': 8263027, 'smApplId': 14642, 'budgetStartDate': '2012-07-01T00:00:00', 'budgetEndDate': '2013-06-30T00:00:00', 'contactPi': 'HARMS, MATTHEW', 'otherPis': None, 'congressionalDistrict': '01', 'dunsId': '068552207', 'latitude': 38.648424, 'longitude': -90.309068, 'orgName': 'WASHINGTON UNIVERSITY', 'orgCity': 'SAINT LOUIS', 'orgState': 'MO', 'orgCountry': 'UNITED STATES', 'orgZipCode': '631304862', 'projectStartDate': '2011-07-01T00:00:00', 'projectEndDate': '2016-06-30T00:00:00', 'cfdaCode': '853', 'abstract': \"DESCRIPTION (provided by applicant): The candidate is a clinical neurologist (MD) with a career goal of investigating the genetic and molecular basis of human neuromuscular diseases, focusing primarily on amyotrophic lateral sclerosis (ALS). The candidate has significant prior laboratory experience, with a track record of successful research projects in neuropharmacology, neurodevelopment, and most recently, neurogenetics. In order to further prepare for successful independent research, the candidate's career development plan includes graduate-level courses in biostatistics, bioinformatics, and genomics, as well as seminars and intensive courses on next-generation sequencing and related methods. The proposed development plan and scientific training will take place at Washington University in St. Louis, an institution with particular strengths in both genetics/genomics and the study of ALS. The neurogenetics community is robust and will provide the candidate with important intellectual assistance and collaborations. The scientific training will be jointly mentored by Dr. Alison Goate, whose lab focuses on the genetics of dementia and addiction, and Dr. Robert Baloh, whose lab studies the molecular basis of neurodegeneration in ALS using tissue culture and animal models. Dr. Goate's expertise in human genetics and sequencing methods coupled with Dr. Baloh's experience in modeling human neuromuscular diseases will provide the candidate with the set of research tools needed to succeed as an independent investigator studying the genetic and molecular basis of neurologic diseases. The candidate's research proposal aims to identify novel genes causing ALS, an untreatable neurodegenerative disease producing loss of motor neurons, progressive paralysis, and death from respiratory failure. Although the majority of ALS cases are sporadic, the genes responsible in familial cases have provided important insights into all forms of the disease. However, known ALS genes still only explain 25% of familial cases. To identify additional familial ALS disease genes, the candidate will: 1) Screen a large cohort of familial and sporadic ALS patients for mutations in known ALS genes. This screen will be performed using pooled-sample sequencing, an innovative method made possible by next-generation sequencing capabilities pioneered by one of our key collaborators. Novel disease-causing mutations will be followed up with phenotypic analysis. This screen will also identify a cohort of ALS families without mutations in any known ALS disease gene to be studied with whole-exome sequencing. 2) Familial cases of ALS without mutations in known genes will then undergo whole-exome analysis, a next- generation sequencing method that allows all coding regions in the human genome to be sequenced simultaneously for variations. To narrow the number of candidate pathogenic mutations, a host of bioinformatic techniques will be used. Candidate ALS genes identified will then be screened for additional mutations in other ALS patients using the same pooled-sample sequencing strategy as in (1). The identification of novel ALS disease genes will pave the way for future studies aimed at understanding the molecular mechanism by which these mutations lead to neurodegeneration, using tissue culture and animal modeling. This work will provide key insight into the molecular basis of ALS and will immediately impact efforts aimed at therapeutic development for this untreatable disease.  PUBLIC HEALTH RELEVANCE: Amyotrophic lateral sclerosis (a.k.a. Lou Gehrig's disease) is an untreatable disease characterized by progressive paralysis and eventual death. Identification of genes responsible for hereditary forms of ALS will shed light on why the disease occurs and guide how therapies might be developed. This project uses state-of- the-art genetic sequencing technologies to uncover these genes.  \", 'terms': ' addiction; Age of Onset; Algorithms; Amyotrophic Lateral Sclerosis; Animal Model; base; Bioinformatics; Biological Assay; Biometry; Brain Stem; Candidate Disease Gene; career; career development; Cessation of life; Clinical; Code; cohort; Collaborations; Communities; Coupled; Cytoplasm; Data; Dementia; Development Plans; Disease; Disease Management; disease-causing mutation; DNA; effective therapy; exome; Exons; experience; Familial Amyotrophic Lateral Sclerosis; Family; follow-up; Future; gene discovery; gene function; Gene Mutation; Genes; Genetic; Genetic Heterogeneity; genetic linkage analysis; genetic pedigree; Genetic Techniques; Genome; Genomics; Genotype; Goals; Human; Human Genetics; Human Genome; Incidence; Individual; Inherited; innovation; insight; Institution; Introns; Laboratories; Lead; Light; Mentors; Methods; Minority; Modeling; Molecular; Molecular Genetics; motor neuron degeneration; Motor Neurons; Mutation; Nerve Degeneration; Neurodegenerative Disorders; neurodevelopment; neurogenetics; Neurologic; Neurologist; Neuromuscular Diseases; Neuropharmacology; next generation; novel; novel strategies; Nuclear; palliation; Paralysed; Pathogenesis; Pathogenicity; Pathway interactions; Patients; Penetrance; proband; protein TDP-43; public health relevance; repository; Research; Research Personnel; Research Project Grants; Research Proposals; respiratory; Respiratory Failure; RNA Processing; RNA-Binding Proteins; Sampling; segregation; Spinal Cord; Symptoms; Techniques; Technology; Testing; therapeutic development; tissue culture; tool; Training; Universities; Variant; Washington; Work'}, {'projectNumber': '1U01MH094411-01A1', 'fy': 2012, 'title': '3/4-PSYCHIATRIC GWAS CONSORTIUM: GENOMIC FOLLOW-UP NEXT-GEN SEQUENCING & GENOTYPI', 'department': 'HHS', 'agency': 'NIH', 'ic': 'NIMH', 'totalCostAmount': 504093, 'nihApplId': 8238068, 'smApplId': 18451, 'budgetStartDate': '2012-05-10T00:00:00', 'budgetEndDate': '2013-03-31T00:00:00', 'contactPi': 'SEBAT, JONATHAN', 'otherPis': None, 'congressionalDistrict': '49', 'dunsId': '804355790', 'latitude': 37.240049, 'longitude': -119.503236, 'orgName': 'UNIVERSITY OF CALIFORNIA SAN DIEGO', 'orgCity': 'LA JOLLA', 'orgState': 'CA', 'orgCountry': 'UNITED STATES', 'orgZipCode': '920930934', 'projectStartDate': '2012-05-10T00:00:00', 'projectEndDate': '2016-03-31T00:00:00', 'cfdaCode': '242', 'abstract': \"DESCRIPTION (provided by applicant): This is the final submission of the collaborative R01 MH094421 Psychiatric GWAS Consortium: Genomic Follow-Up Next-Gen Sequencing & Genotyping. The overall goal of this application is ambitious: we aim to generate a trustworthy, high-confidence 'maps' of the genetic architecture of centrally important psychiatric diseases. Such maps consist of systematic and comprehensive evaluation of the allelic spectrum for these disorders including rare exonic, common SNP, and copy number variation. The feasibility of our aims is supported by the track record of the first iteration of the Psychiatric GWAS Consortium ('PGC1') where we united nearly all major groups in the field into a harmonious and functional entity and completed our initial aims to a high scientific standard. Psychiatric diseases are compelling targets for intensive research: they are mostly idiopathic, first-rank public health problems, and cause enormous morbidity, mortality, and personal/societal cost. Consistent with the NIH mission, our goal is to elucidate fundamental knowledge of these diseases. In this 'PGC2' application, we propose to capitalize on prior NIH investments and on the success of PGC1 for the next logical set of aims. The PGC2 aims are 'large-scale' (largest sample sizes ever in the field) and 'comprehensive' (via the careful application of multiple genomic-scale technologies). Our focus is comprehensive in other senses - the PGC encompasses nearly the entire field, and we aim to elucidate the allelic spectrum of these disorders by integrating empirical data for all readily measurable types of genetic variation of etiological relevance (common SNP, rare exonic, and rare and common copy number variation - what we term the 'map'). There are three analytic aims - systematically to assess copy number variation, to create a pipeline for the analysis of exome data (and eventually whole-genome data), and to investigate genetic associations that span traditional disease boundaries. Finally, we propose to develop the 'PsychChip' a custom 20,000 probe array targeting common SNP, exonic, and CNVs that would then be used to genotype 115,082 subjects. The PGC2 impact is potentially very large - a fundamental understanding of the genetics of these diseases would be a major milestone in psychiatry and in biomedicine.  PUBLIC HEALTH RELEVANCE: Psychiatric disorders cause enormous human suffering and cost to society. Our goal is rapidly to learn more about the genetics of some of these disorders. We propose to do this via the largest consortium ever constructed in the field, and by the use of several genetic technologies.  \", 'terms': ' Architecture; Attention deficit hyperactivity disorder; Autistic Disorder; Bioinformatics; Biological; Bipolar Disorder; Budgets; Chromosome Mapping; Comorbidity; Copy Number Polymorphism; cost; Country; Custom; Data; Disease; Disease Association; Epidemiology; Evaluation; exome; follow-up; Funding; Genetic; genetic association; genetic technology; genetic variant; Genetic Variation; Genome; genome sequencing; genome wide association study; Genomics; Genotype; Goals; Grant; Hereditary Disease; Human; Individual; Institution; Intention; Investments; Knowledge; Learning; Major Depressive Disorder; Maps; Measurable; Mental disorders; Mission; Morbidity - disease rate; mortality; National Institute of Mental Health; next generation; Phenotype; Process; Psychiatry; psychogenetics; Public Health; public-private partnership; Recording of previous events; Reporting; Research; research study; Risk; Role; Sample Size; Sampling; Schizophrenia; Scientist; Sequence Analysis; Societies; Source; success; Symptoms; Technology; Testing; Time; Trust; United States National Institutes of Health; Validation; Variant; Work; working group'}, {'projectNumber': '1U01MH094421-01A1', 'fy': 2012, 'title': '1/4 PSYCHIATRIC GWAS CONSORTIUM: GENOMIC FOLLOW-UP NEXT-GEN SEQUENCING & GENOTYPI', 'department': 'HHS', 'agency': 'NIH', 'ic': 'NIMH', 'totalCostAmount': 491979, 'nihApplId': 8237640, 'smApplId': 18452, 'budgetStartDate': '2012-05-10T00:00:00', 'budgetEndDate': '2013-03-31T00:00:00', 'contactPi': 'SULLIVAN, PATRICK F', 'otherPis': None, 'congressionalDistrict': '04', 'dunsId': '608195277', 'latitude': 35.366248, 'longitude': -79.429394, 'orgName': 'UNIVERSITY OF NORTH CAROLINA CHAPEL HILL', 'orgCity': 'CHAPEL HILL', 'orgState': 'NC', 'orgCountry': 'UNITED STATES', 'orgZipCode': '275990001', 'projectStartDate': '2012-05-10T00:00:00', 'projectEndDate': '2016-03-31T00:00:00', 'cfdaCode': '242', 'abstract': \"DESCRIPTION (provided by applicant): This is the final submission of the collaborative R01 MH094421 Psychiatric GWAS Consortium: Genomic Follow-Up Next-Gen Sequencing & Genotyping. The overall goal of this application is ambitious: we aim to generate a trustworthy, high-confidence 'maps' of the genetic architecture of centrally important psychiatric diseases. Such maps consist of systematic and comprehensive evaluation of the allelic spectrum for these disorders including rare exonic, common SNP, and copy number variation. The feasibility of our aims is supported by the track record of the first iteration of the Psychiatric GWAS Consortium ('PGC1') where we united nearly all major groups in the field into a harmonious and functional entity and completed our initial aims to a high scientific standard. Psychiatric diseases are compelling targets for intensive research: they are mostly idiopathic, first-rank public health problems, and cause enormous morbidity, mortality, and personal/societal cost. Consistent with the NIH mission, our goal is to elucidate fundamental knowledge of these diseases. In this 'PGC2' application, we propose to capitalize on prior NIH investments and on the success of PGC1 for the next logical set of aims. The PGC2 aims are 'large-scale' (largest sample sizes ever in the field) and 'comprehensive' (via the careful application of multiple genomic-scale technologies). Our focus is comprehensive in other senses - the PGC encompasses nearly the entire field, and we aim to elucidate the allelic spectrum of these disorders by integrating empirical data for all readily measurable types of genetic variation of etiological relevance (common SNP, rare exonic, and rare and common copy number variation - what we term the 'map'). There are three analytic aims - systematically to assess copy number variation, to create a pipeline for the analysis of exome data (and eventually whole-genome data), and to investigate genetic associations that span traditional disease boundaries. Finally, we propose to develop the 'PsychChip' a custom 20,000 probe array targeting common SNP, exonic, and CNVs that would then be used to genotype 115,082 subjects. The PGC2 impact is potentially very large - a fundamental understanding of the genetics of these diseases would be a major milestone in psychiatry and in biomedicine.  PUBLIC HEALTH RELEVANCE: Psychiatric disorders cause enormous human suffering and cost to society. Our goal is rapidly to learn more about the genetics of some of these disorders. We propose to do this via the largest consortium ever constructed in the field, and by the use of several genetic technologies.  \", 'terms': ' Architecture; Attention deficit hyperactivity disorder; Autistic Disorder; Bioinformatics; Biological; Bipolar Disorder; Budgets; Chromosome Mapping; Comorbidity; Copy Number Polymorphism; cost; Country; Custom; Data; Disease; Disease Association; Epidemiology; Evaluation; exome; follow-up; Funding; Genetic; genetic association; genetic technology; genetic variant; Genetic Variation; Genome; genome sequencing; genome wide association study; Genomics; Genotype; Goals; Grant; Hereditary Disease; Human; Individual; Institution; Intention; Investments; Knowledge; Learning; Major Depressive Disorder; Maps; Measurable; Mental disorders; Mission; Morbidity - disease rate; mortality; National Institute of Mental Health; next generation; Phenotype; Process; Psychiatry; psychogenetics; Public Health; public-private partnership; Recording of previous events; Reporting; Research; research study; Risk; Role; Sample Size; Sampling; Schizophrenia; Scientist; Sequence Analysis; Societies; Source; success; Symptoms; Technology; Testing; Time; Trust; United States National Institutes of Health; Validation; Variant; Work; working group'}, {'projectNumber': '1U01MH094432-01A1', 'fy': 2012, 'title': '2/4-PSYCHIATRIC GWAS CONSORTIUM: GENOMIC FOLLOW-UP NEXT-GEN SEQUENCING & GENOTYPI', 'department': 'HHS', 'agency': 'NIH', 'ic': 'NIMH', 'totalCostAmount': 559381, 'nihApplId': 8238211, 'smApplId': 18453, 'budgetStartDate': '2012-05-10T00:00:00', 'budgetEndDate': '2013-03-31T00:00:00', 'contactPi': 'DALY, MARK JOSEPH', 'otherPis': None, 'congressionalDistrict': '08', 'dunsId': '073130411', 'latitude': 42.347463, 'longitude': -71.082039, 'orgName': 'MASSACHUSETTS GENERAL HOSPITAL', 'orgCity': 'BOSTON', 'orgState': 'MA', 'orgCountry': 'UNITED STATES', 'orgZipCode': '021142696', 'projectStartDate': '2012-05-10T00:00:00', 'projectEndDate': '2016-03-31T00:00:00', 'cfdaCode': '242', 'abstract': \"DESCRIPTION (provided by applicant): This is the final submission of the collaborative R01 MH094421 Psychiatric GWAS Consortium: Genomic Follow-Up Next-Gen Sequencing & Genotyping. The overall goal of this application is ambitious: we aim to generate a trustworthy, high-confidence 'maps' of the genetic architecture of centrally important psychiatric diseases. Such maps consist of systematic and comprehensive evaluation of the allelic spectrum for these disorders including rare exonic, common SNP, and copy number variation. The feasibility of our aims is supported by the track record of the first iteration of the Psychiatric GWAS Consortium ('PGC1') where we united nearly all major groups in the field into a harmonious and functional entity and completed our initial aims to a high scientific standard. Psychiatric diseases are compelling targets for intensive research: they are mostly idiopathic, first-rank public health problems, and cause enormous morbidity, mortality, and personal/societal cost. Consistent with the NIH mission, our goal is to elucidate fundamental knowledge of these diseases. In this 'PGC2' application, we propose to capitalize on prior NIH investments and on the success of PGC1 for the next logical set of aims. The PGC2 aims are 'large-scale' (largest sample sizes ever in the field) and 'comprehensive' (via the careful application of multiple genomic-scale technologies). Our focus is comprehensive in other senses - the PGC encompasses nearly the entire field, and we aim to elucidate the allelic spectrum of these disorders by integrating empirical data for all readily measurable types of genetic variation of etiological relevance (common SNP, rare exonic, and rare and common copy number variation - what we term the 'map'). There are three analytic aims - systematically to assess copy number variation, to create a pipeline for the analysis of exome data (and eventually whole-genome data), and to investigate genetic associations that span traditional disease boundaries. Finally, we propose to develop the 'PsychChip' a custom 20,000 probe array targeting common SNP, exonic, and CNVs that would then be used to genotype 115,082 subjects. The PGC2 impact is potentially very large - a fundamental understanding of the genetics of these diseases would be a major milestone in psychiatry and in biomedicine.  PUBLIC HEALTH RELEVANCE: Psychiatric disorders cause enormous human suffering and cost to society. Our goal is rapidly to learn more about the genetics of some of these disorders. We propose to do this via the largest consortium ever constructed in the field, and by the use of several genetic technologies.  \", 'terms': ' Architecture; Attention deficit hyperactivity disorder; Autistic Disorder; Bioinformatics; Biological; Bipolar Disorder; Budgets; Chromosome Mapping; Comorbidity; Copy Number Polymorphism; cost; Country; Custom; Data; Disease; Disease Association; Epidemiology; Evaluation; exome; follow-up; Funding; Genetic; genetic association; genetic technology; genetic variant; Genetic Variation; Genome; genome sequencing; genome wide association study; Genomics; Genotype; Goals; Grant; Hereditary Disease; Human; Individual; Institution; Intention; Investments; Knowledge; Learning; Major Depressive Disorder; Maps; Measurable; Mental disorders; Mission; Morbidity - disease rate; mortality; National Institute of Mental Health; next generation; Phenotype; Process; Psychiatry; psychogenetics; Public Health; public-private partnership; Recording of previous events; Reporting; Research; research study; Risk; Role; Sample Size; Sampling; Schizophrenia; Scientist; Sequence Analysis; Societies; Source; success; Symptoms; Technology; Testing; Time; Trust; United States National Institutes of Health; Validation; Variant; Work; working group'}, {'projectNumber': '1U01MH096296-01', 'fy': 2012, 'title': '4/4-PSYCHIATRIC GWAS CONSORTIUM:GENOMIC FOLLOW-UP NEXT-GEN SEQUENCING & GENOTYPI', 'department': 'HHS', 'agency': 'NIH', 'ic': 'NIMH', 'totalCostAmount': 297635, 'nihApplId': 8238750, 'smApplId': 18454, 'budgetStartDate': '2012-05-10T00:00:00', 'budgetEndDate': '2013-03-31T00:00:00', 'contactPi': 'SKLAR, PAMELA', 'otherPis': None, 'congressionalDistrict': '13', 'dunsId': '078861598', 'latitude': 40.790552, 'longitude': -73.95385, 'orgName': 'MOUNT SINAI SCHOOL OF MEDICINE', 'orgCity': 'NEW YORK', 'orgState': 'NY', 'orgCountry': 'UNITED STATES', 'orgZipCode': '100296574', 'projectStartDate': '2012-05-10T00:00:00', 'projectEndDate': '2016-03-31T00:00:00', 'cfdaCode': '242', 'abstract': \"DESCRIPTION (provided by applicant): This is the final submission of the collaborative R01 MH094421 Psychiatric GWAS Consortium: Genomic Follow-Up Next-Gen Sequencing & Genotyping. The overall goal of this application is ambitious: we aim to generate a trustworthy, high-confidence 'maps' of the genetic architecture of centrally important psychiatric diseases. Such maps consist of systematic and comprehensive evaluation of the allelic spectrum for these disorders including rare exonic, common SNP, and copy number variation. The feasibility of our aims is supported by the track record of the first iteration of the Psychiatric GWAS Consortium ('PGC1') where we united nearly all major groups in the field into a harmonious and functional entity and completed our initial aims to a high scientific standard. Psychiatric diseases are compelling targets for intensive research: they are mostly idiopathic, first-rank public health problems, and cause enormous morbidity, mortality, and personal/societal cost. Consistent with the NIH mission, our goal is to elucidate fundamental knowledge of these diseases. In this 'PGC2' application, we propose to capitalize on prior NIH investments and on the success of PGC1 for the next logical set of aims. The PGC2 aims are 'large-scale' (largest sample sizes ever in the field) and 'comprehensive' (via the careful application of multiple genomic-scale technologies). Our focus is comprehensive in other senses - the PGC encompasses nearly the entire field, and we aim to elucidate the allelic spectrum of these disorders by integrating empirical data for all readily measurable types of genetic variation of etiological relevance (common SNP, rare exonic, and rare and common copy number variation - what we term the 'map'). There are three analytic aims - systematically to assess copy number variation, to create a pipeline for the analysis of exome data (and eventually whole-genome data), and to investigate genetic associations that span traditional disease boundaries. Finally, we propose to develop the 'PsychChip' a custom 20,000 probe array targeting common SNP, exonic, and CNVs that would then be used to genotype 115,082 subjects. The PGC2 impact is potentially very large - a fundamental understanding of the genetics of these diseases would be a major milestone in psychiatry and in biomedicine.  PUBLIC HEALTH RELEVANCE: Psychiatric disorders cause enormous human suffering and cost to society. Our goal is rapidly to learn more about the genetics of some of these disorders. We propose to do this via the largest consortium ever constructed in the field, and by the use of several genetic technologies.  \", 'terms': ' Architecture; Attention deficit hyperactivity disorder; Autistic Disorder; Bioinformatics; Biological; Bipolar Disorder; Budgets; Chromosome Mapping; Comorbidity; Copy Number Polymorphism; cost; Country; Custom; Data; Disease; Disease Association; Epidemiology; Evaluation; exome; follow-up; Funding; Genetic; genetic association; genetic technology; genetic variant; Genetic Variation; Genome; genome sequencing; genome wide association study; Genomics; Genotype; Goals; Grant; Hereditary Disease; Human; Individual; Institution; Intention; Investments; Knowledge; Learning; Major Depressive Disorder; Maps; Measurable; Mental disorders; Mission; Morbidity - disease rate; mortality; National Institute of Mental Health; next generation; Phenotype; Process; Psychiatry; psychogenetics; Public Health; public-private partnership; Recording of previous events; Reporting; Research; research study; Risk; Role; Sample Size; Sampling; Schizophrenia; Scientist; Sequence Analysis; Societies; Source; success; Symptoms; Technology; Testing; Time; Trust; United States National Institutes of Health; Validation; Variant; Work; working group'}, {'projectNumber': '1R01HG006870-01', 'fy': 2012, 'title': 'MACHINE LEARNING METHODS TO INCREASE GENOMIC ACCESSIBILITY BY NEXT-GEN SEQUENCING', 'department': 'HHS', 'agency': 'NIH', 'ic': 'NHGRI', 'totalCostAmount': 220000, 'nihApplId': 8350385, 'smApplId': 39872, 'budgetStartDate': '2012-08-01T00:00:00', 'budgetEndDate': '2013-06-30T00:00:00', 'contactPi': 'XIE, XIAOHUI', 'otherPis': None, 'congressionalDistrict': '45', 'dunsId': '046705849', 'latitude': 33.678011, 'longitude': -117.773633, 'orgName': 'UNIVERSITY OF CALIFORNIA IRVINE', 'orgCity': 'IRVINE', 'orgState': 'CA', 'orgCountry': 'UNITED STATES', 'orgZipCode': '926173213', 'projectStartDate': '2012-08-01T00:00:00', 'projectEndDate': '2015-06-30T00:00:00', 'cfdaCode': '172', 'abstract': 'DESCRIPTION (provided by applicant): DNA sequencing has become an indispensable tool in many areas of biology and medicine. Recent techno- logical breakthroughs in next-generation sequencing (NGS) have made it possible to sequence billions of bases quickly and cheaply. A number of NGS-based tools have been created, including ChIP-seq, RNA-seq, Methyl- seq and exon/whole-genome sequencing, enabling a fundamentally new way of studying diseases, genomes and epigenomes. The widespread use of NGS-based methods calls for better and more efficient tools for the analysis and interpretation of the NGS high-throughput data. Although a number of computational tools have been devel- oped, they are insufficient in mapping and studying genome features located within repeat, duplicated and other so-called unmappable regions of genomes. In this project, computational algorithms and software that expand genomic accessibility of NGS to these previously understudied regions will be developed. The algorithms will begin with a new way of mapping raw reads from NGS to the reference genome, followed by a machine learning method to resolve ambiguously mapped reads, and will be integrated into a comprehen- sive analysis pipeline for ChIP-seq. More specifically, the three aims of the research are to develop: (1) Data structures and efficient algorithms for read mapping to rapidly identify all mapping locations. Unlike existing methods, the focus of this research is to rapidly identify all candidate locations of each read, instead of one or only a few locations. (2) Machine learning algorithms for read analysis to resolve ambiguously mapped reads for both ChIP-seq analysis and genetic variation detection. This work will develop probabilistic models to resolve ambiguously mapped reads by pooling information from the entire collection of reads. (3) A comprehensive ChIP- seq analysis pipeline to systematically study genomic features located within unmappable regions of genomes. These algorithms will be tested and refined using both publicly available data and data from established wet-lab collaborators. In addition to discovering new genomic features located within repeat, duplicated or other previously unac- cessible regions, this work will provide the NGS community with (a) a faster and more accurate tool for mapping short sequence reads, (b) a general methodology for expanding genomic accessibility of NGS, and (c) a versatile, modular, open-source toolbox of algorithms for NGS data analysis, (d) a comprehensive analysis of protein-DNA interactions in repeat regions in all publicly available ChIP-seq datasets. This work is a close collaboration between computer scientists and web-lab biologists who are developing NGS assays to study biomedical problems. In particular, we will collaborate with Timothy Osborne of Sanford- Burnham Medical Research Institute to study regulators involved in cholesterol and fatty acid metabolism, with Kyoko Yokomori of UC Irvine to study Cohesin, Nipbl and their roles in Cornelia de Lange syndrome, and Ken Cho of UC Irvine to study the roles of FoxH1 and Schnurri in development and growth control.  PUBLIC HEALTH RELEVANCE: DNA-sequencing has become an indispensable tool for basic biomedical research as well as for discovering new treatments and helping biomedical researchers understand disease mechanisms. Next-generation sequencing, which enables rapid generation of billions of bases at relatively low cost, poses a significant computational challenge on how to analyze the large amount of sequence data efficiently and accurately. The goal of this research is to develop open-source software to improve both the efficiency and accuracy of the next-generation sequencing analysis tools, and thereby allowing biomedical researchers to take full advantage of next-generation sequencing to study biology and disease.  ', 'terms': ' Algorithms; Anus; Area; base; Base Sequence; Binding; Biological; Biological Assay; Biology; Biomedical Research; Bruck-de Lange syndrome; ChIP-seq; Cholesterol; Chromatin; cohesin; Collaborations; Collection; Communities; Computational algorithm; Computer software; computerized tools; Computers; cost; Data; Data Analyses; Data Set; Detection; Disease; DNA Sequence; DNA-Protein Interaction; Exons; Facioscapulohumeral; fatty acid metabolism; Foundations; functional genomics; Generations; Genetic Variation; Genome; genome sequencing; genome-wide; Genomics; Goals; Growth and Development function; improved; insertion/deletion mutation; Internet; Location; Machine Learning; Maps; Medical Research; Medicine; Methodology; Methods; Muscular Dystrophies; next generation; novel; open source; Procedures; Publishing; Reading; Research; Research Institute; Research Personnel; RNA; Role; Scientist; Sequence Analysis; Software Engineering; Speed; Statistical Models; Structure; Testing; tool; transcription factor; Uncertainty; Work; xenopus development'}, {'projectNumber': '5F32GM096711-02', 'fy': 2012, 'title': 'NEXT-GENERATION SEQUENCING AS A PLATFORM FOR ACTINOMYCETE NATURAL PRODUCT DISCOVE', 'department': 'HHS', 'agency': 'NIH', 'ic': 'NIGMS', 'totalCostAmount': 8124, 'nihApplId': 8218059, 'smApplId': 40082, 'budgetStartDate': '2012-02-01T00:00:00', 'budgetEndDate': '2012-03-05T00:00:00', 'contactPi': 'BERNHARDT, PETER', 'otherPis': None, 'congressionalDistrict': '49', 'dunsId': '804355790', 'latitude': 37.240049, 'longitude': -119.503236, 'orgName': 'UNIVERSITY OF CALIFORNIA SAN DIEGO', 'orgCity': 'LA JOLLA', 'orgState': 'CA', 'orgCountry': 'UNITED STATES', 'orgZipCode': '920930934', 'projectStartDate': '2011-02-01T00:00:00', 'projectEndDate': '2012-03-05T00:00:00', 'cfdaCode': '859', 'abstract': 'DESCRIPTION (provided by applicant): Actinomycetes have a tremendous potential to produce natural products with medicinal value. The long-term goal of this project is to develop novel agents for the treatment of human disease facilitated by the application of next-generation DNA sequencing to rapidly access actinomycete genetic information. The marine Streptomyces sp. CNH-189 is a producer of structurally fascinating meroterpenoids with potent activity against a broad panel of multi-drug resistant Staphylococcus aureus strains. The overall research objectives are to beta-test Illumina DNA sequencing for rapidly accessing natural product gene cluster sequences in Streptomyces sp. CNH-189, and to use genetics and biochemistry to link meroterpenoid biosynthesis to the identified gene clusters. Preliminary studies include the construction of genomic DNA fragment libraries that are ready for paired-end Illumina sequencing, and the isolation of a fosmid clone that contains a large part of the biosynthetic gene cluster for meroterpenoid biosynthesis. Through a series of experiments, I propose to evaluate how Illumina sequencing can be used to find gene clusters in Streptomyces sp. CNH-189 and other actinomycetes, to obtain a complete coverage of the gene cluster, and to probe the biosynthesis through in vitro biochemistry, gene disruption, and comparative metabolite profiling. The unique chemical structures and biological activities of the natural products isolated from Streptomyces sp. CNH-189 may lead to new chemotherapies and fundamental biochemical knowledge.  PUBLIC HEALTH RELEVANCE: Many pharmaceuticals currently used in the clinic are natural product based. Therefore, natural product discovery, understanding of biosynthetic pathways, and engineering the production of natural product analogs are key for the development of new human chemotherapies. Next-generation DNA sequencing and short read assembly ofers a new and potentialy general platform for accessing gene clusters that could facilitate the genome mining and understanding of clustered biosynthetic pathways in microorganisms. The proposed research will for the first time evaluate Illumina sequencing as a platform for rapidly accessing biosynthetic gene clusters in actinomycetes. Successful completion of this research may advance the field of natural product discovery and biosynthesis.   ', 'terms': ' Actinobacteria class; Anabolism; analog; Antibiotics; base; Biochemical; Biochemistry; Bioinformatics; Biological; Biological Assay; Biological Factors; Cell Culture Techniques; Chemical Structure; chemotherapy; Clinic; comparative; Data Set; Development; DNA; DNA Sequence; Engineering; Enzymes; Event; fascinate; Future; Gene Cluster; Generations; Genes; Genetic; Genome; genome sequencing; Genomic Library; Genomics; Goals; Human; human disease; In Vitro; in vivo; Knock-out; Knowledge; Laboratories; Lead; Libraries; Link; Marines; methicillin resistant Staphylococcus aureus; microorganism; Mining; Molecular Biology; Multi-Drug Resistance; next generation; novel; Pathway interactions; Pharmacologic Substance; Phase; Play; Production; public health relevance; Reading; Research; research study; Role; Screening procedure; Series; Staphylococcus aureus; Streptomyces; Technology; Testing; Time; tool'}, {'projectNumber': '5R01HD068578-02', 'fy': 2012, 'title': 'NON-INVASIVE DETECTION OF FETAL ANEUPLOIDY BY NEXT-GENERATION DNA SEQUENCING', 'department': 'HHS', 'agency': 'NIH', 'ic': 'NICHD', 'totalCostAmount': 458611, 'nihApplId': 8263034, 'smApplId': 60793, 'budgetStartDate': '2012-02-01T00:00:00', 'budgetEndDate': '2013-01-31T00:00:00', 'contactPi': 'PETERS, DAVID GERARD', 'otherPis': None, 'congressionalDistrict': '14', 'dunsId': '119132785', 'latitude': 40.434217, 'longitude': -79.954766, 'orgName': \"MAGEE-WOMEN'S RES INST AND FOUNDATION\", 'orgCity': 'PITTSBURGH', 'orgState': 'PA', 'orgCountry': 'UNITED STATES', 'orgZipCode': '152134430', 'projectStartDate': '2011-05-15T00:00:00', 'projectEndDate': '2016-01-31T00:00:00', 'cfdaCode': '865', 'abstract': 'DESCRIPTION (provided by applicant): Prenatal diagnosis of fetal genetic disease has evolved to reach a prominent position in obstetric clinical care. Established screening methods targeted towards serum proteins are used routinely alongside ultrasonography to identify potentially abnormal pregnancies. Definitive diagnosis is then undertaken using interventional procedures such as amniocentesis and chorionic villus sampling (CVS) that obtain fetal or placental cells, respectively, for karyotype analysis. However, these invasive procedures involve a risk of associated miscarriage. This is significant because, for trisomy 21, current non-invasive first trimester screening methods have detection rates of 82 to 87% and false positive rates of approximately 5%. Therefore, up to 18% of true positives are missed and one expectant mother in every twenty who are screened will undergo an unnecessary invasive diagnostic procedure that could result in the avoidable miscarriage of her baby. In addition to the risk of mortality and morbidity, invasive procedures invoke considerable parental anxiety. Our goal is to dramatically reduce these avoidable miscarriages and other associated risks by developing a diagnostic method that significantly improves the sensitivity and specificity of non-invasive prenatal detection of aneuploidy in the first trimester. To achieve this goal we will expand on our recently published work to test the hypothesis that shotgun next generation sequencing of first trimester maternal plasma DNA provides improved sensitivity and specificity over existing combinations of serum screening and ultrasound. Significantly, earlier economic and logistical barriers preventing the translation of this approach to clinical practice have very recently been overcome by the emergence of methods for high-throughput DNA sequencing that are cost-effective for clinical diagnosis. Specifically, in Aim 1, we will carry out shotgun next generation sequencing on samples of maternal plasma DNA obtained in the first trimester of pregnancy from large cohorts of confirmed aneuploidy and control pregnancies (combined n = 70). We will then undertake a formal statistical analysis to determine the sensitivity and specificity of next-generation DNA sequencing for the detection of aneuploidy on chromosomes 13, 18, 21 and X and compare these results to sensitivity and specificity data obtained using existing first trimester screening methods in the same cohort (Aim 2). Finally we will develop a software package with graphical user interface that can be utilized by non-specialist end users for the rapid analysis of next generation sequencing data and the detection of aneuploidy (Aim 3). We anticipate that this new first trimester test will increase the detection rate of fetal aneuploidy to 95% and reduce the false positive rate to 1%, resulting in an 80% reduction in unnecessary miscarriages associated with invasive prenatal diagnosis after first trimester screening.  PUBLIC HEALTH RELEVANCE: The primary goal of this study is to test the hypothesis that next-generation sequencing of maternal plasma DNA provides significantly improved sensitivity and specificity over existing combinations of serum screening and ultrasound. Specifically, in Aim 1, we will carry out shotgun next generation sequencing on samples of maternal plasma DNA obtained in the first trimester of pregnancy from large cohorts of confirmed aneuploidy and control pregnancies (combined n = 700). We will then undertake a formal statistical analysis to determine the sensitivity and specificity of next-generation DNA sequencing for the detection of aneuploidy on chromosomes 13, 18, 21 and X and compare these results to sensitivity and specificity data obtained using existing first trimester screening methods in the same cohort (Aim 2). Finally we will develop a software package with graphical user interface that can be utilized by non-specialist end users for the rapid analysis of next generation sequencing data and the detection of aneuploidy (Aim 3). We anticipate that this new first trimester test will increase the detection rate of fetal aneuploidy from 82% to 95% and reduce the false positive rate from 5% to 1%, resulting in an 80% reduction in the rate of avoidable fetal miscarriage associated with invasive prenatal diagnosis.   ', 'terms': ' Alleles; Amniocentesis; analytical method; analytical tool; Aneuploidy; Anxiety; Apoptotic; Biological Assay; Blood Circulation; Cells; Chorionic villi; Chorionic Villi Sampling; Chromosomes, Human, Pair 13; clinical care; clinical Diagnosis; clinical practice; cohort; Collaborations; Communities; computer science; Computer software; cost; cost effective; Data; Data Analyses; Detection; Development; Diagnosis; Diagnostic Procedure; Diagnostics Research; Discipline of obstetrics; DNA; DNA Sequence; Down Syndrome; Economics; fetal; Fetal Diseases; Fetus; First Pregnancy Trimester; Genomics; Genotype; Goals; graphical user interface; Hereditary Disease; high throughput analysis; improved; Inherited; Karyotype; Laboratory Technicians; Methods; minimally invasive; Morbidity - disease rate; mortality; Mothers; next generation; Non-Invasive Cancer Detection; novel; Nucleic Acids; open source; Plasma; Positioning Attribute; Pregnancy; Pregnant Women; prenatal; Prenatal Diagnosis; prevent; Principal Investigator; Procedures; programs; public health relevance; Publishing; Research; research study; Risk; RNA; Sampling; Screening procedure; Sensitivity and Specificity; Serum; Serum Proteins; Shotguns; Spontaneous abortion; statistics; Technology; Testing; tool; Translations; Ultrasonography; Work'}, {'projectNumber': '5R01HG005859-02', 'fy': 2012, 'title': 'STATISTICAL METHODS FOR POPULATION GENOMICS AND \"NEXT-GEN\" SEQUENCING DATA', 'department': 'HHS', 'agency': 'NIH', 'ic': 'NHGRI', 'totalCostAmount': 384477, 'nihApplId': 8288682, 'smApplId': 60915, 'budgetStartDate': '2012-06-01T00:00:00', 'budgetEndDate': '2013-05-31T00:00:00', 'contactPi': 'SCHEET, PAUL A', 'otherPis': None, 'congressionalDistrict': '09', 'dunsId': '800772139', 'latitude': 29.706319, 'longitude': -95.397195, 'orgName': 'UNIVERSITY OF TEXAS MD ANDERSON CAN CTR', 'orgCity': 'HOUSTON', 'orgState': 'TX', 'orgCountry': 'UNITED STATES', 'orgZipCode': '770304009', 'projectStartDate': '2011-09-01T00:00:00', 'projectEndDate': '2016-05-31T00:00:00', 'cfdaCode': '172', 'abstract': \"DESCRIPTION (provided by applicant): Massively-parallel ('next-generation') shotgun DNA sequencing projects will provide the highest resolution to date for genetic variation of human populations. This new technology offers great promise for interrogating the genetic etiology of complex disease. However, with this promise come challenges. These new sequencing methods are prone to nontrivial error rates and sparse coverage of mapped reads, confounding polymorphism discovery and genotyping. Copy number variation must often be inferred indirectly. The massive size of these data sets requires rapid and scaleable analytic approaches. In this proposal, we present statistical methods to address these challenges directly, using computationally tractable models for population genetic variation. Our methods take account of the dependence among nearby alleles (linkage disequilibrium) with a clusterbased model for haplotype variation, and utilize this information to aid inferences about the underlying genetic architecture of the samples. Specifically, we propose to call genotypes and detect novel polymorphic loci from next- generation shotgun sequence data, detect rare disease risk alleles for follow-up sequencing studies, and simultaneously model single nucleotide and copy number polymorphism in population data to facilitate studies of association between phenotype and genotype. Our experienced team of medical and statistical geneticists have the technical expertise and access to data sets necessary for achieving these aims. We will implement our methods in our widely-used software package fastPHASE.  \", 'terms': ' Accounting; Address; Affect; Alleles; Architecture; base; Cardiovascular system; Chromosome Deletion; Clinical; Complex; Computer software; Copy Number Polymorphism; Data; Data Set; Databases; density; Dependence; Detection; Disease; disorder risk; DNA; DNA Resequencing; DNA Sequence; Event; exome; experience; follow-up; Frequencies; Genes; Genetic; Genetic Polymorphism; Genetic Predisposition to Disease; Genetic Variation; Genome; genome wide association study; genome-wide; Genomics; Genotype; Germ Lines; Goals; Haplotypes; Health; Human; human disease; Human Genetics; improved; Individual; Joints; Linkage Disequilibrium; Loss of Heterozygosity; lung melanoma; malformation; Malignant neoplasm of lung; Maps; markov model; Medical; Medical center; Medical Genetics; Methods; Modeling; new technology; next generation; novel; Nucleotides; Pattern; Phenotype; Population; Population Genetics; Predisposition; Procedures; Process; Quality Control; Rare Diseases; Reading; Resolution; Resources; Risk; Risk Assessment; Running; Sampling; Sardinia; Shotgun Sequencing; Shotguns; Signal Transduction; Simulate; SNP genotyping; Statistical Methods; Statistical Models; Structure; Surveys; Technical Expertise; Technology; Testing; Texas; trait; Translations; tumor; Variant'}, {'projectNumber': '4R00HG005846-03', 'fy': 2012, 'title': 'HIGH-THROUGHPUT MOBILE ELEMENT GENOTYPING USING NEXT-GENERATION SEQUENCING', 'department': 'HHS', 'agency': 'NIH', 'ic': 'NHGRI', 'totalCostAmount': 249000, 'nihApplId': 8382966, 'smApplId': 66120, 'budgetStartDate': '2012-04-01T00:00:00', 'budgetEndDate': '2013-02-28T00:00:00', 'contactPi': 'XING, JINCHUAN', 'otherPis': None, 'congressionalDistrict': '06', 'dunsId': '001912864', 'latitude': 40.481906, 'longitude': -74.457066, 'orgName': 'RUTGERS THE ST UNIV OF NJ NEW BRUNSWICK', 'orgCity': 'PISCATAWAY', 'orgState': 'NJ', 'orgCountry': 'UNITED STATES', 'orgZipCode': '088543925', 'projectStartDate': '2010-09-26T00:00:00', 'projectEndDate': '2015-02-28T00:00:00', 'cfdaCode': '172', 'abstract': 'Project Summary/Abstract Mobile DNA elements are DNA segments that can mobilize from one genomic location to another,some making copies of themselves during the process. Mobile elements are ubiquitous in eukaryotic genomes,and roughly 50% of the human genome consists of mobile, repetitive DNA sequences such as Alu and LINE1elements. Mobile elements have profoundly influenced primate genomic evolution, and alterations caused bymobile elements have been identified in more than 100 human genetic disorders. In addition, mobile elementsare highly useful as genetic markers in tracing relationships of populations and species because of their uniquemanner of propagating in the genome. Therefore, it is of great interest to understand the biology of mobileelements in human populations. Currently available genotyping methods, however, provide only limited insightinto the distribution of Alu and L1 elements in the human genome. In this proposal, I will first describe a novel, high-throughput method to genotype a large number ofindividuals for Alu insertion polymorphisms. Implementation of this method will permit, for the first time, rapid,inexpensive genotyping of whole families of mobile elements not only in humans but in any organism. Next, Ioutline my plans to construct a computational pipeline that can efficiently handle and analyze the high-throughput genomic data generated by my genotyping method. Finally I describe my plan to apply thegenotyping procedure to 540 HapMap samples and use my computational pipeline to analyze the results.Specifically, I will use the insertion polymorphism data to assess the genomic diversity associated with mobileelement insertions, as well as to answer questions related to mobile element biology and to reconstruct ancienthuman demographic history. In addition, the Alu insertion polymorphism dataset generated in this project willprovide a valuable supplementary resource of structural variants for the HapMap and 1000 Genomes Projects. My long-term goal is to pursue a career in human genomic variation and population genetics researchwith an emphasis on mobile elements. Using a combination of computational and experimental tools, Iwill study the impact of mobile DNA elements on the human genome and on genetic diversity among humanpopulations. This award will provide the necessary training for me to become an independent investigator ontwo fronts: I will acquire essential training in computational biology and bioinformatics to handle large-scalegenomic data, including data generated from next-generation sequencing technology. In addition, researchtraining offered by the institution and my mentor will allow me to acquire knowledge about research resources,grant writing, responsible conduct in research, and other information necessary to become a successful,independent investigator.', 'terms': ' abstracting; Alu Elements; Award; Beryllium; Bioinformatics; Biology; career; Computational Biology; cost; Data; Data Analyses; data management; Data Set; DNA; DNA Insertion Elements; DNA Sequence; Elements; Etiology; Evolution; Family; Genes; Genetic Markers; Genetic Polymorphism; Genetic Research; Genetic Variation; Genome; genome sequencing; Genomics; Genotype; Goals; Grant; Guanine + Cytosine Composition; Hereditary Disease; high throughput technology; Human; human disease; Human Genetics; Human Genome; indexing; Indium; Individual; insight; Institution; interest; Knowledge; L1 Elements; Libraries; Location; Mentors; Methods; next generation; novel; Organism; Phase; Population; Population Genetics; Primates; Procedures; Process; Property; Protocols documentation; Recording of previous events; Research; Research Personnel; Research Training; Resources; Sampling; Scanning; Site; Structure; Techniques; Technology; Time; tool; Training; transposon/insertion element; Variant; Writing'}, {'projectNumber': '5R25HG006110-02', 'fy': 2012, 'title': 'SHORT COURSE ON NEXT-GENERATION SEQUENCING: TECHNOLOGY AND STATISTICAL METHODS', 'department': 'HHS', 'agency': 'NIH', 'ic': 'NHGRI', 'totalCostAmount': 52471, 'nihApplId': 8260817, 'smApplId': 68734, 'budgetStartDate': '2012-02-01T00:00:00', 'budgetEndDate': '2013-01-31T00:00:00', 'contactPi': 'TIWARI, HEMANT K.', 'otherPis': None, 'congressionalDistrict': '07', 'dunsId': '063690705', 'latitude': 32.635016, 'longitude': -86.809379, 'orgName': 'UNIVERSITY OF ALABAMA AT BIRMINGHAM', 'orgCity': 'BIRMINGHAM', 'orgState': 'AL', 'orgCountry': 'UNITED STATES', 'orgZipCode': '352940001', 'projectStartDate': '2011-04-27T00:00:00', 'projectEndDate': '2014-01-31T00:00:00', 'cfdaCode': '172', 'abstract': \"DESCRIPTION (provided by applicant): Major efforts are under way to uncover the specific genetic components of many complex human disorders and quantitative traits. These efforts include rapidly evolving technologies for next- generation sequencing that will generate an unprecedented wealth of data. In that turn that will require sophisticated statistical analysis in order to maximize the information gleaned from each study. Likewise, many advances have been made in statistical methods for the study of complex genetic traits, and new statistical methods continue to be promulgated. To expedite and significantly advance the search for specific genes that predispose to these complex traits, we propose a series of annual short courses on next-generation sequencing technologies and statistical genomics which will enable a far greater number of researchers, including clinical researchers, to participate in, contribute to, and lead such research. These short courses will help equip students, post-doctoral fellows, junior investigators and senior investigators to expedite genomic discovery. The courses will be taught by leading experts in next-generation sequencing and statistical genetics/genomics. Each 4-day course will provide substantial expertise both in technologies for next- generation sequencing and in statistical methods pertaining to these technologies that will effectively increase the number and the expertise of investigators who are pursuing genetic and genomic research. This is a unique course containing instruction in both technological and statistical aspects of next-generation sequencing. There is currently no course of this type offered in US. We expect approximately 50 students to attend each course, allowing for intensive interaction between students and faculty. Lectures will be supplemented with extensive discussion sessions, handouts, and presentation of worked examples.  PUBLIC HEALTH RELEVANCE: We propose an annual short course each year for 3 years on next-generation sequencing and analysis methods for investigators studying complex traits. Taught by leading experts in next-generation sequencing technology and statistical genomics, these courses will be aimed at statisticians and non- statistician investigators who wish to learn both the rapidly evolving technology and 'language' of statistical genomics. Attendees should be either current investigators or in the early stages of a career in genetic/genomic research.  \", 'terms': ' Alabama; Applications Grants; Bioinformatics; Biotechnology; career; Characteristics; Clinical; Communities; Complex; Complex Genetic Trait; Consultations; Data; Data Analyses; Development; disability; Disease; Education; education evaluation; Educational process of instructing; Evaluation; experience; Faculty; Fees; Funding; Genetic; Genomics; Glean; graduate student; Human; Institutes; Instruction; Laboratories; Language; Lead; Learning; lectures; Location; meetings; Methods; Modification; Molecular; next generation; Participant; Postdoctoral Fellow; programs; public health relevance; Request for Proposals; Research; Research Personnel; Resources; responsible research conduct; Selection Criteria; Sequence Analysis; Series; Staging; Statistical Methods; Students; Susceptibility Gene; Technology; Training; trait; Universities; web site; Work'}, {'projectNumber': '5R25HG006243-02', 'fy': 2012, 'title': 'ANALYZING NEXT-GENERATION SEQUENCING DATA', 'department': 'HHS', 'agency': 'NIH', 'ic': 'NHGRI', 'totalCostAmount': 53653, 'nihApplId': 8323321, 'smApplId': 68735, 'budgetStartDate': '2012-07-01T00:00:00', 'budgetEndDate': '2013-08-31T00:00:00', 'contactPi': 'BROWN, C. TITUS', 'otherPis': None, 'congressionalDistrict': '08', 'dunsId': '193247145', 'latitude': 42.748478, 'longitude': -84.483276, 'orgName': 'MICHIGAN STATE UNIVERSITY', 'orgCity': 'EAST LANSING', 'orgState': 'MI', 'orgCountry': 'UNITED STATES', 'orgZipCode': '488242600', 'projectStartDate': '2011-08-23T00:00:00', 'projectEndDate': '2013-08-31T00:00:00', 'cfdaCode': '172', 'abstract': \"DESCRIPTION (provided by applicant): Modern biomedical research is increasingly making use of genome-scale from next-generation sequencing platforms, including Roche 454, Illumina GA2, and ABI SOLiD. These platforms make it possible for individual labs to quickly and cheaply generate vast amounts of genomic and transcriptomic data from de novo sequencing, resequencing, ChIP-seq, mRNA-seq, and allelotyping experiments. Despite this ability to generate large data sets, biomedical researchers are rarely trained in the computational and statistical techniques necessary to make sense of this data. Thus, many researchers must rely on others - often computational scientists with little biological training - to design and implement appropriate data reduction and data mining techniques. Moreover, most institutions do not have access to computational resources necessary to run these analyses. Our specific aims are to help bridge this gap in a short, two-week course, by teaching biomedical researchers to (1) run analyses on remote UNIX servers hosted in the Amazon Web Services 'cloud'; (2) perform mapping and assembly on large short-read data sets; (3) tackle specific biological problems with existing short-read data; and (4) design computational pipelines capable of addressing their own research questions. All specific aims will be accompanied by in-depth hands-on practical training in the relevant techniques. Our experience is that this practical training leads to a substantial improvement in the basic computational sophistication of participants. This short course will help train the current and next generation of independent biomedical re- searchers in basic computational thinking and procedure, as well as teaching them how to make use of scalable Internet computing resources for their own research. Our end goal is increase the efficiency and sophistication with which biomedical researchers make use of novel sequencing technologies.  \", 'terms': ' Address; Allelotyping; Automation; base; Bioinformatics; Biological; Biomedical Research; ChIP-seq; Computer software; computerized tools; computing resources; Data; Data Analyses; data mining; data reduction; Data Set; design; DNA Resequencing; Educational process of instructing; experience; Experimental Designs; Future; Gene Expression Profile; Genome; genome sequencing; Genomics; Goals; Graph; Individual; Institution; Internet; Learning; Maps; Memory; Messenger RNA; next generation; novel; Operating System; Participant; Paste substance; Procedures; programs; Reading; Recipe; Research; Research Personnel; research study; Running; Sampling; Scientist; Sequence Analysis; skills; Source; Students; System; Techniques; Technology; Thinking; tool; Training; transcriptomics; web services; Work'}, {'projectNumber': '1222718', 'fy': 2012, 'title': 'COLLABORATIVE RESEARCH: ATD: INTEGRATED STATISTICAL ALGORITHMS WITH ULTRA-HIGH PERFORMANCE COMPUTING FOR DISCOVERING SNPS FROM MASSIVE NEXT-GENERATION METAGENOMIC SEQUENCING DATA', 'department': 'NSF', 'agency': 'NSF', 'ic': None, 'totalCostAmount': 372098, 'nihApplId': None, 'smApplId': 79028, 'budgetStartDate': None, 'budgetEndDate': None, 'contactPi': 'MA, PING', 'otherPis': None, 'congressionalDistrict': '15', 'dunsId': '041544081', 'latitude': 40.092617, 'longitude': -88.247723, 'orgName': 'UNIVERSITY OF ILLINOIS URBANA-CHAMPAIGN', 'orgCity': 'CHAMPAIGN', 'orgState': 'IL', 'orgCountry': 'UNITED STATES', 'orgZipCode': '61820-7473', 'projectStartDate': '2012-08-15T00:00:00', 'projectEndDate': '2016-07-31T00:00:00', 'cfdaCode': '47.049', 'abstract': 'Recently, the emerging new field of metagenomics facilitated by the advent of next-generation sequencing technology enables genome sequencing of unculturable and often unknown microbes in natural environments, offering researchers an unprecedented opportunity to delineate bio-diversity of any microbial organism. Mining single nucleotide polymorphisms (SNPs) from metagenomic sequencing data offers an unique opportunity to rapidly and accurately detect known or novel strains related to multiple biothreat agents. While the sequencing technologies are evolving at unprecedented speed, researchers engaged in this enterprise are facing major computational, algorithmic and statistical challenges in the analysis of the massive metagenomic data. It is clear that both current analytical and computational methods are inadequate for this challenge. In this project, the investigator and his colleagues develop a family of statistically sound and computationally efficient algorithms to detect SNPs from metagenomic data to characterize microbial diversity in natural environments. <br/><br/>The proposed project provides the national security and biodefense agencies new tools for rapid and accurate detection of biothreat agents. It also provides researchers in microbiology with new tools for producing abundant, high throughput SNPs for detailed analysis of the genetic basis of microbial diversity and evolution. Since this informatics tool can be used to study a wide variety of microbial communities, it helps accelerating scientific advancements of our knowledge in microbiology and evolution. The multidisciplinary nature of the project will promote collaboration between biologists, computer scientists and statistician. The multidisciplinary nature of the project will also provide postdoctoral fellows and graduate students training in statistics, genomics and scientific computing through hands-on experience.', 'terms': ' Algorithms; analytical method; base; biodefense; biothreat; Collaborations; Computers; Computing Methodologies; Data; Detection; Environment; Evolution; experience; Family; genetic analysis; genome sequencing; Genomics; graduate student; Informatics; Knowledge; metagenomic sequencing; Metagenomics; Microbe; microbial; microbial community; Microbiology; Mining; multidisciplinary; National Security; Nature; next generation; novel; Organism; Postdoctoral Fellow; Research Personnel; scientific computing; Scientist; Single Nucleotide Polymorphism; sound; statistics; Technology; tool; Training'}, {'projectNumber': '1223107', 'fy': 2012, 'title': 'COLLABORATIVE RESEARCH: ATD: INTEGRATED STATISTICAL ALGORITHMS WITH ULTRA-HIGH PERFORMANCE COMPUTING FOR DISCOVERING SNPS FROM MASSIVE NEXT-GENERATION METAGENOMIC SEQUENCING DATA', 'department': 'NSF', 'agency': 'NSF', 'ic': None, 'totalCostAmount': 532114, 'nihApplId': None, 'smApplId': 79048, 'budgetStartDate': None, 'budgetEndDate': None, 'contactPi': 'CUI, XINPING', 'otherPis': None, 'congressionalDistrict': '44', 'dunsId': '627797426', 'latitude': 33.968143, 'longitude': -117.3358533, 'orgName': 'UNIVERSITY OF CALIFORNIA RIVERSIDE', 'orgCity': 'RIVERSIDE', 'orgState': 'CA', 'orgCountry': 'UNITED STATES', 'orgZipCode': '92521-1000', 'projectStartDate': '2012-08-15T00:00:00', 'projectEndDate': '2016-07-31T00:00:00', 'cfdaCode': '47.049', 'abstract': 'Recently, the emerging new field of metagenomics facilitated by the advent of next-generation sequencing technology enables genome sequencing of unculturable and often unknown microbes in natural environments, offering researchers an unprecedented opportunity to delineate bio-diversity of any microbial organism. Mining single nucleotide polymorphisms (SNPs) from metagenomic sequencing data offers an unique opportunity to rapidly and accurately detect known or novel strains related to multiple biothreat agents. While the sequencing technologies are evolving at unprecedented speed, researchers engaged in this enterprise are facing major computational, algorithmic and statistical challenges in the analysis of the massive metagenomic data. It is clear that both current analytical and computational methods are inadequate for this challenge. In this project, the investigator and his colleagues develop a family of statistically sound and computationally efficient algorithms to detect SNPs from metagenomic data to characterize microbial diversity in natural environments. <br/><br/>The proposed project provides the national security and biodefense agencies new tools for rapid and accurate detection of biothreat agents. It also provides researchers in microbiology with new tools for producing abundant, high throughput SNPs for detailed analysis of the genetic basis of microbial diversity and evolution. Since this informatics tool can be used to study a wide variety of microbial communities, it helps accelerating scientific advancements of our knowledge in microbiology and evolution. The multidisciplinary nature of the project will promote collaboration between biologists, computer scientists and statistician. The multidisciplinary nature of the project will also provide postdoctoral fellows and graduate students training in statistics, genomics and scientific computing through hands-on experience.', 'terms': ' Algorithms; analytical method; base; biodefense; biothreat; Collaborations; Computers; Computing Methodologies; Data; Detection; Environment; Evolution; experience; Family; genetic analysis; genome sequencing; Genomics; graduate student; Informatics; Knowledge; metagenomic sequencing; Metagenomics; Microbe; microbial; microbial community; Microbiology; Mining; multidisciplinary; National Security; Nature; next generation; novel; Organism; Postdoctoral Fellow; Research Personnel; scientific computing; Scientist; Single Nucleotide Polymorphism; sound; statistics; Technology; tool; Training'}, {'projectNumber': '7R01HG005692-03', 'fy': 2012, 'title': 'STATISTICAL TOOLS AND METHODS FOR NEXT-GENERATION SEQUENCING IN EPIGENETICS', 'department': 'HHS', 'agency': 'NIH', 'ic': 'NHGRI', 'totalCostAmount': 336105, 'nihApplId': 8235963, 'smApplId': 80190, 'budgetStartDate': '2012-03-01T00:00:00', 'budgetEndDate': '2013-02-28T00:00:00', 'contactPi': 'JOHNSON, WILLIAM EVAN', 'otherPis': None, 'congressionalDistrict': '07', 'dunsId': '604483045', 'latitude': 42.33625, 'longitude': -71.070668, 'orgName': 'BOSTON UNIVERSITY MEDICAL CAMPUS', 'orgCity': 'BOSTON', 'orgState': 'MA', 'orgCountry': 'UNITED STATES', 'orgZipCode': '021182340', 'projectStartDate': '2010-06-01T00:00:00', 'projectEndDate': '2015-02-28T00:00:00', 'cfdaCode': '172', 'abstract': 'DESCRIPTION (provided by applicant): Next-generation sequencing technologies are capable of producing tens of millions of sequence reads during each instrument run, and are quickly being applied in diverse types of experiments (e.g. RNA-Seq, miRNA-Seq, ChIP-Seq, BS-seq, CNV-Seq) to address biomedical questions by cost-effectively generating genome-wide datasets. While sequencing has been promoted as overcoming longstanding limitations of microarray-based studies, its data files are much larger than for microarrays, and its diverse data types raise similar as well as novel statistical and computational challenges. There is a pressing need for statistical and computational tools to address what leaders in the field have stated are the largest problems: data analysis and data integration. We propose to develop a comprehensive and coordinated set of statistical methods for high throughput sequencing (HTS) that directly address many important data analysis problems in epigenomics. Specifically we plan to address the following computational and statistical challenges facing researchers conducting HTS experiments: 1) develop sensitive statistical methods for the analysis of ChIP-seq data both for single- and paired-end-tag runs, particularly the focusing on applications in genome-wide profiling of nucleosome positions. 2) develop statistical methods for the analysis of BS-seq data, producing base-level DNA methylation profiles. 3) develop new statistical tools and methods for data integration in order to gain new biological insights about global transcription and regulation. We also plan to apply these approaches to a variety of high throughput sequencing data sets to demonstrate the relevance and utility of our methods. We plan to work with stimulated STAT1 and STAT3 data, and data from the ETS transcription factor family and its cofactors, for which we have already gathered significant data through our collaborations, including transcription factors, histone marks, DNAse I hypersensitivity and gene expression.  PUBLIC HEALTH RELEVANCE: We propose to develop a comprehensive and coordinated set of statistical methods for high throughput sequencing (HTS) that directly address many important data analysis problems in epigenomics. In particular, we plan to integrate data from multiple sources including expression, transcription factor binding, nucleosome positioning, histone marks and DNA methylation to better understand the mechanisms that regulate the behavior of a cell. Much of our proposal involves not just the development of new statistical and computational methods, but also the design, implementation and delivery of software tools that support these ideas. The many useful applications of next-generation sequencing with assure that or well- developed methods will have a broad impact in molecular biology, specifically in transcription regulation, chromatin dynamics, development, and cancer.     ', 'terms': ' Address; Algorithms; base; Behavior; Binding; Biological; bisulfite; Cells; ChIP-seq; Chromatin; cofactor; Collaborations; Computer software; computerized tools; Computing Methodologies; cost; Cytosine; Data; Data Analyses; Data Files; data integration; Data Set; design; Development; DNA; DNA Methylation; Ensure; Epigenetic Process; epigenomics; Family; Future; Gene Expression; Genetic Transcription; genome-wide; Genomics; Goals; histone modification; Histones; Hypersensitivity; insight; Institutes; instrument; Linker DNA; Malignant Neoplasms; Methods; MicroRNAs; Modeling; Molecular Biology; Names; National Human Genome Research Institute; next generation; novel; Nucleosomes; Positioning Attribute; Process; programs; public health relevance; Reading; Research Personnel; research study; Resolution; RNA; Running; Sequence Analysis; Software Tools; Source; STAT1 gene; STAT3 gene; Statistical Methods; Technology; Thymine; tool; transcription factor; Transcriptional Regulation; United States National Institutes of Health; Variant; Work'}, {'projectNumber': '1R44HG006578-01', 'fy': 2012, 'title': 'ASSOCIATION ANALYSIS SOFTWARE FOR MINING CLINICAL NEXT-GEN SEQUENCING DATA', 'department': 'HHS', 'agency': 'NIH', 'ic': 'NHGRI', 'totalCostAmount': 150000, 'nihApplId': 8236680, 'smApplId': 82415, 'budgetStartDate': '2012-09-25T00:00:00', 'budgetEndDate': '2013-06-24T00:00:00', 'contactPi': 'DURFEE, TIMOTHY J', 'otherPis': None, 'congressionalDistrict': '02', 'dunsId': '130194947', 'latitude': 43.067896, 'longitude': -89.438617, 'orgName': 'DNASTAR INC', 'orgCity': 'MADISON', 'orgState': 'WI', 'orgCountry': 'UNITED STATES', 'orgZipCode': '537055202', 'projectStartDate': '2012-09-25T00:00:00', 'projectEndDate': '2013-06-24T00:00:00', 'cfdaCode': '172', 'abstract': \"DESCRIPTION (provided by applicant): Remarkable improvements in throughput, accuracy and cost-effectiveness of next-generation sequencing (next-gen) technologies are ushering in a new era of clinical medicine. Genome wide association studies (GWAS) in particular have begun to leverage these advances to determine the complete catalog of common and rare variants for each member of a cohort. The resolving power of this approach has the potential to greatly accelerate our understanding, diagnosis and treatment of human disease. Unfortunately, analysis of these massive data sets requires that several disparate pieces of software be cobbled together including a large capacity next-gen sequencing assembler, variation detection modules, mapping and comparison tools for tens to hundreds of variant reports, statistical analysis packages, reporting tools, and so on. Combining and using these tools typically requires extensive bioinformatic expertise as the software is rarely well documented or supported and often depends on having elaborate hardware. These hurdles makes next-gen based GWAS inaccessible to the vast majority of the crucial user base, the physician researchers. The goal of this proposal is to assemble the essential next-gen based GWAS software components into a single coherent pipeline that that is fully equipped to meet the needs of the medical research community. Consistent with DNASTAR's 28 year tradition, the software will be easy to use, run on a reasonably priced (<$3000) desktop computer, and will be fully documented and supported. The pipeline will consist of two modules already available through DNASTAR, SeqMan NGen 3.0 (SM NGen 3.0) and ArrayStar. SM NGen 3.0, our recently released human genome scale assembly and analysis package, forms the front end of pipeline. Reference-guided assemblies of whole human genome or exome next-gen data sets produce variation reports including impact on gene features and associations with the dbSNP database. Putative variations can be verified by direct inspection of the alignment through the SeqMan Pro component of the package. Variation reports from each member of a GWAS cohort will then be fed into our multi-sample comparison and analysis program, ArrayStar, at the back end of the pipeline. ArrayStar has the infrastructure for multi-sample management and processing which can be easily adapted to GWAS analysis. These adaptations and their documentation are a central focus of this application. Critical to the successful development of this software is our collaboration with Dr. Douglas McNeel (Dept. of Oncology, UW-Madison). The exomes from a panel of prostate cancer vaccine recipients, including responders and non-responders, from the McNeel lab will be sequenced as input from which to build the pipeline using iterative cycles of development followed by evaluation by the McNeel group. This relationship offers an ideal opportunity to build the analysis and reporting software needed by physician researchers to form, test and validate GWAS generated hypotheses.  PUBLIC HEALTH RELEVANCE: The easy to use tools to be developed and integrated in this project will dramatically enhance the efficiency of clinical and diagnostic research for a wide range of life scientists and medical professionals using next-generation DNA sequencing technologies, allowing new treatments to be brought to market sooner, enhancing scientists' understanding of treatment efficacy, and supporting the tailoring of different treatments to specific groups of individuals based on their genetic composition. These tools will be flexible enough to support critical analysis of large populations for clinical research and easy enough to use for all life scientists and medical professionals to feel comfortable with them.   \", 'terms': ' Automatic Data Processing; Back; base; Bioinformatics; Cataloging; Catalogs; Cereals; Clinical; Clinical Medicine; Clinical Research; clinically relevant; cohort; Collaborations; Communities; Computer software; Computers; cost effectiveness; Data; Data Set; Databases; Detection; Development; Diagnosis; Diagnostics Research; Disease; DNA Sequence; Documentation; Ensure; Evaluation; exome; feeding; flexibility; Foundations; Generations; Genes; Genetic; Genome; genome wide association study; genome-wide; Goals; Hour; Housing; human disease; Human Genome; Individual; insight; Laboratories; Life; Maps; Marketing; Medical; Medical Research; meetings; member; Metadata; Mining; Modification; next generation; oncology; Performance; Phase; Physicians; Population; Price; Process; programs; Prostate Cancer Vaccine; Reporting; Research Infrastructure; Research Personnel; response; Running; Sample Size; Sampling; Scientist; Sequence Analysis; software development; Solid; Solutions; Suggestion; Technology; Testing; tool; Treatment Efficacy; Vaccines; Variant'}, {'projectNumber': '1F32HG004825-01', 'fy': 2008, 'title': 'INTERROGATING GENETIC INTERACTION NETWORKS VIA NEXT-GENERATION SEQUENCING', 'department': 'HHS', 'agency': 'NIH', 'ic': 'NHGRI', 'totalCostAmount': 46826, 'nihApplId': 7545124, 'smApplId': 203847, 'budgetStartDate': '2008-12-01T00:00:00', 'budgetEndDate': '2009-11-30T00:00:00', 'contactPi': 'MELLOR, JOSEPH C', 'otherPis': None, 'congressionalDistrict': '07', 'dunsId': '047006379', 'latitude': 42.335814, 'longitude': -71.105033, 'orgName': 'HARVARD UNIVERSITY', 'orgCity': 'BOSTON', 'orgState': 'MA', 'orgCountry': 'UNITED STATES', 'orgZipCode': '021156027', 'projectStartDate': '2008-12-01T00:00:00', 'projectEndDate': '2010-11-30T00:00:00', 'cfdaCode': '172', 'abstract': \"  DESCRIPTION (provided by applicant): When two genes are perturbed simultaneously, a surprising phenotype often emerges. Genetic interaction-defined by this phenomenon-suggests that the interacting genes have related functions. Genetic interactions have shaped our understanding of nearly all known biological pathways. Examples of genetic interaction encompass complex human diseases such as cancer that require multiple mutations. The multigenic origins of such diseases motivate the need to discover and understand how genetic interactions yield extreme phenotypes. Due to its facile genetics, S. cerevisiae has been a key model organism for the systematic study of genetic interactions. Technology that can more efficiently map genetic interactions would accelerate completion of the current interaction mapping effort to encompass all -18 million gene pairs in yeast. Moreover, genetic interactions are strongly influenced by or dependent on environment. Recent advances in next-generation DNA sequencing technology have reduced the cost of sequencing short sequence tags by at least a factor of 1000 relative to conventional Sanger dideoxy sequencing coupled with capillary electrophoresis. Future improvements in scale and cost are imminently expected over the next several years. Here I propose to develop and apply a new approach termed 'barcode fusion genetics' BFG) on a pilot scale to study factors influencing DNA repair in yeast. This project would demonstrate :feasibility for a technology that could produce a global genetic interaction map in a particular environmental condition by a single technician within a year. Effects on DNA maintenance and cellular response by endogenous and exogenous mutagenesis is both of basic science interest and is also critical to our understanding of cancer etiology in humans. There are well over a hundred genes in yeast that encode different parts of the DNA repair process, including protein complexes for recognizing different forms of DNA breaks and lesions, signaling proteins for activating specific repair pathways, and enzymes for catalyzing repair operations. Applied to the unraveling of genetic interactions among DNA repair genes, the proposal outlined here will significantly enhance our understanding of this crucial biological function.        \", 'terms': ' Animal Model; Bar Codes; Basic Science; Biological; Biological Process; Cancer Etiology; Capillary Electrophoresis; Chromosome Mapping; Classification; Complex; Condition; cost; Coupled; Disease; DNA; DNA Maintenance; DNA Repair; DNA Repair Gene; DNA Sequence; Environment; enzyme pathway; Future; Genes; Genetic; genetic technology; Growth; Human; human disease; interest; Lesion; Malignant Neoplasms; Maps; Measurement; Methods; Mutagenesis; Mutation; next generation; novel; novel strategies; Operative Surgical Procedures; Pathway interactions; Phenotype; Process; Proteins; Rate; Relative (related person); repaired; response; Saccharomyces cerevisiae; Shapes; Signaling Protein; Technology; Yeasts'}, {'projectNumber': '5F32HG004825-02', 'fy': 2010, 'title': 'INTERROGATING GENETIC INTERACTION NETWORKS VIA NEXT-GENERATION SEQUENCING', 'department': 'HHS', 'agency': 'NIH', 'ic': 'NHGRI', 'totalCostAmount': 50474, 'nihApplId': 7742679, 'smApplId': 279498, 'budgetStartDate': '2009-12-01T00:00:00', 'budgetEndDate': '2010-11-30T00:00:00', 'contactPi': 'MELLOR, JOSEPH C', 'otherPis': None, 'congressionalDistrict': '07', 'dunsId': '047006379', 'latitude': 42.335814, 'longitude': -71.105033, 'orgName': 'HARVARD UNIVERSITY', 'orgCity': 'BOSTON', 'orgState': 'MA', 'orgCountry': 'UNITED STATES', 'orgZipCode': '021156027', 'projectStartDate': '2008-12-01T00:00:00', 'projectEndDate': '2010-11-30T00:00:00', 'cfdaCode': '172', 'abstract': \"  DESCRIPTION (provided by applicant): When two genes are perturbed simultaneously, a surprising phenotype often emerges. Genetic interaction-defined by this phenomenon-suggests that the interacting genes have related functions. Genetic interactions have shaped our understanding of nearly all known biological pathways. Examples of genetic interaction encompass complex human diseases such as cancer that require multiple mutations. The multigenic origins of such diseases motivate the need to discover and understand how genetic interactions yield extreme phenotypes. Due to its facile genetics, S. cerevisiae has been a key model organism for the systematic study of genetic interactions. Technology that can more efficiently map genetic interactions would accelerate completion of the current interaction mapping effort to encompass all -18 million gene pairs in yeast. Moreover, genetic interactions are strongly influenced by or dependent on environment. Recent advances in next-generation DNA sequencing technology have reduced the cost of sequencing short sequence tags by at least a factor of 1000 relative to conventional Sanger dideoxy sequencing coupled with capillary electrophoresis. Future improvements in scale and cost are imminently expected over the next several years. Here I propose to develop and apply a new approach termed 'barcode fusion genetics' BFG) on a pilot scale to study factors influencing DNA repair in yeast. This project would demonstrate :feasibility for a technology that could produce a global genetic interaction map in a particular environmental condition by a single technician within a year. Effects on DNA maintenance and cellular response by endogenous and exogenous mutagenesis is both of basic science interest and is also critical to our understanding of cancer etiology in humans. There are well over a hundred genes in yeast that encode different parts of the DNA repair process, including protein complexes for recognizing different forms of DNA breaks and lesions, signaling proteins for activating specific repair pathways, and enzymes for catalyzing repair operations. Applied to the unraveling of genetic interactions among DNA repair genes, the proposal outlined here will significantly enhance our understanding of this crucial biological function.        \", 'terms': ' Animal Model; Bar Codes; Basic Science; Biological; Biological Process; Cancer Etiology; Capillary Electrophoresis; Chromosome Mapping; Complex; cost; Coupled; Disease; DNA; DNA Maintenance; DNA Repair; DNA Repair Gene; DNA Sequence; Environment; enzyme pathway; Future; Genes; Genetic; genetic technology; Growth; Human; human disease; interest; Lesion; Malignant Neoplasms; Maps; Measurement; Methods; Mutagenesis; Mutation; next generation; novel; novel strategies; operation; Pathway interactions; Phenotype; Process; protein complex; Relative (related person); repaired; response; Saccharomyces cerevisiae; Shapes; Signaling Protein; Technology; Yeasts'}, {'projectNumber': '1S10RR026342-01', 'fy': 2010, 'title': 'BIOMEDICAL GENOMICS CENTER:  NEXT-GENERATION SEQUENCING INSTRUMENTATION', 'department': 'HHS', 'agency': 'NIH', 'ic': 'NCRR', 'totalCostAmount': 500000, 'nihApplId': 7790889, 'smApplId': 293960, 'budgetStartDate': '2010-03-11T00:00:00', 'budgetEndDate': '2011-03-10T00:00:00', 'contactPi': 'BECKMAN, KENNETH B', 'otherPis': None, 'congressionalDistrict': '05', 'dunsId': '555917996', 'latitude': 44.974119, 'longitude': -93.227014, 'orgName': 'UNIVERSITY OF MINNESOTA TWIN CITIES', 'orgCity': 'MINNEAPOLIS', 'orgState': 'MN', 'orgCountry': 'UNITED STATES', 'orgZipCode': '554552070', 'projectStartDate': '2010-03-11T00:00:00', 'projectEndDate': '2011-03-10T00:00:00', 'cfdaCode': '701', 'abstract': \"  DESCRIPTION (provided by applicant): Biological research is at the cusp of a technological revolution of the type that has repeatedly transformed research in recent decades. With the arrival of 'next-generation' DNA sequencers, individual laboratories are able to ask questions that just five years ago required the resources of an entire institution. The transformational aspect of the new sequencers stems not only from their higher overall throughput in terms of bases per run, but also from their ability to permit digital DNA 'counting' of DNA tags. Indeed, the newest of the next-gen sequencers count hundreds of millions of DNA tags per run, which in combination with up-stream sequence selection and preparation protocols, are poised to transform measurements of gene expression, regulation, and epigenetics. This Shared Instrumentation Grant requests such a next-generation sequencer-the Illumina GA IIx.  Scientists at the UMN have imagined dozens of interesting applications for next-generation sequencing, but currently, they are hampered by our lack of a truly high-powered sequencer. They would like to use next-gen counting of transcripts to study control of cancer cells and to discover regulatory RNAs; they would like to carry out targeted re-sequencing of patient DNA to discover genetic variants responsible for differences in chemotherapy response; they would like to use sequence-based characterization of insertional mutagenesis to study carcinogenesis and to uncover mutations controlling the fundamental basis of meiosis. All of these questions are made possible once one can read, at a reasonable cost, and in a reasonable time frame, tens of billions of bases of DNA. Without this capability, however, interesting questions remain unanswered, and important hypotheses untested.  The UMN's Biomedical Genomics Center (BMGC) currently provides genomic services at the UMN, and the BMGC proposes to put the GA IIx into shared use for the entire university. Our twelve scientists have the expertise, facilities, and other resources required to make best use of a GA IIx. Although we presently operate an early model of next-generation sequencer in the BMGC, in the critical metric of number of sequencing reads per run and cost per base, this now two-year old instrument is not a viable option. In short, the GAIIx will enable inquiries that are simply not achievable at present, and will therefore help UMN researchers move science closer to cures for important human diseases.        \", 'terms': ' 2 year old; base; Base Sequence; biological research; cancer cell; carcinogenesis; chemotherapy; Controlled Study; cost; digital; DNA; Epigenetic Process; Gene Expression Regulation; genetic variant; Genomics; Grant; human disease; Individual; Insertional Mutagenesis; Institution; instrument; instrumentation; interest; Laboratories; Measurement; Meiosis; Metric; Modeling; Mutation; next generation; Patients; Preparation; Protocols documentation; Reading; Research; Research Personnel; Resources; response; Running; Science; Scientist; Services; stem; Stream; Time; Transcript; Universities'}, {'projectNumber': '1S10RR027754-01', 'fy': 2010, 'title': 'UNMC SIG SUPPORTING NEXT-GENERATION DNA SEQUENCING TECHNOLOGY', 'department': 'HHS', 'agency': 'NIH', 'ic': 'NCRR', 'totalCostAmount': 500000, 'nihApplId': 7795514, 'smApplId': 295629, 'budgetStartDate': '2010-04-15T00:00:00', 'budgetEndDate': '2011-04-14T00:00:00', 'contactPi': 'EUDY, JAMES', 'otherPis': None, 'congressionalDistrict': '02', 'dunsId': '168559177', 'latitude': 41.265884, 'longitude': -96.053824, 'orgName': 'UNIVERSITY OF NEBRASKA MEDICAL CENTER', 'orgCity': 'OMAHA', 'orgState': 'NE', 'orgCountry': 'UNITED STATES', 'orgZipCode': '681987835', 'projectStartDate': '2010-04-15T00:00:00', 'projectEndDate': '2011-04-14T00:00:00', 'cfdaCode': '701', 'abstract': '  DESCRIPTION (provided by applicant):   Project Summary/Abstract The University of Nebraska Medical Center (UNMC) is requesting funds to purchase an Illumina Genome Analyzer IIX (GAIIX) to provide state-of-the-art genomic services to the UNMC biomedical research community. The GAIIX instrument will be housed and operated as a core facility by well-trained and highly capable scientists and technical personnel. This placement and management strategy will ensure the efficient and maximal use of the instrument, as well as ensure that the researchers needing the instrument and associated technological applications have the support they need in a timely fashion. The GAIIX is a next- generation DNA sequencing instrument capable of generating gigabase (Gb) quantities of DNA nucleotide information from a variety of different experimental contexts. The instrument will facilitate genomic research involving DNA sequencing to investigate genetic variation, genome-wide gene regulatory processes to include protein / DNA interactions as well as epigenetic mechanisms, and a variety of whole transcriptome applications to include mRNA and miRNA sequencing and digital gene expression-tag profiling. This technology is currently unavailable on the UNMC campus. The NIH supported projects outlined in this proposal include molecular studies of biomedical relevance to breast cancer, HIV disease, lymphoma, methamphetamine abuse, infectious disease, neurological disorders and neurodegeneration, neurosensory systems of vision and hearing, disorders of language and learning, and respiratory disorders. The nine NIH supported major users come from a variety of academic departments, institutes, and centers including the Eppley Cancer Institute, an NCI designated cancer center since 1982, the Center for Lymphoma and Leukemia Research, and the Center for Neurovirology and Neurodegenerative Diseases. There is considerable interest in next-generation sequencing applications in the UNMC biomedical research community as evidenced by the nine major user and eleven minor user projects outlined in this proposal, as well as many other researchers here at UNMC that have also expressed an interest in using the technology. The local availability of the Ilumina GAIIX instrument and next-generation sequencing services will have a major impact on a number of different research programs at UNMC and the local biomedical research community as a whole, thus enhancing research and facilitating the ability of researchers to compete effectively for existing and new grant applications at NIH and other funding agencies.      ', 'terms': ' abstracting; Applications Grants; Arts; Biomedical Research; Communicable Diseases; Communities; Core Facility; digital; Disease; DNA; DNA Sequence; DNA-Protein Interaction; Ensure; Epigenetic Process; Funding; Funding Agency; Gene Expression; Gene Expression Profile; Genetic Variation; Genome; genome-wide; Genomics; Hearing problem; HIV; Housing; Human Resources; Institutes; instrument; interest; Language Development; leukemia/lymphoma; Lymphoma; malignant breast neoplasm; Malignant Neoplasms; Medical center; Messenger RNA; methamphetamine abuse; MicroRNAs; Minor; Molecular; NCI-Designated Cancer Center; Nebraska; Nerve Degeneration; nervous system disorder; Neurodegenerative Disorders; Neurovirology; next generation; Nucleotides; Process; programs; Regulator Genes; Research; Research Personnel; Respiration Disorders; Scientist; Services; System; Technology; Training; United States National Institutes of Health; Universities; Vision'}, {'projectNumber': '1RC1HG005482-01', 'fy': 2009, 'title': 'STATISTICAL METHODS FOR ESTIMATION OF COPY NUMBER FROM NEXT-GENERATION SEQUENCING', 'department': 'HHS', 'agency': 'NIH', 'ic': 'NHGRI', 'totalCostAmount': 397116, 'nihApplId': 7829623, 'smApplId': 305381, 'budgetStartDate': '2009-09-22T00:00:00', 'budgetEndDate': '2010-06-30T00:00:00', 'contactPi': 'PARK, PETER J', 'otherPis': None, 'congressionalDistrict': '07', 'dunsId': '047006379', 'latitude': 42.335814, 'longitude': -71.105033, 'orgName': 'HARVARD UNIVERSITY', 'orgCity': 'BOSTON', 'orgState': 'MA', 'orgCountry': 'UNITED STATES', 'orgZipCode': '021156027', 'projectStartDate': '2009-09-22T00:00:00', 'projectEndDate': '2011-06-30T00:00:00', 'cfdaCode': '701', 'abstract': '  DESCRIPTION (provided by applicant):  This application addresses board Challenge Area (08) Genomics and specific challenge topic, 08-DA-102 Improved Bioinformatics Analysis for Deep Sequencing. The number of human samples undergoing whole-genome sequencing is expected to increase dramatically in the next few years, as advances in next-generation sequencing technologies continue to lower the cost of sequencing. In addition to detection of sequence variation, these data can be used to estimate DNA copy number variation and subsequently to examine correlation between copy number and phenotype. In this proposal, we aim to develop a series of computational steps and integrated analysis pipeline for accurate estimation of copy number from next-generation sequencing data. This involves efficient processing of the sequencing data, including appropriate alignment procedures and correction for experiment artifacts. For estimation of the copy number along chromosomal location, we will develop novel segmentation procedures, both for a single sample and for multiple samples, to take advantage of the specific nature of sequencing data. Importantly, we also address issues in experimental design, especially the effect of depth of sequencing (genome coverage) and read length on the resolution and accuracy of copy number profiles. We use data from a number of platforms including Solexa, SOLiD, and CompleteGenomes for our studies. The pipeline developed in this proposal will be implemented on a powerful distributed computing system and will be made available freely to the research community. The results of this project will thus enable efficient extraction of copy number from whole-genome sequencing data and will facilitate rapid translation of next-generation sequencing technology to identify structural variations associated with normal or disease phenotypes.           ', 'terms': ' Address; Algorithms; Area; Attention; base; Bioinformatics; Biological; chromatin immunoprecipitation; Clinical; cluster computing; Communities; comparative genomic hybridization; Computer Systems; computerized data processing; Computing Methodologies; Confidence Intervals; Copy Number Polymorphism; cost; Data; Data Sources; design; Detection; Development; Disease; disease phenotype; DNA copy number; experience; Experimental Designs; Genome; genome sequencing; Genomics; Health; Human; Human Genome; improved; Individual; interest; Length; Location; Malignant Neoplasms; Maps; markov model; method development; Modeling; Morphologic artifacts; Nature; next generation; novel; Patients; Phenotype; Population; Procedures; Process; Reading; Research; Research Design; Research Infrastructure; Research Personnel; research study; Resolution; Resources; Sampling; Sensitivity and Specificity; Series; Single Nucleotide Polymorphism; Solid; Statistical Methods; Structure; System; Techniques; Technology; tool; Translating; Translations; user-friendly; Variant; web interface; Work'}, {'projectNumber': '1RC2HG005596-01', 'fy': 2009, 'title': 'A UNIVERSAL FRONT END TO IMPROVE ASSEMBLY OUTCOMES FOR NEXT-GEN SEQUENCING AND RE', 'department': 'HHS', 'agency': 'NIH', 'ic': 'NHGRI', 'totalCostAmount': 732668, 'nihApplId': 7853052, 'smApplId': 312158, 'budgetStartDate': '2009-09-30T00:00:00', 'budgetEndDate': '2010-07-31T00:00:00', 'contactPi': 'BARRON, ANNELISE EMILY', 'otherPis': 'BATZOGLOU, SERAFIM  ; QUAKE, STEPHEN R ; SHAQFEH, ERIC S ; ', 'congressionalDistrict': '18', 'dunsId': '009214214', 'latitude': 37.430351, 'longitude': -122.17821, 'orgName': 'STANFORD UNIVERSITY', 'orgCity': 'STANFORD', 'orgState': 'CA', 'orgCountry': 'UNITED STATES', 'orgZipCode': '943041222', 'projectStartDate': '2009-09-30T00:00:00', 'projectEndDate': '2011-07-31T00:00:00', 'cfdaCode': '701', 'abstract': \"  DESCRIPTION (provided by applicant):  DNA sequencing is currently in the midst of disruptive technological shifts, with 454, Illumina, and Solid providing us with enormous throughput increases and large reductions in cost per base. Massively parallel technologies deliver a few Gbp of sequence per week as short fragments, or reads. New applications of sequencing only recently considered impractical are enabled: personal genome sequencing, 'metagenomics' analysis of 'soups' containing several, to hundreds of unique organisms, and finally, de novo sequencing of novel genomes of complex organisms. No matter how the sequencing is done, reads must be assembled computationally, if they are to be useful. Given the read length and read quality limitations of new instruments and the massive volume of data generated, the computational assembly problem is becoming critical, with the cost of computational infrastructure and personnel exceeding reagent and instrument-related costs. Moreover, the results of assembly are currently far from ideal; for example, much of the human genome remains invisible due to high percentage of repeats.  We propose to develop a new 'front end' to next-gen sequencers for DNA preparation, the 'Read-Cloud Method', which can reduce computational cost of genome assembly by 2-3 orders of magnitude, produce more complete and accurate genomes, and make metagenomics tractable. We propose a hierarchical sequencing approach, without any need for bacterial cloning. We will achieve this by handling single DNA molecules, tiled across the genome with high redundancy, on microfluidic devices. We will design, prototype, and thoroughly test technology to (i) shear genomic DNA into 200- kbp fragments with narrow size distributions; (ii) randomly amplify each individual, 200-kbp DNA in isolation, within a porous gel microcontainer that will be formed around the dsDNA molecule within a microdevice; (iii) digest micro-encapsulated DNA into small fragments, of tunable size; (iv) bar-code the progeny of each 200-kbp DNA with a 12mer oligonucleotide, to identify each read as associated with a particular 200-kbp DNA. A planar microfluidic device will be fabricated to allow one unique bar- code sequence to be blunt-end-ligated to both DNA termini. Bar-coded DNA is pooled, and next-gen sequencing is done. The results are a highly reducible data set. The method and algorithm are applicable universally, to next-generation platforms. The PIs (Batzoglou, Barron, Shaqfeh, Quake) will collaborate to make an efficient approach to hierarchical sequencing in microfluidic devices.      PUBLIC HEALTH RELEVANCE:  Project Narrative Gene sequencing is important to medicine. Our DNA sequencing method has the potential for reducing computational cost by orders of magnitude while making the assembled genomes significantly more complete and accurate. The key to this step is using microfluidic handling technologies to subdivide genomic DNA into 200kbp fragments, which are then amplified in isolation from each other and uniquely-labeled to form a highly reducible dataset for genomic assembly.      \", 'terms': ' Algorithms; Animal Model; Bar Codes; base; Cloning; Complex; computer infrastructure; Computing Methodologies; cost; Data; Data Set; design; Development; Devices; Digestion; DNA; DNA Sequence; Encapsulated; Escherichia coli; Frequencies; Gel; Genes; Genetic; Genetic Variation; Genome; genome sequencing; Genomics; Goals; Human Genome; Human Resources; Hydrogels; improved; Individual; Informatics; instrument; Label; Laws; Length; Manufacturer Name; Maps; Medicine; metagenomic sequencing; Metagenomics; Methods; Microbe; Microcapsules drug delivery system; Microfluidic Microchips; Microfluidics; microorganism; Modification; new technology; next generation; novel; Oligonucleotides; Organism; Outcome; Preparation; Process; Protocols documentation; prototype; public health relevance; Reading; Reagent; Relative (related person); Research; Route; Sampling; Shotguns; Solid; System; Technology; Testing; Time'}, {'projectNumber': '1RC2HG005697-01', 'fy': 2009, 'title': 'NEXT-GENERATION MEDICAL RESEQUENCING OF GOUT DISEASE GENES IN THE ARIC COHORT', 'department': 'HHS', 'agency': 'NIH', 'ic': 'NHGRI', 'totalCostAmount': 1317671, 'nihApplId': 7853113, 'smApplId': 312170, 'budgetStartDate': '2009-09-30T00:00:00', 'budgetEndDate': '2010-08-31T00:00:00', 'contactPi': 'HIXSON, JAMES E.', 'otherPis': None, 'congressionalDistrict': '07', 'dunsId': '800771594', 'latitude': 29.703025, 'longitude': -95.403303, 'orgName': 'UNIVERSITY OF TEXAS HLTH SCI CTR HOUSTON', 'orgCity': 'HOUSTON', 'orgState': 'TX', 'orgCountry': 'UNITED STATES', 'orgZipCode': '77030', 'projectStartDate': '2009-09-30T00:00:00', 'projectEndDate': '2011-08-31T00:00:00', 'cfdaCode': '701', 'abstract': \"  DESCRIPTION (provided by applicant):  This proposal responds to the GO Program 'ARRA Medical Sequencing Discovery Projects' to establish next-generation technologies for medical resequencing in smaller academic laboratories compared to larger facilities like the Genomic Sequencing Centers. In this application, we propose to use next-generation sequencing for medical resequencing of genes that have shown highly significant associations with gout and serum uric acid levels in genome-wide association studies (GWAS) in the 'Cohorts for Heart and Aging Research in Genomic Epidemiology (CHARGE)'. Resequencing will characterize the overall 'genetic architecture' by identification of both common functional variants that underlie GWAS statistical associations, as well as rare variants with larger phenotypic effects. Our laboratories have established key elements of next-generation sequencing including a robust and cost-effective process for subgenomic capture to enrich gene targets for resequencing, as well as implementation of the SOLID System v.3 (Applied Biosystems) and associated pipelines for data management and quality control. We have selected 11 genes from CHARGE GWAS results for resequencing of functional regions (promoters, exons, conserved regions) in gout cases and controls (total n=1,199) from the 'Atherosclerosis Risk in Communities (ARIC)' cohort. After resequencing, we will genotype variants in the entire ARIC cohort (n=16,000) to verify resequencing results, and to increase power for statistical analysis. Statistical analyses will include standard association studies for relatively common alleles, as well as analyses of rare variants by tests for differences in numbers of rare variant carriers in cases versus controls, and comparisons of mean uric acid concentrations in carriers versus the overall cohort. For both common and rare variants that show significant associations, we will use bioinformatics to identify possible functional consequences like non-conservative amino acid replacements and premature stop codons, disruption of normal mRNA splicing, or alterations in control elements that regulate gene expression. We propose to replicate our findings by genotyping and statistical analysis in two additional CHARGE cohorts including the 'Framingham Health Study (FHS)' and the 'Cardiovascular Health Study (CHS)'.    PUBLIC HEALTH RELEVANCE:  We propose to use next-generation DNA sequencing technologies to identify genetic variants that influence gout, one of the most common forms of arthritis affecting nearly 3 million adults in the US. Our subjects are from the 'Atherosclerosis Risk in Community (ARIC)' study, a large multi-ethnic epidemiological cohort (16,000 subjects) that has been measured for multiple disease-related risk factors and clinical endpoints. The identification of genetic variants will provide an improved understanding of molecular mechanisms that regulate serum levels of uric acid (the major risk factor for gout), and eventually lead to novel drug targets to improve treatment of gout.        \", 'terms': ' Accounting; Adult; Affect; African American; Age; Aging; Alleles; Amino Acids; Architecture; Arthritis; Atherosclerosis; Bioinformatics; Candidate Disease Gene; Cardiovascular system; case control; Caucasians; Caucasoid Race; Clinical; cohort; Communities; Complex; cost; Data; data management; database of Genotypes and Phenotypes; Deposition; design; Disease; Disease Progression; DNA Resequencing; DNA Sequence; Drug Delivery Systems; Educational workshop; Elements; Epidemiology; Etiology; Exons; follow-up; Framingham Heart Study; Gene Expression; Gene Frequency; Gene Targeting; Genes; Genetic; genetic variant; genome sequencing; genome wide association study; Genomics; Genotype; Gout; Health; Heart; Human; Human Genome; improved; Individual; Intercistronic Region; interest; Investigation; Joints; Laboratories; Lead; Learning; Measures; Medical; Medical Technology; Messenger RNA; Molecular; multidisciplinary; next generation; Nonsense Codon; novel; Outpatients; Pain; Phenotype; Prevention; Process; programs; Promoter Regions; protein structure; public health relevance; Quality Control; Research; Research Personnel; Resources; Risk; Risk Factors; RNA Splicing; Serum; Solid; success; Susceptibility Gene; System; Technology; Testing; Tissues; tool; trait; United States National Institutes of Health; Urate; Uric Acid; Variant; Visit'}, {'projectNumber': '1RC2HG005605-01', 'fy': 2009, 'title': 'STATISTICAL METHODS FOR NEXT-GEN SEQUENCING IN DISEASE ASSOCIATION STUDIES', 'department': 'HHS', 'agency': 'NIH', 'ic': 'NHGRI', 'totalCostAmount': 500000, 'nihApplId': 7853195, 'smApplId': 312178, 'budgetStartDate': '2009-09-30T00:00:00', 'budgetEndDate': '2010-07-31T00:00:00', 'contactPi': 'MARTIN, EDEN R.', 'otherPis': None, 'congressionalDistrict': '27', 'dunsId': '052780918', 'latitude': 25.713376, 'longitude': -80.277293, 'orgName': 'UNIVERSITY OF MIAMI SCHOOL OF MEDICINE', 'orgCity': 'CORAL GABLES', 'orgState': 'FL', 'orgCountry': 'UNITED STATES', 'orgZipCode': '331462926', 'projectStartDate': '2009-09-30T00:00:00', 'projectEndDate': '2011-07-31T00:00:00', 'cfdaCode': '701', 'abstract': \"Statistical Methods for Next-Generation Sequencing in Disease Association Studies Through this project we propose to develop statistical approaches and software for genotype calling andassociation testing in next-generation sequence data. The field is driven by molecular advances that allow foraffordable, massively parallel sequencing. The rapid development of statistical methods for next-generationsequence data in disease studies is necessary to keep pace with the advancing molecular technology. Next-generation sequencing is based on random, short-read technology; thus the coverage of any nucleotide ishighly variable and subject to error. Distinguishing random error from truly variable sites is required for 'SNP-calling'. One step beyond this is identifying the individual's actual genotype at the site. This is a highlystatistical problem and we have yet to see this problem addressed in a statistically rigorous manner. The solution that we propose, and what makes our approach novel, assumes that we have a sample ofindividuals, each with next-generation sequence data. We anticipate that sequencing may ultimately replaceGWAS SNP arrays for disease-association studies. While this may be several years away for whole-genomesequencing, sequencing enough people individually for a small association study is already becoming practicalwith target capture arrays. We can leverage the information from a sample of individuals with next-generationsequence data to more accurately estimate an individual's genotype and the position-specific error rate. Ourapproach is to express the genotype probabilities and error rate in a likelihood framework. We can then usestandard statistical theory to help us call genotypes. This approach should perform better than callinggenotypes for a single individual at a time based on an arbitrary filter as is currently done. A distinct advantage of this statistical framework is that the uncertainty in the genotype calls can beincorporated directly into our disease-association tests (e.g., case-control and rare variant analysis). In thisway we will increase power of our association tests and reduce bias due to error or systematic missingness.Incorporation of next-generation sequence data into the association tests provides a complete analysis pipelinefrom sequence to association.\", 'terms': ' Acceleration; Address; Algorithms; Area; base; case control; Classification; Classification Scheme; Computer software; Data; Data Set; Databases; design; Development; Disease; Disease Association; Evaluation; Genome; genome sequencing; genome wide association study; Genotype; Goals; Grant; Individual; Infusion procedures; meetings; method development; Molecular; new technology; next generation; novel strategies; Nucleotides; One-Step dentin bonding system; Performance; Positioning Attribute; Probability; programs; Reading; Sampling; Simulate; Site; Solutions; Statistical Methods; Technology; Testing; theories; Time; Uncertainty; Variant'}, {'projectNumber': '1RC2CA148317-01', 'fy': 2009, 'title': 'AN INFRASTRUCTURE FOR CANCER VIRUS DISCOVERY FROM NEXT-GENERATION SEQUENCING DATA', 'department': 'HHS', 'agency': 'NIH', 'ic': 'NCI', 'totalCostAmount': 765137, 'nihApplId': 7856252, 'smApplId': 312611, 'budgetStartDate': '2009-09-30T00:00:00', 'budgetEndDate': '2010-09-29T00:00:00', 'contactPi': 'MEYERSON, MATTHEW L.', 'otherPis': None, 'congressionalDistrict': '07', 'dunsId': '076580745', 'latitude': 42.337854, 'longitude': -71.108326, 'orgName': 'DANA-FARBER CANCER INSTITUTE', 'orgCity': 'BOSTON', 'orgState': 'MA', 'orgCountry': 'UNITED STATES', 'orgZipCode': '022155450', 'projectStartDate': '2009-09-30T00:00:00', 'projectEndDate': '2011-08-31T00:00:00', 'cfdaCode': '701', 'abstract': \"  DESCRIPTION (provided by applicant): Our goal is to build an infrastructure to discover novel viruses associated with human cancer from next-generation sequencing data, using a sequence-based computational subtraction approach that we developed. This proposed project responds to the ARRA Research and Research Infrastructure Grand Opportunities RFA on 'Identifying Potential Viral Signatures in Large Scale Studies of Germline and Somatic Changes in Cancer Genomes Pilot Program'. Large-scale genome projects, including The Cancer Genome Atlas (TCGA) as well as studies of germ-line genetic correlations with cancer, are now moving towards the application of ultra-high- throughput next-generation sequencing approaches, both on the cDNA level ('RNA-seq') and the DNA level, especially whole genome sequencing ('WGS'). The study is focused on the computational analysis of large next-generation sequencing data sets for virus discovery. Specifically, we plan to build an infrastructure to apply sequence-based computational subtraction, a method developed by the PI and co-investigator jointly, to evaluate the presence of novel non-human nucleic acid sequences in databases generated by these large-scale cancer genome projects. This approach starts with the assumption that virally-induced cancers contain both human and viral nucleic acids, and that subtraction of the human genome from cancer-derived sequences will leave residual candidate non-human and potentially viral sequences. First, we will build a software pipeline for computational subtraction-based data analysis and candidate pathogen sequence discovery. Second, we will apply this pipeline to the incoming flood of next-generation sequencing data from TCGA and other large-scale data sets. Third, we will experimentally test non-human sequences that we have identified for their presence in validation cohorts for the cancers in which they were discovered. Fourth, we will use the validation data to circle back and improve the quality of our computational pipeline. In the long run, we anticipate that we can build a sustainable pipeline that could be supported either as an academic or industrial effort. Identification of a novel infectious agent associated with human cancer would have immediate preventive, diagnostic and therapeutic significance. The infrastructure that we develop in this two-year project pilot will lay the groundwork for discovering additional cancer-associated pathogens in the future, by analyzing the ever-increasing quantities of next-generation cancer sequencing data.      PUBLIC HEALTH RELEVANCE: Viruses are among the major causes of human cancer. Discovering these viruses can lead to major improvements in public health, because virally induced cancers can be prevented by vaccination. In recent years, hepatitis B vaccination has led to a dramatic decrease in the occurrence of liver cancer, and human papillomavirus vaccination has been shown to decrease the rates of cervical carcinoma. Genome analysis and sequencing technologies are being used to discover the causes of human cancer, in projects such as The Cancer Genome Atlas, or TCGA. These technologies can also lead to the discovery of new viruses. Therefore the National Cancer Institute is investing funds from the American Recovery and Reinvestment Act of 2009 to support the discovery of new viruses in data from cancer genome projects such as TCGA. Our proposal is responsive to the National Cancer Institute request, entitled 'Identifying Potential Viral Signatures in Large Scale Studies of Germline and Somatic Changes in Cancer Genomes Pilot Program'. We have developed a powerful computational approach to compare DNA and RNA sequences from cancer, or from cancer patients, to the normal human genome. Sequences that are unique to cancers, or to cancer patients, may represent novel cancer- causing viruses. In this plan, we will build a stable software infrastructure to perform this sequence comparison, apply this infrastructure to data from large-scale cancer genome projects, test candidate sequences for whether they are likely to represent viruses, and then continue to improve the software infrastructure. This effort will enable discovery of viruses by the entire cancer research community.       \", 'terms': ' Algorithms; American; anticancer research; Atlases; Back; base; Base Sequence; Cancer Etiology; cancer genome; cancer genomics; Cancer Patient; cancer type; Cervix carcinoma; Cloning; cohort; Communities; Complementary DNA; Computer Analysis; Computer software; Data; Data Analyses; Data Set; Databases; Diagnostic; DNA; Floods; Formalin; Funding; Future; Gene Expression Profile; Genetic; Genome; genome sequencing; Genomics; Germ Lines; Goals; Hepatitis B Vaccination; Human; Human Genome; Human Papillomavirus; improved; Infectious Agent; Lead; Left; Malignant neoplasm of liver; Malignant Neoplasms; Methods; National Cancer Institute; next generation; novel; novel virus; Nucleic acid sequencing; Nucleic Acids; Oncogenic Viruses; Paraffin Embedding; pathogen; Phase; Pilot Projects; prevent; Preventive; programs; Public Health; public health relevance; Reading; Recovery; Recurrence; Research; Research Infrastructure; Research Personnel; Residual state; response; RNA; RNA Sequences; Running; Sampling; Sequence Analysis; Technology; Testing; Therapeutic; Tissues; transcriptomics; Vaccination; Validation; Viral; Virus'}, {'projectNumber': '1R01HG005692-01', 'fy': 2010, 'title': 'METHODS FOR THE ANALYSIS AND INTEGRATIONS OF NEXT-GENERATION SEQUENCING WITH APPL', 'department': 'HHS', 'agency': 'NIH', 'ic': 'NHGRI', 'totalCostAmount': 349700, 'nihApplId': 7865503, 'smApplId': 314959, 'budgetStartDate': '2010-06-01T00:00:00', 'budgetEndDate': '2011-02-28T00:00:00', 'contactPi': 'JOHNSON, WILLIAM EVAN', 'otherPis': None, 'congressionalDistrict': '03', 'dunsId': '009094012', 'latitude': 40.24569, 'longitude': -111.644678, 'orgName': 'BRIGHAM YOUNG UNIVERSITY', 'orgCity': 'PROVO', 'orgState': 'UT', 'orgCountry': 'UNITED STATES', 'orgZipCode': '846021001', 'projectStartDate': '2010-06-01T00:00:00', 'projectEndDate': '2015-02-28T00:00:00', 'cfdaCode': '172', 'abstract': '  DESCRIPTION (provided by applicant): Next-generation sequencing technologies are capable of producing tens of millions of sequence reads during each instrument run, and are quickly being applied in diverse types of experiments (e.g. RNA-Seq, miRNA-Seq, ChIP-Seq, BS-seq, CNV-Seq) to address biomedical questions by cost-effectively generating genome-wide datasets. While sequencing has been promoted as overcoming longstanding limitations of microarray-based studies, its data files are much larger than for microarrays, and its diverse data types raise similar as well as novel statistical and computational challenges. There is a pressing need for statistical and computational tools to address what leaders in the field have stated are the largest problems: data analysis and data integration. We propose to develop a comprehensive and coordinated set of statistical methods for high throughput sequencing (HTS) that directly address many important data analysis problems in epigenomics. Specifically we plan to address the following computational and statistical challenges facing researchers conducting HTS experiments: 1) develop sensitive statistical methods for the analysis of ChIP-seq data both for single- and paired-end-tag runs, particularly the focusing on applications in genome-wide profiling of nucleosome positions. 2) develop statistical methods for the analysis of BS-seq data, producing base-level DNA methylation profiles. 3) develop new statistical tools and methods for data integration in order to gain new biological insights about global transcription and regulation. We also plan to apply these approaches to a variety of high throughput sequencing data sets to demonstrate the relevance and utility of our methods. We plan to work with stimulated STAT1 and STAT3 data, and data from the ETS transcription factor family and its cofactors, for which we have already gathered significant data through our collaborations, including transcription factors, histone marks, DNAse I hypersensitivity and gene expression.    PUBLIC HEALTH RELEVANCE: We propose to develop a comprehensive and coordinated set of statistical methods for high throughput sequencing (HTS) that directly address many important data analysis problems in epigenomics. In particular, we plan to integrate data from multiple sources including expression, transcription factor binding, nucleosome positioning, histone marks and DNA methylation to better understand the mechanisms that regulate the behavior of a cell. Much of our proposal involves not just the development of new statistical and computational methods, but also the design, implementation and delivery of software tools that support these ideas. The many useful applications of next-generation sequencing with assure that or well- developed methods will have a broad impact in molecular biology, specifically in transcription regulation, chromatin dynamics, development, and cancer.            ', 'terms': ' Address; Algorithms; base; Behavior; Binding; Biological; bisulfite; Cells; Chromatin; cofactor; Collaborations; Computer software; computerized tools; Computing Methodologies; cost; Cytosine; Data; Data Analyses; Data Files; data integration; Data Set; Deoxyribonuclease I; design; Development; DNA; DNA Methylation; Ensure; epigenomics; Family; Future; Gene Expression; genome-wide; Genomics; Goals; histone modification; Histones; Hypersensitivity; insight; Institutes; instrument; Linker DNA; Malignant Neoplasms; Methods; MicroRNAs; Modeling; Molecular Biology; Names; National Human Genome Research Institute; next generation; novel; Nucleosomes; Positioning Attribute; Process; programs; public health relevance; Reading; Research Personnel; research study; Resolution; Running; Sequence Analysis; Software Tools; Source; STAT1 gene; STAT3 gene; Statistical Methods; Technology; Thymine; tool; transcription factor; Transcriptional Regulation; United States National Institutes of Health; Variant; Work'}, {'projectNumber': '1F32CA150631-01', 'fy': 2010, 'title': 'NEXT-GENERATION SEQUENCING OF THERAPY-RELATED ACUTE MYELOID LEUKEMIA', 'department': 'HHS', 'agency': 'NIH', 'ic': 'NCI', 'totalCostAmount': 35815, 'nihApplId': 7915136, 'smApplId': 337995, 'budgetStartDate': '2010-12-01T00:00:00', 'budgetEndDate': '2011-06-30T00:00:00', 'contactPi': 'MCNERNEY, MEGAN ELIZABETH', 'otherPis': None, 'congressionalDistrict': '01', 'dunsId': '005421136', 'latitude': 41.789554, 'longitude': -87.601172, 'orgName': 'UNIVERSITY OF CHICAGO', 'orgCity': 'CHICAGO', 'orgState': 'IL', 'orgCountry': 'UNITED STATES', 'orgZipCode': '606375418', 'projectStartDate': '2010-12-01T00:00:00', 'projectEndDate': '2011-06-30T00:00:00', 'cfdaCode': '398', 'abstract': '  DESCRIPTION (provided by applicant): Therapy-related acute myeloid eukemia/myelodysplastic syndrome (t-AML) is a complication that occurs in up to 10% of patients receiving chemotherapy with alkylating agents, topoisomerase II inhibitors, or radiation. This disease has increased in incidence as patients achieve longer remissions. T-AML remains refractory to current therapy and carries a uniformly fatal prognosis, with a median survival of 8-10 months. In this proposal, it is hypothesized that transformed hematopoietic cells have genetic mutations that drive leukemogenesis and can serve as therapeutic targets and biomarkers of disease. The specific aims are designed to test this hypothesis by sequencing the leukemic mRNA of t-AML patients using next-generation technology to identify genetic abnormalities, including potential gene fusions and mutations. Specific Aim 1 is the discovery and prioritization of molecular lesions in t-AML leukemia samples by sequencing all mRNAs in t- AML human samples. Specific Aim 2 is the validation of mutations by molecular techniques using the large number of t-AML samples available to us at the University of Chicago. Specific Aim 3 is to study the functional significance of mutations in vitro using a multitude of approaches dictated by the nature of the mutated gene products discovered. This information will allow the identification of pathways altered in t-AML, correlate these aberrations with clinical outcome, identify potential biomarkers, and generate candidate targets for leukemia therapeutics.    PUBLIC HEALTH RELEVANCE: Therapy-related acute myeloid leukemia (t-AML) is a side-effect of chemotherapy that occurs in up to 10% of cancer survivors and does not have an effective treatment. Cutting-edge sequencing technology will be used to analyze the genome of t-AML cancer cells in this project. The goal of this proposal is to understand the genetics of t-AML and identify the cause and a treatment for this disease.       ', 'terms': ' Acute; Adverse effects; Alkylating Agents; Biological Markers; cancer cell; Cancer Survivor; Cells; chemotherapy; Chicago; Clinical; Complication; design; Disease; Disease remission; Dysmyelopoietic Syndromes; effective therapy; Gene Fusion; Gene Mutation; gene therapy; Genome; Goals; Hematopoietic; Human; In Vitro; Incidence; Lesion; leukemia; leukemogenesis; Messenger RNA; Molecular; Molecular Abnormality; Mutate; Mutation; Myelogenous; Nature; next generation; Outcome; outcome forecast; Pathway interactions; Patients; Proteins; public health relevance; Radiation; Refractory; Sampling; Techniques; Technology; Testing; Therapeutic; therapeutic target; Therapy-Related Acute Myeloid Leukemia; Topoisomerase-II Inhibitor; Universities; Validation'}, {'projectNumber': '5RC1HG005482-02', 'fy': 2010, 'title': 'STATISTICAL METHODS FOR ESTIMATION OF COPY NUMBER FROM NEXT-GENERATION SEQUENCING', 'department': 'HHS', 'agency': 'NIH', 'ic': 'NHGRI', 'totalCostAmount': 376168, 'nihApplId': 7935506, 'smApplId': 348794, 'budgetStartDate': '2010-07-01T00:00:00', 'budgetEndDate': '2012-06-30T00:00:00', 'contactPi': 'PARK, PETER J', 'otherPis': None, 'congressionalDistrict': '07', 'dunsId': '047006379', 'latitude': 42.335814, 'longitude': -71.105033, 'orgName': 'HARVARD UNIVERSITY', 'orgCity': 'BOSTON', 'orgState': 'MA', 'orgCountry': 'UNITED STATES', 'orgZipCode': '021156027', 'projectStartDate': '2009-09-22T00:00:00', 'projectEndDate': '2012-06-30T00:00:00', 'cfdaCode': '701', 'abstract': '  DESCRIPTION (provided by applicant):  This application addresses board Challenge Area (08) Genomics and specific challenge topic, 08-DA-102 Improved Bioinformatics Analysis for Deep Sequencing. The number of human samples undergoing whole-genome sequencing is expected to increase dramatically in the next few years, as advances in next-generation sequencing technologies continue to lower the cost of sequencing. In addition to detection of sequence variation, these data can be used to estimate DNA copy number variation and subsequently to examine correlation between copy number and phenotype. In this proposal, we aim to develop a series of computational steps and integrated analysis pipeline for accurate estimation of copy number from next-generation sequencing data. This involves efficient processing of the sequencing data, including appropriate alignment procedures and correction for experiment artifacts. For estimation of the copy number along chromosomal location, we will develop novel segmentation procedures, both for a single sample and for multiple samples, to take advantage of the specific nature of sequencing data. Importantly, we also address issues in experimental design, especially the effect of depth of sequencing (genome coverage) and read length on the resolution and accuracy of copy number profiles. We use data from a number of platforms including Solexa, SOLiD, and CompleteGenomes for our studies. The pipeline developed in this proposal will be implemented on a powerful distributed computing system and will be made available freely to the research community. The results of this project will thus enable efficient extraction of copy number from whole-genome sequencing data and will facilitate rapid translation of next-generation sequencing technology to identify structural variations associated with normal or disease phenotypes.           ', 'terms': ' Address; Algorithms; Area; Attention; base; Bioinformatics; Biological; chromatin immunoprecipitation; Clinical; cluster computing; Communities; comparative genomic hybridization; Computer Systems; computerized data processing; Computing Methodologies; Confidence Intervals; Copy Number Polymorphism; cost; Data; Data Sources; design; Detection; Development; Disease; disease phenotype; DNA copy number; experience; Experimental Designs; Genome; genome sequencing; Genomics; Health; Human; Human Genome; improved; Individual; interest; Length; Location; Malignant Neoplasms; Maps; markov model; method development; Modeling; Morphologic artifacts; Nature; next generation; novel; Patients; Phenotype; Population; Procedures; Process; Reading; Research; Research Design; Research Infrastructure; Research Personnel; research study; Resolution; Resources; Sampling; Sensitivity and Specificity; Series; Single Nucleotide Polymorphism; Solid; Statistical Methods; Structure; System; Techniques; Technology; tool; Translating; Translations; user-friendly; Variant; web interface; Work'}, {'projectNumber': '5RC2CA148317-02', 'fy': 2010, 'title': 'AN INFRASTRUCTURE FOR CANCER VIRUS DISCOVERY FROM NEXT-GENERATION SEQUENCING DATA', 'department': 'HHS', 'agency': 'NIH', 'ic': 'NCI', 'totalCostAmount': 786413, 'nihApplId': 7941869, 'smApplId': 351863, 'budgetStartDate': '2010-09-30T00:00:00', 'budgetEndDate': '2012-08-31T00:00:00', 'contactPi': 'MEYERSON, MATTHEW L.', 'otherPis': None, 'congressionalDistrict': '07', 'dunsId': '076580745', 'latitude': 42.337854, 'longitude': -71.108326, 'orgName': 'DANA-FARBER CANCER INSTITUTE', 'orgCity': 'BOSTON', 'orgState': 'MA', 'orgCountry': 'UNITED STATES', 'orgZipCode': '022155450', 'projectStartDate': '2009-09-30T00:00:00', 'projectEndDate': '2012-08-31T00:00:00', 'cfdaCode': '701', 'abstract': \"  DESCRIPTION (provided by applicant): Our goal is to build an infrastructure to discover novel viruses associated with human cancer from next-generation sequencing data, using a sequence-based computational subtraction approach that we developed. This proposed project responds to the ARRA Research and Research Infrastructure Grand Opportunities RFA on 'Identifying Potential Viral Signatures in Large Scale Studies of Germline and Somatic Changes in Cancer Genomes Pilot Program'. Large-scale genome projects, including The Cancer Genome Atlas (TCGA) as well as studies of germ-line genetic correlations with cancer, are now moving towards the application of ultra-high- throughput next-generation sequencing approaches, both on the cDNA level ('RNA-seq') and the DNA level, especially whole genome sequencing ('WGS'). The study is focused on the computational analysis of large next-generation sequencing data sets for virus discovery. Specifically, we plan to build an infrastructure to apply sequence-based computational subtraction, a method developed by the PI and co-investigator jointly, to evaluate the presence of novel non-human nucleic acid sequences in databases generated by these large-scale cancer genome projects. This approach starts with the assumption that virally-induced cancers contain both human and viral nucleic acids, and that subtraction of the human genome from cancer-derived sequences will leave residual candidate non-human and potentially viral sequences. First, we will build a software pipeline for computational subtraction-based data analysis and candidate pathogen sequence discovery. Second, we will apply this pipeline to the incoming flood of next-generation sequencing data from TCGA and other large-scale data sets. Third, we will experimentally test non-human sequences that we have identified for their presence in validation cohorts for the cancers in which they were discovered. Fourth, we will use the validation data to circle back and improve the quality of our computational pipeline. In the long run, we anticipate that we can build a sustainable pipeline that could be supported either as an academic or industrial effort. Identification of a novel infectious agent associated with human cancer would have immediate preventive, diagnostic and therapeutic significance. The infrastructure that we develop in this two-year project pilot will lay the groundwork for discovering additional cancer-associated pathogens in the future, by analyzing the ever-increasing quantities of next-generation cancer sequencing data.      PUBLIC HEALTH RELEVANCE: Viruses are among the major causes of human cancer. Discovering these viruses can lead to major improvements in public health, because virally induced cancers can be prevented by vaccination. In recent years, hepatitis B vaccination has led to a dramatic decrease in the occurrence of liver cancer, and human papillomavirus vaccination has been shown to decrease the rates of cervical carcinoma. Genome analysis and sequencing technologies are being used to discover the causes of human cancer, in projects such as The Cancer Genome Atlas, or TCGA. These technologies can also lead to the discovery of new viruses. Therefore the National Cancer Institute is investing funds from the American Recovery and Reinvestment Act of 2009 to support the discovery of new viruses in data from cancer genome projects such as TCGA. Our proposal is responsive to the National Cancer Institute request, entitled 'Identifying Potential Viral Signatures in Large Scale Studies of Germline and Somatic Changes in Cancer Genomes Pilot Program'. We have developed a powerful computational approach to compare DNA and RNA sequences from cancer, or from cancer patients, to the normal human genome. Sequences that are unique to cancers, or to cancer patients, may represent novel cancer- causing viruses. In this plan, we will build a stable software infrastructure to perform this sequence comparison, apply this infrastructure to data from large-scale cancer genome projects, test candidate sequences for whether they are likely to represent viruses, and then continue to improve the software infrastructure. This effort will enable discovery of viruses by the entire cancer research community.       \", 'terms': ' Algorithms; American; anticancer research; Atlases; Back; base; Base Sequence; Cancer Etiology; cancer genome; cancer genomics; Cancer Patient; cancer type; Cervix carcinoma; Cloning; cohort; Communities; Complementary DNA; Computer Analysis; Computer software; Data; Data Analyses; Data Set; Databases; Diagnostic; DNA; Floods; Formalin; Funding; Future; Gene Expression Profile; Genetic; Genome; genome sequencing; Genomics; Germ Lines; Goals; Hepatitis B Vaccination; Human; Human Genome; Human Papillomavirus; improved; Infectious Agent; Lead; Left; Malignant neoplasm of liver; Malignant Neoplasms; Methods; National Cancer Institute; next generation; novel; novel virus; Nucleic acid sequencing; Nucleic Acids; Oncogenic Viruses; Paraffin Embedding; pathogen; Phase; Pilot Projects; prevent; Preventive; programs; Public Health; public health relevance; Reading; Recovery; Recurrence; Research; Research Infrastructure; Research Personnel; Residual state; response; RNA; RNA Sequences; Running; Sampling; Sequence Analysis; Technology; Testing; Therapeutic; Tissues; transcriptomics; Vaccination; Validation; Viral; Virus'}, {'projectNumber': '5RC2HG005605-02', 'fy': 2010, 'title': 'STATISTICAL METHODS FOR NEXT-GEN SEQUENCING IN DISEASE ASSOCIATION STUDIES', 'department': 'HHS', 'agency': 'NIH', 'ic': 'NHGRI', 'totalCostAmount': 500000, 'nihApplId': 7943996, 'smApplId': 352943, 'budgetStartDate': '2010-08-01T00:00:00', 'budgetEndDate': '2012-07-31T00:00:00', 'contactPi': 'MARTIN, EDEN R.', 'otherPis': None, 'congressionalDistrict': '27', 'dunsId': '052780918', 'latitude': 25.713376, 'longitude': -80.277293, 'orgName': 'UNIVERSITY OF MIAMI SCHOOL OF MEDICINE', 'orgCity': 'CORAL GABLES', 'orgState': 'FL', 'orgCountry': 'UNITED STATES', 'orgZipCode': '331462926', 'projectStartDate': '2009-09-30T00:00:00', 'projectEndDate': '2012-07-31T00:00:00', 'cfdaCode': '701', 'abstract': \"Statistical Methods for Next-Generation Sequencing in Disease Association Studies Through this project we propose to develop statistical approaches and software for genotype calling andassociation testing in next-generation sequence data. The field is driven by molecular advances that allow foraffordable, massively parallel sequencing. The rapid development of statistical methods for next-generationsequence data in disease studies is necessary to keep pace with the advancing molecular technology. Next-generation sequencing is based on random, short-read technology; thus the coverage of any nucleotide ishighly variable and subject to error. Distinguishing random error from truly variable sites is required for 'SNP-calling'. One step beyond this is identifying the individual's actual genotype at the site. This is a highlystatistical problem and we have yet to see this problem addressed in a statistically rigorous manner. The solution that we propose, and what makes our approach novel, assumes that we have a sample ofindividuals, each with next-generation sequence data. We anticipate that sequencing may ultimately replaceGWAS SNP arrays for disease-association studies. While this may be several years away for whole-genomesequencing, sequencing enough people individually for a small association study is already becoming practicalwith target capture arrays. We can leverage the information from a sample of individuals with next-generationsequence data to more accurately estimate an individual's genotype and the position-specific error rate. Ourapproach is to express the genotype probabilities and error rate in a likelihood framework. We can then usestandard statistical theory to help us call genotypes. This approach should perform better than callinggenotypes for a single individual at a time based on an arbitrary filter as is currently done. A distinct advantage of this statistical framework is that the uncertainty in the genotype calls can beincorporated directly into our disease-association tests (e.g., case-control and rare variant analysis). In thisway we will increase power of our association tests and reduce bias due to error or systematic missingness.Incorporation of next-generation sequence data into the association tests provides a complete analysis pipelinefrom sequence to association.\", 'terms': ' Acceleration; Address; Algorithms; Area; base; case control; Classification Scheme; Computer software; Data; Data Set; Databases; design; Development; Disease; Disease Association; Evaluation; Genome; genome sequencing; genome wide association study; Genotype; Goals; Grant; Individual; Infusion procedures; meetings; method development; Molecular; new technology; next generation; novel strategies; Nucleotides; One-Step dentin bonding system; Performance; Positioning Attribute; Probability; programs; Reading; Sampling; Simulate; Site; Solutions; Statistical Methods; Technology; Testing; theories; Time; Uncertainty; Variant'}, {'projectNumber': '5RC2HG005596-02', 'fy': 2010, 'title': 'A UNIVERSAL FRONT END TO IMPROVE ASSEMBLY OUTCOMES FOR NEXT-GEN SEQUENCING AND RE', 'department': 'HHS', 'agency': 'NIH', 'ic': 'NHGRI', 'totalCostAmount': 739589, 'nihApplId': 7945357, 'smApplId': 353570, 'budgetStartDate': '2010-08-01T00:00:00', 'budgetEndDate': '2012-07-31T00:00:00', 'contactPi': 'BARRON, ANNELISE EMILY', 'otherPis': 'BATZOGLOU, SERAFIM  ; QUAKE, STEPHEN R ; SHAQFEH, ERIC S ; ', 'congressionalDistrict': '18', 'dunsId': '009214214', 'latitude': 37.430351, 'longitude': -122.17821, 'orgName': 'STANFORD UNIVERSITY', 'orgCity': 'STANFORD', 'orgState': 'CA', 'orgCountry': 'UNITED STATES', 'orgZipCode': '943041222', 'projectStartDate': '2009-09-30T00:00:00', 'projectEndDate': '2012-07-31T00:00:00', 'cfdaCode': '701', 'abstract': \"  DESCRIPTION (provided by applicant):  DNA sequencing is currently in the midst of disruptive technological shifts, with 454, Illumina, and Solid providing us with enormous throughput increases and large reductions in cost per base. Massively parallel technologies deliver a few Gbp of sequence per week as short fragments, or reads. New applications of sequencing only recently considered impractical are enabled: personal genome sequencing, 'metagenomics' analysis of 'soups' containing several, to hundreds of unique organisms, and finally, de novo sequencing of novel genomes of complex organisms. No matter how the sequencing is done, reads must be assembled computationally, if they are to be useful. Given the read length and read quality limitations of new instruments and the massive volume of data generated, the computational assembly problem is becoming critical, with the cost of computational infrastructure and personnel exceeding reagent and instrument-related costs. Moreover, the results of assembly are currently far from ideal; for example, much of the human genome remains invisible due to high percentage of repeats.  We propose to develop a new 'front end' to next-gen sequencers for DNA preparation, the 'Read-Cloud Method', which can reduce computational cost of genome assembly by 2-3 orders of magnitude, produce more complete and accurate genomes, and make metagenomics tractable. We propose a hierarchical sequencing approach, without any need for bacterial cloning. We will achieve this by handling single DNA molecules, tiled across the genome with high redundancy, on microfluidic devices. We will design, prototype, and thoroughly test technology to (i) shear genomic DNA into 200- kbp fragments with narrow size distributions; (ii) randomly amplify each individual, 200-kbp DNA in isolation, within a porous gel microcontainer that will be formed around the dsDNA molecule within a microdevice; (iii) digest micro-encapsulated DNA into small fragments, of tunable size; (iv) bar-code the progeny of each 200-kbp DNA with a 12mer oligonucleotide, to identify each read as associated with a particular 200-kbp DNA. A planar microfluidic device will be fabricated to allow one unique bar- code sequence to be blunt-end-ligated to both DNA termini. Bar-coded DNA is pooled, and next-gen sequencing is done. The results are a highly reducible data set. The method and algorithm are applicable universally, to next-generation platforms. The PIs (Batzoglou, Barron, Shaqfeh, Quake) will collaborate to make an efficient approach to hierarchical sequencing in microfluidic devices.      PUBLIC HEALTH RELEVANCE:  Project Narrative Gene sequencing is important to medicine. Our DNA sequencing method has the potential for reducing computational cost by orders of magnitude while making the assembled genomes significantly more complete and accurate. The key to this step is using microfluidic handling technologies to subdivide genomic DNA into 200kbp fragments, which are then amplified in isolation from each other and uniquely-labeled to form a highly reducible dataset for genomic assembly.      \", 'terms': ' Algorithms; Animal Model; Bar Codes; base; Cloning; Complex; computer infrastructure; Computing Methodologies; cost; Data; Data Set; design; Development; Devices; Digestion; DNA; DNA Sequence; Encapsulated; Escherichia coli; Frequencies; Gel; Genes; Genetic; Genetic Variation; Genome; genome sequencing; Genomics; Goals; Human Genome; Human Resources; Hydrogels; improved; Individual; Informatics; instrument; Label; Laws; Length; Manufacturer Name; Maps; Medicine; metagenomic sequencing; Metagenomics; Methods; Microbe; Microcapsules drug delivery system; Microfluidic Microchips; Microfluidics; microorganism; Modification; new technology; next generation; novel; Oligonucleotides; Organism; Outcome; Preparation; Process; Protocols documentation; prototype; public health relevance; Reading; Reagent; Relative (related person); Research; Route; Sampling; Shotguns; Solid; System; Technology; Testing; Time'}, {'projectNumber': '5RC2HG005697-02', 'fy': 2010, 'title': 'NEXT-GENERATION MEDICAL RESEQUENCING OF GOUT DISEASE GENES IN THE ARIC COHORT', 'department': 'HHS', 'agency': 'NIH', 'ic': 'NHGRI', 'totalCostAmount': 1448896, 'nihApplId': 7945359, 'smApplId': 353572, 'budgetStartDate': '2010-09-01T00:00:00', 'budgetEndDate': '2012-08-31T00:00:00', 'contactPi': 'HIXSON, JAMES E.', 'otherPis': None, 'congressionalDistrict': '07', 'dunsId': '800771594', 'latitude': 29.703025, 'longitude': -95.403303, 'orgName': 'UNIVERSITY OF TEXAS HLTH SCI CTR HOUSTON', 'orgCity': 'HOUSTON', 'orgState': 'TX', 'orgCountry': 'UNITED STATES', 'orgZipCode': '77030', 'projectStartDate': '2009-09-30T00:00:00', 'projectEndDate': '2012-08-31T00:00:00', 'cfdaCode': '701', 'abstract': \"  DESCRIPTION (provided by applicant):  This proposal responds to the GO Program 'ARRA Medical Sequencing Discovery Projects' to establish next-generation technologies for medical resequencing in smaller academic laboratories compared to larger facilities like the Genomic Sequencing Centers. In this application, we propose to use next-generation sequencing for medical resequencing of genes that have shown highly significant associations with gout and serum uric acid levels in genome-wide association studies (GWAS) in the 'Cohorts for Heart and Aging Research in Genomic Epidemiology (CHARGE)'. Resequencing will characterize the overall 'genetic architecture' by identification of both common functional variants that underlie GWAS statistical associations, as well as rare variants with larger phenotypic effects. Our laboratories have established key elements of next-generation sequencing including a robust and cost-effective process for subgenomic capture to enrich gene targets for resequencing, as well as implementation of the SOLID System v.3 (Applied Biosystems) and associated pipelines for data management and quality control. We have selected 11 genes from CHARGE GWAS results for resequencing of functional regions (promoters, exons, conserved regions) in gout cases and controls (total n=1,199) from the 'Atherosclerosis Risk in Communities (ARIC)' cohort. After resequencing, we will genotype variants in the entire ARIC cohort (n=16,000) to verify resequencing results, and to increase power for statistical analysis. Statistical analyses will include standard association studies for relatively common alleles, as well as analyses of rare variants by tests for differences in numbers of rare variant carriers in cases versus controls, and comparisons of mean uric acid concentrations in carriers versus the overall cohort. For both common and rare variants that show significant associations, we will use bioinformatics to identify possible functional consequences like non-conservative amino acid replacements and premature stop codons, disruption of normal mRNA splicing, or alterations in control elements that regulate gene expression. We propose to replicate our findings by genotyping and statistical analysis in two additional CHARGE cohorts including the 'Framingham Health Study (FHS)' and the 'Cardiovascular Health Study (CHS)'.    PUBLIC HEALTH RELEVANCE:  We propose to use next-generation DNA sequencing technologies to identify genetic variants that influence gout, one of the most common forms of arthritis affecting nearly 3 million adults in the US. Our subjects are from the 'Atherosclerosis Risk in Community (ARIC)' study, a large multi-ethnic epidemiological cohort (16,000 subjects) that has been measured for multiple disease-related risk factors and clinical endpoints. The identification of genetic variants will provide an improved understanding of molecular mechanisms that regulate serum levels of uric acid (the major risk factor for gout), and eventually lead to novel drug targets to improve treatment of gout.        \", 'terms': ' Accounting; Adult; Affect; African American; Age; Aging; Alleles; Amino Acids; Architecture; Arthritis; Atherosclerosis; Bioinformatics; Candidate Disease Gene; Cardiovascular system; case control; Caucasians; Caucasoid Race; Clinical; cohort; Communities; Complex; cost; Data; data management; database of Genotypes and Phenotypes; Deposition; design; Disease; Disease Progression; DNA Resequencing; DNA Sequence; Drug Delivery Systems; Educational workshop; Elements; Epidemiology; Etiology; Exons; follow-up; Framingham Heart Study; Gene Expression; Gene Frequency; Gene Targeting; Genes; Genetic; genetic variant; genome sequencing; genome wide association study; Genomics; Genotype; Gout; Health; Heart; Human; Human Genome; improved; Individual; Intercistronic Region; interest; Investigation; Joints; Laboratories; Lead; Learning; Measures; Medical; Medical Technology; Messenger RNA; Molecular; multidisciplinary; next generation; Nonsense Codon; novel; Outpatients; Pain; Phenotype; Prevention; Process; programs; Promoter Regions; protein structure; public health relevance; Quality Control; Research; Research Personnel; Resources; Risk; Risk Factors; RNA Splicing; Serum; Solid; success; Susceptibility Gene; System; Technology; Testing; Tissues; tool; trait; United States National Institutes of Health; Urate; Uric Acid; Variant; Visit'}, {'projectNumber': '1K99HG005846-01', 'fy': 2010, 'title': 'HIGH-THROUGHPUT MOBILE ELEMENT GENOTYPING USING NEXT-GENERATION SEQUENCING', 'department': 'HHS', 'agency': 'NIH', 'ic': 'NHGRI', 'totalCostAmount': 88508, 'nihApplId': 7959278, 'smApplId': 360744, 'budgetStartDate': '2010-09-26T00:00:00', 'budgetEndDate': '2011-06-30T00:00:00', 'contactPi': 'XING, JINCHUAN', 'otherPis': None, 'congressionalDistrict': '02', 'dunsId': '009095365', 'latitude': 40.76776, 'longitude': -111.832926, 'orgName': 'UNIVERSITY OF UTAH', 'orgCity': 'SALT LAKE CITY', 'orgState': 'UT', 'orgCountry': 'UNITED STATES', 'orgZipCode': '841128930', 'projectStartDate': '2010-09-26T00:00:00', 'projectEndDate': '2012-06-30T00:00:00', 'cfdaCode': '172', 'abstract': '  DESCRIPTION (provided by applicant):  Mobile DNA elements are DNA segments that can mobilize from one genomic location to another, some making copies of themselves during the process.  Mobile elements are ubiquitous in eukaryotic genomes, and roughly 50% of the human genome consists of mobile, repetitive DNA sequences such as Alu and LINE1 elements.  Mobile elements have profoundly influenced primate genomic evolution, and alterations caused by mobile elements have been identified in more than 100 human genetic disorders.  In addition, mobile elements are highly useful as genetic markers in tracing relationships of populations and species because of their unique manner of propagating in the genome.  Therefore, it is of great interest to understand the biology of mobile elements in human populations.  Currently available genotyping methods, however, provide only limited insight into the distribution of Alu and L1 elements in the human genome.  In this proposal, I will first describe a novel, high-throughput method to genotype a large number of individuals for Alu insertion polymorphisms.  Implementation of this method will permit, for the first time, rapid, inexpensive genotyping of whole families of mobile elements not only in humans but in any organism.  Next, I outline my plans to construct a computational pipeline that can efficiently handle and analyze the high- throughput genomic data generated by my genotyping method.  Finally I describe my plan to apply the genotyping procedure to 540 HapMap samples and use my computational pipeline to analyze the results.  Specifically, I will use the insertion polymorphism data to assess the genomic diversity associated with mobile element insertions, as well as to answer questions related to mobile element biology and to reconstruct ancient human demographic history.  In addition, the Alu insertion polymorphism dataset generated in this project will provide a valuable supplementary resource of structural variants for the HapMap and 1000 Genomes Projects.  My long-term goal is to pursue a career in human genomic variation and population genetics research with an emphasis on mobile elements.  Using a combination of computational and experimental tools, I will study the impact of mobile DNA elements on the human genome and on genetic diversity among human populations.  This award will provide the necessary training for me to become an independent investigator on two fronts:  I will acquire essential training in computational biology and bioinformatics to handle large-scale genomic data, including data generated from next-generation sequencing technology. In addition, research training offered by the institution and my mentor will allow me to acquire knowledge about research resources, grant writing, responsible conduct in research, and other information necessary to become a successful, independent investigator.    PUBLIC HEALTH RELEVANCE:  Mobile DNA elements have had profound and widespread effects on the human genome, and they have been implicated in the causation of dozens of human diseases.  This proposal will develop a low-cost, high-throughput technology that can be used to genotype mobile element insertions in humans and other organisms.  This technology will greatly enhance our understanding of mobile element biology in human populations and provide a valuable structural variant data resource for the 1000 Genomes Project.      ', 'terms': ' Award; Bioinformatics; Biology; career; Computational Biology; cost; Data; Data Analyses; data management; Data Set; DNA; DNA Insertion Elements; DNA Sequence; Elements; Etiology; Evolution; Family; Genes; Genetic Markers; Genetic Polymorphism; Genetic Research; Genetic Variation; Genome; genome sequencing; Genomics; Genotype; Goals; Grant; Guanine + Cytosine Composition; Hereditary Disease; high throughput technology; Human; human disease; Human Genetics; Human Genome; indexing; Indium; Individual; insight; Institution; interest; Knowledge; L1 Elements; Libraries; Location; Mentors; Methods; next generation; novel; Organism; Phase; Population; Population Genetics; Primates; Procedures; Process; Property; Protocols documentation; public health relevance; Recording of previous events; Research; Research Personnel; Research Training; Resources; Sampling; Scanning; Site; Structure; Techniques; Technology; Time; tool; Training; transposon/insertion element; Variant; Writing'}, {'projectNumber': '1R43HG006081-01', 'fy': 2011, 'title': 'SLIPCHIP FOR TARGET ENRICHMENT FOR NEXT-GEN SEQUENCING BY MULTIPLEXED PCR', 'department': 'HHS', 'agency': 'NIH', 'ic': 'NHGRI', 'totalCostAmount': 173381, 'nihApplId': 8057982, 'smApplId': 387332, 'budgetStartDate': '2011-04-01T00:00:00', 'budgetEndDate': '2013-12-31T00:00:00', 'contactPi': 'SHEN, FENG', 'otherPis': 'LI, LIANG  ; ', 'congressionalDistrict': '07', 'dunsId': '962040890', 'latitude': 41.8781136, 'longitude': -87.6297982, 'orgName': 'SLIPCHIP LLC', 'orgCity': 'MENLO PARK', 'orgState': 'CA', 'orgCountry': 'UNITED STATES', 'orgZipCode': '94025', 'projectStartDate': '2011-04-01T00:00:00', 'projectEndDate': '2013-12-31T00:00:00', 'cfdaCode': '172', 'abstract': \"  DESCRIPTION (provided by applicant): 1. For the past 30 years, Sanger sequencing has enabled the determination of a sequence of nucleotide  2 bases on a small segment of DNA, with the crowning achievement of the technology being the complete  3 elucidation of the human genome at a cost of several hundreds of millions of dollars. Recently, 'next-  4 generation' sequencing technologies have been developed capable of processing millions of DNA templates in  5 parallel. Compared to the astronomical cost of the original human genome project, the current cost to 6 sequence a complete human genome using next-generation sequencing technology is currently under  7 $50,000. However, even this relatively lower cost is still too high for the majority of scientific inquiry or clinical  8 studies that require deep sequencing of focused regions of the genome. Thus, targeted enrichment strategies  9 that enrich only desired segments of the genome for subsequent sequencing are necessary. And, there is 10 currently no solution for targeted enrichment exists that is simple, affordable to individual research or clinical 11 laboratories, and that incorporates quality control. 12 SlipChip is an attractive and innovative technology that can potentially solve this problem. SlipChip has 13 already been applied to complex protocols such as enzyme assays, immunoassays, and PCR assays. The PIs 14 have demonstrated that SlipChips fabricated in glass can be used to perform multiplexed PCR and digital PCR. 15 The goal of this Phase I proposal is to establish the feasibility and applicability of this innovation in the area of 16 multiplexed PCR for targeted enrichment and subsequent sequencing, while testing the feasibility of upstream 17 and downstream steps in the sequencing workflow: from Aim 1) where we will test and verify the robustness of 18 multiplexed PCR in the SlipChip using human DNA, Aim 2) where we will optimize the density of SlipChip and 19 finally, Aim 3) where We will verify that on-chip quality control of PCR before committing a sample is feasible. 20 Reaching these specific aims would firmly establish feasibility and viability of SlipChip technology as a 21 sample processing tool for targeted enrichment for next generation sequencing by multiplexed PCR, and would 22 reduce the technical risk of Phase II work in this area, which would include sequencing experiments in the 23 context of biomedical problems of high significance, and will allow for the study of the validated SlipChip 24 system for complete integration in the workflow of an existing next-generation sequencing tool, specifically the 25 Illumina GA IIe.    PUBLIC HEALTH RELEVANCE: 1 Technologies for the targeted enrichment of specific genomic regions are urgently needed to bring the 2 power of next-generation sequencing to fundamental and applied biomedical research. However, no solution 3 for targeted enrichment exists that is simple and affordable to individual research or clinical laboratories. This 4 proposal describes a SlipChip technology to address this unmet need.      \", 'terms': ' Achievement; Address; Area; base; Base Sequence; Biological Assay; Biomedical Research; cancer type; Characteristics; Chicago; Clinical; clinical application; clinical practice; Commit; Complex; cost; Crystallization; density; Dental crowns; design; Detection; developmental disease/disorder; Devices; Dideoxy Chain Termination DNA Sequencing; digital; Disease; DNA; DNA biosynthesis; DNA Microarray Chip; Enzymes; gel electrophoresis; Gene Mutation; Generations; Genome; Genomics; Genotype; Glass; Goals; Guanine + Cytosine Composition; human DNA; human DNA sequencing; Human Genome; Human Genome Project; Immunoassay; Individual; Industry; innovation; innovative technologies; Institutes; interest; Laboratories; Lead; Letters; Malignant Neoplasms; Measures; meetings; Methods; next generation; Performance; Phase; Plastics; Problem Solving; Process; Proteins; Protocols documentation; Qualifying; Quality Control; Research; research study; Risk; Sampling; Sensitivity and Specificity; Solutions; Specificity; Spottings; System; Systems Biology; Technology; Testing; tool; Universities; Validation; Work'}, {'projectNumber': '1F32GM096711-01', 'fy': 2011, 'title': 'NEXT-GENERATION SEQUENCING AS A PLATFORM FOR ACTINOMYCETE NATURAL PRODUCT DISCOVE', 'department': 'HHS', 'agency': 'NIH', 'ic': 'NIGMS', 'totalCostAmount': 48398, 'nihApplId': 8060837, 'smApplId': 388280, 'budgetStartDate': '2011-02-01T00:00:00', 'budgetEndDate': '2012-01-31T00:00:00', 'contactPi': 'BERNHARDT, PETER', 'otherPis': None, 'congressionalDistrict': '49', 'dunsId': '804355790', 'latitude': 37.240049, 'longitude': -119.503236, 'orgName': 'UNIVERSITY OF CALIFORNIA SAN DIEGO', 'orgCity': 'LA JOLLA', 'orgState': 'CA', 'orgCountry': 'UNITED STATES', 'orgZipCode': '920930934', 'projectStartDate': '2011-02-01T00:00:00', 'projectEndDate': '2013-04-30T00:00:00', 'cfdaCode': '859', 'abstract': '  DESCRIPTION (provided by applicant): Actinomycetes have a tremendous potential to produce natural products with medicinal value. The long-term goal of this project is to develop novel agents for the treatment of human disease facilitated by the application of next-generation DNA sequencing to rapidly access actinomycete genetic information. The marine Streptomyces sp. CNH-189 is a producer of structurally fascinating meroterpenoids with potent activity against a broad panel of multi-drug resistant Staphylococcus aureus strains. The overall research objectives are to beta-test Illumina DNA sequencing for rapidly accessing natural product gene cluster sequences in Streptomyces sp. CNH-189, and to use genetics and biochemistry to link meroterpenoid biosynthesis to the identified gene clusters. Preliminary studies include the construction of genomic DNA fragment libraries that are ready for paired-end Illumina sequencing, and the isolation of a fosmid clone that contains a large part of the biosynthetic gene cluster for meroterpenoid biosynthesis. Through a series of experiments, I propose to evaluate how Illumina sequencing can be used to find gene clusters in Streptomyces sp. CNH-189 and other actinomycetes, to obtain a complete coverage of the gene cluster, and to probe the biosynthesis through in vitro biochemistry, gene disruption, and comparative metabolite profiling. The unique chemical structures and biological activities of the natural products isolated from Streptomyces sp. CNH-189 may lead to new chemotherapies and fundamental biochemical knowledge.    PUBLIC HEALTH RELEVANCE: Many pharmaceuticals currently used in the clinic are natural product based. Therefore, natural product discovery, understanding of biosynthetic pathways, and engineering the production of natural product analogs are key for the development of new human chemotherapies. Next-generation DNA sequencing and short read assembly ofers a new and potentialy general platform for accessing gene clusters that could facilitate the genome mining and understanding of clustered biosynthetic pathways in microorganisms. The proposed research will for the first time evaluate Illumina sequencing as a platform for rapidly accessing biosynthetic gene clusters in actinomycetes. Successful completion of this research may advance the field of natural product discovery and biosynthesis.        ', 'terms': ' Actinobacteria class; Anabolism; analog; Antibiotics; base; Biochemical; Biochemistry; Bioinformatics; Biological; Biological Assay; Biological Factors; Cell Culture Techniques; Chemical Structure; chemotherapy; Clinic; comparative; Data Set; Development; DNA; DNA Sequence; Engineering; Enzymes; Event; fascinate; Future; Gene Cluster; Generations; Genes; Genetic; Genome; genome sequencing; Genomic Library; Genomics; Goals; Human; human disease; In Vitro; in vivo; Knock-out; Knowledge; Laboratories; Lead; Libraries; Link; Marines; methicillin resistant Staphylococcus aureus; microorganism; Mining; Molecular Biology; Multi-Drug Resistance; next generation; novel; Pathway interactions; Pharmacologic Substance; Phase; Play; Production; Reading; Research; research study; Role; Screening procedure; Series; Staphylococcus aureus; Streptomyces; Technology; Testing; Time; tool'}, {'projectNumber': '1R25HG006110-01', 'fy': 2011, 'title': 'SHORT COURSE ON NEXT-GENERATION SEQUENCING: TECHNOLOGY AND STATISTICAL METHODS', 'department': 'HHS', 'agency': 'NIH', 'ic': 'NHGRI', 'totalCostAmount': 52471, 'nihApplId': 8077692, 'smApplId': 396750, 'budgetStartDate': '2011-04-27T00:00:00', 'budgetEndDate': '2012-01-31T00:00:00', 'contactPi': 'TIWARI, HEMANT K.', 'otherPis': None, 'congressionalDistrict': '07', 'dunsId': '063690705', 'latitude': 32.635016, 'longitude': -86.809379, 'orgName': 'UNIVERSITY OF ALABAMA AT BIRMINGHAM', 'orgCity': 'BIRMINGHAM', 'orgState': 'AL', 'orgCountry': 'UNITED STATES', 'orgZipCode': '352940001', 'projectStartDate': '2011-04-27T00:00:00', 'projectEndDate': '2014-01-31T00:00:00', 'cfdaCode': '172', 'abstract': \"  DESCRIPTION (provided by applicant): Major efforts are under way to uncover the specific genetic components of many complex human disorders and quantitative traits. These efforts include rapidly evolving technologies for next- generation sequencing that will generate an unprecedented wealth of data. In that turn that will require sophisticated statistical analysis in order to maximize the information gleaned from each study. Likewise, many advances have been made in statistical methods for the study of complex genetic traits, and new statistical methods continue to be promulgated. To expedite and significantly advance the search for specific genes that predispose to these complex traits, we propose a series of annual short courses on next-generation sequencing technologies and statistical genomics which will enable a far greater number of researchers, including clinical researchers, to participate in, contribute to, and lead such research. These short courses will help equip students, post-doctoral fellows, junior investigators and senior investigators to expedite genomic discovery. The courses will be taught by leading experts in next-generation sequencing and statistical genetics/genomics. Each 4-day course will provide substantial expertise both in technologies for next- generation sequencing and in statistical methods pertaining to these technologies that will effectively increase the number and the expertise of investigators who are pursuing genetic and genomic research. This is a unique course containing instruction in both technological and statistical aspects of next-generation sequencing. There is currently no course of this type offered in US. We expect approximately 50 students to attend each course, allowing for intensive interaction between students and faculty. Lectures will be supplemented with extensive discussion sessions, handouts, and presentation of worked examples.    PUBLIC HEALTH RELEVANCE: We propose an annual short course each year for 3 years on next-generation sequencing and analysis methods for investigators studying complex traits. Taught by leading experts in next-generation sequencing technology and statistical genomics, these courses will be aimed at statisticians and non- statistician investigators who wish to learn both the rapidly evolving technology and 'language' of statistical genomics. Attendees should be either current investigators or in the early stages of a career in genetic/genomic research.      \", 'terms': ' Alabama; Applications Grants; Bioinformatics; Biotechnology; career; Characteristics; Clinical; Communities; Complex; Complex Genetic Trait; Consultations; Data; Data Analyses; Development; disability; Disease; Education; education evaluation; Educational process of instructing; Evaluation; experience; Faculty; Fees; Funding; Genetic; Genomics; Glean; graduate student; Human; Institutes; Instruction; Laboratories; Language; Lead; Learning; lectures; Location; meetings; Methods; Modification; Molecular; next generation; Participant; Postdoctoral Fellow; programs; Request for Proposals; Research; Research Personnel; Resources; responsible research conduct; Selection Criteria; Sequence Analysis; Series; Staging; Statistical Methods; Students; Susceptibility Gene; Technology; Training; trait; Universities; web site; Work'}, {'projectNumber': '5R01HG005692-02', 'fy': 2011, 'title': 'METHODS FOR THE ANALYSIS AND INTEGRATIONS OF NEXT-GENERATION SEQUENCING WITH APPL', 'department': 'HHS', 'agency': 'NIH', 'ic': 'NHGRI', 'totalCostAmount': 333828, 'nihApplId': 8078848, 'smApplId': 397320, 'budgetStartDate': '2011-03-01T00:00:00', 'budgetEndDate': '2011-09-30T00:00:00', 'contactPi': 'JOHNSON, WILLIAM EVAN', 'otherPis': None, 'congressionalDistrict': '03', 'dunsId': '009094012', 'latitude': 40.24569, 'longitude': -111.644678, 'orgName': 'BRIGHAM YOUNG UNIVERSITY', 'orgCity': 'PROVO', 'orgState': 'UT', 'orgCountry': 'UNITED STATES', 'orgZipCode': '846021001', 'projectStartDate': '2010-06-01T00:00:00', 'projectEndDate': '2011-09-30T00:00:00', 'cfdaCode': '172', 'abstract': '  DESCRIPTION (provided by applicant): Next-generation sequencing technologies are capable of producing tens of millions of sequence reads during each instrument run, and are quickly being applied in diverse types of experiments (e.g. RNA-Seq, miRNA-Seq, ChIP-Seq, BS-seq, CNV-Seq) to address biomedical questions by cost-effectively generating genome-wide datasets. While sequencing has been promoted as overcoming longstanding limitations of microarray-based studies, its data files are much larger than for microarrays, and its diverse data types raise similar as well as novel statistical and computational challenges. There is a pressing need for statistical and computational tools to address what leaders in the field have stated are the largest problems: data analysis and data integration. We propose to develop a comprehensive and coordinated set of statistical methods for high throughput sequencing (HTS) that directly address many important data analysis problems in epigenomics. Specifically we plan to address the following computational and statistical challenges facing researchers conducting HTS experiments: 1) develop sensitive statistical methods for the analysis of ChIP-seq data both for single- and paired-end-tag runs, particularly the focusing on applications in genome-wide profiling of nucleosome positions. 2) develop statistical methods for the analysis of BS-seq data, producing base-level DNA methylation profiles. 3) develop new statistical tools and methods for data integration in order to gain new biological insights about global transcription and regulation. We also plan to apply these approaches to a variety of high throughput sequencing data sets to demonstrate the relevance and utility of our methods. We plan to work with stimulated STAT1 and STAT3 data, and data from the ETS transcription factor family and its cofactors, for which we have already gathered significant data through our collaborations, including transcription factors, histone marks, DNAse I hypersensitivity and gene expression.    PUBLIC HEALTH RELEVANCE: We propose to develop a comprehensive and coordinated set of statistical methods for high throughput sequencing (HTS) that directly address many important data analysis problems in epigenomics. In particular, we plan to integrate data from multiple sources including expression, transcription factor binding, nucleosome positioning, histone marks and DNA methylation to better understand the mechanisms that regulate the behavior of a cell. Much of our proposal involves not just the development of new statistical and computational methods, but also the design, implementation and delivery of software tools that support these ideas. The many useful applications of next-generation sequencing with assure that or well- developed methods will have a broad impact in molecular biology, specifically in transcription regulation, chromatin dynamics, development, and cancer.            ', 'terms': ' Address; Algorithms; base; Behavior; Binding; Biological; bisulfite; Cells; Chromatin; cofactor; Collaborations; Computer software; computerized tools; Computing Methodologies; cost; Cytosine; Data; Data Analyses; Data Files; data integration; Data Set; design; Development; DNA; DNA Methylation; Ensure; epigenomics; Family; Future; Gene Expression; Genetic Transcription; genome-wide; Genomics; Goals; histone modification; Histones; Hypersensitivity; insight; Institutes; instrument; Linker DNA; Malignant Neoplasms; Methods; MicroRNAs; Modeling; Molecular Biology; Names; National Human Genome Research Institute; next generation; novel; Nucleosomes; Positioning Attribute; Process; programs; public health relevance; Reading; Research Personnel; research study; Resolution; RNA; Running; Sequence Analysis; Software Tools; Source; STAT1 gene; STAT3 gene; Statistical Methods; Technology; Thymine; tool; transcription factor; Transcriptional Regulation; United States National Institutes of Health; Variant; Work'}, {'projectNumber': '1R01HD068578-01', 'fy': 2011, 'title': 'NON-INVASIVE DETECTION OF FETAL ANEUPLOIDY BY NEXT-GENERATION DNA SEQUENCING', 'department': 'HHS', 'agency': 'NIH', 'ic': 'NICHD', 'totalCostAmount': 481483, 'nihApplId': 8084747, 'smApplId': 399590, 'budgetStartDate': '2011-05-15T00:00:00', 'budgetEndDate': '2012-01-31T00:00:00', 'contactPi': 'PETERS, DAVID GERARD', 'otherPis': None, 'congressionalDistrict': '14', 'dunsId': '119132785', 'latitude': 40.434217, 'longitude': -79.954766, 'orgName': \"MAGEE-WOMEN'S RES INST AND FOUNDATION\", 'orgCity': 'PITTSBURGH', 'orgState': 'PA', 'orgCountry': 'UNITED STATES', 'orgZipCode': '152134430', 'projectStartDate': '2011-05-15T00:00:00', 'projectEndDate': '2016-01-31T00:00:00', 'cfdaCode': '865', 'abstract': '  DESCRIPTION (provided by applicant): Prenatal diagnosis of fetal genetic disease has evolved to reach a prominent position in obstetric clinical care. Established screening methods targeted towards serum proteins are used routinely alongside ultrasonography to identify potentially abnormal pregnancies. Definitive diagnosis is then undertaken using interventional procedures such as amniocentesis and chorionic villus sampling (CVS) that obtain fetal or placental cells, respectively, for karyotype analysis. However, these invasive procedures involve a risk of associated miscarriage. This is significant because, for trisomy 21, current non-invasive first trimester screening methods have detection rates of 82 to 87% and false positive rates of approximately 5%. Therefore, up to 18% of true positives are missed and one expectant mother in every twenty who are screened will undergo an unnecessary invasive diagnostic procedure that could result in the avoidable miscarriage of her baby. In addition to the risk of mortality and morbidity, invasive procedures invoke considerable parental anxiety. Our goal is to dramatically reduce these avoidable miscarriages and other associated risks by developing a diagnostic method that significantly improves the sensitivity and specificity of non-invasive prenatal detection of aneuploidy in the first trimester. To achieve this goal we will expand on our recently published work to test the hypothesis that shotgun next generation sequencing of first trimester maternal plasma DNA provides improved sensitivity and specificity over existing combinations of serum screening and ultrasound. Significantly, earlier economic and logistical barriers preventing the translation of this approach to clinical practice have very recently been overcome by the emergence of methods for high-throughput DNA sequencing that are cost-effective for clinical diagnosis. Specifically, in Aim 1, we will carry out shotgun next generation sequencing on samples of maternal plasma DNA obtained in the first trimester of pregnancy from large cohorts of confirmed aneuploidy and control pregnancies (combined n = 70). We will then undertake a formal statistical analysis to determine the sensitivity and specificity of next-generation DNA sequencing for the detection of aneuploidy on chromosomes 13, 18, 21 and X and compare these results to sensitivity and specificity data obtained using existing first trimester screening methods in the same cohort (Aim 2). Finally we will develop a software package with graphical user interface that can be utilized by non-specialist end users for the rapid analysis of next generation sequencing data and the detection of aneuploidy (Aim 3). We anticipate that this new first trimester test will increase the detection rate of fetal aneuploidy to 95% and reduce the false positive rate to 1%, resulting in an 80% reduction in unnecessary miscarriages associated with invasive prenatal diagnosis after first trimester screening.    PUBLIC HEALTH RELEVANCE: The primary goal of this study is to test the hypothesis that next-generation sequencing of maternal plasma DNA provides significantly improved sensitivity and specificity over existing combinations of serum screening and ultrasound. Specifically, in Aim 1, we will carry out shotgun next generation sequencing on samples of maternal plasma DNA obtained in the first trimester of pregnancy from large cohorts of confirmed aneuploidy and control pregnancies (combined n = 700). We will then undertake a formal statistical analysis to determine the sensitivity and specificity of next-generation DNA sequencing for the detection of aneuploidy on chromosomes 13, 18, 21 and X and compare these results to sensitivity and specificity data obtained using existing first trimester screening methods in the same cohort (Aim 2). Finally we will develop a software package with graphical user interface that can be utilized by non-specialist end users for the rapid analysis of next generation sequencing data and the detection of aneuploidy (Aim 3). We anticipate that this new first trimester test will increase the detection rate of fetal aneuploidy from 82% to 95% and reduce the false positive rate from 5% to 1%, resulting in an 80% reduction in the rate of avoidable fetal miscarriage associated with invasive prenatal diagnosis.        ', 'terms': ' Alleles; Amniocentesis; analytical method; analytical tool; Aneuploidy; Anxiety; Apoptotic; Biological Assay; Blood Circulation; Cells; Chorionic villi; Chorionic Villi Sampling; Chromosomes, Human, Pair 13; clinical care; clinical Diagnosis; clinical practice; cohort; Collaborations; Communities; computer science; Computer software; cost; cost effective; Data; Data Analyses; Detection; Development; Diagnosis; Diagnostic Procedure; Diagnostics Research; Discipline of obstetrics; DNA; DNA Sequence; Down Syndrome; Economics; fetal; Fetal Diseases; Fetus; First Pregnancy Trimester; Genomics; Genotype; Goals; graphical user interface; Hereditary Disease; high throughput analysis; improved; Inherited; Karyotype; Laboratory Technicians; Methods; minimally invasive; Morbidity - disease rate; mortality; Mothers; next generation; Non-Invasive Cancer Detection; novel; Nucleic Acids; open source; Plasma; Positioning Attribute; Pregnancy; Pregnant Women; prenatal; Prenatal Diagnosis; prevent; Procedures; Publishing; Research; research study; Risk; RNA; Sampling; Screening procedure; Sensitivity and Specificity; Serum; Serum Proteins; Shotguns; Spontaneous abortion; statistics; Technology; Testing; tool; Translations; Ultrasonography; Work'}, {'projectNumber': '1R01HG005859-01A1', 'fy': 2011, 'title': 'STATISTICAL METHODS FOR POPULATION GENOMICS AND \"NEXT-GEN\" SEQUENCING DATA', 'department': 'HHS', 'agency': 'NIH', 'ic': 'NHGRI', 'totalCostAmount': 399128, 'nihApplId': 8109726, 'smApplId': 408768, 'budgetStartDate': '2011-09-01T00:00:00', 'budgetEndDate': '2012-05-31T00:00:00', 'contactPi': 'SCHEET, PAUL A', 'otherPis': None, 'congressionalDistrict': '09', 'dunsId': '800772139', 'latitude': 29.706319, 'longitude': -95.397195, 'orgName': 'UNIVERSITY OF TEXAS MD ANDERSON CAN CTR', 'orgCity': 'HOUSTON', 'orgState': 'TX', 'orgCountry': 'UNITED STATES', 'orgZipCode': '770304009', 'projectStartDate': '2011-09-01T00:00:00', 'projectEndDate': '2016-05-31T00:00:00', 'cfdaCode': '172', 'abstract': \"  DESCRIPTION (provided by applicant): Massively-parallel ('next-generation') shotgun DNA sequencing projects will provide the highest resolution to date for genetic variation of human populations. This new technology offers great promise for interrogating the genetic etiology of complex disease. However, with this promise come challenges. These new sequencing methods are prone to nontrivial error rates and sparse coverage of mapped reads, confounding polymorphism discovery and genotyping. Copy number variation must often be inferred indirectly. The massive size of these data sets requires rapid and scaleable analytic approaches. In this proposal, we present statistical methods to address these challenges directly, using computationally tractable models for population genetic variation. Our methods take account of the dependence among nearby alleles (linkage disequilibrium) with a clusterbased model for haplotype variation, and utilize this information to aid inferences about the underlying genetic architecture of the samples. Specifically, we propose to call genotypes and detect novel polymorphic loci from next- generation shotgun sequence data, detect rare disease risk alleles for follow-up sequencing studies, and simultaneously model single nucleotide and copy number polymorphism in population data to facilitate studies of association between phenotype and genotype. Our experienced team of medical and statistical geneticists have the technical expertise and access to data sets necessary for achieving these aims. We will implement our methods in our widely-used software package fastPHASE.    PUBLIC HEALTH RELEVANCE: High throughput DNA sequencing technology is providing unparalleled detail of human genetic variation. This will allow finer resolution in locating disease genes that affect human health and disease. Both the large quantity and the uneven quality of this new technology demand new statistical methods for inference, risk assessment and eventually clinical translation.      \", 'terms': ' Accounting; Address; Affect; Alleles; Architecture; base; Cardiovascular system; Chromosome Deletion; Clinical; Complex; Computer software; Copy Number Polymorphism; Data; Data Set; Databases; density; Dependence; Detection; Disease; disorder risk; DNA; DNA Resequencing; DNA Sequence; Event; exome; experience; follow-up; Frequencies; Genes; Genetic; Genetic Polymorphism; Genetic Predisposition to Disease; Genetic Variation; Genome; genome wide association study; genome-wide; Genomics; Genotype; Germ Lines; Goals; Haplotypes; Health; Human; human disease; Human Genetics; improved; Individual; Joints; Linkage Disequilibrium; Loss of Heterozygosity; lung melanoma; malformation; Malignant neoplasm of lung; Maps; markov model; Medical; Medical center; Medical Genetics; Methods; Modeling; new technology; next generation; novel; Nucleotides; Pattern; Phenotype; Population; Population Genetics; Predisposition; Procedures; Process; Quality Control; Rare Diseases; Reading; Resolution; Resources; Risk; Risk Assessment; Running; Sampling; Sardinia; Shotgun Sequencing; Shotguns; Signal Transduction; Simulate; SNP genotyping; Statistical Methods; Statistical Models; Structure; Surveys; Technical Expertise; Technology; Testing; Texas; trait; Translations; tumor; Variant'}, {'projectNumber': '1R25HG006243-01', 'fy': 2011, 'title': 'ANALYZING NEXT-GENERATION SEQUENCING DATA', 'department': 'HHS', 'agency': 'NIH', 'ic': 'NHGRI', 'totalCostAmount': 54000, 'nihApplId': 8150859, 'smApplId': 428121, 'budgetStartDate': '2011-08-23T00:00:00', 'budgetEndDate': '2012-06-30T00:00:00', 'contactPi': 'BROWN, C. TITUS', 'otherPis': None, 'congressionalDistrict': '08', 'dunsId': '193247145', 'latitude': 42.748478, 'longitude': -84.483276, 'orgName': 'MICHIGAN STATE UNIVERSITY', 'orgCity': 'EAST LANSING', 'orgState': 'MI', 'orgCountry': 'UNITED STATES', 'orgZipCode': '488242600', 'projectStartDate': '2011-08-23T00:00:00', 'projectEndDate': '2013-06-30T00:00:00', 'cfdaCode': '172', 'abstract': \"  DESCRIPTION (provided by applicant): Modern biomedical research is increasingly making use of genome-scale from next-generation sequencing platforms, including Roche 454, Illumina GA2, and ABI SOLiD. These platforms make it possible for individual labs to quickly and cheaply generate vast amounts of genomic and transcriptomic data from de novo sequencing, resequencing, ChIP-seq, mRNA-seq, and allelotyping experiments. Despite this ability to generate large data sets, biomedical researchers are rarely trained in the computational and statistical techniques necessary to make sense of this data. Thus, many researchers must rely on others - often computational scientists with little biological training - to design and implement appropriate data reduction and data mining techniques. Moreover, most institutions do not have access to computational resources necessary to run these analyses. Our specific aims are to help bridge this gap in a short, two-week course, by teaching biomedical researchers to (1) run analyses on remote UNIX servers hosted in the Amazon Web Services 'cloud'; (2) perform mapping and assembly on large short-read data sets; (3) tackle specific biological problems with existing short-read data; and (4) design computational pipelines capable of addressing their own research questions. All specific aims will be accompanied by in-depth hands-on practical training in the relevant techniques. Our experience is that this practical training leads to a substantial improvement in the basic computational sophistication of participants. This short course will help train the current and next generation of independent biomedical re- searchers in basic computational thinking and procedure, as well as teaching them how to make use of scalable Internet computing resources for their own research. Our end goal is increase the efficiency and sophistication with which biomedical researchers make use of novel sequencing technologies.    PUBLIC HEALTH RELEVANCE: Many biomedical researchers are not well trained in computational tools that would  help them make use of genomic and other bioinformatics data.  We propose a two- week short course for advanced researchers that will help train to take advantage of  sequence data in their research.       \", 'terms': ' Address; Allelotyping; Automation; base; Bioinformatics; Biological; Biomedical Research; Computer software; computerized tools; computing resources; Data; Data Analyses; data mining; data reduction; Data Set; design; DNA Resequencing; Educational process of instructing; experience; Experimental Designs; Future; Gene Expression Profile; Genome; genome sequencing; Genomics; Goals; Graph; Individual; Institution; Internet; Learning; Maps; Memory; Messenger RNA; next generation; novel; Operating System; Participant; Paste substance; Procedures; programs; Reading; Recipe; Research; Research Personnel; research study; Running; Sampling; Scientist; Sequence Analysis; skills; Source; Students; System; Techniques; Technology; Thinking; tool; Training; transcriptomics; web services; Work'}, {'projectNumber': '5K99HG005846-02', 'fy': 2011, 'title': 'HIGH-THROUGHPUT MOBILE ELEMENT GENOTYPING USING NEXT-GENERATION SEQUENCING', 'department': 'HHS', 'agency': 'NIH', 'ic': 'NHGRI', 'totalCostAmount': 90000, 'nihApplId': 8151098, 'smApplId': 428232, 'budgetStartDate': '2011-07-01T00:00:00', 'budgetEndDate': '2012-03-31T00:00:00', 'contactPi': 'XING, JINCHUAN', 'otherPis': None, 'congressionalDistrict': '02', 'dunsId': '009095365', 'latitude': 40.76776, 'longitude': -111.832926, 'orgName': 'UNIVERSITY OF UTAH', 'orgCity': 'SALT LAKE CITY', 'orgState': 'UT', 'orgCountry': 'UNITED STATES', 'orgZipCode': '841128930', 'projectStartDate': '2010-09-26T00:00:00', 'projectEndDate': '2012-03-31T00:00:00', 'cfdaCode': '172', 'abstract': '  DESCRIPTION (provided by applicant):  Mobile DNA elements are DNA segments that can mobilize from one genomic location to another, some making copies of themselves during the process.  Mobile elements are ubiquitous in eukaryotic genomes, and roughly 50% of the human genome consists of mobile, repetitive DNA sequences such as Alu and LINE1 elements.  Mobile elements have profoundly influenced primate genomic evolution, and alterations caused by mobile elements have been identified in more than 100 human genetic disorders.  In addition, mobile elements are highly useful as genetic markers in tracing relationships of populations and species because of their unique manner of propagating in the genome.  Therefore, it is of great interest to understand the biology of mobile elements in human populations.  Currently available genotyping methods, however, provide only limited insight into the distribution of Alu and L1 elements in the human genome.  In this proposal, I will first describe a novel, high-throughput method to genotype a large number of individuals for Alu insertion polymorphisms.  Implementation of this method will permit, for the first time, rapid, inexpensive genotyping of whole families of mobile elements not only in humans but in any organism.  Next, I outline my plans to construct a computational pipeline that can efficiently handle and analyze the high- throughput genomic data generated by my genotyping method.  Finally I describe my plan to apply the genotyping procedure to 540 HapMap samples and use my computational pipeline to analyze the results.  Specifically, I will use the insertion polymorphism data to assess the genomic diversity associated with mobile element insertions, as well as to answer questions related to mobile element biology and to reconstruct ancient human demographic history.  In addition, the Alu insertion polymorphism dataset generated in this project will provide a valuable supplementary resource of structural variants for the HapMap and 1000 Genomes Projects.  My long-term goal is to pursue a career in human genomic variation and population genetics research with an emphasis on mobile elements.  Using a combination of computational and experimental tools, I will study the impact of mobile DNA elements on the human genome and on genetic diversity among human populations.  This award will provide the necessary training for me to become an independent investigator on two fronts:  I will acquire essential training in computational biology and bioinformatics to handle large-scale genomic data, including data generated from next-generation sequencing technology. In addition, research training offered by the institution and my mentor will allow me to acquire knowledge about research resources, grant writing, responsible conduct in research, and other information necessary to become a successful, independent investigator.    PUBLIC HEALTH RELEVANCE:  Mobile DNA elements have had profound and widespread effects on the human genome, and they have been implicated in the causation of dozens of human diseases.  This proposal will develop a low-cost, high-throughput technology that can be used to genotype mobile element insertions in humans and other organisms.  This technology will greatly enhance our understanding of mobile element biology in human populations and provide a valuable structural variant data resource for the 1000 Genomes Project.      ', 'terms': ' Alu Elements; Award; Beryllium; Bioinformatics; Biology; career; Computational Biology; cost; Data; Data Analyses; data management; Data Set; DNA; DNA Insertion Elements; DNA Sequence; Elements; Etiology; Evolution; Family; Genes; Genetic Markers; Genetic Polymorphism; Genetic Research; Genetic Variation; Genome; genome sequencing; Genomics; Genotype; Goals; Grant; Guanine + Cytosine Composition; Hereditary Disease; high throughput technology; Human; human disease; Human Genetics; Human Genome; indexing; Indium; Individual; insight; Institution; interest; Knowledge; L1 Elements; Libraries; Location; Mentors; Methods; next generation; novel; Organism; Phase; Population; Population Genetics; Primates; Procedures; Process; Property; Protocols documentation; public health relevance; Recording of previous events; Research; Research Personnel; Research Training; Resources; Sampling; Scanning; Site; Structure; Techniques; Technology; Time; tool; Training; transposon/insertion element; Variant; Writing'}, {'projectNumber': '1K08NS075094-01', 'fy': 2011, 'title': 'NEXT-GENERATION SEQUENCING APPROACHES FOR NOVEL GENE DISCOVERY IN ALS', 'department': 'HHS', 'agency': 'NIH', 'ic': 'NINDS', 'totalCostAmount': 157414, 'nihApplId': 8165082, 'smApplId': 429276, 'budgetStartDate': '2011-07-01T00:00:00', 'budgetEndDate': '2012-06-30T00:00:00', 'contactPi': 'HARMS, MATTHEW', 'otherPis': None, 'congressionalDistrict': '01', 'dunsId': '068552207', 'latitude': 38.648424, 'longitude': -90.309068, 'orgName': 'WASHINGTON UNIVERSITY', 'orgCity': 'SAINT LOUIS', 'orgState': 'MO', 'orgCountry': 'UNITED STATES', 'orgZipCode': '631304862', 'projectStartDate': '2011-07-01T00:00:00', 'projectEndDate': '2016-06-30T00:00:00', 'cfdaCode': '853', 'abstract': \"  DESCRIPTION (provided by applicant): The candidate is a clinical neurologist (MD) with a career goal of investigating the genetic and molecular basis of human neuromuscular diseases, focusing primarily on amyotrophic lateral sclerosis (ALS). The candidate has significant prior laboratory experience, with a track record of successful research projects in neuropharmacology, neurodevelopment, and most recently, neurogenetics. In order to further prepare for successful independent research, the candidate's career development plan includes graduate-level courses in biostatistics, bioinformatics, and genomics, as well as seminars and intensive courses on next-generation sequencing and related methods. The proposed development plan and scientific training will take place at Washington University in St. Louis, an institution with particular strengths in both genetics/genomics and the study of ALS. The neurogenetics community is robust and will provide the candidate with important intellectual assistance and collaborations. The scientific training will be jointly mentored by Dr. Alison Goate, whose lab focuses on the genetics of dementia and addiction, and Dr. Robert Baloh, whose lab studies the molecular basis of neurodegeneration in ALS using tissue culture and animal models. Dr. Goate's expertise in human genetics and sequencing methods coupled with Dr. Baloh's experience in modeling human neuromuscular diseases will provide the candidate with the set of research tools needed to succeed as an independent investigator studying the genetic and molecular basis of neurologic diseases. The candidate's research proposal aims to identify novel genes causing ALS, an untreatable neurodegenerative disease producing loss of motor neurons, progressive paralysis, and death from respiratory failure. Although the majority of ALS cases are sporadic, the genes responsible in familial cases have provided important insights into all forms of the disease. However, known ALS genes still only explain 25% of familial cases. To identify additional familial ALS disease genes, the candidate will: 1) Screen a large cohort of familial and sporadic ALS patients for mutations in known ALS genes. This screen will be performed using pooled-sample sequencing, an innovative method made possible by next-generation sequencing capabilities pioneered by one of our key collaborators. Novel disease-causing mutations will be followed up with phenotypic analysis. This screen will also identify a cohort of ALS families without mutations in any known ALS disease gene to be studied with whole-exome sequencing. 2) Familial cases of ALS without mutations in known genes will then undergo whole-exome analysis, a next- generation sequencing method that allows all coding regions in the human genome to be sequenced simultaneously for variations. To narrow the number of candidate pathogenic mutations, a host of bioinformatic techniques will be used. Candidate ALS genes identified will then be screened for additional mutations in other ALS patients using the same pooled-sample sequencing strategy as in (1). The identification of novel ALS disease genes will pave the way for future studies aimed at understanding the molecular mechanism by which these mutations lead to neurodegeneration, using tissue culture and animal modeling. This work will provide key insight into the molecular basis of ALS and will immediately impact efforts aimed at therapeutic development for this untreatable disease.    PUBLIC HEALTH RELEVANCE: Amyotrophic lateral sclerosis (a.k.a. Lou Gehrig's disease) is an untreatable disease characterized by progressive paralysis and eventual death. Identification of genes responsible for hereditary forms of ALS will shed light on why the disease occurs and guide how therapies might be developed. This project uses state-of- the-art genetic sequencing technologies to uncover these genes.      \", 'terms': ' addiction; Age of Onset; Algorithms; Amyotrophic Lateral Sclerosis; Animal Model; base; Bioinformatics; Biological Assay; Biometry; Brain Stem; Candidate Disease Gene; career; career development; Cessation of life; Clinical; Code; cohort; Collaborations; Communities; Coupled; Cytoplasm; Data; Dementia; Development Plans; Disease; Disease Management; disease-causing mutation; DNA; effective therapy; exome; Exons; experience; Familial Amyotrophic Lateral Sclerosis; Family; follow-up; Future; gene discovery; gene function; Gene Mutation; Genes; Genetic; Genetic Heterogeneity; genetic linkage analysis; genetic pedigree; Genetic Techniques; Genome; Genomics; Genotype; Goals; Human; Human Genetics; Human Genome; Incidence; Individual; Inherited; innovation; insight; Institution; Introns; Laboratories; Lead; Light; Mentors; Methods; Minority; Modeling; Molecular; Molecular Genetics; motor neuron degeneration; Motor Neurons; Mutation; Nerve Degeneration; Neurodegenerative Disorders; neurodevelopment; neurogenetics; Neurologic; Neurologist; Neuromuscular Diseases; Neuropharmacology; next generation; novel; novel strategies; Nuclear; palliation; Paralysed; Pathogenesis; Pathogenicity; Pathway interactions; Patients; Penetrance; proband; protein TDP-43; repository; Research; Research Personnel; Research Project Grants; Research Proposals; respiratory; Respiratory Failure; RNA Processing; RNA-Binding Proteins; Sampling; segregation; Spinal Cord; Symptoms; Techniques; Technology; Testing; therapeutic development; tissue culture; tool; Training; Universities; Variant; Washington; Work'}, {'projectNumber': '1R43HG006407-01', 'fy': 2011, 'title': 'A DESKTOP ASSEMBLY AND ANALYSIS PIPELINE FOR NEXT-GEN METAGENOMIC SEQUENCING', 'department': 'HHS', 'agency': 'NIH', 'ic': 'NHGRI', 'totalCostAmount': 152857, 'nihApplId': 8200467, 'smApplId': 440965, 'budgetStartDate': '2011-08-05T00:00:00', 'budgetEndDate': '2013-01-31T00:00:00', 'contactPi': 'DURFEE, TIMOTHY J', 'otherPis': None, 'congressionalDistrict': '02', 'dunsId': '130194947', 'latitude': 43.067896, 'longitude': -89.438617, 'orgName': 'DNASTAR INC', 'orgCity': 'MADISON', 'orgState': 'WI', 'orgCountry': 'UNITED STATES', 'orgZipCode': '537055202', 'projectStartDate': '2011-08-05T00:00:00', 'projectEndDate': '2013-01-31T00:00:00', 'cfdaCode': '172', 'abstract': \"  DESCRIPTION (provided by applicant): Over the last decade, bacterial and archaeal communities have been identified in virtually every ecological niche examined, from acid mine drainage to tropospheric clouds to the human body. Understanding the make-up and interactions within these communities is crucial in understanding how each ecosystem functions. Metagenomics (or community genomics) is the approach whereby the collective genomic content of a naturally occurring microbial community can be determined without the need to isolate and culture its constituents independently. When assembled, the genetic framework of the community, including critical information on population structure, phylogenetic diversity, as well as novel genetic and biochemical activities can be obtained. The potential impact of such findings on biotechnology, medicine and ecology are enormous.  Recently, next-gen sequencing technologies (e.g. Roche/454, Illumina, and Life Technologies (SOLiD)) have replaced traditional Sanger sequencing for metagenomic data generation. These are cost effective, clone-free, massively parallel technologies capable of producing as many as 250 gigabases of data in a single machine run. That level of sequencing makes it feasible to reconstruct even low abundance genomes as well as determine intraspecies heterogeneity from a community sample. However, the next-gen technologies also present their own computational challenges including the sheer volume of data to be processed together with the different read lengths, error models and formats unique to each technology. These complexities together with those posed the communities themselves together has left researchers to cobble together various combinations of software tools to process and analyze their data. Most software that can handle these large, complex data sets also require substantial computing resources and computer expertise beyond that of a normally equipped lab. These difficulties continue to have a serious stifling effect on the advances in science and technology that metagenomics offers.  The long term goal of this proposal is to develop a seamless, commercial-grade metagenomic sequence assembly and analysis pipeline that is fully scalable to any size project. The proposed software will be easy to use and run on a desktop computer costing less than $5000 so that any reasonably funded laboratory or clinic can exploit metagenomic technology. Toward that goal, this Phase I proposal focuses on a solution to the central task of processing massive next-gen metagenomic data sets on a desktop computer. We will evaluate whether our new non-memory bound assembly engine, XNG, can meet the challenges in three crucial steps: 1) removing reads derived from contaminating host DNA, 2) 'recruiting' the potentially hundreds of millions remaining reads into appropriate phylogenetic bins based on matches to a local reference genome database, and 3) converting genome sequences from multiple strains of a given species into a single annotated entry (the 'pan-genome') for enhanced read recruitment and downstream annotation.    PUBLIC HEALTH RELEVANCE: Human health and medicine are greatly influenced by the microbial communities that are integral parts of our bodies as well as those that produce useful antibiotics for example. To better exploit the potential of these communities, metagenomic studies are producing vast amounts of Next-gen DNA sequence data to decipher their collective genetic content. This project focuses on developing computer software capable of reconstructing microbial community genomes and analyzing their content.      \", 'terms': ' Acids; Antibiotics; base; Basic Science; Binding; Biochemical; Bioinformatics; Biotechnology; Cells; Clinic; Cloning; Communities; Complex; Computer software; Computers; computing resources; cost; cost effective; Data; Data Set; Databases; Dideoxy Chain Termination DNA Sequencing; DNA; DNA Sequence; Drainage procedure; Ecology; Ecosystem; Excision; Funding; Generations; Genes; Genetic; Genome; genome database; genome sequencing; Genomics; Goals; Gold; Health; Heterogeneity; Hour; Human; Human body; Human Genome; Human Microbiome; Imagery; Laboratories; Left; Length; Life; Marketing; Medicine; meetings; Memory; metagenomic sequencing; Metagenomics; Methodology; Methods; Microbe; microbial; microbial community; microbial genome; Mining; Modeling; next generation; novel; Organism; Performance; Phase; Phylogenetic Analysis; Play; Population; Positioning Attribute; Price; Process; Reading; Recruitment Activity; Research Personnel; Role; Running; Sampling; Science; Shotguns; Site; software development; Software Tools; Solid; Solutions; Sorting - Cell Movement; Source; Structure; Technology; Testing; Time; Uncertainty'}, {'projectNumber': '1320273', 'fy': 2013, 'title': 'AF: SMALL: ALGORITHMS FOR HAPLOTYPE ASSEMBLY FROM NEXT-GENERATION SEQUENCING DATA', 'department': 'NSF', 'agency': 'NSF', 'ic': None, 'totalCostAmount': 400000, 'nihApplId': None, 'smApplId': 470580, 'budgetStartDate': None, 'budgetEndDate': None, 'contactPi': 'VIKALO, HARIS', 'otherPis': None, 'congressionalDistrict': '25', 'dunsId': '170230239', 'latitude': 30.2917167, 'longitude': -97.7377704, 'orgName': 'UNIVERSITY OF TEXAS AUSTIN', 'orgCity': 'AUSTIN', 'orgState': 'TX', 'orgCountry': 'UNITED STATES', 'orgZipCode': '78712-1532', 'projectStartDate': '2013-09-01T00:00:00', 'projectEndDate': '2016-08-31T00:00:00', 'cfdaCode': '47.070', 'abstract': 'Humans are diploid organisms with two sets of chromosomes: 22 pairs of autosomes and one pair of sex chromosomes. The two chromosomes in a pair of autosomes are homologous, i.e., they have similar DNA sequences and essentially carry the same type of information but are not identical. The most common type of variation between chromosomes in a pair is that where the base in a specific location differs between the two sequences, i.e., the corresponding alleles on the homologous chromosomes are different. The complete information about DNA variations in an individual genome is provided by haplotypes, the list of alleles at contiguous sites in a region of a single chromosome. Haplotype information is essential for medical and pharmaceutical studies, including understanding variations in gene expressions and recombination patterns.<br/><br/>Intellectual Merit:<br/><br/>This research aims to develop and analyze novel algorithms for haplotype assembly from next-generation sequencing data. It consists of three main thrusts: (1) Haplotype assembly from next-generation sequencing data is computationally challenging. The first thrust proposes branch-and-bound algorithms that exploit certain structural features of the problem to efficiently find the exact solution. (2) As the size of the haplotype assembly problem grows, the exact solution is increasingly more difficult to obtain. The second thrust is focused on the development of fast heuristic methods with guaranteed performance bounds that enable explicit complexity-accuracy trade-offs. (3) Existing haplotype assembly schemes process DNA fragments comprising nucleotides whose order is already determined by the sequencing platform. The third thrust is focused on the development of algorithms for finding joint solution to the base-calling and haplotype assembly problems, enabling significant improvements in accuracy.<br/><br/>Broader Impact:<br/><br/>The results of this research will have a major impact on a number of fields that rely on accurate haplotype assembly, including medicine and pharmacogenomics, and will enrich the educational experience of engineering students at the University of Texas at Austin.', 'terms': ' Algorithms; Alleles; austin; autosome; base; Chromosomes; Chromosomes, Human, Pair 22; Data; Development; Diploidy; DNA; DNA Sequence; Engineering; experience; Gene Expression; Genetic Recombination; Genome; Haplotypes; heuristics; Human; Individual; Joints; Location; Medical; Medicine; Methods; next generation; novel; Nucleotides; Organism; Pattern; Performance; Pharmacogenomics; Pharmacologic Substance; Process; Research; Scheme; Sex Chromosomes; Site; Solutions; Students; Texas; Universities; Variant'}, {'projectNumber': '1248096', 'fy': 2013, 'title': 'RCN-UBE - GCAT-SEEK: THE GENOME CONSORTIUM FOR ACTIVE UNDERGRADUATE RESEARCH AND TEACHING USING NEXT-GENERATION SEQUENCING', 'department': 'NSF', 'agency': 'NSF', 'ic': None, 'totalCostAmount': 445039, 'nihApplId': None, 'smApplId': 475416, 'budgetStartDate': None, 'budgetEndDate': None, 'contactPi': 'BUONACCORSI, VINCENT P', 'otherPis': 'NEWMAN, JEFFREY; TOBIN-JANZEN, TAMMY; TRUN, NANCY; GROVE, DEBORAH', 'congressionalDistrict': '05', 'dunsId': '072845035', 'latitude': 40.5006386, 'longitude': -78.0172906, 'orgName': 'JUNIATA COLLEGE', 'orgCity': 'HUNTINGDON', 'orgState': 'PA', 'orgCountry': 'UNITED STATES', 'orgZipCode': '16652-2119', 'projectStartDate': '2013-02-01T00:00:00', 'projectEndDate': '2018-01-31T00:00:00', 'cfdaCode': '47.074', 'abstract': 'The focus of this RCN-UBE project is to support and expand participation in the newly formed Genome Consortium for Active Teaching using Next-generation Sequencing (GCAT-SEEK).  Genomics and bioinformatics are dynamic fields well-suited for capturing the imagination of undergraduates in both research laboratories and classrooms.  However, high costs, demanding technical skills, and high rates of change inhibit integration of next-generation sequencing approaches into the undergraduate curriculum.  GCAT-SEEK aims to generate and use massively parallel sequencing data associated with the research interests of network members as the catalyst for producing innovative and broadly disseminated educational modules that offer authentic research experiences to students.  The goals of this project are to increase network participation, produce and disseminate educational modules that are assessed for student learning gains, foster a sense of community among network faculty and students, and support network communications.  Hands-on faculty/student workshops centered on participant research projects are the key implementation strategy. <br/> <br/>Anticipated outcomes include improved faculty expertise, increased authentic student research experiences for undergraduates, and production of innovative, effective educational modules that improve student preparation for graduate, technical and research careers. We anticipate impacting thousands of students via this project.  Workshops will be moved around the country to include three minority serving host institutions to bolster participation by groups under-represented in STEM disciplines.<br/> <br/>This project is being jointly funded by the Directorate for Biological Sciences and the Directorate for Education and Human Resources, Division of Undergraduate Education as part of their efforts towards support of Vision and Change in Undergraduate Biology Education.', 'terms': ' Bioinformatics; Biological Sciences; Biology; career; catalyst; Communication; Communities; cost; Country; Data; Discipline; Education; Educational Curriculum; Educational process of instructing; Educational workshop; experience; Faculty; Fostering; Funding; GCAT; Genome; Genomics; Goals; Hand; Human Resources; Imagination; improved; innovation; Institution; interest; Laboratory Research; Learning; member; Minority; next generation; Outcome; Participant; Preparation; Production; Research; Research Project Grants; skills; Students; Vision'}, {'projectNumber': '4R44HG006578-02', 'fy': 2013, 'title': 'ASSOCIATION ANALYSIS SOFTWARE FOR MINING CLINICAL NEXT-GEN SEQUENCING DATA', 'department': 'HHS', 'agency': 'NIH', 'ic': 'NHGRI', 'totalCostAmount': 485252, 'nihApplId': 8624982, 'smApplId': 477420, 'budgetStartDate': '2013-07-18T00:00:00', 'budgetEndDate': '2014-06-30T00:00:00', 'contactPi': 'DURFEE, TIMOTHY J', 'otherPis': None, 'congressionalDistrict': '02', 'dunsId': '130194947', 'latitude': 43.067896, 'longitude': -89.438617, 'orgName': 'DNASTAR INC', 'orgCity': 'MADISON', 'orgState': 'WI', 'orgCountry': 'UNITED STATES', 'orgZipCode': '537055202', 'projectStartDate': '2012-09-25T00:00:00', 'projectEndDate': '2015-06-30T00:00:00', 'cfdaCode': '172', 'abstract': \"   DESCRIPTION (provided by applicant): Remarkable improvements in throughput, accuracy and cost-effectiveness of next-generation sequencing (next-gen) technologies are ushering in a new era of clinical medicine. Genome wide association studies (GWAS) in particular have begun to leverage these advances to determine the complete catalog of common and rare variants for each member of a cohort. The resolving power of this approach has the potential to greatly accelerate our understanding, diagnosis and treatment of human disease. Unfortunately, analysis of these massive data sets requires that several disparate pieces of software be cobbled together including a large capacity next-gen sequencing assembler, variation detection modules, mapping and comparison tools for tens to hundreds of variant reports, statistical analysis packages, reporting tools, and so on. Combining and using these tools typically requires extensive bioinformatic expertise as the software is rarely well documented or supported and often depends on having elaborate hardware. These hurdles makes next-gen based GWAS inaccessible to the vast majority of the crucial user base, the physician researchers.  The goal of this proposal is to assemble the essential next-gen based GWAS software components into a single coherent pipeline that that is fully equipped to meet the needs of the medical research community. Consistent with DNASTAR's 28 year tradition, the software will be easy to use, run on a reasonably priced (<$3000) desktop computer, and will be fully documented and supported. The pipeline will consist of two modules already available through DNASTAR, SeqMan NGen 3.0 (SM NGen 3.0) and ArrayStar. SM NGen 3.0, our recently released human genome scale assembly and analysis package, forms the front end of pipeline. Reference-guided assemblies of whole human genome or exome next-gen data sets produce variation reports including impact on gene features and associations with the dbSNP database. Putative variations can be verified by direct inspection of the alignment through the SeqMan Pro component of the package. Variation reports from each member of a GWAS cohort will then be fed into our multi-sample comparison and analysis program, ArrayStar, at the back end of the pipeline. ArrayStar has the infrastructure for multi-sample management and processing which can be easily adapted to GWAS analysis. These adaptations and their documentation are a central focus of this application.  Critical to the successful development of this software is our collaboration with Dr. Douglas McNeel (Dept. of Oncology, UW-Madison). The exomes from a panel of prostate cancer vaccine recipients, including responders and non-responders, from the McNeel lab will be sequenced as input from which to build the pipeline using iterative cycles of development followed by evaluation by the McNeel group. This relationship offers an ideal opportunity to build the analysis and reporting software needed by physician researchers to form, test and validate GWAS generated hypotheses.      \", 'terms': ' Automatic Data Processing; Back; base; Bioinformatics; Cataloging; Catalogs; Cereals; Clinical; Clinical Medicine; Clinical Research; clinically relevant; cohort; Collaborations; Communities; Computer software; Computers; cost effectiveness; Data; Data Set; Databases; Detection; Development; Diagnosis; Diagnostics Research; Disease; DNA Sequence; Documentation; Ensure; Evaluation; exome; exome sequencing; feeding; flexibility; Foundations; Generations; Genes; Genetic; Genome; genome wide association study; genome-wide; Goals; Hour; Housing; human disease; Human Genome; Individual; insight; Laboratories; Life; Maps; Marketing; Medical; Medical Research; meetings; member; Metadata; Mining; Modification; next generation; next generation sequencing; oncology; Performance; Phase; Physicians; Population; Price; Process; programs; Prostate Cancer Vaccine; Reporting; Research Infrastructure; Research Personnel; response; Running; Sample Size; Sampling; Scientist; Sequence Analysis; software development; Solid; Solutions; Suggestion; Technology; Testing; tool; Treatment Efficacy; Vaccines; Variant'}, {'projectNumber': '5R00HG005846-04', 'fy': 2013, 'title': 'HIGH-THROUGHPUT MOBILE ELEMENT GENOTYPING USING NEXT-GENERATION SEQUENCING', 'department': 'HHS', 'agency': 'NIH', 'ic': 'NHGRI', 'totalCostAmount': 245540, 'nihApplId': 8449671, 'smApplId': 481196, 'budgetStartDate': '2013-03-01T00:00:00', 'budgetEndDate': '2014-02-28T00:00:00', 'contactPi': 'XING, JINCHUAN', 'otherPis': None, 'congressionalDistrict': '06', 'dunsId': '001912864', 'latitude': 40.481906, 'longitude': -74.457066, 'orgName': 'RUTGERS THE ST UNIV OF NJ NEW BRUNSWICK', 'orgCity': 'PISCATAWAY', 'orgState': 'NJ', 'orgCountry': 'UNITED STATES', 'orgZipCode': '088543925', 'projectStartDate': '2010-09-26T00:00:00', 'projectEndDate': '2015-02-28T00:00:00', 'cfdaCode': '172', 'abstract': 'Project Summary/Abstract Mobile DNA elements are DNA segments that can mobilize from one genomic location to another,some making copies of themselves during the process. Mobile elements are ubiquitous in eukaryotic genomes,and roughly 50% of the human genome consists of mobile, repetitive DNA sequences such as Alu and LINE1elements. Mobile elements have profoundly influenced primate genomic evolution, and alterations caused bymobile elements have been identified in more than 100 human genetic disorders. In addition, mobile elementsare highly useful as genetic markers in tracing relationships of populations and species because of their uniquemanner of propagating in the genome. Therefore, it is of great interest to understand the biology of mobileelements in human populations. Currently available genotyping methods, however, provide only limited insightinto the distribution of Alu and L1 elements in the human genome. In this proposal, I will first describe a novel, high-throughput method to genotype a large number ofindividuals for Alu insertion polymorphisms. Implementation of this method will permit, for the first time, rapid,inexpensive genotyping of whole families of mobile elements not only in humans but in any organism. Next, Ioutline my plans to construct a computational pipeline that can efficiently handle and analyze the high-throughput genomic data generated by my genotyping method. Finally I describe my plan to apply thegenotyping procedure to 540 HapMap samples and use my computational pipeline to analyze the results.Specifically, I will use the insertion polymorphism data to assess the genomic diversity associated with mobileelement insertions, as well as to answer questions related to mobile element biology and to reconstruct ancienthuman demographic history. In addition, the Alu insertion polymorphism dataset generated in this project willprovide a valuable supplementary resource of structural variants for the HapMap and 1000 Genomes Projects. My long-term goal is to pursue a career in human genomic variation and population genetics researchwith an emphasis on mobile elements. Using a combination of computational and experimental tools, Iwill study the impact of mobile DNA elements on the human genome and on genetic diversity among humanpopulations. This award will provide the necessary training for me to become an independent investigator ontwo fronts: I will acquire essential training in computational biology and bioinformatics to handle large-scalegenomic data, including data generated from next-generation sequencing technology. In addition, researchtraining offered by the institution and my mentor will allow me to acquire knowledge about research resources,grant writing, responsible conduct in research, and other information necessary to become a successful,independent investigator.', 'terms': ' abstracting; Alu Elements; Award; Beryllium; Bioinformatics; Biology; career; Computational Biology; cost; Data; Data Analyses; data management; Data Set; DNA; DNA Insertion Elements; DNA Sequence; Elements; Etiology; Evolution; Family; Genes; Genetic Markers; Genetic Polymorphism; Genetic Research; Genetic Variation; Genome; genome sequencing; Genomics; Genotype; Goals; Grant; Guanine + Cytosine Composition; Hereditary Disease; high throughput technology; Human; human disease; Human Genetics; Human Genome; indexing; Indium; Individual; insight; Institution; interest; Knowledge; L1 Elements; Libraries; Location; Mentors; Methods; next generation sequencing; novel; Organism; Phase; Population; Population Genetics; Primates; Procedures; Process; Property; Protocols documentation; Recording of previous events; Research; Research Personnel; Research Training; Resources; Sampling; Scanning; Site; Structure; Techniques; Technology; Time; tool; Training; transposon/insertion element; Variant; Writing'}, {'projectNumber': '3U01MH094432-02S1', 'fy': 2013, 'title': '2/4-PSYCHIATRIC GWAS CONSORTIUM: GENOMIC FOLLOW-UP NEXT-GEN SEQUENCING & GENOTYPI', 'department': 'HHS', 'agency': 'NIH', 'ic': 'NIMH', 'totalCostAmount': 109996, 'nihApplId': 8663986, 'smApplId': 481448, 'budgetStartDate': '2013-06-01T00:00:00', 'budgetEndDate': '2014-03-31T00:00:00', 'contactPi': 'DALY, MARK JOSEPH', 'otherPis': None, 'congressionalDistrict': '08', 'dunsId': '073130411', 'latitude': 42.347463, 'longitude': -71.082039, 'orgName': 'MASSACHUSETTS GENERAL HOSPITAL', 'orgCity': 'BOSTON', 'orgState': 'MA', 'orgCountry': 'UNITED STATES', 'orgZipCode': '021142696', 'projectStartDate': '2012-05-10T00:00:00', 'projectEndDate': '2016-03-31T00:00:00', 'cfdaCode': '242', 'abstract': \"  DESCRIPTION (provided by applicant): This is the final submission of the collaborative R01 MH094421 Psychiatric GWAS Consortium: Genomic Follow-Up Next-Gen Sequencing & Genotyping. The overall goal of this application is ambitious: we aim to generate a trustworthy, high-confidence 'maps' of the genetic architecture of centrally important psychiatric diseases. Such maps consist of systematic and comprehensive evaluation of the allelic spectrum for these disorders including rare exonic, common SNP, and copy number variation. The feasibility of our aims is supported by the track record of the first iteration of the Psychiatric GWAS Consortium ('PGC1') where we united nearly all major groups in the field into a harmonious and functional entity and completed our initial aims to a high scientific standard. Psychiatric diseases are compelling targets for intensive research: they are mostly idiopathic, first-rank public health problems, and cause enormous morbidity, mortality, and personal/societal cost. Consistent with the NIH mission, our goal is to elucidate fundamental knowledge of these diseases. In this 'PGC2' application, we propose to capitalize on prior NIH investments and on the success of PGC1 for the next logical set of aims. The PGC2 aims are 'large-scale' (largest sample sizes ever in the field) and 'comprehensive' (via the careful application of multiple genomic-scale technologies). Our focus is comprehensive in other senses - the PGC encompasses nearly the entire field, and we aim to elucidate the allelic spectrum of these disorders by integrating empirical data for all readily measurable types of genetic variation of etiological relevance (common SNP, rare exonic, and rare and common copy number variation - what we term the 'map'). There are three analytic aims - systematically to assess copy number variation, to create a pipeline for the analysis of exome data (and eventually whole-genome data), and to investigate genetic associations that span traditional disease boundaries. Finally, we propose to develop the 'PsychChip' a custom 20,000 probe array targeting common SNP, exonic, and CNVs that would then be used to genotype 115,082 subjects. The PGC2 impact is potentially very large - a fundamental understanding of the genetics of these diseases would be a major milestone in psychiatry and in biomedicine.    \", 'terms': ' Architecture; Attention deficit hyperactivity disorder; Autistic Disorder; Bioinformatics; Biological; Bipolar Disorder; Budgets; Chromosome Mapping; Comorbidity; Copy Number Polymorphism; cost; Country; Custom; Data; Disease; Disease Association; Epidemiology; Evaluation; exome; follow-up; Funding; Genetic; genetic association; genetic technology; genetic variant; Genetic Variation; Genome; genome sequencing; genome wide association study; Genomics; Genotype; Goals; Grant; Hereditary Disease; Human; Individual; Institution; Intention; Investments; Knowledge; Learning; Major Depressive Disorder; Maps; Measurable; Mental disorders; Mission; Morbidity - disease rate; mortality; National Institute of Mental Health; next generation sequencing; Phenotype; Process; Psychiatry; psychogenetics; Public Health; public-private partnership; Recording of previous events; Reporting; Research; research study; Risk; Role; Sample Size; Sampling; Schizophrenia; Scientist; Sequence Analysis; Societies; Source; success; Symptoms; Technology; Testing; Time; Trust; United States National Institutes of Health; Validation; Variant; Work; working group'}, {'projectNumber': '3U01MH094411-02S1', 'fy': 2013, 'title': '3/4-PSYCHIATRIC GWAS CONSORTIUM: GENOMIC FOLLOW-UP NEXT-GEN SEQUENCING & GENOTYPI', 'department': 'HHS', 'agency': 'NIH', 'ic': 'NIMH', 'totalCostAmount': 143835, 'nihApplId': 8664514, 'smApplId': 485996, 'budgetStartDate': '2013-06-15T00:00:00', 'budgetEndDate': '2014-03-31T00:00:00', 'contactPi': 'SEBAT, JONATHAN', 'otherPis': None, 'congressionalDistrict': '49', 'dunsId': '804355790', 'latitude': 37.240049, 'longitude': -119.503236, 'orgName': 'UNIVERSITY OF CALIFORNIA SAN DIEGO', 'orgCity': 'LA JOLLA', 'orgState': 'CA', 'orgCountry': 'UNITED STATES', 'orgZipCode': '920930934', 'projectStartDate': '2012-05-10T00:00:00', 'projectEndDate': '2016-03-31T00:00:00', 'cfdaCode': '242', 'abstract': \"  DESCRIPTION (provided by applicant): This is the final submission of the collaborative R01 MH094421 Psychiatric GWAS Consortium: Genomic Follow-Up Next-Gen Sequencing & Genotyping. The overall goal of this application is ambitious: we aim to generate a trustworthy, high-confidence 'maps' of the genetic architecture of centrally important psychiatric diseases. Such maps consist of systematic and comprehensive evaluation of the allelic spectrum for these disorders including rare exonic, common SNP, and copy number variation. The feasibility of our aims is supported by the track record of the first iteration of the Psychiatric GWAS Consortium ('PGC1') where we united nearly all major groups in the field into a harmonious and functional entity and completed our initial aims to a high scientific standard. Psychiatric diseases are compelling targets for intensive research: they are mostly idiopathic, first-rank public health problems, and cause enormous morbidity, mortality, and personal/societal cost. Consistent with the NIH mission, our goal is to elucidate fundamental knowledge of these diseases. In this 'PGC2' application, we propose to capitalize on prior NIH investments and on the success of PGC1 for the next logical set of aims. The PGC2 aims are 'large-scale' (largest sample sizes ever in the field) and 'comprehensive' (via the careful application of multiple genomic-scale technologies). Our focus is comprehensive in other senses - the PGC encompasses nearly the entire field, and we aim to elucidate the allelic spectrum of these disorders by integrating empirical data for all readily measurable types of genetic variation of etiological relevance (common SNP, rare exonic, and rare and common copy number variation - what we term the 'map'). There are three analytic aims - systematically to assess copy number variation, to create a pipeline for the analysis of exome data (and eventually whole-genome data), and to investigate genetic associations that span traditional disease boundaries. Finally, we propose to develop the 'PsychChip' a custom 20,000 probe array targeting common SNP, exonic, and CNVs that would then be used to genotype 115,082 subjects. The PGC2 impact is potentially very large - a fundamental understanding of the genetics of these diseases would be a major milestone in psychiatry and in biomedicine.    \", 'terms': ' Architecture; Attention deficit hyperactivity disorder; Autistic Disorder; Bioinformatics; Biological; Bipolar Disorder; Budgets; Chromosome Mapping; Comorbidity; Copy Number Polymorphism; cost; Country; Custom; Data; Disease; Disease Association; Epidemiology; Evaluation; exome; follow-up; Funding; Genetic; genetic association; genetic technology; genetic variant; Genetic Variation; Genome; genome sequencing; genome wide association study; Genomics; Genotype; Goals; Grant; Hereditary Disease; Human; Individual; Institution; Intention; Investments; Knowledge; Learning; Major Depressive Disorder; Maps; Measurable; Mental disorders; Mission; Morbidity - disease rate; mortality; National Institute of Mental Health; next generation sequencing; Phenotype; Process; Psychiatry; psychogenetics; Public Health; public-private partnership; Recording of previous events; Reporting; Research; research study; Risk; Role; Sample Size; Sampling; Schizophrenia; Scientist; Sequence Analysis; Societies; Source; success; Symptoms; Technology; Testing; Time; Trust; United States National Institutes of Health; Validation; Variant; Work; working group'}, {'projectNumber': '5R25HG006110-03', 'fy': 2013, 'title': 'SHORT COURSE ON NEXT-GENERATION SEQUENCING: TECHNOLOGY AND STATISTICAL METHODS', 'department': 'HHS', 'agency': 'NIH', 'ic': 'NHGRI', 'totalCostAmount': 52471, 'nihApplId': 8447107, 'smApplId': 488737, 'budgetStartDate': '2013-02-01T00:00:00', 'budgetEndDate': '2014-05-07T00:00:00', 'contactPi': 'TIWARI, HEMANT K.', 'otherPis': None, 'congressionalDistrict': '07', 'dunsId': '063690705', 'latitude': 32.635016, 'longitude': -86.809379, 'orgName': 'UNIVERSITY OF ALABAMA AT BIRMINGHAM', 'orgCity': 'BIRMINGHAM', 'orgState': 'AL', 'orgCountry': 'UNITED STATES', 'orgZipCode': '352940001', 'projectStartDate': '2011-04-27T00:00:00', 'projectEndDate': '2014-05-07T00:00:00', 'cfdaCode': '172', 'abstract': \"  DESCRIPTION (provided by applicant): Major efforts are under way to uncover the specific genetic components of many complex human disorders and quantitative traits. These efforts include rapidly evolving technologies for next- generation sequencing that will generate an unprecedented wealth of data. In that turn that will require sophisticated statistical analysis in order to maximize the information gleaned from each study. Likewise, many advances have been made in statistical methods for the study of complex genetic traits, and new statistical methods continue to be promulgated. To expedite and significantly advance the search for specific genes that predispose to these complex traits, we propose a series of annual short courses on next-generation sequencing technologies and statistical genomics which will enable a far greater number of researchers, including clinical researchers, to participate in, contribute to, and lead such research. These short courses will help equip students, post-doctoral fellows, junior investigators and senior investigators to expedite genomic discovery. The courses will be taught by leading experts in next-generation sequencing and statistical genetics/genomics. Each 4-day course will provide substantial expertise both in technologies for next- generation sequencing and in statistical methods pertaining to these technologies that will effectively increase the number and the expertise of investigators who are pursuing genetic and genomic research. This is a unique course containing instruction in both technological and statistical aspects of next-generation sequencing. There is currently no course of this type offered in US. We expect approximately 50 students to attend each course, allowing for intensive interaction between students and faculty. Lectures will be supplemented with extensive discussion sessions, handouts, and presentation of worked examples.    PUBLIC HEALTH RELEVANCE: We propose an annual short course each year for 3 years on next-generation sequencing and analysis methods for investigators studying complex traits. Taught by leading experts in next-generation sequencing technology and statistical genomics, these courses will be aimed at statisticians and non- statistician investigators who wish to learn both the rapidly evolving technology and 'language' of statistical genomics. Attendees should be either current investigators or in the early stages of a career in genetic/genomic research.      \", 'terms': ' Alabama; Applications Grants; Bioinformatics; Biotechnology; career; Characteristics; Clinical; Communities; Complex; Complex Genetic Trait; Consultations; Data; Data Analyses; Development; disability; Disease; Education; education evaluation; Educational process of instructing; Evaluation; experience; Faculty; Fees; Funding; Genetic; Genomics; Glean; graduate student; Human; Institutes; Instruction; Laboratories; Language; Lead; Learning; lectures; Location; meetings; Methods; Modification; Molecular; next generation sequencing; Participant; Postdoctoral Fellow; programs; public health relevance; Request for Proposals; Research; Research Personnel; Resources; responsible research conduct; Selection Criteria; Sequence Analysis; Series; Staging; Statistical Methods; Students; Susceptibility Gene; Technology; Training; trait; Universities; web site; Work'}]\n"
     ]
    }
   ],
   "source": [
    "response = requests.get('https://api.federalreporter.nih.gov/v1/projects/search?query=text:next-gen+sequencing').json()\n",
    "print((response['items']))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 39,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style>\n",
       "    .dataframe thead tr:only-child th {\n",
       "        text-align: right;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: left;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>keyterm</th>\n",
       "      <th>client_list_frequency</th>\n",
       "      <th>NIH_result_output</th>\n",
       "      <th>url</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>next-gen sequencing</td>\n",
       "      <td>31</td>\n",
       "      <td>1778</td>\n",
       "      <td>https://api.federalreporter.nih.gov/v1/project...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>RNA sequencing</td>\n",
       "      <td>14</td>\n",
       "      <td>6953</td>\n",
       "      <td>https://api.federalreporter.nih.gov/v1/project...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>RNAseq</td>\n",
       "      <td>17</td>\n",
       "      <td>4369</td>\n",
       "      <td>https://api.federalreporter.nih.gov/v1/project...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>RNA seq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://api.federalreporter.nih.gov/v1/project...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>ribonucleic acid sequencing</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://api.federalreporter.nih.gov/v1/project...</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                       keyterm client_list_frequency NIH_result_output  \\\n",
       "0          next-gen sequencing                    31              1778   \n",
       "1               RNA sequencing                    14              6953   \n",
       "2                       RNAseq                    17              4369   \n",
       "3                      RNA seq                   NaN               NaN   \n",
       "4  ribonucleic acid sequencing                   NaN               NaN   \n",
       "\n",
       "                                                 url  \n",
       "0  https://api.federalreporter.nih.gov/v1/project...  \n",
       "1  https://api.federalreporter.nih.gov/v1/project...  \n",
       "2  https://api.federalreporter.nih.gov/v1/project...  \n",
       "3  https://api.federalreporter.nih.gov/v1/project...  \n",
       "4  https://api.federalreporter.nih.gov/v1/project...  "
      ]
     },
     "execution_count": 39,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "key_terms_df['url'] = url_list\n",
    "key_terms_df.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 40,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "IOPub data rate exceeded.\n",
      "The notebook server will temporarily stop sending output\n",
      "to the client in order to avoid crashing it.\n",
      "To change this limit, set the config variable\n",
      "`--NotebookApp.iopub_data_rate_limit`.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "0\n",
      "300\n",
      "https://api.federalreporter.nih.gov/v1/projects/search?query=text:mtDNAseq\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "IOPub data rate exceeded.\n",
      "The notebook server will temporarily stop sending output\n",
      "to the client in order to avoid crashing it.\n",
      "To change this limit, set the config variable\n",
      "`--NotebookApp.iopub_data_rate_limit`.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "1\n",
      "[{'projectNumber': '1U24CA224309-01', 'fy': 2017, 'title': 'IMMUNE MONITORING AND ANALYSIS OF CANCER AT STANFORD (IMACS)', 'department': 'HHS', 'agency': 'NIH', 'ic': 'NCI', 'totalCostAmount': 12560000, 'nihApplId': 9456826, 'smApplId': 965720, 'budgetStartDate': '2017-09-30T00:00:00', 'budgetEndDate': '2022-08-31T00:00:00', 'contactPi': 'MAECKER, HOLDEN T.', 'otherPis': 'BENDALL, SEAN CURTIS ; ', 'congressionalDistrict': '18', 'dunsId': '009214214', 'latitude': 37.426852, 'longitude': -122.17047, 'orgName': 'STANFORD UNIVERSITY', 'orgCity': 'STANFORD', 'orgState': 'CA', 'orgCountry': 'UNITED STATES', 'orgZipCode': '943041222', 'projectStartDate': '2017-09-30T00:00:00', 'projectEndDate': '2022-08-31T00:00:00', 'cfdaCode': '353', 'abstract': '\\xa0IMMUNE\\xa0MONITORING\\xa0AND\\xa0ANALYSIS\\xa0OF\\xa0CANCER\\xa0AT\\xa0STANFORD\\xa0(IMACS)\\xa0\\xa0Abstract\\xa0The\\xa0 Center\\xa0 for\\xa0 Immune\\xa0 Monitoring\\xa0 and\\xa0 Analysis\\xa0 of\\xa0 Cancer\\xa0 at\\xa0 Stanford\\xa0 (IMACS)\\xa0 will\\xa0 perform\\xa0 highly\\xa0comprehensive\\xa0assays\\xa0of\\xa0immune\\xa0phenotype\\xa0and\\xa0function\\xa0for\\xa0NCI-\\xadidentified\\xa0clinical\\xa0trials.\\xa0\\xa0These\\xa0will\\xa0include\\xa0standardized\\xa0 assays\\xa0 already\\xa0 developed\\xa0 on\\xa0 CyTOF,\\xa0 high-\\xaddimensional\\xa0 flow\\xa0 cytometry,\\xa0 Luminex,\\xa0 TCRseq,\\xa0 and\\xa0RNAseq\\xa0 platforms.\\xa0 \\xa0 As\\xa0 part\\xa0 of\\xa0 the\\xa0 program\\xa0 we\\xa0 will\\xa0 also\\xa0 standardize\\xa0 and\\xa0 offer\\xa0 as\\xa0 assays\\xa0 Stanford-\\xadinvented\\xa0technologies\\xa0under\\xa0development,\\xa0including\\xa0Multiplexed\\xa0Ion\\xa0Beam\\xa0Imaging\\xa0(MIBI)\\xa0and\\xa0Assays\\xa0of\\xa0Transposon-\\xadAccessible\\xa0Chromatin\\xa0(ATAC-\\xadseq).\\xa0\\xa0We\\xa0have\\xa0designed\\xa0our\\xa0center\\xa0structure\\xa0to\\xa0work\\xa0with\\xa0investigators\\xa0to\\xa0define\\xa0the\\xa0assays\\xa0best\\xa0suited\\xa0to\\xa0the\\xa0immunological\\xa0questions\\xa0being\\xa0posed,\\xa0and\\xa0match\\xa0these\\xa0with\\xa0the\\xa0required\\xa0sample\\xa0types.\\xa0\\xa0We\\xa0will\\xa0perform\\xa0quality\\xa0control\\xa0measures\\xa0on\\xa0all\\xa0assays,\\xa0as\\xa0well\\xa0as\\xa0generate\\xa0a\\xa0standard\\xa0report\\xa0for\\xa0each\\xa0assay\\xa0 and\\xa0 project.\\xa0 \\xa0 Data\\xa0 will\\xa0 be\\xa0 organized\\xa0 via\\xa0 our\\xa0 online\\xa0 database,\\xa0 Stanford\\xa0 Data\\xa0 Miner,\\xa0 to\\xa0 ensure\\xa0 data\\xa0longevity\\xa0and\\xa0transferability,\\xa0as\\xa0well\\xa0as\\xa0access\\xa0to\\xa0both\\xa0raw\\xa0data\\xa0files\\xa0and\\xa0analyzed\\xa0results.\\xa0\\xa0Finally,\\xa0we\\xa0will\\xa0work\\xa0with\\xa0investigators\\xa0on\\xa0novel\\xa0bioinformatics\\xa0approaches\\xa0to\\xa0mining\\xa0these\\xa0high-\\xaddimensional\\xa0data\\xa0sets.\\xa0\\xa0These\\xa0will\\xa0include\\xa0 approaches\\xa0 designed\\xa0 for\\xa0 a\\xa0 single\\xa0 data\\xa0 type\\xa0 (e.g.,\\xa0 viSNE\\xa0 and\\xa0 Citrus\\xa0 for\\xa0 CyTOF\\xa0 and\\xa0 flow\\xa0 cytometry\\xa0data),\\xa0as\\xa0well\\xa0as\\xa0approaches\\xa0for\\xa0integrating\\xa0data\\xa0across\\xa0assays,\\xa0using\\xa0appropriate\\xa0machine\\xa0learning\\xa0algorithms\\xa0to\\xa0aid\\xa0NCI\\xa0researchers\\xa0in\\xa0identifying\\xa0immune\\xa0hallmarks\\xa0central\\xa0to\\xa0their\\xa0trials.\\xa0\\xa0Relevance:\\xa0\\xa0The\\xa0IMACS\\xa0center\\xa0will\\xa0provide\\xa0access\\xa0to\\xa0a\\xa0suite\\xa0of\\xa0state-\\xadof-\\xadthe-\\xadart\\xa0immune\\xa0assays,\\xa0many\\xa0of\\xa0them\\xa0developed\\xa0 or\\xa0 refined\\xa0 at\\xa0 Stanford.\\xa0 \\xa0 This\\xa0 unmatched\\xa0 set\\xa0 of\\xa0 technologies\\xa0 will\\xa0 facilitate\\xa0 the\\xa0 discovery\\xa0 of\\xa0 new\\xa0biomarkers\\xa0 for\\xa0 predicting\\xa0 cancer\\xa0 outcome\\xa0 or\\xa0 therapeutic\\xa0 response,\\xa0 as\\xa0 well\\xa0 as\\xa0 defining\\xa0 potential\\xa0 new\\xa0mechanisms\\xa0of\\xa0immune\\xa0control\\xa0of\\xa0cancer.\\xa0\\xa0\\xa0\\xa0\\xa0', 'terms': ' Adaptive Immune System; Algorithms; Alpha Cell; Archives; Bioinformatics; Biological Assay; cancer clinical trial; Cancer Control; Cells; Chromatin; Citrus; Clinical; Clinical Trials; Complex; Custom; cytokine; Data; Data Files; Data Set; Databases; design; Development; Disease; Ensure; Epigenetic Process; experience; Flow Cytometry; Genomics; Health; high dimensionality; Human; Image; Image Reconstructions; Immune; Immunologic Monitoring; Immunologics; Immunology procedure; Informatics; innovation; interest; Longevity; Machine Learning; Malignant Neoplasms; Measures; Methods; Mining; Multiplexed Ion Beam Imaging; Nature; Network-based; new technology; next generation; novel; operation; Outcome; Phenotype; Population; predictive marker; programs; Quality Control; Reporting; Research; Research Personnel; response; Sampling; Specimen; Standardization; Structure; Technology; Tissues; transcriptome sequencing; treatment response; Work'}]\n",
      "551\n",
      "https://api.federalreporter.nih.gov/v1/projects/search?query=text:TCRseq\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "IOPub data rate exceeded.\n",
      "The notebook server will temporarily stop sending output\n",
      "to the client in order to avoid crashing it.\n",
      "To change this limit, set the config variable\n",
      "`--NotebookApp.iopub_data_rate_limit`.\n"
     ]
    },
    {
     "ename": "KeyboardInterrupt",
     "evalue": "",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mTypeError\u001b[0m                                 Traceback (most recent call last)",
      "\u001b[0;32m~/anaconda3/lib/python3.6/site-packages/urllib3/connectionpool.py\u001b[0m in \u001b[0;36m_make_request\u001b[0;34m(self, conn, method, url, timeout, chunked, **httplib_request_kw)\u001b[0m\n\u001b[1;32m    379\u001b[0m             \u001b[0;32mtry\u001b[0m\u001b[0;34m:\u001b[0m  \u001b[0;31m# Python 2.7, use buffering of HTTP responses\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 380\u001b[0;31m                 \u001b[0mhttplib_response\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mconn\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mgetresponse\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mbuffering\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0;32mTrue\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    381\u001b[0m             \u001b[0;32mexcept\u001b[0m \u001b[0mTypeError\u001b[0m\u001b[0;34m:\u001b[0m  \u001b[0;31m# Python 2.6 and older, Python 3\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;31mTypeError\u001b[0m: getresponse() got an unexpected keyword argument 'buffering'",
      "\nDuring handling of the above exception, another exception occurred:\n",
      "\u001b[0;31mKeyboardInterrupt\u001b[0m                         Traceback (most recent call last)",
      "\u001b[0;32m<ipython-input-40-2af83e7a75d0>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[1;32m     27\u001b[0m \u001b[0;32mfor\u001b[0m \u001b[0murl\u001b[0m \u001b[0;32min\u001b[0m \u001b[0murl_list\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     28\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m---> 29\u001b[0;31m     \u001b[0mresponse\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mrequests\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mget\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0murl\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mjson\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m     30\u001b[0m     \u001b[0mprint\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mlen\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mresponse\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0;34m'items'\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     31\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;32m~/anaconda3/lib/python3.6/site-packages/requests/api.py\u001b[0m in \u001b[0;36mget\u001b[0;34m(url, params, **kwargs)\u001b[0m\n\u001b[1;32m     70\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     71\u001b[0m     \u001b[0mkwargs\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0msetdefault\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m'allow_redirects'\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;32mTrue\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m---> 72\u001b[0;31m     \u001b[0;32mreturn\u001b[0m \u001b[0mrequest\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m'get'\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0murl\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mparams\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0mparams\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;34m**\u001b[0m\u001b[0mkwargs\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m     73\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     74\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;32m~/anaconda3/lib/python3.6/site-packages/requests/api.py\u001b[0m in \u001b[0;36mrequest\u001b[0;34m(method, url, **kwargs)\u001b[0m\n\u001b[1;32m     56\u001b[0m     \u001b[0;31m# cases, and look like a memory leak in others.\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     57\u001b[0m     \u001b[0;32mwith\u001b[0m \u001b[0msessions\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mSession\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m \u001b[0;32mas\u001b[0m \u001b[0msession\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m---> 58\u001b[0;31m         \u001b[0;32mreturn\u001b[0m \u001b[0msession\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mrequest\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mmethod\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0mmethod\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0murl\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0murl\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;34m**\u001b[0m\u001b[0mkwargs\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m     59\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     60\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;32m~/anaconda3/lib/python3.6/site-packages/requests/sessions.py\u001b[0m in \u001b[0;36mrequest\u001b[0;34m(self, method, url, params, data, headers, cookies, files, auth, timeout, allow_redirects, proxies, hooks, stream, verify, cert, json)\u001b[0m\n\u001b[1;32m    506\u001b[0m         }\n\u001b[1;32m    507\u001b[0m         \u001b[0msend_kwargs\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mupdate\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0msettings\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 508\u001b[0;31m         \u001b[0mresp\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0msend\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mprep\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;34m**\u001b[0m\u001b[0msend_kwargs\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    509\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    510\u001b[0m         \u001b[0;32mreturn\u001b[0m \u001b[0mresp\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;32m~/anaconda3/lib/python3.6/site-packages/requests/sessions.py\u001b[0m in \u001b[0;36msend\u001b[0;34m(self, request, **kwargs)\u001b[0m\n\u001b[1;32m    616\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    617\u001b[0m         \u001b[0;31m# Send the request\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 618\u001b[0;31m         \u001b[0mr\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0madapter\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0msend\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mrequest\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;34m**\u001b[0m\u001b[0mkwargs\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    619\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    620\u001b[0m         \u001b[0;31m# Total elapsed time of the request (approximately)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;32m~/anaconda3/lib/python3.6/site-packages/requests/adapters.py\u001b[0m in \u001b[0;36msend\u001b[0;34m(self, request, stream, timeout, verify, cert, proxies)\u001b[0m\n\u001b[1;32m    438\u001b[0m                     \u001b[0mdecode_content\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0;32mFalse\u001b[0m\u001b[0;34m,\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    439\u001b[0m                     \u001b[0mretries\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mmax_retries\u001b[0m\u001b[0;34m,\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 440\u001b[0;31m                     \u001b[0mtimeout\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0mtimeout\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    441\u001b[0m                 )\n\u001b[1;32m    442\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;32m~/anaconda3/lib/python3.6/site-packages/urllib3/connectionpool.py\u001b[0m in \u001b[0;36murlopen\u001b[0;34m(self, method, url, body, headers, retries, redirect, assert_same_host, timeout, pool_timeout, release_conn, chunked, body_pos, **response_kw)\u001b[0m\n\u001b[1;32m    599\u001b[0m                                                   \u001b[0mtimeout\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0mtimeout_obj\u001b[0m\u001b[0;34m,\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    600\u001b[0m                                                   \u001b[0mbody\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0mbody\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mheaders\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0mheaders\u001b[0m\u001b[0;34m,\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 601\u001b[0;31m                                                   chunked=chunked)\n\u001b[0m\u001b[1;32m    602\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    603\u001b[0m             \u001b[0;31m# If we're going to release the connection in ``finally:``, then\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;32m~/anaconda3/lib/python3.6/site-packages/urllib3/connectionpool.py\u001b[0m in \u001b[0;36m_make_request\u001b[0;34m(self, conn, method, url, timeout, chunked, **httplib_request_kw)\u001b[0m\n\u001b[1;32m    381\u001b[0m             \u001b[0;32mexcept\u001b[0m \u001b[0mTypeError\u001b[0m\u001b[0;34m:\u001b[0m  \u001b[0;31m# Python 2.6 and older, Python 3\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    382\u001b[0m                 \u001b[0;32mtry\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 383\u001b[0;31m                     \u001b[0mhttplib_response\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mconn\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mgetresponse\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    384\u001b[0m                 \u001b[0;32mexcept\u001b[0m \u001b[0mException\u001b[0m \u001b[0;32mas\u001b[0m \u001b[0me\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    385\u001b[0m                     \u001b[0;31m# Remove the TypeError from the exception chain in Python 3;\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;32m~/anaconda3/lib/python3.6/http/client.py\u001b[0m in \u001b[0;36mgetresponse\u001b[0;34m(self)\u001b[0m\n\u001b[1;32m   1329\u001b[0m         \u001b[0;32mtry\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m   1330\u001b[0m             \u001b[0;32mtry\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m-> 1331\u001b[0;31m                 \u001b[0mresponse\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mbegin\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m   1332\u001b[0m             \u001b[0;32mexcept\u001b[0m \u001b[0mConnectionError\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m   1333\u001b[0m                 \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mclose\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;32m~/anaconda3/lib/python3.6/http/client.py\u001b[0m in \u001b[0;36mbegin\u001b[0;34m(self)\u001b[0m\n\u001b[1;32m    295\u001b[0m         \u001b[0;31m# read until we get a non-100 response\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    296\u001b[0m         \u001b[0;32mwhile\u001b[0m \u001b[0;32mTrue\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 297\u001b[0;31m             \u001b[0mversion\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mstatus\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mreason\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0m_read_status\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    298\u001b[0m             \u001b[0;32mif\u001b[0m \u001b[0mstatus\u001b[0m \u001b[0;34m!=\u001b[0m \u001b[0mCONTINUE\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    299\u001b[0m                 \u001b[0;32mbreak\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;32m~/anaconda3/lib/python3.6/http/client.py\u001b[0m in \u001b[0;36m_read_status\u001b[0;34m(self)\u001b[0m\n\u001b[1;32m    256\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    257\u001b[0m     \u001b[0;32mdef\u001b[0m \u001b[0m_read_status\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mself\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 258\u001b[0;31m         \u001b[0mline\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mstr\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mfp\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mreadline\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0m_MAXLINE\u001b[0m \u001b[0;34m+\u001b[0m \u001b[0;36m1\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;34m\"iso-8859-1\"\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    259\u001b[0m         \u001b[0;32mif\u001b[0m \u001b[0mlen\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mline\u001b[0m\u001b[0;34m)\u001b[0m \u001b[0;34m>\u001b[0m \u001b[0m_MAXLINE\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    260\u001b[0m             \u001b[0;32mraise\u001b[0m \u001b[0mLineTooLong\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m\"status line\"\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;32m~/anaconda3/lib/python3.6/socket.py\u001b[0m in \u001b[0;36mreadinto\u001b[0;34m(self, b)\u001b[0m\n\u001b[1;32m    584\u001b[0m         \u001b[0;32mwhile\u001b[0m \u001b[0;32mTrue\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    585\u001b[0m             \u001b[0;32mtry\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 586\u001b[0;31m                 \u001b[0;32mreturn\u001b[0m \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0m_sock\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mrecv_into\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mb\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    587\u001b[0m             \u001b[0;32mexcept\u001b[0m \u001b[0mtimeout\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    588\u001b[0m                 \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0m_timeout_occurred\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;32mTrue\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;32m~/anaconda3/lib/python3.6/site-packages/urllib3/contrib/pyopenssl.py\u001b[0m in \u001b[0;36mrecv_into\u001b[0;34m(self, *args, **kwargs)\u001b[0m\n\u001b[1;32m    278\u001b[0m     \u001b[0;32mdef\u001b[0m \u001b[0mrecv_into\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mself\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;34m*\u001b[0m\u001b[0margs\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;34m**\u001b[0m\u001b[0mkwargs\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    279\u001b[0m         \u001b[0;32mtry\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 280\u001b[0;31m             \u001b[0;32mreturn\u001b[0m \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mconnection\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mrecv_into\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m*\u001b[0m\u001b[0margs\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;34m**\u001b[0m\u001b[0mkwargs\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    281\u001b[0m         \u001b[0;32mexcept\u001b[0m \u001b[0mOpenSSL\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mSSL\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mSysCallError\u001b[0m \u001b[0;32mas\u001b[0m \u001b[0me\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    282\u001b[0m             \u001b[0;32mif\u001b[0m \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0msuppress_ragged_eofs\u001b[0m \u001b[0;32mand\u001b[0m \u001b[0me\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0margs\u001b[0m \u001b[0;34m==\u001b[0m \u001b[0;34m(\u001b[0m\u001b[0;34m-\u001b[0m\u001b[0;36m1\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;34m'Unexpected EOF'\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;32m~/anaconda3/lib/python3.6/site-packages/OpenSSL/SSL.py\u001b[0m in \u001b[0;36mrecv_into\u001b[0;34m(self, buffer, nbytes, flags)\u001b[0m\n\u001b[1;32m   1622\u001b[0m             \u001b[0mresult\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0m_lib\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mSSL_peek\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0m_ssl\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mbuf\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mnbytes\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m   1623\u001b[0m         \u001b[0;32melse\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m-> 1624\u001b[0;31m             \u001b[0mresult\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0m_lib\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mSSL_read\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0m_ssl\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mbuf\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mnbytes\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m   1625\u001b[0m         \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0m_raise_ssl_error\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0m_ssl\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mresult\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m   1626\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;31mKeyboardInterrupt\u001b[0m: "
     ]
    }
   ],
   "source": [
    "# NIH query using key words from list\n",
    "# collect the following info \n",
    "\n",
    "key_words = []\n",
    "project_abstract = []\n",
    "project_title = []\n",
    "project_number = []\n",
    "project_start_date = []\n",
    "project_end_date = []\n",
    "department = []\n",
    "agency = []\n",
    "PI = []\n",
    "latitude = []\n",
    "longitude = []\n",
    "org_name = []\n",
    "org_city = []\n",
    "org_state = []\n",
    "org_zip = []\n",
    "org_country = []\n",
    "budget_start_date = []\n",
    "budget_end_date = []\n",
    "CFDA_code = []\n",
    "fiscal_year = []\n",
    "total_cost = []\n",
    "\n",
    "\n",
    "for url in url_list:\n",
    "    \n",
    "    response = requests.get(url).json()\n",
    "    print(len(response['items']))\n",
    "    \n",
    "    for item in response['items']:\n",
    "        print(response['items'])\n",
    "        \n",
    "        key_words.append(item['terms'])\n",
    "        project_abstract.append(item['abstract'])\n",
    "        project_title.append(item['title'])\n",
    "        project_number.append(item['projectNumber'])\n",
    "        project_start_date.append(item['projectStartDate'])\n",
    "        project_end_date.append(item['projectEndDate'])\n",
    "        department.append(item['department'])                        \n",
    "        agency.append(item['agency'])                \n",
    "        PI.append(item['contactPi'])\n",
    "        latitude.append(item['latitude'])\n",
    "        longitude.append(item['longitude'])\n",
    "        org_name.append(item['orgName'])                 \n",
    "        org_city.append(item['orgCity'])\n",
    "        org_state.append(item['orgState'])                \n",
    "        org_zip.append(item['orgZipCode'])\n",
    "        org_country.append(item['orgCountry'])               \n",
    "        budget_start_date.append(item['projectStartDate'])               \n",
    "        budget_end_date.append(item['projectEndDate'])\n",
    "        CFDA_code.append(item['cfdaCode'])                      \n",
    "        fiscal_year.append(item['fy'])                       \n",
    "        total_cost.append(item['totalCostAmount'])\n",
    "    print(len(key_words))\n",
    "    print(url)\n",
    "    \n",
    "        \n",
    "    time.sleep(0.4)\n",
    "        \n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[]\n"
     ]
    }
   ],
   "source": [
    "response = requests.get(url).json()\n",
    "print(response['items'])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "# key_words,\n",
    "# project_abstract,\n",
    "# project_title,\n",
    "# project_number,\n",
    "# project_start_date,\n",
    "# project_end_date,\n",
    "# department,\n",
    "# agency,\n",
    "# PI,\n",
    "# latitude,\n",
    "# longitude,\n",
    "# org_name,\n",
    "# org_city,\n",
    "# org_state,\n",
    "# org_zip,\n",
    "# org_country,\n",
    "# budget_start_date,\n",
    "# budget_end_date,\n",
    "# CFDA_code,\n",
    "# fiscal_year,"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style>\n",
       "    .dataframe thead tr:only-child th {\n",
       "        text-align: right;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: left;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Agency</th>\n",
       "      <th>Budget End Date</th>\n",
       "      <th>Budget Start Date</th>\n",
       "      <th>CFDA code</th>\n",
       "      <th>Department</th>\n",
       "      <th>Fiscal Year</th>\n",
       "      <th>Key Words</th>\n",
       "      <th>Latitude</th>\n",
       "      <th>Longitude</th>\n",
       "      <th>Organization City</th>\n",
       "      <th>...</th>\n",
       "      <th>Organization Name</th>\n",
       "      <th>Organization State</th>\n",
       "      <th>Organization ZIP</th>\n",
       "      <th>Principal Investigator</th>\n",
       "      <th>Project Abstract</th>\n",
       "      <th>Project End Date</th>\n",
       "      <th>Project Number</th>\n",
       "      <th>Project Start Date</th>\n",
       "      <th>Project Title</th>\n",
       "      <th>Total Cost</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>NIH</td>\n",
       "      <td>2016-06-30T00:00:00</td>\n",
       "      <td>2011-07-01T00:00:00</td>\n",
       "      <td>853</td>\n",
       "      <td>HHS</td>\n",
       "      <td>2012</td>\n",
       "      <td>addiction; Age of Onset; Algorithms; Amyotrop...</td>\n",
       "      <td>38.648424</td>\n",
       "      <td>-90.309068</td>\n",
       "      <td>SAINT LOUIS</td>\n",
       "      <td>...</td>\n",
       "      <td>WASHINGTON UNIVERSITY</td>\n",
       "      <td>MO</td>\n",
       "      <td>631304862</td>\n",
       "      <td>HARMS, MATTHEW</td>\n",
       "      <td>DESCRIPTION (provided by applicant): The candi...</td>\n",
       "      <td>2016-06-30T00:00:00</td>\n",
       "      <td>5K08NS075094-02</td>\n",
       "      <td>2011-07-01T00:00:00</td>\n",
       "      <td>NEXT-GENERATION SEQUENCING APPROACHES FOR NOVE...</td>\n",
       "      <td>157414.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>NIH</td>\n",
       "      <td>2016-03-31T00:00:00</td>\n",
       "      <td>2012-05-10T00:00:00</td>\n",
       "      <td>242</td>\n",
       "      <td>HHS</td>\n",
       "      <td>2012</td>\n",
       "      <td>Architecture; Attention deficit hyperactivity...</td>\n",
       "      <td>37.240049</td>\n",
       "      <td>-119.503236</td>\n",
       "      <td>LA JOLLA</td>\n",
       "      <td>...</td>\n",
       "      <td>UNIVERSITY OF CALIFORNIA SAN DIEGO</td>\n",
       "      <td>CA</td>\n",
       "      <td>920930934</td>\n",
       "      <td>SEBAT, JONATHAN</td>\n",
       "      <td>DESCRIPTION (provided by applicant): This is t...</td>\n",
       "      <td>2016-03-31T00:00:00</td>\n",
       "      <td>1U01MH094411-01A1</td>\n",
       "      <td>2012-05-10T00:00:00</td>\n",
       "      <td>3/4-PSYCHIATRIC GWAS CONSORTIUM: GENOMIC FOLLO...</td>\n",
       "      <td>504093.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>NIH</td>\n",
       "      <td>2016-03-31T00:00:00</td>\n",
       "      <td>2012-05-10T00:00:00</td>\n",
       "      <td>242</td>\n",
       "      <td>HHS</td>\n",
       "      <td>2012</td>\n",
       "      <td>Architecture; Attention deficit hyperactivity...</td>\n",
       "      <td>35.366248</td>\n",
       "      <td>-79.429394</td>\n",
       "      <td>CHAPEL HILL</td>\n",
       "      <td>...</td>\n",
       "      <td>UNIVERSITY OF NORTH CAROLINA CHAPEL HILL</td>\n",
       "      <td>NC</td>\n",
       "      <td>275990001</td>\n",
       "      <td>SULLIVAN, PATRICK F</td>\n",
       "      <td>DESCRIPTION (provided by applicant): This is t...</td>\n",
       "      <td>2016-03-31T00:00:00</td>\n",
       "      <td>1U01MH094421-01A1</td>\n",
       "      <td>2012-05-10T00:00:00</td>\n",
       "      <td>1/4 PSYCHIATRIC GWAS CONSORTIUM: GENOMIC FOLLO...</td>\n",
       "      <td>491979.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>NIH</td>\n",
       "      <td>2016-03-31T00:00:00</td>\n",
       "      <td>2012-05-10T00:00:00</td>\n",
       "      <td>242</td>\n",
       "      <td>HHS</td>\n",
       "      <td>2012</td>\n",
       "      <td>Architecture; Attention deficit hyperactivity...</td>\n",
       "      <td>42.347463</td>\n",
       "      <td>-71.082039</td>\n",
       "      <td>BOSTON</td>\n",
       "      <td>...</td>\n",
       "      <td>MASSACHUSETTS GENERAL HOSPITAL</td>\n",
       "      <td>MA</td>\n",
       "      <td>021142696</td>\n",
       "      <td>DALY, MARK JOSEPH</td>\n",
       "      <td>DESCRIPTION (provided by applicant): This is t...</td>\n",
       "      <td>2016-03-31T00:00:00</td>\n",
       "      <td>1U01MH094432-01A1</td>\n",
       "      <td>2012-05-10T00:00:00</td>\n",
       "      <td>2/4-PSYCHIATRIC GWAS CONSORTIUM: GENOMIC FOLLO...</td>\n",
       "      <td>559381.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>NIH</td>\n",
       "      <td>2016-03-31T00:00:00</td>\n",
       "      <td>2012-05-10T00:00:00</td>\n",
       "      <td>242</td>\n",
       "      <td>HHS</td>\n",
       "      <td>2012</td>\n",
       "      <td>Architecture; Attention deficit hyperactivity...</td>\n",
       "      <td>40.790552</td>\n",
       "      <td>-73.953850</td>\n",
       "      <td>NEW YORK</td>\n",
       "      <td>...</td>\n",
       "      <td>MOUNT SINAI SCHOOL OF MEDICINE</td>\n",
       "      <td>NY</td>\n",
       "      <td>100296574</td>\n",
       "      <td>SKLAR, PAMELA</td>\n",
       "      <td>DESCRIPTION (provided by applicant): This is t...</td>\n",
       "      <td>2016-03-31T00:00:00</td>\n",
       "      <td>1U01MH096296-01</td>\n",
       "      <td>2012-05-10T00:00:00</td>\n",
       "      <td>4/4-PSYCHIATRIC GWAS CONSORTIUM:GENOMIC FOLLOW...</td>\n",
       "      <td>297635.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>5 rows Ã— 21 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "  Agency      Budget End Date    Budget Start Date CFDA code Department  \\\n",
       "0    NIH  2016-06-30T00:00:00  2011-07-01T00:00:00       853        HHS   \n",
       "1    NIH  2016-03-31T00:00:00  2012-05-10T00:00:00       242        HHS   \n",
       "2    NIH  2016-03-31T00:00:00  2012-05-10T00:00:00       242        HHS   \n",
       "3    NIH  2016-03-31T00:00:00  2012-05-10T00:00:00       242        HHS   \n",
       "4    NIH  2016-03-31T00:00:00  2012-05-10T00:00:00       242        HHS   \n",
       "\n",
       "   Fiscal Year                                          Key Words   Latitude  \\\n",
       "0         2012   addiction; Age of Onset; Algorithms; Amyotrop...  38.648424   \n",
       "1         2012   Architecture; Attention deficit hyperactivity...  37.240049   \n",
       "2         2012   Architecture; Attention deficit hyperactivity...  35.366248   \n",
       "3         2012   Architecture; Attention deficit hyperactivity...  42.347463   \n",
       "4         2012   Architecture; Attention deficit hyperactivity...  40.790552   \n",
       "\n",
       "    Longitude Organization City    ...      \\\n",
       "0  -90.309068       SAINT LOUIS    ...       \n",
       "1 -119.503236          LA JOLLA    ...       \n",
       "2  -79.429394       CHAPEL HILL    ...       \n",
       "3  -71.082039            BOSTON    ...       \n",
       "4  -73.953850          NEW YORK    ...       \n",
       "\n",
       "                          Organization Name Organization State  \\\n",
       "0                     WASHINGTON UNIVERSITY                 MO   \n",
       "1        UNIVERSITY OF CALIFORNIA SAN DIEGO                 CA   \n",
       "2  UNIVERSITY OF NORTH CAROLINA CHAPEL HILL                 NC   \n",
       "3            MASSACHUSETTS GENERAL HOSPITAL                 MA   \n",
       "4            MOUNT SINAI SCHOOL OF MEDICINE                 NY   \n",
       "\n",
       "  Organization ZIP Principal Investigator  \\\n",
       "0        631304862         HARMS, MATTHEW   \n",
       "1        920930934        SEBAT, JONATHAN   \n",
       "2        275990001    SULLIVAN, PATRICK F   \n",
       "3        021142696      DALY, MARK JOSEPH   \n",
       "4        100296574          SKLAR, PAMELA   \n",
       "\n",
       "                                    Project Abstract     Project End Date  \\\n",
       "0  DESCRIPTION (provided by applicant): The candi...  2016-06-30T00:00:00   \n",
       "1  DESCRIPTION (provided by applicant): This is t...  2016-03-31T00:00:00   \n",
       "2  DESCRIPTION (provided by applicant): This is t...  2016-03-31T00:00:00   \n",
       "3  DESCRIPTION (provided by applicant): This is t...  2016-03-31T00:00:00   \n",
       "4  DESCRIPTION (provided by applicant): This is t...  2016-03-31T00:00:00   \n",
       "\n",
       "      Project Number   Project Start Date  \\\n",
       "0    5K08NS075094-02  2011-07-01T00:00:00   \n",
       "1  1U01MH094411-01A1  2012-05-10T00:00:00   \n",
       "2  1U01MH094421-01A1  2012-05-10T00:00:00   \n",
       "3  1U01MH094432-01A1  2012-05-10T00:00:00   \n",
       "4    1U01MH096296-01  2012-05-10T00:00:00   \n",
       "\n",
       "                                       Project Title Total Cost  \n",
       "0  NEXT-GENERATION SEQUENCING APPROACHES FOR NOVE...   157414.0  \n",
       "1  3/4-PSYCHIATRIC GWAS CONSORTIUM: GENOMIC FOLLO...   504093.0  \n",
       "2  1/4 PSYCHIATRIC GWAS CONSORTIUM: GENOMIC FOLLO...   491979.0  \n",
       "3  2/4-PSYCHIATRIC GWAS CONSORTIUM: GENOMIC FOLLO...   559381.0  \n",
       "4  4/4-PSYCHIATRIC GWAS CONSORTIUM:GENOMIC FOLLOW...   297635.0  \n",
       "\n",
       "[5 rows x 21 columns]"
      ]
     },
     "execution_count": 12,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# put in DataFrame\n",
    "NIH_dict = {\n",
    "    \"Key Words\": key_words,\n",
    "    \"Project Abstract\": project_abstract,\n",
    "    \"Project Title\": project_title,\n",
    "    \"Project Number\": project_number,\n",
    "    \"Project Start Date\": project_start_date,\n",
    "    \"Project End Date\": project_end_date,\n",
    "    \"Department\": department,\n",
    "    \"Agency\": agency,\n",
    "    \"Principal Investigator\": PI,\n",
    "    \"Latitude\": latitude,\n",
    "    \"Longitude\":longitude,\n",
    "    \"Organization Name\": org_name,\n",
    "    \"Organization City\": org_city,\n",
    "    \"Organization State\": org_state,\n",
    "    \"Organization ZIP\": org_zip,\n",
    "    \"Organization Country\": org_country,\n",
    "    \"Budget Start Date\": budget_start_date,\n",
    "    \"Budget End Date\": budget_end_date,\n",
    "    \"CFDA code\": CFDA_code,\n",
    "    \"Fiscal Year\": fiscal_year,\n",
    "    \"Total Cost\": total_cost\n",
    "}\n",
    "NIH_data = pd.DataFrame(NIH_dict)\n",
    "NIH_data.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "# save DF to a csv\n",
    "NIH_data.to_csv(\"FedReporter.csv\", encoding=\"utf-8\", index=False)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.6.3"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
