{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "#load dependencies\n",
    "import requests\n",
    "import json\n",
    "import pandas as pd\n",
    "import time"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 29,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "# URL for API call\n",
    "#url = \"https://api.federalreporter.nih.gov/v1/Projects/search?query=text:cancer biology\"\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 37,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "# we will need to add all our search terms to this URL\n",
    "url = \"https://api.federalreporter.nih.gov/v1/projects/search?query=text:cancer biology$textFields:terms$textOperator:OR$biotech\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 38,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "<Response [200]>\n"
     ]
    }
   ],
   "source": [
    "# check to see if response is valid\n",
    "# wait \n",
    "print(requests.get(url))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 39,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{\n",
      "    \"items\": [\n",
      "        {\n",
      "            \"abstract\": \"DESCRIPTION (provided by applicant): Vitamin E (tocopherol) is the major fat-soluble antioxidant in humans, and serves a critical function in protection against a plethora of oxidative-stress-related pathologies. However, surprisingly little is known about the molecular mechanisms that regulate the levels and activities of the vitamin in the body. The hepatic alpha tocopherol transfer protein (TTP) is a critical regulator of vitamin E status; humans carrying mutated TTP alleles develop severe vitamin E deficiency, accompanied by neuronal degeneration. TTP is thought to function as a 'gatekeeper' in the liver, where it selectively retains the most biologically active form of vitamin E (RRR-1-tocopherol), and 'directs' it to plasma lipoproteins that deliver the vitamin to peripheral cells. Here, we propose to continue our in vivo and in vitro studies on TTP with the overall goal of understanding the molecular bases for the function of the protein in mediating the effects of vitamin E, both in health and in disease. We will use our combined expertise in synthetic chemistry, biophysics, biochemistry and cell-biology to address four specific aims: 1)- to characterize the steps involved in the transport of vitamin E in hepatocytes, and the exact role of TTP in stimulating this process. Using novel fluorescent analogs of vitamin E, we will monitor the intracellular trafficking of the vitamin in cultured hepatocytes. We will then use molecular- and cell-biology approaches to define the steps that TTP catalyzes, and the molecular mechanisms that underlie these events. 2)- To determine the factors and mechanisms that regulate the expression and degradation of TTP, and the roles that vitamin E status and / or oxidative stress play in these processes. 3)- To examine the functions of TTP in neurons, where the protein is expressed in significant levels, and which are the primary site of damage due to vitamin E deficiency. 4)- To understand the molecular mechanisms by which TTP catalyzes the inter-membrane transfer of vitamin E in vitro. Studies proposed in this application will provide critical information that will improve our understanding of vitamin E function as a mediator of human health. PUBLIC HEALTH RELEVANCE: Vitamin E is an essential nutrient that is required for protection of all cells from oxidative damage. TTP is the only known protein that specifically regulates the levels of vitamin E in the body. We propose to determine what exactly TTP does in liver cells, and how mutations in TTP that are found in humans affect its activity.  \",\n",
      "            \"agency\": \"NIH\",\n",
      "            \"budgetEndDate\": \"2013-03-31T00:00:00\",\n",
      "            \"budgetStartDate\": \"2012-04-01T00:00:00\",\n",
      "            \"cfdaCode\": \"847\",\n",
      "            \"congressionalDistrict\": \"11\",\n",
      "            \"contactPi\": \"MANOR, DANNY\",\n",
      "            \"department\": \"HHS\",\n",
      "            \"dunsId\": \"077758407\",\n",
      "            \"fy\": 2012,\n",
      "            \"ic\": \"NIDDK\",\n",
      "            \"latitude\": 41.502739,\n",
      "            \"longitude\": -81.607334,\n",
      "            \"nihApplId\": 8237041,\n",
      "            \"orgCity\": \"CLEVELAND\",\n",
      "            \"orgCountry\": \"UNITED STATES\",\n",
      "            \"orgName\": \"CASE WESTERN RESERVE UNIVERSITY\",\n",
      "            \"orgState\": \"OH\",\n",
      "            \"orgZipCode\": \"441061712\",\n",
      "            \"otherPis\": null,\n",
      "            \"projectEndDate\": \"2013-03-31T00:00:00\",\n",
      "            \"projectNumber\": \"5R01DK067494-10\",\n",
      "            \"projectStartDate\": \"2004-04-15T00:00:00\",\n",
      "            \"smApplId\": 20787,\n",
      "            \"terms\": \" Address; Affect; Alleles; alpha-tocopherol transfer protein; Amino Acids; analog; Antioxidants; Ataxia; base; Biochemistry; Biological Assay; Biophysics; Brain; Brain region; Cardiovascular Diseases; Catalysis; Cells; Cellular biology; Coupled; Cultured Cells; Disease; Event; Fatty acid glycerol esters; Gatekeeping; Genes; Genetic Transcription; Goals; Health; Hepatic; Hepatocyte; Human; Human Pathology; improved; In Vitro; in vivo; Indium; insight; Lipids; Lipoproteins; Liver; Malignant Neoplasms; Maps; Mediating; Mediator of activation protein; Membrane; Membrane Lipids; Molecular; Molecular Biology; Molecular Profiling; Monitor; Movement; multidisciplinary; Mutate; Mutation; Nerve Degeneration; Neuraxis; Neuronal Dysfunction; Neurons; novel; Nutrient; Optics; Organic Chemistry; oxidative damage; Oxidative Stress; Pathology; Pattern; Peripheral; Placenta; Plasma; Play; Population; primary outcome; Process; promoter; protein function; Proteins; Reaction; Recombinants; Reporter; Role; Route; selective expression; Site; skills; Structure; Synthesis Chemistry; Time; tissue processing; Tocopherols; trafficking; transcription factor; Travel; uptake; Variant; Vitamin E; Vitamin E Deficiency; Vitamins\",\n",
      "            \"title\": \"FUNCTIONS OF TOCOPHEROL TRANSFER PROTEIN\",\n",
      "            \"totalCostAmount\": 319889\n",
      "        },\n",
      "        {\n",
      "            \"abstract\": \"DESCRIPTION (provided by applicant): This proposal describes a five year program for a clinical research based career in Infectious Diseases. The candidate, Mini Kamboj MD, an Instructor in Medicine and Associate-Director of Hospital Infection Control will conduct the program under the supervision of Dr. Kent Sepkowitz, Professor of Medicine, and Director of the Infection Control Program at Memorial Sloan-Kettering Cancer Center (MSKCC), to study the role of novel risk factors for a multi-drug resistant organism (MDRO) Vancomycin resistant enterococcus (VRE).These infections are more common in persons with cancer and result in devastating outcomes and greater resource utilization. The study will be conducted at MSKCC, one of the largest programs of clinical research in the world. The candidate has enlisted an advisory committee of esteemed leaders in the field of microbial pathogenesis, infection control and genome based testing for guidance and advice during this program. The proposal is guided by the hypothesis that many anti-cancer drugs have anti microbial properties and therefore are likely to influence gastrointestinal flora including the microbial burden (density) of VRE. We also hypothesize that a change in density of VRE in turn has an effect on clinical outcome. To examine this hypothesis we propose to (1)Determine VRE density in stool in 50 cancer patients at MSKCC using a new quantitative PCR assay and standard microbiologic techniques (2) Determine the effect of commonly used chemotherapeutic agents with strong antimicrobial activity, (5-Fluorouracil and Methotrexate) upon density of VRE among colonized individuals. (3) Determine the predictive value of VRE density determined by standard techniques and quantitative PCR on clinical outcome, including mortality and resource utilization, among patients with cancer at highest risk for invasive VRE disease. The candidate's immediate goal is to develop into an independent clinical investigator in patient-oriented clinical research with a focus on hospital acquired infections in persons with cancer and long term goal is to implement studies on devising and testing therapeutic approaches including decolonizing and other control strategies for MDRO's that are common in the cancer population.  Hospital-acquired infections are a major cause of morbidity and mortality in persons with cancer. In this population, colonization and infection with vancomycin-resistant enterococci are more common. The studies in this proposal will provide novel insight into common but previously unrecognized risk factors for VRE in persons with cancer and provide a basis for devising and testing control strategies against this multi-drug resistant organism.  \",\n",
      "            \"agency\": \"NIH\",\n",
      "            \"budgetEndDate\": \"2013-05-31T00:00:00\",\n",
      "            \"budgetStartDate\": \"2012-06-01T00:00:00\",\n",
      "            \"cfdaCode\": \"855\",\n",
      "            \"congressionalDistrict\": \"12\",\n",
      "            \"contactPi\": \"KAMBOJ, MINI\",\n",
      "            \"department\": \"HHS\",\n",
      "            \"dunsId\": \"064931884\",\n",
      "            \"fy\": 2012,\n",
      "            \"ic\": \"NIAID\",\n",
      "            \"latitude\": 40.764386,\n",
      "            \"longitude\": -73.956799,\n",
      "            \"nihApplId\": 8260345,\n",
      "            \"orgCity\": \"NEW YORK\",\n",
      "            \"orgCountry\": \"UNITED STATES\",\n",
      "            \"orgName\": \"SLOAN-KETTERING INSTITUTE FOR CANCER RES\",\n",
      "            \"orgState\": \"NY\",\n",
      "            \"orgZipCode\": \"100656007\",\n",
      "            \"otherPis\": null,\n",
      "            \"projectEndDate\": \"2015-05-31T00:00:00\",\n",
      "            \"projectNumber\": \"5K23AI083880-03\",\n",
      "            \"projectStartDate\": \"2010-06-01T00:00:00\",\n",
      "            \"smApplId\": 20860,\n",
      "            \"terms\": \" Address; Advisory Committees; Affect; Animals; Anti-Bacterial Agents; Antibiotics; antimicrobial; antimicrobial drug; Antimicrobial Effect; Antineoplastic Agents; Applications Grants; attributable mortality; base; Biological Assay; cancer care; Cancer Patient; cancer therapy; career; Caring; chemotherapeutic agent; Clinical; Clinical Investigator; Clinical Research; commensal microbes; Communicable Diseases; Critical Illness; density; Diagnosis; Diagnostic; Disease; Environment; Failure; Feces; Fluorouracil; Foundations; gastrointestinal; Genome; Genomics; Goals; Growth; Healthcare; Hematologic Neoplasms; high risk; Human; Immunocompromised Host; Immunologic Receptors; In Vitro; Incidence; Individual; Infection; Infection Control; insight; Institution; instructor; Laboratories; Malignant Neoplasms; Measures; Medicine; Memorial Sloan-Kettering Cancer Center; methicillin resistant Staphylococcus aureus; Methods; Methotrexate; microbial; Microbiological Techniques; Microbiology; Molecular; Morbidity - disease rate; mortality; Multi-Drug Resistance; Mus; Natural History; Nosocomial Infections; novel; Organism; Outcome; pathogen; Pathogenesis; patient oriented; patient population; Patients; Persons; Pharmaceutical Preparations; Polymerase Chain Reaction; Population; Predictive Value; Predisposition; pressure; Prevalence; professor; programs; Property; Publishing; Regimen; Reporting; Resistance; Resources; Risk; Risk Factors; Role; Series; Supervision; System; Techniques; Technology; Testing; Therapeutic; Time; transmission process; United States; Vancomycin; Vancomycin resistant enterococcus; Work\",\n",
      "            \"title\": \"NOVEL RISK FACTORS FOR VANCOMYCIN RESISTANT ENTEROCOCCI IN PERSONS WITH CANCER\",\n",
      "            \"totalCostAmount\": 130437\n",
      "        },\n",
      "        {\n",
      "            \"abstract\": \"DESCRIPTION (provided by applicant): The stomach epithelium comprises two main compartments with distinct turnover rates and cell compositions termed antrum or pylorus and corpus or main body. While classical mutagenesis experiments suggested the existence of adult stem cells that continuously replenish these cell types, their identity and localization remains elusive. Notably, a recent report identified Lgr5+ stem cells in the antrum, which are capable of multilineage differentiation over long-term and can serve as the cell type of origin for adenomas upon deletion of the tumor suppressor gene Apc. We have recently identified rare Sox2+ cells in the antrum and corpus of mice. Genetic lineage tracing demonstrates that Sox2+ cells can, like antral Lgr5+ cells, give rise to all mature cell types in the stomach for up to 22 months, thus qualifying as bona fide stem cells. These observations raise the following key questions relevant to the biology of stomach turnover and stomach cancer: (1) When in development are Sox2+ stomach stem cells formed; (2) Are Sox2+ stem cells and Lgr5+ stem cells part of the same lineage; (3) What are the molecular and cellular features of Sox2+ stem cells in antrum and corpus; (4) Do Sox2+ stem cells divide symmetrically or asymmetrically and at which rate; (5) Are Sox2+ stem cells responsive to tissue injury and inflammation; (6) Are Sox2+ cells more amenable to tumorigenesis than differentiated stomach cells; and (7) Is Sox2 itself required for stomach development, tissue homeostasis and cancer? We have developed several novel transgenic tools in mice to address each of these fundamental questions in the context of three major aims and multiple subaims. Specifically, we have generated Sox2-GFP reporter mice as well as Sox2-CreER lineage tracing mice to characterize Sox2+ cells at the molecular and cellular levels at different stages of development. Aim 1 entails characterization of the ultrastructure and transcriptome of Sox2+ cells as well as their self-renewal and differentiation kinetics, establishment of an in vitro culture system and evaluation of the lineage relationship between Sox2+ cells and Lgr5+ cells. In Aim 2, we will cross a novel conditional allele for Sox2 to different Cre drivers to assess the requirement for Sox2 at various stages of pre-and postnatal development and in the context of stomach cell injury inflicted by several genetic and chemotoxic models. Given that stomach cancer is the second-most common cause of cancer-related deaths worldwide with relatively little known about its underlying genetic and cellular origins, we propose to test in Aim 3 the susceptibility of Sox2+ stem cells to malignant transformation. Here, we will evaluate whether Sox2+ stem cells are amenable to transformation into adenomas/adenocarcinomas upon deletion of the Apc tumor suppressor and whether Sox2 protein itself is required for tumor formation. Lastly, we will test the hypothesis that the differentiation state of stomach cells influences their amenability to transformation by deleting the E-Cadherin gene, which is mutated in 50% of human diffuse gastric cancer cases, in Sox2+ stem cells, transit-amplifying cells and differentiated chief cells.  PUBLIC HEALTH RELEVANCE: Stomach cancer is the second-most common cancer-related cause of death worldwide with an increase seen specifically in diffuse gastric cancer in the United States. Elucidating the fundamental biology of stomach development and homeostasis is imperative for understanding how stomach cancer develops and for identifying cellular and molecular targets for treatment. Thus, by studying Sox2 and Sox2+ stem cells in the normal and malignant stomach, we will gain new basic insights that can be exploited for regenerative medicine as well as for stomach cancer prevention and treatment.   \",\n",
      "            \"agency\": \"NIH\",\n",
      "            \"budgetEndDate\": \"2013-06-30T00:00:00\",\n",
      "            \"budgetStartDate\": \"2012-07-01T00:00:00\",\n",
      "            \"cfdaCode\": \"847\",\n",
      "            \"congressionalDistrict\": \"08\",\n",
      "            \"contactPi\": \"HOCHEDLINGER, KONRAD\",\n",
      "            \"department\": \"HHS\",\n",
      "            \"dunsId\": \"073130411\",\n",
      "            \"fy\": 2012,\n",
      "            \"ic\": \"NIDDK\",\n",
      "            \"latitude\": 42.347463,\n",
      "            \"longitude\": -71.082039,\n",
      "            \"nihApplId\": 8348185,\n",
      "            \"orgCity\": \"BOSTON\",\n",
      "            \"orgCountry\": \"UNITED STATES\",\n",
      "            \"orgName\": \"MASSACHUSETTS GENERAL HOSPITAL\",\n",
      "            \"orgState\": \"MA\",\n",
      "            \"orgZipCode\": \"021142696\",\n",
      "            \"otherPis\": null,\n",
      "            \"projectEndDate\": \"2016-06-30T00:00:00\",\n",
      "            \"projectNumber\": \"1R01DK096034-01\",\n",
      "            \"projectStartDate\": \"2012-07-01T00:00:00\",\n",
      "            \"smApplId\": 20877,\n",
      "            \"terms\": \" Ablation; Address; Adenocarcinoma; adenoma; Adult; adult stem cell; Alleles; Antral; base; Biochemical; Biology; Cancer Etiology; cancer prevention; cancer therapy; Cause of Death; cell injury; cell type; Cells; cellular targeting; Cessation of life; chemical genetics; Chief Cell; Complement; Development; Diffuse; Doxycycline; E-Cadherin; Electron Microscopy; Embryo; Epithelial; Epithelium; Evaluation; Exhibits; Fetal Development; gain of function; Gastric Parietal Cells; Gene Expression Profile; Genes; Genetic; Goals; Homeostasis; Human; In Vitro; in vivo; Inflammation; Injury; insight; Interferons; Intestines; Kinetics; Location; loss of function; Malignant - descriptor; Malignant Neoplasms; malignant stomach neoplasm; Maps; Messenger RNA; MicroRNAs; Modeling; Molecular; Molecular Target; mouse model; Mus; Mutagenesis; Mutate; Natural regeneration; novel; Oncogenic; Organoids; overexpression; Pathway interactions; Pattern; Phenotype; Play; postnatal; Predisposition; prenatal; Process; progenitor; Proteins; Pylorus; Qualifying; Regenerative Medicine; repaired; Reporter; Reporting; research study; response; Role; self-renewal; Signal Transduction; Staging; stem; stem cell differentiation; stem cell niche; Stem cells; Stimulus; Stomach; System; Testing; Tissues; tool; Transgenic Organisms; tumor; tumor initiation; Tumor Suppressor Genes; Tumor Suppressor Proteins; tumorigenesis; United States; villin\",\n",
      "            \"title\": \"ROLE OF SOX2 IN STOMACH DEVELOPMENT, REGENERATION AND CANCER\",\n",
      "            \"totalCostAmount\": 353155\n",
      "        },\n",
      "        {\n",
      "            \"abstract\": \"DESCRIPTION (provided by applicant): The radiation exposure resulting from medical imaging has become a public safety concern. Dose reduction for pediatric patients is particularly important since such patients are at increased risk owing to the enhanced radiosensitivity of their tissues and the longer time-period over which stochastic radiation effects may manifest. The objective of this proposal is to develop and validate techniques that can be used to minimize the radiation exposure to pediatric patients undergoing molecular imaging procedures without adversely impacting the diagnostic quality of the images. The hypothesis of this proposal is that it is possible to reduce the absorbed dose received by pediatric molecular imaging patients without compromising the image quality of the study by: (1) individualizing the amount of administered activity and the image acquisition time to better account for the effects that patient-specific body size and shape may have on the detector count-rate density; (2) adjusting acquisition parameters to account for the smaller field of view in pediatric imaging; (3) improving image reconstruction methods to enable image generation from these lower-count, smaller field-of-view images. In current clinical practice, dosing for pediatric molecular imaging has focused on rule-of-thumb approaches to identifying the activity administered to children for different imaging procedures. By using state-of-the-art simulation, image quality evaluation and radiation dosimetry tools, the work proposed in this grant application will examine dose reduction methods in a much more rigorous and, scientifically validated manner. This approach will lead to clinically implementable dose reduction techniques for pediatric (and potentially adult) molecular imaging on par with national efforts to reduce doses in pediatric and adult CT and fluoroscopically guided interventions without adversely impacting the diagnostic quality of the images.  PUBLIC HEALTH RELEVANCE: The radiation exposure resulting from medical imaging has become a public safety concern. The objective of this proposal is to develop and validate techniques that can be used to reduce the radiation exposure to pediatric patients undergoing molecular imaging procedures using methods that retain the diagnostic quality of the images.  \",\n",
      "            \"agency\": \"NIH\",\n",
      "            \"budgetEndDate\": \"2013-02-28T00:00:00\",\n",
      "            \"budgetStartDate\": \"2012-03-01T00:00:00\",\n",
      "            \"cfdaCode\": \"286\",\n",
      "            \"congressionalDistrict\": \"07\",\n",
      "            \"contactPi\": \"SGOUROS, GEORGE\",\n",
      "            \"department\": \"HHS\",\n",
      "            \"dunsId\": \"001910777\",\n",
      "            \"fy\": 2012,\n",
      "            \"ic\": \"NIBIB\",\n",
      "            \"latitude\": 39.298737,\n",
      "            \"longitude\": -76.594342,\n",
      "            \"nihApplId\": 8235635,\n",
      "            \"orgCity\": \"BALTIMORE\",\n",
      "            \"orgCountry\": \"UNITED STATES\",\n",
      "            \"orgName\": \"THE JOHNS HOPKINS UNIVERSITY\",\n",
      "            \"orgState\": \"MD\",\n",
      "            \"orgZipCode\": \"212051832\",\n",
      "            \"otherPis\": \"FREY, ERIC C. ; \",\n",
      "            \"projectEndDate\": \"2016-02-29T00:00:00\",\n",
      "            \"projectNumber\": \"1R01EB013558-01A1\",\n",
      "            \"projectStartDate\": \"2012-03-01T00:00:00\",\n",
      "            \"smApplId\": 20932,\n",
      "            \"terms\": \" Accounting; Adult; Age; age group; Algorithms; Applications Grants; attenuation; base; Body Size; Breast; cancer risk; Characteristics; Child; Childhood; clinical practice; Collimator; Data; density; design; detector; Diagnostic; Diagnostic Imaging; Discipline of Nuclear Medicine; Dose; dosimetry; Evaluation; experience; Exposure to; Fluoroscopy; Generations; Guidelines; Height; Image; Image Reconstructions; imaging modality; Imaging Techniques; improved; Incidence; indexing; Intervention; Lead; Longevity; Long-Term Effects; Malignant Neoplasms; Medical Imaging; Methods; Modeling; molecular imaging; mortality; Nuclear; Organ; patient population; Patients; Positioning Attribute; Radiation; radiation effect; Radiation Tolerance; Radiometry; Radiopharmaceuticals; reconstruction; Risk; Safety; sex; Shapes; simulation; Specific qualifier value; Techniques; Thyroid Gland; Time; Tissues; tool; Weight; Width; Work\",\n",
      "            \"title\": \"DOSE REDUCTION IN PEDIATRIC MOLECULAR IMAGING\",\n",
      "            \"totalCostAmount\": 527727\n",
      "        },\n",
      "        {\n",
      "            \"abstract\": \"DESCRIPTION (provided by applicant): The objective of this research proposal is to develop a series of steroid-based magnetic resonance imaging (MRI) contrast agents to facilitate molecular characterization of the status and function of steroid receptors in hormone-dependent disease and development. The ability to detect the location of cell receptors and their concentration throughout a living organism is of vital importance as it allows for further understanding of cell signaling mechanics. Progesterone and estrogen are steroid hormones that bind to their receptors and function as a transcription factors in the nucleus. A non-invasivemeans of determining the hormone receptor status of hormone-dependent tumors and benign lesions could assist with treatment options, identification of the size and exact location of the tumor, and provide additional tools when traditional imaging strategies miss or confuse lesions of the breast and uterus. Unlike fluorescence and optical microscopy, MRI is not limited by depth or transparency of the specimen. MRI does not use ionizing radiation or radioactivity like positron emission tomography (PET) and X-ray/CT, and it allows for 3D reconstructions and high resolution imaging over time without the need to sacrifice the organism. It is the hypothesis of this grant proposal that gadolinium(III) conjugated hormone-based contrast agents can target and accumulate in hormone receptor positive cells to non-invasively image receptor status. Our preliminary studies have identified gadolinium (Gd)-conjugated progesterone derivatives as compounds capable of traversing the cell membrane, binding to progesterone receptors, initiating gene transcription, and enhancing contrast in mammalian tissues and tumors imaged in vivo using magnetic resonance, the most promising of which is termed 'ProGlo'. This proposal focuses on the application and expansion of ProGlo to enhance the imaging of steroid receptor tumors and tissues in vivo. AIM 1. To synthesize and test a series of CAs that targets the estrogen receptor. Estrogen-based contrast agents will be designed and synthesized with varying polarities, charges, and water solubilities that will selectively probe membrane bound receptors or receptors located inside the cells and tumors. AIM 2. To investigate if hormone receptor disease of the breast and uterus can be classified as receptor positive using functional hormone MR agents. AIM 3. To chemically modify CAs to enhance in vivo relaxivity and reduce toxicity by developing and testing bio-orthogonal, water-soluble, and multi-chelated hormone CAs. Steroid receptors have emerged as attractive targets for molecular imaging due to their role in promoting the growth of breast and uterine lesions. This proposal will develop steroid contrast agents that could provide a molecular profile of hormone receptor status in cells, validate responsiveness to therapy, and improve diagnoses. Functional contrast agents will provide valuable tools for use in humans and for immediate use in animal models of hormone receptor dependent development and disease without the need to euthanize the animal thereby increasing knowledge about developmental biology, disease etiology, and progression.  PUBLIC HEALTH RELEVANCE: Steroid receptors have emerged as attractive targets for molecular imaging due to their role in promoting the growth of breast and uterine lesions. This proposal will develop steroid magnetic resonance imaging contrast agents that could provide a molecular profile of hormone receptor status in cells and tissues, validate responsiveness to therapy, and improve diagnoses. This is an innovative and far-reaching proposal that is likely to impact the fields of imaging, cancer diagnosis, and women's health.  \",\n",
      "            \"agency\": \"NIH\",\n",
      "            \"budgetEndDate\": \"2013-08-31T00:00:00\",\n",
      "            \"budgetStartDate\": \"2012-09-15T00:00:00\",\n",
      "            \"cfdaCode\": \"286\",\n",
      "            \"congressionalDistrict\": \"09\",\n",
      "            \"contactPi\": \"MEADE, THOMAS J\",\n",
      "            \"department\": \"HHS\",\n",
      "            \"dunsId\": \"160079455\",\n",
      "            \"fy\": 2012,\n",
      "            \"ic\": \"NIBIB\",\n",
      "            \"latitude\": 42.05122,\n",
      "            \"longitude\": -87.684661,\n",
      "            \"nihApplId\": 8265545,\n",
      "            \"orgCity\": \"EVANSTON\",\n",
      "            \"orgCountry\": \"UNITED STATES\",\n",
      "            \"orgName\": \"NORTHWESTERN UNIVERSITY\",\n",
      "            \"orgState\": \"IL\",\n",
      "            \"orgZipCode\": \"602013149\",\n",
      "            \"otherPis\": null,\n",
      "            \"projectEndDate\": \"2016-08-31T00:00:00\",\n",
      "            \"projectNumber\": \"1R01EB014806-01\",\n",
      "            \"projectStartDate\": \"2012-09-15T00:00:00\",\n",
      "            \"smApplId\": 20947,\n",
      "            \"terms\": \" Aging; Animal Model; Animals; Applications Grants; base; Benign; Binding; Biological; Biological Assay; Breast Diseases; breast lesion; cancer diagnosis; Cell membrane; Cell Nucleus; Cells; Charge; Chemicals; Chemistry; Conjugated Estrogens; Contrast Media; design; Development; Developmental Biology; Diagnosis; dimethylbenzanthracene; Disease; Disease Progression; Endocrine System Diseases; endometriosis; Estrogen receptor positive; Estrogen Receptors; Estrogens; Etiology; Fluorescence; Gadolinium; gadolinium oxide; Genetic Transcription; Growth; Hormone Receptor; Hormones; Human; Image; Imaging Techniques; Implant; improved; In Vitro; in vitro testing; in vivo; innovation; Ionizing radiation; Knowledge; Lesion; Life; Location; Magnetic Resonance; Magnetic Resonance Imaging; malignant breast neoplasm; Malignant Neoplasms; Mammary Neoplasms; Mechanics; Membrane; Microscopy; Modeling; Modification; Molecular; molecular imaging; Molecular Profiling; Monitor; Mus; Mutagenesis; neoplastic cell; New Agents; Nuclear; Nude Mice; Oncogenic; Optics; Organism; Palpable; Pathologic Processes; Physiological Processes; Positron-Emission Tomography; Process; Progesterone; Progesterone Receptors; Property; Radioactivity; receptor; receptor binding; Receptor Cell; receptor function; recombinase; reconstruction; Research Proposals; research study; Resolution; Rodent Model; Role; Safety; Series; Signal Transduction; soft tissue; Solubility; spatiotemporal; Specimen; steroid hormone; Steroid Receptors; Steroids; Testing; Time; Tissues; Tomography, Emission-Computed, Single-Photon; tool; Toxic effect; transcription factor; tumor; Tumor Tissue; Uterine Lesion; Uterus; Water; water solubility; Women's Health; Xenograft procedure; X-Ray Computed Tomography\",\n",
      "            \"title\": \"STEROID-BASED CONTRAST AGENTS FOR MAGNETIC RESONANCE IMAGING OF ENDOCRINE DISEASE\",\n",
      "            \"totalCostAmount\": 356617\n",
      "        },\n",
      "        {\n",
      "            \"abstract\": \"DESCRIPTION (provided by applicant): Cerenkov radiation is a phenomenon in which optical photons are emitted when a charged particle moves faster than the speed of light in a dielectric medium such as tissue. Recently we, and others, have discovered that measurable visible light due to the Cerenkov effect is produced in vivo following administration of ?-emitting radionuclides to small animals. Furthermore, the amounts of injected activity required to produce a detectable signal are consistent with small animal molecular imaging applications. This observation has led to the development of a new hybrid molecular imaging modality known as Cerenkov luminescence imaging (CLI) that allows the spatial distribution of biomolecules labeled with ?-emitting radionuclides to be imaged in vivo using sensitive charge-coupled device (CCD) cameras. This is especially valuable for preclinical studies with important therapeutic ?-emitting radionuclides, such as 90Y, that cannot readily be imaged at tracer doses using any other in vivo imaging technique. CLI also provides a fast, low-cost, readily accessible approach to image PET radiotracers in small animals. Translational opportunities for Cerenkov luminescence also exist. CLI might be possible for skin, eye and oral cavity, or using endoscopic or catheter-based devices inside the body. Another possible application that is being explored is to use the abundant blue/UV Cerenkov light produced by radiotracers as a source of internal light delivery deep inside tissues for photoactivation, for example in photodynamic therapy and photoactivated drug delivery. The goal of this proposal is to provide a quantitative understanding of Cerenkov luminescence as it applies to biomedical applications and to produce data that serves as a platform for guiding the development of small-animal and translational Cerenkov applications by the molecular imaging and therapeutics community. The specific aims address obtaining a detailed quantitative understanding of Cerenkov signals and spectra that can be measured from tissues through a combination of computation/simulation, phantom experiments and in vivo studies. We will compare the performance of two different camera technologies for the detection of weak Cerenkov signals, and determine detection limits. The ability to exploit spectral information for Cerenkov tomography, and for spectral unmixing in multimodal studies that involve both Cerenkov and bioluminescence signals, also will be characterized. Finally, we will estimate the light energy delivered to tissues by radiotracers via the Cerenkov effect, to determine if this mechanism can feasibly be used as a source of internal excitation for photoactivation. The anticipated outcome of this work is a detailed understanding of the production, distribution and transport of visible Cerenkov radiation by systemically-administered radiotracers, that will serve as a critical foundation on which to develop biomedical applications based on this newly-observed phenomenon.  PUBLIC HEALTH RELEVANCE: Radiotracers commonly administered for medical imaging examinations in nuclear medicine have recently been observed to emit detectable amounts of visible light via the Cerenkov effect. This opens up new opportunities for in vivo imaging of diagnostic and therapeutic radionuclides using sensitive optical imaging cameras, especially in preclinical small-animal studies. There also are prospects for using this Cerenkov radiation to assist in delivering light-activated therapies deep inside the body without the use of an external laser. This research in this proposal seeks to determine the feasibility of using Cerenkov light fo both imaging and therapeutic applications.  \",\n",
      "            \"agency\": \"NIH\",\n",
      "            \"budgetEndDate\": \"2013-04-30T00:00:00\",\n",
      "            \"budgetStartDate\": \"2012-07-01T00:00:00\",\n",
      "            \"cfdaCode\": \"286\",\n",
      "            \"congressionalDistrict\": \"03\",\n",
      "            \"contactPi\": \"CHERRY, SIMON R\",\n",
      "            \"department\": \"HHS\",\n",
      "            \"dunsId\": \"047120084\",\n",
      "            \"fy\": 2012,\n",
      "            \"ic\": \"NIBIB\",\n",
      "            \"latitude\": 38.543366,\n",
      "            \"longitude\": -121.72946,\n",
      "            \"nihApplId\": 8342753,\n",
      "            \"orgCity\": \"DAVIS\",\n",
      "            \"orgCountry\": \"UNITED STATES\",\n",
      "            \"orgName\": \"UNIVERSITY OF CALIFORNIA DAVIS\",\n",
      "            \"orgState\": \"CA\",\n",
      "            \"orgZipCode\": \"956186153\",\n",
      "            \"otherPis\": null,\n",
      "            \"projectEndDate\": \"2016-04-30T00:00:00\",\n",
      "            \"projectNumber\": \"1R01EB015471-01\",\n",
      "            \"projectStartDate\": \"2012-07-01T00:00:00\",\n",
      "            \"smApplId\": 20960,\n",
      "            \"terms\": \" 90Y; Address; Algorithms; Animal Experiments; Animals; base; Biological; Bioluminescence; Catheters; Cells; Characteristics; Charge; charge coupled device camera; Clinic; Complex; computer studies; cost; Coupled; Data; density; Dependence; design; Detection; Development; Devices; Diagnostic Imaging; Dimensions; Discipline of Nuclear Medicine; Dose; Drug Delivery Systems; Electrons; Equation; Eye; Foundations; Gamma Rays; Goals; Gold; Half-Life; Hybrids; Image; imaging modality; Imaging Techniques; improved; In Vitro; in vivo; indexing; interest; Label; Lasers; Light; Luciferases; luminescence; magnetic field; Malignant neoplasm of liver; Measurable; Measurement; Measures; Medical Imaging; Methods; Microspheres; molecular imaging; multimodality; Non-Hodgkin's Lymphoma; Nuclear; optical imaging; Optics; Oral cavity; Outcome; particle; Pathway interactions; Performance; photoactivation; Photochemotherapy; Photons; Phototherapy; Positron; Positron-Emission Tomography; pre-clinical; preclinical study; Production; Property; Publications; Quantitative Evaluations; Quantum Dots; Radiation; Radioimmunotherapy; Radioisotopes; radiotracer; reconstruction; Reporter; Research; research study; Side; Signal Transduction; simulation; Skin; Source; Spatial Distribution; Speed; Surface; System; Techniques; Technology; theories; Therapeutic; Therapeutic community technique; Time; Tissues; tomography; Tomography, Emission-Computed, Single-Photon; Tracer; uptake; Visible Radiation; Weight; Work\",\n",
      "            \"title\": \"QUANTITATIVE EVALUATION OF CERENKOV LUMINESCENCE FOR IMAGING AND THERAPY\",\n",
      "            \"totalCostAmount\": 340933\n",
      "        },\n",
      "        {\n",
      "            \"abstract\": \"DESCRIPTION (provided by applicant): The aryl hydrocarbon receptor (AHR) is a ligand-activated transcription factor that mediates the biological effects of polycyclic aromatic hydrocarbons (PAHs), halogenated-dioxins ('dioxins'), - dibenzofurans, -biphenyls (PCBs), and -diphenylethers. Because of its central role in acute toxicity, teratogenesis and carcinogenesis of a large number of environmental contaminants, mechanistic information about AHR signaling is used by regulatory agencies to establish acceptable exposure levels for these pollutants. The overarching hypothesis of this proposal is that at least three distinct AHR signaling pathways exist that are of toxicological significance to human populations and vertebrate species in general. We propose that these three pathways can be distinguished by their unique impacts on transcriptional profiles within target cell populations and that these pathways can be further defined based upon their dependence on fractional activation of the AHR, by their cellular context, and their influence on gene expression through enhancer elements known as AHREs. The goal of this proposal is to identify biomarkers that reflect exposure to agonists and their induced toxic responses. We will accomplish this through the identification of low and high affinity AHREs in the hepatocyte and endothelial cell compartments of novel recombinant mouse models. To these ends, we propose the following specific aims: Specific Aim 1: Use gene targeting in ES cells to generate recombinant alleles derived from a ligand responsive version of Ahr. Specific Aim 2: Demonstrate the importance of the cellular expression and concentration of AHR. Specific Aim 3: Classify the high and low affinity AHRE batteries using genomic approaches. Specific Aim 4: Provide evidence that environmental AHR agonists can stimulate the endogenous AHR pathway. We propose that the results of this effort will provide new models that will inform the mechanism of dioxin toxicity in humans, characterize DNA signatures of toxicity, and test a novel hypothesis that developmental toxicity arises from interference with the endogenous AHR pathway. Collectively these studies will set the stage for more accurate and informative risk assessment in man.  PUBLIC HEALTH RELEVANCE: This project aims to understand the mechanisms of AHR-mediated transcriptional activation which leads to the adaptive, toxic, and developmental consequences of dioxin exposure and to search for biomarkers that are predictive of these disparate outcomes. These experiments will allow more relevant assessments of AHR-mediated dioxin toxicity as well as generate biomarkers predictive for relevant endpoints of dioxin exposure.   \",\n",
      "            \"agency\": \"NIH\",\n",
      "            \"budgetEndDate\": \"2012-11-30T00:00:00\",\n",
      "            \"budgetStartDate\": \"2012-01-10T00:00:00\",\n",
      "            \"cfdaCode\": \"113\",\n",
      "            \"congressionalDistrict\": \"02\",\n",
      "            \"contactPi\": \"BRADFIELD, CHRISTOPHER A\",\n",
      "            \"department\": \"HHS\",\n",
      "            \"dunsId\": \"161202122\",\n",
      "            \"fy\": 2012,\n",
      "            \"ic\": \"NIEHS\",\n",
      "            \"latitude\": 43.060421,\n",
      "            \"longitude\": -89.397322,\n",
      "            \"nihApplId\": 8258122,\n",
      "            \"orgCity\": \"MADISON\",\n",
      "            \"orgCountry\": \"UNITED STATES\",\n",
      "            \"orgName\": \"UNIVERSITY OF WISCONSIN MADISON\",\n",
      "            \"orgState\": \"WI\",\n",
      "            \"orgZipCode\": \"537151218\",\n",
      "            \"otherPis\": null,\n",
      "            \"projectEndDate\": \"2016-11-30T00:00:00\",\n",
      "            \"projectNumber\": \"1R01ES020668-01A1\",\n",
      "            \"projectStartDate\": \"2012-01-10T00:00:00\",\n",
      "            \"smApplId\": 20999,\n",
      "            \"terms\": \" activating transcription factor; Acute; Affinity; Agonist; Alleles; Animal Model; ARNT gene; Aromatic Polycyclic Hydrocarbons; Aryl Hydrocarbon Receptor; aryl hydrocarbon receptor ligand; base; Biological; Biological Markers; Candidate Disease Gene; carcinogenesis; Cells; Chemical Structure; Complex; Coupled; Data Set; Dependence; Development; Developmental Biology; dibenzofuran; Dioxins; diphenyl; DNA; embryonic stem cell; Endothelial Cells; Environment; environmental chemical; Environmental Health; Environmental Pollution; Exposure to; Gene Expression; Gene Targeting; Genetic Enhancer Element; Genetic Polymorphism; Genomics; Goals; Grant; hazard; Health Policy; Hepatocyte; Human; Individual; insight; Ligands; Liver neoplasms; man; Mediating; Modeling; mouse model; Mus; mutant; novel; Outcome; Pathway interactions; Physiological; Play; pollutant; Polychlorinated Biphenyls; Population; receptor; Receptor Signaling; Recombinants; research study; response; Risk; Risk Assessment; Rodent; Role; Signal Pathway; Signal Transduction; Staging; Testing; Toxic effect; Toxicity Tests; Toxicology; Transcript; Transcriptional Activation; Transgenic Model; Tumor Promotion\",\n",
      "            \"title\": \"TRANSGENIC MODELS OF AH RECEPTOR ACTION\",\n",
      "            \"totalCostAmount\": 338625\n",
      "        },\n",
      "        {\n",
      "            \"abstract\": \"DESCRIPTION (provided by applicant): Translesion synthesis (TLS) DNA polymerases (Pols) help ensure the continued progression of the replication fork by promoting replication through DNA lesions. In the proposed studies, we will determine the roles of a number of human TLS Pols in promoting replication through a variety of DNA lesions induced by environmental pollutants and carcinogens and by cellular oxidative damage. In particular, we will test the hypothesis that replication through DNA lesions in human cells occurs via two distinct modes in which Pols, ?, ?, k, Rev1, and ? mediate predominantly error-free TLS and act in a highly specialized manner dependent upon the DNA lesion, whereas Pol? performs lesion bypass in a more generalized and error-prone manner. Further, we will test the hypothesis that the more generalized and error-prone role of Pol? emanates from its ability to insert a purine nucleotide (nt), preferentially an A, opposite DNA lesions via a 'protein-template'-directed mechanism. To elucidate the genetic bases of error-free and mutagenic replication through DNA lesions in humans, we will carry out the following studies. In Aim 1, we will examine the contributions of various TLS Pols to error-free vs. mutagenic lesion bypass in human and mouse cells. The lesions to be studied include (6- 4) photoproduct induced by UV irradiation, 7,8-dihydro-8-oxogunaine (8-oxoG) generated from free-radical attack on guanine in DNA, 1,N6-ethenodeoxyadenosine (edA) generated from interaction of adenine with products of lipid peroxidation resulting from cellular oxidative damage and from exposure to chemical carcinogens, 1,N2-propano-2'-deoxyguanosine (PdG), a ring-closed form of acrolein generated from lipid peroxidation and which also is a ubiquitous environmental pollutant, and N2-dG adduct of the environmental carcinogen (+) anti-benzo[a]pyrene diol expoxide (BPDE). In Aim 2, biochemical studies will be done to examine the proficiency of Pol? in synthesizing DNA opposite (6-4) TT photoproduct, 8-oxoG, edA, PdG, and N2-dG BPDE. By steady-state kinetic analyses we will determine the catalytic efficiency of Pol? for inserting nucleotides opposite the DNA lesion and for carrying out the subsequent extension reaction, and biochemical studies will be done to test the hypothesis that Pol? inserts a purine nt opposite DNA lesions via a protein- template-directed mechanism. The genetic and biochemical studies we propose here are highly relevant for delineating the roles of TLS Pols in promoting error-free vs. mutagenic lesion bypass during replication, and for providing a comprehensive understanding of the genetic bases of mutagenesis and carcinogenesis induced by environmental and cellular DNA damaging agents in human cells. Our proposal for a predominantly error-free mode of TLS by Pols ?, ?, k, Rev1, and ? would predict a role for these Pols in cancer suppression, whereas a mutagenic mode of TLS by Pol? would predict a role for this Pol in enhancing genomic instability and carcinogenesis.  PUBLIC HEALTH RELEVANCE: DNA lesions are generated in human cells from cellular oxidative damage and from exposure to chemical and environmental carcinogens. The determination of roles of various DNA polymerases in promoting error-free vs. mutagenic lesion bypass during replication in human cells is important for providing a comprehensive understanding of the genetic bases of mutagenesis and carcinogenesis induced by environmental and cellular DNA damaging agents.  \",\n",
      "            \"agency\": \"NIH\",\n",
      "            \"budgetEndDate\": \"2012-11-30T00:00:00\",\n",
      "            \"budgetStartDate\": \"2012-01-27T00:00:00\",\n",
      "            \"cfdaCode\": \"113\",\n",
      "            \"congressionalDistrict\": \"14\",\n",
      "            \"contactPi\": \"PRAKASH, SATYA\",\n",
      "            \"department\": \"HHS\",\n",
      "            \"dunsId\": \"800771149\",\n",
      "            \"fy\": 2012,\n",
      "            \"ic\": \"NIEHS\",\n",
      "            \"latitude\": 29.243715,\n",
      "            \"longitude\": -94.856994,\n",
      "            \"nihApplId\": 8216401,\n",
      "            \"orgCity\": \"GALVESTON\",\n",
      "            \"orgCountry\": \"UNITED STATES\",\n",
      "            \"orgName\": \"UNIVERSITY OF TEXAS MEDICAL BR GALVESTON\",\n",
      "            \"orgState\": \"TX\",\n",
      "            \"orgZipCode\": \"775555302\",\n",
      "            \"otherPis\": null,\n",
      "            \"projectEndDate\": \"2016-11-30T00:00:00\",\n",
      "            \"projectNumber\": \"1R01ES020833-01\",\n",
      "            \"projectStartDate\": \"2012-01-27T00:00:00\",\n",
      "            \"smApplId\": 21001,\n",
      "            \"terms\": \" 7,8-dihydro-8-oxoguanine; Acrolein; adduct; Adenine; base; Benzo(a)pyrene; Biochemical; Biochemical Reaction; Biological; Bypass; carcinogenesis; Cells; chemical carcinogen; Deoxyguanosine; DNA; DNA biosynthesis; DNA Damage; DNA lesion; DNA-Directed DNA Polymerase; Ensure; Environmental Carcinogens; Environmental Pollutants; Epoxy Compounds; Exposure to; Family; Free Radicals; Frequencies; Genes; Genetic; Genomic Instability; Glycols; Guanine; Human; in vivo; Kinetics; Lesion; Lipid Peroxidation; Malignant Neoplasms; Mediating; Mus; Mutagenesis; Mutation; Nucleotides; oxidative damage; Pathway interactions; Plasmids; Play; Polymerase; Proteins; Purine Nucleotides; Relative (related person); Role; Scheme; Simian virus 40; Small Interfering RNA; System; Testing; ultraviolet irradiation\",\n",
      "            \"title\": \"GENETIC CONTROL OF REPLICATION THROUGH DNA LESIONS IN HUMANS, AND CARCINOGENESIS\",\n",
      "            \"totalCostAmount\": 344250\n",
      "        },\n",
      "        {\n",
      "            \"abstract\": \"DESCRIPTION (provided by applicant): Cross-links From Abasic Sites in Duplex DNA. A significant ongoing endeavor in cancer research and environmental toxicology involves the identification of important DNA-damage lesions and characterization of their ability to induce celldeath or cancer-causing mutations. The proposed work will characterize a novel family of DNA lesions that are derived from apurinic/abasic (AP) sites in duplex DNA. AP sites are generated by a wide variety of processes and may be the most common type of damage sustained by cellular DNA. This proposal builds upon our recent observations that AP sites can generate interstrand DNA-DNA cross-links via reactions with nucleobases on the opposing strand of the double helix. It is striking that AP sites can generate interstrand cross-links, which generally ar thought to be the most deleterious of all DNA lesions. Cross-links present an exceptional challenge to the DNA-repair machinery in human cells. The repair of cross-links may be error- prone and the resulting mutations in the genetic code could contribute to the etiology of both spontaneous and chemical-induced cancers. Left unrepaired, cross-links may block DNA transcription and replication. The exact nature of the cellular response(s) to these cross-links may be influenced by an individual's genetic makeup, especially with regard to defects or polymorphisms in their DNA damage response and repair machinery. The proposed work is significant because it will characterize how cross-links contribute to the propensity for endogenous or chemically-induced abasic sites to cause mutagenesis, cancer, cell dysfunction, senescence, and aging in humans. Our studies represent the first efforts to characterize the formation and biological consequences of these recently discovered DNA lesions. The proposed work will: (Aims 1 and 2) characterize the formation and chemical structures of two different AP-derived cross-links in duplex DNA, (Aim 3) use mass spectrometric methods to quantitatively measure the formation and repair of AP-derived cross-links in human cells exposed to agents that induce AP sites, including radiolysis and clinically-used alkylating agents and, (Aim 4) assess transcriptional bypass, replication, and repair of the dG-AP cross- link in human cells using a shuttle-vector methodology.  PUBLIC HEALTH RELEVANCE: - Cross-links From Abasic Sites in Duplex DNA Abasic sites are generated by the loss of a nucleobase 'letter' from the genetic code in cellular DNA. DNA abasic sites occur spontaneously in cells and also can be induced by a variety of natural toxins, industrial chemicals, pesticides, and anticancer drugs. The proposed work will examine the occurrence and repair of a novel family of interstrand DNA cross-links derived from abasic sites in duplex DNA. Cross-links present an exceptional challenge to cells because they prevent separation of the two strands of the double helix. The repair of the abasic-site-derived cross-links may be error prone and the resulting mutations in the genetic code could contribute to the etiology of both spontaneous and chemical-induced cancers. Left unrepaired, these cross-links may block DNA transcription and replication, leading to cell dysfunction, cell death, senescence, and aging. The exact nature of the cellular response(s) to these cross-links may be influenced by an individual's genetic makeup, especially with regard to defects or polymorphisms in their DNA damage response and repair machinery. This work is significant because it will characterize how AP-derived cross-links contribute to the propensity for a very common DNA lesion (the abasic site) to cause mutagenesis, cancer, cell dysfunction, senescence, and aging in humans.  \",\n",
      "            \"agency\": \"NIH\",\n",
      "            \"budgetEndDate\": \"2013-04-30T00:00:00\",\n",
      "            \"budgetStartDate\": \"2012-08-15T00:00:00\",\n",
      "            \"cfdaCode\": \"113\",\n",
      "            \"congressionalDistrict\": \"04\",\n",
      "            \"contactPi\": \"GATES, KENT S\",\n",
      "            \"department\": \"HHS\",\n",
      "            \"dunsId\": \"153890272\",\n",
      "            \"fy\": 2012,\n",
      "            \"ic\": \"NIEHS\",\n",
      "            \"latitude\": 38.948231,\n",
      "            \"longitude\": -92.327335,\n",
      "            \"nihApplId\": 8372731,\n",
      "            \"orgCity\": \"COLUMBIA\",\n",
      "            \"orgCountry\": \"UNITED STATES\",\n",
      "            \"orgName\": \"UNIVERSITY OF MISSOURI-COLUMBIA\",\n",
      "            \"orgState\": \"MO\",\n",
      "            \"orgZipCode\": \"652110001\",\n",
      "            \"otherPis\": \"WANG, YINSHENG  ; \",\n",
      "            \"projectEndDate\": \"2017-04-30T00:00:00\",\n",
      "            \"projectNumber\": \"1R01ES021007-01A1\",\n",
      "            \"projectStartDate\": \"2012-08-15T00:00:00\",\n",
      "            \"smApplId\": 21014,\n",
      "            \"terms\": \" Adenine; Aging; Alkylating Agents; anticancer research; Antineoplastic Agents; Biological; Biology; Bypass; cancer cell; Cancer Etiology; carcinogenesis; Cell Death; Cells; Chemical Structure; Chemicals; Chemistry; crosslink; cytotoxic; cytotoxicity; Defect; Diagnosis; DNA; DNA crosslink; DNA Damage; DNA lesion; DNA Repair; Employee Strikes; environmental toxicology; Etiology; Family; Functional disorder; Genetic; Genetic Code; Genetic Polymorphism; Genetic Transcription; Genomics; Guanine; Hot Spot; Human; Human Cell Line; In Vitro; Individual; Lead; Left; Lesion; Letters; Malignant Neoplasms; Measures; Methodology; Methods; Molecular; Mutagenesis; Mutation; Nature; novel; nucleobase; Pesticides; Physiological; prevent; Process; Reaction; repaired; response; Risk Assessment; Scheme; senescence; Shuttle Vectors; Site; Structure; Toxin; Work\",\n",
      "            \"title\": \"CROSS-LINKS AT ABASIC SITES IN DUPLEX DNA\",\n",
      "            \"totalCostAmount\": 330360\n",
      "        },\n",
      "        {\n",
      "            \"abstract\": \"DESCRIPTION (provided by applicant): DNA double-strand breaks (DSBs) are highly toxic DNA lesions that can occur spontaneously during DNA replication, but also can be introduced by exposure to ionizing radiation (IR) or to chemical mutagens. DSBs can lead to cell death, mutations and cancer, and defects in DSB repair (DSBR) underlie many human diseases, including disorders associated with cancer predisposition, radiosensitivity, immune dysfunction, neurodegeneration and premature aging. Nuclear Ubiquitous Casein and Cyclin-dependent Kinases Substrate (NUCKS) is a 27 kD, DNA-binding and vertebrate-specific protein with unknown function. NUCKS is abundant in rapidly growing cells and overexpressed in a variety of human cancers. Of particular interest, the NUCKS gene is located on human chromosome 1q32.1, a region that is commonly gained in breast cancer. Based on sequence homology to RAD51 Associated Protein 1 (RAD51AP1), a critical factor in homologous recombination repair (HRR), we have tested and determined a role for NUCKS in DSBR. Our results show that human cells with NUCKS knockdown are hypersensitive to IR and to chemical mutagens, and that NUCKS regulates not only HRR, but also cellular resistance to IR in G1-phase cells. In preliminary experiments using mice with heterozygous Trp53, we find that impaired Nucks function significantly increases the susceptibility to IR-induced tumors, alters the tumor spectrum and promotes metastasis. We now intend to further define how NUCKS regulates DSBR capacity using biochemical and cell-based assays (Aims 1 & 2). In addition, we will carry out a systematic study to further determine the phenotypic consequences of Nucks disruption in mice (Aim 2). Our findings showing a role for NUCKS in DSBR and in preventing radiation carcinogenesis are the first evidence for a biological function of NUCKS, a protein that was discovered more than 25 years ago. Given the importance of DSBR and HRR in tumor suppression and in the removal of DNA lesions induced by IR and other environmental mutagens, the results from our investigation will have direct relevance to risk predictions for health from environmental factors. In addition, our studies may also contribute to the improved diagnosis, prevention and treatment of cancer.  PUBLIC HEALTH RELEVANCE: Homologous recombination repair (HRR) is indispensable for maintaining genomic stability and preventing cancer, and this repair mechanism is impaired in BRCA1/2-related breast cancers. NUCKS (Nuclear Ubiquitous Casein and Cyclin-dependent Kinases Substrate) is an important protein involved in the HRR pathway, and is required for the repair of DNA damage caused by ionizing radiation and several chemotherapeutic agents. The proposed studies are designed to further elucidate the role of NUCKS in HRR and cancer-relevant processes, and will make an important contribution to understanding the role of DNA repair in cancer avoidance.  \",\n",
      "            \"agency\": \"NIH\",\n",
      "            \"budgetEndDate\": \"2013-03-31T00:00:00\",\n",
      "            \"budgetStartDate\": \"2012-06-28T00:00:00\",\n",
      "            \"cfdaCode\": \"113\",\n",
      "            \"congressionalDistrict\": \"13\",\n",
      "            \"contactPi\": \"WIESE, CLAUDIA\",\n",
      "            \"department\": \"HHS\",\n",
      "            \"dunsId\": \"078576738\",\n",
      "            \"fy\": 2012,\n",
      "            \"ic\": \"NIEHS\",\n",
      "            \"latitude\": 37.873838,\n",
      "            \"longitude\": -122.254934,\n",
      "            \"nihApplId\": 8400362,\n",
      "            \"orgCity\": \"BERKELEY\",\n",
      "            \"orgCountry\": \"UNITED STATES\",\n",
      "            \"orgName\": \"UNIVERSITY OF CALIF-LAWRENC BERKELEY LAB\",\n",
      "            \"orgState\": \"CA\",\n",
      "            \"orgZipCode\": \"947208050\",\n",
      "            \"otherPis\": null,\n",
      "            \"projectEndDate\": \"2017-03-31T00:00:00\",\n",
      "            \"projectNumber\": \"1R01ES021454-01A1\",\n",
      "            \"projectStartDate\": \"2012-06-28T00:00:00\",\n",
      "            \"smApplId\": 21023,\n",
      "            \"terms\": \" 1q32.1; Affect; base; Biochemical; Biological; Biological Assay; Biological Process; BRCA1 gene; Cancer Biology; cancer therapy; cancer type; Caseins; Cell Death; Cell Survival; Cells; Chemicals; chemotherapeutic agent; Complement; crosslink; Cyclin-Dependent Kinases; Defect; Dependence; Diagnosis; Disease; DNA; DNA Binding; DNA biosynthesis; DNA Damage; DNA Double Strand Break; DNA lesion; DNA Repair; DNA-Protein Interaction; Double Strand Break Repair; Embryo; environmental mutagens; Environmental Risk Factor; Excision; Exhibits; Exposure to; Fibroblasts; G1 Phase; General Population; Genes; Genetic Recombination; Genome Stability; Genomic Instability; Health; homologous recombination; Human; Human Chromosomes; human disease; Immune System Diseases; improved; in vivo; interest; Investigation; Ionizing radiation; Knockout Mice; Lead; malignant breast neoplasm; Malignant Neoplasms; Medical; Mus; Mutagens; Mutation; Neoplasm Metastasis; Nerve Degeneration; novel; Nuclear; overexpression; paralogous gene; Pathway interactions; Phenotype; Phosphorylation; Play; Post-Translational Protein Processing; Predisposition; Premature aging syndrome; prevent; Prevention; Process; Proteins; Radiation; radiation carcinogenesis; Radiation Tolerance; recombinase; recombinational repair; repaired; Research; Research Design; research study; Resistance; Risk; Role; Sequence Homology; Testing; tumor; Tumor Suppression; Tumor Suppressor Proteins; XRCC3 gene\",\n",
      "            \"title\": \"NUCKS, A NOVEL DOUBLE-STRAND BREAK REPAIR GENE, IMPLICATED IN CANCER BIOLOGY\",\n",
      "            \"totalCostAmount\": 416736\n",
      "        },\n",
      "        {\n",
      "            \"abstract\": \"DESCRIPTION (provided by applicant): Tobacco-smoking is the single major cause of cancer mortality in the US, and is a risk factor for a number of cancers including lung, upper aero-digestive tract, bladder and pancreas. One of the most powerful carcinogens in tobacco smoke is 4-(methylnitrosamino)-1-(3-pyridyl)-1-butanone (NNK). NNK is bioactivated to potent electrophiles that react to form methyl and 4-(3-pyridyl)-4-oxobutyl (POB) DNA adducts. Role of methyl-DNA adducts in carcinogenesis have been well characterized and the current paradigm is that methyl- DNA adducts are more important than POB-DNA adducts in the etiology of tobacco-induced cancers. However, recently it was found that O2-POB-dT adduct is the most persistent POB adduct in NNK-treated rodents. Our preliminary results show that O2-POB-dT is inefficiently repaired in human cells and is mutagenic in SOS-induced E. coli and mammalian cells. The objective of this application is to determine the mechanisms by which O2-POB-dT forms mutations and is repaired in mammalian cells. These goals will be examined in two specific aims: (1) to determine the polymerases involved in accurate and mutagenic bypass of O2-POB-dT and (2) to determine the mechanisms by which the POB-adducts are repaired. The polymerases involved in the bypass will be determined in cells in which specific polymerases are down regulated by siRNA, and in vitro with purified polymerases. The role of strand switching during translesion synthesis will be examined with cell- free extracts. The repair mechanisms will be evaluated by a combination of ex vivo and in vitro experiments. The roles of NER and BER will be evaluated ex vivo using cells with deficient repair proteins using a HPLC- MS/MS assay to measure levels of DNA adducts. The repair of O2-POB-dT via NER and BER will examined in vitro using synthetic oligodeoxynucleotides. The role of transcription-coupled NER for O6-POB-dG and O2- POB-dT will be probed with a novel modified host cell reactivation.  PUBLIC HEALTH RELEVANCE: One way by which tobacco smoke causes cancer is by damaging the DNA in the cell. If the damage is not repaired then when the cell replicates its DNA mistakes (mutations) are made that can lead to cancer. The goal of this project is to understand how the cell repairs and replicates an important class of DNA damage.  \",\n",
      "            \"agency\": \"NIH\",\n",
      "            \"budgetEndDate\": \"2013-03-31T00:00:00\",\n",
      "            \"budgetStartDate\": \"2012-07-20T00:00:00\",\n",
      "            \"cfdaCode\": \"113\",\n",
      "            \"congressionalDistrict\": \"15\",\n",
      "            \"contactPi\": \"SPRATT, THOMAS E.\",\n",
      "            \"department\": \"HHS\",\n",
      "            \"dunsId\": \"129348186\",\n",
      "            \"fy\": 2012,\n",
      "            \"ic\": \"NIEHS\",\n",
      "            \"latitude\": 40.273192,\n",
      "            \"longitude\": -76.675739,\n",
      "            \"nihApplId\": 8345834,\n",
      "            \"orgCity\": \"HERSHEY\",\n",
      "            \"orgCountry\": \"UNITED STATES\",\n",
      "            \"orgName\": \"PENNSYLVANIA STATE UNIV HERSHEY MED CTR\",\n",
      "            \"orgState\": \"PA\",\n",
      "            \"orgZipCode\": \"170332360\",\n",
      "            \"otherPis\": null,\n",
      "            \"projectEndDate\": \"2017-03-31T00:00:00\",\n",
      "            \"projectNumber\": \"1R01ES021762-01\",\n",
      "            \"projectStartDate\": \"2012-07-20T00:00:00\",\n",
      "            \"smApplId\": 21035,\n",
      "            \"terms\": \" 2'-Deoxythymidine; adduct; Affect; base; Base Excision Repairs; Base Pairing; Biological; Biological Assay; Biological Markers; Bladder; Butanones; Bypass; Cancer Etiology; carcinogenesis; Carcinogens; Cells; Chemicals; cigarette smoking; Deoxyguanosine; DNA; DNA Adducts; DNA Damage; DNA Repair; DNA-3-methyladenine glycosidase II; DNA-Directed DNA Polymerase; Embryo; Escherichia coli; Etiology; Gastrointestinal tract structure; genotoxicity; Goals; High Pressure Liquid Chromatography; Human; In Vitro; in vivo; Individual; innovation; insight; Ions; kidney cell; Knowledge; Lead; Lung; Malignant Neoplasms; Mammalian Cell; Measures; Methylation; Minor Groove; Modeling; Molecular; mortality; Mutagenesis; Mutation; Nitrosamines; novel; Nucleotide Excision Repair; Pancreas; Pathway interactions; Play; Polymerase; Predisposition; Property; Proteins; Pyrimidine; repaired; Research; research study; Resistance; Risk Factors; Rodent; Role; Small Interfering RNA; Smoking; Son of Sevenless Proteins; Time; Tobacco; Tobacco smoke; Transcription-Coupled Repair; United States; Work\",\n",
      "            \"title\": \"GENOTOXICITY AND REPAIR OF TOBACCO-SPECIFIC NITROSAMINE DNA ADDUCTS\",\n",
      "            \"totalCostAmount\": 350691\n",
      "        },\n",
      "        {\n",
      "            \"abstract\": \"DESCRIPTION (provided by applicant): The objective of this proposal is to identify novel biomarkers and molecular mechanisms of multi-walled carbon nanotube (MWCNT)-induced pulmonary diseases, including fibrosis, for early detection and treatment interventions. MWCNT have been widely used for various industrial applications. However, concern over potential MWCNT-induced toxicity has emerged, particularly due to the structural similarity between asbestos and MWCNT. In our preliminary studies, exposure of mice to MWCNT by pharyngeal aspiration results in significant pulmonary inflammation, damage, and fibrosis at 56 days post-exposure. We have identified MWCNT-induced gene signatures in the mouse model that could predict human lung cancer risk and progression. In the computational evaluation of more than 800 pathways, VEGF, ICAM-1, MCP-1, and TGF- are among the most significantly represented pathways in the mouse model. The involvement of these signaling pathways has been further validated in MWCNT-treated human small airway epithelial cells (SAEC). A co-culture model of SAEC with human microvascular endothelial cells (HMVEC) was recently developed by our laboratory to elucidate mechanisms for MWCNT-induced cellular response. Nevertheless, the long-term pulmonary responses to MWCNT in mice and their underlying molecular mechanisms remain to be resolved. Moreover, there are currently no clinically available biomarkers for early detection and no effective treatment inventions for MWCNT-induced pulmonary diseases, particularly lung fibrosis. We hypothesize that genomic profiling and computational toxicology analysis of in vitro and in vivo studies can identify novel mechanisms and biomarkers predictive of MWCNT-induced pulmonary injuries in humans, which will lead to early diagnostic detection and treatment interventions for MWCNT-induced pulmonary diseases, including fibrosis. We will utilize multidisciplinary approaches, including in vivo animal toxicology assays, in vitro cellulartoxicology assays, and computational modeling, to establish in vitro genomic signatures predictive of MWCNT-induced pulmonary diseases in the in vivo animal model and in humans and to explore the potential molecular targets for early treatment interventions. Aim 1 will evaluate the pulmonary dose-response and time course responses to MWCNT exposure in mice for up to 1 year. Aim 2 will determine the molecular mechanisms of MWCNT- induced injuries to the lung using an alveolo-capillary co-culture model. Aim 3 will identify mechanism-based gene signatures predictive of MWCNT-induced human pulmonary diseases using in vivo, in vitro, and patient data for early detection and treatment interventions. Aim 4 will perform integrated analyses of MWCNT-induced mRNA and miRNA changes and identify miRNA markers for early detection of MWCNT-induced lung fibrosis using non-invasive blood tests. We anticipate that this project will transform toxicological research of MWCNT- induced pulmonary diseases into strategies for environmental health protection and intervention. Results will fill th gap between in vivo/in vitro MWCNT-induced toxicity studies and risk assessment in humans.  PUBLIC HEALTH RELEVANCE: Multi-walled carbon nanotubes (MWCNT) have broad industrial applications. This project will assess the risk of MWCNT exposure in humans through animal and human cell model studies, and will develop early detection and treatment strategies for MWCNT-induced lung diseases, particularly fibrosis.  \",\n",
      "            \"agency\": \"NIH\",\n",
      "            \"budgetEndDate\": \"2013-05-31T00:00:00\",\n",
      "            \"budgetStartDate\": \"2012-09-18T00:00:00\",\n",
      "            \"cfdaCode\": \"113\",\n",
      "            \"congressionalDistrict\": \"01\",\n",
      "            \"contactPi\": \"GUO, NANCY LAN\",\n",
      "            \"department\": \"HHS\",\n",
      "            \"dunsId\": \"191510239\",\n",
      "            \"fy\": 2012,\n",
      "            \"ic\": \"NIEHS\",\n",
      "            \"latitude\": 39.63764,\n",
      "            \"longitude\": -79.946403,\n",
      "            \"nihApplId\": 8341498,\n",
      "            \"orgCity\": \"MORGANTOWN\",\n",
      "            \"orgCountry\": \"UNITED STATES\",\n",
      "            \"orgName\": \"WEST VIRGINIA UNIVERSITY\",\n",
      "            \"orgState\": \"WV\",\n",
      "            \"orgZipCode\": \"265066845\",\n",
      "            \"otherPis\": null,\n",
      "            \"projectEndDate\": \"2017-05-31T00:00:00\",\n",
      "            \"projectNumber\": \"1R01ES021764-01\",\n",
      "            \"projectStartDate\": \"2012-09-18T00:00:00\",\n",
      "            \"smApplId\": 21036,\n",
      "            \"terms\": \" Affect; Alveolar; angiogenesis; Animal Model; Animals; Apoptosis; Asbestos; base; Biological; Biological Assay; Biological Markers; Blood capillaries; Blood specimen; Blood Tests; Caliber; cancer initiation; cancer risk; capillary; carcinogenesis; CCL2 gene; Cell model; cell type; Characteristics; chemokine; Cicatrix; Coculture Techniques; Communication; Computer Simulation; Cultured Cells; cytokine; cytotoxicity; Data; Detection; Development; Diagnostic; Dose; Early Diagnosis; Early treatment; effective therapy; Endothelial Cells; Environmental Health; Epithelial; Epithelial Cells; Evaluation; Exhibits; Exposure to; Fiber; Fibrosis; Future; Gases; Genes; genome-wide; Genomics; Health Hazards; Health protection; Human; Hyperplasia; In Vitro; in vitro Assay; in vivo; Inflammation; Injury; Intercellular adhesion molecule 1; interdisciplinary approach; Intervention; Laboratories; Lead; Length; Lesion; Lung; Lung diseases; Malignant neoplasm of lung; Malignant Neoplasms; Measurement; Measures; Mesenchymal; Messenger RNA; MicroRNAs; Modeling; Molecular; Molecular Analysis; Molecular Profiling; Molecular Target; mouse model; mRNA Expression; multi walled carbon nanotube; Mus; Mutation; nanoscale; novel; particle; Pathway interactions; Patients; Pattern; Permeability; Pharmaceutical Preparations; Phosphorylation; Pneumonia; Production; Pulmonary Fibrosis; Reaction Time; Reactive Oxygen Species; Research; response; Risk; Risk Assessment; Screening procedure; Shapes; Signal Pathway; Signal Transduction; Signal Transduction Alteration; Structure of parenchyma of lung; Systems Biology; Testing; Tissues; Toxic effect; Toxicology; treatment strategy; tumor progression; Vascular Endothelial Growth Factors\",\n",
      "            \"title\": \"SYSTEMATIC ASSESSMENT OF MULTI-WALLED CARBON NANOTUBES IN PULMONARY DISEASE\",\n",
      "            \"totalCostAmount\": 333000\n",
      "        },\n",
      "        {\n",
      "            \"abstract\": \"DESCRIPTION (provided by applicant): Epidemiological studies have long suggested a critical, but poorly understood, link between toxic environmental exposures early in life and the development of human disease(s) later in life. Environmental toxins produce oxidative stress in cells, and a link between oxidative stress and epigenetic changes in a cell has implications for a variety of human diseases including cancer, infertility, and multiple neurodegenerative disorders. Though limited data exists, it has also been suggested that environmental toxins may disrupt both DNA methylation patterns and chromatin structure, which can confer not only heritable changes in gene expression, but also affect overall genomic stability. However, a direct link between environmental toxins, oxidative damage and epigenetic changes has not yet been established and is the primary focus of this proposal. It is proposed that exposure to common environmental contaminants during development or during key windows of susceptibility (WOS) throughout a lifetime may induce abnormal epigenetic, pre-malignant changes in developing/differentiating cells. It is expected that this window of susceptibility coincides with the normal epigenetic remodeling required for lineage commitment during development or periods of cellular repair/regeneration. To test this idea the following three specific aims are proposed: Specific Aim 1: Determine the in vitro WOS during which the environmental toxin paraquat induces DNA methylation changes in differentiating stem cells. It is hypothesized that stem cells are vulnerable to errors in epigenetic remodeling when they receive a signal to actively remodel their chromatin, i.e. during development and tissue regeneration; Specific Aim 2: Investigate whether paraquat exposure disrupts the composition and/or functional recruitment of stem cell transcriptional repressive complexes to tumor suppressor gene promoters. It is postulated that the introduction of DNA damaging agents and prolonged exposure of cells to environmental toxins during a window of active epigenetic remodeling may cause disruption and/or abnormal recruitment of polycomb repressive complexes (PRC), the stress response protein SIRT1, and/or DNA methyltransferases to gene promoter regions with abnormal DNA methylation changes; and Specific Aim 3: Determine whether low-dose, chronic, in utero paraquat exposure induces DNA methylation changes in the developing mouse brain, disrupts neural cell plasticity, and/or enhances the malignant potential of neural stem cells in the F1 generation. It is also hypothesized that the dynamic nature of epigenetic remodeling during embryonic development may leave cells particularly vulnerable to the effects of environmental toxins. Prolonged exposure may result in the accumulation of abnormal, promoter associated DNA methylation, inhibit the ability of neuronal stem cells to properly differentiate, and may induce pre-malignant changes including abnormally high mitotic rates, nuclear atypia, or focal necrosis in these cells.  PUBLIC HEALTH RELEVANCE: A role for environmental toxins in tumor initiation has long been suggested and continues to be an area of particular interest for cancer researchers, as populations with increased exposure to pesticides, herbicides, fungicides, and heavy metals have been shown to have increased cancer incidence. Chemical exposures in utero are of particular concern, as development is dependent on a tightly orchestrated epigenetic remodeling program. This proposal seeks to directly assay the role of exposure to environmental toxins in the induction of abnormal epigenetic drift and disruption of stem cell pluripotency, and investigates a potential role for these toxins in tumor etiology. This proposal will investigate the mechanisms by which oxidative damage may be mechanistically linked to these epigenetic changes and mediate pre- malignant changes at a molecular level in this vulnerable stem cell population. We will also identify novel relationships between the stem cell and developmental proteins that may be involved (including the strongly implicated Polycomb family of transcriptional repressors) and both known and unknown epigenetic regulatory protein complexes. This study has significant potential for increasing our understanding the role of environmental epigenomics in tumor-initiating events in humans, and seeks to identify key mechanisms responsible for these changes. This study is also expected to identify embryonic factors and epigenetic modifications that play a role in tumor initiation and may be exploited for therapeutic purposes.   \",\n",
      "            \"agency\": \"NIH\",\n",
      "            \"budgetEndDate\": \"2013-05-31T00:00:00\",\n",
      "            \"budgetStartDate\": \"2012-09-01T00:00:00\",\n",
      "            \"cfdaCode\": \"113\",\n",
      "            \"congressionalDistrict\": \"00\",\n",
      "            \"contactPi\": \"OHM, JOYCE ELLEN\",\n",
      "            \"department\": \"HHS\",\n",
      "            \"dunsId\": \"102280781\",\n",
      "            \"fy\": 2012,\n",
      "            \"ic\": \"NIEHS\",\n",
      "            \"latitude\": 47.921895,\n",
      "            \"longitude\": -97.070834,\n",
      "            \"nihApplId\": 8390167,\n",
      "            \"orgCity\": \"GRAND FORKS\",\n",
      "            \"orgCountry\": \"UNITED STATES\",\n",
      "            \"orgName\": \"UNIVERSITY OF NORTH DAKOTA\",\n",
      "            \"orgState\": \"ND\",\n",
      "            \"orgZipCode\": \"582026059\",\n",
      "            \"otherPis\": null,\n",
      "            \"projectEndDate\": \"2017-05-31T00:00:00\",\n",
      "            \"projectNumber\": \"1R01ES022030-01\",\n",
      "            \"projectStartDate\": \"2012-09-01T00:00:00\",\n",
      "            \"smApplId\": 21051,\n",
      "            \"terms\": \" Affect; Age; Area; biological adaptation to stress; Biological Assay; Brain; Cells; Chemical Exposure; ChIP-seq; Chromatin; chromatin remodeling; Chromatin Structure; Chronic; Complex; Cytosine; Data; Development; Disease susceptibility; disorder risk; DNA; DNA Damage; DNA Methylation; DNA Methyltransferase; DNA Modification Methylases; Dose; Embryo; Embryonic Development; Environmental Exposure; Environmental Pollution; Epidemiologic Studies; Epigenetic Process; epigenomics; Etiology; Event; Exposure to; Family; Functional disorder; Gene Expression; Gene Expression Profile; Gene Expression Regulation; Generations; Genes; genetic regulatory protein; Genome Stability; Heavy Metals; Herbicides; Human; human disease; in utero; In Vitro; Incidence; Industrial fungicide; Infertility; interest; Lead; Left; Life; Link; Maintenance; Malignant - descriptor; Malignant Neoplasms; Mediating; metaplastic cell transformation; Mitotic; Modeling; Modification; Molecular; Mus; Natural regeneration; Nature; Necrosis; nerve stem cell; Neurodegenerative Disorders; Neuronal Differentiation; Neurons; novel; Nuclear Atypia; oxidative damage; Oxidative Stress; Paraquat; Pattern; Pesticides; Play; pluripotency; Polycomb; Population; Predisposition; Premalignant; programs; promoter; Promoter Regions; protein complex; Proteins; repaired; Research; Research Personnel; RNA Sequences; Role; Signal Transduction; stem cell biology; stem cell population; Stem cells; Stimulus; Stress; Testing; theories; Therapeutic; tissue regeneration; tool; Toxic effect; Toxic Environmental Substances; Toxin; Transcription Repressor/Corepressor; tumor; Tumor Biology; tumor initiation; Tumor Suppressor Genes\",\n",
      "            \"title\": \"ENVIRONMENTAL TOXINS AND STEM CELL EPIGENETIC REMODELING\",\n",
      "            \"totalCostAmount\": 310500\n",
      "        },\n",
      "        {\n",
      "            \"abstract\": \"DESCRIPTION (provided by applicant): Oncogenic HPV, mucosal immune response and risk of HIV acquisition. Approximately 33 million people are infected with HIV worldwide, with 2.7 million new infections in 2008 alone. The infection disproportionately affects sub-Saharan Africa with 67% of all HIV infections and 75% of all HIV/AIDS deaths. High-risk human papillomavirus (HR-HPV) infection leads to anogenital cancers, including cervical cancer (the third leading cause of cancer mortality in women worldwide). HR-HPV may also be a risk factor for HIV acquisition, presumably due to recruitment of HIV target cells into the mucosa. However, the genital mucosal immunologic microenvironment and HIV/HR-HPV interactions are poorly understood. Male circumcision reduces the risk of HIV and HR-HPV, demonstrating the critical role of the foreskin in acquisition and transmission of these viral infections. Foreskin tissue, genital swabs of men and their female partners, and epidemiologic data from male circumcisions in Rakai, Uganda, provide biological samples to assess HIV and HR-HPV mucosal immunologic interactions. We hypothesize that HR-HPV clearance induces a distinct proinflammatory cytokine/chemokine profile and increased foreskin mucosal T-cell densities which are associated with increased risk of HIV acquisition. We propose the following aims. 1) Assess whether clearance of pre-existing HR-HPV infection and/or acquisition of a new HR-HPV genotype are associated with an increased risk of HIV acquisition in men and women using a case-control design in which cases will be HIV seroconverters and controls will be persistently HIV-negative individuals. 2) Characterize the foreskin cellular mucosal immunologic response associated with HR-HPV in HIV+ and HIV-negative men. 3) Compare cytokine/chemokine concentrations in penile swabs associated with HR-HPV and HIV infection status. <4) Determine whether increased foreskin immunologic cell densities and penile proinflammatory cytokines/chemokines are associated with HR-HPV clearance.> The foreskin mucosa, genital swabs and epidemiologic data provide a unique opportunity to study HIV and HR-HPV infection and could potentially inform the development of HIV preventive measures. This K23 grant will contribute to the professional and scientific development of a physician scientist. The applicant has completed an NIH-funded MD/PhD program and a clinical pathology residency. This grant will enable the applicant to develop proficiency in epidemiology and biostatistics and, building on the proposed research program, to design and manage an independent RO1-level research program in molecular/clinical epidemiology. The applicant will utilize the strengths of his mentors and the Rakai Health Science Program in Uganda (one of the largest and longest-running population-based HIV and infectious diseases cohorts in sub- Saharan Africa) to learn how to design, implement, manage and analyze international HIV research studies.  \",\n",
      "            \"agency\": \"NIH\",\n",
      "            \"budgetEndDate\": \"2013-06-30T00:00:00\",\n",
      "            \"budgetStartDate\": \"2012-07-01T00:00:00\",\n",
      "            \"cfdaCode\": \"855\",\n",
      "            \"congressionalDistrict\": \"07\",\n",
      "            \"contactPi\": \"TOBIAN, AARON A\",\n",
      "            \"department\": \"HHS\",\n",
      "            \"dunsId\": \"001910777\",\n",
      "            \"fy\": 2012,\n",
      "            \"ic\": \"NIAID\",\n",
      "            \"latitude\": 39.298737,\n",
      "            \"longitude\": -76.594342,\n",
      "            \"nihApplId\": 8281450,\n",
      "            \"orgCity\": \"BALTIMORE\",\n",
      "            \"orgCountry\": \"UNITED STATES\",\n",
      "            \"orgName\": \"THE JOHNS HOPKINS UNIVERSITY\",\n",
      "            \"orgState\": \"MD\",\n",
      "            \"orgZipCode\": \"212051832\",\n",
      "            \"otherPis\": null,\n",
      "            \"projectEndDate\": \"2016-06-30T00:00:00\",\n",
      "            \"projectNumber\": \"5K23AI093152-02\",\n",
      "            \"projectStartDate\": \"2011-07-01T00:00:00\",\n",
      "            \"smApplId\": 21096,\n",
      "            \"terms\": \" Adult; Affect; Africa; Africa South of the Sahara; AIDS prevention; AIDS/HIV problem; Anogenital cancer; arm; Biological; Biological Assay; Biometry; Biopsy; Cancer Etiology; case control; Case-Control Studies; CD8-Positive T-Lymphocytes; Cell Density; Cells; Cervical; Cervical dysplasia; Cessation of life; chemokine; clinical epidemiology; Clinical Pathology; cohort; Communicable Diseases; Communities; condoms; cytokine; Dendritic Cells; design; Detection; Development; Doctor of Philosophy; Dysplasia; Enrollment; Epidemic; epidemiologic data; Epidemiology; Female; Funding; Future; Genital system; Genotype; Grant; Health Sciences; HIV; HIV Infections; HIV prevention trial; HPV-High Risk; Human papilloma virus infection; Human Papilloma Virus Vaccine; Human Papillomavirus; Immune; Immunoassay; Immunobiology; Immunohistochemistry; Immunologic Markers; Immunologics; improved; Incidence; Individual; Infection; International; Learning; Logistic Regressions; male; Male Circumcision; Malignant neoplasm of cervix uteri; Malignant neoplasm of penis; Measurement; Measures; men; Mentors; Modeling; Molecular; mortality; Mucosal Immune Responses; Mucous Membrane; Natural immunosuppression; Neoplasms; Odds Ratio; Oncogenic; penis foreskin; Physicians; population based; Prevalence; Prevention strategy; Preventive; Procedures; programs; Proteins; Recruitment Activity; Research; research study; Residencies; response; Risk; Risk Behaviors; Risk Factors; Role; Running; Sampling; Scientist; Screening procedure; Services; Swab; Time; Tissue Sample; Tissues; T-Lymphocyte; transmission process; Uganda; Ulcer; United States National Institutes of Health; Virus; Virus Diseases; Woman\",\n",
      "            \"title\": \"ONCOGENIC HPV, MUCOSAL IMMUNE RESPONSE AND RISK OF HIV ACQUISITION\",\n",
      "            \"totalCostAmount\": 136754\n",
      "        },\n",
      "        {\n",
      "            \"abstract\": \"DESCRIPTION (provided by applicant): The candidate, Dr. Ying Taur, an infectious diseases instructor at Memorial Sloan-Kettering Cancer Center, seeks to achieve an academic career as a clinical researcher in the field of infectious diseases. This project proposes a five-year plan where he can take steps to become an independent physician-scientist, under the mentorship of Drs. Eric Pamer and Kent Sepkowitz. This project seeks to study the intestinal flora of patients undergoing allogeneic hematopoietic stem cell transplantation (HSCT), and determine the relationship between flora and the risk for two infections, bacteremia with vancomycin-resistant Enterococcus (VRE) and Clostridium difficile-associated diarrhea (CDAD). This will be done by collecting stool samples serially from patients undergoing HSCT. Using high-throughput sequencing methods, each stool sample will be analyzed in order to characterize the microbial composition. A variety of microbial ecology measures will be calculated and used to describe the intestinal flora. Correlations will be made with risk of VRE bacteremia and CDAD. The specific aims of the project are as follows: (1) Characterize the changes in intestinal flora that occur during HSCT, (2) Identify intestinal flora profiles that impart risk for VRE bacteremia, (3) Identify intestinal flora profiles that impart risk for CDAD. This project will be able to identify the 'key players' of the gut flora, whose presence or absence play an essential role in the development of infection. Armed with a thorough understanding of the etiologic relationship between the gut flora and infectious diseases, it will be possible to design intelligent strategies to prevent, diagnose, or treat these infections in a definitive manner. Throughout the five-year period, the candidate will meet regularly with his mentors, interact with other pertinent experts at MSKCC, and supplement his training with structured meetings, conferences, and coursework. At the end of the proposal period, he will be able to achieve independent status as a clinical researcher in infectious diseases.  \",\n",
      "            \"agency\": \"NIH\",\n",
      "            \"budgetEndDate\": \"2013-06-30T00:00:00\",\n",
      "            \"budgetStartDate\": \"2012-07-01T00:00:00\",\n",
      "            \"cfdaCode\": \"855\",\n",
      "            \"congressionalDistrict\": \"12\",\n",
      "            \"contactPi\": \"TAUR, YING\",\n",
      "            \"department\": \"HHS\",\n",
      "            \"dunsId\": \"064931884\",\n",
      "            \"fy\": 2012,\n",
      "            \"ic\": \"NIAID\",\n",
      "            \"latitude\": 40.764386,\n",
      "            \"longitude\": -73.956799,\n",
      "            \"nihApplId\": 8303439,\n",
      "            \"orgCity\": \"NEW YORK\",\n",
      "            \"orgCountry\": \"UNITED STATES\",\n",
      "            \"orgName\": \"SLOAN-KETTERING INSTITUTE FOR CANCER RES\",\n",
      "            \"orgState\": \"NY\",\n",
      "            \"orgZipCode\": \"100656007\",\n",
      "            \"otherPis\": null,\n",
      "            \"projectEndDate\": \"2016-06-30T00:00:00\",\n",
      "            \"projectNumber\": \"5K23AI095398-02\",\n",
      "            \"projectStartDate\": \"2011-07-20T00:00:00\",\n",
      "            \"smApplId\": 21104,\n",
      "            \"terms\": \" Adult; Allogenic; Antibiotics; antineoplastic antibiotics; arm; Bacteremia; Bacteria; base; Bone Marrow Transplantation; career; Characteristics; Clinical; Clostridium difficile; Communicable Diseases; design; Development; Diagnosis; Diarrhea; Disease; Ecology; Elements; Feces; Five-Year Plans; Gastrointestinal tract structure; Grouping; Health; Healthcare; Hematopoietic Stem Cell Transplantation; improved; indexing; Infection; instructor; interest; Intestines; Measures; meetings; Memorial Sloan-Kettering Cancer Center; Mentors; Mentorship; Methods; microbial; microbial community; microbiome; Outcome; Pathogenesis; patient population; Patients; Phylogenetic Analysis; Physicians; Play; Population; Predisposition; prevent; prospective; Recombinant DNA; Research; Research Personnel; Risk; Risk Factors; Role; Running; Sampling; Scientist; Specimen; Structure; symposium; Techniques; Time; Training; Transplantation; Vancomycin resistant enterococcus\",\n",
      "            \"title\": \"EFFECTS OF THE INTESTINAL MICROBIOTA ON INFECTIONS DURING BONE MARROW TRANSPLANT\",\n",
      "            \"totalCostAmount\": 130113\n",
      "        },\n",
      "        {\n",
      "            \"abstract\": \"DESCRIPTION (provided by applicant): Dr. Jun Mao, an Assistant Professor of Family Medicine and certified acupuncturist, is applying for a Mentored Patient-Oriented Research Career Development Award (K23) to become an independent translational researcher focusing on complementary and alternative medicine (CAM). The candidate seeks to conduct multidisciplinary research on the effects and mechanisms of acupuncture for symptom management in cancer. The substantial placebo effect observed in previous acupuncture trials presents a major limitation for studying acupuncture and interpreting experimental results. Therefore, the candidate proposes a comprehensive research and training program to investigate the placebo effect of acupuncture using hot flashes among breast cancer survivors as a clinical model. This K23 will provide the candidate with additional training in advanced research design, neuroimaging, and behavioral science to augment his initial research training in epidemiology. It will also provide mentored research experience through the conduct of a randomized controlled trial (RCT) to determine the magnitude of response to placebo acupuncture vs. placebo pill for hot flashes among breast cancer survivors. Furthermore, a feasibility study on functional brain imaging has been proposed among a subset of acupuncture trial participants to explore the use of serotonin transport binding as a potential biomarker to visualize acupuncture's effects for hot flashes. A group of multidisciplinary mentors has been assembled to support the candidate's career and research development, including Drs. John Farrar (acupuncture research methodology), Marjorie Bowman (leadership development and women's health), Andrew Newberg (neuroimaging), Caryn Lerman (behavioral science), Linda Jacobs (cancer survivorship), Deborah Bruner (symptom assessment), Sharon Xie (biostatistics) and Jay Amsterdam (biological psychiatry). The proposed research will provide an important methodological step leading to further investigation of the effects and mechanisms of acupuncture for hot flashes. Because hot flashes are a common symptom experienced by breast cancer survivors, and because existing conventional therapies are highly limited, this research and its follow-up investigation have the potential to benefit millions of women with a history of breast cancer. Furthermore, the mentored research experience and protected time provided by this K23 will allow Dr. Mao to develop into a leading translational scientist who can bridge neurobiology and clinical response of acupuncture for symptom management in cancer.  \",\n",
      "            \"agency\": \"NIH\",\n",
      "            \"budgetEndDate\": \"2014-04-30T00:00:00\",\n",
      "            \"budgetStartDate\": \"2012-05-01T00:00:00\",\n",
      "            \"cfdaCode\": \"213\",\n",
      "            \"congressionalDistrict\": \"02\",\n",
      "            \"contactPi\": \"MAO, JUN J\",\n",
      "            \"department\": \"HHS\",\n",
      "            \"dunsId\": \"042250712\",\n",
      "            \"fy\": 2012,\n",
      "            \"ic\": \"NCCAM\",\n",
      "            \"latitude\": 39.952982,\n",
      "            \"longitude\": -75.192768,\n",
      "            \"nihApplId\": 8248617,\n",
      "            \"orgCity\": \"PHILADELPHIA\",\n",
      "            \"orgCountry\": \"UNITED STATES\",\n",
      "            \"orgName\": \"UNIVERSITY OF PENNSYLVANIA\",\n",
      "            \"orgState\": \"PA\",\n",
      "            \"orgZipCode\": \"191046205\",\n",
      "            \"otherPis\": null,\n",
      "            \"projectEndDate\": \"2014-04-30T00:00:00\",\n",
      "            \"projectNumber\": \"5K23AT004112-05\",\n",
      "            \"projectStartDate\": \"2008-08-01T00:00:00\",\n",
      "            \"smApplId\": 21141,\n",
      "            \"terms\": \" Acupuncture procedure; Animals; Area; arm; base; Behavioral Sciences; Binding; Biological; Biological Markers; Biological Psychiatry; Biometry; Brain imaging; Cancer Patient; Cancer Survivor; Cancer Survivorship; cancer therapy; career development; Caring; Clinical; Clinical Trials; Complementary and alternative medicine; Consent; Control Groups; Controlled Clinical Trials; conventional therapy; Data; Development; diaries; Distress; Education; Effectiveness; efficacy evaluation; Epidemiology; experience; Family Practice; Feasibility Studies; follow-up; gabapentin; General Population; Goals; Grant; Hot flushes; Image; Individual; Interdisciplinary Study; interest; Intervention; Investigation; Leadership; Literature; malignant breast neoplasm; Malignant Neoplasms; Mao-to; Measures; Medical; Mentored Patient-Oriented Research Career Development Award; Mentors; Modeling; multidisciplinary; Needles; Neuraxis; Neurobiology; neuroimaging; Neurotransmitters; Participant; Patients; Physiologic Thermoregulation; pill; Placebo Control; Placebo Effect; Placebos; Play; Population; preference; Process; professor; programs; psychosocial; Public Health; Puncture procedure; Quality of life; Randomized; Randomized Controlled Trials; Recording of previous events; Research; research and development; Research Design; Research Methodology; Research Personnel; Research Training; response; Role; Scientist; Serotonin; serotonin transporter; Skin; Study models; symptom management; Symptoms; System; Time; Tomography, Emission-Computed, Single-Photon; Training; Training Programs; treatment effect; Woman; Women's Health\",\n",
      "            \"title\": \"ELUCIDATING THE PLACEBO EFFECTS OF ACUPUNCTURE: HOT FLASHES AS A CLINICAL MODEL\",\n",
      "            \"totalCostAmount\": 131895\n",
      "        },\n",
      "        {\n",
      "            \"abstract\": \"DESCRIPTION (provided by applicant): This mentored career award outlines a five-year training program for Dr. Ahuja's transition to scientific independence in the field of cancer biology. Dr. Ahuja proposes to expand her scientific skills and knowledge and it is expected that she would attain independent investigator status at the end of this award period. Dr. Ahuja will continue an established scientific mentorship with Dr. Baylin, an expert in epigenetics. She will also be mentored by Dr. Leach, an expert in pancreatic developmental biology and a fellow surgeon who will provide valuable clinical guidance. She has the full support of her Department to protect at least 75% of her time during the award period. Dr. Ahuja has laboratory and office space within Dr. Baylin's laboratory and is fully immersed in the research environment. The laboratory is located in the Cancer Research Buildings at Johns Hopkins and has access to many established scientists with an environment that favors collaboration and support of clinician scientists. The research plans focuses on understanding the role of DNA methylation in colorectal cancer metastases and recurrence and understand its applicability as translational markers. Epigenetic gene silencing involving aberrant DNA methylation of promoter-associated CpG islands functions as a central mechanism of inactivation of tumor-suppressor genes in colorectal cancer. We hypothesize that methylation profiles hold enormous potential as prognostic markers for recurrence in colorectal cancer and methylation profiles may allow molecular staging of colorectal cancer. In order to test these hypotheses, the following aims are proposed: 1) To utilize methylation profiles to predict recurrence in early-stage colon cancers. 2) To establish the methylation changes that underlie the process of metastasis in colorectal cancers in order to identify novel markers for predicting recurrence and metastases 3) To develop a molecular staging system for colorectal cancer. The research will build on exciting preliminary results demonstrating the utility of methylation profiles to predict recurrence. In addition, a systematic understanding of late events in colorectal carcinogenesis will be performed. Relevance: Colorectal cancer is a leading cause of cancer death worldwide and in the United States. The long-term goal of this proposal is to utilize knowledge of the newly emerging field, termed epigenetics, to establish novel markers for predicting patients at high-risk for colorectal cancer recurrence and to define novel cancer treatments.  \",\n",
      "            \"agency\": \"NIH\",\n",
      "            \"budgetEndDate\": \"2015-07-31T00:00:00\",\n",
      "            \"budgetStartDate\": \"2012-08-01T00:00:00\",\n",
      "            \"cfdaCode\": \"398\",\n",
      "            \"congressionalDistrict\": \"07\",\n",
      "            \"contactPi\": \"AHUJA, NITA\",\n",
      "            \"department\": \"HHS\",\n",
      "            \"dunsId\": \"001910777\",\n",
      "            \"fy\": 2012,\n",
      "            \"ic\": \"NCI\",\n",
      "            \"latitude\": 39.298737,\n",
      "            \"longitude\": -76.594342,\n",
      "            \"nihApplId\": 8304362,\n",
      "            \"orgCity\": \"BALTIMORE\",\n",
      "            \"orgCountry\": \"UNITED STATES\",\n",
      "            \"orgName\": \"THE JOHNS HOPKINS UNIVERSITY\",\n",
      "            \"orgState\": \"MD\",\n",
      "            \"orgZipCode\": \"212051832\",\n",
      "            \"otherPis\": null,\n",
      "            \"projectEndDate\": \"2015-07-31T00:00:00\",\n",
      "            \"projectNumber\": \"5K23CA127141-05\",\n",
      "            \"projectStartDate\": \"2008-08-01T00:00:00\",\n",
      "            \"smApplId\": 21156,\n",
      "            \"terms\": \" Aberrant DNA Methylation; adenoma; Adjuvant; anticancer research; Area; Award; base; Biological Markers; Blinded; Breast; Cancer Biology; cancer cell; Cancer cell line; Cancer Etiology; Cancer Patient; Cancer Prognosis; cancer recurrence; cancer therapy; carcinogenesis; career; case control; Cessation of life; Chromatin; Clinical; Collaborations; Colon; colon carcinogenesis; Colon Carcinoma; Colorectal; Colorectal Cancer; Colorectal Neoplasms; CpG dinucleotide; CpG Islands; Data; demethylation; Development; Developmental Biology; Disease; DNA Methylation; Environment; Epigenetic Process; Event; follow-up; gene discovery; Gene Expression; Gene Expression Profile; Gene Silencing; Genes; Genetic Transcription; Genome; Genomics; Goals; high risk; Human; Hypermethylation; indexing; Knowledge; Laboratories; Lung; lymph nodes; Malignant neoplasm of lung; Malignant neoplasm of pancreas; Malignant Neoplasms; Mentors; Mentorship; metastatic colorectal; Methylation; Modification; Molecular; Neoplasm Metastasis; Nested Case-Control Study; novel; novel marker; Oncogenes; Pancreas; Pathogenesis; Patients; Pattern; Pilot Projects; Primary Neoplasm; Process; Prognostic Marker; programs; promoter; Recurrence; Regimen; Research; Research Personnel; research study; Risk; Role; Scientist; skills; Staging; Staging System; Subgroup; Surgeon; Testing; Therapeutic; Time; Training Programs; tumor; Tumor Suppressor Genes; Tumor-Suppressor Gene Inactivation; United States\",\n",
      "            \"title\": \"EPIGENETIC MARKERS INVOLVED IN COLORECTAL CANCER RECURRENCE AND METASTASES\",\n",
      "            \"totalCostAmount\": 140940\n",
      "        },\n",
      "        {\n",
      "            \"abstract\": \"DESCRIPTION (provided by applicant): Therapies for breast cancer (BC) should be guided by biological features, such as endocrine or HER2/neu dependence. Pre-clinical and early clinical data suggest synergism between PI3K/Akt and EGFR signaling inhibition in BC. We hypothesize that BC with low PTEN levels or Akt hyperactivation will have a higher response to combined blockade of the EGFR and PI3K/Akt signaling, and the microarray analysis of tumors that respond to the treatment combination will resemble established EGFR and PI3K/Akt/mTOR dependence gene signatures. As a future independent breast cancer translational researcher, with the necessary mentorship from Dr. Arteaga and Dr. Rothenberg and full institutional support beyond the allowable costs of this award, I propose a clinical trial in the metastatic setting, in which the correlative studies would help predict clinical responses, impacting future patient selection and trial design. In this research proposal, we will: (1) Determine the safety profile and efficacy of the mTOR inhibitor everolimus and the EGFR inhibitor erlotinib combination in a phase l/ll clinical trial for patients with metastatic BC; (2) Determine biomarkers of response to the erlotinib/everolimus combination in BC. PTEN, pAkt, pS6, and EGFR (IHC) and PI3K mutations on exons 9 and 20 sequenced in collected paraffin blocks will be correlated with clinical outcome in Aim 1; and (3) Determine if gene expression profiling can identify gene expression signatures that will predict sensitivity to EGFR and mTOR inhibitors. We will identify a signature suggestive of EGFR and mTOR dependence using the clinical outcome on the clinical trial as the supervising parameter, in addition to mining the generated data sets from all collected material to answer if previously established EGFR and PI3K/Akt/mTOR gene signatures will predict response to treatment. We will also correlate IHC results of Aim 2 with the subtypes of breast cancer determined by molecular classification based on their gene expression (luminal A, luminal B, HER2?, and basal-like). Relevance to public: This research will focus on discovery of a 'molecular signature' based on genes and proteins involved in breast cancer's growth and survival, which will help identify groups of breast cancer patients that may benefit from new treatment combinations that target these particular genes and proteins. This research could impact design of future clinical trials, their answers could ultimately reduce breast cancer mortality.  \",\n",
      "            \"agency\": \"NIH\",\n",
      "            \"budgetEndDate\": \"2013-08-31T00:00:00\",\n",
      "            \"budgetStartDate\": \"2012-09-01T00:00:00\",\n",
      "            \"cfdaCode\": \"398\",\n",
      "            \"congressionalDistrict\": \"05\",\n",
      "            \"contactPi\": \"MAYER, INGRID A\",\n",
      "            \"department\": \"HHS\",\n",
      "            \"dunsId\": \"965717143\",\n",
      "            \"fy\": 2012,\n",
      "            \"ic\": \"NCI\",\n",
      "            \"latitude\": 36.140134,\n",
      "            \"longitude\": -86.796274,\n",
      "            \"nihApplId\": 8312344,\n",
      "            \"orgCity\": \"NASHVILLE\",\n",
      "            \"orgCountry\": \"UNITED STATES\",\n",
      "            \"orgName\": \"VANDERBILT UNIVERSITY\",\n",
      "            \"orgState\": \"TN\",\n",
      "            \"orgZipCode\": \"372407749\",\n",
      "            \"otherPis\": null,\n",
      "            \"projectEndDate\": \"2014-08-31T00:00:00\",\n",
      "            \"projectNumber\": \"5K23CA127469-04\",\n",
      "            \"projectStartDate\": \"2009-09-24T00:00:00\",\n",
      "            \"smApplId\": 21157,\n",
      "            \"terms\": \" analog; Attenuated; Award; base; Biochemical; Biological; Biological Markers; Breast; Cancer Biology; Cancer cell line; Cancer Etiology; Cancer Patient; cDNA Arrays; Cell Proliferation; Cells; Cessation of life; Chromosomes, Human, Pair 10; Classification; Clinical; Clinical Data; Clinical Trials; Clinical Trials Design; Combined Modality Therapy; Correlative Study; cost; Data; Data Set; Day Surgery; Dependence; design; Development; Disease; DNA Microarray Chip; Down-Regulation; Endocrine; Enrollment; Epidermal Growth Factor Receptor; Epidermal Growth Factor Receptor Tyrosine Kinase Inhibitor; ERBB2 gene; Erlotinib; Estrogens; Event; Exhibits; Exons; Feedback; Female; Formalin; Future; Gefitinib; Gene Amplification; Gene Expression; Gene Expression Profile; Gene Expression Profiling; Gene Proteins; Genes; Genetic; Growth; hormone sensitivity; hormone therapy; human FRAP1 protein; Immunohistochemistry; improved; In Situ; inhibitor/antagonist; insulin receptor substrate 1 protein; Insulin-Like Growth Factor I; Lead; malignant breast neoplasm; Malignant Neoplasms; MAP Kinase Gene; Measures; Mentorship; Microarray Analysis; Mining; Molecular; Molecular Profiling; Molecular Target; mortality; mTOR inhibition; mTOR Inhibitor; Mutation; neoplastic cell; Newly Diagnosed; open label; Operative Surgical Procedures; Outcome; outcome forecast; overexpression; Paraffin; Pathway interactions; Patient Selection; Patients; Personal Communication; Phase; Phenotype; pre-clinical; prevent; Progesterone Receptors; prognostic; programs; PTEN gene; receptor; Receptor Down-Regulation; receptor expression; Receptor Inhibition; Receptor Protein-Tyrosine Kinases; Research; Research Design; Research Personnel; Research Proposals; Resistance; response; Role; Safety; SDZ RAD; Signal Transduction; Sirolimus; Subgroup; synergism; Testing; The Vanderbilt-Ingram Cancer Center at the Vanderbilt University; Therapeutic; Time; Trastuzumab; triple-negative invasive breast carcinoma; tumor; tumorigenesis; Tyrosine Kinase Inhibitor; United States; Woman\",\n",
      "            \"title\": \"TARGETED THERAPIES IN BREAST CANCER\",\n",
      "            \"totalCostAmount\": 163944\n",
      "        },\n",
      "        {\n",
      "            \"abstract\": \"DESCRIPTION (provided by applicant): The protein FOX3 acts on the DNA of T cells important to immune regulation. Researchers have suspected that this critical FOXP3+ T cell is important in at least the top three most common human autoimmune diseases and the four most deadly cancers. As there are over 8.5 million U.S. residents with an autoimmune disease, it is easy to appreciate the importance of understanding the mechanisms behind the development of this critical FOXP3+ T cell. Our long-term goal is to understand how the cytokine TGF, an extremely important cytokine to the intestinal environment, regulates the generation of these FOXP3+ T cells in the intestine. Our laboratory has discovered that the deficiency of a KLF family member in mice results in a block in intestinal FOXP3+ T cell generation and a tendency to develop colitis similar to human inflammatory bowel disease. Thus, our objective in this application is to determine the mechanism by which KLF family proteins mediate TGF-dependent Foxp3 gene transcription, and how disruption of this pathway leads to colitis. The three specific aims designed to achieve this objective include: (1) Test the hypothesis that KLF10 expression in T cells mediates a mechanism of resistance to colitis, (2) Test the hypothesis that KLF10 regulates distinct genomic control regions of Foxp3, and (3) Test the hypothesis that KLF10 works via distinct chromatin remodeling required for Foxp3 activation. We will use the KLF10-deficient mouse colony and adoptive transfer of immune cell compartments to determine the precise cell type in which KLF10 is critically required to prevent colitis. Subsequent TCR transgenic animals will be used to determine the mechanism by which KLF10 regulates the development of intestinal FOXP3+ T regulatory cells specific for a defined gut antigen. Human T cells lines and transient transfection methodology will be used to characterize the specific regulatory domains of KLF10 required for Foxp3 promoter function. Finally, a novel human T cell line with a defined Foxp3 promoter and luciferase reporter construct stably integrated into the host cell genome will be used to characterize KLF10-dependent chromatin modifications required for Foxp3 promoter activation. This project is significant because upon conclusion, we will understand the role for KLF10 in the regulation of Foxp3, elucidating pathobiological information into the pathogenesis of chronic inflammation and neoplasia. This in turn will stimulate new areas for experimental therapeutics in human chronic inflammatory diseases.  PUBLIC HEALTH RELEVANCE: The public health burden of autoimmune disease is significant as these unregulated immune responses affect widely disparate organ systems causing such diseases as inflammatory bowel disease, rheumatoid arthritis and diabetes mellitus. This proposal addresses the mechanisms of development of T regulatory cells, cells expected to protect against autoimmune disease. Results from this proposal will stimulate new areas for experimental therapeutics in human chronic inflammatory diseases.  \",\n",
      "            \"agency\": \"NIH\",\n",
      "            \"budgetEndDate\": \"2013-01-31T00:00:00\",\n",
      "            \"budgetStartDate\": \"2012-02-01T00:00:00\",\n",
      "            \"cfdaCode\": \"855\",\n",
      "            \"congressionalDistrict\": \"01\",\n",
      "            \"contactPi\": \"FAUBION, WILLIAM A\",\n",
      "            \"department\": \"HHS\",\n",
      "            \"dunsId\": \"006471700\",\n",
      "            \"fy\": 2012,\n",
      "            \"ic\": \"NIAID\",\n",
      "            \"latitude\": 44.055948,\n",
      "            \"longitude\": -92.525906,\n",
      "            \"nihApplId\": 8223136,\n",
      "            \"orgCity\": \"ROCHESTER\",\n",
      "            \"orgCountry\": \"UNITED STATES\",\n",
      "            \"orgName\": \"MAYO CLINIC\",\n",
      "            \"orgState\": \"MN\",\n",
      "            \"orgZipCode\": \"559050001\",\n",
      "            \"otherPis\": null,\n",
      "            \"projectEndDate\": \"2016-01-31T00:00:00\",\n",
      "            \"projectNumber\": \"5R01AI089714-02\",\n",
      "            \"projectStartDate\": \"2011-02-15T00:00:00\",\n",
      "            \"smApplId\": 22071,\n",
      "            \"terms\": \" Address; Adoptive Transfer; Affect; Antigens; Area; Autoimmune Diseases; base; Binding; Biology; body system; Cancer Prognosis; Cell Line; cell type; Cells; Chromatin; chromatin modification; chromatin remodeling; Chronic; Clinic; Colitis; Colorectal Cancer; Crohn's disease; cytokine; Data; design; Development; Diabetes Mellitus; Disease; DNA; Environment; Enzymes; Epigenetic Process; Equilibrium; Event; experience; Experimental Designs; Experimental Models; Family; Family member; Gene Activation; Gene Expression; Gene Expression Regulation; Gene Targeting; Generations; Genes; Genetic Transcription; Genome; Genomics; Goals; Histone Acetylation; Histones; Human; Immune; Immune response; Immunology; Inflammation; Inflammatory; Inflammatory Bowel Diseases; Intestines; Laboratories; Lifetime Risk; Luciferases; Malignant Neoplasms; Mediating; member; Methodology; Mus; Neoplasms; novel; Pathogenesis; Pathway interactions; Predisposition; Prevalence; prevent; Process; programs; promoter; Protein Family; Proteins; Public Health; public health relevance; Qualifying; Recruitment Activity; Regulation; Regulatory Pathway; Regulatory T-Lymphocyte; Reporter; Research Personnel; resistance mechanism; Rheumatoid Arthritis; Role; Severities; System; Testing; Therapeutic; T-Lymphocyte; T-Lymphocyte Subsets; transcription factor; Transfection; Transgenic Animals; Transgenic Organisms; Work; Zinc Fingers\",\n",
      "            \"title\": \"KLF10 REGULATES COLITIS THROUGH MEDIATING TGFB INDUCTION OF FOXP3 IN TREG CELLS.\",\n",
      "            \"totalCostAmount\": 394250\n",
      "        },\n",
      "        {\n",
      "            \"abstract\": \"DESCRIPTION (provided by applicant): NF-kB signaling has a crucial role in regulating the activation, proliferation and effector functions of lymphocytes in adaptive immune responses. Deregulation of this process results in immunodeficiency, autoimmune diseases, or neoplastic disorders. In the presence of additional co-stimulatory signals, NF-kB is activated by the engagement of TCR with MHC-bound antigen peptides or the interaction of BCR with antigens. The protein kinase C ? (PKC?) of T-cells and PKC\\u00bf of B-cells are recruited to lipid rafts during antigen-receptor triggering and play key roles in TCR- and BCR-induced NF-kB activation, respectively. The ternary complex of CARMA1 [caspase-recruitment domain (CARD) membrane-associated guanylate kinase (MAGUK) protein 1, also known as CARD11], Bcl10 (B-cell lymphoma 10) and MALT1 (mucosa-associated lymphoid tissue lymphoma translocation protein 1) acts downstream of PKC? and PKC\\u00bf to induced activation of the IkB kinase (IKK) in both B and T lymphocytes (Figure 1). CARMA1 appears to be constitutively associated with the cytoplasmic membrane. Upon TCR stimulation, CARMA1 is redistributed to the lipid rafts at the immunological synapse to recruit Bcl10 and MALT1 to form the CARMA1-Bcl10-MALT1 (CBM) complex. No structural information is currently available on the CBM complex or its component proteins. In addition, despite having domains that belong to known protein families, CARMA1, Bcl10 and MALT1 exhibit very limited sequence homology to any of the known proteins. To fill this gap, we propose a complete comprehensive structural analysis on the CBM complex, including its assembly mechanisms, conformational changes, enzymatic activities and post-translational modifications. These studies will inevitably provide insights into this important signaling complex.  PUBLIC HEALTH RELEVANCE: The proteins in the CBM complex play critical roles in Non-Hodgkin's lymphomas and may be bona fide oncogenes. CARMA1 overexpression has been found in adult T-cell leukemia, primary gastric B-cell lymphoma, and diffuse large B-cell lymphoma (DLBCL). In DLBCL, the most common form of non-Hodgkin's lymphoma, missense mutations of CARMA1 have been found at the coiled coil region of CARMA1, which correlate with constitutive NF-kB activation. Chromosomal translocations of both Bcl10 and MALT1 have been found in mucosa-associated lymphoid tissue (MALT) lymphoma. The most frequent chromosomal aberrations in MALT lymphoma are the t(11;18)(q21:q21) and t(14;18)(q32:q21) translocations, which account for about half of all MALT lymphoma. The former translocation creates a fusion protein comprising the BIR domains of cIAP2 and the meta-caspase domain of MALT1 and constitutive NF-kB activation. The t(14;18) translocation results in positioning of the MALT1 genes to the Ig heavy chain locus. Similarly for Bcl10, the t(1;14)(p22:q32) translocation places the Bcl10 to the Ig heavy chain locus. These translocations likely activate NF-kB through MALT1 and Bcl10 overexpression. Collectively, these studies suggest that the CBM complex is an attractive therapeutic target neoplastic disorders of the lymphocytes.  \",\n",
      "            \"agency\": \"NIH\",\n",
      "            \"budgetEndDate\": \"2012-06-30T00:00:00\",\n",
      "            \"budgetStartDate\": \"2012-05-01T00:00:00\",\n",
      "            \"cfdaCode\": \"855\",\n",
      "            \"congressionalDistrict\": \"12\",\n",
      "            \"contactPi\": \"WU, HAO\",\n",
      "            \"department\": \"HHS\",\n",
      "            \"dunsId\": \"060217502\",\n",
      "            \"fy\": 2012,\n",
      "            \"ic\": \"NIAID\",\n",
      "            \"latitude\": 40.76491,\n",
      "            \"longitude\": -73.955055,\n",
      "            \"nihApplId\": 8260245,\n",
      "            \"orgCity\": \"NEW YORK\",\n",
      "            \"orgCountry\": \"UNITED STATES\",\n",
      "            \"orgName\": \"WEILL MEDICAL COLLEGE OF CONRELL UNIVERSITY\",\n",
      "            \"orgState\": \"NY\",\n",
      "            \"orgZipCode\": \"100654805\",\n",
      "            \"otherPis\": null,\n",
      "            \"projectEndDate\": \"2012-06-30T00:00:00\",\n",
      "            \"projectNumber\": \"5R01AI089882-03\",\n",
      "            \"projectStartDate\": \"2010-05-01T00:00:00\",\n",
      "            \"smApplId\": 22092,\n",
      "            \"terms\": \" Accounting; adaptive immunity; Adult; Anniversary; antigen binding; Antigen Receptors; Antigens; Autoimmune Diseases; Autoimmunity; Baltimore; base; B-Cell Lymphomas; Binding; Biochemical; Biological; Biological Process; BIR Domain; B-Lymphocytes; Calorimetry; Caspase; Cell membrane; Cell Nucleus; Cell surface; Cells; Chimeric Proteins; Chromosomal translocation; Chromosome abnormality; Collection; Complex; C-terminal; Cytoplasm; Death Domain; design; Development; dimer; Dimerization; Disease; DNA Binding Domain; Dorsal; Drosophila genus; electron crystallography; Electron Microscopy; Exhibits; Family; Genes; Guanylate kinase; Homologous Gene; Immune response; Immunoglobulins; Immunologic Deficiency Syndromes; immunological synapse; Inflammatory Response; inhibitor/antagonist; insight; Interleukin-1 Receptors; large cell Diffuse non-Hodgkin's lymphoma; Legal patent; Length; Ligation; Lymphocyte; Lymphocyte Activation; Lymphocyte Function; lymphocyte proliferation; Maps; Measurement; Mediating; meetings; member; Membrane Microdomains; membrane-associated guanylate kinase; Missense Mutation; Molecular; molecular domain; mucosa-associated lymphoid tissue lymphoma; Mutagenesis; Mutation; Names; Natural Immunity; neoplastic; NF-kappa B; Non-Hodgkin's Lymphoma; N-terminal; Nuclear; Nuclear Translocation; Oncogenes; Oncogenic; overexpression; Peptides; Phosphorylation; Phosphotransferases; Play; Positioning Attribute; Post-Translational Protein Processing; Process; Protein Family; Protein Kinase C; Protein translocation; Proteins; public health relevance; Publications; receptor; Receptors, Antigen, B-Cell; reconstitution; Recruitment Activity; Regulation; research study; response; Role; Scaffolding Protein; Sequence Homology; SH3 Domains; Signal Pathway; Signal Transduction; small molecule; Specificity; Stomach; Structure; Surface Plasmon Resonance; System; T-Cell Leukemia; T-Cell Receptor; therapeutic target; Titrations; T-Lymphocyte; Toll-like receptors; transcription factor; Tumor Necrosis Factor Receptor; tumorigenesis; Viral; v-rel Oncogenes; web site\",\n",
      "            \"title\": \"MOLECULAR ELUCIDATION OF THE CBM COMPLEX IN NF-KAPPAB ACTIVATION BY ANTIGEN RECEP\",\n",
      "            \"totalCostAmount\": 55769\n",
      "        },\n",
      "        {\n",
      "            \"abstract\": \"DESCRIPTION (provided by applicant):  Owing to the recent setbacks in vaccine and microbicide trails, novel strategies to slow the HIV-pandemic are desperately needed. Development of a small molecule that blocked a host process required for HIV-1 replication could provide such a strategy. Like all retroviruses, HIV-1 requires host proteins to complete its life cycle, and these intracellular host molecules represent an undeveloped pool of novel anti-HIV therapeutics. The goal of this application is to use functional proteomics to identify host proteins that are dispensable to the host but essential for viral replication. A generic proteomic screen would likely identify numerous host proteins induced by HIV-infection, most of which would be poor therapeutic candidates. A functional proteomics screen, which queries a subset of proteins with strong therapeutic potential, would greatly simplify the search for host targets; the host purinome has this potential. The purinome comprises any protein that binds purine- containing molecules (e.g. ATP, NADH), and it includes heat shock proteins, dehydrogenases, and protein kinases. Inhibition of purinome proteins forms the basis of many current therapies, including those targeting cancer, hypertension, and bacterial infections. The central hypothesis of this application states that inhibition of purinome proteins, which are induced or regulated by HIV-infection, will block HIV-1 replication. Our research team has designed a protein affinity media that captures the entire purinome through its purine-binding pocket. The proposed research will proceed in four broad steps: use SILAC (stable isotope labeling with amino acids in cell culture) to quantify purinome proteins induced by HIV-1 infection of THP1 and Jurkat cells (target identification); determine which purinome proteins are essential for HIV-1 replication (target validation); identify drugs that selectively compete validated targets from the purinome-binding resin (drug discovery); confirm that these compounds block HIV-replication in primary cells (drug validation). During this process, we will also identify host proteins that interact with HIV-1, and in the event that a viable purinome target and/or drug is not identified, viral biology will be advanced by a greater understanding of how HIV-1 interacts with its host. The proposed research uses innovative proteomics technology (SILAC) and an unconventional, yet validated, approach. First, conventional anti-HIV therapeutics inhibit virally-encoded proteins, which makes them more susceptible to the development of resistance. By targeting a host enzyme that is required for HIV-1 replication, it is likely that the virus will have to undergo more radical changes to offset the changes in the host milieu. Second, conventional drug discovery uses a reductionist approach to separately identify a target and a drug, while our approach combines the target/drug identification process, which makes discovery more efficient. At the completion of this project, our combination of experience and technology will enable us to characterize host purine-binding proteins that are essential for HIV-1 replication and identify several lead compounds that will inhibit host proteins necessary for HIV-1 replication.  \",\n",
      "            \"agency\": \"NIH\",\n",
      "            \"budgetEndDate\": \"2013-05-31T00:00:00\",\n",
      "            \"budgetStartDate\": \"2012-06-01T00:00:00\",\n",
      "            \"cfdaCode\": \"855\",\n",
      "            \"congressionalDistrict\": \"03\",\n",
      "            \"contactPi\": \"KWIEK, JESSE J\",\n",
      "            \"department\": \"HHS\",\n",
      "            \"dunsId\": \"832127323\",\n",
      "            \"fy\": 2012,\n",
      "            \"ic\": \"NIAID\",\n",
      "            \"latitude\": 39.999598,\n",
      "            \"longitude\": -83.033131,\n",
      "            \"nihApplId\": 8272655,\n",
      "            \"orgCity\": \"COLUMBUS\",\n",
      "            \"orgCountry\": \"UNITED STATES\",\n",
      "            \"orgName\": \"OHIO STATE UNIVERSITY\",\n",
      "            \"orgState\": \"OH\",\n",
      "            \"orgZipCode\": \"432101016\",\n",
      "            \"otherPis\": null,\n",
      "            \"projectEndDate\": \"2014-05-31T00:00:00\",\n",
      "            \"projectNumber\": \"5R01AI090644-03\",\n",
      "            \"projectStartDate\": \"2010-06-01T00:00:00\",\n",
      "            \"smApplId\": 22133,\n",
      "            \"terms\": \" Affinity; Amino Acids; Antineoplastic Agents; Antiviral Agents; Attenuated; Bacterial Infections; base; Binding; Binding Proteins; Biology; CD4 Positive T Lymphocytes; Cell Culture Techniques; Cells; Clinical Trials; Communicable Diseases; conventional therapy; Coupled; Databases; design; Development; DNA; Down-Regulation; Drug Delivery Systems; drug discovery; Drug resistance; Enzymes; Erectile dysfunction; Event; experience; functional genomics; Funding; Generations; Generic Drugs; Goals; Heat shock proteins; helicase; HIV; HIV drug resistance; HIV Infections; HIV-1; Human; human disease; hypercholesterolemia; Hypertension; Infection; innovation; innovative technologies; Jurkat Cells; Lead; Life Cycle Stages; Link; macrophage; Malignant Neoplasms; Mass Spectrum Analysis; Methods; microbicide; Modeling; monocyte; Monomeric GTP-Binding Proteins; NADH; Non-Insulin-Dependent Diabetes Mellitus; novel; novel strategies; Online Mendelian Inheritance In Man; Oxidoreductase; pandemic disease; Pathway interactions; Pharmaceutical Preparations; Phase; Plant Resins; Predisposition; Process; protein activation; Protein Binding; protein expression; protein folding; Protein Kinase; Proteins; Proteome; Proteomics; purine; Purines; Regulation; Research; Resistance development; Retroviridae; Reverse Transcription; RNA chemical synthesis; Role; Small Interfering RNA; small molecule; Stable Isotope Labeling; System; Technology; Therapeutic; therapeutic target; Time; tissue culture; tissue/cell culture; Transcription Process; TRIM Gene; Vaccines; Validation; Vertebral column; Viral; Viral Proteins; Virus; Wolves; Work\",\n",
      "            \"title\": \"A METHOD TO STOP HIV REPLICATION:INHIBITION OF HUMAN PURINE UTILIZING PROTEINS\",\n",
      "            \"totalCostAmount\": 432020\n",
      "        },\n",
      "        {\n",
      "            \"abstract\": \"DESCRIPTION (provided by applicant): 2G12 is a rare broadly-neutralizing antibody which binds to the HIV envelope glycoprotein gp120 by recognizing a cluster of carbohydrates. The goal of this project is to design a good structural mimic of the 2G12 epitope and test its ability to elicit a focused 2G12-like antibody response that can neutralize and protect against HIV infection. In Specific Aim 1, we will approach this problem from the standpoint of rational design. We have designed a tunable peptide scaffold for presentation of carbohydrates which can be used to present 2-6 carbohydrates with systematically varied inter-carbohydrate distances. We will synthesize a range of possible di-, tri- and tetravalent carbohydrate clusters this way, and compare their binding affinities for 2G12 by ELISA and surface plasmon resonance. These measurements will guide the refinement of our design. We will also study the effect of certain rigidifying cross-linkers on the synthetic antigen's ability to bind 2G12. In Specific Aim 2, we will approach the problem from the standpoint of directed evolution. We will use 'click chemistry' to attach carbohydrate-azides to a random mRNA display library of ~1013 different unnatural alkynyl peptides. We will then select from this glycopeptide library the best binders to 2G12. Because every peptide in the library is attached to its encoding RNA, we can amplify and/or diversify the selection winners by PCR. Translation and click-chemistry glycosylation will yield a second-generation library with an increased population of 2G12 binders. Increasingly stringent cycles of screening with 2G12 and amplification/diversification will give rise to high affinity glycopeptide binders of 2G12, which should be good mimics of the 2G12 epitope. The best mimics will be sequenced and synthesized on preparative scale for further studies. Biophysical measurements will asses the ability of the best 2G12 epitope mimics from Aims 1 and 2 to compete with gp120 for binding to 2G12. In Specific Aim 3, we will test the immunogenicity of the best Aim1/Aim2 antigens in a two-phase rabbit study. A small pilot study will test 5 antigens in small groups. A follow-up study will reexamine the best 2 antigens with larger groups, varying doses, and varying adjuvants. In both studies, animal sera will be screened for neutralization activity against a range of HIV viral strains. Additionally, sera will be tested for the ability to bind gp120, as well as several glycopeptide test antigens. Whether or not a neutralizing antibody response is raised, this research will shed light on the 2G12-gp120 interaction. Moreover, the glycopeptide evolution technique developed in Aim 2 will have numerous other applications in fields such as cell signaling, cancer biology, and virology, and so will have wide- ranging impact across disciplines. Although the effort to create a neutralizing antibody vaccine against HIV-1 has so far met with little success, studies of the HIV-infected population have brought into the spotlight several examples of broadly neutralizing antibodies. The focus of this project is to apply knowledge gained from the study of one such antibody, 2G12, to design a vaccine which elicits a neutralizing antibody response by mimicking the 2G12 epitope.  \",\n",
      "            \"agency\": \"NIH\",\n",
      "            \"budgetEndDate\": \"2013-05-31T00:00:00\",\n",
      "            \"budgetStartDate\": \"2012-06-01T00:00:00\",\n",
      "            \"cfdaCode\": \"855\",\n",
      "            \"congressionalDistrict\": \"05\",\n",
      "            \"contactPi\": \"KRAUSS, ISAAC\",\n",
      "            \"department\": \"HHS\",\n",
      "            \"dunsId\": \"616845814\",\n",
      "            \"fy\": 2012,\n",
      "            \"ic\": \"NIAID\",\n",
      "            \"latitude\": 42.38892,\n",
      "            \"longitude\": -71.242325,\n",
      "            \"nihApplId\": 8287717,\n",
      "            \"orgCity\": \"WALTHAM\",\n",
      "            \"orgCountry\": \"UNITED STATES\",\n",
      "            \"orgName\": \"BRANDEIS UNIVERSITY\",\n",
      "            \"orgState\": \"MA\",\n",
      "            \"orgZipCode\": \"024532728\",\n",
      "            \"otherPis\": null,\n",
      "            \"projectEndDate\": \"2014-05-31T00:00:00\",\n",
      "            \"projectNumber\": \"5R01AI090745-03\",\n",
      "            \"projectStartDate\": \"2010-06-01T00:00:00\",\n",
      "            \"smApplId\": 22147,\n",
      "            \"terms\": \" Acquired Immunodeficiency Syndrome; Adjuvant; Affinity; Amino Acids; Animals; Antibodies; Antibody Formation; Antigens; Asses; Azides; Binding; Cancer Biology; Carbohydrates; Cessation of life; Chemistry; combinatorial; Coupling; design; directed evolution; Discipline; Dose; Elements; Enzyme-Linked Immunosorbent Assay; Epitopes; Evaluation; Evolution; falls; follow-up; Generations; Glycopeptides; glycosylation; Goals; HIV; HIV Envelope Protein gp120; HIV Infections; HIV vaccine; HIV-1; Immune response; immunogenicity; Immunology; Individual; Infection; Knowledge; Libraries; Light; Measurement; meetings; member; Messenger RNA; neutralizing antibody; Oligosaccharides; Oryctolagus cuniculus; Peptide Library; Peptides; Phase; Pilot Projects; polypeptide; Population; Research; RNA; scaffold; Screening procedure; Series; Serum; Signal Transduction; Structure; success; Surface Plasmon Resonance; Synthetic Antigens; Techniques; Testing; Translations; vaccine candidate; vaccine development; Vaccines; Viral; virology; Virus\",\n",
      "            \"title\": \"RATIONAL AND COMBINATORIAL DESIGN OF IMMUNOGENS TO ELICIT 2G12-LIKE ANTIBODIES\",\n",
      "            \"totalCostAmount\": 540103\n",
      "        },\n",
      "        {\n",
      "            \"abstract\": \"DESCRIPTION (provided by applicant): Previous funding by an R03 application allowed us to develop novel methods that permit the amplification of Plasmodium sequences from fecal DNA. Screening ape samples from over 50 field sites throughout central Africa, we found that P. falciparum is of gorilla origin, and not of chimpanzee, bonobo or ancient human origin. We also found that chimpanzees and gorillas harbor at least nine Plasmodium species, including parasites that are near identical to P. vivax, P. ovale and P. malariae. Given the magnitude of this Plasmodium reservoir and the fact that gorilla P. falciparum has crossed the species to humans already once, the question arises whether additional cross-infections of ape Plasmodium parasites are occurring. This is of critical public health importance, not because such transmissions would be expected to contribute to current malaria morbidity and mortality, but because they would give an indication of the potential of ape malaria parasites to colonize humans should a reduction of P. falciparum transmission rates generate a new ecological niche. In this application, we will examine whether wild apes serve as a recurrent source of human malaria. We will continue to screen wild ape populations for Plasmodium infections, and test humans who live in close proximity to these apes for evidence of cross-species infection. Our hypothesis is that ape parasites have the potential to infect humans but fail to establish persistent infections because of the predominance of P. falciparum. Determining the types, locations and frequencies of ape Plasmodium cross-species infections will be critical to gauge the success of future P. falciparum eradication campaigns. 1. To determine the prevalence of P. vivax and other non-Laverania species in wild apes. We will determine the prevalence, host specificities and distribution of non-Laverania parasites in wild apes, and determine whether chimpanzees and/or gorillas represent a reservoir for human P. vivax in west central Africa. 2. To determine whether wild chimpanzees or gorillas serve as recurrent sources of human infection. We will use ultradeep (454) sequencing to screen humans who live in close proximity to wild apes for evidence of zoonotic Plasmodium infections. This approach will identify Laverania and non-Laverania parasites even if they infect humans at very low frequencies and in the context of multispecies infections. 3. To determine the natural history of ape Plasmodium infections. We will prospectively follow Plasmodium infected sanctuary apes to examine their clinical status, validate our non-invasive detection methods, and obtain blood samples for ape Plasmodium isolation and whole genome sequencing studies. 4. To determine the biological properties that distinguish human P. falciparum from related ape species. We will express ape Plasmodium erythrocyte binding ligands and test their ability to bind to human erythrocytes in order to investigate the molecular basis of Plasmodium host specificity and to assess which ape Plasmodium species have the capacity to cause a blood stage infection in humans.  \",\n",
      "            \"agency\": \"NIH\",\n",
      "            \"budgetEndDate\": \"2013-08-31T00:00:00\",\n",
      "            \"budgetStartDate\": \"2012-09-01T00:00:00\",\n",
      "            \"cfdaCode\": \"855\",\n",
      "            \"congressionalDistrict\": \"02\",\n",
      "            \"contactPi\": \"HAHN, BEATRICE H\",\n",
      "            \"department\": \"HHS\",\n",
      "            \"dunsId\": \"042250712\",\n",
      "            \"fy\": 2012,\n",
      "            \"ic\": \"NIAID\",\n",
      "            \"latitude\": 39.952982,\n",
      "            \"longitude\": -75.192768,\n",
      "            \"nihApplId\": 8334500,\n",
      "            \"orgCity\": \"PHILADELPHIA\",\n",
      "            \"orgCountry\": \"UNITED STATES\",\n",
      "            \"orgName\": \"UNIVERSITY OF PENNSYLVANIA\",\n",
      "            \"orgState\": \"PA\",\n",
      "            \"orgZipCode\": \"191046205\",\n",
      "            \"otherPis\": null,\n",
      "            \"projectEndDate\": \"2014-08-31T00:00:00\",\n",
      "            \"projectNumber\": \"5R01AI091595-02\",\n",
      "            \"projectStartDate\": \"2011-09-20T00:00:00\",\n",
      "            \"smApplId\": 22179,\n",
      "            \"terms\": \" Accounting; Address; Africa; African; Area; base; Binding; Biological; Biological Assay; Biology; Blood; Blood specimen; Centers for Disease Control and Prevention (U.S.); Central Africa; Clinical; Collection; Communicable Diseases; Cross Infection; Data; DNA; Erythrocytes; Event; Evolution; follow-up; Frequencies; Funding; Future; genome sequencing; Geographic Locations; Gorilla gorilla; High Prevalence; Human; In Vitro; Infection; insight; Institutes; Knowledge; Life; Ligand Binding; Light; Location; Malaria; man; Methods; Microscopy; Mitochondria; Molecular; Morbidity - disease rate; mortality; Natural History; Non-Invasive Cancer Detection; novel; Nuclear; Pan Genus; Pan paniscus; pandemic disease; Parasites; Pathogenesis; Pathway interactions; Phenotype; Phylogenetic Analysis; Plasmodium; Plasmodium falciparum; Pongidae; Population; Prevalence; Process; Property; Public Health; Recurrence; Reporting; Sampling; Screening procedure; Shipping; Ships; Site; Source; Specificity; Staging; success; Surveys; Testing; transmission process; Validation; Variant; Veterinarians; Work\",\n",
      "            \"title\": \"GREAT APE RESERVOIRS OF HUMAN MALARIA\",\n",
      "            \"totalCostAmount\": 669393\n",
      "        },\n",
      "        {\n",
      "            \"abstract\": \"DESCRIPTION (provided by applicant): Allogeneic T cell responses drive reactivity to foreign tissues in the setting of solid organ or hematopoietic stem cell transplantation (allo-HSCT). After allo-HSCT, donor alloreactive T cells induce both beneficial graft-versus-tumor activity and harmful graft-versus-host-disease, a life- threatening complication that limits the effectiveness of allo-HSCT. Graft-versus-host-disease is a serious medical problem for which existing therapeutic interventions are often ineffective. In addition, existing strategies to control graft-versus-host disease impair anti-tumor responses, leading to an increased risk of cancer relapse. Discovering novel immunomodulatory approaches to control the harmful effects of allogeneic T cell responses without eliminating their beneficial anti-cancer activity is essential to improve the long-term success and widespread applicability of allo-HSCT. We have identified a new critical role for Notch signaling in alloreactive T cells mediating graft-versus-host disease after allo-HSCT. Inhibition of canonical Notch signaling in donor T cells markedly reduced the severity and mortality of graft-versus-host disease in several mouse models of allo-HSCT. Notch-deprived T cells proliferated normally and showed increased expansion in lympho-hematopoietic organs, demonstrating the absence of global immunosuppression. Notably, Notch-deprived alloreactive T cells acquired efficient cytotoxicity in vivo and retained potent anti-leukemia activity, leading to markedly improved overall survival of the recipients. However, their ability to produce multiple inflammatory cytokines was reduced. Notch inhibition also decreased the accumulation of alloreactive T cells in the intestine, a key GVHD target organ. Thus, Notch signaling represents a promising therapeutic target to control graft-versus-host disease while preserving significant anti- cancer activity in donor T cells after allo-HSCT. We hypothesize that Notch is a new essential regulator of T cell function in allogeneic T cell responses. To explore this hypothesis in detail, we will determine the specific Notch ligands and receptors that mediate Notch activation in T cells after allogeneic HSCT; investigate the cellular and molecular mechanisms underlying the decreased induction of GVHD by Notch-deficient alloreactive T cells; and identify the cytotoxic pathways that mediate the persistent anti-cancer activity of CD4+ and CD8+ T cells upon Notch inhibition. These studies will bring novel insights into the molecular regulation of alloimmunity and might lead to the development of new approaches to limit damaging consequences of T cell reactivity after allogeneic transplantation. .  PUBLIC HEALTH RELEVANCE: Allogeneic T cell responses against host antigens mediate graft-versus-host disease, the most serious complication of allogeneic hematopoietic stem cell transplantation. The goal of this proposal is to evaluate the role of Notch signaling in the regulation of T cell alloimmunity. Completion of this project will not only reveal novel functions of the Notch signaling pathway in the immune system, but also pave the way for therapeutic manipulation of Notch signaling in T cell-mediated disorders, including graft-versus-host disease, organ rejection and autoimmunity.   \",\n",
      "            \"agency\": \"NIH\",\n",
      "            \"budgetEndDate\": \"2013-03-31T00:00:00\",\n",
      "            \"budgetStartDate\": \"2012-04-01T00:00:00\",\n",
      "            \"cfdaCode\": \"855\",\n",
      "            \"congressionalDistrict\": \"12\",\n",
      "            \"contactPi\": \"MAILLARD, IVAN\",\n",
      "            \"department\": \"HHS\",\n",
      "            \"dunsId\": \"073133571\",\n",
      "            \"fy\": 2012,\n",
      "            \"ic\": \"NIAID\",\n",
      "            \"latitude\": 42.244005,\n",
      "            \"longitude\": -83.73915,\n",
      "            \"nihApplId\": 8260313,\n",
      "            \"orgCity\": \"ANN ARBOR\",\n",
      "            \"orgCountry\": \"UNITED STATES\",\n",
      "            \"orgName\": \"UNIVERSITY OF MICHIGAN AT ANN ARBOR\",\n",
      "            \"orgState\": \"MI\",\n",
      "            \"orgZipCode\": \"481091276\",\n",
      "            \"otherPis\": null,\n",
      "            \"projectEndDate\": \"2016-03-31T00:00:00\",\n",
      "            \"projectNumber\": \"5R01AI091627-02\",\n",
      "            \"projectStartDate\": \"2011-04-22T00:00:00\",\n",
      "            \"smApplId\": 22181,\n",
      "            \"terms\": \" Adverse effects; Alloantigen; Allogenic; Antigen-Presenting Cells; Antigens; Autoimmunity; base; Biochemical; cancer cell; Cancer Relapse; CD8B1 gene; Cell physiology; Complication; cytokine; cytotoxic; cytotoxicity; Development; Disease; Effectiveness; Elements; Epithelial; Gastrointestinal tract structure; Genetic; Goals; graft versus host disease induction; Graft-Versus-Tumor Induction; Graft-vs-Host Disease; Hematopoietic; Hematopoietic Stem Cell Transplantation; hematopoietic tissue; Homologous Transplantation; Immune; Immune system; Immunobiology; immunoregulation; improved; in vivo; Individual; Infiltration; Inflammatory; Inflammatory Response; insight; Intestines; isoimmunity; Killings; Lead; leukemia; Life; Ligands; Lymphocyte; Malignant Neoplasms; Mediating; Medical; Methods; Molecular; mortality; mouse model; Natural immunosuppression; neutralizing monoclonal antibodies; notch protein; Notch Signaling Pathway; novel; novel strategies; Organ; Pathway interactions; Play; prevent; Production; Proliferating; public health relevance; receptor; Regulation; Regulatory T-Lymphocyte; response; Risk; Role; Severities; Severity of illness; Signal Transduction; Solid; success; Synapses; T cell regulation; T cell response; Testing; Therapeutic; Therapeutic Intervention; Therapeutic Studies; therapeutic target; Tissues; T-Lymphocyte; tumor; Work\",\n",
      "            \"title\": \"NOTCH SIGNALING IN ALLOIMMUNITY\",\n",
      "            \"totalCostAmount\": 383034\n",
      "        },\n",
      "        {\n",
      "            \"abstract\": \"DESCRIPTION (provided by applicant): Viruses pose a significant threat to human health based on their ability to cause diseases such as AIDS, influenza, hepatitis, cancer and the common cold. Basic research has revealed much about the molecular details of processes that are essential for viruses to reproduce, but a critical barrier to further progress is a lack of understanding of how factors of the host environment quantitatively impact virus growth. The proposed research aims to address this barrier by advancing new quantitative experiments and computational models of virus growth. The approach will employ, combine and extend perspectives drawn from the fields of experimental evolution and systems biology. Specific aims of the proposed project will be to: (1) advance computational models of virus growth to provide an integrated dynamic measure of cellular resource use, (2) quantify how resource use is altered during evolutionary adaptation of virus to host cells, and (3) elucidate mechanisms by which viral adaptation alters resource use. The methodology established by this research will set a basis for advancing the development of data-driven predictive models of virus growth. Results of the research will have the potential to significantly impact several fields: the design anti-viral therapeutics, applications of viruses in oncolytic (anti-tumor) cancer therapies, and development of new vaccines. At a more basic level, predictive models of virus growth will be useful for better understanding how viruses grow, evolve and persist in Nature.  PUBLIC HEALTH RELEVANCE: Viruses cause a diversity of human diseases including AIDS, influenza, cancer, hepatitis, and the common cold. The proposed research develops new experimental and computational approaches to study and understand how viruses grow. The broad goal of this research is to learn how to better fight diseases caused by viruses. Their work some concerns which serve to spread the enthusiasm somewhat, nevertheless, it came to rest in the exceptional range.   \",\n",
      "            \"agency\": \"NIH\",\n",
      "            \"budgetEndDate\": \"2012-11-30T00:00:00\",\n",
      "            \"budgetStartDate\": \"2011-12-01T00:00:00\",\n",
      "            \"cfdaCode\": \"855\",\n",
      "            \"congressionalDistrict\": \"02\",\n",
      "            \"contactPi\": \"YIN, JOHN\",\n",
      "            \"department\": \"HHS\",\n",
      "            \"dunsId\": \"161202122\",\n",
      "            \"fy\": 2012,\n",
      "            \"ic\": \"NIAID\",\n",
      "            \"latitude\": 43.060421,\n",
      "            \"longitude\": -89.397322,\n",
      "            \"nihApplId\": 8204496,\n",
      "            \"orgCity\": \"MADISON\",\n",
      "            \"orgCountry\": \"UNITED STATES\",\n",
      "            \"orgName\": \"UNIVERSITY OF WISCONSIN MADISON\",\n",
      "            \"orgState\": \"WI\",\n",
      "            \"orgZipCode\": \"537151218\",\n",
      "            \"otherPis\": null,\n",
      "            \"projectEndDate\": \"2015-11-30T00:00:00\",\n",
      "            \"projectNumber\": \"5R01AI091646-02\",\n",
      "            \"projectStartDate\": \"2010-12-03T00:00:00\",\n",
      "            \"smApplId\": 22183,\n",
      "            \"terms\": \" Acquired Immunodeficiency Syndrome; Address; Advanced Development; base; Basic Science; cancer therapy; Cell physiology; Cells; Common Cold; Computer Simulation; Data; design; Disease; Environment; Evolution; Exhibits; fighting; fitness; Gene Order; Goals; Growth; Health; Hepatitis; Human; human disease; Infection; Influenza; insight; Integration Host Factors; Knowledge; Laboratories; Learning; Malignant Neoplasms; Measures; Medical; Methodology; Methods; Modeling; Molecular; molecular modeling; Molecular Models; mutant; Nature; novel vaccines; Oncolytic; Organism; Population; predictive modeling; Process; Protein Biosynthesis; public health relevance; Relative (related person); Research; research study; Resources; Rest; Serial Passage; Structure; Systems Biology; Therapeutic; therapy development; Time; tumor; Variant; Vesicular stomatitis Indiana virus; Viral; Virus; Virus Diseases; virus host interaction; Virus-Cell Membrane Interaction; Work\",\n",
      "            \"title\": \"EVOLUTION OF VIRUS-HOST INTERACTIONS\",\n",
      "            \"totalCostAmount\": 379546\n",
      "        },\n",
      "        {\n",
      "            \"abstract\": \"DESCRIPTION (provided by applicant): Type I interferons (IFNs) are well-characterized cytokines that inhibit a wide range of viruses. The cellular proteins that mediate the antiviral functions of IFN are derived from interferon-stimulated genes (ISGs), and are potentially exploitable for the development of pan-tropic antiviral drugs. Unfortunately, very little is known about which ISGs are antiviral, their specificity is largely undetermined, and their mechanisms of action remain elusive. We propose to address this gap in scientific knowledge by conducting a series of large-scale screens to determine the 'ISG profile' of diverse viruses. We have cloned over 380 commonly upregulated ISGs into a lentiviral vector and optimized conditions to correlate the expression of these genes with viral replication using a two-color flow cytometric assay. The ISG library will be used to probe ISG antiviral function against an array of viruses form diverse families, with a special emphasis on hepatitis C virus (HCV) and influenza virus A (FluA). Each of these viruses has a significant impact on global health, with pathologies ranging from hepatocellular carcinoma to severe respiratory disease, and potential outcomes spanning life-long chronic infection to acute pandemic events. Comparison of ISG profiles from across viral species, genera, and families will begin to reveal the range of ISG specificities. Screens of select viruses under different cellular conditions will address the depth of ISG function. Validation and mechanistic dissection of the most interesting hits will clarify the roles of ISGs in combating infection, and may yield new strategies for translating naturally occurring broad-spectrum antiviral activities into clinically useful compounds.  PUBLIC HEALTH RELEVANCE: Hepatitis C, influenza A, and other infectious diseases cause nearly 15 million deaths each year. Therapeutic options for viral infections are often severely limited. Here we propose to investigate the activity of cellular interferon-stimulated genes (ISGs) against diverse viruses in the hopes of identifying candidates to guide development of broad-spectrum antiviral drugs.  \",\n",
      "            \"agency\": \"NIH\",\n",
      "            \"budgetEndDate\": \"2013-02-28T00:00:00\",\n",
      "            \"budgetStartDate\": \"2012-03-01T00:00:00\",\n",
      "            \"cfdaCode\": \"855\",\n",
      "            \"congressionalDistrict\": \"12\",\n",
      "            \"contactPi\": \"RICE, CHARLES M\",\n",
      "            \"department\": \"HHS\",\n",
      "            \"dunsId\": \"071037113\",\n",
      "            \"fy\": 2012,\n",
      "            \"ic\": \"NIAID\",\n",
      "            \"latitude\": 40.762377,\n",
      "            \"longitude\": -73.955649,\n",
      "            \"nihApplId\": 8230498,\n",
      "            \"orgCity\": \"NEW YORK\",\n",
      "            \"orgCountry\": \"UNITED STATES\",\n",
      "            \"orgName\": \"ROCKEFELLER UNIVERSITY\",\n",
      "            \"orgState\": \"NY\",\n",
      "            \"orgZipCode\": \"100656399\",\n",
      "            \"otherPis\": null,\n",
      "            \"projectEndDate\": \"2016-02-29T00:00:00\",\n",
      "            \"projectNumber\": \"5R01AI091707-02\",\n",
      "            \"projectStartDate\": \"2011-03-01T00:00:00\",\n",
      "            \"smApplId\": 22187,\n",
      "            \"terms\": \" Acute; Address; Adverse effects; Antiviral Agents; Arteriviridae; base; Biological Assay; Biology; Cataloging; Catalogs; cell type; Cells; Cessation of life; Chikungunya virus; Chronic; Color; combat; Communicable Diseases; cytokine; Data; Development; Dissection; DNA Viruses; Environment; Event; Exhibits; Family; Flaviviridae; Flow Cytometry; Gene Expression; Gene Expression Profile; gene function; Gene Library; Genes; global health; HCV screening; Hepatitis C; Hepatitis C virus; Herpesviridae; HIV; HIV-1; Human; Immune response; Individual; Infection; Influenza; Influenza A virus; influenzavirus; insight; interest; Interferon Type I; Interferons; IRF1 gene; Knowledge; Lentivirus Vector; Life Cycle Stages; Longevity; Lung diseases; LY6E gene; MAP3K14 gene; Mediating; member; Molecular Probes; Orthomyxoviridae; Outcome; overexpression; pandemic disease; Paramyxoviridae; pathogen; Pathology; Pathway interactions; Phylogenetic Analysis; Poxviridae; Primary carcinoma of the liver cells; Proteins; public health relevance; Reoviridae; Reporting; response; Rhabdoviridae; RNA Viruses; Role; Screening procedure; Series; Signal Pathway; Specificity; System; Testing; Therapeutic; Time; Togaviridae; Translating; TREX1 gene; Validation; Venezuelan Equine Encephalitis Virus; Viral; Virus; Virus Diseases; West Nile virus; Yellow fever virus\",\n",
      "            \"title\": \"TYPE I INTERFERON-STIMULATED GENES AND THE ANTIVIRAL IMMUNE RESPONSE\",\n",
      "            \"totalCostAmount\": 422500\n",
      "        },\n",
      "        {\n",
      "            \"abstract\": \"DESCRIPTION (provided by applicant): Hepatitis C virus (HCV) infection continues to be a major burden on public health, affecting approximately 170 million people worldwide and 3-4 million Americans. HCV-associated end-stage liver disease is the leading indicator of liver transplantation. Current standard therapy with pegylated interferon-? in combination with ribavirin only achieves limited (<50%) antiviral response and causes severe side effects. The HCV protease- and polymerase-specific inhibitors currently in clinical trials are promising but are undermined by rapid emergence of drug-resistant HCV mutants. Future antiviral therapy for hepatitis C likely requires a combination of several drugs targeting different steps of the HCV life cycle. The lack of knowledge about the molecular details of the HCV life cycle has significantly impeded the discovery and development of antiviral drugs against HCV infection. A more complete understanding of the roles of viral and cellular proteins in the HCV life cycle will provide additional novel targets for anti-HCV drug discovery. Our recent studies have demonstrated that human apolipoprotein E (apoE) is an important determinant for the outcomes of HCV infection and assembly. ApoE was also found to interact with the HCV NS5A in our preliminary studies. In Specific Aim 1, we will dissect apoE domains and amino acid residues important for its dual functions in HCV infection and assembly. We will also determine the importance and the underlying molecular mechanisms of apoE and its interaction with NS5A in HCV infection and assembly, respectively. In preliminary studies, we have demonstrated that adaptive mutations in the HCV non-structural (NS) proteins greatly enhanced HCV production and cytopathogenicity in cell culture. We have also found that HCV NS proteins interact with viral structural proteins, as demonstrated by experiments with the mammalian two-hybrid system and co-immunoprecipitation assay. Consistent with our findings, recent genetic studies done by others suggest that HCV NS proteins play important roles in HCV assembly and/or production. However, the underlying molecular mechanisms of the HCV NS proteins in the HCV life cycle have not been defined. In Specific Aim 2, we will decipher the roles and mechanisms of action of cell culture adaptive mutations in HCV assembly and induction of cytopathogenicity. In Specific Aim 3, we will determine the importance of viral NS proteins and their interactions with structural proteins in HCV assembly. We will accomplish these specific aims using a robust HCV reverse genetics system recently developed in our lab in conjunction with mutagenesis studies and the use of biochemical, immunological, and cell biological approaches. New knowledge derived from these studies will lead to a paradigm shift in virology with respect to the roles of viral NS proteins in HCV assembly. Our studies will also provide novel targets for discovery of antiviral drugs to effectively treat hepatitis C.  \",\n",
      "            \"agency\": \"NIH\",\n",
      "            \"budgetEndDate\": \"2012-09-30T00:00:00\",\n",
      "            \"budgetStartDate\": \"2012-06-01T00:00:00\",\n",
      "            \"cfdaCode\": \"855\",\n",
      "            \"congressionalDistrict\": \"06\",\n",
      "            \"contactPi\": \"LUO, GUANGXIANG GEORGE\",\n",
      "            \"department\": \"HHS\",\n",
      "            \"dunsId\": \"939017877\",\n",
      "            \"fy\": 2012,\n",
      "            \"ic\": \"NIAID\",\n",
      "            \"latitude\": 38.028302,\n",
      "            \"longitude\": -84.50314,\n",
      "            \"nihApplId\": 8282631,\n",
      "            \"orgCity\": \"LEXINGTON\",\n",
      "            \"orgCountry\": \"UNITED STATES\",\n",
      "            \"orgName\": \"UNIVERSITY OF KENTUCKY\",\n",
      "            \"orgState\": \"KY\",\n",
      "            \"orgZipCode\": \"405260001\",\n",
      "            \"otherPis\": null,\n",
      "            \"projectEndDate\": \"2012-09-30T00:00:00\",\n",
      "            \"projectNumber\": \"5R01AI091953-02\",\n",
      "            \"projectStartDate\": \"2011-06-15T00:00:00\",\n",
      "            \"smApplId\": 22205,\n",
      "            \"terms\": \" Address; Adverse effects; Affect; American; Amino Acids; anti-hepatitis C; Antiviral Agents; Antiviral Response; Antiviral Therapy; Apolipoprotein E; base; Binding; Biochemical; Biological; Biological Assay; Cell Culture System; Cell Culture Techniques; Cells; Cessation of life; Cirrhosis; Clinical Trials; Co-Immunoprecipitations; Combined Modality Therapy; Development; Drug Delivery Systems; drug development; drug discovery; Drug resistance; Ectopic Expression; Future; Genetic; Genome; Genotype; Goals; HCV Vaccine; Hepatitis; Hepatitis C; Hepatitis C Therapy; Hepatitis C virus; Human; Infectious hepatitides; inhibitor/antagonist; Interferons; Knowledge; Lead; Life Cycle Stages; Liver diseases; liver transplantation; Molecular; Mutagenesis; mutant; Mutation; novel; Outcome; Peptide Hydrolases; Play; Polymerase; positional cloning; Primary carcinoma of the liver cells; Production; Protein Isoforms; Proteins; Public Health; Research; Research Design; research study; Ribavirin; RNA replication; RNA Sequences; Role; Sequence Analysis; Staging; Structural Protein; Structure; System; United States; Variant; Viral; Viral Nonstructural Proteins; Viral Proteins; viral RNA; Viral Structural Proteins; Virion; virology; Virus; Virus Assembly; yeast two hybrid system\",\n",
      "            \"title\": \"VIRAL AND CELLULAR DETERMINANTS OF HCV ASSEMBLY\",\n",
      "            \"totalCostAmount\": 59523\n",
      "        },\n",
      "        {\n",
      "            \"abstract\": \"DESCRIPTION (provided by applicant): Natural Killer T (iNKT) cells have evolved to recognize glycolipid antigens presented by CD1d molecules. Following stimulation through their T cell receptor (TCR), iNKT cells respond very rapidly, as is characteristic of innate rather than adaptive responses. iNKT cells have been implicated in the regulation of immune responses associated with a broad range of diseases, including autoimmunity, allergy, infectious diseases and cancer. iNKT cells can be activated through two major pathways. Microbe-specific CD1d-restricted lipids can stimulate iNKT cells directly. Alternatively, iNKT cells can be activated via recognition of CD1d-restricted self-antigens, in combination with inflammatory cytokines. This suggests that iNKT cells might play a role in the response to nearly all infectious agents. Furthermore, recognition of 'self' by iNKT cells might play an important role in cancer immunity as iNKT cells have been shown to play a critical role in the immune surveillance of carcinoma, sarcoma and hematopoietic malignancies. Similarly, the inflammatory cascade triggered during ischemia-reperfusion injury and sickle cell disease appears to be a direct consequence of iNKT cells activation by self-derived lipids. Importantly, self-antigens that can trigger an iNKT cell response may be responsible for the positive selection of these cells during their development. The nature of the self-lipid(s) that are involved in these processes is currently a subject of controversy. We have engineered iNKT T cell receptors that conserve the same specificity than 'regular iNKT TCR' but have a higher affinity for the antigen-CD1d complex. Using these unique tools, we have started to identify self-antigens that can bind the TCR when presented by CD1d molecules. Through distinctive and multi-pronged approaches we propose to examine what is/are the self-ligands of iNKT cells and how they are recognized. Finally, we will assess, using a highly innovative approach, how these newly identified self-ligands affect iNKT cell development. These studies will define the guidelines to optimize iNKT cell ligands and ultimately regulate iNKT cell function, with important implications for glycolipid-based vaccine development.  \",\n",
      "            \"agency\": \"NIH\",\n",
      "            \"budgetEndDate\": \"2013-07-31T00:00:00\",\n",
      "            \"budgetStartDate\": \"2012-08-01T00:00:00\",\n",
      "            \"cfdaCode\": \"855\",\n",
      "            \"congressionalDistrict\": \"06\",\n",
      "            \"contactPi\": \"GAPIN, LAURENT\",\n",
      "            \"department\": \"HHS\",\n",
      "            \"dunsId\": \"041096314\",\n",
      "            \"fy\": 2012,\n",
      "            \"ic\": \"NIAID\",\n",
      "            \"latitude\": 39.747483,\n",
      "            \"longitude\": -104.838638,\n",
      "            \"nihApplId\": 8311617,\n",
      "            \"orgCity\": \"AURORA\",\n",
      "            \"orgCountry\": \"UNITED STATES\",\n",
      "            \"orgName\": \"UNIVERSITY OF COLORADO DENVER\",\n",
      "            \"orgState\": \"CO\",\n",
      "            \"orgZipCode\": \"800452571\",\n",
      "            \"otherPis\": null,\n",
      "            \"projectEndDate\": \"2015-07-31T00:00:00\",\n",
      "            \"projectNumber\": \"5R01AI092108-02\",\n",
      "            \"projectStartDate\": \"2011-08-04T00:00:00\",\n",
      "            \"smApplId\": 22217,\n",
      "            \"terms\": \" Address; Affect; Affinity; alpha-galactosylceramide; Antigens; Autoantigens; Autoimmunity; autoreactivity; base; Binding; Biochemistry; Carcinoma; CD1d antigen; Cell physiology; Cells; Cellular biology; Characteristics; Collection; Communicable Diseases; Complex; cytokine; Data; Dendritic Cells; Detection; Development; Disease; Engineering; Future; genetic analysis; Glycolipids; Goals; Grant; Guidelines; Health; Hematopoietic Neoplasms; Histocompatibility Antigens Class I; Human; Hypersensitivity; Immune; Immune response; Immunity; Immunologic Surveillance; improved; in vivo; Infection; Infectious Agent; Inflammatory; innovation; interest; killer T cell; Lecithin; Ligands; Lipids; Lymphocyte; Malignant Neoplasms; Marines; Mass Spectrum Analysis; Microbe; microbial; Mus; Nature; novel; novel strategies; Pathway interactions; Peripheral; Phosphatidylserines; Play; Population; Porifera; Process; Regulation; Reperfusion Injury; response; Role; sarcoma; Sickle Cell Anemia; Specificity; Structure; Surface Plasmon Resonance; T cell response; T-Cell Activation; T-Cell Development; T-Cell Receptor; Testing; Therapeutic; Thymus Gland; tool; Trademark; tumor; vaccine development\",\n",
      "            \"title\": \"INKT CELL RECOGNITION OF ENDOGENOUS LIPID ANTIGENS\",\n",
      "            \"totalCostAmount\": 368595\n",
      "        },\n",
      "        {\n",
      "            \"abstract\": \"DESCRIPTION (provided by applicant): WU polyomavirus (WUV) is a newly discovered virus that has been widely detected in respiratory secretions of many patients suffering acute respiratory disease. Seroepidemiologic studies have determined that ~90% of adults in the general population have anti-WUV antibodies and thus the vast majority of the world's population has been infected at some point in time by WUV. Currently, the pathogenic potential of WUV is undefined. In some patients with acute respiratory disease, WUV is the only virus detected despite extensive screening for other respiratory viruses, raising the possibility that WUV causes respiratory disease. Based on paradigms provided by the known human pathogens BK and JC polyomaviruses, infection by WUV may result in persistent, lifelong infection that may lead to significant human disease in immunocompromised individuals. In addition, there has long been controversy as to whether SV40, BK and JC viruses play a role in human cancers; the recent discovery of Merkel polyomavirus and its strong association with Merkel cell carcinoma has provided the most compelling link to date between a polyomavirus and cancer. These studies raise the question of whether WUV might be associated with any tumors. Efforts to address these questions, as well as studies aimed at characterizing the fundamental virologic properties of WUV, would be greatly facilitated by the development of a cell culture replication system for WUV. The lack of any culture system for propagation of WUV is currently the fundamental barrier to further characterization of the virus and assessment of its relationship to human disease. For example, fulfillment of Koch's postulates, the gold standard for establishing microbial disease causality, necessarily requires that the agent be grown in culture. In addition, studies of receptors for WUV entry (and thus virus tropism), viral replication mechanisms and detailed analysis of intracellular WUV-host interactions are dependent on having robust culture systems. Successful culture of WUV is a critical milestone that would facilitate many future, long-term experiments essential for understanding of WUV biology.  \",\n",
      "            \"agency\": \"NIH\",\n",
      "            \"budgetEndDate\": \"2014-06-30T00:00:00\",\n",
      "            \"budgetStartDate\": \"2012-07-01T00:00:00\",\n",
      "            \"cfdaCode\": \"855\",\n",
      "            \"congressionalDistrict\": \"01\",\n",
      "            \"contactPi\": \"WANG, DAVID\",\n",
      "            \"department\": \"HHS\",\n",
      "            \"dunsId\": \"068552207\",\n",
      "            \"fy\": 2012,\n",
      "            \"ic\": \"NIAID\",\n",
      "            \"latitude\": 38.648424,\n",
      "            \"longitude\": -90.309068,\n",
      "            \"nihApplId\": 8282707,\n",
      "            \"orgCity\": \"SAINT LOUIS\",\n",
      "            \"orgCountry\": \"UNITED STATES\",\n",
      "            \"orgName\": \"WASHINGTON UNIVERSITY\",\n",
      "            \"orgState\": \"MO\",\n",
      "            \"orgZipCode\": \"631304862\",\n",
      "            \"otherPis\": null,\n",
      "            \"projectEndDate\": \"2014-06-30T00:00:00\",\n",
      "            \"projectNumber\": \"5R21AI095922-02\",\n",
      "            \"projectStartDate\": \"2011-07-01T00:00:00\",\n",
      "            \"smApplId\": 22300,\n",
      "            \"terms\": \" Acute; Address; Adult; Antibodies; base; Biology; BK Virus; Cell Culture System; Cell Culture Techniques; cell type; Child; Clinical; Development; Disease; Etiology; Excretory function; Future; General Population; Gold; Human; human disease; Human Virus; Immunocompromised Host; immunosuppressed; In Vitro; Individual; Infection; JC Virus; Lead; Link; Lung diseases; Malignant Neoplasms; Merkel cell carcinoma; microbial disease; novel; pathogen; Pathogenesis; Patients; Play; Polyomavirus; Polyomavirus Infections; Population; Property; Publishing; receptor; Recombinants; research study; respiratory; Respiratory Tract Infections; respiratory virus; Role; Screening procedure; Seroepidemiologic Studies; Simian virus 40; Specimen; System; Time; tissue tropism; Transfection; tumor; Urine; Viral; virology; Virus; Virus Diseases; virus tropism\",\n",
      "            \"title\": \"ESTABLISHMENT OF A REPLICATION SYSTEM FOR WU POLYOMAVIRUS\",\n",
      "            \"totalCostAmount\": 190000\n",
      "        },\n",
      "        {\n",
      "            \"abstract\": \"DESCRIPTION (provided by applicant): Glycosylphosphatidylinositol (GPI)-anchored proteins (GPI-APs) are ubiquitous in eukaryotes. Examples of GPI-APs include folate receptor, acetylcholinesterase, renal dipeptidase and the variant surface glycoproteins of Trypanosoma brucei, the causative agent of African sleeping sickness. Inability to synthesize GPI-APs results in embryonic lethality in mammals. Defective GPI biosynthesis in multipotent hematopoietic human stem cells causes paroxysmal nocturnal hemoglobinuria, an acquired hemolytic disease. GPI-APs are needed for fungal cell viability and they are important in diseases such as trypanosomiasis, malaria and leishmaniasis that are caused by parasitic protozoa. The GPI assembly pathway is a drug target for fungal and protozoal diseases. GPI anchoring is catalyzed by GPI transamidase (GPIT), a 5-subunit membrane- bound complex located in the endoplasmic reticulum (ER). The catalytic subunit, Gpi8, shares homology with caspases; the functional role of the other subunits is unclear, but all are required for GPIT activity. Three of the non-catalytic subunits are over-expressed in certain cancers, indicating a link between GPIT and oncogenesis. In this R21 application we propose to initiate structure-function studies of the GPIT complex using electron microscopy and X-ray crystallography. We are ultimately interested in establishing the structural organization of GPIT, delineating the role of its subunits, and understanding how this important enzyme is regulated. In two specific aims we propose to (1) analyze the endogenous GPIT complex from yeast by electron microscopy and (2) express GPIT subunits and sub-complexes for X-ray crystallographic studies. Our efforts are expected to yield a medium-resolution structure of GPIT and pave the way for a future high-resolution structure of the intact complex. These studies will have high impact as there is no structural information on GPIT; also, results obtained here will shed light on other multi-subunit membrane bound enzymes in the ER such as oligosaccharyltransferase and signal peptidase that play a critical role in processing a wide range of ER-translocated proteins, including proteins destined for GPI anchoring, but whose functional architecture remains largely a mystery.  \",\n",
      "            \"agency\": \"NIH\",\n",
      "            \"budgetEndDate\": \"2013-05-31T00:00:00\",\n",
      "            \"budgetStartDate\": \"2012-06-01T00:00:00\",\n",
      "            \"cfdaCode\": \"855\",\n",
      "            \"congressionalDistrict\": \"12\",\n",
      "            \"contactPi\": \"MENON, ANANT K\",\n",
      "            \"department\": \"HHS\",\n",
      "            \"dunsId\": \"060217502\",\n",
      "            \"fy\": 2012,\n",
      "            \"ic\": \"NIAID\",\n",
      "            \"latitude\": 40.76491,\n",
      "            \"longitude\": -73.955055,\n",
      "            \"nihApplId\": 8267601,\n",
      "            \"orgCity\": \"NEW YORK\",\n",
      "            \"orgCountry\": \"UNITED STATES\",\n",
      "            \"orgName\": \"WEILL MEDICAL COLLEGE OF CONRELL UNIVERSITY\",\n",
      "            \"orgState\": \"NY\",\n",
      "            \"orgZipCode\": \"100654805\",\n",
      "            \"otherPis\": null,\n",
      "            \"projectEndDate\": \"2013-05-31T00:00:00\",\n",
      "            \"projectNumber\": \"5R21AI096554-02\",\n",
      "            \"projectStartDate\": \"2011-06-01T00:00:00\",\n",
      "            \"smApplId\": 22324,\n",
      "            \"terms\": \" Abbreviations; Acetylcholinesterase; African Trypanosomiasis; Alkaline Phosphatase; Amino Acids; Anabolism; Architecture; Area; Binding; Biochemistry; Biological; Biological Assay; Biology; Canis familiaris; Caspase; Catalytic Domain; Cell Adhesion Molecules; Cell surface; Cell Survival; Cellular biology; Cleaved cell; Complex; COOH-terminal signal transamidase; Cysteine Protease; Defect; design; Disease; dolichyl-diphosphooligosaccharide - protein glycotransferase; Drug Delivery Systems; Electron Microscopy; Embryo; Endoplasmic Reticulum; enzyme structure; Enzymes; Epitopes; Eukaryota; experience; Family; folate-binding protein; Future; glycosylation; Glycosylphosphatidylinositols; Glypican; GPI Membrane Anchors; Hematopoietic; Human; human stem cells; inhibitor/antagonist; insight; interest; Laboratories; Learning; Leishmaniasis; Light; Link; Location; Malaria; Malignant Neoplasms; Mammals; man; Mannose; Maps; Membrane; membrane dipeptidase; Membrane Glycoproteins; Membrane Proteins; Multienzyme Complexes; NCAM1 gene; pathogen; Pathway interactions; Peptide Signal Sequences; Phosphatidylinositols; phosphoethanolamine; PI-Glycan; Play; Prions; Process; protein complex; protein structure; Protein Subunits; Proteins; Protozoa; rat Piga protein; reconstitution; Recruitment Activity; Reporting; Resolution; Roentgen Rays; Role; signal peptidase; structural biology; Structure; tool; transamidases; Trypanosoma brucei brucei; Trypanosomiasis; tumorigenesis; Universities; Variant; Work; X-Ray Crystallography; Yeasts\",\n",
      "            \"title\": \"STRUCTURAL ANALYSIS OF THE GPI TRANSAMIDASE COMPLEX\",\n",
      "            \"totalCostAmount\": 211250\n",
      "        },\n",
      "        {\n",
      "            \"abstract\": \"DESCRIPTION (provided by applicant): Plakophilin 3 (PKP3) is a desmosomal protein that belongs to the armadillo family of cell adhesion and signaling proteins. Our preliminary data demonstrate that loss of Pkp3 function in epidermal keratinocytes leads to an up regulation of several armadillo proteins (plakophilin 1, plakoglobin, \\u00bf-catenin), reduced cell migration and an increased susceptibility to oncogene-mediated transformation. Consequently, we concluded that PKP3 acts as a tumor suppressor gene in stratified epithelia. Our data provide a mechanistic explanation for the observed correlation between PKP3 downregulation and increased malignancy observed in human squamous cell carcinomas. We also found that PKP3 interacts with G3BP and Rack-1 proteins that are thought to regulate cellular stress response, such as cell death and cell survival decisions in cells exposed to heat or chemical stress. The successful completion of this project will lead to the identification of novel cell signaling pathways in keratinocytes that are important for the cellular stress response, cell migration and the susceptibility of keratinocytes to tumor development.  \",\n",
      "            \"agency\": \"NIH\",\n",
      "            \"budgetEndDate\": \"2014-04-30T00:00:00\",\n",
      "            \"budgetStartDate\": \"2012-05-01T00:00:00\",\n",
      "            \"cfdaCode\": \"846\",\n",
      "            \"congressionalDistrict\": \"06\",\n",
      "            \"contactPi\": \"KOCH, PETER J.\",\n",
      "            \"department\": \"HHS\",\n",
      "            \"dunsId\": \"041096314\",\n",
      "            \"fy\": 2012,\n",
      "            \"ic\": \"NIAMS\",\n",
      "            \"latitude\": 39.747483,\n",
      "            \"longitude\": -104.838638,\n",
      "            \"nihApplId\": 8293310,\n",
      "            \"orgCity\": \"AURORA\",\n",
      "            \"orgCountry\": \"UNITED STATES\",\n",
      "            \"orgName\": \"UNIVERSITY OF COLORADO DENVER\",\n",
      "            \"orgState\": \"CO\",\n",
      "            \"orgZipCode\": \"800452571\",\n",
      "            \"otherPis\": null,\n",
      "            \"projectEndDate\": \"2014-04-30T00:00:00\",\n",
      "            \"projectNumber\": \"5R21AR060145-02\",\n",
      "            \"projectStartDate\": \"2011-07-01T00:00:00\",\n",
      "            \"smApplId\": 22343,\n",
      "            \"terms\": \" Adhesions; Affect; armadillo proteins; Binding Proteins; biological adaptation to stress; Biology; carcinogenesis; Cell Adhesion; Cell Adhesion Molecules; Cell Death; cell motility; Cell physiology; Cell Survival; cell transformation; cell type; Cell-Cell Adhesion; Cells; Cellular Stress; Cellular Stress Response; Chemicals; Complex; Cytoplasmic Granules; Data; Defect; Development; Down-Regulation; Exposure to; Family; Family Dasypodidae; Goals; hazard; Health; Heating; Human; In Vitro; in vivo; In-Migration; keratinocyte; Lead; Link; Malignant Neoplasms; Mammalian Cell; Mediating; migration; mouse model; Mus; mutant; novel; Oncogenes; Oncogenic; Pathway interactions; Phenotype; plakoglobin; plakophilins; Play; polypeptide D6; Predisposition; Process; Property; protein expression; protein function; Protein Isoforms; Protein Kinase C; Proteins; reconstitution; Regulation; research study; response; Role; Scaffolding Protein; Signal Pathway; Signal Transduction; Signaling Protein; Skin; Skin Neoplasms; Squamous cell carcinoma; Stratified Epithelium; Stress; Testing; tumor; tumor initiation; tumor progression; Tumor Suppressor Genes; Tumor Suppressor Proteins; tumorigenesis; Ultraviolet Rays; Up-Regulation\",\n",
      "            \"title\": \"THE ROLE OF PLAKOPHILIN 3 IN KERATINOCYTE BIOLOGY\",\n",
      "            \"totalCostAmount\": 204003\n",
      "        },\n",
      "        {\n",
      "            \"abstract\": \"DESCRIPTION (provided by applicant):  Muscle wasting occurs during aging, HIV infection, cancer, and numerous other pathological conditions, resulting in a significant decrease in quality of life and a financial burden of $18.5 billion in 2000. While 3D in vitro models of skin and lung tissue have proven essential in elucidating mechanisms of homeostasis and disease progression, analogous models of skeletal muscle do not exist. We propose a 3D model of primary human skeletal muscle that utilizes an engineered extracellular matrix (eECM), gradients of chemotactic cues, and cellular patterning. This collaborative proposal combines complementary expertise in cell microenvironment engineering and human muscle progenitor cell (hMuPC) and myoblast biology. Aim 1 is to develop and optimize a 3D eECM to enhance the proliferation of hMuPCs. Previous results show that hMuPCs are critically responsive to the biochemistry and biomechanics of the microenvironment and have diminished proliferation and regeneration following 2D culture. Customized eECM will be made from a protein-engineered biomaterial that enables independent tuning of biomechanics (elastic moduli = 1-100 kPa) and cell-ligand density (0-100,000 ligands/micron3). Viability, proliferation, and myogenic differentiation of hMuPCs will be directly compared between 2D and 3D cultures utilizing identical eECM. Aim 2 is to develop a 3D in vitro model of hMuPC migration. Little is known about the soluble cues that regulate hMuPC migration to sites of regeneration in vivo. Time-lapse imaging of hMuPC migration speed, directional persistence, and filopodia extension will be performed in a microfluidic device that enables the formation of stable concentration profiles. Migration will be compared on 2D and in 3D eECM in response to gradients and uniform concentrations of putative chemotactic cues. Migration in response to cell lysates from young (18-25 years old), old (60-80 years old), and dystrophic human skeletal muscle biopsies will be quantified to identify potential novel regulators of chemotaxis. Aim 3 is to develop a 3D patterned mimic of human skeletal muscle tissue. Human myoblasts will be cultured on patterned eECM to induce myotube fusion and alignment. Fiber fusion rate, maturity, nuclear index, and alignment will be compared on eECM of varying pattern geometry, biomechanics, and biochemistry. Multiple sheets of aligned myotubes will be layered together with hMuPCs to create a dynamic model of regenerating muscle tissue. These aims will lead to new 3D technologies for tissue culture, fundamental new insights in skeletal muscle biology, and potential new clinical therapies to activate hMuPCs and stimulate regeneration of muscle damaged during wasting and aging. (End of Abstract)  \",\n",
      "            \"agency\": \"NIH\",\n",
      "            \"budgetEndDate\": \"2013-12-31T00:00:00\",\n",
      "            \"budgetStartDate\": \"2012-09-01T00:00:00\",\n",
      "            \"cfdaCode\": \"846\",\n",
      "            \"congressionalDistrict\": \"18\",\n",
      "            \"contactPi\": \"HEILSHORN, SARAH C\",\n",
      "            \"department\": \"HHS\",\n",
      "            \"dunsId\": \"009214214\",\n",
      "            \"fy\": 2012,\n",
      "            \"ic\": \"NIAMS\",\n",
      "            \"latitude\": 37.430351,\n",
      "            \"longitude\": -122.17821,\n",
      "            \"nihApplId\": 8325582,\n",
      "            \"orgCity\": \"STANFORD\",\n",
      "            \"orgCountry\": \"UNITED STATES\",\n",
      "            \"orgName\": \"STANFORD UNIVERSITY\",\n",
      "            \"orgState\": \"CA\",\n",
      "            \"orgZipCode\": \"943041222\",\n",
      "            \"otherPis\": \"BLAU, HELEN M ; \",\n",
      "            \"projectEndDate\": \"2013-12-31T00:00:00\",\n",
      "            \"projectNumber\": \"5R21AR062359-02\",\n",
      "            \"projectStartDate\": \"2011-09-01T00:00:00\",\n",
      "            \"smApplId\": 22382,\n",
      "            \"terms\": \" abstracting; Adult; aged; Aged, 80 and over; Aging; Biochemical; Biochemistry; Biocompatible Materials; Biology; Biomechanics; Biomedical Engineering; Biopsy; Cachexia; cell motility; Cell physiology; Cell Proliferation; Cell Survival; Cells; cellular engineering; Cellular Morphology; Chemotaxis; Chronic; Clinical; Collaborations; Conditioned Culture Media; Cues; Cultured Cells; density; design; Development; Disease; Disease Progression; Encapsulated; Engineering; Extracellular Matrix; Fiber; Filopodia; Goals; high risk; high throughput screening; HIV Infections; Homeostasis; Human; Human Engineering; Image; In Vitro; in vitro Model; in vivo regeneration; indexing; injured; In-Migration; innovation; insight; Knowledge; Laboratories; Lead; Life; Ligands; loss of function; Malignant Neoplasms; Microfluidic Microchips; Microfluidics; migration; Modeling; Molecular; Mus; Muscle; Muscle Fibers; muscle form; Muscle function; muscle regeneration; Muscle satellite cell; Muscular Dystrophies; Myoblasts; Natural regeneration; novel; Nuclear; Pattern; Pharmacologic Substance; Physiology; Process; progenitor; Property; Protein Engineering; Public Health; Quality of life; regenerative; Regulation; repaired; response; Rest; sarcopenia; satellite cell; scaffold; Scientist; Screening procedure; Site; Skeletal Muscle; skeletal muscle wasting; Skin Tissue; Speed; Stem cells; Structure of parenchyma of lung; Technology; Testing; Therapeutic Agents; three-dimensional modeling; Time; tissue culture; Tissues; wasting\",\n",
      "            \"title\": \"3D BIOENGINEERING STRATEGIES TO MIMIC HUMAN SKELETAL MUSCLE PROGENITOR CELL NICHE\",\n",
      "            \"totalCostAmount\": 165629\n",
      "        },\n",
      "        {\n",
      "            \"abstract\": \"DESCRIPTION (provided by applicant): This proposal concerns molecular mechanisms leading to the development of neural crest cells. The neural crest is a population of stem-cell-like cells, which form in vertebrate embryos at the neural plate border and migrate to diverse locations in the body to give rise to diverse cell types, including face cartilage, melanocytes and the peripheral nervous system. Neural crest forms as a result of inductive interactions of neuroectoderm, epidermal ectoderm and the underlying mesoderm. One embryonic signaling pathway that plays an essential role in neural crest induction is the Wnt pathway, known to control cell proliferation, fates, cell polarity and motility in many cell types. Whereas the role for the Wnt/2-catenin pathway in neural crest regulation has been established, the functions of 2-catenin-independent Wnt signals in this process remain largely unknown. Our preliminary studies reveal a function for 2-catenin-independent Wnt proteins in neural crest specification. We have also discovered that PAR proteins are involved in the activation of neural crest-specific genes. To connect cell polarization to neural crest specification, we plan to investigate the mechanism, by which PAR proteins influence neural crest fates. The proposed studies will evaluate at which developmental step PAR function is required, whether the response to known neural crest regulators is altered and whether the proper subcellular localization of PAR proteins is critical for this process. Using an unbiased protein interaction screen and the candidate protein approach, we will identify downstream targets of PAR proteins, which contribute to neural crest development. Other studies will determine how upstream signaling pathways regulate PAR localization, levels and activity to specify neural crest fates. These experiments will be carried out in Xenopus embryos, which represent a rapid in vivo system for gene function and allow classical cell biological and biochemical experiments. By connecting PAR and Wnt signaling during the initial stages of craniofacial development, the proposed experiments will contribute to the understanding of both the basic developmental mechanisms and will be relevant to the disease. A large number of human diseases, such as craniosynostosis, Waardenburg and Hirschsprung's syndromes, and cancer, have been associated with neural crest abnormalities. Studying the mechanisms underlying the development of the neural crest cells should provide insights into human diseases associated with stem cell disorders and cancer.  PUBLIC HEALTH RELEVANCE: This application concerns molecular mechanisms leading to the development of neural crest cells. Since neural crest abnormalities are known to lead to craniosynostosis, Waardenburg and Hirschsprung's syndromes, and cancer, the proposed studies are highly relevant to human diseases associated with stem cell disorders and tumorigenesis.  \",\n",
      "            \"agency\": \"NIH\",\n",
      "            \"budgetEndDate\": \"2013-02-28T00:00:00\",\n",
      "            \"budgetStartDate\": \"2012-03-01T00:00:00\",\n",
      "            \"cfdaCode\": \"853\",\n",
      "            \"congressionalDistrict\": \"13\",\n",
      "            \"contactPi\": \"SOKOL, SERGEI\",\n",
      "            \"department\": \"HHS\",\n",
      "            \"dunsId\": \"078861598\",\n",
      "            \"fy\": 2012,\n",
      "            \"ic\": \"NINDS\",\n",
      "            \"latitude\": 40.790552,\n",
      "            \"longitude\": -73.95385,\n",
      "            \"nihApplId\": 8238037,\n",
      "            \"orgCity\": \"NEW YORK\",\n",
      "            \"orgCountry\": \"UNITED STATES\",\n",
      "            \"orgName\": \"MOUNT SINAI SCHOOL OF MEDICINE\",\n",
      "            \"orgState\": \"NY\",\n",
      "            \"orgZipCode\": \"100296574\",\n",
      "            \"otherPis\": null,\n",
      "            \"projectEndDate\": \"2016-02-29T00:00:00\",\n",
      "            \"projectNumber\": \"2R01NS040972-11A1\",\n",
      "            \"projectStartDate\": \"2001-02-01T00:00:00\",\n",
      "            \"smApplId\": 22503,\n",
      "            \"terms\": \" base; Biochemical; Biological; Biological Assay; Cartilage; Cell Communication; cell fate specification; cell motility; Cell Polarity; Cell Proliferation; cell type; Cells; cellular targeting; Congenital Abnormality; craniofacial; Craniosynostosis; Development; Developmental Process; Disease; Dorsal; Ectoderm; Ectoderm Cell; Embryo; Epithelial; Face; Gene Expression; gene function; Genes; Goals; Human; human disease; human JTB protein; in vivo; insight; Lead; Ligands; Link; Location; loss of function; Malignant Neoplasms; Mass Spectrum Analysis; Mediating; melanocyte; melanoma; Mesenchymal; Mesoderm; Molecular; Morphogenesis; Myosin Type II; Nervous system structure; Neural Crest; Neural Crest Cell; neural plate; Neuroblastoma; neurodevelopment; Neuroectoderm; non-muscle myosin; novel; Pathway interactions; Pattern; Peripheral Nervous System; Play; Population; Positioning Attribute; Process; progenitor; prospective; Proteins; receptor; Regulation; Research; research study; response; rho GTP-Binding Proteins; Role; Signal Pathway; Signal Transduction; Staging; stem cell biology; stem cell population; Stem cells; Syndrome; System; Testing; To specify; Transcriptional Regulation; tumorigenesis; Wnt proteins; Xenopus\",\n",
      "            \"title\": \"PATTERNING OF THE VERTEBRATE NERVOUS SYSTEM\",\n",
      "            \"totalCostAmount\": 379748\n",
      "        },\n",
      "        {\n",
      "            \"abstract\": \"DESCRIPTION (provided by applicant): The focus of this proposal is the nicotinic-receptor superfamily of ion channels. Our long-term goal is to elucidate how the biophysical properties of these receptor-channels shape their function under both normal and pathological conditions. To this end, we will apply a multi-faceted approach that includes single-channel and ensemble electrophysiology, protein engineering, X-ray crystallography and theoretical calculations. Most of the experiments will be conducted on the muscle nicotinic acetylcholine receptor (AChR) and its bacterial counterparts, ELIC and GLIC. Because of its large single-channel conductance, robust stoichiometry, tractable kinetics and high expression levels in heterologous expression systems, the muscle AChR remains an unparalleled model for the entire superfamily. Furthermore, throughout the years, research on this particular receptor has provided answers to fundamental questions on the physics, chemistry and biology of ion channels that cannot be addressed with (probably) any other channel. In this application, we propose to address the three pillars of ion-channel function, namely, ion conduction, gating, and modulation of gating. Our three Specific Aims are: 1) To characterize the ring of glutamates in the charge selectivity filter of the AChR in terms of number of total negative charges and acid-base behavior; 2) To identify the nature of the gate(s) and its rearrangement upon channel opening, closing and desensitization; and 3) To characterize the regions of the AChR hypothesized to be involved in the metabolic and intramembrane modulation of function. Because of its role as a paradigmatic neurotransmitter-gated ion channel, the knowledge derived from the proposed experiments on the muscle AChR is poised to have a broad impact on our understanding of postsynaptic receptors and fast synaptic transmission in general. It should also be emphasized that AChR-mediated neurotransmission supports not only motor, but also, autonomic and cognitive function. As a result, in addition to being compromised in myasthenic syndromes, cholinergic neurotransmission is also impaired in cardiovascular diseases and neurodegenerative disorders such as Alzheimer's disease and schizophrenia. Even more broadly, AChRs are also expressed in the airways, keratinocytes, lymphocytes and endothelial cells where they are mediators of cell differentiation, proliferation and resistance to apoptosis, and hence, of cancer.  PUBLIC HEALTH RELEVANCE: The main subject of this research proposal is the biophysical characterization of the nicotinic acetylcholine receptor. In the central and autonomic nervous system, nicotinic receptors play important roles in learning, memory and the modulation of cardiac function. It follows then a full understanding of not only motor, but also, cardiovascular and cognitive pathological conditions (such as Alzheimer's disease and schizophrenia) requires a thorough knowledge of the molecular underpinnings of these receptors. Furthermore, there is growing evidence that nicotinic receptors are also expressed in the lung, skin, platelets, immune cells and the cells that line the blood vessels, where they mediate cell growth and, eventually, the development of malignant tumors.  \",\n",
      "            \"agency\": \"NIH\",\n",
      "            \"budgetEndDate\": \"2013-05-31T00:00:00\",\n",
      "            \"budgetStartDate\": \"2012-06-01T00:00:00\",\n",
      "            \"cfdaCode\": \"853\",\n",
      "            \"congressionalDistrict\": \"13\",\n",
      "            \"contactPi\": \"GROSMAN, CLAUDIO F\",\n",
      "            \"department\": \"HHS\",\n",
      "            \"dunsId\": \"041544081\",\n",
      "            \"fy\": 2012,\n",
      "            \"ic\": \"NINDS\",\n",
      "            \"latitude\": 40.094459,\n",
      "            \"longitude\": -88.238635,\n",
      "            \"nihApplId\": 8315329,\n",
      "            \"orgCity\": \"CHAMPAIGN\",\n",
      "            \"orgCountry\": \"UNITED STATES\",\n",
      "            \"orgName\": \"UNIVERSITY OF ILLINOIS URBANA-CHAMPAIGN\",\n",
      "            \"orgState\": \"IL\",\n",
      "            \"orgZipCode\": \"618207406\",\n",
      "            \"otherPis\": null,\n",
      "            \"projectEndDate\": \"2017-05-31T00:00:00\",\n",
      "            \"projectNumber\": \"2R01NS042169-09\",\n",
      "            \"projectStartDate\": \"2001-07-01T00:00:00\",\n",
      "            \"smApplId\": 22506,\n",
      "            \"terms\": \" Acids; Address; Alzheimer's Disease; Amino Acids; Apoptosis; Autonomic nervous system; base; Behavior; Biology; Blood Platelets; Blood Vessels; Cardiac; Cardiovascular Diseases; Cardiovascular system; Catalytic Domain; Cations; Cell Differentiation process; cell growth; Cell Line; Cells; Charge; Chemistry; cholinergic; Cholinergic Receptors; Cognitive; cognitive function; Complex; C-terminal; cyclic-nucleotide gated ion channels; Cysteine; Data; desensitization; design; Development; DNA Sequence Rearrangement; Electrophysiology (science); Endothelial Cells; Engineering; Excision; extracellular; Gated Ion Channel; Glutamates; Glutamine; Glycine Receptors; Goals; Immune; Ion Channel; Ions; keratinocyte; Kinetics; Knowledge; Lead; Learning; Lung; Lymphocyte; Malignant Neoplasms; Mediating; Mediator of activation protein; member; Membrane; Memory; Metabolic; Methods; Modeling; Molecular; Motor; Muscle; muscular system; Mutagenesis; mutant; Mutation; Myasthenic Syndrome; Nature; Neurodegenerative Disorders; Neurons; neurotransmission; Neurotransmitters; Nicotinic Receptors; Outcome; Oxidation-Reduction; patch clamp; Perfusion; Phosphorylation; Physics; Play; postsynaptic; prevent; Property; Protein Engineering; Protein Region; receptor; receptor function; Research; Research Proposals; research study; Resistance; Role; Schizophrenia; Serine; Shapes; Side; Skin; stoichiometry; success; Synaptic Transmission; Tail; Testing; Tyrosine; Ursidae Family; voltage; Water; Work; X-Ray Crystallography\",\n",
      "            \"title\": \"MECHANISMS OF NEUROTRANSMITTER-GATED ION CHANNELS\",\n",
      "            \"totalCostAmount\": 352736\n",
      "        },\n",
      "        {\n",
      "            \"abstract\": \"DESCRIPTION (provided by applicant): In 2002, we cloned Tim-3 as a molecule differentially expressed of IFN?-producing Th1/Tc1 cells. However, we have found that Tim-3 is also expressed on pro-inflammatory Th17 cells. Subsequently, we identified Galectin- 9 as the Tim-3 ligand and found that the Galectin-9/Tim-3 interaction led to cell death and termination of Th1/Tc1 responses. These studies, together with other in vivo blockade studies with soluble Tim-3, suggested that Tim-3 is an inhibitory molecule that dampens effector Th1/Tc1 responses. Exciting new data suggest that Tim-3 is also involved in inducing T cell exhaustion. Tim-3 expression is increased on effector T cells in chronic viral infections (HIV, HCV) and cancers, rendering them dysfunctional. However, in human autoimmune diseases there is loss of Tim-3 expression on effector T cells, making them highly proinflammatory and pathogenic. Therefore, Tim-3 plays a central regulatory role in multiple human diseases yet very little is known about the functional biology of Tim-3. It is not clear what induces Tim-3 expression on T cells and how it mediates its inhibitory effects. We have made a novel observation that IL-27, an immunosuppressive cytokine, is a potent inducer of Tim-3 expression on T cells. To understand the intracellular pathways that mediate Tim-3 function, we undertook a yeast two-hybrid screen and identified Bat3 (HLA-B associated transcript 3) as a molecule that binds to the Tim-3 cytoplasmic tail. Based on our preliminary data, we hypothesize that IL-27 produced during the resolution phase of autoimmune diseases induces Tim-3 expression on effector T cells to promote T cell death/exhaustion and resolve tissue inflammation and that Bat3, when bound to the Tim-3 tail, acts as a molecular 'gate-keeper' that regulates the inhibitory function of Tim-3. To address this hypothesis we propose two specific aims: 1) Analyze the molecular mechanisms by which Tim-3 is induced on effector T cells. We have found that IL-27 induces the transcription factor NFIL3, which may be involved in Tim-3 expression. Using NFIL3-/- mice, we will determine whether the IL-27/NFIL3/Tim-3 axis is critical for the development of experimental autoimmune encephalomyelitis (EAE) and induction of T cell exhaustion/dysfunction. 2) Identify the molecular mechanisms by which Tim-3 signals into T cells to induce T cell dysfunction/exhaustion and inhibit EAE. Since we have identified Bat3 as a binding partner for the Tim-3 tail, we will analyze whether conditional loss of Bat-3 in T cells induces T cell dysfunction and study how Bat3 bound to Tim-3 regulates proximal CD3-TcR signaling. The proposed studies will identify the molecular mechanisms by which Tim-3 mediates its inhibitory function, which could be exploited to dampen pathogenic T cells in autoimmune diseases.  PUBLIC HEALTH RELEVANCE: Tim-3:galectin-9 pathway has emerged as an important player in inducing T cell inhibition/exhaustion in effector T cells in chronic human viral infectios like HIV and HCV. In contrast in a number of human autoimmune diseases including multiple sclerosis, Psoriasis, Rheumatoid arthritis and type 1 diabetes, there is evidence that Tim-3 is functioning suboptimally resulting in generation of highly pathogenic effector T cells. This proposal will provide a greater understanding of how Tim-3 can be induced and how it functions in effector T cells so that the pathway can be therapeutically exploited in chronic viral infection, cancers and autoimmune diseases.   \",\n",
      "            \"agency\": \"NIH\",\n",
      "            \"budgetEndDate\": \"2013-07-31T00:00:00\",\n",
      "            \"budgetStartDate\": \"2012-09-30T00:00:00\",\n",
      "            \"cfdaCode\": \"853\",\n",
      "            \"congressionalDistrict\": \"07\",\n",
      "            \"contactPi\": \"KUCHROO, VIJAY K.\",\n",
      "            \"department\": \"HHS\",\n",
      "            \"dunsId\": \"030811269\",\n",
      "            \"fy\": 2012,\n",
      "            \"ic\": \"NINDS\",\n",
      "            \"latitude\": 42.334978,\n",
      "            \"longitude\": -71.105987,\n",
      "            \"nihApplId\": 8438835,\n",
      "            \"orgCity\": \"BOSTON\",\n",
      "            \"orgCountry\": \"UNITED STATES\",\n",
      "            \"orgName\": \"BRIGHAM AND WOMEN'S HOSPITAL\",\n",
      "            \"orgState\": \"MA\",\n",
      "            \"orgZipCode\": \"021156110\",\n",
      "            \"otherPis\": null,\n",
      "            \"projectEndDate\": \"2017-07-31T00:00:00\",\n",
      "            \"projectNumber\": \"2R01NS045937-10\",\n",
      "            \"projectStartDate\": \"2003-03-15T00:00:00\",\n",
      "            \"smApplId\": 22518,\n",
      "            \"terms\": \" Address; Antigens; Applications Grants; Autoimmune Diseases; Autoimmunity; base; Binding; Biology; CD28 gene; CD3 Antigens; CD8B1 gene; Cell Death; Cell physiology; Cell surface; Cells; Cessation of life; Chiroptera; Chronic; Communicable Diseases; cytokine; Cytoplasmic Tail; Data; Defect; Development; Disease Progression; exhaust; exhaustion; Experimental Autoimmune Encephalomyelitis; Functional disorder; Galactose Binding Lectin; Galectin 3; Gene Family; Generations; Genes; Genetic; Hepatitis C virus; HIV; HLA-B Antigens; Human; human disease; Immune; Immune system; Immunoglobulins; Immunosuppressive Agents; In Vitro; in vivo; Incidence; Individual; Infection Control; Inflammation; Inflammatory; Insulin-Dependent Diabetes Mellitus; Interferons; Interleukin-10; Interleukin-2; Knockout Mice; Laboratories; Ligands; Malignant Neoplasms; Mediating; member; Molecular; mouse model; Mucins; Multiple Sclerosis; Mus; novel; Pathway interactions; Phase; Phenotype; Play; Predisposition; Production; programs; Psoriasis; Regulation; Research; research study; Resolution; response; Rheumatoid Arthritis; Role; Signal Transduction; T cell response; Tail; T-bet protein; T-Cell Development; Tissues; T-Lymphocyte; TNF gene; Transcript; transcription factor; Up-Regulation; Viral; Virus Diseases; Work; yeast two hybrid system; Yeasts\",\n",
      "            \"title\": \"ROLE OF TH1 SPECIFIC CELL SURFACE MOLECULE TIM-3 IN EAE\",\n",
      "            \"totalCostAmount\": 350447\n",
      "        },\n",
      "        {\n",
      "            \"abstract\": \"DESCRIPTION (provided by applicant): Our lab has identified an interesting relation between members of the Myc and E2f family of cell cycle regulators and chromatin changes and transcriptional regulation of differentiation of oligodendrocyte progenitors. We now propose to use primary cultures and genetic mouse models to test a new mechanistic model of oligodendrocyte differentiation. The results from the proposed experimental plan are likely to advance our knowledge not only in neural development and repair, but also in cancer and nuclear reprogramming and have wide implications for the development of novel therapeutic strategies.  PUBLIC HEALTH RELEVANCE: The relationship between cell cycle and differentiation in progenitor cells is critical for a vast number of pathologies, ranging from cancer to repair. Although considerable progress has been made in the study of oncogenes in cellular transformation, still very little is known about their role in physiological conditions. This projet defines the role of Myc and E2F in the oligodendrocyte lineage and proposes a novel model of progenitor differentiation, linking cell cycle exit to chromatin condensation and transcriptional modulation.  \",\n",
      "            \"agency\": \"NIH\",\n",
      "            \"budgetEndDate\": \"2013-02-28T00:00:00\",\n",
      "            \"budgetStartDate\": \"2012-03-01T00:00:00\",\n",
      "            \"cfdaCode\": \"853\",\n",
      "            \"congressionalDistrict\": \"13\",\n",
      "            \"contactPi\": \"CASACCIA, PATRIZIA\",\n",
      "            \"department\": \"HHS\",\n",
      "            \"dunsId\": \"078861598\",\n",
      "            \"fy\": 2012,\n",
      "            \"ic\": \"NINDS\",\n",
      "            \"latitude\": 40.790552,\n",
      "            \"longitude\": -73.95385,\n",
      "            \"nihApplId\": 8319125,\n",
      "            \"orgCity\": \"NEW YORK\",\n",
      "            \"orgCountry\": \"UNITED STATES\",\n",
      "            \"orgName\": \"MOUNT SINAI SCHOOL OF MEDICINE\",\n",
      "            \"orgState\": \"NY\",\n",
      "            \"orgZipCode\": \"100296574\",\n",
      "            \"otherPis\": null,\n",
      "            \"projectEndDate\": \"2016-02-29T00:00:00\",\n",
      "            \"projectNumber\": \"2R01NS052738-06A1\",\n",
      "            \"projectStartDate\": \"2007-06-15T00:00:00\",\n",
      "            \"smApplId\": 22535,\n",
      "            \"terms\": \" Affect; Binding; Cancer Biology; cancer cell; cdc Genes; Cell Cycle; Cell Cycle Proteins; Cell Differentiation process; Cells; ChIP-seq; Chromatin; chromatin immunoprecipitation; Chromatin Structure; c-Myc Staining Method; Co-Immunoprecipitations; Collaborations; Complex; Data; Deacetylation; Development; developmental neurobiology; Discipline; Dominant-Negative Mutation; Down-Regulation; E2F1 gene; Energy Metabolism; Enzymes; Family; Family member; Funding; Gene Cluster; Gene Expression; Gene Expression Profile; gene repression; Gene Targeting; Genes; Genetic; Glioblastoma; Grant; HDAC1 gene; Histone Acetylation; histone acetyltransferase; Histones; In Vitro; in vivo; interest; Internal Ribosome Entry Site; Kinetics; Knowledge; Link; Malignant Neoplasms; Mediating; member; Metabolic; metaplastic cell transformation; Modeling; Molecular; mouse model; Mus; myelination; nervous system disorder; neurodevelopment; neuro-oncology; Nodal; novel; novel therapeutics; nuclear reprogramming; oligodendrocyte lineage; Oligodendroglia; Oncogenes; Pathology; Physical condensation; Physiological; prevent; progenitor; programs; Proliferating; Proteins; Regulator Genes; repaired; Role; Shapes; Sorting - Cell Movement; Specificity; Stem cells; Testing; Time; Transcriptional Regulation; Up-Regulation; Validation\",\n",
      "            \"title\": \"ROLE OF CELL CYCLE REGULATORS IN DIFFERENTIATION\",\n",
      "            \"totalCostAmount\": 370781\n",
      "        },\n",
      "        {\n",
      "            \"abstract\": \"DESCRIPTION (provided by applicant): Lyme disease (LD) is the most commonly reported vector-borne disease in the United States with approximately 38,000 confirmed and probable cases in 2009. The currently recommended method for diagnosing LD is a two-tiered serology-based test that is limited in its ability to differentiate active from previous infection, diagnose early LD, and is not standardized among laboratories. These limitations can result in misinterpretations and have led to millions of diagnostic tests (FDA approved and non-approved) being performed each year for this infection. Thus, a fresh approach for improved diagnostic development is critical to the field. Studies applying the global evaluation of small molecule biomarkers, 'metabolomics', for the discovery of biosignatures and biomarkers for monitoring cancers and metabolic diseases have shown tremendous promise. Although this technology has not been widely applied for infectious diseases, it is recognized that infectious diseases are manifested based on alterations to the biochemistry of a biological system. Thus, as is now being shown with other infections, we hypothesize that metabolomics offers an innovative approach for identification of biosignatures and biomarkers of LD. This R21/R33 application proposes to, in the first two years, elucidate multiple metabolome-based biosignatures of LD; initiate identification of products that comprise these biosignatures; and establish tools for the elucidation of the Borrelia burgdorferi metabolome. Successful completion of these initial studies and tool development efforts will lead to efforts in Years 3-5 to qualify the LD biosignatures; complete the molecular identificationof the biosignature products; and provide a comprehensive analysis of the B. burgdorferi metabolome. These activities encompassing the entirety of the R21/R33 program are framed within three Specific Aims: 1) Develop and qualify metabolic biosignatures that differentiate active LD from look-alike infections, and define the stages of disease including cured LD; 2) Identify and validate the chemical structures of the metabolites that comprise the biosignatures of LD; and 3) Evaluate the metabolic profile of B. burgdorferi. The overall goal at the end of fiveyears will be to have qualified LD biosignatures that can be applied in clinical laboratories usingmass spectrometry or another analytical technique suitable for a multi-analyte small molecule diagnostic. PUBLIC HEALTH RELEVANCE: The proposed research will identify novel small molecule diagnostic biomarkers for Lyme disease. The diagnostic marker profile will include both products of the human host and the pathogen, Borrelia burgdorferi. These will be used to develop biosignatures for improved Lyme disease diagnostic assays that are not based on classical serology and that can differentiate between the stages of Lyme disease and provide a test of cure.  \",\n",
      "            \"agency\": \"NIH\",\n",
      "            \"budgetEndDate\": \"2013-05-31T00:00:00\",\n",
      "            \"budgetStartDate\": \"2012-06-01T00:00:00\",\n",
      "            \"cfdaCode\": \"855\",\n",
      "            \"congressionalDistrict\": \"02\",\n",
      "            \"contactPi\": \"BELISLE, JOHN T\",\n",
      "            \"department\": \"HHS\",\n",
      "            \"dunsId\": \"785979618\",\n",
      "            \"fy\": 2012,\n",
      "            \"ic\": \"NIAID\",\n",
      "            \"latitude\": 40.548455,\n",
      "            \"longitude\": -105.06609,\n",
      "            \"nihApplId\": 8301968,\n",
      "            \"orgCity\": \"FORT COLLINS\",\n",
      "            \"orgCountry\": \"UNITED STATES\",\n",
      "            \"orgName\": \"COLORADO STATE UNIVERSITY-FORT COLLINS\",\n",
      "            \"orgState\": \"CO\",\n",
      "            \"orgZipCode\": \"805231062\",\n",
      "            \"otherPis\": null,\n",
      "            \"projectEndDate\": \"2014-05-31T00:00:00\",\n",
      "            \"projectNumber\": \"1R21AI100228-01\",\n",
      "            \"projectStartDate\": \"2012-06-01T00:00:00\",\n",
      "            \"smApplId\": 22566,\n",
      "            \"terms\": \" Amoxicillin; Antibiotics; Antigens; Bacteria; base; Biochemistry; Bioinformatics; Biological Assay; Biological Markers; biological systems; biosignature; Bite; Borrelia burgdorferi; Cancer Biology; Centers for Disease Control and Prevention (U.S.); Characteristics; Chemical Structure; Chronic; Clinical; Collaborations; Colorado; Communicable Diseases; comparative; Complex; Computer software; Data; Data Set; Deposition; Development; Diagnosis; Diagnostic; Diagnostic tests; Disease; disorder prevention; DNA; Doxycycline; Early Diagnosis; Enzyme-Linked Immunosorbent Assay; erythema migrans; Etiology; Europe; Evaluation; Exanthema; FDA approved; Goals; Growth; Human; improved; Individual; Infection; innovation; instrumentation; Laboratories; latent infection; Lead; Logistic Regressions; Lyme Disease; Malignant Neoplasms; Mass Spectrum Analysis; medical schools; Metabolic; Metabolic Diseases; metabolomics; Methods; Modeling; Molecular; Monitor; Morbidity - disease rate; New York; novel; novel diagnostics; Order Spirochaetales; pathogen; Patients; Phase; Phased Innovation Awards; Process; Qualifying; Reporting; Research; Resources; Sampling; Scientist; Serologic tests; Serum; Site; skin lesion; small molecule; Source; Specificity; Staging; Standardization; Technical Expertise; Techniques; Technology; Testing; Ticks; tool; tool development; tuberculosis treatment; United States; Universities; Urine; Vector-transmitted infectious disease; Visual; Western Blotting\",\n",
      "            \"title\": \"METABOLIC BIOMARKERS AND BIOSIGNATURES FOR IMPROVED DIAGNOSIS OF LYME DISEASE\",\n",
      "            \"totalCostAmount\": 216156\n",
      "        },\n",
      "        {\n",
      "            \"abstract\": \"DESCRIPTION (provided by applicant): Investigations on hepatitis B virus (HBV) and related animal viruses, particularly duck hepatitis B virus (DHBV), have provided a detailed model of the hepadnavirus replication cycle. However, the mechanism responsible for the conversion of the relaxed circular (RC) viral DNA genome into covalently closed circular (CCC) DNA, the template for viral RNA transcription, is not yet known. Likewise, the mechanism regulating amplification of CCC DNA in mammalian hepadnaviruses is still elusive. Research from our laboratory showed that the conversion of DHBV RC into CCC DNA occurs independently of viral enzymatic functions, and requires instead cellular DNA repair enzymes that are resistant to aphidicolin and dideoxynucleotides. Consistent with these results, we have obtained compelling evidence that the Y DNA polymerase kappa, central to nucleotide excision repair (NER), is required for DHBV CCC DNA synthesis. We propose a novel hypothesis stipulating that HBV, and DHBV, use the cellular nucleotide excision repair machinery for the conversion of RC DNA into CCC DNA. The hypothesis will be investigated through the following specific aims: Aim 1. Identification of cellular factors required for the conversion of HBV and DHBV RC into CCC DNA. The purpose of this aim is to determine if polymerase \\u00bf is also required for HBV CCC DNA synthesis and to determine if the endonucleases and DNA ligases involved in NER play a role in the conversion of DHBV, as well as HBV RC to CCC DNA. Aim 2. Identification of intermediates of minus and plus strand DNA. We propose experiments to identify DNA intermediates predicted to accumulate in cells lacking pol \\u00bf and investigate whether the repair of minus and plus strand DNA occurs by the same or different mechanisms. This proposal will fill a major gap in our understanding of hepadnavirus biology, the mechanism of CCC DNA synthesis, and reveal a completely novel strategy used by an animal virus to replicate DNA. The information gained from our research into the mechanism of CCC DNA synthesis could be exploited for the identification of novel cellular genes that might provide targets for future antiviral therapies, required for the cure of over 400 million HBV infected people. Moreover, the aims of the application represent an exploratory effort that will set the stage for future investigations into he mechanism used by HBV to recruit the NER system and on the mechanism controlling the observed species specificity of HBV CCC DNA synthesis.  PUBLIC HEALTH RELEVANCE: Over 400 million people are currently infected with hepatitis B virus and are at significantly elevated risk for the development of liver cancer. Our research intothe mechanism of CCC DNA synthesis is directed at identification of cellular genes that represent novel targets for future antiviral therapies that, in combination with existing drugs, might cure HBV infections.  \",\n",
      "            \"agency\": \"NIH\",\n",
      "            \"budgetEndDate\": \"2013-06-30T00:00:00\",\n",
      "            \"budgetStartDate\": \"2012-07-01T00:00:00\",\n",
      "            \"cfdaCode\": \"855\",\n",
      "            \"congressionalDistrict\": \"13\",\n",
      "            \"contactPi\": \"SEEGER, CHRISTOPH\",\n",
      "            \"department\": \"HHS\",\n",
      "            \"dunsId\": \"064367329\",\n",
      "            \"fy\": 2012,\n",
      "            \"ic\": \"NIAID\",\n",
      "            \"latitude\": 40.066768,\n",
      "            \"longitude\": -75.09275,\n",
      "            \"nihApplId\": 8355689,\n",
      "            \"orgCity\": \"PHILADELPHIA\",\n",
      "            \"orgCountry\": \"UNITED STATES\",\n",
      "            \"orgName\": \"INSTITUTE FOR CANCER RESEARCH\",\n",
      "            \"orgState\": \"PA\",\n",
      "            \"orgZipCode\": \"191112434\",\n",
      "            \"otherPis\": null,\n",
      "            \"projectEndDate\": \"2014-06-30T00:00:00\",\n",
      "            \"projectNumber\": \"1R21AI101558-01\",\n",
      "            \"projectStartDate\": \"2012-07-01T00:00:00\",\n",
      "            \"smApplId\": 22630,\n",
      "            \"terms\": \" Animal Viruses; Antiviral Therapy; Aphidicolin; base; Biological Assay; Biology; Cells; Circular DNA; design; Development; dideoxynucleotide; DNA; DNA biosynthesis; DNA ligase I; DNA Ligases; DNA Repair; DNA Repair Enzymes; DNA-Directed DNA Polymerase; Duck Hepatitis B Virus; endonuclease; Future; Genes; Genetic Transcription; Genome; Goals; Hepadnaviridae; Hepatitis B Virus; Human; Investigation; Laboratories; Life Cycle Stages; Ligase; Malignant neoplasm of liver; Maps; Modeling; Monitor; novel; novel strategies; Nucleotide Excision Repair; Oncogenic; pathogen; Pharmaceutical Preparations; Play; Polymerase; Primer Extension; Recruitment Activity; repaired; Research; research study; Resistance; Risk; Role; small hairpin RNA; Species Specificity; Staging; System; Viral; viral DNA; viral RNA; Virus Diseases\",\n",
      "            \"title\": \"MECHANISM OF CCC DNA SYNTHESIS IN HEPATITIS B VIRUS\",\n",
      "            \"totalCostAmount\": 223125\n",
      "        },\n",
      "        {\n",
      "            \"abstract\": \"DESCRIPTION (provided by applicant): The specific aims of this project are designed to better understand the biology and virulence of Aspergillus fumigatus, the causative agent of invasive aspergillosis (IA), the host immune response, and the role of iron in the interaction between fungus and host. IA is an understudied disease of growing significance worldwide that causes over 3,500 annual deaths in the U.S. alone. The significance of IA is growing due to substantial increases in the number of allogenic and autologous stem cell transplants, as well as advances in medical technologies that are expanding the spectrum of patients susceptible to A. fumigatus infections. For example, while transplant patients remain the patient population most at risk, immunosuppression therapies are now routinely used to treat more common diseases including, asthma, rheumatoid arthritis, and gastrointestinal disorders. With the rising number of immunocompromised patients, the significance of invasive infections caused by A. fumigatus and other invasive moulds will undoubtedly continue to increase. In order to identify new and promising therapeutic targets to treat infection with this ubiquitous fungus, researchers have incorporated a systems biology approach to understand the fungal pathways contributing to virulence. The project proposed here applies this approach to the host response, a step toward the ultimate goal of integrating host and fungal pathways. While the host response is multi-pronged, a central factor in the interplay between the host and A. fumigatus is the manipulation of iron levels in both macrophages and epithelial cells. Iron homeostasis is regulated by a control system of proteins that are connected through several intertwined feedback loops. This complicates an understanding of the mechanisms that mediate the effect of cytokine production on iron levels. The focus of the proposed project is to elucidate the network of immune proteins and key proteins involved in iron homeostasis in both alveolar macrophages and lung epithelial cells, with the goal of identifying key drivers of iron regulation during Aspergillus infection. Th central discovery tool in the project will be a dynamic mathematical model of the network to be constructed that encodes the dynamic response and the effect of feedback loops under perturbations of key proteins.  PUBLIC HEALTH RELEVANCE: This project is designed to better understand the biology and virulence of Aspergillus fumigatus, the causative agent of invasive aspergillosis (IA), the host immune response, and the role of iron in the interaction between fungus and host. It impacts our ability to identify novel therapeutic treatments for this understudied disease.  \",\n",
      "            \"agency\": \"NIH\",\n",
      "            \"budgetEndDate\": \"2013-05-31T00:00:00\",\n",
      "            \"budgetStartDate\": \"2012-06-01T00:00:00\",\n",
      "            \"cfdaCode\": \"855\",\n",
      "            \"congressionalDistrict\": \"09\",\n",
      "            \"contactPi\": \"LAUBENBACHER, REINHARD\",\n",
      "            \"department\": \"HHS\",\n",
      "            \"dunsId\": \"003137015\",\n",
      "            \"fy\": 2012,\n",
      "            \"ic\": \"NIAID\",\n",
      "            \"latitude\": 37.232792,\n",
      "            \"longitude\": -80.420657,\n",
      "            \"nihApplId\": 8359974,\n",
      "            \"orgCity\": \"BLACKSBURG\",\n",
      "            \"orgCountry\": \"UNITED STATES\",\n",
      "            \"orgName\": \"VIRGINIA POLYTECHNIC INSTITUTE AND STATE UNIVERSITY\",\n",
      "            \"orgState\": \"VA\",\n",
      "            \"orgZipCode\": \"240616100\",\n",
      "            \"otherPis\": \"LAWRENCE, CHRISTOPHER BRIAN ; \",\n",
      "            \"projectEndDate\": \"2014-05-31T00:00:00\",\n",
      "            \"projectNumber\": \"1R21AI101619-01\",\n",
      "            \"projectStartDate\": \"2012-06-01T00:00:00\",\n",
      "            \"smApplId\": 22637,\n",
      "            \"terms\": \" AIDS/HIV problem; Allergens; allergic airway disease; Allogenic; Alveolar Macrophages; Antifungal Agents; Aspergillosis; Aspergillus; Aspergillus fumigatus; Asthma; Autologous; Bioinformatics; Biology; Breast; Cell Culture System; Cell Culture Techniques; cell type; Cessation of life; chemotherapy; Clinic; Collaborations; Communicable Diseases; Complex; Computational Biology; computational network modeling; Computer Simulation; cytokine; Data; design; Disease; Enrollment; Enzyme-Linked Immunosorbent Assay; Epithelial Cells; Epithelium; Epstein-Barr Virus Infections; experience; Exploratory/Developmental Grant for Diagnostic Cancer Imaging; Feedback; fungus; Gastrointestinal Diseases; Generations; Goals; Grant; Growth; Homeostasis; Human; Immune; Immune response; Immunocompromised Host; Immunology; In Vitro; Incidence; Infection; Infectious Disease Immunology; Inflammatory Response; Institutes; interest; Iron; iron metabolism; Laboratories; Lead; Letters; Lung; macrophage; Malignant Neoplasms; Mammals; mathematical model; Mathematics; Measurement; Mediating; Medical Technology; meetings; member; Messenger RNA; Microbiology; Modeling; Molds; Montana; mortality; Mus; National Cancer Institute; National Institute of Allergy and Infectious Disease; Natural Immunity; Natural immunosuppression; novel therapeutics; Organ Transplantation; Outcome; pathogen; Pathogenesis; Pathway interactions; patient population; Patients; Production; professor; programs; Proteins; Recording of previous events; Regulation; Research; Research Personnel; research study; resistant strain; response; Reverse Transcriptase Polymerase Chain Reaction; Rheumatoid Arthritis; Risk; Role; Siderophores; Stem cell transplant; System; Systems Biology; Techniques; therapeutic target; Time; tool; Training; Transplant Recipients; Universities; Virginia; Virulence; Western Blotting; working group\",\n",
      "            \"title\": \"THE SYSTEMS BIOLOGY OF IRON HOMEOSTASIS AND THE IMMUNE RESPONSE TO ASPERGILLUS\",\n",
      "            \"totalCostAmount\": 192569\n",
      "        },\n",
      "        {\n",
      "            \"abstract\": \"DESCRIPTION (provided by applicant): The standard treatment regimen for active tuberculosis (TB) is >40 years old and has suboptimal effectiveness, because it requires daily drug adherence for ?6 months, long after symptoms have resolved. Now, for the first time in decades, the TB drug pipeline is full of new, highly potent and possibly treatment- shortening agents nearing readiness for phase 2/3 clinical trials. Unfortunately, evaluating these drugs poses major challenges, because of the large sample sizes (~2500 patients) and long follow-up periods (~30 months) required to show their non-inferiority to the standard regimen in preventing treatment failure and relapse. Although two-month mycobacterial culture conversion is currently used as a surrogate marker, it has numerous disadvantages, such as the need to collect sputum from patients who are no longer coughing, transport it to specialized reference laboratories, and wait weeks for a dichotomous result which has poor statistical power. The objective of this proposal is to develop a quantitative, blood-based, TB biomarker assay using a proteomic approach that targets human exosomes. Exosomes are vesicular nanoparticles released by host cells during active infection. Consistent with their postulated role in immune signaling, exosomes contain high concentrations of M. tuberculosis (Mtb) proteins, which seem to decrease with clearance of Mtb from infected mice. Thus, we hypothesize that exosome proteins can serve as suitable biomarkers for predicting microbiologic response to TB treatment among groups randomized to different drug regimens in clinical trials, as well as among individuals on standard regimens. A non-sputum-dependent, non-growth-based biomarker assay that could be applied early in treatment in either or both of these scenarios could transform the pace and scope of TB drug development and global TB control. This assay would also be useful for monitoring treatment of paucibacillary disease as is common in childhood TB, extrapulmonary TB, and HIV-TB co-infection. To achieve this goal, we have assembled a multidisciplinary team of TB researchers with expertise in exosome biology, proteomics, diagnostics, novel prediction methods, and clinical trials. We will build on the existing infrastructure of my K23 mentor's NIH-funded cohort study of pneumonia by collecting serial serum specimens from patients followed through the initiation phase of treatment for culture-confirmed pulmonary TB. We will use these data and samples to 1) optimize methods for processing serum exosomes for Mtb- biomarker assays; 2) evaluate the kinetics of a serologic, exosome-based Mtb-biomarker panel in response to treatment, and determine the optimal sampling frame for assessing that response. Upon completing these proof-of-principle studies, we expect to have the preliminary data to justify larger clinical studies to determine the accuracyof the marker in predicting treatment response in test and validation cohorts; to further develop and validate a clinical testing platform for these markers; and ultimately to evaluate the resultin refined assay as a surrogate marker in Phase 3 clinical trial banks currently under development.  PUBLIC HEALTH RELEVANCE: Because current tuberculosis (TB) treatments are long and hard to tolerate, many people around the world fail treatment and die. New drugs for TB patients are now in development but will take years to come to market, unless we can create tests to predict early in treatment which drugs are most effective. Such tests would speed new drug development and improve individual care by predict when treatments are likely to fail. We plan to develop such a test using human 'exosomes,' newly identified bloodstream particles which carry TB proteins.  \",\n",
      "            \"agency\": \"NIH\",\n",
      "            \"budgetEndDate\": \"2013-06-30T00:00:00\",\n",
      "            \"budgetStartDate\": \"2012-07-10T00:00:00\",\n",
      "            \"cfdaCode\": \"855\",\n",
      "            \"congressionalDistrict\": \"12\",\n",
      "            \"contactPi\": \"DAVIS, JOHN LUCIAN\",\n",
      "            \"department\": \"HHS\",\n",
      "            \"dunsId\": \"094878337\",\n",
      "            \"fy\": 2012,\n",
      "            \"ic\": \"NIAID\",\n",
      "            \"latitude\": 37.786097,\n",
      "            \"longitude\": -122.448391,\n",
      "            \"nihApplId\": 8358854,\n",
      "            \"orgCity\": \"SAN FRANCISCO\",\n",
      "            \"orgCountry\": \"UNITED STATES\",\n",
      "            \"orgName\": \"UNIVERSITY OF CALIFORNIA SAN FRANCISCO\",\n",
      "            \"orgState\": \"CA\",\n",
      "            \"orgZipCode\": \"941186215\",\n",
      "            \"otherPis\": null,\n",
      "            \"projectEndDate\": \"2014-06-30T00:00:00\",\n",
      "            \"projectNumber\": \"1R21AI101714-01\",\n",
      "            \"projectStartDate\": \"2012-07-10T00:00:00\",\n",
      "            \"smApplId\": 22644,\n",
      "            \"terms\": \" Address; Adherence; adverse outcome; Aliquot; Amplifiers; Antigens; Anti-Retroviral Agents; Area; base; Binding; Biological Assay; Biological Markers; Biology; Biometry; Blinded; Blood; Blood Circulation; Caring; Cell membrane; Cells; Child; Childhood; Clinical; Clinical Research; Clinical Trials; cohort; Cohort Studies; Collection; Complex; cost; Coughing; Data; Development; Diagnosis; Diagnostic; Disadvantaged; Disease; drug development; effective therapy; Effectiveness; Evaluation; follow-up; Funding; Goals; Gold; HIV; Human; Immune; improved; Individual; Infection; Infection Control; International; Kinetics; Laboratories; Logistics; macrophage; Marketing; mass spectrometer; mathematical model; Measurement; Measures; medication compliance; Membrane; Mentors; Methods; Metric; MicroRNAs; Monitor; mouse model; multidisciplinary; Multidrug-Resistant Tuberculosis; multiple reaction monitoring; Mus; mycobacterial; Mycobacterium tuberculosis; nanoparticle; Nested Case-Control Study; Noise; novel; novel diagnostics; novel strategies; Outcome; particle; Patients; Pharmaceutical Preparations; Pharmacotherapy; Phase; Phase III Clinical Trials; Pneumonia; point of care; prevent; Process; Prognostic Marker; Property; Proteins; Proteomics; Pulmonary Tuberculosis; Randomized; Readiness; Regimen; Relapse; Renal carcinoma; Reporting; Research; research clinical testing; Research Infrastructure; Research Personnel; Resistance to infection; response; Role; Sample Size; Sampling; Sensitivity and Specificity; Serological; Serum; Signal Transduction; Specimen; Speed; Sputum; Staging; standard care; success; Surrogate Endpoint; Surrogate Markers; Symptoms; Testing; Time; Treatment Failure; Treatment Protocols; treatment response; Tuberculosis; tuberculosis drugs; tuberculosis treatment; United States National Institutes of Health; Urine; Validation; Vesicle; Viral Load result; Work\",\n",
      "            \"title\": \"A NOVEL PROTEOMIC APPROACH TO TB BIOMARKER DISCOVERY USING HUMAN EXOSOMES\",\n",
      "            \"totalCostAmount\": 222750\n",
      "        },\n",
      "        {\n",
      "            \"abstract\": \"DESCRIPTION (provided by applicant): Professional phagocytes play a critical role in anti-microbial defense. Uptake of microbes by phagocytosis results in the formation of a dynamic vesicular compartment, termed the phagolysosome. The phagolysosome physically and functionally defines a critical separation of the microbe from the host cell, allowing the host to target degradative anti-microbial mechanisms to this confined space. We find that bacterial infection triggers the unfolded protein response (UPR), a cellular program associated with ER stress and innate immune function. Activation of the UPR increases the capacity of the cell to degrade proteins, but the mechanisms responsible for this degradation are incompletely understood. Our preliminary studies suggest that UPR activation during bacterial infection results in increased association of methicillin resistant Staphylococcus aureus (MRSA) with the degradative lysosomal compartment and bacterial killing. Inhibition of specific UPR regulators results in decreased association with lysosomes and decreased MRSA killing. The central hypothesis of this proposal is that activation of the UPR in phagocytes results in increased trafficking and degradative capacity of the phagolysosomal network, leading to enhanced degradative, and thus anti-microbial, function. To test this hypothesis, we will (1) measure mobilization of the lysosomal network using physical and functional markers upon activation of the UPR; (2) define the role of the UPR sensors, Ire1, ATF6 and PERK in regulating specific aspects of lysosomal trafficking and function in response to innate immune signals during infection. Regulation of the degradative capacity of the cell by the UPR is a fundamental strategy by which cells can respond to perturbations in the production or secretion of proteins. Our studies now highlight a novel connection between UPR-mediated degradation and anti-microbial function, and will define key druggable targets that shape the macrophage anti-microbial arsenal for development of anti-infective strategies.  PUBLIC HEALTH RELEVANCE: Virulent pathogens may evade host-antimicrobial defenses and are a major cause of morbidity and mortality worldwide. Our studies have revealed a fundamental cellular stress response that broadly enhances host defense, increasing killing of bacteria such as methicillin-resistant Staphylococcus aureus by immune cells. These findings highlight a new mechanism for anti-microbial defense that may provide potential targets for development of anti-infective therapies.   \",\n",
      "            \"agency\": \"NIH\",\n",
      "            \"budgetEndDate\": \"2013-07-31T00:00:00\",\n",
      "            \"budgetStartDate\": \"2012-08-01T00:00:00\",\n",
      "            \"cfdaCode\": \"855\",\n",
      "            \"congressionalDistrict\": \"12\",\n",
      "            \"contactPi\": \"O'RIORDAN, MARY\",\n",
      "            \"department\": \"HHS\",\n",
      "            \"dunsId\": \"073133571\",\n",
      "            \"fy\": 2012,\n",
      "            \"ic\": \"NIAID\",\n",
      "            \"latitude\": 42.244005,\n",
      "            \"longitude\": -83.73915,\n",
      "            \"nihApplId\": 8364443,\n",
      "            \"orgCity\": \"ANN ARBOR\",\n",
      "            \"orgCountry\": \"UNITED STATES\",\n",
      "            \"orgName\": \"UNIVERSITY OF MICHIGAN AT ANN ARBOR\",\n",
      "            \"orgState\": \"MI\",\n",
      "            \"orgZipCode\": \"481091276\",\n",
      "            \"otherPis\": null,\n",
      "            \"projectEndDate\": \"2014-07-31T00:00:00\",\n",
      "            \"projectNumber\": \"1R21AI101777-01\",\n",
      "            \"projectStartDate\": \"2012-08-01T00:00:00\",\n",
      "            \"smApplId\": 22651,\n",
      "            \"terms\": \" Abscess; Animal Model; Anti-Bacterial Agents; Antibiotic Therapy; Anti-Infective Agents; antimicrobial; Autophagocytosis; Bacteria; Bacterial Infections; bafilomycin A; biological adaptation to stress; Biology; Cell Culture Techniques; Cell physiology; Cells; Cellular Stress Response; Chemicals; Confined Spaces; Cutaneous; cytokine; Cytosol; Data; Defense Mechanisms; Development; Endoplasmic Reticulum; endoplasmic reticulum stress; Garbage; Heart Diseases; Homeostasis; Host Defense; Imaging Techniques; Immune; immune function; Infection; Inflammatory; Inflammatory Response; inhibitor/antagonist; innate immune function; Invaded; Killings; Lead; Lysosomes; macrophage; Malignant Neoplasms; man; Measures; Mediating; Messenger RNA; methicillin resistant Staphylococcus aureus; Microbe; Molecular; Morbidity - disease rate; mortality; novel; OmpR protein; Oxidases; pathogen; Pathway interactions; Peptide Hydrolases; Phagocytes; Phagocytosis; Phagolysosome; Play; Positioning Attribute; Process; Production; programs; protein degradation; Protein Secretion; Proteins; Receptor Signaling; Regulation; research study; response; RNA Splicing; Role; sensor; Shapes; Signal Transduction; Site; Testing; Toll-like receptors; trafficking; Transcriptional Regulation; uptake; vacuolar H+-ATPase; Virulent; Yeasts\",\n",
      "            \"title\": \"MOBILIZATION OF LYSOSOME ANTI-MICROBIAL DEFENSES BY THE UNFOLDED PROTEIN RESPONSE\",\n",
      "            \"totalCostAmount\": 227540\n",
      "        },\n",
      "        {\n",
      "            \"abstract\": \"DESCRIPTION (provided by applicant): Single cell transcriptional analysis has already demonstrated its ability to identify novel cell subsets but is currently limited by the number of cells and analytes that can be measured in parallel. We plan to increase both the number of cells and analytes that can be monitored by one to two orders of magnitude by using DNA-bar coded nanoliter well (nanowell) arrays to label each individual transcript with a DNA-encoded address. Next generation sequencing will identify both the transcript identity and the attached barcode, thereby tracing each sequenced transcript back to a single cell. This will be accomplished by transferring a million DNA barcodes synthesized on the surface of a microarray to primer-conjugated nanoparticles in the nanowells through asymmetric PCR while the nanowells are sealed by the microarray surface. The PCR reaction will also add a poly(dT) tail to each barcode. Single cells will then be sealed into the bar coded nanowells. Following lysis of the cells, the poly(dT) probe will capture the mRNA and reverse transcriptase will extend the poly(dT) sequence, thereby fusing the barcode with each transcript. The bar coded cDNA will be amplified and integrated into next generation sequencing workflows. The technique will be validated by comparing the measured transcript levels to the levels measured in the same cell population by single cell qPCR using the Fluidigm platform. We will also demonstrate that the barcodes identify individual cell transcripts by sequencing B and T cell receptor transcripts and demonstrating that each unique BCR or TCR transcript has a unique barcode fused to it and the barcode maps back to a well that originally contained the correct cell type. Furthermore, the single cell transcript data will be integrated with single cell secretion data from the same cells obtained prior to cell lysis through our previously described microengraving methodology, thereby establishing the first platform that can create highly multiplexed single cell transcript ad proteomic data from the same population of single cells. Application of this technology will greatly accelerate our understanding of single cell biology and heterogeneous cell populations.  PUBLIC HEALTH RELEVANCE: Many important medical processes involve the interaction of a large number of distinct cell types that act together to produce an observed biological state. Traditional research approaches that average measurements over the entire population cannot determine the cell-to- cell interactions that are occurring between unique cell subsets within the population, which is critical for understanding the system as a whole and misses an opportunity for focusing drug discovery efforts on very specific cell types that are critical for the process. ur proposed method for highly parallel single cell transcriptome analysis will open the door to defining and understanding the biology of rare subsets of cells in heterogenous populations and thereby accelerate our understanding in fields as far ranging as immunology, cancer, stem cells and neurobiology.   \",\n",
      "            \"agency\": \"NIH\",\n",
      "            \"budgetEndDate\": \"2013-08-31T00:00:00\",\n",
      "            \"budgetStartDate\": \"2012-09-01T00:00:00\",\n",
      "            \"cfdaCode\": \"310\",\n",
      "            \"congressionalDistrict\": \"07\",\n",
      "            \"contactPi\": \"LOVE, JOHN CHRISTOPHER\",\n",
      "            \"department\": \"HHS\",\n",
      "            \"dunsId\": \"001425594\",\n",
      "            \"fy\": 2012,\n",
      "            \"ic\": \"NIAID\",\n",
      "            \"latitude\": 42.359888,\n",
      "            \"longitude\": -71.094312,\n",
      "            \"nihApplId\": 8413936,\n",
      "            \"orgCity\": \"CAMBRIDGE\",\n",
      "            \"orgCountry\": \"UNITED STATES\",\n",
      "            \"orgName\": \"MASSACHUSETTS INSTITUTE OF TECHNOLOGY\",\n",
      "            \"orgState\": \"MA\",\n",
      "            \"orgZipCode\": \"021421029\",\n",
      "            \"otherPis\": null,\n",
      "            \"projectEndDate\": \"2014-08-31T00:00:00\",\n",
      "            \"projectNumber\": \"1R21AI106025-01\",\n",
      "            \"projectStartDate\": \"2012-09-01T00:00:00\",\n",
      "            \"smApplId\": 22698,\n",
      "            \"terms\": \" Address; Back; Bar Codes; Biological; Biology; B-Lymphocytes; Cancer stem cell; cDNA Arrays; Cell Communication; Cell Count; Cell secretion; cell type; Cells; Cellular biology; Complementary DNA; Custom; Cytolysis; Data; data integration; Data Set; design; DNA; drug discovery; Gene Expression Profile; Genetic Transcription; Immunology; Individual; Label; Magnetism; Maps; Measurement; Measures; Medical; Messenger RNA; Methodology; Methods; Monitor; nanolitre; nanoparticle; Neurobiology; next generation; novel; Oligonucleotide Microarrays; Oligonucleotides; particle; Phenotype; Poly T; Population; Procedures; Process; Proteomics; Protocols documentation; Reaction; receptor; Research; Reverse Transcription; RNA-Directed DNA Polymerase; seal; single cell analysis; Solutions; Surface; System; Tail; T-Cell Receptor; Techniques; Technology; tool; Transcript; Work\",\n",
      "            \"title\": \"HIGHLY MULTIPLEXED SINGLE-CELL TRANSCRIPT ANALYSIS USING DNA-BARCODED NANOWELLS\",\n",
      "            \"totalCostAmount\": 183685\n",
      "        },\n",
      "        {\n",
      "            \"abstract\": \"DESCRIPTION (provided by applicant): Melanin in the skin protects the epidermis and the underlying dermis from the deleterious effects of ultraviolet radiation in the sun light. Therefore melanocyte homeostasis is essential for a functional epidermal melanin unit and healthy skin. Disruption of melanocyte homeostasis leads to skin disorders. Most notably, loss due to death of melanocyte causes depigmentary disorder vitiligo and accompanying skin photosensitivity, and gain of melanocytes due to unregulated proliferation as occurs in cutaneous melanoma could eventually lead to death. Exposure of intracellular proteins following death of normal and/or malignant melanocytes leads to break in tolerance and immune response to several melanosomal self antigens in vitiligo and melanoma. Therefore, understanding the mechanisms that regulate melanocyte homeostasis, specifically melanocyte death, is important for developing strategies to prevent and treat melanocytic disorders. While melanocyte death due to necrosis and apoptosis has been well investigated, mechanisms of autophagic death of melanocytes and the role of autophagy in the behavior of malignant melanocytes are not well understood. For example, it is not known whether a) susceptibility to develop cutaneous melanoma is related to ability to activate or inhibit autophagy upon oncogenic transformation of melanocytes, b) differences exist in activation of autophagy and autophagic cell death between melanocytes in darker and lighter skin individuals, and c) differences in activation of autophagy can account for susceptibility of resistance of melanocytes to malignant transformation. Autophagy, which normally serves as cell survival mechanism when cells are under stress, is regulated by mammalian Target of Rapamycin (mTOR) signaling pathway that responds to nutrient deprivation and other cell survival signals. Inhibition of mTOR signaling by rapamycin induces autophagy, which can be monitored by morphological (light microscopic immunofluorescence and electron microscopy) and biochemical methods. In our preliminary studies, we found that melanocytes from European and African ancestry show differential sensitivity to mTOR and autophagic cell death can be induced in malignant melanocytes by hyperactivation of oncogene BRAF. We hypothesized that autophagy plays a role in homeostasis of epidermal melanin unit. Here, we propose to address the following questions: Is autophagic cell death a normal homeostatic mechanism that maintains epidermal melanin unit? What role do keratinocytes play in sensitivity of melanocytes to autophagy? What is the role of Atg5 and Atg7 in sensitivity of melanocytes to mTOR inhibition? Experiments proposed here are expected to generate new scientific knowledge that could lead to better understanding of the pathogenesis of melanocytic disorders and may eventually lead to more effective treatments.  PUBLIC HEALTH RELEVANCE: Pigment producing cells in the skin are maintained in equilibrium with respect to the surrounding keratinocytes to which they export melanin pigment that provides protection from damaging ultraviolet radiation in the sun light. However, if melanocytes undergo unscheduled cell death or fail to die when damaged, it can lead to skin diseases such as vitiligo and skin cancer melanoma. The aim of this proposal to investigate melanocyte cell death mechanism that scontribute to melanocytic disorders.  \",\n",
      "            \"agency\": \"NIH\",\n",
      "            \"budgetEndDate\": \"2013-06-30T00:00:00\",\n",
      "            \"budgetStartDate\": \"2012-07-01T00:00:00\",\n",
      "            \"cfdaCode\": \"846\",\n",
      "            \"congressionalDistrict\": \"02\",\n",
      "            \"contactPi\": \"SETALURI, VIJAYASARADHI\",\n",
      "            \"department\": \"HHS\",\n",
      "            \"dunsId\": \"161202122\",\n",
      "            \"fy\": 2012,\n",
      "            \"ic\": \"NIAMS\",\n",
      "            \"latitude\": 43.060421,\n",
      "            \"longitude\": -89.397322,\n",
      "            \"nihApplId\": 8398642,\n",
      "            \"orgCity\": \"MADISON\",\n",
      "            \"orgCountry\": \"UNITED STATES\",\n",
      "            \"orgName\": \"UNIVERSITY OF WISCONSIN MADISON\",\n",
      "            \"orgState\": \"WI\",\n",
      "            \"orgZipCode\": \"537151218\",\n",
      "            \"otherPis\": \"BRILLIANT, MURRAY H ; \",\n",
      "            \"projectEndDate\": \"2014-06-30T00:00:00\",\n",
      "            \"projectNumber\": \"1R21AR062734-01A1\",\n",
      "            \"projectStartDate\": \"2012-07-01T00:00:00\",\n",
      "            \"smApplId\": 22736,\n",
      "            \"terms\": \" 1-Phosphatidylinositol 3-Kinase; Accounting; Acute; Address; African; Apoptosis; Apoptotic; Asians; Attention; Autoantigens; Autophagocytosis; base; Behavior; Biochemical; Biology; BRAF gene; Caucasians; Caucasoid Race; Cell Death; Cell surface; Cell Survival; cell type; Cells; Cessation of life; Complex; Cutaneous Melanoma; Data; Defect; deprivation; Dermatologic; Dermis; Development; Disease; effective therapy; Electron Microscopy; Embryo; Epidermis; Equilibrium; European; Fluorescence; Functional disorder; Homeostasis; Housekeeping; Human; Immune response; Immunofluorescence Microscopy; in vivo; Individual; inhibition of autophagy; inhibitor/antagonist; interest; keratinocyte; Knowledge; Lead; Light; Lysosomes; Malignant - descriptor; Malignant Neoplasms; Melanins; melanocyte; Melanocytic nevus; Melanogenesis; melanoma; Methods; Microscopic; Monitor; mTOR protein; Necrosis; Neurodegenerative Disorders; Nuclear Envelope; Nutrient; Oncogenes; Oncogenic; Organelles; Pathogenesis; Pathway interactions; Phosphorylation; Photosensitivity; Pigmentation physiologic function; Pigments; Play; Predisposition; prevent; programs; protein aggregate; Proteins; Reporting; research study; Resistance; response; Role; senescence; Signal Pathway; Signal Transduction; Sirolimus; Skin; Skin Cancer; skin disorder; Skin Pigmentation; Starvation; Stress; Tamoxifen; Testing; The Sun; Tissues; tumor growth; Ultraviolet Rays; Vitiligo; Xenopus\",\n",
      "            \"title\": \"AUTOPHAGY IN EPIDERMAL MELANOCYTE: A PROTECTIVE OR A DESTRUCTIVE ROLE?\",\n",
      "            \"totalCostAmount\": 209651\n",
      "        },\n",
      "        {\n",
      "            \"abstract\": \"DESCRIPTION (provided by applicant): MAGE proteins are normally expressed only in developing germ cells but are aberrantly expressed in some nevi, most melanomas, and many malignancies. Their selective expression makes them nearly tumor specific targets but the dominant paradigm regards MAGE expression as a functionally irrelevant byproduct of cellular transformation. This project challenges this paradigm by proposing that MAGE regulate cell growth and contribute to tumor development by regulating KAP1, a scaffolding protein that in turn mediates gene regulation by the large family of KRAB domain containing zinc finger transcription factors (KZNFs). The long term goals are: 1. To show how MAGE proteins contribute to cell growth and survival by functioning as master regulators of KZNFs, and 2. To develop novel therapies for malignancies based on interfering with MAGE expression or function. The Specific Aims are: Aim 1: To identify and manipulate MAGE effects on gene regulation in human melanocytes. To determine MAGE effects in melanocytes, we will correlate MAGE expression with genome wide KAP1 binding, chromatin compaction, and gene repression using ChIP-seq and RNA-seq of normal human melanocytes, with and without ectopic MAGE expression. Controls will use low passage human melanoma cell lines, with and without shRNA mediated MAGE knockdown, and chromatin from melanocytic nevi removed for cosmetic purposes. Analysis will include comprehensive gene annotation, clustering, and network analysis for multi-parametric identification of global effects of MAGE on cellular processes and critical downstream components. Validation will test the role of downstream network components with assays for proliferation, survival and cell cycle regulation, by selective inhibition of pathway components with RNAi. To begin to develop and validate small molecules that inhibit MAGE effects, MAGE knockdown effects on KAP1 binding and gene expression profiles will be compared to the effects of small molecule MAGE inhibitors. Aim 2: To determine mechanisms for differential MAGE effects on different genes. We have discovered that MAGE proteins can enhance or decrease KZNF and KAP1 mediated repression of specific genes. To determine mechanisms for differential MAGE effects on gene regulation, we will test the hypothesis that differences in the composition of KRAB domains determines whether MAGE expression increases, decreases, or has no effect on KZNF mediated gene repression. These experiments will test MAGE effects with a comprehensive series of KRAB domains for: a. KAP1- KRAB binding in a mammalian two hybrid assay, b. KAP1 mediated gene repression in a luciferase reporter gene assay, and c. KAP1 mediated KZNF ubiquitination by immunoblotting.  PUBLIC HEALTH RELEVANCE: MAGE expression has been correlated with aggressive clinical course, the acquisition of resistance to chemotherapy and poor clinical outcome in many types of tumors. This project will provide novel insights into mechanisms and patterns of MAGE effects that underline transition from normal melanocyte to aggressive melanoma. Developing therapies based on MAGE functions will save lives and have a huge economical impact.  \",\n",
      "            \"agency\": \"NIH\",\n",
      "            \"budgetEndDate\": \"2013-06-30T00:00:00\",\n",
      "            \"budgetStartDate\": \"2012-07-01T00:00:00\",\n",
      "            \"cfdaCode\": \"846\",\n",
      "            \"congressionalDistrict\": \"02\",\n",
      "            \"contactPi\": \"LONGLEY, B JACK\",\n",
      "            \"department\": \"HHS\",\n",
      "            \"dunsId\": \"161202122\",\n",
      "            \"fy\": 2012,\n",
      "            \"ic\": \"NIAMS\",\n",
      "            \"latitude\": 43.060421,\n",
      "            \"longitude\": -89.397322,\n",
      "            \"nihApplId\": 8385968,\n",
      "            \"orgCity\": \"MADISON\",\n",
      "            \"orgCountry\": \"UNITED STATES\",\n",
      "            \"orgName\": \"UNIVERSITY OF WISCONSIN MADISON\",\n",
      "            \"orgState\": \"WI\",\n",
      "            \"orgZipCode\": \"537151218\",\n",
      "            \"otherPis\": null,\n",
      "            \"projectEndDate\": \"2014-06-30T00:00:00\",\n",
      "            \"projectNumber\": \"1R21AR062773-01A1\",\n",
      "            \"projectStartDate\": \"2012-07-01T00:00:00\",\n",
      "            \"smApplId\": 22742,\n",
      "            \"terms\": \" Affect; Aggressive Clinical Course; Animal Care and Use Committees; Apoptosis; base; Binding; Biological Assay; Biology; Boxing; Cell Cycle Regulation; cell growth; Cell Line; Cell physiology; Cell Survival; Characteristics; Chemistry; chemotherapy; ChIP-seq; Chromatin; Chromatin Structure; Clinical; Cosmetics; Critical Pathways; Cutaneous; Development; DNA Sequence; Drug Delivery Systems; Family; Funding; Future; Gene Activation; Gene Expression; Gene Expression Regulation; gene repression; Gene Targeting; Genes; genome-wide; Germ Cells; Goals; Growth; Human; human ZNF45 protein; Immunoblotting; In Vitro; in vivo; Individual; inhibitor/antagonist; insight; Luciferases; Malignant Neoplasms; Mediating; melanocyte; Melanocytic nevus; melanoma; Melanoma Cell; melanoma-associated antigen; metaplastic cell transformation; Molecular Bank; Molecular Profiling; Mus; Neoplastic Cell Transformation; Nevus; novel; novel strategies; Outcome; Pathway Analysis; Pathway interactions; Patients; Pattern; Pharmaceutical Chemistry; Production; Protein Binding; Proteins; Protocols documentation; Reporter Genes; Repression; research study; Resistance; Resources; RNA; RNA Interference; Role; Scaffolding Protein; Screening procedure; selective expression; Series; Signal Pathway; skills; small hairpin RNA; small molecule; Testing; therapy development; transcription factor; tumor; Tumor Suppressor Proteins; Two-Hybrid System Techniques; ubiquitin ligase; Ubiquitination; United States National Institutes of Health; Validation; Work; Zinc Fingers\",\n",
      "            \"title\": \"THE CUTANEOUS BIOLOGY OF MAGE TRANSCRIPTION FACTORS\",\n",
      "            \"totalCostAmount\": 203175\n",
      "        },\n",
      "        {\n",
      "            \"abstract\": \"DESCRIPTION (provided by applicant): UV radiation has recently been classified as a known human carcinogen by the US Department of Health and Human Services. Nevertheless, the voluntary exposure to sunlight continues unabated despite progressively increasing rates of ultraviolet radiation (UVR)-induced illness and death, particularly skin cancer. An increasingly common form of UVR administration is through the use of indoor tanning salons. Almost 30 million Americans, including 20% of 18-39 year olds, visit indoor tanning salons each year. Frequent and excessive tanning, despite a growing understanding by those who tan of the morbidity and mortality associated with tanning, suggests that UVR may impart rewarding effects beyond the assumed cosmetic benefits. Recent studies, in fact, have shown that up to 40% of both frequent beach and salon tanners exhibit signs and symptoms consistent with an addictive disorder, including an inability to decrease their tanning frequencies, compulsive tanning, and/or continued tanning despite adverse consequences. To empirically measure putative centrally rewarding properties of UVR, we assessed the effects of a commercially available tanning bed upon regional cerebral blood flow (rCBF), a measure of brain activity, using single-photon emission computed tomography (SPECT). Compulsive salon bed tanners were placed under a UVA/UVB tanning light during two sessions; one session with active UVR and the other with filtered UVR (sham UVR). During the active UVR session, relative to sham UVR session, subjects demonstrated both a significant decrease in Desire to Tan and a relative increase in rCBF of the dorsal striatum (including caudate and putamen), anterior insula, and medial orbitofrontal cortex. These preliminary findings suggest that UVR may have centrally rewarding properties that encourage excessive tanning. As the mesostriatal dopaminergic pathway plays a key role in reward and addiction, we propose to extend these novel findings by using 123I-IBZM to determine if 1) dopamine is released during UVR (as is observed in response to other rewards), 2) basal D2/D3 receptors are lower in compulsive tanners relative to infrequent tanners (a finding commonly observed in other addictive disorders), and 3) whether dopamine efflux in response to UVR, relative to sham UVR, differs between compulsive and infrequent tanners. If our hypotheses are proven correct, these findings will support a dermatologic-mesostriatal reward pathway, confirm a neurobiological, reward-based mechanism for compulsive tanning behaviors, demonstrate underlying biological characteristics similar to other addictive disorders (supporting an 'addiction' diagnosis for compulsive tanners), offer a framework for future studies to examine dermatologic-mesostriatal pathways, and provide a rationale for the development of pharmacological treatments for compulsive tanning. Data obtained by the R21 mechanism will also inform prospective studies regarding the power necessary to demonstrate statistical significance between hypothesized comparisons not reaching significance in this preliminary study.  PUBLIC HEALTH RELEVANCE: Exposure to tanning beds continues unabated despite progressively increasing rates of ultraviolet radiation- induced illness and death, particularly skn cancer. Frequent and excessive tanning, in spite of a growing understanding by the general population (as well as by those who tan) of its associated morbidity and mortality, suggests that the ultraviolet radiation from sunlight and tanning beds may impart rewarding effects beyond the assumed cosmetic benefits. An understanding of the central nervous system reward mechanisms involved in excessive tanning will provide targets for treatment interventions to decrease excessive tanning.   \",\n",
      "            \"agency\": \"NIH\",\n",
      "            \"budgetEndDate\": \"2013-07-31T00:00:00\",\n",
      "            \"budgetStartDate\": \"2012-08-01T00:00:00\",\n",
      "            \"cfdaCode\": \"846\",\n",
      "            \"congressionalDistrict\": \"30\",\n",
      "            \"contactPi\": \"ADINOFF, BRYON H.\",\n",
      "            \"department\": \"HHS\",\n",
      "            \"dunsId\": \"800771545\",\n",
      "            \"fy\": 2012,\n",
      "            \"ic\": \"NIAMS\",\n",
      "            \"latitude\": 32.812451,\n",
      "            \"longitude\": -96.838171,\n",
      "            \"nihApplId\": 8285568,\n",
      "            \"orgCity\": \"DALLAS\",\n",
      "            \"orgCountry\": \"UNITED STATES\",\n",
      "            \"orgName\": \"UNIVERSITY OF TEXAS SW MED CTR\",\n",
      "            \"orgState\": \"TX\",\n",
      "            \"orgZipCode\": \"753909105\",\n",
      "            \"otherPis\": null,\n",
      "            \"projectEndDate\": \"2014-07-31T00:00:00\",\n",
      "            \"projectNumber\": \"1R21AR063018-01\",\n",
      "            \"projectStartDate\": \"2012-08-01T00:00:00\",\n",
      "            \"smApplId\": 22750,\n",
      "            \"terms\": \" 123I-IBZM; addiction; adverse outcome; age group; Alcohols; American; Anterior; base; Beds; Behavior; Binding; Biologic Characteristic; Biological; Brain; Brain region; Carcinogens; Cerebrovascular Circulation; Cessation of life; Clinical; Cocaine; Corpus striatum structure; Cosmetics; craving; Data; Dermatologic; Development; Diagnosis; Dopamine; Dorsal; Emission-Computed Tomography; Environment; Exhibits; experience; Exploratory/Developmental Grant; Exposure to; Frequencies; Future; Gender; General Population; Genes; Genetic Polymorphism; Goals; Heating; Human; Incidence; insight; Insula of Reil; Intervention; Life; Light; Malignant Neoplasms; Measures; Medial; melanoma; Mental disorders; Monitor; Morbidity - disease rate; mortality; Neuraxis; Neurobiology; Nicotine; novel; Obesity; Opioid; Pathway interactions; Pharmacological Treatment; Photons; Play; Population; programs; Property; Prospective Studies; putamen; Radiation-Induced Change; receptor; receptor density; Recording of previous events; relating to nervous system; Relative (related person); Reporting; response; Rewards; Sample Size; Severities; sex; Signs and Symptoms; Skin; Skin Cancer; Skin Carcinoma; Skin tanning; Sun Exposure; Sunburn; Sunlight; Synaptic Receptors; System; Testing; Tomography, Emission-Computed, Single-Photon; Ultraviolet B Radiation; Ultraviolet Rays; United Nations; United States Dept. of Health and Human Services; UV Radiation Exposure; Visit; World Health Organization\",\n",
      "            \"title\": \"STRIATAL DOPAMINE RELEASE IN RESPONSE TO ULTRAVIOLET LIGHT IN COMPULSIVE TANNERS\",\n",
      "            \"totalCostAmount\": 214594\n",
      "        },\n",
      "        {\n",
      "            \"abstract\": \"DESCRIPTION (provided by applicant): Approximately 350,000 anterior cruciate ligament (ACL) reconstruction surgeries are performed each year, and billions of dollars are spent on the acute care costs involved in this procedure. The most common ACL replacement strategies involve grafts (allografts from cadavers or autografts of the patients' own patellar or hamstring tendons), although newer, more experimental treatments use engineered tissues that typically involve seeding cells on a polymeric scaffold. In these procedures, the graft or engineered tissue never fully integrates with the bone tunnel created during surgery and the joint never recovers pre-surgical level biomechanics. Not surprisingly, more than half of the patients who have undergone this procedure develop early-onset osteoarthritis (OA). With acute ACL injuries becoming increasingly prevalent in children, the number of young adults developing OA is increasing each year. The Larkin and Arruda laboratories have spent the last several years engineering an ACL replacement construct generated from bone marrow-derived stromal cells (BMSCs) that exhibits the structural and functional interface characteristics of native ACL when transplanted in vivo. The construct is composed of an engineered ligament with engineered bone at each end; it fully integrates into the recipient bone and differentiates in vivo to form a mechanically appropriate and biologically matched interface between the two tissues. Work from these laboratories demonstrates many advantages to this strategy and evidence in a large animal model supports its superiority for replacement therapy. One critical concern regarding the use of this novel and innovative therapy in humans, however, is the safety of use of stem/precursor cell-derived tissue in patients. It is imperative to demonstrate that the BMSCs used for the generation of these constructs pose no long-term threat after transplantation. The main concern in the field is the ability to ascertain that no undifferentiated cells persist in thetransplanted construct that might later lead to aberrant cellular behavior such as cancer. In collaboration with Dr. Wellik's laboratory, a recent examination of the fate of the BMSC-derived BLB construct led to the surprising discovery that within several months after implantation of the BLB, donor construct cells are replaced entirely by recipient cells and donor-derived cells are no longer present. Thus, by transplanting a developmentally immature, exogenous live tissue template for replacement, adult recipient cells are induced to fully regenerate a viable and mechanically appropriate replacement ligament! This proposal seeks to confirm these preliminary results and initiate exploration into the mechanisms of this remarkable regenerative process, with the longer-term goal of translating this innovative therapy into humans and vastly improving the outcomes of acute ACL and potentially other joint and connective tissue injuries.  PUBLIC HEALTH RELEVANCE: Billions of dollars are spent each year on anterior cruciate ligament (ACL) reconstructions after acute knee injury and more than half of patients who undergo this procedure develop early- onset osteoarthritis. New technology developed by this research team uses bone marrow stromal cell-derived, living bone-ligament-bone templates to replace the injured ACL. Transplanted tissue does not survive long-term in the recipient (patient), but induces the regeneration of a new ligament made entirely of the patient's own cells! The regenerating ACL integrates into bone and forms a viable and mechanically appropriate replacement ligament. If these preliminary results are confirmed, we will be one large step closer to translating these results into a vastly improved human therapy. Further, understanding the mechanisms by which this regeneration occurs will open new avenues in the regeneration of musculoskeletal tissue.  \",\n",
      "            \"agency\": \"NIH\",\n",
      "            \"budgetEndDate\": \"2013-06-30T00:00:00\",\n",
      "            \"budgetStartDate\": \"2012-07-01T00:00:00\",\n",
      "            \"cfdaCode\": \"846\",\n",
      "            \"congressionalDistrict\": \"12\",\n",
      "            \"contactPi\": \"WELLIK, DENEEN M\",\n",
      "            \"department\": \"HHS\",\n",
      "            \"dunsId\": \"073133571\",\n",
      "            \"fy\": 2012,\n",
      "            \"ic\": \"NIAMS\",\n",
      "            \"latitude\": 42.244005,\n",
      "            \"longitude\": -83.73915,\n",
      "            \"nihApplId\": 8383131,\n",
      "            \"orgCity\": \"ANN ARBOR\",\n",
      "            \"orgCountry\": \"UNITED STATES\",\n",
      "            \"orgName\": \"UNIVERSITY OF MICHIGAN AT ANN ARBOR\",\n",
      "            \"orgState\": \"MI\",\n",
      "            \"orgZipCode\": \"481091276\",\n",
      "            \"otherPis\": null,\n",
      "            \"projectEndDate\": \"2014-06-30T00:00:00\",\n",
      "            \"projectNumber\": \"1R21AR063444-01\",\n",
      "            \"projectStartDate\": \"2012-07-01T00:00:00\",\n",
      "            \"smApplId\": 22759,\n",
      "            \"terms\": \" Acute; Address; Adult; Allogenic; Allografting; Animal Model; Anterior Cruciate Ligament; anterior cruciate ligament reconstruction; Autologous Transplantation; base; Behavior; Biology; Biomechanics; bone; bone engineering; Bone Marrow; Bone Tissue; Cadaver; Caring; cell motility; Cells; Characteristics; Child; Childhood; Chromosome Painting; Collaborations; Connective Tissue; cost; Data; Degenerative polyarthritis; Development; early onset; Embryonic Development; Engineering; Ensure; Exhibits; Female; Generations; Goals; Growth Factor; hamstring; Healthcare; high risk; Human; Implant; implantation; improved; in vivo; in vivo regeneration; injured; Injury; innovation; Innovative Therapy; Joints; Knee Injuries; Laboratories; Lead; Life; ligament injury; Ligaments; male; Malignant Neoplasms; Mammals; Maps; Mesenchymal; Molecular; molecular marker; Musculoskeletal; musculoskeletal injury; Natural regeneration; new technology; novel; Operative Surgical Procedures; Organ Transplantation; Outcome; Pathway interactions; Patients; Population; precursor cell; Procedures; Process; Quality of life; regenerative; regenerative therapy; repaired; Replacement Therapy; Research; response; Risk; Safety; scaffold; Sheep; Signal Transduction; spatiotemporal; stem; Stem cells; Stromal Cells; success; Technology; Tendon structure; Time; Tissue Differentiation; Tissue Engineering; Tissues; Translating; Transplantation; Transplanted tissue; Treatment Efficacy; Undifferentiated; Work; Y Chromosome; young adult\",\n",
      "            \"title\": \"USING BMSC-DERIVED BONE-LIGAMENT-BONE TISSUE AS A LIVE TEMPLATE FOR ACL REGENERAT\",\n",
      "            \"totalCostAmount\": 209925\n",
      "        },\n",
      "        {\n",
      "            \"abstract\": \"DESCRIPTION (provided by applicant): The goal of Imaging in 2020 is to foster the development of enduring associations between numerous disciplines that will lead to innovative solutions and applications of imaging technologies. The proven 'secret ingredient' for the success of previous I2020 meetings has been the diverse participants and location. These key components have successfully provided a unique opportunity for education between disciplines and lasting collaborations. This year (as for all past meetings) Imaging in 2020 VIII will be held at the Jackson Lake Lodge in Jackson Hole, Wyoming. This location in Grand Teton National Forest is easily accessible from Jackson Airport. Breathtaking views of the Grand Tetons provide a colorful backdrop and this environment has minimal distractions providing members the opportunity to commune with each other.  PUBLIC HEALTH RELEVANCE: The theme of Imaging in 2020 for 2012, will be to explore the use of molecular imaging to understand complex biology. New links between molecular imaging and systems biology will be developed. A fundamental understanding of the biological systems in both normal and diseased physiology will play a critical role in the creation, monitoring and control of new highly potent interventions. With new therapeutic technologies such as RNA interference, nanotherapeutics and stem cells entering the clinic, a deeper understanding of underlying systems biology (that can only be attained through molecular imaging technology) is required. The meeting will address imaging of complexity in single cells, system pharmacology, cancer stem cells, immunological systems, animal and theoretical models to assess biological systems and the use of imaging in computational systems analysis. This year will expand the diversity of the scientists that gather in Jackson Hole to include systems biologists and mathematicians that will benefit from applying imaging technologies to their specific disciplines and, importantly, introduce new technologies and challenges for molecular imaging scientists. The focuses of the 2014 and 2016 meetings will be decided in June 2013 and June 2015 respectively. The chosen topics will be germane to the most up-to-date research in the field. The general themes will be the influence of molecular imaging in the areas of biology and medicine.  \",\n",
      "            \"agency\": \"NIH\",\n",
      "            \"budgetEndDate\": \"2014-08-31T00:00:00\",\n",
      "            \"budgetStartDate\": \"2012-09-01T00:00:00\",\n",
      "            \"cfdaCode\": \"394\",\n",
      "            \"congressionalDistrict\": \"09\",\n",
      "            \"contactPi\": \"MEADE, THOMAS J\",\n",
      "            \"department\": \"HHS\",\n",
      "            \"dunsId\": \"160079455\",\n",
      "            \"fy\": 2012,\n",
      "            \"ic\": \"NCI\",\n",
      "            \"latitude\": 42.05122,\n",
      "            \"longitude\": -87.684661,\n",
      "            \"nihApplId\": 8400285,\n",
      "            \"orgCity\": \"EVANSTON\",\n",
      "            \"orgCountry\": \"UNITED STATES\",\n",
      "            \"orgName\": \"NORTHWESTERN UNIVERSITY\",\n",
      "            \"orgState\": \"IL\",\n",
      "            \"orgZipCode\": \"602013149\",\n",
      "            \"otherPis\": null,\n",
      "            \"projectEndDate\": \"2018-08-31T00:00:00\",\n",
      "            \"projectNumber\": \"2R13CA162890-02\",\n",
      "            \"projectStartDate\": \"2011-09-01T00:00:00\",\n",
      "            \"smApplId\": 22788,\n",
      "            \"terms\": \" Address; Animal Model; Area; biological systems; Biology; Cancer Diagnostics; Cancer stem cell; Cells; cellular imaging; Clinic; Collaborations; commune; Communication; Complex; Development; Discipline; distraction; Education; Engineering; Environment; Fertilization; flexibility; forest; Fostering; Foundations; Goals; Image; Imaging technology; Individual; innovation; Institution; Intervention; Lead; Link; Location; Medicine; meetings; member; Molecular; molecular imaging; Monitor; nanotherapeutic; Neurosciences; new technology; novel therapeutics; Outcome; Participant; Pharmacology; Physiological; Physiology; Play; Research; Research Personnel; RNA Interference; Role; Scientist; Societies; Solutions; Stem cells; Structure; success; symposium; System; Systems Analysis; Systems Biology; Technology; Theoretical model; Wyoming\",\n",
      "            \"title\": \"IMAGING IN 2020-VIII: INTEGRATING MOLECULAR IMAGING AND COMPLEX BIOLOGY\",\n",
      "            \"totalCostAmount\": 52450\n",
      "        },\n",
      "        {\n",
      "            \"abstract\": \"DESCRIPTION (provided by applicant): The discovery of the gastric pathogen Helicobacter pylori nearly 30 years ago has drastically altered the treatment of peptic ulcer disease. By means of a horizontally acquired pathogenicity island, the cag PAI, H. pylori translocates an effector protein into host gastric mucosal cells mediating severe inflammation and carcinogenesis. Although the cag PAI is a critical determinant of virulence, not all persons infected with H. pylor develop severe disease. This is in part due to the existence of H. pylori strains that do not possess this pathogenicity island. Our lab has found genetic loci present in these cag PAI negative isolates that are frequently absent or highly mutated in the more virulent, cag PAI positive H. pylori isolates. The specific aims of this project are to characterize a gene found frequently among the H. pylori strains of lesser virulence, but infrequently among those associated with more severe disease. We hypothesize this gene, p7, is variably expressed in those cag PAI negative strains that possess it. We also aim to examine more cag PAI negative isolates for a bacteriophage found frequently as a prophage in these strains, but found very rarely among the more virulent, cag PAI positive isolates. Lastly, we propose to determine the complete genome sequence of four cag PAI negative H. pylori isolates from patients with no known disease associated with H. pylori. This effort will take advantage of next generation sequencing technology and be accomplished as part of a freshman biology laboratory class.  PUBLIC HEALTH RELEVANCE: While some people infected with the stomach pathogenic bacterium, Helicobacter pylori develop gastric or duodenal ulcers or even gastric cancer, most of those infected with this pathogen remain asymptomatic for their lives. While many strains of this bacterium have a section of the genome dedicated to secreting a bacterial protein into stomach cells to induce inflammation and cancer, some strains of this bacterium do not. To examine the evolution of virulence in this significant human pathogen, more research is needed on the genomes of H. pylori strains that produce disease less frequently  \",\n",
      "            \"agency\": \"NIH\",\n",
      "            \"budgetEndDate\": \"2015-06-30T00:00:00\",\n",
      "            \"budgetStartDate\": \"2012-07-01T00:00:00\",\n",
      "            \"cfdaCode\": \"855\",\n",
      "            \"congressionalDistrict\": \"01\",\n",
      "            \"contactPi\": \"FORSYTH, MARK H\",\n",
      "            \"department\": \"HHS\",\n",
      "            \"dunsId\": \"074762238\",\n",
      "            \"fy\": 2012,\n",
      "            \"ic\": \"NIAID\",\n",
      "            \"latitude\": 37.267538,\n",
      "            \"longitude\": -76.720837,\n",
      "            \"nihApplId\": 8366981,\n",
      "            \"orgCity\": \"WILLIAMSBURG\",\n",
      "            \"orgCountry\": \"UNITED STATES\",\n",
      "            \"orgName\": \"COLLEGE OF WILLIAM AND MARY\",\n",
      "            \"orgState\": \"VA\",\n",
      "            \"orgZipCode\": \"231878795\",\n",
      "            \"otherPis\": null,\n",
      "            \"projectEndDate\": \"2015-06-30T00:00:00\",\n",
      "            \"projectNumber\": \"2R15AI053062-04\",\n",
      "            \"projectStartDate\": \"2003-05-15T00:00:00\",\n",
      "            \"smApplId\": 22809,\n",
      "            \"terms\": \" Architecture; Bacteria; Bacterial Genome; Bacterial Proteins; Bacteriophages; Benign; Bioinformatics; Biology; carcinogenesis; Cells; Clinical; college; comparative genomics; Core Facility; design; Disease; Duodenal Ulcer; Educational process of instructing; Elements; Event; Evolution; Gastric ulcer; Genes; Genetic; Genetic Transcription; Genome; genome sequencing; Genomics; Helicobacter Infections; Helicobacter pylori; Horizontal Gene Transfer; Human; Inflammation; Inflammatory; insight; interest; Ions; Laboratories; Learning; Length; Life Style; Malignant Neoplasms; malignant stomach neoplasm; Mediating; Molecular; Mutate; next generation; novel; pathogen; pathogenic bacteria; Pathogenicity Island; Patients; Peptic Ulcer; Persons; Poly G; Population; Promoter Regions; Prophages; Proteins; Pseudogenes; Research; research study; Role; Signal Transduction; Stomach; Technology; Transcriptional Regulation; Ulcer; undergraduate research; Virulence; Virulent\",\n",
      "            \"title\": \"EVOLUTION OF A BACTERIAL GENOME ENCODING INCREASED VIRULENCE: THE GENOMICS OF HEL\",\n",
      "            \"totalCostAmount\": 346000\n",
      "        },\n",
      "        {\n",
      "            \"abstract\": \"DESCRIPTION (provided by applicant): Nervous system development encompasses generation of neural cells, followed by their positioning and differentiation to culminate with establishment of specific connections that enable the execution of innumerable functions. Although both the developing and mature nervous system exhibit tremendous plasticity, comparatively, the former has a greater capacity for remodeling. When the neural tissue is injured or its function is imbalanced, as for patients suffering from spinal cord injury or epilepsy, recreating the remarkable plasticity of the developing nervous system towards repairing and regenerating damaged cells and connections is the ultimate goal. Understanding developmental processes becomes crucial to devising therapies targeting nervous system regeneration and repair. Two major developmental cues are key for nervous system development, the morphogentic protein, Sonic hedgehog (Shh), and early electrical activity. Although many aspects of their action are known, there has been no previous consideration of the interaction between them. This research project wil study the molecular mechanisms underlying the interplay between Shh and electrical activity and how their interaction affects nervous system development and maturation. This study will test the hypothesis that the Shh gradient across the dorsoventral axis of the nervous system contributes to establish a gradient of calcium-dependent electrical activity across the dorsoventral developing spinal cord that in turn regulates neuronal differentiation. Pharmacological and molecular manipulations of Shh gradient and signaling wil be implemented in developing Xenopus embryos. Calcium dynamics will be imaged in neurons on the dorsal and ventral surfaces of the neural tube of control and experimentally perturbed embryos. Reciprocally, manipulations of calcium spike activity will be carried out and the consequences to Shh signaling in the developing spinal cord will be assessed. To investigate the functional consequences of the interplay between Shh signaling and calcium spike activity, a crucial biological outcome will be studied: dorsoventral differentiation of developing spinal neurons. The involvement of calcium spike activity-dependent pathways in this process will be studied in vivo following pharmacological and molecular perturbations of Shh signaling and electrical activity. This project may lead to new ways of thinking of how the nervous system develops and how different pathways interact to promote its formation and maturation. When generation and differentiation of neural cells is needed to reestablish lost connections, the ability to direct the affected system toward the production of appropriate number and type of cells is paramount. This investigation of mechanisms underlying neuronal differentiation may set the basis for devising treatments for neurological disorders such as pediatric epilepsy and spinal cord injury, in which both the reestablishment of balanced excitability and the reposition of damaged cells are key events for promoting recovery.  \",\n",
      "            \"agency\": \"NIH\",\n",
      "            \"budgetEndDate\": \"2013-06-30T00:00:00\",\n",
      "            \"budgetStartDate\": \"2012-07-01T00:00:00\",\n",
      "            \"cfdaCode\": \"853\",\n",
      "            \"congressionalDistrict\": \"03\",\n",
      "            \"contactPi\": \"BORODINSKY, LAURA NOEMI\",\n",
      "            \"department\": \"HHS\",\n",
      "            \"dunsId\": \"047120084\",\n",
      "            \"fy\": 2012,\n",
      "            \"ic\": \"NINDS\",\n",
      "            \"latitude\": 38.543366,\n",
      "            \"longitude\": -121.72946,\n",
      "            \"nihApplId\": 8290336,\n",
      "            \"orgCity\": \"DAVIS\",\n",
      "            \"orgCountry\": \"UNITED STATES\",\n",
      "            \"orgName\": \"UNIVERSITY OF CALIFORNIA DAVIS\",\n",
      "            \"orgState\": \"CA\",\n",
      "            \"orgZipCode\": \"956186153\",\n",
      "            \"otherPis\": null,\n",
      "            \"projectEndDate\": \"2016-06-30T00:00:00\",\n",
      "            \"projectNumber\": \"5R01NS073055-02\",\n",
      "            \"projectStartDate\": \"2011-07-01T00:00:00\",\n",
      "            \"smApplId\": 22855,\n",
      "            \"terms\": \" Address; Affect; axon guidance; axonal pathfinding; base; Biological; Biological Assay; Biosensor; Brain; Calcium; Calcium Spikes; Cancer stem cell; cell injury; Cell Proliferation; cell type; Cells; Childhood; Cilia; Complex; Cues; Development; Developmental Biology; Developmental Process; Dominant-Negative Mutation; Dorsal; Embryo; Epilepsy; Equilibrium; Erinaceidae; Event; Exhibits; Generations; Goals; human SMO protein; Image; in vivo; injured; innovation; Investigation; Knowledge; Label; Lead; Life; Link; Luciferases; Mediating; migration; Molecular; morphogens; Natural regeneration; Nerve Regeneration; nerve stem cell; nervous system development; nervous system disorder; Nervous system structure; Neural tube; Neuronal Differentiation; Neurons; Neurosciences; Neurotransmitters; novel; Outcome; Pathway interactions; Patients; Pattern; Phenotype; Positioning Attribute; Process; Production; Proteins; receptor; Recovery; relating to nervous system; repaired; Research; Research Project Grants; Role; second messenger; Second Messenger Systems; Signal Transduction; Signal Transduction Pathway; smoothened signaling pathway; Sonic hedgehog protein; Spinal; Spinal Cord; Spinal cord injury; Staging; Surface; Synapses; synaptogenesis; System; Testing; Thinking; tissue repair; Tissues; Transcriptional Regulation; Western Blotting; Xenopus\",\n",
      "            \"title\": \"SPINAL CORD DEVELOPMENT: INTERPLAY BETWEEN ELECTRICAL ACTIVITY AND SONIC HEDGEHOG\",\n",
      "            \"totalCostAmount\": 332176\n",
      "        },\n",
      "        {\n",
      "            \"abstract\": \"DESCRIPTION (provided by applicant): The ability to study whole organisms makes it possible to study complex in vivo processes that cannot be replicated in vitro such as organ development, liver, pancreas, heart, and neuronal regeneration, cancer metastasis, neural degeneration, infectious disease progression, pathogenesis, cardiovascular, immune, endocrine, and nervous system functions. Cells are not transformed and are in their normal physiological environment of cell-cell, extracellular matrix, and other interactions. Microarray studies and in vitro screens using cell lines and millions of combinatorially synthesized compounds have generated thousands of possible genetic targets and drug candidates. Identification of specificity, potency, toxicity, and biodistribution of pharmaceuticals as well as functions of thousands genes on entire organs like kidney, liver, heart, and brain cannot be done in vitro, and require use of in vivo animal models. Currently, there is significant gap between the throughput and capabilities of in vitro and in vivo assays on vertebrates. As a result, during early stages of drug screening and development, pharmaceuticals cannot be tested in vivo. Failure of tests on animals at later stages of development not only costs dearly, but also slows progress significantly. Yet, high-throughput testing of gene functions and compounds using in vivo vertebrate animal models has so far been significantly limited due to the absence of key technologies. Here, we propose a highly transformative technology that will allow, for the first time, large- scale in vivo genetic and chemical screens at cellular resolution on complex organs of vertebrates such as heart, liver, kidney, pancreas, vision, immune system, and central nervous system. This technology can impact a broad spectrum of fields ranging from neurobiology to regenerative biology, and cancer biology. The proposed high-speed whole-animal manipulation, orientation, immobilization, imaging, microsurgery, and injection platform will enable a dramatic increase in the throughput and complexity with which in vivo assays can be performed (~5-10 seconds per animal depending on the observed phenotype and manipulation complexity instead of the 10-30 minutes it currently takes). Our proposal is highly relevant to NIH's roadmap goals as it will allow systematic and unbiased genome-wide vertebrate studies to dramatically accelerate both fundamental and translational research in identification of gene functions as well as in discovery of drug leads. To demonstrate system capabilities, we will perform the first large-scale in vivo chemical screen for regenerating micro-surgically injured spinal-cord fibers.\",\n",
      "            \"agency\": \"NIH\",\n",
      "            \"budgetEndDate\": \"2013-05-31T00:00:00\",\n",
      "            \"budgetStartDate\": \"2012-06-01T00:00:00\",\n",
      "            \"cfdaCode\": \"310\",\n",
      "            \"congressionalDistrict\": \"07\",\n",
      "            \"contactPi\": \"YANIK, MEHMET FATIH\",\n",
      "            \"department\": \"HHS\",\n",
      "            \"dunsId\": \"001425594\",\n",
      "            \"fy\": 2012,\n",
      "            \"ic\": \"NINDS\",\n",
      "            \"latitude\": 42.359888,\n",
      "            \"longitude\": -71.094312,\n",
      "            \"nihApplId\": 8268464,\n",
      "            \"orgCity\": \"CAMBRIDGE\",\n",
      "            \"orgCountry\": \"UNITED STATES\",\n",
      "            \"orgName\": \"MASSACHUSETTS INSTITUTE OF TECHNOLOGY\",\n",
      "            \"orgState\": \"MA\",\n",
      "            \"orgZipCode\": \"021421029\",\n",
      "            \"otherPis\": null,\n",
      "            \"projectEndDate\": \"2015-05-31T00:00:00\",\n",
      "            \"projectNumber\": \"5R01NS073127-03\",\n",
      "            \"projectStartDate\": \"2010-09-27T00:00:00\",\n",
      "            \"smApplId\": 22865,\n",
      "            \"terms\": \" Animal Model; Animals; Biodistribution; Biological Assay; Biology; Brain; Cancer Biology; cancer type; Cardiovascular system; Cell Line; Cells; cellular targeting; chemical genetics; Chemicals; Communicable Diseases; Complex; cost; Development; Disease; Disease model; Disease Progression; drug candidate; drug development; Embryo; Endocrine system; Environment; Extracellular Matrix; Failure; Fiber; fundamental research; gene function; Gene Silencing; Genes; Genetic; Genetic Screening; genome-wide; Goals; Habitats; Heart; high throughput screening; human disease; Image; Immobilization; Immune system; In Vitro; in vivo; Injection of therapeutic agent; Injury; innovation; Kidney; Larva; Lead; Liver; Mammals; Microfluidics; Microinjections; Microsurgery; Modeling; mutant; Natural regeneration; Neoplasm Metastasis; Nerve Degeneration; Nervous System Physiology; Neuraxis; Neurobiology; Neurons; Optics; Organ; Pancreas; Pathogenesis; Pharmaceutical Preparations; Pharmacologic Substance; Phenotype; Physiological; Preclinical Drug Evaluation; Process; regenerative; Resolution; Screening procedure; small molecule; Specificity; Speed; Spinal cord injury; Staging; System; Techniques; Technology; Testing; Time; Toxic effect; Translational Research; United States National Institutes of Health; Vertebrates; Vision; Whole Organism; Zebrafish\",\n",
      "            \"title\": \"HIGH-THROUGHPUT IN VIVO SUBCELLULAR-RESOLUTION VERTEBRATE SCREENING PLATFORM\",\n",
      "            \"totalCostAmount\": 401143\n",
      "        }\n",
      "    ],\n",
      "    \"limit\": 50,\n",
      "    \"offset\": 1,\n",
      "    \"totalCount\": 335495,\n",
      "    \"totalPages\": 6710\n",
      "}\n"
     ]
    }
   ],
   "source": [
    "response = requests.get(url).json()\n",
    "\n",
    "# print results from API call\n",
    "print(json.dumps(response, indent=4, sort_keys=True))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "dict_keys(['totalCount', 'offset', 'limit', 'totalPages', 'items'])"
      ]
     },
     "execution_count": 7,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "response.keys()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[' Aging; Agriculture; base; Biotechnology; career; Climate; Clinical; college; Communities; Community Developments; Computers; design; Development; Discipline; Economics; Education; Educational workshop; Employment; Enrollment; experience; Face; Forensic Medicine; Future; Goals; Healthcare; high school; Individual; information model; Internships; Intervention; Life; Mentors; Occupations; Pathway interactions; Pharmacy (field); Population; Preparation; Program Evaluation; programs; Radial; remediation; Research; Resort; retiree; Science; science education; Science, Technology, Engineering and Mathematics; Services; Students; Technology; tool; Training Programs; university student; Vocational Guidance; Wages; Work', ' Biotechnology; Training Programs', ' Biotechnology; Leadership; Pharmacologic Substance; Training', ' Biotechnology; Molecular; Training', ' Biotechnology; Marines; Training Programs', ' Biotechnology; Engineering; Science; Training', ' Attitude; Biotechnology; college; Communities; Development; Education; Educational process of instructing; high school; improved; Industry; Institutes; junior high school; Knowledge; Laboratories; Learning; Methodology; middle school; New York City; Outcome; research study; Schools; Science; Services; simulation; skills; Students; symposium; teacher; virtual', ' Area; Biology; Biotechnology; Collaborations; college; Communities; Development; digital imaging; Education; Educational workshop; elementary school; Employment; Fluorescence Microscopy; Government; high school; Industry; Internships; junior high school; Laboratories; Microscopy; middle school; outreach; Phase; programs; Protocols documentation; response; Services; Students; teacher; Training; Update', ' Achievement; Address; Biotechnology; Cells; Centrifugation; Chimeric Proteins; Chromatography; Class; Commit; Condition; cost; Cultured Cells; design; Diagnostic; Dialysis procedure; DNA; Doctor of Philosophy; Escherichia coli; Fermentation; Fiber; Filtration; Fractionation; Future; Goals; Healthcare; improved; Industry; innovation; instrument; interest; Label; Laboratory Research; Licensing; Marketing; Membrane; Methods; Molecular; Monitor; nanomedicine; Nanotubes; new technology; NIH Program Announcements; novel; novel therapeutics; Performance; Personal Satisfaction; Pharmaceutical Preparations; Phase; Physical Dialysis; Plasmids; point of care; Polymers; Precipitation; present value; Process; protein expression; Proteins; Proteomics; Protocols documentation; prototype; Public Health; Pump; Rate; Research; Sampling; scale up; seal; Sodium Chloride; Solvents; Staging; Stream; success; System; Techniques; Technology; technology development; Time; Ubiquitin; United States National Institutes of Health; Vaccine Production; Vaccines', ' Biotechnology; pre-doctoral; Training Programs', ' Biotechnology; Training Programs', ' Biotechnology; pre-doctoral; Training Programs', ' Biotechnology; Development; Future; Training', ' Biotechnology; Training Programs', ' Biotechnology; Grant; Research Training', ' anticancer research; Area; Arts; Biochemical; Biochemistry; Biotechnology; Cancer Etiology; Collaborations; Complex; Depth; Discipline; Disease; DNA biosynthesis; DNA chemical synthesis; DNA Sequence Analysis; Doctor of Philosophy; Fluorescent Probes; Fox Chase Cancer Center; Funding; Gene Expression; Genes; Genetic Polymorphism; Grant; Human Resources; Investigation; Laboratories; Mass Spectrum Analysis; Modification; Mutation; Oligonucleotides; Participant; Peer Review; Peptides; Performance; Pharmaceutical Preparations; Polymerase Chain Reaction; programs; Proteins; Proteomics; Quality Control; Regulation; Research; Research Personnel; Resources; response; Science; Services; Structure; Technology; Time; tool; Transcript', ' Achievement; Address; Annual Reports; Area; Arts; Basic Science; Benchmarking; Biotechnology; career; Collaborations; Communication; Communities; Consult; Development; Educational workshop; Engineering; Enrollment; Equipment; Evaluation; experience; Faculty; Funding; Future; Goals; Grant; Health; health disparity; Image; improved; Incentives; Institution; instrumentation; Journals; meetings; Mentors; Minority; minority health; Monitor; Natural Sciences; Outcome; Paper; peer; physical science; Play; Principal Investigator; programs; Protocols documentation; Publications; Publishing; Recruitment Activity; Research; Research Infrastructure; Research Personnel; Research Project Grants; Research Technics; Role; Schools; Science; Series; skills training; stem; Structure; Students; success; symposium; Time; Training; Training Support; Travel; Universities; Writing', \" base; Base Pairing; Bioinformatics; Bioinformatics Shared Resource; Biotechnology; Cancer Center; Cancer Center Support Grant; Capillary Electrophoresis; Chemistry; Childhood; Complementary DNA; Computer software; computing resources; Data; data management; Data Storage and Retrieval; Databases; Detection; Development; Discipline; Disease; DNA; DNA Microarray Chip; DNA Sequence; DNA Sequence Analysis; Doctor of Philosophy; Environment; Ependymoma; functional genomics; Funding; Gel; Genome; Genotype; High Performance Computing; instrumentation; Internet; Laboratories; Malignant Neoplasms; Measurement; medulloblastoma; Methylation; Mission; NCI Center for Cancer Research; Oligonucleotides; Online Systems; Operative Surgical Procedures; Postdoctoral Fellow; Process; Production; programs; Protein Chemistry; Proteins; Proteomics; Research Personnel; Resource Sharing; Resources; Robotics; Saint Jude Children's Research Hospital; Science; Security; Seminal; Sequence Analysis; Services; Slice; software development; Solutions; Spottings; System; Technology\", ' Address; Admission activity; Applications Grants; Area; Automobiles; Award; base; Biotechnology; Businesses; Cities; Climate; college; Communication; Communities; Competence; Cost Savings; Country; Data Collection; design; Development; economic impact; Economics; Education; Educational process of instructing; Effectiveness; Elements; Engineering; Ensure; Environmental Health; Equipment; Evaluation; experience; follow-up; Fostering; Future; Gene Mutation; Genetic Code; Gifts; Goals; Grant; Growth; health science research; high school; Impact evaluation; Individual; innovation; Institutes; interest; Internet; Knowledge; Laboratories; Learning; Marketing; Mathematics; Methodology; Methods; Michigan; Monitor; Occupations; Outcome; Positioning Attribute; Postdoctoral Fellow; Program Sustainability; programs; prospective; Protein Engineering; public health relevance; Qualifying; Reporting; Research; Research Personnel; Resources; Schools; Science; science education; Secure; Security; Services; skills; Small Business Innovation Research Grant; Students; Surveys; symposium; teacher; Techniques; Technology; Testing; Time; TimeLine; tool; trend; Universities; virtual; Work', ' Biotechnology; Development; Future; Training', ' Biotechnology; Training Programs', ' Biotechnology; Molecular; Training', ' Biotechnology; Engineering; Science; Training', ' Biotechnology; Protein Chemistry; Training', ' Biotechnology; Leadership; Pharmacologic Substance; Training', ' Biotechnology; Marines; Training Programs', ' Biotechnology; Training Programs', ' Biotechnology; Grant; Research Training', ' Biotechnology; Training Programs', ' Biotechnology; Training Programs', ' Biotechnology; pre-doctoral; Training Programs', ' anticancer research; Area; Arts; Biochemical; Biochemistry; Biotechnology; Cancer Etiology; Collaborations; Comprehensive Cancer Center; Discipline; Disease; DNA biosynthesis; DNA Sequence Analysis; Doctor of Philosophy; Fluorescent Probes; Fox Chase Cancer Center; Foxes; Funding; Gene Expression; Genes; Genetic Polymorphism; Grant; Human Resources; Investigation; Laboratories; Mass Spectrum Analysis; Modification; Mutation; Oligonucleotides; Participant; Peer Review; Peptides; Performance; Pharmaceutical Preparations; programs; protein complex; Proteins; Proteomics; Quality Control; Regulation; Research; Research Personnel; Resources; response; Science; Services; Structure; Technology; Time; tool; Transcript', ' Biotechnology; pre-doctoral; Training Programs', ' Biotechnology; Pharmacologic Substance; Training', ' Biotechnology; pre-doctoral; Training Programs', ' Biotechnology; Training Programs', ' Biotechnology; Science; Training', ' Area; Basic Science; Biochemical; Biology; biomedical scientist; Biotechnology; Businesses; Colorado; Computers; design; Engineering; Entrepreneurship; Faculty; Functional RNA; Funding; Industry; innovation; Interdisciplinary Study; Laboratories; Medicine; Molecular; Occupations; programs; Request for Proposals; Schools; Scientist; Signal Transduction; Structural Biologist; structural biology; System; Translations; Universities; Work', ' Biotechnology; Molecular; Training', ' Biotechnology; Training Programs', ' Biotechnology; Training Programs', ' Address; Area; base; Binding; Biological; Biomedical Engineering; Biomedical Research; Biotechnology; Cell physiology; Cellular Structures; Client; Collaborations; Communities; Consult; Coupled; crosslink; Data; Detection; Development; Device or Instrument Development; Devices; Diffusion; digital; DNA-protein crosslink; Effectiveness; Electronics; Engineering; Equipment; Evaluation; experience; Fluorescence; Fluorescence Spectroscopy; Generations; Goals; Grant; Hand; Image; Imagery; improved; in vivo; Information Dissemination; innovation; instrumentation; Investigation; Lasers; Measurement; Mediating; meetings; Methods; Microscopy; Mission; Molecular; Monitor; Multiphoton Fluorescence Microscopy; nano; nanofabrication; Nanostructures; Nanotechnology; National Center for Research Resources; new technology; novel; optical fiber; optical imaging; Optics; Participant; Photochemistry; Physiologic pulse; pre-doctoral; Prions; programs; protein folding; Provider; Publishing; Pump; Refractory; Research; research and development; Research Design; Research Infrastructure; Research Project Grants; research study; Resources; second harmonic; Services; single molecule; Solutions; Spectrum Analysis; Students; synergism; System; Techniques; technological innovation; Technology; technology development; Technology Transfer; Testing; Time; Tissues; tool; Training; Universities; user-friendly', ' Address; Admission activity; Applications Grants; Area; Automobiles; Award; base; Biotechnology; Businesses; Cities; Climate; college; Communication; Communities; Competence; Cost Savings; Country; Data Collection; design; Development; economic impact; Economics; Education; Educational process of instructing; Effectiveness; Elements; Engineering; Ensure; Environmental Health; Equipment; Evaluation; experience; follow-up; Fostering; Future; Gene Mutation; Genetic Code; Gifts; Goals; Grant; Growth; health science research; high school; Impact evaluation; Individual; innovation; Institutes; interest; Internet; Knowledge; Laboratories; Learning; Marketing; Mathematics; Methodology; Methods; Michigan; Monitor; Occupations; Outcome; Positioning Attribute; Postdoctoral Fellow; Program Sustainability; programs; prospective; Protein Engineering; public health relevance; Qualifying; Reporting; Research; Research Personnel; Resources; Schools; Science; science education; Secure; Security; Services; skills; Small Business Innovation Research Grant; Students; Surveys; symposium; teacher; Techniques; Technology; Testing; Time; TimeLine; tool; trend; Universities; virtual; Work', ' Address; Animal Model; Area; Bacteria; Biodiversity; Biomedical Engineering; Bioremediations; Biotechnology; Catalytic Domain; Chemicals; Coupled; Cytochrome P450; Drug Metabolic Detoxication; Engineering; Enzyme Gene; Enzymes; Evolution; Exposure to; Fostering; Funding; Gene Proteins; Genes; Goals; graduate student; Insecta; insight; International; Knowledge; meetings; microbial; novel; Pathway interactions; Pharmaceutical Preparations; plant fungi; Plants; Postdoctoral Fellow; Process; public health relevance; remediation; Research; response; Risk Assessment; Role; Scientist; Structural Protein; Structure; Students; symposium; System; Toxicology; Training; Xenobiotic Metabolism; Xenobiotics; Zebrafish', \" base; Base Pairing; Bioinformatics; Bioinformatics Shared Resource; Biotechnology; Cancer Center; Cancer Center Support Grant; Capillary Electrophoresis; Chemistry; Childhood; Complementary DNA; Computer software; computing resources; Data; data management; Data Storage and Retrieval; Databases; Detection; Development; Discipline; Disease; DNA; DNA Microarray Chip; DNA Sequence; DNA Sequence Analysis; Doctor of Philosophy; Environment; Ependymoma; functional genomics; Funding; Gel; Genome; Genotype; High Performance Computing; instrumentation; Internet; Laboratories; Malignant Neoplasms; Measurement; medulloblastoma; Methylation; Mission; NCI Center for Cancer Research; Oligonucleotides; Online Systems; operation; Postdoctoral Fellow; Process; Production; programs; Protein Chemistry; Proteins; Proteomics; Research Personnel; Resource Sharing; Resources; Robotics; Saint Jude Children's Research Hospital; Science; Security; Seminal; Sequence Analysis; Services; Slice; software development; Solutions; Spottings; System; Technology\", ' Address; Area; base; Binding; Biological; Biomedical Engineering; Biomedical Research; Biotechnology; Cell physiology; Cellular Structures; Client; Collaborations; Communities; Consult; Coupled; crosslink; Data; Detection; Development; Device or Instrument Development; Devices; Diffusion; digital; DNA-protein crosslink; Effectiveness; Electronics; Engineering; Equipment; Evaluation; experience; Fluorescence; Fluorescence Spectroscopy; Generations; Goals; Grant; Hand; Image; Imagery; improved; in vivo; Information Dissemination; innovation; instrumentation; Investigation; Lasers; Measurement; Mediating; meetings; Methods; Microscopy; Mission; Molecular; Monitor; Multiphoton Fluorescence Microscopy; nano; nanofabrication; Nanostructures; Nanotechnology; National Center for Research Resources; new technology; novel; optical fiber; optical imaging; Optics; Participant; Photochemistry; Physiologic pulse; pre-doctoral; Prions; programs; protein folding; Provider; Publishing; Pump; Refractory; Research; research and development; Research Design; Research Infrastructure; Research Project Grants; research study; Resources; second harmonic; Services; single molecule; Solutions; Spectrum Analysis; Students; synergism; System; Techniques; technological innovation; Technology; technology development; Technology Transfer; Testing; Time; Tissues; tool; Training; Universities; user-friendly', ' absorption; Address; Adoption; Area; base; Biochemical; Biology; Biomedical Research; Biotechnology; Blood; Breast; cell type; Cells; Chemistry; Clinical Medicine; Collaborations; commercialization; Complement; Contrast Media; cost; data acquisition; Data Analyses; design; Development; Device or Instrument Development; Diagnostic; Diagnostic Imaging; Discipline; Engineering; Ensure; Environment; Fertilization; Florida; Fluorescence; fluorophore; Funding; Head; Health; Hemoglobin; Human; Imagery; Imaging Device; Imaging Techniques; in vivo; in vivo Model; Individual; Industry; Knowledge; Laser Surgery; Learning; Length; Lesion; Light; Malignant - descriptor; Mathematical Computing; Mathematics; Medical; Medicine; meetings; Microscopy; Modality; Molecular; Operative Surgical Procedures; optical imaging; Optics; Perfusion; Physics; Postdoctoral Fellow; posters; Property; Proteins; Research; skills; social; Specificity; Students; symposium; Techniques; Therapeutic; Time; Tissues; Training; Transgenic Organisms; Travel; United States National Institutes of Health; Viola; Water; Work', ' Address; Admission activity; Applications Grants; Area; Automobiles; Award; base; Biotechnology; Businesses; Cities; Climate; college; Communication; Communities; Competence; Cost Savings; Country; Data Collection; design; Development; economic impact; Economics; Education; Educational process of instructing; Effectiveness; Elements; Engineering; Ensure; Environmental Health; Equipment; Evaluation; experience; follow-up; Fostering; Future; Gene Mutation; Genetic Code; Gifts; Goals; Grant; Growth; health science research; high school; Impact evaluation; Individual; innovation; Institutes; interest; Internet; Knowledge; Laboratories; Learning; Marketing; Mathematics; Methodology; Methods; Michigan; Monitor; Occupations; Outcome; Positioning Attribute; Postdoctoral Fellow; Program Sustainability; programs; prospective; Protein Engineering; public health relevance; Qualifying; Reporting; Research; Research Personnel; Resources; Schools; Science; science education; Secure; Security; Services; skills; Small Business Innovation Research Grant; Students; Surveys; symposium; teacher; Techniques; Technology; Testing; Time; TimeLine; tool; trend; Universities; virtual; Work', ' Biotechnology; pre-doctoral; Training Programs', ' Biotechnology; Leadership; Pharmacologic Substance; Training', ' Aging; Agriculture; base; Biotechnology; career; Climate; Clinical; college; Communities; Community Developments; Computers; design; Development; Discipline; Economics; Education; Educational workshop; Employment; Enrollment; experience; Face; Forensic Medicine; Future; Goals; Healthcare; high school; Individual; information model; Internships; Intervention; Life; Mentors; Occupations; Pathway interactions; Pharmacy (field); Population; Preparation; Program Evaluation; programs; Radial; remediation; Research; Resort; retiree; Science; science education; Science, Technology, Engineering and Mathematics; Services; Students; Technology; tool; Training Programs; university student; Vocational Guidance; Wages; Work', ' Biotechnology; Training Programs', ' Biotechnology; Leadership; Pharmacologic Substance; Training', ' Biotechnology; Molecular; Training', ' Biotechnology; Marines; Training Programs', ' Biotechnology; Engineering; Science; Training', ' Attitude; Biotechnology; college; Communities; Development; Education; Educational process of instructing; high school; improved; Industry; Institutes; junior high school; Knowledge; Laboratories; Learning; Methodology; middle school; New York City; Outcome; research study; Schools; Science; Services; simulation; skills; Students; symposium; teacher; virtual', ' Area; Biology; Biotechnology; Collaborations; college; Communities; Development; digital imaging; Education; Educational workshop; elementary school; Employment; Fluorescence Microscopy; Government; high school; Industry; Internships; junior high school; Laboratories; Microscopy; middle school; outreach; Phase; programs; Protocols documentation; response; Services; Students; teacher; Training; Update', ' Achievement; Address; Biotechnology; Cells; Centrifugation; Chimeric Proteins; Chromatography; Class; Commit; Condition; cost; Cultured Cells; design; Diagnostic; Dialysis procedure; DNA; Doctor of Philosophy; Escherichia coli; Fermentation; Fiber; Filtration; Fractionation; Future; Goals; Healthcare; improved; Industry; innovation; instrument; interest; Label; Laboratory Research; Licensing; Marketing; Membrane; Methods; Molecular; Monitor; nanomedicine; Nanotubes; new technology; NIH Program Announcements; novel; novel therapeutics; Performance; Personal Satisfaction; Pharmaceutical Preparations; Phase; Physical Dialysis; Plasmids; point of care; Polymers; Precipitation; present value; Process; protein expression; Proteins; Proteomics; Protocols documentation; prototype; Public Health; Pump; Rate; Research; Sampling; scale up; seal; Sodium Chloride; Solvents; Staging; Stream; success; System; Techniques; Technology; technology development; Time; Ubiquitin; United States National Institutes of Health; Vaccine Production; Vaccines', ' Biotechnology; pre-doctoral; Training Programs', ' Biotechnology; Training Programs', ' Biotechnology; pre-doctoral; Training Programs', ' Biotechnology; Development; Future; Training', ' Biotechnology; Training Programs', ' Biotechnology; Grant; Research Training', ' anticancer research; Area; Arts; Biochemical; Biochemistry; Biotechnology; Cancer Etiology; Collaborations; Complex; Depth; Discipline; Disease; DNA biosynthesis; DNA chemical synthesis; DNA Sequence Analysis; Doctor of Philosophy; Fluorescent Probes; Fox Chase Cancer Center; Funding; Gene Expression; Genes; Genetic Polymorphism; Grant; Human Resources; Investigation; Laboratories; Mass Spectrum Analysis; Modification; Mutation; Oligonucleotides; Participant; Peer Review; Peptides; Performance; Pharmaceutical Preparations; Polymerase Chain Reaction; programs; Proteins; Proteomics; Quality Control; Regulation; Research; Research Personnel; Resources; response; Science; Services; Structure; Technology; Time; tool; Transcript', ' Achievement; Address; Annual Reports; Area; Arts; Basic Science; Benchmarking; Biotechnology; career; Collaborations; Communication; Communities; Consult; Development; Educational workshop; Engineering; Enrollment; Equipment; Evaluation; experience; Faculty; Funding; Future; Goals; Grant; Health; health disparity; Image; improved; Incentives; Institution; instrumentation; Journals; meetings; Mentors; Minority; minority health; Monitor; Natural Sciences; Outcome; Paper; peer; physical science; Play; Principal Investigator; programs; Protocols documentation; Publications; Publishing; Recruitment Activity; Research; Research Infrastructure; Research Personnel; Research Project Grants; Research Technics; Role; Schools; Science; Series; skills training; stem; Structure; Students; success; symposium; Time; Training; Training Support; Travel; Universities; Writing', \" base; Base Pairing; Bioinformatics; Bioinformatics Shared Resource; Biotechnology; Cancer Center; Cancer Center Support Grant; Capillary Electrophoresis; Chemistry; Childhood; Complementary DNA; Computer software; computing resources; Data; data management; Data Storage and Retrieval; Databases; Detection; Development; Discipline; Disease; DNA; DNA Microarray Chip; DNA Sequence; DNA Sequence Analysis; Doctor of Philosophy; Environment; Ependymoma; functional genomics; Funding; Gel; Genome; Genotype; High Performance Computing; instrumentation; Internet; Laboratories; Malignant Neoplasms; Measurement; medulloblastoma; Methylation; Mission; NCI Center for Cancer Research; Oligonucleotides; Online Systems; Operative Surgical Procedures; Postdoctoral Fellow; Process; Production; programs; Protein Chemistry; Proteins; Proteomics; Research Personnel; Resource Sharing; Resources; Robotics; Saint Jude Children's Research Hospital; Science; Security; Seminal; Sequence Analysis; Services; Slice; software development; Solutions; Spottings; System; Technology\", ' Address; Admission activity; Applications Grants; Area; Automobiles; Award; base; Biotechnology; Businesses; Cities; Climate; college; Communication; Communities; Competence; Cost Savings; Country; Data Collection; design; Development; economic impact; Economics; Education; Educational process of instructing; Effectiveness; Elements; Engineering; Ensure; Environmental Health; Equipment; Evaluation; experience; follow-up; Fostering; Future; Gene Mutation; Genetic Code; Gifts; Goals; Grant; Growth; health science research; high school; Impact evaluation; Individual; innovation; Institutes; interest; Internet; Knowledge; Laboratories; Learning; Marketing; Mathematics; Methodology; Methods; Michigan; Monitor; Occupations; Outcome; Positioning Attribute; Postdoctoral Fellow; Program Sustainability; programs; prospective; Protein Engineering; public health relevance; Qualifying; Reporting; Research; Research Personnel; Resources; Schools; Science; science education; Secure; Security; Services; skills; Small Business Innovation Research Grant; Students; Surveys; symposium; teacher; Techniques; Technology; Testing; Time; TimeLine; tool; trend; Universities; virtual; Work', ' Biotechnology; Development; Future; Training', ' Biotechnology; Training Programs', ' Biotechnology; Molecular; Training', ' Biotechnology; Engineering; Science; Training', ' Biotechnology; Protein Chemistry; Training', ' Biotechnology; Leadership; Pharmacologic Substance; Training', ' Biotechnology; Marines; Training Programs', ' Biotechnology; Training Programs', ' Biotechnology; Grant; Research Training', ' Biotechnology; Training Programs', ' Biotechnology; Training Programs', ' Biotechnology; pre-doctoral; Training Programs', ' anticancer research; Area; Arts; Biochemical; Biochemistry; Biotechnology; Cancer Etiology; Collaborations; Comprehensive Cancer Center; Discipline; Disease; DNA biosynthesis; DNA Sequence Analysis; Doctor of Philosophy; Fluorescent Probes; Fox Chase Cancer Center; Foxes; Funding; Gene Expression; Genes; Genetic Polymorphism; Grant; Human Resources; Investigation; Laboratories; Mass Spectrum Analysis; Modification; Mutation; Oligonucleotides; Participant; Peer Review; Peptides; Performance; Pharmaceutical Preparations; programs; protein complex; Proteins; Proteomics; Quality Control; Regulation; Research; Research Personnel; Resources; response; Science; Services; Structure; Technology; Time; tool; Transcript', ' Biotechnology; pre-doctoral; Training Programs', ' Biotechnology; Pharmacologic Substance; Training', ' Biotechnology; pre-doctoral; Training Programs', ' Biotechnology; Training Programs', ' Biotechnology; Science; Training', ' Area; Basic Science; Biochemical; Biology; biomedical scientist; Biotechnology; Businesses; Colorado; Computers; design; Engineering; Entrepreneurship; Faculty; Functional RNA; Funding; Industry; innovation; Interdisciplinary Study; Laboratories; Medicine; Molecular; Occupations; programs; Request for Proposals; Schools; Scientist; Signal Transduction; Structural Biologist; structural biology; System; Translations; Universities; Work', ' Biotechnology; Molecular; Training', ' Biotechnology; Training Programs', ' Biotechnology; Training Programs', ' Address; Area; base; Binding; Biological; Biomedical Engineering; Biomedical Research; Biotechnology; Cell physiology; Cellular Structures; Client; Collaborations; Communities; Consult; Coupled; crosslink; Data; Detection; Development; Device or Instrument Development; Devices; Diffusion; digital; DNA-protein crosslink; Effectiveness; Electronics; Engineering; Equipment; Evaluation; experience; Fluorescence; Fluorescence Spectroscopy; Generations; Goals; Grant; Hand; Image; Imagery; improved; in vivo; Information Dissemination; innovation; instrumentation; Investigation; Lasers; Measurement; Mediating; meetings; Methods; Microscopy; Mission; Molecular; Monitor; Multiphoton Fluorescence Microscopy; nano; nanofabrication; Nanostructures; Nanotechnology; National Center for Research Resources; new technology; novel; optical fiber; optical imaging; Optics; Participant; Photochemistry; Physiologic pulse; pre-doctoral; Prions; programs; protein folding; Provider; Publishing; Pump; Refractory; Research; research and development; Research Design; Research Infrastructure; Research Project Grants; research study; Resources; second harmonic; Services; single molecule; Solutions; Spectrum Analysis; Students; synergism; System; Techniques; technological innovation; Technology; technology development; Technology Transfer; Testing; Time; Tissues; tool; Training; Universities; user-friendly', ' Address; Admission activity; Applications Grants; Area; Automobiles; Award; base; Biotechnology; Businesses; Cities; Climate; college; Communication; Communities; Competence; Cost Savings; Country; Data Collection; design; Development; economic impact; Economics; Education; Educational process of instructing; Effectiveness; Elements; Engineering; Ensure; Environmental Health; Equipment; Evaluation; experience; follow-up; Fostering; Future; Gene Mutation; Genetic Code; Gifts; Goals; Grant; Growth; health science research; high school; Impact evaluation; Individual; innovation; Institutes; interest; Internet; Knowledge; Laboratories; Learning; Marketing; Mathematics; Methodology; Methods; Michigan; Monitor; Occupations; Outcome; Positioning Attribute; Postdoctoral Fellow; Program Sustainability; programs; prospective; Protein Engineering; public health relevance; Qualifying; Reporting; Research; Research Personnel; Resources; Schools; Science; science education; Secure; Security; Services; skills; Small Business Innovation Research Grant; Students; Surveys; symposium; teacher; Techniques; Technology; Testing; Time; TimeLine; tool; trend; Universities; virtual; Work', ' Address; Animal Model; Area; Bacteria; Biodiversity; Biomedical Engineering; Bioremediations; Biotechnology; Catalytic Domain; Chemicals; Coupled; Cytochrome P450; Drug Metabolic Detoxication; Engineering; Enzyme Gene; Enzymes; Evolution; Exposure to; Fostering; Funding; Gene Proteins; Genes; Goals; graduate student; Insecta; insight; International; Knowledge; meetings; microbial; novel; Pathway interactions; Pharmaceutical Preparations; plant fungi; Plants; Postdoctoral Fellow; Process; public health relevance; remediation; Research; response; Risk Assessment; Role; Scientist; Structural Protein; Structure; Students; symposium; System; Toxicology; Training; Xenobiotic Metabolism; Xenobiotics; Zebrafish', \" base; Base Pairing; Bioinformatics; Bioinformatics Shared Resource; Biotechnology; Cancer Center; Cancer Center Support Grant; Capillary Electrophoresis; Chemistry; Childhood; Complementary DNA; Computer software; computing resources; Data; data management; Data Storage and Retrieval; Databases; Detection; Development; Discipline; Disease; DNA; DNA Microarray Chip; DNA Sequence; DNA Sequence Analysis; Doctor of Philosophy; Environment; Ependymoma; functional genomics; Funding; Gel; Genome; Genotype; High Performance Computing; instrumentation; Internet; Laboratories; Malignant Neoplasms; Measurement; medulloblastoma; Methylation; Mission; NCI Center for Cancer Research; Oligonucleotides; Online Systems; operation; Postdoctoral Fellow; Process; Production; programs; Protein Chemistry; Proteins; Proteomics; Research Personnel; Resource Sharing; Resources; Robotics; Saint Jude Children's Research Hospital; Science; Security; Seminal; Sequence Analysis; Services; Slice; software development; Solutions; Spottings; System; Technology\", ' Address; Area; base; Binding; Biological; Biomedical Engineering; Biomedical Research; Biotechnology; Cell physiology; Cellular Structures; Client; Collaborations; Communities; Consult; Coupled; crosslink; Data; Detection; Development; Device or Instrument Development; Devices; Diffusion; digital; DNA-protein crosslink; Effectiveness; Electronics; Engineering; Equipment; Evaluation; experience; Fluorescence; Fluorescence Spectroscopy; Generations; Goals; Grant; Hand; Image; Imagery; improved; in vivo; Information Dissemination; innovation; instrumentation; Investigation; Lasers; Measurement; Mediating; meetings; Methods; Microscopy; Mission; Molecular; Monitor; Multiphoton Fluorescence Microscopy; nano; nanofabrication; Nanostructures; Nanotechnology; National Center for Research Resources; new technology; novel; optical fiber; optical imaging; Optics; Participant; Photochemistry; Physiologic pulse; pre-doctoral; Prions; programs; protein folding; Provider; Publishing; Pump; Refractory; Research; research and development; Research Design; Research Infrastructure; Research Project Grants; research study; Resources; second harmonic; Services; single molecule; Solutions; Spectrum Analysis; Students; synergism; System; Techniques; technological innovation; Technology; technology development; Technology Transfer; Testing; Time; Tissues; tool; Training; Universities; user-friendly', ' absorption; Address; Adoption; Area; base; Biochemical; Biology; Biomedical Research; Biotechnology; Blood; Breast; cell type; Cells; Chemistry; Clinical Medicine; Collaborations; commercialization; Complement; Contrast Media; cost; data acquisition; Data Analyses; design; Development; Device or Instrument Development; Diagnostic; Diagnostic Imaging; Discipline; Engineering; Ensure; Environment; Fertilization; Florida; Fluorescence; fluorophore; Funding; Head; Health; Hemoglobin; Human; Imagery; Imaging Device; Imaging Techniques; in vivo; in vivo Model; Individual; Industry; Knowledge; Laser Surgery; Learning; Length; Lesion; Light; Malignant - descriptor; Mathematical Computing; Mathematics; Medical; Medicine; meetings; Microscopy; Modality; Molecular; Operative Surgical Procedures; optical imaging; Optics; Perfusion; Physics; Postdoctoral Fellow; posters; Property; Proteins; Research; skills; social; Specificity; Students; symposium; Techniques; Therapeutic; Time; Tissues; Training; Transgenic Organisms; Travel; United States National Institutes of Health; Viola; Water; Work', ' Address; Admission activity; Applications Grants; Area; Automobiles; Award; base; Biotechnology; Businesses; Cities; Climate; college; Communication; Communities; Competence; Cost Savings; Country; Data Collection; design; Development; economic impact; Economics; Education; Educational process of instructing; Effectiveness; Elements; Engineering; Ensure; Environmental Health; Equipment; Evaluation; experience; follow-up; Fostering; Future; Gene Mutation; Genetic Code; Gifts; Goals; Grant; Growth; health science research; high school; Impact evaluation; Individual; innovation; Institutes; interest; Internet; Knowledge; Laboratories; Learning; Marketing; Mathematics; Methodology; Methods; Michigan; Monitor; Occupations; Outcome; Positioning Attribute; Postdoctoral Fellow; Program Sustainability; programs; prospective; Protein Engineering; public health relevance; Qualifying; Reporting; Research; Research Personnel; Resources; Schools; Science; science education; Secure; Security; Services; skills; Small Business Innovation Research Grant; Students; Surveys; symposium; teacher; Techniques; Technology; Testing; Time; TimeLine; tool; trend; Universities; virtual; Work', ' Biotechnology; pre-doctoral; Training Programs', ' Biotechnology; Leadership; Pharmacologic Substance; Training']\n",
      "['MIDDLETON, NATAVIA', 'LAURIE, GORDON WILLIAM', 'RANDOLPH, THEODORE W', 'KELLY, ROBERT M', 'GERWICK, WILLIAM HENRY', 'TANG, YI', 'TARAS, LORETTA BRANCACCIO', 'MANFRE, ALICIA', 'KNIGHT, MARTHA', 'SHEA, LONNIE D', 'SWANSON, JOEL A', 'QUINN, DANIEL M.', 'HU, WEI-SHOU', 'PEDERSEN, HENRIK', 'WEST, JENNIFER L', 'YOUNG, ROBERT C', 'FREEMAN, EVERETTE J', 'NAEVE, CLAYTON W.', 'DERESKI, MARY O', 'HU, WEI-SHOU', 'SWARTZ, JAMES R', 'KELLY, ROBERT M', 'MONBOUQUETTE, HAROLD GEORGE', 'REEVES, RAYMOND', 'RANDOLPH, THEODORE W', 'GERWICK, WILLIAM HENRY', 'LAURIE, GORDON WILLIAM', 'WEST, JENNIFER L', 'SWANSON, JOEL A', 'WITTRUP, KARL D', 'QUINN, DANIEL M.', 'YOUNG, ROBERT C', 'SHEA, LONNIE D', 'SIAHAAN, TERUNA J.', 'SHEA, LONNIE D', 'PEDERSEN, HENRIK', 'MONBOUQUETTE, HAROLD GEORGE', 'CECH, THOMAS ROBERT', 'KELLY, ROBERT M', 'FOX, BRIAN G', 'LAURIE, GORDON WILLIAM', 'ZIPFEL, WARREN R', 'DERESKI, MARY O', 'STEGEMAN, JOHN J', 'NAEVE, CLAYTON W.', 'ZIPFEL, WARREN R', 'HILLMAN, ELIZABETH M', 'DERESKI, MARY O', 'QUINN, DANIEL M.', 'RANDOLPH, THEODORE W', 'MIDDLETON, NATAVIA', 'LAURIE, GORDON WILLIAM', 'RANDOLPH, THEODORE W', 'KELLY, ROBERT M', 'GERWICK, WILLIAM HENRY', 'TANG, YI', 'TARAS, LORETTA BRANCACCIO', 'MANFRE, ALICIA', 'KNIGHT, MARTHA', 'SHEA, LONNIE D', 'SWANSON, JOEL A', 'QUINN, DANIEL M.', 'HU, WEI-SHOU', 'PEDERSEN, HENRIK', 'WEST, JENNIFER L', 'YOUNG, ROBERT C', 'FREEMAN, EVERETTE J', 'NAEVE, CLAYTON W.', 'DERESKI, MARY O', 'HU, WEI-SHOU', 'SWARTZ, JAMES R', 'KELLY, ROBERT M', 'MONBOUQUETTE, HAROLD GEORGE', 'REEVES, RAYMOND', 'RANDOLPH, THEODORE W', 'GERWICK, WILLIAM HENRY', 'LAURIE, GORDON WILLIAM', 'WEST, JENNIFER L', 'SWANSON, JOEL A', 'WITTRUP, KARL D', 'QUINN, DANIEL M.', 'YOUNG, ROBERT C', 'SHEA, LONNIE D', 'SIAHAAN, TERUNA J.', 'SHEA, LONNIE D', 'PEDERSEN, HENRIK', 'MONBOUQUETTE, HAROLD GEORGE', 'CECH, THOMAS ROBERT', 'KELLY, ROBERT M', 'FOX, BRIAN G', 'LAURIE, GORDON WILLIAM', 'ZIPFEL, WARREN R', 'DERESKI, MARY O', 'STEGEMAN, JOHN J', 'NAEVE, CLAYTON W.', 'ZIPFEL, WARREN R', 'HILLMAN, ELIZABETH M', 'DERESKI, MARY O', 'QUINN, DANIEL M.', 'RANDOLPH, THEODORE W']\n"
     ]
    }
   ],
   "source": [
    "key_words = []\n",
    "project_abstract = []\n",
    "project_title = []\n",
    "project_number = []\n",
    "project_start_date = []\n",
    "project_end_date = []\n",
    "department = []\n",
    "agency = []\n",
    "PI = []\n",
    "latitude = []\n",
    "longitude = []\n",
    "org_name = []\n",
    "org_city = []\n",
    "org_state = []\n",
    "org_zip = []\n",
    "org_country = []\n",
    "budget_start_date = []\n",
    "budget_end_date = []\n",
    "CFDA_code = []\n",
    "fiscal_year = []\n",
    "total_cost = []\n",
    "i = 0\n",
    "# range needs to be updated.. I just used 2 for testing purposes so the code would execute faster\n",
    "for i in range(2):\n",
    "    response = requests.get(url).json()\n",
    "    for item in response['items']:\n",
    "        key_words.append(item['terms'])\n",
    "        project_abstract.append(item['abstract'])\n",
    "        project_title.append(item['title'])\n",
    "        project_number.append(item['projectNumber'])\n",
    "        project_start_date.append(item['projectStartDate'])\n",
    "        project_end_date.append(item['projectEndDate'])\n",
    "        department.append(item['department'])                        \n",
    "        agency.append(item['agency'])                \n",
    "        PI.append(item['contactPi'])\n",
    "        latitude.append(item['latitude'])\n",
    "        longitude.append(item['longitude'])\n",
    "        org_name.append(item['orgName'])                 \n",
    "        org_city.append(item['orgCity'])\n",
    "        org_state.append(item['orgState'])                \n",
    "        org_zip.append(item['orgZipCode'])\n",
    "        org_country.append(item['orgCountry'])               \n",
    "        budget_start_date.append(item['projectStartDate'])               \n",
    "        budget_end_date.append(item['projectEndDate'])\n",
    "        CFDA_code.append(item['cfdaCode'])                      \n",
    "        fiscal_year.append(item['fy'])                       \n",
    "        total_cost.append(item['totalCostAmount']) \n",
    "i += 1\n",
    "time.sleep(0.5)\n",
    "        \n",
    "print(key_words)\n",
    "print(PI)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style>\n",
       "    .dataframe thead tr:only-child th {\n",
       "        text-align: right;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: left;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Agency</th>\n",
       "      <th>Budget End Date</th>\n",
       "      <th>Budget Start Date</th>\n",
       "      <th>CFDA code</th>\n",
       "      <th>Department</th>\n",
       "      <th>Fiscal Year</th>\n",
       "      <th>Key Words</th>\n",
       "      <th>Latitude</th>\n",
       "      <th>Longitude</th>\n",
       "      <th>Organization City</th>\n",
       "      <th>...</th>\n",
       "      <th>Organization Name</th>\n",
       "      <th>Organization State</th>\n",
       "      <th>Organization ZIP</th>\n",
       "      <th>Principal Investigator</th>\n",
       "      <th>Project Abstract</th>\n",
       "      <th>Project End Date</th>\n",
       "      <th>Project Number</th>\n",
       "      <th>Project Start Date</th>\n",
       "      <th>Project Title</th>\n",
       "      <th>Total Cost</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>NSF</td>\n",
       "      <td>2015-07-31T00:00:00</td>\n",
       "      <td>2012-08-01T00:00:00</td>\n",
       "      <td>47.076</td>\n",
       "      <td>NSF</td>\n",
       "      <td>2012</td>\n",
       "      <td>Aging; Agriculture; base; Biotechnology; care...</td>\n",
       "      <td>32.419564</td>\n",
       "      <td>-80.688428</td>\n",
       "      <td>BEAUFORT</td>\n",
       "      <td>...</td>\n",
       "      <td>TECHNICAL COLLEGE OF THE LOWCOUNTRY</td>\n",
       "      <td>SC</td>\n",
       "      <td>29902-5441</td>\n",
       "      <td>MIDDLETON, NATAVIA</td>\n",
       "      <td>The purpose of this project is to design and i...</td>\n",
       "      <td>2015-07-31T00:00:00</td>\n",
       "      <td>1204850</td>\n",
       "      <td>2012-08-01T00:00:00</td>\n",
       "      <td>EMERGENT USE OF AGRISCIENCE BIOTECHNOLOGY</td>\n",
       "      <td>199200</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>NIH</td>\n",
       "      <td>2015-06-30T00:00:00</td>\n",
       "      <td>2000-07-01T00:00:00</td>\n",
       "      <td>859</td>\n",
       "      <td>HHS</td>\n",
       "      <td>2012</td>\n",
       "      <td>Biotechnology; Training Programs</td>\n",
       "      <td>38.034386</td>\n",
       "      <td>-78.519716</td>\n",
       "      <td>CHARLOTTESVILLE</td>\n",
       "      <td>...</td>\n",
       "      <td>UNIVERSITY OF VIRGINIA CHARLOTTESVILLE</td>\n",
       "      <td>VA</td>\n",
       "      <td>229044195</td>\n",
       "      <td>LAURIE, GORDON WILLIAM</td>\n",
       "      <td>DESCRIPTION (provided by applicant): We seek f...</td>\n",
       "      <td>2015-06-30T00:00:00</td>\n",
       "      <td>5T32GM008715-13</td>\n",
       "      <td>2000-07-01T00:00:00</td>\n",
       "      <td>BIOTECHNOLOGY TRAINING PROGRAM</td>\n",
       "      <td>314148</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>NIH</td>\n",
       "      <td>2016-06-30T00:00:00</td>\n",
       "      <td>2001-07-01T00:00:00</td>\n",
       "      <td>859</td>\n",
       "      <td>HHS</td>\n",
       "      <td>2012</td>\n",
       "      <td>Biotechnology; Leadership; Pharmacologic Subs...</td>\n",
       "      <td>40.027599</td>\n",
       "      <td>-105.251595</td>\n",
       "      <td>BOULDER</td>\n",
       "      <td>...</td>\n",
       "      <td>UNIVERSITY OF COLORADO AT BOULDER</td>\n",
       "      <td>CO</td>\n",
       "      <td>803031058</td>\n",
       "      <td>RANDOLPH, THEODORE W</td>\n",
       "      <td>DESCRIPTION (provided by applicant): In the Le...</td>\n",
       "      <td>2016-06-30T00:00:00</td>\n",
       "      <td>5T32GM008732-12</td>\n",
       "      <td>2001-07-01T00:00:00</td>\n",
       "      <td>LEADERSHIP TRAINING IN PHARMACEUTICAL BIOTECHN...</td>\n",
       "      <td>216969</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>NIH</td>\n",
       "      <td>2015-06-30T00:00:00</td>\n",
       "      <td>2000-07-01T00:00:00</td>\n",
       "      <td>859</td>\n",
       "      <td>HHS</td>\n",
       "      <td>2012</td>\n",
       "      <td>Biotechnology; Molecular; Training</td>\n",
       "      <td>35.832653</td>\n",
       "      <td>-78.645194</td>\n",
       "      <td>RALEIGH</td>\n",
       "      <td>...</td>\n",
       "      <td>NORTH CAROLINA STATE UNIVERSITY RALEIGH</td>\n",
       "      <td>NC</td>\n",
       "      <td>276950001</td>\n",
       "      <td>KELLY, ROBERT M</td>\n",
       "      <td>DESCRIPTION (provided by applicant): Training ...</td>\n",
       "      <td>2015-06-30T00:00:00</td>\n",
       "      <td>5T32GM008776-13</td>\n",
       "      <td>2000-07-01T00:00:00</td>\n",
       "      <td>GRADUATE TRAINING IN MOLECULAR BIOTECHNOLOGY A...</td>\n",
       "      <td>228855</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>NIH</td>\n",
       "      <td>2014-06-30T00:00:00</td>\n",
       "      <td>2003-07-01T00:00:00</td>\n",
       "      <td>859</td>\n",
       "      <td>HHS</td>\n",
       "      <td>2012</td>\n",
       "      <td>Biotechnology; Marines; Training Programs</td>\n",
       "      <td>37.240049</td>\n",
       "      <td>-119.503236</td>\n",
       "      <td>LA JOLLA</td>\n",
       "      <td>...</td>\n",
       "      <td>UNIVERSITY OF CALIFORNIA SAN DIEGO</td>\n",
       "      <td>CA</td>\n",
       "      <td>920930934</td>\n",
       "      <td>GERWICK, WILLIAM HENRY</td>\n",
       "      <td>DESCRIPTION (provided by applicant): This revi...</td>\n",
       "      <td>2014-06-30T00:00:00</td>\n",
       "      <td>5T32GM067550-08</td>\n",
       "      <td>2003-07-01T00:00:00</td>\n",
       "      <td>TRAINING PROGRAM IN MARINE BIOTECHNOLOGY</td>\n",
       "      <td>145722</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>5 rows  21 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "  Agency      Budget End Date    Budget Start Date CFDA code Department  \\\n",
       "0    NSF  2015-07-31T00:00:00  2012-08-01T00:00:00    47.076        NSF   \n",
       "1    NIH  2015-06-30T00:00:00  2000-07-01T00:00:00       859        HHS   \n",
       "2    NIH  2016-06-30T00:00:00  2001-07-01T00:00:00       859        HHS   \n",
       "3    NIH  2015-06-30T00:00:00  2000-07-01T00:00:00       859        HHS   \n",
       "4    NIH  2014-06-30T00:00:00  2003-07-01T00:00:00       859        HHS   \n",
       "\n",
       "   Fiscal Year                                          Key Words   Latitude  \\\n",
       "0         2012   Aging; Agriculture; base; Biotechnology; care...  32.419564   \n",
       "1         2012                   Biotechnology; Training Programs  38.034386   \n",
       "2         2012   Biotechnology; Leadership; Pharmacologic Subs...  40.027599   \n",
       "3         2012                 Biotechnology; Molecular; Training  35.832653   \n",
       "4         2012          Biotechnology; Marines; Training Programs  37.240049   \n",
       "\n",
       "    Longitude Organization City    ...      \\\n",
       "0  -80.688428          BEAUFORT    ...       \n",
       "1  -78.519716   CHARLOTTESVILLE    ...       \n",
       "2 -105.251595           BOULDER    ...       \n",
       "3  -78.645194           RALEIGH    ...       \n",
       "4 -119.503236          LA JOLLA    ...       \n",
       "\n",
       "                         Organization Name Organization State  \\\n",
       "0      TECHNICAL COLLEGE OF THE LOWCOUNTRY                 SC   \n",
       "1   UNIVERSITY OF VIRGINIA CHARLOTTESVILLE                 VA   \n",
       "2        UNIVERSITY OF COLORADO AT BOULDER                 CO   \n",
       "3  NORTH CAROLINA STATE UNIVERSITY RALEIGH                 NC   \n",
       "4       UNIVERSITY OF CALIFORNIA SAN DIEGO                 CA   \n",
       "\n",
       "  Organization ZIP  Principal Investigator  \\\n",
       "0       29902-5441      MIDDLETON, NATAVIA   \n",
       "1        229044195  LAURIE, GORDON WILLIAM   \n",
       "2        803031058    RANDOLPH, THEODORE W   \n",
       "3        276950001         KELLY, ROBERT M   \n",
       "4        920930934  GERWICK, WILLIAM HENRY   \n",
       "\n",
       "                                    Project Abstract     Project End Date  \\\n",
       "0  The purpose of this project is to design and i...  2015-07-31T00:00:00   \n",
       "1  DESCRIPTION (provided by applicant): We seek f...  2015-06-30T00:00:00   \n",
       "2  DESCRIPTION (provided by applicant): In the Le...  2016-06-30T00:00:00   \n",
       "3  DESCRIPTION (provided by applicant): Training ...  2015-06-30T00:00:00   \n",
       "4  DESCRIPTION (provided by applicant): This revi...  2014-06-30T00:00:00   \n",
       "\n",
       "    Project Number   Project Start Date  \\\n",
       "0          1204850  2012-08-01T00:00:00   \n",
       "1  5T32GM008715-13  2000-07-01T00:00:00   \n",
       "2  5T32GM008732-12  2001-07-01T00:00:00   \n",
       "3  5T32GM008776-13  2000-07-01T00:00:00   \n",
       "4  5T32GM067550-08  2003-07-01T00:00:00   \n",
       "\n",
       "                                       Project Title Total Cost  \n",
       "0          EMERGENT USE OF AGRISCIENCE BIOTECHNOLOGY     199200  \n",
       "1                     BIOTECHNOLOGY TRAINING PROGRAM     314148  \n",
       "2  LEADERSHIP TRAINING IN PHARMACEUTICAL BIOTECHN...     216969  \n",
       "3  GRADUATE TRAINING IN MOLECULAR BIOTECHNOLOGY A...     228855  \n",
       "4           TRAINING PROGRAM IN MARINE BIOTECHNOLOGY     145722  \n",
       "\n",
       "[5 rows x 21 columns]"
      ]
     },
     "execution_count": 9,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# create a dictionary and store data in pandas dataframe\n",
    "NIH_dict = {\n",
    "    \"Key Words\": key_words,\n",
    "    \"Project Abstract\": project_abstract,\n",
    "    \"Project Title\": project_title,\n",
    "    \"Project Number\": project_number,\n",
    "    \"Project Start Date\": project_start_date,\n",
    "    \"Project End Date\": project_end_date,\n",
    "    \"Department\": department,\n",
    "    \"Agency\": agency,\n",
    "    \"Principal Investigator\": PI,\n",
    "    \"Latitude\": latitude,\n",
    "    \"Longitude\":longitude,\n",
    "    \"Organization Name\": org_name,\n",
    "    \"Organization City\": org_city,\n",
    "    \"Organization State\": org_state,\n",
    "    \"Organization ZIP\": org_zip,\n",
    "    \"Organization Country\": org_country,\n",
    "    \"Budget Start Date\": budget_start_date,\n",
    "    \"Budget End Date\": budget_end_date,\n",
    "    \"CFDA code\": CFDA_code,\n",
    "    \"Fiscal Year\": fiscal_year,\n",
    "    \"Total Cost\": total_cost\n",
    "}\n",
    "NIH_data = pd.DataFrame(NIH_dict)\n",
    "NIH_data.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [],
   "source": [
    "# save DF to a csv\n",
    "NIH_data.to_csv(\"NIH_data.csv\", encoding=\"utf-8\", index=False)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.6.3"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
